
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/base_style-CPPDhfna.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-BepKMsUl.js"></script>

  <link rel="stylesheet" href="/static/assets/article_style-DUfI1jZp.css" />
<link rel="stylesheet" href="/static/assets/style-BOrsu84o.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-D3wJHomh.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Biosynthesis of Gold Nanostructures and Their Virucidal Activity Against Influenza A Virus - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="18DE24BD930AFC630024BD00451F7E76.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ijms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11899802/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="International Journal of Molecular Sciences">
<meta name="citation_title" content="Biosynthesis of Gold Nanostructures and Their Virucidal Activity Against Influenza A Virus">
<meta name="citation_author" content="Fernanda Contreras">
<meta name="citation_author_institution" content="Laboratorio de Microbiología Molecular, Facultad de Química y Biología, Universidad de Santiago de Chile, Santiago 9170022, Chile; fernanda.contreras.t@usach.cl (F.C.); katherine.rivero@usach.cl (K.R.); nicole.neira@usach.cl (N.N.)">
<meta name="citation_author" content="Katherine Rivero">
<meta name="citation_author_institution" content="Laboratorio de Microbiología Molecular, Facultad de Química y Biología, Universidad de Santiago de Chile, Santiago 9170022, Chile; fernanda.contreras.t@usach.cl (F.C.); katherine.rivero@usach.cl (K.R.); nicole.neira@usach.cl (N.N.)">
<meta name="citation_author" content="Jaime Andrés Rivas-Pardo">
<meta name="citation_author_institution" content="Laboratorio de Genómica Microbiana, Centro de Genómica y Bioinformática, Facultad de Ciencias, Ingeniería y Tecnología, Universidad Mayor, Santiago 8580745, Chile; jaime.rivas@umayor.cl">
<meta name="citation_author" content="Fabiana Liendo">
<meta name="citation_author_institution" content="Laboratorio de Electroanálisis, Facultad de Química y Biología, Universidad de Santiago de Chile, Santiago 9170022, Chile; fabiana.liendo@usach.cl (F.L.); rodrigo.segura@usach.cl (R.S.)">
<meta name="citation_author" content="Rodrigo Segura">
<meta name="citation_author_institution" content="Laboratorio de Electroanálisis, Facultad de Química y Biología, Universidad de Santiago de Chile, Santiago 9170022, Chile; fabiana.liendo@usach.cl (F.L.); rodrigo.segura@usach.cl (R.S.)">
<meta name="citation_author" content="Nicole Neira">
<meta name="citation_author_institution" content="Laboratorio de Microbiología Molecular, Facultad de Química y Biología, Universidad de Santiago de Chile, Santiago 9170022, Chile; fernanda.contreras.t@usach.cl (F.C.); katherine.rivero@usach.cl (K.R.); nicole.neira@usach.cl (N.N.)">
<meta name="citation_author" content="Mauricio Arenas-Salinas">
<meta name="citation_author_institution" content="Centro de Bioinformática, Simulación y Modelado (CBSM), Facultad de Ingeniería, Universidad de Talca, Talca 3460000, Chile; marenas@utalca.cl">
<meta name="citation_author" content="Marcelo Cortez-San Martín">
<meta name="citation_author_institution" content="Laboratorio de Virología Molecular y Control de Patógenos, Facultad de Química y Biología, Universidad de Santiago de Chile, Santiago 9170022, Chile">
<meta name="citation_author" content="Felipe Arenas">
<meta name="citation_author_institution" content="Laboratorio de Microbiología Molecular, Facultad de Química y Biología, Universidad de Santiago de Chile, Santiago 9170022, Chile; fernanda.contreras.t@usach.cl (F.C.); katherine.rivero@usach.cl (K.R.); nicole.neira@usach.cl (N.N.)">
<meta name="citation_publication_date" content="2025 Feb 24">
<meta name="citation_volume" content="26">
<meta name="citation_issue" content="5">
<meta name="citation_firstpage" content="1934">
<meta name="citation_doi" content="10.3390/ijms26051934">
<meta name="citation_pmid" content="40076560">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11899802/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11899802/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11899802/pdf/ijms-26-01934.pdf">
<meta name="description" content="Bacteria in natural environments often encounter high concentrations of metal ions, leading to the development of defense mechanisms such as chemical reduction. This process can result in the formation of nanostructures (NS) ranging from 1–100 nm, ...">
<meta name="og:title" content="Biosynthesis of Gold Nanostructures and Their Virucidal Activity Against Influenza A Virus">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Bacteria in natural environments often encounter high concentrations of metal ions, leading to the development of defense mechanisms such as chemical reduction. This process can result in the formation of nanostructures (NS) ranging from 1–100 nm, ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11899802/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://pmc.ncbi.nlm.nih.gov/search/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="11899802">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.3390/ijms26051934"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/ijms-26-01934.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11899802%2F%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/11899802/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/11899802/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11899802/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ijms.png" alt="International Journal of Molecular Sciences logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to International Journal of Molecular Sciences" title="Link to International Journal of Molecular Sciences" shape="default" href="http://www.mdpi.com/journal/ijms/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Int J Mol Sci</button></div>. 2025 Feb 24;26(5):1934. doi: <a href="https://doi.org/10.3390/ijms26051934" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.3390/ijms26051934</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://pmc.ncbi.nlm.nih.gov/search/?term=%22Int%20J%20Mol%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Int%20J%20Mol%20Sci%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Int%20J%20Mol%20Sci%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Int%20J%20Mol%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Biosynthesis of Gold Nanostructures and Their Virucidal Activity Against Influenza A Virus</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Contreras%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Fernanda Contreras</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Fernanda Contreras</span></h3>
<div class="p">
<sup>1</sup>Laboratorio de Microbiología Molecular, Facultad de Química y Biología, Universidad de Santiago de Chile, Santiago 9170022, Chile; fernanda.contreras.t@usach.cl (F.C.); katherine.rivero@usach.cl (K.R.); nicole.neira@usach.cl (N.N.)</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Contreras%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Fernanda Contreras</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rivero%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Katherine Rivero</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Katherine Rivero</span></h3>
<div class="p">
<sup>1</sup>Laboratorio de Microbiología Molecular, Facultad de Química y Biología, Universidad de Santiago de Chile, Santiago 9170022, Chile; fernanda.contreras.t@usach.cl (F.C.); katherine.rivero@usach.cl (K.R.); nicole.neira@usach.cl (N.N.)</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rivero%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Katherine Rivero</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rivas-Pardo%20JA%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Jaime Andrés Rivas-Pardo</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Jaime Andrés Rivas-Pardo</span></h3>
<div class="p">
<sup>2</sup>Laboratorio de Genómica Microbiana, Centro de Genómica y Bioinformática, Facultad de Ciencias, Ingeniería y Tecnología, Universidad Mayor, Santiago 8580745, Chile; jaime.rivas@umayor.cl</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rivas-Pardo%20JA%22%5BAuthor%5D" class="usa-link"><span class="name western">Jaime Andrés Rivas-Pardo</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liendo%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Fabiana Liendo</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Fabiana Liendo</span></h3>
<div class="p">
<sup>3</sup>Laboratorio de Electroanálisis, Facultad de Química y Biología, Universidad de Santiago de Chile, Santiago 9170022, Chile; fabiana.liendo@usach.cl (F.L.); rodrigo.segura@usach.cl (R.S.)</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liendo%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Fabiana Liendo</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Segura%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Rodrigo Segura</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Rodrigo Segura</span></h3>
<div class="p">
<sup>3</sup>Laboratorio de Electroanálisis, Facultad de Química y Biología, Universidad de Santiago de Chile, Santiago 9170022, Chile; fabiana.liendo@usach.cl (F.L.); rodrigo.segura@usach.cl (R.S.)</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Segura%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rodrigo Segura</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Neira%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Nicole Neira</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Nicole Neira</span></h3>
<div class="p">
<sup>1</sup>Laboratorio de Microbiología Molecular, Facultad de Química y Biología, Universidad de Santiago de Chile, Santiago 9170022, Chile; fernanda.contreras.t@usach.cl (F.C.); katherine.rivero@usach.cl (K.R.); nicole.neira@usach.cl (N.N.)</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Neira%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nicole Neira</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Arenas-Salinas%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Mauricio Arenas-Salinas</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Mauricio Arenas-Salinas</span></h3>
<div class="p">
<sup>4</sup>Centro de Bioinformática, Simulación y Modelado (CBSM), Facultad de Ingeniería, Universidad de Talca, Talca 3460000, Chile; marenas@utalca.cl</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Arenas-Salinas%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mauricio Arenas-Salinas</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cortez-San%20Mart%C3%ADn%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Marcelo Cortez-San Martín</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Marcelo Cortez-San Martín</span></h3>
<div class="p">
<sup>5</sup>Laboratorio de Virología Molecular y Control de Patógenos, Facultad de Química y Biología, Universidad de Santiago de Chile, Santiago 9170022, Chile</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cortez-San%20Mart%C3%ADn%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marcelo Cortez-San Martín</span></a>
</div>
</div>
<sup>5,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Arenas%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Felipe Arenas</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Felipe Arenas</span></h3>
<div class="p">
<sup>1</sup>Laboratorio de Microbiología Molecular, Facultad de Química y Biología, Universidad de Santiago de Chile, Santiago 9170022, Chile; fernanda.contreras.t@usach.cl (F.C.); katherine.rivero@usach.cl (K.R.); nicole.neira@usach.cl (N.N.)</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Arenas%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Felipe Arenas</span></a>
</div>
</div>
<sup>1,</sup><sup>*</sup>
</div>
<div class="cg p">Editor: <span class="name western">Kleoniki Giannousi</span>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="af1-ijms-26-01934">
<sup>1</sup>Laboratorio de Microbiología Molecular, Facultad de Química y Biología, Universidad de Santiago de Chile, Santiago 9170022, Chile; fernanda.contreras.t@usach.cl (F.C.); katherine.rivero@usach.cl (K.R.); nicole.neira@usach.cl (N.N.)</div>
<div id="af2-ijms-26-01934">
<sup>2</sup>Laboratorio de Genómica Microbiana, Centro de Genómica y Bioinformática, Facultad de Ciencias, Ingeniería y Tecnología, Universidad Mayor, Santiago 8580745, Chile; jaime.rivas@umayor.cl</div>
<div id="af3-ijms-26-01934">
<sup>3</sup>Laboratorio de Electroanálisis, Facultad de Química y Biología, Universidad de Santiago de Chile, Santiago 9170022, Chile; fabiana.liendo@usach.cl (F.L.); rodrigo.segura@usach.cl (R.S.)</div>
<div id="af4-ijms-26-01934">
<sup>4</sup>Centro de Bioinformática, Simulación y Modelado (CBSM), Facultad de Ingeniería, Universidad de Talca, Talca 3460000, Chile; marenas@utalca.cl</div>
<div id="af5-ijms-26-01934">
<sup>5</sup>Laboratorio de Virología Molecular y Control de Patógenos, Facultad de Química y Biología, Universidad de Santiago de Chile, Santiago 9170022, Chile</div>
<div class="author-notes p"><div class="fn" id="c1-ijms-26-01934">
<sup>*</sup><p class="display-inline">Correspondence: <span>marcelo.cortez@usach.cl</span> (M.C.-S.M.); <span>felipe.arenass@usach.cl</span> (F.A.)</p>
</div></div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Kleoniki Giannousi</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jan 18; Revised 2025 Feb 19; Accepted 2025 Feb 20; Collection date 2025 Mar.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 by the authors.</div>
<p>Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (<a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://creativecommons.org/licenses/by/4.0/</a>).</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC11899802  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40076560/" class="usa-link">40076560</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Bacteria in natural environments often encounter high concentrations of metal ions, leading to the development of defense mechanisms such as chemical reduction. This process can result in the formation of nanostructures (NS) ranging from 1–100 nm, which have valuable properties for various applications, including as virucidal agents. Currently, metallic NS with virucidal activity are used in disinfectants and surface protection products. However, their production mainly relies on physical and chemical methods, which are often complex, toxic, and energy-intensive. A sustainable alternative is the biosynthesis of nanostructures. Our research focuses on the biosynthesis of gold nanostructures (AuNS) using environmental bacteria and their proteins, with the aim of exploring their potential as agents to destroy the influenza A virus. We screened bacteria under conditions with HAuCl<sub>4</sub>, identifying eight microorganisms capable of growing in high gold concentrations. <em>Staphylococcus haemolyticus</em> BNF01 showed the highest resistance and Au(III) reduction, growing up to 0.25 mM in HAuCl<sub>4</sub>. Bioinformatic analysis revealed five proteins with potential Au(III)-reductase activity, which were cloned and expressed in <em>Escherichia coli</em>. These proteins reduced gold to form AuNPs, which were purified, characterized for size, shape, and surface charge, and tested against influenza A, showing significant virucidal effects, likely due to interactions with viral proteins.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> metal reduction, biosynthesis, gold nanostructures, virucidal, influenza A</p></section></section><section id="sec1-ijms-26-01934"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>The process of industrialization has caused significant ecological and environmental challenges, mainly due to the contamination of soils, sediments, and water bodies with metals. These pollutants are a major concern because metals accumulate in nature and do not degrade, leading to long-lasting effects on ecosystems and human health [<a href="#B1-ijms-26-01934" class="usa-link" aria-describedby="B1-ijms-26-01934">1</a>,<a href="#B2-ijms-26-01934" class="usa-link" aria-describedby="B2-ijms-26-01934">2</a>]. Microorganisms, as key components of terrestrial ecosystems, play a crucial role in nutrient cycles and energy transfer. However, they are highly sensitive to environmental changes, including metal pollution, which can disrupt their metabolic processes and community structures [<a href="#B3-ijms-26-01934" class="usa-link" aria-describedby="B3-ijms-26-01934">3</a>].</p>
<p>Metals vary in their biological role and toxicity. Some are essential for metabolic processes, while others are highly toxic even at low concentrations. Based on their toxicity, metals can be categorized as (i) essential and relatively non-toxic (e.g., calcium, magnesium); (ii) essential but toxic at high levels (e.g., iron, manganese, zinc); and (iii) highly toxic with no known biological function (e.g., mercury, cadmium, silver, and gold) [<a href="#B4-ijms-26-01934" class="usa-link" aria-describedby="B4-ijms-26-01934">4</a>]. The toxic effects of metals on microorganisms have been extensively studied, with mechanisms including the generation of reactive oxygen species (ROS), enzyme inhibition, and DNA damage. These toxic interactions disrupt redox balance, primarily through excessive ROS production, which leads to oxidative stress and cellular damage [<a href="#B5-ijms-26-01934" class="usa-link" aria-describedby="B5-ijms-26-01934">5</a>,<a href="#B6-ijms-26-01934" class="usa-link" aria-describedby="B6-ijms-26-01934">6</a>].</p>
<p>To survive in metal-contaminated environments, bacteria have evolved diverse resistance mechanisms, including metal efflux systems, extracellular sequestration, enzymatic detoxification, and biotransformation. One key detoxification strategy is the ability of certain bacteria to reduce metal ions into their elemental form, a process known as bioreduction, which can lead to the formation of metal nanostructures (NS) [<a href="#B7-ijms-26-01934" class="usa-link" aria-describedby="B7-ijms-26-01934">7</a>,<a href="#B8-ijms-26-01934" class="usa-link" aria-describedby="B8-ijms-26-01934">8</a>,<a href="#B9-ijms-26-01934" class="usa-link" aria-describedby="B9-ijms-26-01934">9</a>]. This microbial-driven metal transformation has gained significant attention as an eco-friendly alternative to traditional nanomaterial synthesis, which often relies on hazardous chemicals and energy-intensive processes.</p>
<p>Biogenic nanostructures, produced via bacterial enzymatic activity, offer multiple advantages over chemically synthesized counterparts. They are biocompatible, environmentally friendly, and free of toxic stabilizing agents, making them ideal for biomedical applications [<a href="#B10-ijms-26-01934" class="usa-link" aria-describedby="B10-ijms-26-01934">10</a>,<a href="#B11-ijms-26-01934" class="usa-link" aria-describedby="B11-ijms-26-01934">11</a>]. Among these, gold nanostructures (AuNS) have attracted interest due to their unique physicochemical properties, including their stability, tunable surface chemistry, and optical characteristics [<a href="#B12-ijms-26-01934" class="usa-link" aria-describedby="B12-ijms-26-01934">12</a>]. Studies have demonstrated the potential of AuNS as antimicrobial agents, capable of disrupting bacterial membranes and interfering with microbial metabolism. Additionally, AuNS have shown antiviral activity, particularly against enveloped viruses, including influenza A virus, by inhibiting viral replication and surface protein interactions [<a href="#B13-ijms-26-01934" class="usa-link" aria-describedby="B13-ijms-26-01934">13</a>]. However, the virucidal activity of metal nanoparticles, in general, has been poorly explored, and it remains an interesting area for future research.</p>
<p>Despite its promising applications, the microbial biosynthesis of AuNS remains poorly understood, particularly regarding the specific proteins and enzymatic mechanisms involved in Au(III) reduction. Most studies have focused on identifying bacterial species capable of metal bioreduction, but few have investigated the molecular basis of bacterial gold reduction and nanostructure formation. Understanding these biological processes is critical for optimizing green nanotechnology approaches and expanding the biomedical applications of biosynthesized nanomaterials [<a href="#B14-ijms-26-01934" class="usa-link" aria-describedby="B14-ijms-26-01934">14</a>,<a href="#B15-ijms-26-01934" class="usa-link" aria-describedby="B15-ijms-26-01934">15</a>].</p>
<p>This study aims to identify bacterial isolates with high gold resistance, characterize their ability to reduce Au(III), and investigate key reductase proteins involved in AuNS biosynthesis. By elucidating the microbial mechanisms responsible for gold nanostructure formation, this research seeks to provide a sustainable alternative for nanomaterial production with potential applications in biomedicine, particularly in the development of antiviral agents against influenza A virus.</p></section><section id="sec2-ijms-26-01934"><h2 class="pmc_sec_title">2. Results</h2>
<section id="sec2dot1-ijms-26-01934"><h3 class="pmc_sec_title">2.1. Evaluation of Resistance Levels and Reducing Capacity of Environmental Isolates Toward Au(III)</h3>
<p>We evaluated the resistance of 18 environmental bacterial isolates to gold (HAuCl<sub>4</sub>) using minimum inhibitory concentration (MIC), minimum bactericidal concentration (MBC), and bacterial growth curves as proxy for the determination. To better understand the gold resistance potential among environmental bacterial isolates, we focused on six isolates that exhibited the highest Au(III) tolerance and reduction capacity, selecting them for further analysis. These isolates, originally from different sources in extreme environments such as Antarctica and Salars, exhibited quite different optimal growing temperatures ranging from 15 to 37 °C.</p>
<p>After completing the temperature characterization and optimization, we proceeded to explore the isolates’ ability to grow in the presence of gold. A comparative analysis of MIC and MBC values allowed the identification of the most resistant isolates, providing a basis for selecting the strains with the greatest potential for gold nanostructure biosynthesis. <a href="#ijms-26-01934-t001" class="usa-link">Table 1</a> summarizes the minimum inhibitory concentration (MIC) and minimum bactericidal concentration (MBC) values for gold for the environmental bacterial isolates. The highest resistance to gold was determined by comparing the MIC observed in each isolate with the control strain, <em>E. coli</em> BW25113. The MIC values for gold ranged from 0.25 mM to less than 0.0078125 mM, with isolates MF16, BNF01, BNF05, M53, and R37A showing the highest resistance. On the other hand, the highest resistance to gold was determined by comparing the MBC observed in each isolate with that of the control strain, <em>E. coli</em> BW25113. The MBC for gold ranged from 0.125 mM to 0.5 mM, with environmental isolates M53, M53B, and R37A demonstrating the highest resistance to the metal.</p>
<section class="tw xbox font-sm" id="ijms-26-01934-t001"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Minimum inhibitory concentrations (MIC), minimum bactericidal concentrations (MBC), and growth parameters of environmental isolates in the presence of HAuCl<sub>4</sub>.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Isolate</th>
<th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">MIC (mM)</th>
<th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">MBC (mM)</th>
<th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">OD<sub>600nm</sub> max (%)</th>
<th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Growth Rate (%)</th>
<th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Generation Time (%)</th>
</tr></thead>
<tbody>
<tr>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">MF01</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.125</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.125</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">WOV</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">WOV</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">WOV</td>
</tr>
<tr>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">MF02</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.125</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.25</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
</tr>
<tr>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">MF07</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.125</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.25</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
</tr>
<tr>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">MF10</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.125</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.125</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
</tr>
<tr>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">MF15</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.125</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.125</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
</tr>
<tr>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">MF16</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.25</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.25</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">WOV</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">WOV</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">WOV</td>
</tr>
<tr>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">MF17</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.125</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.25</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
</tr>
<tr>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">MF18</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.125</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.125</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
</tr>
<tr>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">MF19</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.125</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.125</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
</tr>
<tr>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">MF20</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.125</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.125</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
</tr>
<tr>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">BNF01</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.25</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.25</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">WOV</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">↓ 44</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">↑ 78.9</td>
</tr>
<tr>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">BNF05</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.25</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.25</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">WOV</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">↓ 28.8</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">↑ 40.9</td>
</tr>
<tr>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">BNF20</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.125</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.25</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
</tr>
<tr>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">BNF22</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.03125</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.25</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
</tr>
<tr>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">M53</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.25</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.5</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">↓ 16.1</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">↓ 32.5</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">WOV</td>
</tr>
<tr>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">M53B</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.0625</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.5</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">↓ 10.6</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">↓ 40.3</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">WOV</td>
</tr>
<tr>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">R37A</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.25</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.5</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">↓ 13.3</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">WOV</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">↓ 13.3</td>
</tr>
<tr>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">BW 25113</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.125</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.25</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ND</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/ijms-26-01934-t001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="fn2"><p>This table presents a comprehensive summary of the resistance of various environmental bacterial isolates to HAuCl<sub>4</sub>. The MIC and MBC values are shown alongside the main variations in bacterial growth parameters, including maximum optical density at 600 nm (OD<sub>600nm</sub>), growth rate, and generation time. The MIC and MBC values were determined based on six independent measurements, while the growth parameters were evaluated in the presence of sublethal concentrations of HAuCl<sub>4</sub>. Isolates that exhibited the highest resistance, as indicated by the MIC and MBC values, and significant variations in growth parameters are highlighted. The arrows ‘↓’ and ‘↑’ indicate decreases or increases in the parameters with respect to the control condition (no treatment with HAuCl<sub>4</sub>), respectively. ‘WOV’ indicates ‘without variation’, meaning no significant change in the measured parameter compared to the control condition. ‘ND’ stands for ‘not determined’, indicating that the data are not available or not applicable.</p></div></div></section><p>Based on these results, the environmental isolates that exhibited the highest resistance to gold compared to the <em>E. coli</em> BW25113 strain were selected for subsequent activities. The isolates MF01, MF16, BNF01, BNF05, BNF20, M53, M53B, and R37A were finally selected for further assays, evaluating growth curves in the presence of sublethal concentrations of gold. The growth parameters analyzed included maximum optical density at 600 nm (OD<sub>600nm</sub>), growth rate, and generation time (<a href="#ijms-26-01934-t001" class="usa-link">Table 1</a>). Bacterial identification of the isolates selected was accomplished by sequencing the 16S rRNA gene using the universal primer 8F (forward) and 1492R (reverse). GeneBank accession numbers for the 16S rRNA nucleotide sequences are <a href="https://www.ncbi.nlm.nih.gov/nuccore/MG461635" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MG461635</a> (MF01), <a href="https://www.ncbi.nlm.nih.gov/nuccore/PV094902" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV094902</a> (MF16), <a href="https://www.ncbi.nlm.nih.gov/nuccore/KF701037" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">KF701037</a> (BNF01), <a href="https://www.ncbi.nlm.nih.gov/nuccore/KF701038" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">KF701038</a> (BNF05), <a href="https://www.ncbi.nlm.nih.gov/nuccore/MF806171" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MF806171</a> (BNF20), <a href="https://www.ncbi.nlm.nih.gov/nuccore/PV101475" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV101475</a> (M53), <a href="https://www.ncbi.nlm.nih.gov/nuccore/PV101473" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV101473</a> (M53B), and <a href="https://www.ncbi.nlm.nih.gov/nuccore/PV096985" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV096985</a> (R37A).</p>
<p>In <a href="#ijms-26-01934-f001" class="usa-link">Figure 1</a>A,B, we show in detail the performance of BNF01 and MF16, environmental isolates with the best MIC results, in the presence of increasing gold concentrations. The environmental isolate MF16—<em>Staphylococcus</em> sp.—exhibited a growth similar to that of the control, although it was assayed at 62.5 µM of HAuCl<sub>4</sub> (equivalent to 1/4 of the MIC) (<a href="#ijms-26-01934-f001" class="usa-link">Figure 1</a>B). Notably, this environmental isolate did not show any variations in the growth parameters evaluated in the presence of gold salt (<a href="#ijms-26-01934-t001" class="usa-link">Table 1</a>). The other six environmental isolates were also assayed, showing less capacity to grow in gold; however, based on the results obtained, we identified a total of six isolates resistant to gold: MF16, BNF01, BNF05, M53, M53B, and R37A. Moreover, some of these environmental isolates demonstrated resistance to more than one metal, such as isolates M53, M53B, and R37A, which showed resistance to both Au (III) and Zn (II). In particular, the isolate MF16 exhibited resistance to multiple metals, suggesting not only its ability to grow at high concentrations of toxic metal but also its resistance to different metals.</p>
<figure class="fig xbox font-sm" id="ijms-26-01934-f001"><h4 class="obj_head">Figure 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11899802_ijms-26-01934-g001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f8df/11899802/dd771f66add9/ijms-26-01934-g001.jpg" loading="lazy" height="795" width="727" alt="Figure 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/ijms-26-01934-f001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Evaluation of the gold resistance performance of environmental isolates. Growth curves of the environmental isolates BNF01 (<strong>A</strong>) and MF16 (<strong>B</strong>) resistant to Au(III) exposed to sublethal concentrations of HAuCl<sub>4</sub> (based on the MIC). The environmental isolates were grown in LB medium in the absence (black circles) or presence of HAuCl<sub>4</sub> (concentration indicated next to each graph). The curves shown represents the environmental isolates most resistant to gold. Each point represents the mean of three independent experiments ± standard deviation. (<strong>C</strong>,<strong>D</strong>) Gold-reducing activity mediated by crude extracts from environmental isolates resistant to Au(III). The measurement of gold-reducing activity with HAuCl<sub>4</sub> was performed in the presence of NADH (<strong>C</strong>) and NADPH (<strong>D</strong>) cofactors across different pH levels (colored bars). “ND” indicates no reducing activity observed under the tested conditions. The bars represent the mean of three independent experiments ± standard deviation.</p></figcaption></figure><p>Later, we assayed the gold-reducing capacity of protein lysates from these six gold-resistant environmental bacterial isolates (MF16, BNF01, BNF05, M53, M53B, and R37A) at various pH from 6.0 to 9.0 in the presence of the cofactors NADH and NADPH (<a href="#ijms-26-01934-f001" class="usa-link">Figure 1</a>C,D). In general, when assessing the gold-reducing activity of the crude extracts, we observed that most extracts exhibited activity across the majority of the pH levels tested in the presence of NADH, with the exception of the R37A extract. In contrast, activity in the presence of NADPH was observed only at specific pH levels (<a href="#ijms-26-01934-f001" class="usa-link">Figure 1</a>D). The highest gold-reducing activity in the presence of NADH was detected at pH 8.0, for the protein lysate from BNF05 and MF16, while for NADPH, the MF16 protein extract at pH 7.4 shows the highest activity (<a href="#ijms-26-01934-f001" class="usa-link">Figure 1</a>C,D).</p>
<p>In summary, we successfully identified a total of eight environmental isolates resistant to gold; however, only six showed outstanding performance at high gold concentrations: MF16, BNF01, BNF05, M53, M53B, and R37A. Additionally, the MF16 isolate displays the highest gold-reducing activity, suggesting that this isolate could be a strong candidate for subsequent in vivo or/and in vitro synthesis of gold nanostructures.</p></section><section id="sec2dot2-ijms-26-01934"><h3 class="pmc_sec_title">2.2. Identification, Amplification, and Cloning of Proteins with Potential Au(III) Reductase Activity in Environmental Isolates</h3>
<p>To investigate the molecular basis of gold reduction in bacterial isolates, a targeted analysis was performed to identify key proteins associated with Au(III) reductase activity, prioritizing those with the highest predicted catalytic efficiency. We conducted a genome search to identify proteins with potential Au (III) reductase activity. We used the genome sequence of BNF01, which was available from previous studies [<a href="#B16-ijms-26-01934" class="usa-link" aria-describedby="B16-ijms-26-01934">16</a>].</p>
<p>After a literature search for proteins with known metal reductase activity, we performed an amino acid sequence analysis, yielding a dataset of 13,500 proteins classified by function and conserved motifs. We narrowed down the dataset to 63 proteins using the PDB, focusing only on proteins with structure available. Based on these results, five proteins with potential Au(III) reductase activity were predicted in the environmental isolate BNF01 (<a href="#ijms-26-01934-t002" class="usa-link">Table 2</a>). Among the proteins identified, Gor exhibited the highest Au(III) reductase activity, making it the primary candidate for further biochemical characterization and gold nanostructure synthesis.</p>
<section class="tw xbox font-sm" id="ijms-26-01934-t002"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>In silico identification of proteins with probable Au(III) reductase activity in the environmental isolate BNF01.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Protein Name</th>
<th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Gene Name</th>
<th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Gene Size (bp)</th>
</tr></thead>
<tbody>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Thioredoxin reductase</td>
<td align="center" valign="middle" rowspan="1" colspan="1">
<em>trxB</em>
</td>
<td align="center" valign="middle" rowspan="1" colspan="1">942</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Glutathione reductase</td>
<td align="center" valign="middle" rowspan="1" colspan="1">
<em>gor</em>
</td>
<td align="center" valign="middle" rowspan="1" colspan="1">1524</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">L-amino acid oxidase</td>
<td align="center" valign="middle" rowspan="1" colspan="1">
<em>oxi</em>
</td>
<td align="center" valign="middle" rowspan="1" colspan="1">724</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Methylenetetrahydrofolate-tRNA-(uracil-5)-methyltransferase (TrmFO)</td>
<td align="center" valign="middle" rowspan="1" colspan="1">
<em>trmFO</em>
</td>
<td align="center" valign="middle" rowspan="1" colspan="1">1308</td>
</tr>
<tr>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Coenzyme A disulfide reductase</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">
<em>cdr</em>
</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1323</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/ijms-26-01934-t002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>Following the in silico identification of proteins with potential Au(III) reductase activity, the genes encoding these proteins were amplified, expressed, and purified. Specific primers were then designed (<a href="#app1-ijms-26-01934" class="usa-link">Table S1</a>) to amplify the genes encoding proteins with probable Au(III) reductase activity from the genome of BNF01. Initially, PCR was performed using GoTaq DNA polymerase to confirm the functionality of the designed primers (<a href="#app1-ijms-26-01934" class="usa-link">Figure S1A</a>). Subsequently, high-fidelity Q5 DNA polymerase was used for PCR amplification of the target genes, and the products were visualized via agarose gel electrophoresis (<a href="#app1-ijms-26-01934" class="usa-link">Figure S1B</a>). PCR products of the expected molecular sizes were obtained for the four genes encoding proteins with potential Au(III) reductase activity from the genomic DNA of BNF01: 1524 bp for <em>cdr</em>, 724 bp for <em>oxi</em>, 1308 bp for <em>trmFO</em>, and 1323 bp for <em>gor</em> (<a href="#app1-ijms-26-01934" class="usa-link">Figure S1A,B</a>).</p>
<p>The PCR products of the expected sizes for each gene, obtained with Q5 DNA polymerase, were purified from the agarose gel for cloning into the pET101/D-TOPO and pET21b(+) vectors. The purified PCR products of the genes encoding proteins with probable Au(III) reductase activity were individually cloned into the pET101/D-TOPO (for the <em>cdr</em> gene) and pET21b(+) vectors. Recombinant vectors for each gene were used to transform into chemically competent <em>E. coli</em> TOP10 cells. After plating the bacteria on LB agar plates supplemented with ampicillin and streptomycin, we obtained transformant colonies containing the vectors pET101/<em>cdr</em>, pET21b/<em>gor</em>, pET21b/<em>oxi</em>, and pET21b/<em>trmFO</em>, which encodes the proteins with potential Au(III) reductase activity.</p>
<p>The introduction and correct orientation of the genes into the vectors were confirmed by colony PCR using either a set of specific primers to amplify the gene or a combination of a forward primer of the gene and a reverse primer only present downstream of the cloning site. The amplification yields the exact sizes of the <em>cdr</em>, <em>oxi</em>, <em>trmFO</em>, and <em>gor</em> genes, and for the orientation assay, 150 pb extra bp were founded to each amplicon, matching the extra segment belonging to the vector. Moreover, we used DNA sequencing to confirm the correct orientation in the plasmid and the absence of substitutions during the cloning process.</p></section><section id="sec2dot3-ijms-26-01934"><h3 class="pmc_sec_title">2.3. Expression, Purification, and Evaluation of Reductase Activity of Recombinant Proteins with Au(III) Reductase Activity</h3>
<p>Recombinant expression and purification of the most promising Au(III) reductase candidates were performed to validate their enzymatic activity under controlled conditions and to optimize their use in gold nanostructure biosynthesis. The recombinant vectors containing the genes—pET101/<em>cdr</em>, pET21b/<em>gor</em>, pET21b/<em>oxi</em>, and pET21b/<em>trmFO</em>—were used to transform chemically competent <em>E. coli</em> BL21 (DE3) cells. After plating the bacteria on LB agar plates supplemented with ampicillin, transformant colonies were obtained for all cases. These transformed colonies were used to conduct induction assays at different temperatures (30 °C and 37 °C) and multiple time points (0, 1, 2, 4, 6, 8, and 24 h), aiming to determine the optimal expression conditions for each protein (<a href="#app1-ijms-26-01934" class="usa-link">Figure S1C,D</a>). In <a href="#ijms-26-01934-f002" class="usa-link">Figure 2</a>, we show the purification of Gor protein, as an example of the recombinant protein expressed for the gold-reducing assays. After exploring the best conditions for the protein expression—assaying temperature and time after induction (<a href="#app1-ijms-26-01934" class="usa-link">Figure S1C,D</a>)—induction tests yield, in the case of Gor protein, a 48 kDa band when the bacteria is induced for 6 h at 30 °C. Similar conditions for induction were found for the other three enzymes, CoADR, TrmFO, and Oxi, which were also successfully purified.</p>
<figure class="fig xbox font-sm" id="ijms-26-01934-f002"><h4 class="obj_head">Figure 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11899802_ijms-26-01934-g002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f8df/11899802/461350b560d7/ijms-26-01934-g002.jpg" loading="lazy" height="462" width="749" alt="Figure 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/ijms-26-01934-f002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Recombinant proteins from environmental isolates display gold-reducing activity. (<strong>A</strong>) Protein purification process for Gor with potential Au(III) reductase activity. Left: Elution profile of Gor protein. Right: SDS-PAGE analysis. Lane 1: Molecular weight marker. Lane 2: Crude extract before column purification. Lane 3: Crude extract after column purification. Lane 4: Column wash. Lanes 5–10: Eluted protein fractions. Lane 11: Purified protein after dialysis. (<strong>B</strong>,<strong>C</strong>) Au(III) reductase activity mediated by purified proteins. The measurement of HAuCl<sub>4</sub> reductase activity was performed with 50 µg of purified protein in the presence of NADH (<strong>B</strong>) and NADPH (<strong>C</strong>) cofactors across different pH levels (colored bars). “ND” indicates no reductase activity observed under the tested conditions. Bars represent the mean of three independent experiments ± standard deviation.</p></figcaption></figure><p>Later, we purified the proteins through affinity chromatography, enriching the protein band belonging to the reducing enzyme. For Gor protein purification, the protein elutes in the first two fractions yielded a band above 48 kDa (<a href="#ijms-26-01934-f002" class="usa-link">Figure 2</a>A, lanes 2–11). Subsequently, we pooled the fractions, evaluating the Au(III) reductase activity at multiple pH and in the presence of NADH or NADPH (<a href="#ijms-26-01934-f002" class="usa-link">Figure 2</a>B,C).</p>
<p>The gold reductase activity assays indicate that all the enzymes, CoADR, TrmFO, Oxi, and Gor, display reducing activity across pH and in the presence of NADH and NADPH cofactors (<a href="#ijms-26-01934-f002" class="usa-link">Figure 2</a>B,C). Nevertheless, Gor shows the highest gold reductase activity, registered at pH 9.0 in the presence of either NADH or NADPH, followed by TrmFO, which shows high reducing activity at pH 7.4 in the presence of NADPH (<a href="#ijms-26-01934-f002" class="usa-link">Figure 2</a>C).</p>
<p>Based on the results obtained, the recombinant Gor protein was selected for the in vitro synthesis of gold nanostructures (AuNS), as it demonstrated the highest levels of reduction under the tested conditions, which could be related to the generation of a higher concentration of nanostructures.</p></section><section id="sec2dot4-ijms-26-01934"><h3 class="pmc_sec_title">2.4. Synthesis, Purification, and Characterization of Gold Nanostructures Generated In Vitro by Reductase Proteins</h3>
<p>Gold nanostructures (AuNS) were synthesized in vitro using the purified recombinant protein from the environmental isolate BNF01. The optimal conditions for the reduction and synthesis of AuNS were determined to be 20 mM Tris-HCl buffer at pH 9.0, with 1 mM NADPH, 1 mM HAuCl<sub>4</sub>, and 50 μg of the purified recombinant Gor protein.</p>
<p>Following synthesis, the AuNS were purified and concentrated by centrifugation using 3 kDa filtration units (Amicon<sup>®</sup>, Darmstadt, Germany). The concentration of the biosynthesized AuNS was quantified using atomic absorption spectroscopy (AAS), yielding a concentration of 1149.9 ± 8.8 μg/mL.</p>
<p>The biosynthesized AuNS were then selected for further characterization using various techniques, as shown in <a href="#ijms-26-01934-f003" class="usa-link">Figure 3</a>. Transmission electron microscopy (TEM) was used to determine the morphology and size of the AuNS. Different magnifications (<a href="#ijms-26-01934-f003" class="usa-link">Figure 3</a>A) revealed that the AuNS exhibited some degree of aggregation and a homogeneous morphology, similar to the spiky or “urchin-like” shape described for this type of nanostructure. The sizes of the nanostructures varied, with a predominance of those larger than 50 nm.</p>
<figure class="fig xbox font-sm" id="ijms-26-01934-f003"><h4 class="obj_head">Figure 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11899802_ijms-26-01934-g003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f8df/11899802/1a6554a2115c/ijms-26-01934-g003.jpg" loading="lazy" height="617" width="749" alt="Figure 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/ijms-26-01934-f003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Characterization of gold nanostructures (AuNS) biosynthesized in vitro by purified recombinant Gor protein. (<strong>A</strong>) TEM micrographs show urchin-like structures resulting from gold reduction. (<strong>B</strong>) SEM–EDX analysis indicates that the primary composition of the biosynthesized AuNS is carbon, oxygen, and gold. (<strong>C</strong>) UV–Vis absorption spectrum shows a maximum absorbance at 622 nm. (<strong>D</strong>) Size distribution analysis by DLS shows two predominant distributions in the biosynthesized gold nanostructures. (<strong>E</strong>) Zeta potential distribution determined for the biosynthesized gold nanostructures was −21.4 mV.</p></figcaption></figure><p>The chemical composition of the AuNS was analyzed using scanning electron microscopy coupled with energy-dispersive X-ray spectroscopy (SEM–EDX) (ZEISS, Oberkochen, Germany; Oxford Instruments, Boston, MA, USA), revealing six peaks corresponding to carbon, oxygen, aluminum, and gold (<a href="#ijms-26-01934-f003" class="usa-link">Figure 3</a>B). The AuNS had a gold content of 23%, with high levels of carbon (38.7%) and oxygen (38.0%), likely related to the biological synthesis of the nanostructures.</p>
<p>The UV–Vis absorbance spectrum of the AuNS showed a maximum absorbance at 622 nm, which is within the recorded range for nanostructures with an urchin-like morphology (555–702 nm) (<a href="#ijms-26-01934-f003" class="usa-link">Figure 3</a>C). The size distribution and surface charge of the biosynthesized nanostructures were determined by dynamic light scattering (DLS) and zeta potential measurements (<a href="#ijms-26-01934-f003" class="usa-link">Figure 3</a>D,E). The DLS analysis revealed two main size distributions of the AuNS, corresponding to 74 ± 17 nm and 6.7 ± 0.8 nm, with a polydispersity index (PDI) of 0.53 (<a href="#ijms-26-01934-f003" class="usa-link">Figure 3</a>D). The zeta potential for the AuNS was −21.4 ± 4.05 mV, which suggests a tendency for the biosynthesized nanostructures to agglomerate (<a href="#ijms-26-01934-f003" class="usa-link">Figure 3</a>E).</p>
<p>In summary, gold nanostructures (AuNS) were successfully synthesized using the recombinant Gor protein from the environmental isolate BNF01. These AuNS, already characterized to understand their physical, chemical, and optical properties, were purified, quantified, and stored for further analysis.</p></section><section id="sec2dot5-ijms-26-01934"><h3 class="pmc_sec_title">2.5. Propagation, Confirmation, and Quantification of Influenza A H1N1 Virus for Virucidal Assays</h3>
<p>The influenza A H1N1 virus (ATCC No. VR-1736) was propagated in Madin–Darby canine kidney (MDCK) epithelial cell cultures to prepare a viral inoculum for subsequent virucidal assays in suspension and on non-porous surfaces. MDCK cell monolayers at 80–90% confluence were infected with a multiplicity of infection (MOI) of 0.1 of the influenza A H1N1 virus. The cytopathic effect (CPE) produced by the virus in cell culture was monitored daily (<a href="#app1-ijms-26-01934" class="usa-link">Figure S2</a>). At 48 h post-infection, spaces began to appear in the MDCK cell monolayer (<a href="#app1-ijms-26-01934" class="usa-link">Figure S2A</a>), and by 72 h post-infection, nearly all cells had detached from the monolayer (<a href="#app1-ijms-26-01934" class="usa-link">Figure S2C</a>). This CPE contrasted with uninfected control cell cultures treated with 1X DPBS, where the cell monolayer remained intact at both 48 and 72 h post-infection (<a href="#app1-ijms-26-01934" class="usa-link">Figure S2B,D</a>). The supernatants from the infected cultures were collected and centrifuged to sediment cell debris, and the resulting supernatant was supplemented with 1% fetal bovine serum (FBS) and stored at −80 °C.</p>
<p>To confirm the presence of the influenza A H1N1 virus in the collected supernatants, we conducted an hemagglutination (HA) assay, based on the ability of influenza viruses to bind to red blood cells (<a href="#app1-ijms-26-01934" class="usa-link">Figure S3</a>). Supernatants from two different infections (P0 and P1) were subjected to the hemagglutination assay, yielding 4 HA units for the P0 supernatant and 64 HA units for the P1 supernatant (<a href="#app1-ijms-26-01934" class="usa-link">Figure S3</a>). In contrast, no hemagglutination was observed in the negative control (PBS) (<a href="#app1-ijms-26-01934" class="usa-link">Figure S3</a>). These results confirmed that the post-infection supernatants contained influenza A virus with hemagglutinating activity.</p>
<p>The viral titers in the collected supernatants were then determined using two methods: the Reed–Muench method (TCID<sub>50</sub>/mL) and absolute RT-qPCR. To implement the RT-qPCR method, a recombinant plasmid containing a conserved segment of the influenza A M gene was generated to enable absolute quantification of the PCR product. Specific primers were used to amplify the conserved segment of the influenza A M gene from a cDNA synthesized by reverse transcription (RT) from RNA extracted from virus-containing supernatants. The amplification of the conserved segment of the influenza A M gene resulted in a PCR product of approximately 100 bp, consistent with the expected size of 101 bp (<a href="#app1-ijms-26-01934" class="usa-link">Figure S4A</a>). The PCR product was purified for subsequent cloning into the pGEM-T Easy vector.</p>
<p>The purified PCR product corresponding to the conserved segment of the influenza A M gene was cloned into the pGEM-T Easy vector. The resulting recombinant vector was used to transform chemically competent <em>E. coli</em> TOP10 cells. After plating the bacteria on LB agar plates supplemented with ampicillin and streptomycin, transformant colonies were obtained. These colonies were analyzed to confirm the correct insertion of the gene into the vector through colony PCR, restriction enzyme digestion, and sequencing (<a href="#app1-ijms-26-01934" class="usa-link">Figure S4B</a>). The colony PCR products confirmed the presence of the conserved M gene segment, showing a band of approximately 100 bp, consistent with the expected insert size (<a href="#app1-ijms-26-01934" class="usa-link">Figure S4B</a>). The gene’s cloning directionality in the vector was determined, revealing a band of over 200 bp, corresponding to the insert size plus 150 bp of the vector (<a href="#app1-ijms-26-01934" class="usa-link">Figure S4B</a>).</p>
<p>Recombinant vectors were purified from each of the verified colonies and digested with the restriction enzyme <em>Eco</em>RI. The digestion yielded two bands, one of approximately 3000 bp, matching the pGEM-T Easy vector size (3015 bp), and another of approximately 100 bp, corresponding to the insert size released from the vector (<a href="#app1-ijms-26-01934" class="usa-link">Figure S4C</a>). The correct insertion of the Mseg gene into the purified recombinant vectors was confirmed by sequencing, where the obtained sequence showed significant alignments with the influenza A virus matrix protein-coding segment 7 (<a href="#app1-ijms-26-01934" class="usa-link">Figure S4D</a>).</p>
<p>RT-qPCR amplification of 12 serial dilutions of known concentration of the purified recombinant vector pGEM-T Easy/Mseg was performed, yielding Ct values for each (<a href="#app1-ijms-26-01934" class="usa-link">Table S2</a>). From these data, the number of copies/µL of the conserved M gene segment in the recombinant vector was calculated (<a href="#app1-ijms-26-01934" class="usa-link">Table S2</a>).</p>
<p>Using RT-qPCR and the calibration curve obtained, the number of virus copies/mL in each of the collected viral supernatants was determined (<a href="#app1-ijms-26-01934" class="usa-link">Table S2</a>), with values ranging from 1.6 to 8.2 × 10<sup>5</sup> copies/mL. However, these viral titers were insufficient for performing the virucidal assays in suspension and on non-porous surfaces, which require a minimum of 1.0 × 10<sup>6</sup> copies/mL.</p>
<p>To obtain an inoculum with the necessary viral titers for the virucidal assays, the supernatant from influenza-virus-infected cells was concentrated using Amicon<sup>®</sup> filtration units. RNA extracted from the concentrated viral supernatant was analyzed by RT-qPCR, resulting in a titer 100 times higher, equivalent to 1.43 × 10<sup>7</sup> copies/mL in a final volume of 8 mL. This value was similar to that obtained using the Reed–Muench quantification method for TCID<sub>50</sub>/mL, where the concentrated viral supernatant had a viral titer of 5.6 × 10<sup>7</sup> TCID<sub>50</sub>/mL.</p></section><section id="sec2dot6-ijms-26-01934"><h3 class="pmc_sec_title">2.6. Evaluation of the Virucidal Activity and Mechanism of Action of Biosynthesized AuNS Against Influenza A H1N1 Virus</h3>
<p>The virucidal activity of biosynthesized gold nanostructures (AuNS) was evaluated against influenza A H1N1 virus both in suspension and on non-porous stainless-steel surfaces, following standardized contact times of 10 and 60 min as per the European Committee for Standardization [<a href="#B17-ijms-26-01934" class="usa-link" aria-describedby="B17-ijms-26-01934">17</a>,<a href="#B18-ijms-26-01934" class="usa-link" aria-describedby="B18-ijms-26-01934">18</a>]. A viral inoculum with a titer of 1.0 × 10<sup>6</sup> TCID<sub>50</sub>/mL and a nanostructure concentration of 100 μg/mL was used in all tests. Standardized hard water (WSH) containing 300 ppm CaCO<sub>3</sub> was used as a negative virucidal control, Virkon<sup>®</sup>, a peroxide-based disinfectant diluted 1:600, served as the positive control, and commercial CuNS &lt; 10 nm were used as the biocidal nanostructure control.</p>
<p>In suspension assays, a reduction in initial viral titers was observed in the influenza A virus treatments with Virkon<sup>®</sup> and biosynthesized AuNS at both contact times, and with commercial CuNS after 60 min of treatment (<a href="#ijms-26-01934-f004" class="usa-link">Figure 4</a>A). On non-porous stainless-steel surfaces, a reduction in initial viral titers was observed following treatment with Virkon<sup>®</sup> as well as with commercial CuNS after 60 min of treatment (<a href="#ijms-26-01934-f004" class="usa-link">Figure 4</a>B). However, treatment with WSH and biosynthesized AuNS did not result in any changes in the initial viral titers at either contact time (<a href="#ijms-26-01934-f004" class="usa-link">Figure 4</a>A,B).</p>
<figure class="fig xbox font-sm" id="ijms-26-01934-f004"><h4 class="obj_head">Figure 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11899802_ijms-26-01934-g004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f8df/11899802/ef531bfffe65/ijms-26-01934-g004.jpg" loading="lazy" height="462" width="760" alt="Figure 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/ijms-26-01934-f004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Virucidal activity of AuNS. (<strong>A</strong>,<strong>B</strong>) Virucidal assays against influenza A virus in suspension and on non-porous stainless-steel surfaces. Influenza A virus inactivation was evaluated in suspension (<strong>A</strong>) and on non-porous stainless-steel surfaces (<strong>B</strong>) using standardized-hardness water (WSH), Virkon<sup>®</sup> solution (1:600 dilution), and commercial copper nanostructures (CuNS) and biosynthesized gold nanostructures (AuNS) with short (10 min) and long (60 min) contact times. The Reed–Muench quantification method was used to determine the TCID<sub>50</sub>/mL. Results represent three independent replicates ± standard deviation. (<strong>C</strong>) Inhibition of cytopathic effect caused by influenza A H1N1 virus after treatment with different concentrations of AuNS. R<sup>2</sup> = 0.9722. Each point represents the mean of three independent experiments ± standard deviation. (<strong>D</strong>) Cytotoxicity of AuNS in MDCK cell cultures after 48 h of incubation, determined by the MTT assay. Non-linear regression curves (dashed line) were generated with concentrations expressed on a logarithmic scale. R<sup>2</sup> = 0.9724. Each point represents the mean of three independent experiments ± standard deviation. * <em>p</em> &lt; 0.05; ** <em>p</em> &lt; 0.01; *** <em>p</em> &lt; 0.001; ns, not significant.</p></figcaption></figure><p>Biosynthesized AuNS exhibited virucidal activity only in suspension assays, and showed virucidal efficacy at longer contact times, with a 4 log<sub>10</sub> reduction in initial viral titers, comparable to the positive control (Virkon<sup>®</sup>) in both contact times, meeting the threshold for virucidal efficacy in surface assays (≥4 log<sub>10</sub> reduction) (<a href="#ijms-26-01934-t003" class="usa-link">Table 3</a>). In contrast, none of the tested nanostructures, including AuNS, were effective on non-porous surfaces, as no ≥ 3 log<sub>10</sub> reduction in viral titers was observed (<a href="#ijms-26-01934-t003" class="usa-link">Table 3</a>).</p>
<section class="tw xbox font-sm" id="ijms-26-01934-t003"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Initial and final viral titers obtained in virucidal assays in suspension and on non-porous stainless-steel surfaces.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th rowspan="2" align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" colspan="1">Sample</th>
<th colspan="3" align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1">Viral Title: Suspension</th>
<th colspan="3" align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1">Viral Title: Non-Porous Surfaces</th>
</tr>
<tr>
<th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0 min</th>
<th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">10 min</th>
<th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">60 min</th>
<th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0 min</th>
<th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">10 min</th>
<th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">60 min</th>
</tr>
</thead>
<tbody>
<tr>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">
<strong>WSH</strong>
</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1.1 × 10<sup>7</sup>
</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">7.1 × 10<sup>6</sup>
</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">8.5 × 10<sup>6</sup>
</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">7.1 × 10<sup>5</sup>
</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">7.1 × 10<sup>5</sup>
</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1.4 × 10<sup>6</sup>
</td>
</tr>
<tr>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">
<strong>Virkon<sup>®</sup></strong>
</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2.7 × 10<sup>7</sup>
</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">NE</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">NE</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">4.5 × 10<sup>5</sup>
</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">NE</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">NE</td>
</tr>
<tr>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">
<strong>Commercial CuNS</strong>
</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2.3 × 10<sup>7</sup>
</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">3.2 × 10<sup>7</sup>
</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2.2 × 10<sup>5</sup>
</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">8.5 × 10<sup>5</sup>
</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">9.7 × 10<sup>4</sup>
</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">9.4 × 10<sup>3</sup>
</td>
</tr>
<tr>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">
<strong>Biosynthesized AuNS</strong>
</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2.8 × 10<sup>7</sup>
</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">4.3 × 10<sup>4</sup>
</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">4.0 × 10<sup>3</sup>
</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2.4 × 10<sup>5</sup>
</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1.9 x 10<sup>5</sup>
</td>
<td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">4.8 × 10<sup>4</sup>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/ijms-26-01934-t003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="fn3"><p>Viral titers were determined using the Reed–Muench quantification method to obtain TCID<sub>50</sub>/mL after exposing the virus to the indicated compound for short (10 min) and long (60 min) contact times. NE: No cytopathic effect observed in any of the evaluated virus dilutions.</p></div></div></section><p>Following confirmation of the virucidal efficacy of AuNS after 60 min of treatment in suspension assays, various concentrations of these nanostructures (0–800 μg/mL) were used to determine the 50% inhibitory concentration (IC<sub>50</sub>) of the cytopathic effect in virucidal assays and the cytotoxicity of AuNS in MDCK cell cultures (<a href="#ijms-26-01934-f004" class="usa-link">Figure 4</a>C). The results showed that the IC<sub>50</sub> of AuNS was 77 ± 1 μg/mL (<a href="#ijms-26-01934-f004" class="usa-link">Figure 4</a>C). Regarding cytotoxicity, it was observed that a concentration of 106 ± 1 μg/mL of AuNS resulted in 50% survival of the MDCK cell culture (<a href="#ijms-26-01934-f004" class="usa-link">Figure 4</a>D).</p>
<p>To explore whether the virucidal action of AuNS was associated with the degradation of influenza A H1N1 surface proteins, a hemagglutination assay was conducted, with the virus treated for 60 min with nanostructures (<a href="#app1-ijms-26-01934" class="usa-link">Figure S5</a>). The untreated virus (positive control) showed 16 HA units, while the PBS negative control did not exhibit hemagglutination in any dilution. The virus treated with WSH showed 2 HA units at the evaluated contact times, similar to treatments with commercial CuNS. In comparison, biosynthesized AuNS treatment resulted in 1 HA unit at 10 min and no HA units at 60 min. These results suggest that the hemagglutinin (HA) surface protein of the virus might be compromised after treatment with AuNS, as the hemagglutination capacity of influenza virus depends on the presence and integrity of this surface protein. However, these results do not confirm whether the viral particle itself has suffered structural damage, losing its integrity and, therefore, its hemagglutination capability.</p>
<p>To further investigate, influenza A viral particles were visualized by TEM after treatment with AuNS (<a href="#ijms-26-01934-f005" class="usa-link">Figure 5</a>A,B). The electron micrographs of influenza A H1N1 virus treated with WSH for 60 min showed intact viral particles with an elongated spherical morphology, surrounded by a membrane with inserted surface proteins, with a diameter &gt; 100 nm (average 144 ± 18 nm) (<a href="#ijms-26-01934-f005" class="usa-link">Figure 5</a>A). In contrast, electron micrographs of viruses treated with AuNS for 60 min showed virus-like particles lacking surface proteins, with affected size (average 128 ± 21 nm) and irregular morphology, although maintaining their lipid membrane (<a href="#ijms-26-01934-f005" class="usa-link">Figure 5</a>B). Additionally, electron-dense regions were observed, with an average size of 7.6 ± 1.5 nm, which could correspond to the smaller AuNS, potentially adsorbed on the virion surface (<a href="#ijms-26-01934-f005" class="usa-link">Figure 5</a>B).</p>
<figure class="fig xbox font-sm" id="ijms-26-01934-f005"><h4 class="obj_head">Figure 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11899802_ijms-26-01934-g005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f8df/11899802/d0bf9eaef53d/ijms-26-01934-g005.jpg" loading="lazy" height="955" width="720" alt="Figure 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/ijms-26-01934-f005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Ultrastructure characterization of the AuNS and viral degradative activity. (<strong>A</strong>,<strong>B</strong>) Evaluation of influenza A H1N1 virus integrity by transmission electron microscopy (TEM) post-treatment with biosynthesized AuNS. Enriched influenza A H1N1 virus was incubated with standardized-hardness water (WSH) (<strong>A</strong>) or 100 µg/mL biosynthesized AuNS (<strong>B</strong>) for 60 min with constant agitation at 25 °C and subsequently visualized by TEM. (<strong>C</strong>) Degradative effects of biosynthesized AuNS treatment on influenza A H1N1 virus visualized by silver-stained SDS-PAGE. Lane 1: Molecular weight marker (AccuRuler RGB PLUS Prestained Protein Ladder). Lanes 2–4: Triplicate aliquots of influenza A H1N1 virus treated with WSH for 60 min. Lanes 5–7: Triplicate aliquots of influenza A H1N1 virus treated with 100 µg/mL of AuNS for 60 min.</p></figcaption></figure><p>To confirm whether the smaller AuNS could be adsorbed on the surface of influenza A H1N1 virions, we conducted a gold quantification after the treatment of nanostructures using AAS. The virus–AuNS mixture was incubated for 60 min at a gold concentration of 107 μg/mL. When the treated virus samples were centrifuged after 60 min of incubation, the sediment had a gold concentration of 86.7 μg/mL, while the supernatant had 13.6 μg/mL of gold. In contrast, the AuNS control after centrifugation showed a gold concentration of 103.2 μg/mL in the sediment, with no gold detected in the supernatant. No gold was detected in the sediment or supernatant of the untreated virus control after centrifugation. These results suggest that approximately 13.6% of the AuNS interacted with the influenza A virion after treatment, likely corresponding to the smaller nanostructures observed by TEM and DLS (<a href="#ijms-26-01934-f003" class="usa-link">Figure 3</a>A,D).</p>
<p>The effect of AuNS treatment on virion proteins was further evaluated through SDS-PAGE analysis (<a href="#ijms-26-01934-f005" class="usa-link">Figure 5</a>C). Influenza A H1N1 virus treated with WSH for 60 min showed an intense band pattern corresponding to viral proteins (<a href="#ijms-26-01934-f005" class="usa-link">Figure 5</a>C, lanes 2–4). However, after treatment of the enriched influenza virus aliquot with AuNS for 60 min, a decrease in band intensity was observed, possibly associated with the degradation of some viral proteins (<a href="#ijms-26-01934-f005" class="usa-link">Figure 5</a>C, lanes 5–7).</p>
<p>In summary, the biosynthesized AuNS exhibited virucidal action against influenza A virus in suspension assays, likely related to the degradation of viral proteins, although this effect could not be associated with any specific protein in the influenza A virion. Further investigations would be required to elucidate the precise mechanisms underlying this virucidal activity.</p></section></section><section id="sec3-ijms-26-01934"><h2 class="pmc_sec_title">3. Discussion</h2>
<section id="sec3dot1-ijms-26-01934"><h3 class="pmc_sec_title">3.1. Resistance Levels and Reducing Capacity of Environmental Isolates Toward Au(III)</h3>
<p>The minimum inhibitory concentration (MIC), defined as the lowest concentration of a compound that completely inhibits the growth of a microorganism, has been widely used to identify bacterial resistance and susceptibility to specific compounds [<a href="#B19-ijms-26-01934" class="usa-link" aria-describedby="B19-ijms-26-01934">19</a>]. However, there are limited studies concerning the toxic effects of tetrachloroauric acid on bacteria. For instance, we previously reported in Muñoz-Villagrán et al. (2020) [<a href="#B20-ijms-26-01934" class="usa-link" aria-describedby="B20-ijms-26-01934">20</a>] an MIC of 0.25 mM for tetrachloroauric acid in <em>E. coli</em> under aerobic conditions, which is higher than the 0.125 mM MIC observed for the same strain in this study. The highest MIC observed in this work for tetrachloroauric acid was 0.25 mM, with isolates MF16, BNF01, BNF05, M53, and R37A showing the highest resistance.</p>
<p>The minimum bactericidal concentration (MBC), defined as the minimum concentration of a compound that results in bacterial death [<a href="#B21-ijms-26-01934" class="usa-link" aria-describedby="B21-ijms-26-01934">21</a>], was also evaluated. For <em>E. coli</em> BW25113, the MBC of tetrachloroauric acid was found to be 0.25 mM, while the most resistant environmental isolates (M53, M53B, and R37A) exhibited an MBC of 0.5 mM. These results indicate that gold is particularly toxic to the environmental isolates, requiring low concentrations to inhibit growth and exert bactericidal effects. This high level of toxicity could be attributed to the non-essential nature of gold for cellular processes and the limited resistance mechanisms in bacteria against the oxidative stress induced by gold ions, which include intracellular thiol depletion and superoxide anion accumulation, leading to damage in DNA, proteins, and membranes [<a href="#B20-ijms-26-01934" class="usa-link" aria-describedby="B20-ijms-26-01934">20</a>].</p>
<p>In parallel, the metal-reducing activity of the environmental isolates was evaluated using crude bacterial extracts. Previous research has linked the metal-reducing activity exhibited by bacterial crude extracts with the cell’s ability to reduce metals [<a href="#B16-ijms-26-01934" class="usa-link" aria-describedby="B16-ijms-26-01934">16</a>,<a href="#B22-ijms-26-01934" class="usa-link" aria-describedby="B22-ijms-26-01934">22</a>]. In this study, the reducing activity toward Au(III) was assessed at 37 °C across a pH range of 6.0 to 9.0, in the presence of either NADH or NADPH, to determine the isolates’ capacity to reduce this metal. Although a correlation was observed between the resistance levels of the isolates and their ability to reduce gold, this reduction was not identified as the primary mechanism of resistance.</p>
<p>The crude extracts displayed gold-reducing activity under various conditions, with activity observed at all tested pH levels and in the presence of both NADH and NADPH. The primary component of the crude extracts, given the extraction procedure, consists of the proteins within the microorganisms [<a href="#B23-ijms-26-01934" class="usa-link" aria-describedby="B23-ijms-26-01934">23</a>]. Thus, the metal-reducing activity observed is likely mediated primarily by these protein components. It has been reported that pH influences metal speciation, allowing the formation of complexes or the protonation/deprotonation of functional groups present in amino acids, which participate in stabilizing the enzyme–substrate complex, explaining the activity observed across the tested pH range [<a href="#B24-ijms-26-01934" class="usa-link" aria-describedby="B24-ijms-26-01934">24</a>].</p>
<p>Moreover, other molecules within the crude extracts, such as nucleic acids and polysaccharides, could also mediate the reduction of Au(III). Polysaccharides, under alkaline conditions, are capable of forming highly reactive molecules (such as enediols), which may contribute to the reduction of gold ions [<a href="#B25-ijms-26-01934" class="usa-link" aria-describedby="B25-ijms-26-01934">25</a>]. The preference for either NADH or NADPH as cofactors could be due to their stability within the active site of the proteins involved in the reduction process [<a href="#B26-ijms-26-01934" class="usa-link" aria-describedby="B26-ijms-26-01934">26</a>].</p>
<p>Overall, the study demonstrated that while environmental isolates showed resistance to gold and exhibited reducing activity under various conditions, this reduction activity was not the primary mechanism of resistance, suggesting that these bacteria might possess other mechanisms to cope with gold toxicity.</p></section><section id="sec3dot2-ijms-26-01934"><h3 class="pmc_sec_title">3.2. Identification, Cloning, and Reductase Activity of Proteins with Probable Au(III) Reductase Activity</h3>
<p>In this study, bioinformatic approaches were employed to identify proteins with potential Au(III) reductase activity, crucial for understanding the mechanisms by which bacteria might reduce gold ions. The first approach involved an extensive literature review to identify proteins previously reported to have metal reductase activity. Amino acid sequences of these proteins were analyzed using various databases, including PubMed, Uniprot, and Protein Data Bank, to find conserved motifs and domains that might indicate reductase activity. The second approach focused on a structural analysis of proteins known for their metal reductase functions, applying specific filters to uncover new proteins with similar activities. These methods collectively led to the identification of five proteins in <em>Staphylococcus haemolyticus</em> (BNF01) with probable Au (III) reductase activity, expanding our understanding of bacterial adaptation to toxic metal environments.</p>
<p>These proteins, identified primarily as oxidoreductases, catalyze redox reactions involving the transfer of electrons from a donor molecule to an acceptor molecule, facilitating the reduction of metal ions like Au(III). For example, glutathione reductase, typically involved in reducing glutathione disulfide to glutathione, was also predicted to possess Au(III) reductase activity, suggesting a secondary function beyond its primary enzymatic role [<a href="#B27-ijms-26-01934" class="usa-link" aria-describedby="B27-ijms-26-01934">27</a>]. Similarly, methylenetetrahydrofolate-tRNA-(uracil-5)-methyltransferase (TrmFO), which catalyzes the formation of 5-methyluridine in tRNAs, might reduce Au(III) as a secondary function [<a href="#B28-ijms-26-01934" class="usa-link" aria-describedby="B28-ijms-26-01934">28</a>].</p>
<p>The bioinformatic search used advanced servers and tools like InterProScan, Pfam, Prosite, Superfamily, and CATH to identify conserved domains and motifs. This strategy has been previously employed to identify six proteins with probable tellurite reductase activity in <em>E. coli</em>, including TrxB, AhpF, YkgC, GorA, E3, and NorW [<a href="#B29-ijms-26-01934" class="usa-link" aria-describedby="B29-ijms-26-01934">29</a>]. Of the 16 proteins identified in this study, most were classified as oxidoreductases, except for TrmFO, classified as a transferase. The ability of oxidoreductases to facilitate electron transfer plays a crucial role in reducing metal ions like Au(III), potentially leading to the formation of nanostructures. Moreover, the identified proteins, particularly flavoproteins containing a flavin adenine dinucleotide (FAD) prosthetic group, are known for their role in metal and metalloid reduction [<a href="#B30-ijms-26-01934" class="usa-link" aria-describedby="B30-ijms-26-01934">30</a>]. Notably, flavoproteins like alkyl hydroperoxide reductase, thioredoxin reductase, and flavorubredoxin reductase have shown reductase activity towards tellurite, while glutathione reductase has demonstrated activity against tellurite, selenite, and gold [<a href="#B16-ijms-26-01934" class="usa-link" aria-describedby="B16-ijms-26-01934">16</a>,<a href="#B31-ijms-26-01934" class="usa-link" aria-describedby="B31-ijms-26-01934">31</a>].</p>
<p>The identified genes encoding these proteins were cloned into expression vectors, with some challenges encountered during the process. Specifically, the <em>trxB</em> gene did not yield transformant colonies, likely due to an inappropriate insert-to-vector ratio or potential damage during gel purification [<a href="#B32-ijms-26-01934" class="usa-link" aria-describedby="B32-ijms-26-01934">32</a>]. Despite this, other genes were successfully cloned into pET101/D-TOPO and pET21b(+) vectors, allowing for the production of recombinant proteins under the control of the T7 promoter. This promoter, inducible by IPTG, facilitates high levels of protein expression in <em>E. coli</em> BL21 (DE3) cells. However, induction assays revealed that the optimal expression conditions varied across proteins, with some, like Oxi and TrmFO, showing low expression levels. This may be attributed to rare codons in the sequences, affecting translation efficiency, or high GC content at the 5′ end of the mRNA, impacting stability and translation [<a href="#B33-ijms-26-01934" class="usa-link" aria-describedby="B33-ijms-26-01934">33</a>,<a href="#B34-ijms-26-01934" class="usa-link" aria-describedby="B34-ijms-26-01934">34</a>].</p>
<p>Purification of these recombinant proteins was conducted using Ni<sup>+2</sup>-sepharose affinity chromatography, leveraging a 6xHis tag for selective binding. While CoADR and Gor were purified successfully with single bands representing the expected molecular weight, proteins like Oxi and TrmFO exhibited multiple bands, possibly due to contaminants or partial protein degradation. Such issues underscore the challenges in producing pure recombinant proteins, particularly those with metal reductase activity, necessitating further purification steps, including concentration through filtration units.</p>
<p>The reductase activity of these purified proteins was then assessed under varying pH conditions (6.0 to 9.0) and in the presence of NADH or NADPH, aiming to identify the optimal conditions for Au(III) reduction. The activity levels observed in these assays were lower than those recorded using crude extracts, suggesting that additional cellular components might contribute to metal reduction, as previously discussed. Notably, the proteins CoADR, TrmFO, Oxi, and Gor showed significant gold reductase activity, particularly at pH 9.0 with NADPH for Gor and at pH 7.4 in Tris-HCl buffer with NADPH for TrmFO. These findings align with previous studies reporting similar pH preferences for Au(III) reductase activity in other proteins, such as sulfite reductase and lignin peroxidase [<a href="#B14-ijms-26-01934" class="usa-link" aria-describedby="B14-ijms-26-01934">14</a>,<a href="#B35-ijms-26-01934" class="usa-link" aria-describedby="B35-ijms-26-01934">35</a>].</p>
<p>The reductase activity observed at pH levels of ≥7.4 likely relates to the basic conditions favoring the deprotonation of cysteine residues in the catalytic sites, leading to highly reactive thiolate anions, crucial for the reduction of Au(III) [<a href="#B36-ijms-26-01934" class="usa-link" aria-describedby="B36-ijms-26-01934">36</a>]. The study’s findings highlight the potential of these proteins, particularly under specific pH conditions, to facilitate gold reduction, with implications for nanostructure formation and other biotechnological applications.</p></section><section id="sec3dot3-ijms-26-01934"><h3 class="pmc_sec_title">3.3. Synthesis, Purification, Quantification, and Characterization of AuNS Synthesized In Vitro by the Gor Reductase Protein</h3>
<p>Gold nanostructures (AuNS) were synthesized in vitro using the purified recombinant Gor protein from the environmental isolate <em>Staphylococcus haemolyticus</em> (BNF01). The synthesis was conducted under optimal conditions: 20 mM Tris-HCl buffer at pH 9.0, 1 mM NADPH, 1 mM HAuCl<sub>4</sub>, and 50 µg of Gor protein. The formation of AuNS was visually confirmed by a color change in the reaction mixture from pale yellow to dark purple after 24 h at 37 °C, consistent with the synthesis of AuNS [<a href="#B37-ijms-26-01934" class="usa-link" aria-describedby="B37-ijms-26-01934">37</a>]. The generation of AuNS through biological methods, particularly using proteins, has been documented in various studies. For example, the recombinant gold reductase (GoIR) from <em>Erwinia</em> sp. IMH was shown to generate AuNS when incubated with NADH and HAuCl<sub>4</sub> in Tris-HCl at 30 °C [<a href="#B38-ijms-26-01934" class="usa-link" aria-describedby="B38-ijms-26-01934">38</a>]. Similarly, a NADPH-dependent sulfite reductase from <em>E. coli</em> produced AuNS at 25 °C when mixed with HAuCl<sub>4</sub> and other components in phosphate buffer [<a href="#B14-ijms-26-01934" class="usa-link" aria-describedby="B14-ijms-26-01934">14</a>]. The use of glutathione reductase for AuNS synthesis has also been reported, where the enzyme was incubated with NADPH and HAuCl<sub>4</sub>, leading to electron transfer from NADPH to the FAD moiety of the enzyme, ultimately reducing AuCl<sup>4−</sup> to Au<sup>0</sup> and stabilizing the nanostructures [<a href="#B31-ijms-26-01934" class="usa-link" aria-describedby="B31-ijms-26-01934">31</a>]. The direct role of NADH/NADPH cofactors alone in AuNS synthesis (abiotic synthesis) was not evaluated in this research, as the study was focused on the enzymatic generation of nanoparticles. However, controls of the nanoparticle synthesis reaction were performed without the Gor protein, conditions under which no abiotic AuNS formation was obtained. Despite this, under certain conditions of pH, temperature, and ionic strength, NADH/NADPH cofactors could contribute to metal reduction, and this remains an interesting aspect for future research.</p>
<p>The biosynthesized AuNS were purified and concentrated by centrifugation using 3 kDa filtration units (Amicon<sup>®</sup>), which effectively removed proteins and other reaction components that might interfere with subsequent applications. The quantification of AuNS was performed using atomic absorption spectroscopy (AAS), which determined the concentration of gold in the samples, yielding 1150 μg/mL. AAS is a widely used technique for quantifying metal concentrations in samples, providing accurate and sensitive measurements [<a href="#B39-ijms-26-01934" class="usa-link" aria-describedby="B39-ijms-26-01934">39</a>].</p>
<p>Characterization of the AuNS was conducted using several analytical techniques to ensure that they were within the nanoscale range and to explore their properties. Transmission electron microscopy (TEM) (Hitachi High-Technologies Corporation, Tokyo, Japan) revealed that the AuNS exhibited a homogeneous morphology, predominantly in the shape of branched or spiky structures, often referred to as “urchin-like” gold nanostructures. Such morphologies have been described in previous studies and are known for their unique optical properties and broad absorption spectra [<a href="#B40-ijms-26-01934" class="usa-link" aria-describedby="B40-ijms-26-01934">40</a>]. The synthesized AuNS varied in size, with most falling within the 10–20 nm and 70–80 nm ranges, though some aggregation was observed, likely due to the sample preparation method for TEM analysis.</p>
<p>Scanning electron microscopy coupled with energy-dispersive X-ray spectroscopy (SEM–EDX) confirmed the chemical composition of the AuNS, which contained 23% gold, along with significant amounts of carbon (38.7%) and oxygen (38.0%). The presence of carbon and oxygen is likely associated with stabilizing agents, such as amino acids or other biomolecules from the Gor protein, which were not fully removed during purification [<a href="#B41-ijms-26-01934" class="usa-link" aria-describedby="B41-ijms-26-01934">41</a>]. These stabilizers play a crucial role in determining the surface chemistry, morphology, and size distribution of the nanostructures. UV–Vis spectroscopy revealed that the biosynthesized AuNS had a maximum absorbance at 622 nm, consistent with the absorbance range for urchin-like nanostructures, which typically exhibit broad absorption properties due to their complex morphology and size distribution [<a href="#B42-ijms-26-01934" class="usa-link" aria-describedby="B42-ijms-26-01934">42</a>]. The broad peak observed in the UV–Vis spectrum corresponds to the polydispersity of the AuNS, which was further supported by dynamic light scattering (DLS) analysis. DLS measurements showed two predominant size distributions at 6.68 nm and 74.38 nm, with the smaller particles being more abundant. However, these results do not agree with those obtained through TEM, which is mainly due to sample preparation: (i) for TEM imaging, multiple droplets were deposited on the copper grid, leading to increased local concentrations and aggregation of nanoparticles due to their zeta potential; (ii) for DLS analysis, the sample was sonicated prior to measurement, which prevented aggregation and reflected the true size distribution of individual particles.</p>
<p>The zeta potential of the AuNS was measured as −21.4 mV, indicating that the nanostructures possess moderate stability with a tendency to aggregate. Zeta potential values within the range of −10 to +10 mV are typically associated with neutral particles, while values exceeding ±30 mV indicate strong repulsive forces and greater stability [<a href="#B43-ijms-26-01934" class="usa-link" aria-describedby="B43-ijms-26-01934">43</a>]. Regarding interaction with biological systems, it has been described that nanostructures with positive zeta potential tend to bind to negatively charged surfaces and/or molecules, and vice versa, so the biosynthesized AuNS could interact with components that have a positive charge, such as the hemagglutinin protein of the influenza A virus [<a href="#B44-ijms-26-01934" class="usa-link" aria-describedby="B44-ijms-26-01934">44</a>,<a href="#B45-ijms-26-01934" class="usa-link" aria-describedby="B45-ijms-26-01934">45</a>]. This potential interaction highlights the importance of zeta potential in the stability and biological interactions of AuNS.</p>
<p>Overall, the successful synthesis, purification, quantification, and characterization of AuNS using the Gor protein demonstrate the potential of biological methods for producing nanomaterials with specific properties tailored for various applications. This study adds to the growing body of research on the biosynthesis of metal nanostructures, highlighting the advantages of using recombinant proteins for controlled and efficient nanoparticle production.</p></section><section id="sec3dot4-ijms-26-01934"><h3 class="pmc_sec_title">3.4. Propagation and Quantification of Influenza A H1N1 Virus in Cell Culture</h3>
<p>The propagation of influenza A H1N1 virus in MDCK cell cultures was essential for producing viral inoculum for virucidal assays in suspension and on non-porous stainless-steel surfaces. MDCK cells were selected due to their documented high susceptibility to influenza A virus, which often results in higher viral titers compared to other cell lines such as A549 or Vero [<a href="#B46-ijms-26-01934" class="usa-link" aria-describedby="B46-ijms-26-01934">46</a>]. Upon infection, MDCK cells exhibited a typical cytopathic effect (CPE), including significant cell detachment and loss of monolayer integrity 48 to 72 h post-infection, consistent with established findings [<a href="#B47-ijms-26-01934" class="usa-link" aria-describedby="B47-ijms-26-01934">47</a>,<a href="#B48-ijms-26-01934" class="usa-link" aria-describedby="B48-ijms-26-01934">48</a>].</p>
<p>To verify the presence of the virus in the collected supernatants, a hemagglutination (HA) assay was performed. This assay leverages the hemagglutinin protein’s ability to bind to red blood cells, confirming successful propagation with HA units of 4 and 64. While HA assays can estimate viral quantities, it is important to recognize that various factors, such as erythrocyte quality and ambient temperature, can influence results [<a href="#B49-ijms-26-01934" class="usa-link" aria-describedby="B49-ijms-26-01934">49</a>].</p>
<p>For more precise quantification, a recombinant plasmid containing a conserved segment of the matrix (M) protein gene was constructed to serve as a calibration standard for RT-qPCR. This approach, based on methods developed by Spackman et al. (2002) [<a href="#B50-ijms-26-01934" class="usa-link" aria-describedby="B50-ijms-26-01934">50</a>] and Furuse et al. (2009) [<a href="#B51-ijms-26-01934" class="usa-link" aria-describedby="B51-ijms-26-01934">51</a>], facilitated the absolute quantification of viral RNA. The analysis revealed viral titers ranging from 1.6 to 8.2 × 10<sup>5</sup> copies/mL, insufficient for virucidal assays that require at least 1 × 10<sup>6</sup> copies/mL [<a href="#B52-ijms-26-01934" class="usa-link" aria-describedby="B52-ijms-26-01934">52</a>].</p>
<p>To address this issue, viral supernatants were concentrated using Amicon<sup>®</sup> ultrafiltration units, a method selected for its ability to concentrate viral particles while maintaining infectivity. This procedure resulted in a 100-fold concentration, yielding final viral titers of 1.43 × 10<sup>7</sup> copies/mL by RT-qPCR and 5.6 × 10<sup>7</sup> TCID<sub>50</sub>/mL by Reed–Muench assay, within the expected range for such concentration techniques [<a href="#B53-ijms-26-01934" class="usa-link" aria-describedby="B53-ijms-26-01934">53</a>]. The success of this approach highlights the significance of reliable concentration methods in achieving adequate viral titers for antiviral testing, especially when initial propagation yields are suboptimal. Additionally, the potential use of alternative MDCK variants like MDCK-SIAT1 or MDCK-London, which may produce higher viral titers due to differences in sialic acid receptor expression, could be considered for future studies [<a href="#B54-ijms-26-01934" class="usa-link" aria-describedby="B54-ijms-26-01934">54</a>,<a href="#B55-ijms-26-01934" class="usa-link" aria-describedby="B55-ijms-26-01934">55</a>].</p></section><section id="sec3dot5-ijms-26-01934"><h3 class="pmc_sec_title">3.5. Virucidal Activity of Biosynthesized AuNS Against Influenza A H1N1 Virus</h3>
<p>The virucidal activity of the biosynthesized gold nanostructures (AuNS) and commercial copper nanostructures (CuNS) against influenza A H1N1 virus was evaluated through suspension assays and on non-porous stainless-steel surfaces. The results showed a reduction in viral titers with Virkon<sup>®</sup> and biosynthesized AuNS in both short (10 min) and long (60 min) contact times in suspension assays, while commercial CuNS only reduced viral titers after 60 min. On stainless steel surfaces, viral titer reduction was observed with Virkon<sup>®</sup> at both contact times and with commercial CuNS after 60 min.</p>
<p>The expected reduction in the viral titers with Virkon<sup>®</sup> confirms its effectiveness as a positive control, given its established efficacy against influenza A virus [<a href="#B56-ijms-26-01934" class="usa-link" aria-describedby="B56-ijms-26-01934">56</a>]. The commercial CuNS, with a hydrodynamic diameter of &lt;10 nm, are known biocidal agents, but the specific concentration and contact time required for virucidal activity were not provided, likely necessitating higher concentrations for shorter contact times.</p>
<p>Biosynthesized AuNS demonstrated virucidal activity against influenza A H1N1 in both short and long contact times during suspension assays, effectively reducing viral titers. Previous studies have shown that porous and non-porous AuNS can inactivate the influenza A virus, achieving nearly 100% cell viability at 0.2 mg/mL and 50% at 0.1 mg/mL, indicating viral inactivation and cell survival [<a href="#B57-ijms-26-01934" class="usa-link" aria-describedby="B57-ijms-26-01934">57</a>]. Comparatively, the biosynthesized AuNS in this study exhibited stronger virucidal effects at 100 µg/mL, suggesting enhanced efficacy at lower concentrations.</p>
<p>Additionally, Chaika et al. (2022) [<a href="#B58-ijms-26-01934" class="usa-link" aria-describedby="B58-ijms-26-01934">58</a>] reported that 5 nm and 20 nm AuNS synthesized chemically via sodium citrate reduction exhibited 80% and 50% inhibition of viral infection, respectively, at 1.97 μg/mL. However, these nanostructures were cytotoxic at the tested concentration, contrasting with the lower toxicity of biosynthesized AuNS.</p>
<p>Virucidal efficacy is defined as a reduction of ≥4 log<sub>10</sub> in viral titers in suspension assays or ≥3 log<sub>10</sub> on non-porous surfaces, equating to a 99.9% reduction in microorganisms [<a href="#B17-ijms-26-01934" class="usa-link" aria-describedby="B17-ijms-26-01934">17</a>,<a href="#B18-ijms-26-01934" class="usa-link" aria-describedby="B18-ijms-26-01934">18</a>]. Based on this definition, only biosynthesized AuNS demonstrated effective virucidal activity against influenza A H1N1 during long contact times in suspension assays, achieving a 4 log<sub>10</sub> reduction in viral titers. However, to confirm general virucidal efficacy, these gold nanostructures should be tested against three non-enveloped viruses—poliovirus type 1, adenovirus type 5, and murine norovirus—as these are highly resistant to various agents, and effectiveness against them would indicate broader antiviral potential [<a href="#B17-ijms-26-01934" class="usa-link" aria-describedby="B17-ijms-26-01934">17</a>,<a href="#B18-ijms-26-01934" class="usa-link" aria-describedby="B18-ijms-26-01934">18</a>].</p></section><section id="sec3dot6-ijms-26-01934"><h3 class="pmc_sec_title">3.6. Inhibitory Concentration (IC<sub>50</sub>) and Cytotoxicity of Biosynthesized AuNS</h3>
<p>The biosynthesized gold nanostructures (AuNS) were found to be effective viricidal agents against influenza A H1N1 virus after 60 min of treatment in suspension assays, with an IC<sub>50</sub> determined to be 76.64 ± 1.08 μg/mL. In comparison, a study by Kim et al. (2020) [<a href="#B57-ijms-26-01934" class="usa-link" aria-describedby="B57-ijms-26-01934">57</a>] observed that treatment with non-porous spherical AuNS at a concentration of 0.2 mg/mL and porous AuNS at 0.1 mg/mL led to a 50% inhibition of infection in MDCK cell cultures after 60 min, indicating that biosynthesized AuNS in this study are more effective at lower concentrations [<a href="#B57-ijms-26-01934" class="usa-link" aria-describedby="B57-ijms-26-01934">57</a>]. Similarly, another study reported an IC<sub>50</sub> of 210 µg/mL using AuNS biosynthesized with <em>Glaucium flavum</em> leaf extract [<a href="#B59-ijms-26-01934" class="usa-link" aria-describedby="B59-ijms-26-01934">59</a>], further highlighting the relative efficiency of the biosynthesized AuNS.</p>
<p>Additionally, the cytotoxicity of the biosynthesized AuNS was evaluated in MDCK cell cultures over 48 h, revealing a 50% cell survival rate at an exposure concentration of 106.3 ± 1.2 μg/mL. Other studies have assessed the cytotoxicity of AuNS of different sizes, showing 59% cell survival after exposure to 1.97 μg/mL of 5 nm AuNS and 76% survival with 1.97 μg/mL of 20 nm AuNS [<a href="#B58-ijms-26-01934" class="usa-link" aria-describedby="B58-ijms-26-01934">58</a>]. Despite the concentration used in these viricidal assays being nearly 100 times higher, the 5 nm and 20 nm AuNS exhibited greater toxicity compared to the biosynthesized AuNS. Differences in toxicity could be attributed to various characteristics of the nanostructures, such as shape, size, surface charge, concentration, and exposure time [<a href="#B60-ijms-26-01934" class="usa-link" aria-describedby="B60-ijms-26-01934">60</a>].</p>
<p>Although the biosynthesized AuNS are effective viricidal agents at 100 μg/mL, and exhibit a 50% cell survival rate at this concentration, additional studies are necessary to evaluate their potential as disinfectants or surface protectants. These studies should include further toxicity assessments using in vitro models (e.g., organoids) or in vivo studies (e.g., small animal models) to ensure that they pose no risk to health or the environment [<a href="#B61-ijms-26-01934" class="usa-link" aria-describedby="B61-ijms-26-01934">61</a>].</p></section><section id="sec3dot7-ijms-26-01934"><h3 class="pmc_sec_title">3.7. Probable Mechanism of Virucidal Action of Biosynthesized AuNS Against Influenza A H1N1 Virus</h3>
<p>To elucidate the potential mechanism through which biosynthesized gold nanostructures (AuNS) exert their virucidal effect against the influenza A H1N1 virus, we first evaluated the possible damage to the viral RNA genome after treatment with AuNS for 10 and 60 min using RT-qPCR. The results indicated that treatment with Virkon<sup>®</sup> and commercial CuNS did not lead to significant changes in the initial viral M gene copies, which is consistent with the known action of Virkon<sup>®</sup>, a disinfectant that disrupts cellular membranes through peroxygen-based compounds [<a href="#B62-ijms-26-01934" class="usa-link" aria-describedby="B62-ijms-26-01934">62</a>]. Similarly, CuNS are thought to inactivate viruses by releasing Cu<sup>+1</sup> and Cu<sup>+2</sup> ions, generating highly reactive hydroxyl radicals that can damage cellular components, although these radicals have a very short half-life, potentially explaining the lack of observed viral RNA degradation [<a href="#B63-ijms-26-01934" class="usa-link" aria-describedby="B63-ijms-26-01934">63</a>,<a href="#B64-ijms-26-01934" class="usa-link" aria-describedby="B64-ijms-26-01934">64</a>]. In contrast, AuNS-treated samples showed a slight decrease in viral RNA copies, suggesting some genomic damage, but the reduction was minor (&lt;1 log<sub>10</sub>), indicating that this may not be the primary mechanism of virucidal action.</p>
<p>A more in-depth investigation focused on whether AuNS could cause degradation of viral proteins, particularly hemagglutinin, a critical protein for the virus’s ability to attach to host cells. Hemagglutination (HA) assays revealed that after 10 min of treatment with biosynthesized AuNS, the HA units decreased from 2 to 1, and after 60 min, hemagglutination was completely inhibited. This reduction suggests that hemagglutinin’s structural integrity may be compromised by AuNS, as hemagglutination relies on the intact function of this protein to bind red blood cells [<a href="#B65-ijms-26-01934" class="usa-link" aria-describedby="B65-ijms-26-01934">65</a>]. In contrast, treatment with CuNS, both biosynthesized and commercial, did not reduce HA units, indicating that these nanostructures did not affect hemagglutinin.</p>
<p>To further assess the impact of AuNS on viral particles, transmission electron microscopy (TEM) was employed. Untreated influenza A virions exhibited the expected morphology with a well-defined lipid membrane and surface proteins, consistent with known characteristics of influenza virions [<a href="#B66-ijms-26-01934" class="usa-link" aria-describedby="B66-ijms-26-01934">66</a>]. However, AuNS-treated virions displayed significant alterations, including loss of surface proteins, reduction in average diameter from 144 nm to 128 nm, and irregular shapes, although the lipid membrane remained intact. These observations align with studies in which nanostructures were shown to adsorb onto viruses, causing inactivation without completely destroying the virions [<a href="#B57-ijms-26-01934" class="usa-link" aria-describedby="B57-ijms-26-01934">57</a>,<a href="#B67-ijms-26-01934" class="usa-link" aria-describedby="B67-ijms-26-01934">67</a>]. Additionally, the presence of electron-dense regions within the treated virions, likely corresponding to smaller AuNS adsorbed on the viral surface, was supported by atomic absorption spectroscopy (EAA), which quantified approximately 13.6 µg/mL of AuNS associated with the virions.</p>
<p>Further examination of viral proteins through SDS-PAGE revealed a decrease in the intensity of viral protein bands after AuNS treatment, suggesting that the nanostructures might cause protein degradation. This result is consistent with findings from studies involving CuNS, in which an increase in nanostructure concentration correlated with reduced protein band intensity, indicating protein degradation [<a href="#B68-ijms-26-01934" class="usa-link" aria-describedby="B68-ijms-26-01934">68</a>]. The additional bands observed in the SDS-PAGE analysis could be attributed to cellular proteins incorporated into virions during assembly, a phenomenon previously noted in influenza virus assembly [<a href="#B69-ijms-26-01934" class="usa-link" aria-describedby="B69-ijms-26-01934">69</a>]. These findings suggest that protein degradation may play a key role in the virucidal mechanism of biosynthesized AuNS. However, further experiments, such as Western blotting with specific antibodies, are necessary to confirm which viral proteins are most affected by AuNS treatment.</p>
<p>Spectrophotometric measurements at 280 nm were conducted to detect potential degradation products of viral proteins. The results showed no significant change in absorbance over the 60 min treatment with AuNS, which might be explained by the generation of smaller peptides that continue to absorb at this wavelength despite protein degradation [<a href="#B70-ijms-26-01934" class="usa-link" aria-describedby="B70-ijms-26-01934">70</a>]. Given the known affinity of AuNS for sulfhydryl groups, the Ellman assay was used to determine whether the AuNS affected these groups within viral proteins. The assay results indicated no significant reduction in sulfhydryl content, suggesting that AuNS did not specifically interact with cysteine residues in hemagglutinin or neuraminidase, despite these proteins having multiple cysteine residues [<a href="#B71-ijms-26-01934" class="usa-link" aria-describedby="B71-ijms-26-01934">71</a>]. Previous studies have shown that porous AuNS, which have a larger surface area than non-porous AuNS, are more effective in interacting with viral proteins, potentially explaining the differences in interaction observed in this study [<a href="#B57-ijms-26-01934" class="usa-link" aria-describedby="B57-ijms-26-01934">57</a>].</p>
<p>In summary, while biosynthesized AuNS appear to exert their virucidal action through multiple mechanisms, including potential minor damage to viral RNA and significant protein degradation, particularly of hemagglutinin, the primary mode of action likely involves structural disruption of the virus, leading to loss of infectivity. The findings suggest that AuNS could serve as potent antiviral agents, although further research is needed to fully understand their interactions with viral components and to confirm their efficacy and safety in broader applications.</p></section></section><section id="sec4-ijms-26-01934"><h2 class="pmc_sec_title">4. Materials and Methods</h2>
<section id="sec4dot1-ijms-26-01934"><h3 class="pmc_sec_title">4.1. Bacterial Strains and Growth Conditions</h3>
<p>Environmental bacterial isolates from Chile, Peru, and Bolivia (<a href="#app1-ijms-26-01934" class="usa-link">Table S3</a>) were cultured in Luria Bertani (LB) medium (10 g/L tryptone, 10 g/L NaCl, 5 g/L yeast extract) at their optimal growth temperatures, previously determined in our lab. We used LB-agar (2% <em>w/v</em>), with incubation overnight at the respective optimal temperatures, identifying the multiple strains, including <em>Enterobacter cloacae</em> (MF01), <em>Staphylococcus</em> sp. (MF16), <em>Staphylococcus haemolyticus</em> (BNF01), <em>Staphylococcus sciuri</em> (BNF05), <em>Psychrobacter immobilis</em> (BNF20), <em>Pseudomonas weihenstephanensis</em> (M53, R37A), and <em>Pseudomonas arsenicoxydans</em> (M53B).</p>
<p>On the other hand, <em>Escherichia coli</em> strains (<a href="#app1-ijms-26-01934" class="usa-link">Table S4</a>) used in the recombinant protein expression assays were grown in LB medium at 37 °C with shaking (150 rpm), or solid medium of LB-agar (2% <em>w</em>/<em>v</em>) was used, incubated overnight at 37 °C. <em>E. coli</em> TOP10 cells (Invitrogen<sup>®</sup>, Waltham, MA, USA) transformed with the DNA vectors (pET101/D-TOPO (Invitrogen<sup>®</sup>), pET21b(+) (Novagen<sup>®</sup>, Darmstadt, Germany), and pGEM-T Easy (Promega<sup>®</sup>, Madison, WI, USA), were cultured with 100 µg/mL ampicillin and 50 µg/mL streptomycin, while <em>E. coli</em> BL21(DE3) cells (Invitrogen<sup>®</sup>) were cultured with 100 µg/mL ampicillin.</p></section><section id="sec4dot2-ijms-26-01934"><h3 class="pmc_sec_title">4.2. Minimum Inhibitory Concentration (MIC), Minimum Bactericidal Concentration (MBC), and Bacterial Growth Parameters</h3>
<p>MIC and MBC were determined following Benhalima et al. (2019) [<a href="#B72-ijms-26-01934" class="usa-link" aria-describedby="B72-ijms-26-01934">72</a>]. Serial dilutions of HAuCl<sub>4</sub> (Sigma-Aldrich, Burlington, MA, USA) starting at 1 mM were prepared, and 10 µL of bacterial cultures (OD<sub>600nm</sub> of 0.5) were added to each well in a 48-well plate. After 24 h of incubation with shaking at the optimal temperature, MIC was identified by turbidity, while MBC was confirmed by plating 100 µL from non-turbid wells on LB agar and observing the absence of colonies after 24 h.</p>
<p>For bacterial growth curves, overnight cultures of environmental isolates were diluted and incubated in fresh LB medium containing sublethal metal concentrations. Growth was monitored at 600 nm every 30 min for 18 h using a TECAN Infinite<sup>®</sup> M200 Pro plate reader (Mannedorf, Switzerland). Growth parameters, including maximum OD<sub>600nm</sub>, growth rate (µ), and generation time, were calculated from the logarithmic growth phase and compared with control conditions.</p></section><section id="sec4dot3-ijms-26-01934"><h3 class="pmc_sec_title">4.3. Preparation of Crude Bacterial Extracts</h3>
<p>Crude bacterial extracts were prepared by centrifuging cultures grown to an OD<sub>600nm</sub> of 0.5 at 9000 rpm for 10 min at 4 °C. The pellet was resuspended in 20 mM Tris-HCl buffer (pH 7.4) with 0.1 mM PMSF (Roche<sup>®</sup>, Basel, Switzerland), then sonicated on ice (3 cycles of 5 min, 60% amplitude). The resulting solution was centrifuged at 14,000 rpm for 10 min at 4 °C, and the supernatant was collected as the crude extract. Protein concentrations were determined using the Bradford method (Sigma-Aldrich) with BSA (Sigma-Aldrich) as a standard, measuring absorbance at 595 nm [<a href="#B73-ijms-26-01934" class="usa-link" aria-describedby="B73-ijms-26-01934">73</a>]. Additionally, protein quantification in chromatography fractions and treated virions was done at 280 nm using a TECAN Infinite<sup>®</sup> M200 Pro plate reader.</p></section><section id="sec4dot4-ijms-26-01934"><h3 class="pmc_sec_title">4.4. Gold-Reducing Activity Mediated by Crude Extracts</h3>
<p>The metal-reducing activity of Au(III) contained in the respective salts HAuCl<sub>4</sub> exhibited by crude bacterial extracts from environmental isolates was evaluated, as described by Figueroa et al. (2018) [<a href="#B16-ijms-26-01934" class="usa-link" aria-describedby="B16-ijms-26-01934">16</a>], at the optimal growth temperature for each isolate in a range of pH values, using 20 mM potassium phosphate buffer at pH 6.0, 7.0, and 7.4, and 20 mM Tris-HCl buffer at pH 7.4, 8.0, and 9.0. The reaction was performed in a volume of 200 µL in the corresponding pH buffer containing 50 µg of crude extract from the isolate, 1 mM NAD(P)H (NADH Sigma-Aldrich; NADPH Roche<sup>®</sup>), and the metal salt to be evaluated at a concentration of 1 mM. The production of metal in its elemental state was monitored spectrophotometrically in a range of 400–800 nm for gold. One enzyme unit (U) was defined as the amount of enzyme required to increase the absorbance by 0.001 units per minute under assay conditions. Enzyme activity was normalized by protein concentration.</p></section><section id="sec4dot5-ijms-26-01934"><h3 class="pmc_sec_title">4.5. Identification of Proteins with Potential Gold-Reducing Activity</h3>
<p>An in silico search for proteins with potential metal-reducing activity was conducted through amino acid sequence and structural analysis at the Center for Bioinformatics and Molecular Simulation at the University of Talca. For sequence analysis, a literature search was carried out using PubMed and Oxford Journal platforms, with sequences obtained from the Uniprot platform. The sequences were analyzed using InterProScan (accessed on 20 April 2023), along with Pfam (accessed on 29 May 2023), ProSite (accessed on 3 June 2023), and Superfamily (accessed on 14 June 2023) databases to identify conserved patterns and motifs [<a href="#B74-ijms-26-01934" class="usa-link" aria-describedby="B74-ijms-26-01934">74</a>,<a href="#B75-ijms-26-01934" class="usa-link" aria-describedby="B75-ijms-26-01934">75</a>,<a href="#B76-ijms-26-01934" class="usa-link" aria-describedby="B76-ijms-26-01934">76</a>,<a href="#B77-ijms-26-01934" class="usa-link" aria-describedby="B77-ijms-26-01934">77</a>,<a href="#B78-ijms-26-01934" class="usa-link" aria-describedby="B78-ijms-26-01934">78</a>]. For structural analysis, known metal-reducing proteins such as thioredoxin reductase, dihydrolipoamide dehydrogenase, and oxidoreductase were used. The Protein Data Bank (PDB) platform was utilized to find new proteins that bind metals of interest (gold), contain FAD, and share 70% identity. Identified proteins were classified using CATH, RMSD, and SSAP servers [<a href="#B29-ijms-26-01934" class="usa-link" aria-describedby="B29-ijms-26-01934">29</a>]. These proteins were then subjected to a BLASTp search against available amino acid sequences from environmental isolates that showed the highest resistance and reduction according to assays performed. Once identified, the nucleotide sequences were obtained from the NCBI database using the protein names from the BLASTp search.</p></section><section id="sec4dot6-ijms-26-01934"><h3 class="pmc_sec_title">4.6. Cloning and Molecular Biology Techniques</h3>
<p>Genomic DNA from environmental isolate BNF01 was extracted using the Wizard<sup>®</sup> Genomic DNA Purification Kit (Promega<sup>®</sup>, Madison, WI, USA), with integrity verified via agarose gel electrophoresis and quantification performed as required. We designed specific primers (<a href="#app1-ijms-26-01934" class="usa-link">Table S2</a>) using Snapgene software 8.0 (GSL Biotech, San Diego, CA, USA), which were synthesized by Integrated DNA Technologies (IDT, Coralville, IA, USA), for the amplification of genes with potential metal-reducing activity, such as <em>trxB</em>, <em>cdr</em>, <em>oxi</em>, <em>trmFO</em>, and <em>gor</em>. These primers were tailored for use with the pET101/D-TOPO and pET21b(+) vectors, ensuring correct insertion sites and directional cloning.</p>
<p>The amplification of target genes was carried out using PCR with high-fidelity DNA polymerases like Q5 (NEB<sup>®</sup>, Ipswich, MA, USA) and Gotaq (Promega<sup>®</sup>). Construct preparation involved digesting the pET21b(+) vector and PCR products with <em>Nhe</em>I-HF and <em>Hind</em>III-HF enzymes (NEB<sup>®</sup>) for precise cloning. For pET101/D-TOPO cloning, the <em>cdr</em> gene from BNF01 was utilized, while the conserved segment of the influenza A virus matrix protein gene (<em>Mseg</em>) was cloned into the pGEM-T Easy vector. These constructs were verified by colony PCR, ensuring the correct orientation and presence of inserts.</p>
<p>Following transformation of <em>E. coli</em> TOP10 and BL21(DE3) cells, recombinant plasmids were purified using the FavorPrep™ Plasmid DNA Extraction Mini Kit (Favorgen<sup>®</sup>, Taiwan, China). The constructs generated included pET101/<em>cdr</em>, pET21b/<em>gor</em>, pET21b/<em>oxi</em>, pET21b/<em>trmFO</em>, pET21b/<em>trxB</em>, as well as pGEM-T Easy/<em>Mseg</em> from influenza A virus. Each construct was sequenced in full using Oxford Nanopore Technologies (ONT) long-read sequencing (Plasmidsaurus Inc., Monrovia, CA, USA), with sequence data analyzed via Snapgene and BLASTn databases to confirm accuracy.</p>
<p>Agarose gel electrophoresis (1% or 2% <em>w</em>/<em>v</em>) was routinely used to assess the integrity of genomic DNA, plasmids, and PCR products. Constructs involving genes with potential metal-reducing activity and the influenza A virus matrix protein were critical for subsequent expression and functional studies, with correct gene insertion confirmed through sequencing and structural analysis.</p></section><section id="sec4dot7-ijms-26-01934"><h3 class="pmc_sec_title">4.7. Expression, Purification, and Analysis of Proteins with Potential Gold-Reducing Activity</h3>
<p>Cultures of <em>E. coli</em> BL21(DE3) cells transformed with recombinant vectors (pET101/<em>cdr</em>, pET21b/<em>gor</em>, pET21b/<em>oxi</em>, pET21b/<em>trmFO</em>, and pET21b/<em>trxB</em>) were grown at 30 °C and 37 °C until reaching an OD<sub>600nm</sub> of 0.5. Protein expression was induced with 1 mM IPTG (Invitrogen<sup>®</sup>), and samples were taken at various time points to optimize expression conditions. Crude extracts were analyzed by SDS-PAGE to confirm protein expression. Additionally, silver staining [<a href="#B79-ijms-26-01934" class="usa-link" aria-describedby="B79-ijms-26-01934">79</a>] was performed to assess the effect of gold nanostructures on influenza A H1N1 virus proteins, specifically examining viral hemagglutinin.</p>
<p>Purification of the expressed proteins was performed using Ni<sup>2+</sup> affinity chromatography as we described previously [<a href="#B29-ijms-26-01934" class="usa-link" aria-describedby="B29-ijms-26-01934">29</a>]. In brief, crude extracts were loaded onto a HisTrap HP column (HisTrap<sup>®</sup> HP, Cytiva<sup>®</sup>, Marlborough, MA, USA), washed, and eluted with an imidazole-containing buffer. Eluted fractions were quantified and dialyzed, and the purity of the purified proteins was confirmed by SDS-PAGE. These purified proteins were then tested for their potential gold-reducing activity using HAuCl<sub>4</sub> as a substrate at 25 °C.</p></section><section id="sec4dot8-ijms-26-01934"><h3 class="pmc_sec_title">4.8. Synthesis, Purification, and Characterization of Gold Nanostructures (AuNS)</h3>
<p>Gold nanostructures (AuNS) were synthesized in vitro using purified recombinant Gor protein from the environmental isolate BNF01, following Figueroa et al. (2018) [<a href="#B16-ijms-26-01934" class="usa-link" aria-describedby="B16-ijms-26-01934">16</a>]. The reaction involved 20 mM Tris-HCl buffer (pH 9.0), 1 mM NADPH, 1 mM HAuCl<sub>4</sub>, and 50 μg of the purified protein. Post-synthesis, AuNS were purified and concentrated using Amicon Ultra-4 3 kDa (Amicon<sup>®</sup>, Darmstadt, Germany) filtration units and centrifuged at 7500× <em>g</em> for 10 min. The concentration of gold in the AuNS was measured by flame atomic absorption spectrometry (Analityk Jena Model Anova 350, Jena, Germany).</p>
<p>The size and morphology of AuNS were characterized by transmission electron microscopy (TEM) using a Hitachi HT7700 microscope (CEDENNA, University of Santiago de Chile), whereby a drop of the nanostructures was placed on a copper grid coated with formvar and carbon film and observed at 100 kV. Their chemical composition was determined using energy-dispersive X-ray spectroscopy (EDX) in conjunction with scanning electron microscopy (SEM) on a Zeiss EVO MA10 microscope (Oberkochen, Germany) equipped with an Oxford Instruments X-act system and Penta FET precision detector (Boston, MA, USA). The UV–Vis absorption spectrum was recorded between 400 and 800 nm using a Jasco V-730 spectrophotometer (Tokyo, Japan). Additionally, the size distribution and zeta potential of AuNS were analyzed using a Zetasizer Nano ZS90 (Malvern, UK), with samples sonicated for size measurements and ionic strength adjusted with KCl for zeta potential analysis.</p></section><section id="sec4dot9-ijms-26-01934"><h3 class="pmc_sec_title">4.9. Cultivation, Propagation, and Molecular Analysis of Influenza A H1N1 Virus in MDCK Cells</h3>
<p>Madin–Darby canine kidney (MDCK) cells (ATCC/CCL-34, Manassas, VA, USA) were cultured in high-glucose DMEM (Cytiva<sup>®</sup>) supplemented with 10% fetal bovine serum (FBS) (Gibco<sup>®</sup>, Waltham, MA, USA) and penicillin–streptomycin (Corning<sup>®</sup>, New York, NY, USA) at 37 °C in a 5% CO<sub>2</sub> atmosphere to maintain optimal growth conditions. The cells were subcultured regularly to ensure that they remained healthy and viable for experiments. For the propagation of the influenza A H1N1 virus (A/Virginia/ATCC1/2009 VR-1736), MDCK cells were grown to 80–90% confluence, which is ideal for infection. The cells were then infected with the virus at a multiplicity of infection (MOI) of 0.1. After a 60 min adsorption period to allow the virus to attach to the cells, the medium was replaced with Opti-MEM I (Gibco<sup>®</sup>) supplemented with TPCK-treated trypsin (Thermo Fisher Scientific<sup>®</sup>, Waltham, MA, USA), which facilitates viral entry and replication. The infected cells were incubated for 72 h at 37 °C in a 5% CO<sub>2</sub> atmosphere. Post-incubation, the viral supernatant was harvested, clarified by centrifugation to remove cell debris, concentrated using Spin-X UF 20 50 kDa filtration units (Corning<sup>®</sup>), and stored at −80 °C for further analysis.</p>
<p>The viral titer was determined using the Reed–Muench method [<a href="#B80-ijms-26-01934" class="usa-link" aria-describedby="B80-ijms-26-01934">80</a>]. This involved preparing serial 1:10 dilutions of the viral supernatant, which were then used to infect a monolayer of MDCK cells in 96-well plates. After 48 h, the cells were observed for cytopathic effects, and the endpoint dilution, which indicates the concentration at which 50% of the wells show viral-induced cell death, was used to calculate the viral titer.</p>
<p>RNA was extracted from the viral supernatant using TRIsure reagent (Bioline<sup>®</sup>, Memphis, TN, USA), which effectively lyses the virus and stabilizes RNA. The RNA was then purified through phase separation with chloroform and precipitated with isopropanol. The RNA pellet was washed with ethanol, dried, and resuspended in nuclease-free water. For cDNA synthesis, the purified RNA was first denatured by heating to 70 °C to remove secondary structures. Then, using M-MLV reverse transcriptase (Promega<sup>®</sup>), cDNA was synthesized in a reaction containing specific primers for the influenza A matrix protein gene, ensuring accurate and efficient reverse transcription.</p>
<p>The correct insertion of the influenza A matrix protein gene into the pGEM-T Easy vector was verified by digesting the recombinant plasmid with the <em>Eco</em>RI-HF restriction enzyme. The digestion products were analyzed by agarose gel electrophoresis to confirm the presence and size of the insert.</p>
<p>For absolute quantification of viral RNA by RT-qPCR, a calibration curve was generated using serial dilutions of the recombinant pGEM-T Easy/Mseg vector. The RT-qPCR was performed using the Brilliant III Ultra-Fast QRT-PCR Master Mix (Agilent Technologies<sup>®</sup>, Santa Clara, CA, USA) with specific TaqMan probes designed to target the matrix protein gene. The amplification conditions included reverse transcription at 55 °C for 10 min, followed by an initial denaturation at 95 °C and 40 cycles of denaturation and annealing/extension. The viral load in the samples was calculated based on the calibration curve derived from known quantities of the recombinant vector.</p>
<p>Finally, DNA and RNA quantification was performed using a TECAN Infinite M200 Pro plate reader with a NanoQuant plate, which allows for precise measurements even with small sample volumes. The purity of the nucleic acids was assessed by the absorbance ratios at 260/280 nm and 260/230 nm, ensuring the integrity and quality of the samples for downstream applications.</p></section><section id="sec4dot10-ijms-26-01934"><h3 class="pmc_sec_title">4.10. Evaluation of Virucidal Activity, Cytotoxicity, and Hemagglutination Effects of Gold Nanostructures (AuNS)</h3>
<p>A hemagglutination assay was performed to determine the hemagglutination titer (HA) of influenza A H1N1 virus using a 96-well U-bottom plate. Serial dilutions of the viral supernatant were mixed with 0.5% human erythrocytes, and the HA titer was defined as the highest dilution showing complete agglutination. To assess the effect of gold nanostructures on hemagglutination, the virus was treated with 100 µg/mL AuNS at various incubation times (0, 10, and 60 min), and the same assay was conducted. The 0 min incubation sample serves as a control to establish the initial viral titer before treatment. Untreated viral supernatant served as a positive control, while 1X PBS and 100 µg/mL AuNS solutions were used as negative controls.</p>
<p>The virucidal activity of biosynthesized AuNS was evaluated both in suspension and on non-porous stainless-steel surfaces following the guidelines of the European Committee for Standardization [<a href="#B17-ijms-26-01934" class="usa-link" aria-describedby="B17-ijms-26-01934">17</a>,<a href="#B18-ijms-26-01934" class="usa-link" aria-describedby="B18-ijms-26-01934">18</a>]. The influenza A H1N1 virus (10 × 10<sup>6</sup> TCID<sub>50</sub>/mL) was incubated with 100 µg/mL of AuNS, and viral titers were measured after 10 and 60 min of incubation at 25 °C with shaking. For surface tests, stainless steel disks were used, whereby the virus was adsorbed onto the disks before adding AuNS. After the incubation period (0, 10, and 60 min), samples were processed, and viral titers were determined using the Reed–Muench method [<a href="#B80-ijms-26-01934" class="usa-link" aria-describedby="B80-ijms-26-01934">80</a>]. Controls included standardized-hardness water (WSH), a 1:600 dilution of the peroxide-based disinfectant Virkon<sup>®</sup> as a positive control, and a commercial copper nanostructure biocidal control with a size &lt; 10 nm.</p>
<p>The cytotoxicity of AuNS on MDCK cells was assessed using the MTT assay (3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyl tetrazolium bromide) (Calbiochem<sup>®</sup>, Darmstadt, Germany) as described by Xiang et al. (2011) [<a href="#B81-ijms-26-01934" class="usa-link" aria-describedby="B81-ijms-26-01934">81</a>]. Cells were treated with AuNS concentrations ranging from 0 to 800 μg/mL, and cell viability was measured spectrophotometrically at 550 nm after a 24 h incubation period. Additionally, the IC<sub>50</sub> of AuNS against influenza A H1N1 was determined by incubating various concentrations of AuNS with the virus, followed by viral titer measurement using the Reed–Muench method [<a href="#B80-ijms-26-01934" class="usa-link" aria-describedby="B80-ijms-26-01934">80</a>].</p></section><section id="sec4dot11-ijms-26-01934"><h3 class="pmc_sec_title">4.11. Evaluation of Virion Protein Integrity</h3>
<p>The integrity of influenza A H1N1 virion proteins was assessed following the protocol by Fujimori et al. (2012) [<a href="#B68-ijms-26-01934" class="usa-link" aria-describedby="B68-ijms-26-01934">68</a>]. A mixture of the virus with 100 µg/mL of gold nanostructures was incubated for 60 min at 25 °C with shaking. As a control, AuNS were replaced with standardized-hardness water. After incubation, the mixtures were centrifuged, and the supernatants were quantified by the Bradford method. Proteins were then subjected to SDS-PAGE, followed by silver staining to evaluate protein integrity, and the remaining supernatant was analyzed by transmission electron microscopy (TEM).</p></section><section id="sec4dot12-ijms-26-01934"><h3 class="pmc_sec_title">4.12. Data Graphing and Statistical Analysis</h3>
<p>Data graphing and statistical analysis were performed using GraphPad Prism 8.00 software (GraphPad Software Inc., La Jolla, CA, USA). The <em>t</em>-test and analysis of variance (ANOVA) were performed with a significance level of <em>p</em> &lt; 0.05. Statistical significance was indicated as follows: * for values with <em>p</em> &lt; 0.05; ** for values with <em>p</em> &lt; 0.01; *** for values with <em>p</em> &lt; 0.001; **** for values with <em>p</em> &lt; 0.0001; and ns for non-significant differences.</p></section></section><section id="sec5-ijms-26-01934"><h2 class="pmc_sec_title">5. Conclusions</h2>
<p>This study provides valuable insights into the antiviral potential of biosynthesized gold nanostructures (AuNS) against influenza A H1N1 virus. The research successfully identified environmental bacterial isolates with significant resistance to HAuCl<sub>4</sub>, notably BNF01 and MF16, which also demonstrated a strong ability to reduce Au(III). Through in silico analysis, five proteins with potential Au(III) reductase activity were identified, with the recombinant Gor protein from BNF01 showing high reductive capacity and enabling the biosynthesis of AuNS.</p>
<p>The virucidal assays revealed that these biosynthesized AuNS effectively reduced viral titers in suspension assays, confirming their potential as antiviral agents. The observed reduction in viral titers, particularly in suspension assays, was accompanied by evidence of disruption to viral proteins, as demonstrated by hemagglutination assays, TEM imaging, and SDS-PAGE analysis. Despite these promising results, the AuNS did not show the same level of efficacy on non-porous stainless-steel surfaces, highlighting the need for further investigation into the conditions that influence their antiviral activity.</p>
<p>The findings suggest that the antiviral mechanism of AuNS likely involves the interaction with and degradation of viral surface proteins, impairing the virus’s ability to infect host cells. This study underscores the potential of AuNS as a novel antiviral material, but it also indicates that additional research is necessary to fully understand their mechanism of action and to optimize their use in different environments, particularly for surface disinfection.</p>
<p>In conclusion, the biosynthesized AuNS present a promising avenue for antiviral applications, particularly in suspension environments. Future studies should focus on refining the synthesis process, further characterizing the interaction of AuNS with viral components, and exploring their effectiveness against a broader range of viruses and in various practical settings.</p></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We are grateful to Fabián Araneda for carrying out the SEM–EDX and TEM measurements at CEDENNA (Universidad de Santiago de Chile), to Esteban Vargas for providing the commercial copper nanostructures with biocidal action used in this study as a control, and to all the members of the Molecular Microbiology Laboratory and the Molecular Virology and Pathogen Control Laboratory for their constant support and constructive ideas applied in this research.</p></section><section id="app1-ijms-26-01934" class="app"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<p>The supporting information can be downloaded at: <a href="https://www.mdpi.com/article/10.3390/ijms26051934/s1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.mdpi.com/article/10.3390/ijms26051934/s1</a>.</p>
<section class="sm xbox font-sm" id="ijms-26-01934-s001"><div class="media p"><div class="caption">
<a href="/articles/instance/11899802/bin/ijms-26-01934-s001.zip" data-ga-action="click_feat_suppl" class="usa-link">ijms-26-01934-s001.zip</a><sup> (954.5KB, zip) </sup>
</div></div></section></section><section id="notes1"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>M.A.-S. conducted a bioinformatics search for proteins with probable Au(III)-reducing activity. K.R. and N.N. helped in carrying out the experiments that gave rise to <a href="#ijms-26-01934-f002" class="usa-link">Figure 2</a>, specifically with the expression and purification of proteins with probable Au(III)-reducing activity. F.L. and R.S. performed the measurements of the gold content in the biosynthesized gold nanostructures by atomic absorption Spectroscopy. J.A.R.-P. participated in editing and correcting the manuscript. F.C. conducted the experiments in this study. F.C., F.A. and M.C.-S.M. conceived the study and participated in its design, coordination, scientific drafting, editing, and manuscript correction. All authors have read and agreed to the published version of the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Institutional Review Board Statement</h2>
<p>Ethical approval for this research was granted by the Institutional Ethics Committee (Report N° 329/2022) of the Universidad de Santiago de Chile on 13 October 2020.</p></section><section id="notes3"><h2 class="pmc_sec_title">Informed Consent Statement</h2>
<p>Not applicable.</p></section><section id="notes4"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>All data generated or analyzed during this study are included in this published article and the <a href="#app1-ijms-26-01934" class="usa-link">Supplementary Files</a>.</p></section><section id="notes5"><h2 class="pmc_sec_title">Conflicts of Interest</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>This project was supported by National Doctoral Scholarship 21200787 (2020–2024, ANID, Chile), a research support grant from the Vice Rectorate for Postgraduate Studies, Universidad de Santiago de Chile, a Grant to Support the Development of Multidisciplinary and Interdisciplinary Research Units and/or Theses: Project USA21991 (Chemistry and Biology Faculty, Universidad de Santiago de Chile), and FONDECYT Regular Project 1230724 (2023–2026, CONICYT, Chile).</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Disclaimer/Publisher’s Note:</strong> The statements, opinions and data contained in all publications are solely those of the individual author(s) and contributor(s) and not of MDPI and/or the editor(s). MDPI and/or the editor(s) disclaim responsibility for any injury to people or property resulting from any ideas, methods, instructions or products referred to in the content.</p></div></div></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="B1-ijms-26-01934">
<span class="label">1.</span><cite>Sawut R., Kasim N., Maihemuti B., Hu L., Abliz A., Abdujappar A., Kurban M. Pollution Characteristics and Health Risk Assessment of Heavy Metals in the Vegetable Bases of Northwest China. Sci. Total Environ. 2018;642:864–878. doi: 10.1016/j.scitotenv.2018.06.034.</cite> [<a href="https://doi.org/10.1016/j.scitotenv.2018.06.034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29925057/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Total%20Environ.&amp;title=Pollution%20Characteristics%20and%20Health%20Risk%20Assessment%20of%20Heavy%20Metals%20in%20the%20Vegetable%20Bases%20of%20Northwest%20China&amp;author=R.%20Sawut&amp;author=N.%20Kasim&amp;author=B.%20Maihemuti&amp;author=L.%20Hu&amp;author=A.%20Abliz&amp;volume=642&amp;publication_year=2018&amp;pages=864-878&amp;pmid=29925057&amp;doi=10.1016/j.scitotenv.2018.06.034&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B2-ijms-26-01934">
<span class="label">2.</span><cite>Igiri B.E., Okoduwa S.I.R., Idoko G.O., Akabuogu E.P., Adeyi A.O., Ejiogu I.K. Toxicity and Bioremediation of Heavy Metals Contaminated Ecosystem from Tannery Wastewater: A Review. J. Toxicol. 2018;2018:2568038. doi: 10.1155/2018/2568038.</cite> [<a href="https://doi.org/10.1155/2018/2568038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6180975/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30363677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Toxicol.&amp;title=Toxicity%20and%20Bioremediation%20of%20Heavy%20Metals%20Contaminated%20Ecosystem%20from%20Tannery%20Wastewater:%20A%20Review&amp;author=B.E.%20Igiri&amp;author=S.I.R.%20Okoduwa&amp;author=G.O.%20Idoko&amp;author=E.P.%20Akabuogu&amp;author=A.O.%20Adeyi&amp;volume=2018&amp;publication_year=2018&amp;pages=2568038&amp;pmid=30363677&amp;doi=10.1155/2018/2568038&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B3-ijms-26-01934">
<span class="label">3.</span><cite>Epelde L., Lanzén A., Blanco F., Urich T., Garbisu C. Adaptation of Soil Microbial Community Structure and Function to Chronic Metal Contamination at an Abandoned Pb-Zn Mine. FEMS Microbiol. Ecol. 2015;91:1–11. doi: 10.1093/femsec/fiu007.</cite> [<a href="https://doi.org/10.1093/femsec/fiu007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25764532/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol.%20Ecol.&amp;title=Adaptation%20of%20Soil%20Microbial%20Community%20Structure%20and%20Function%20to%20Chronic%20Metal%20Contamination%20at%20an%20Abandoned%20Pb-Zn%20Mine&amp;author=L.%20Epelde&amp;author=A.%20Lanz%C3%A9n&amp;author=F.%20Blanco&amp;author=T.%20Urich&amp;author=C.%20Garbisu&amp;volume=91&amp;publication_year=2015&amp;pages=1-11&amp;pmid=25764532&amp;doi=10.1093/femsec/fiu007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B4-ijms-26-01934">
<span class="label">4.</span><cite>Chandrangsu P., Rensing C., Helmann J.D. Metal Homeostasis and Resistance in Bacteria. Nat. Rev. Microbiol. 2017;15:338–350. doi: 10.1038/nrmicro.2017.15.</cite> [<a href="https://doi.org/10.1038/nrmicro.2017.15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5963929/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28344348/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Microbiol.&amp;title=Metal%20Homeostasis%20and%20Resistance%20in%20Bacteria&amp;author=P.%20Chandrangsu&amp;author=C.%20Rensing&amp;author=J.D.%20Helmann&amp;volume=15&amp;publication_year=2017&amp;pages=338-350&amp;pmid=28344348&amp;doi=10.1038/nrmicro.2017.15&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B5-ijms-26-01934">
<span class="label">5.</span><cite>Zhao X., Drlica K. Reactive Oxygen Species and the Bacterial Response to Lethal Stress. Curr. Opin. Microbiol. 2014;21:1–6. doi: 10.1016/j.mib.2014.06.008.</cite> [<a href="https://doi.org/10.1016/j.mib.2014.06.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4254325/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25078317/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Microbiol.&amp;title=Reactive%20Oxygen%20Species%20and%20the%20Bacterial%20Response%20to%20Lethal%20Stress&amp;author=X.%20Zhao&amp;author=K.%20Drlica&amp;volume=21&amp;publication_year=2014&amp;pages=1-6&amp;pmid=25078317&amp;doi=10.1016/j.mib.2014.06.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B6-ijms-26-01934">
<span class="label">6.</span><cite>Phaniendra A., Jestadi D.B., Periyasamy L. Free Radicals: Properties, Sources, Targets, and Their Implication in Various Diseases. Indian J. Clin. Biochem. 2015;30:11–26. doi: 10.1007/s12291-014-0446-0.</cite> [<a href="https://doi.org/10.1007/s12291-014-0446-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4310837/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25646037/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Indian%20J.%20Clin.%20Biochem.&amp;title=Free%20Radicals:%20Properties,%20Sources,%20Targets,%20and%20Their%20Implication%20in%20Various%20Diseases&amp;author=A.%20Phaniendra&amp;author=D.B.%20Jestadi&amp;author=L.%20Periyasamy&amp;volume=30&amp;publication_year=2015&amp;pages=11-26&amp;pmid=25646037&amp;doi=10.1007/s12291-014-0446-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B7-ijms-26-01934">
<span class="label">7.</span><cite>Turner R.J., Huang L.-N., Viti C., Mengoni A. Metal-Resistance in Bacteria: Why Care? Genes. 2020;11:1470.  doi: 10.3390/genes11121470.</cite> [<a href="https://doi.org/10.3390/genes11121470" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7764034/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33302493/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes&amp;title=Metal-Resistance%20in%20Bacteria:%20Why%20Care?&amp;author=R.J.%20Turner&amp;author=L.-N.%20Huang&amp;author=C.%20Viti&amp;author=A.%20Mengoni&amp;volume=11&amp;publication_year=2020&amp;pages=1470&amp;pmid=33302493&amp;doi=10.3390/genes11121470&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B8-ijms-26-01934">
<span class="label">8.</span><cite>Vignesh V., Sathiyanarayanan G., Sathishkumar G., Parthiban K., Sathish-Kumar K., Thirumurugan R. Formulation of Iron Oxide Nanoparticles Using Exopolysaccharide: Evaluation of Their Antibacterial and Anticancer Activities. RSC Adv. 2015;5:27794–27804. doi: 10.1039/C5RA03134F.</cite> [<a href="https://doi.org/10.1039/C5RA03134F" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=RSC%20Adv.&amp;title=Formulation%20of%20Iron%20Oxide%20Nanoparticles%20Using%20Exopolysaccharide:%20Evaluation%20of%20Their%20Antibacterial%20and%20Anticancer%20Activities&amp;author=V.%20Vignesh&amp;author=G.%20Sathiyanarayanan&amp;author=G.%20Sathishkumar&amp;author=K.%20Parthiban&amp;author=K.%20Sathish-Kumar&amp;volume=5&amp;publication_year=2015&amp;pages=27794-27804&amp;doi=10.1039/C5RA03134F&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B9-ijms-26-01934">
<span class="label">9.</span><cite>Salunke B.K., Sawant S.S., Lee S.-I., Kim B.S. Microorganisms as Efficient Biosystem for the Synthesis of Metal Nanoparticles: Current Scenario and Future Possibilities. World J. Microbiol. Biotechnol. 2016;32:88.  doi: 10.1007/s11274-016-2044-1.</cite> [<a href="https://doi.org/10.1007/s11274-016-2044-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27038958/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=World%20J.%20Microbiol.%20Biotechnol.&amp;title=Microorganisms%20as%20Efficient%20Biosystem%20for%20the%20Synthesis%20of%20Metal%20Nanoparticles:%20Current%20Scenario%20and%20Future%20Possibilities&amp;author=B.K.%20Salunke&amp;author=S.S.%20Sawant&amp;author=S.-I.%20Lee&amp;author=B.S.%20Kim&amp;volume=32&amp;publication_year=2016&amp;pages=88&amp;pmid=27038958&amp;doi=10.1007/s11274-016-2044-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B10-ijms-26-01934">
<span class="label">10.</span><cite>Ovais M., Khalil A.T., Ayaz M., Ahmad I., Nethi S.K., Mukherjee S. Biosynthesis of Metal Nanoparticles via Microbial Enzymes: A Mechanistic Approach. Int. J. Mol. Sci. 2018;19:4100.  doi: 10.3390/ijms19124100.</cite> [<a href="https://doi.org/10.3390/ijms19124100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6321641/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30567324/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=Biosynthesis%20of%20Metal%20Nanoparticles%20via%20Microbial%20Enzymes:%20A%20Mechanistic%20Approach&amp;author=M.%20Ovais&amp;author=A.T.%20Khalil&amp;author=M.%20Ayaz&amp;author=I.%20Ahmad&amp;author=S.K.%20Nethi&amp;volume=19&amp;publication_year=2018&amp;pages=4100&amp;pmid=30567324&amp;doi=10.3390/ijms19124100&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B11-ijms-26-01934">
<span class="label">11.</span><cite>Singh P., Kim Y.-J., Zhang D., Yang D.-C. Biological Synthesis of Nanoparticles from Plants and Microorganisms. Trends Biotechnol. 2016;34:588–599. doi: 10.1016/j.tibtech.2016.02.006.</cite> [<a href="https://doi.org/10.1016/j.tibtech.2016.02.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26944794/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Biotechnol.&amp;title=Biological%20Synthesis%20of%20Nanoparticles%20from%20Plants%20and%20Microorganisms&amp;author=P.%20Singh&amp;author=Y.-J.%20Kim&amp;author=D.%20Zhang&amp;author=D.-C.%20Yang&amp;volume=34&amp;publication_year=2016&amp;pages=588-599&amp;pmid=26944794&amp;doi=10.1016/j.tibtech.2016.02.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B12-ijms-26-01934">
<span class="label">12.</span><cite>Saravanan A., Kumar P.S., Karishma S., Vo D.-V.N., Jeevanantham S., Yaashikaa P.R., George C.S. A Review on Biosynthesis of Metal Nanoparticles and Its Environmental Applications. Chemosphere. 2021;264:128580. doi: 10.1016/j.chemosphere.2020.128580.</cite> [<a href="https://doi.org/10.1016/j.chemosphere.2020.128580" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33059285/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chemosphere&amp;title=A%20Review%20on%20Biosynthesis%20of%20Metal%20Nanoparticles%20and%20Its%20Environmental%20Applications&amp;author=A.%20Saravanan&amp;author=P.S.%20Kumar&amp;author=S.%20Karishma&amp;author=D.-V.N.%20Vo&amp;author=S.%20Jeevanantham&amp;volume=264&amp;publication_year=2021&amp;pages=128580&amp;pmid=33059285&amp;doi=10.1016/j.chemosphere.2020.128580&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B13-ijms-26-01934">
<span class="label">13.</span><cite>Sarkar P.K., Das Mukhopadhyay C. Ayurvedic Metal Nanoparticles Could Be Novel Antiviral Agents against SARS-CoV-2. Int. Nano Lett. 2021;11:197–203. doi: 10.1007/s40089-020-00323-9.</cite> [<a href="https://doi.org/10.1007/s40089-020-00323-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7786161/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33425283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20Nano%20Lett.&amp;title=Ayurvedic%20Metal%20Nanoparticles%20Could%20Be%20Novel%20Antiviral%20Agents%20against%20SARS-CoV-2&amp;author=P.K.%20Sarkar&amp;author=C.%20Das%20Mukhopadhyay&amp;volume=11&amp;publication_year=2021&amp;pages=197-203&amp;pmid=33425283&amp;doi=10.1007/s40089-020-00323-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B14-ijms-26-01934">
<span class="label">14.</span><cite>Gholami-Shabani M., Shams-Ghahfarokhi M., Gholami-Shabani Z., Akbarzadeh A., Riazi G., Ajdari S., Amani A., Razzaghi-Abyaneh M. Enzymatic Synthesis of Gold Nanoparticles Using Sulfite Reductase Purified from Escherichia coli: A Green Eco-Friendly Approach. Process Biochem. 2015;50:1076–1085. doi: 10.1016/j.procbio.2015.04.004.</cite> [<a href="https://doi.org/10.1016/j.procbio.2015.04.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Process%20Biochem.&amp;title=Enzymatic%20Synthesis%20of%20Gold%20Nanoparticles%20Using%20Sulfite%20Reductase%20Purified%20from%20Escherichia%20coli:%20A%20Green%20Eco-Friendly%20Approach&amp;author=M.%20Gholami-Shabani&amp;author=M.%20Shams-Ghahfarokhi&amp;author=Z.%20Gholami-Shabani&amp;author=A.%20Akbarzadeh&amp;author=G.%20Riazi&amp;volume=50&amp;publication_year=2015&amp;pages=1076-1085&amp;doi=10.1016/j.procbio.2015.04.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B15-ijms-26-01934">
<span class="label">15.</span><cite>Ribeiro B.D., Coelho M.A.Z., Machado De Castro A.  Principles of Green Chemistry and White Biotechnology. In: Coelho M.A., Ribeiro B.D., editors. White Biotechnology for Sustainable Chemistry. The Royal Society of Chemistry; London, UK: 2015. pp. 1–8.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=White%20Biotechnology%20for%20Sustainable%20Chemistry&amp;author=B.D.%20Ribeiro&amp;author=M.A.Z.%20Coelho&amp;author=A.%20Machado%20De%20Castro&amp;publication_year=2015&amp;isbn=978-1-84973-816-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B16-ijms-26-01934">
<span class="label">16.</span><cite>Figueroa M., Fernandez V., Arenas-Salinas M., Ahumada D., Muñoz-Villagrán C., Cornejo F., Vargas E., Latorre M., Morales E., Vásquez C., et al.  Synthesis and Antibacterial Activity of Metal(Loid) Nanostructures by Environmental Multi-Metal(Loid) Resistant Bacteria and Metal(Loid)-Reducing Flavoproteins. Front. Microbiol. 2018;9:959.  doi: 10.3389/fmicb.2018.00959.</cite> [<a href="https://doi.org/10.3389/fmicb.2018.00959" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5962736/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29869640/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&amp;title=Synthesis%20and%20Antibacterial%20Activity%20of%20Metal(Loid)%20Nanostructures%20by%20Environmental%20Multi-Metal(Loid)%20Resistant%20Bacteria%20and%20Metal(Loid)-Reducing%20Flavoproteins&amp;author=M.%20Figueroa&amp;author=V.%20Fernandez&amp;author=M.%20Arenas-Salinas&amp;author=D.%20Ahumada&amp;author=C.%20Mu%C3%B1oz-Villagr%C3%A1n&amp;volume=9&amp;publication_year=2018&amp;pages=959&amp;pmid=29869640&amp;doi=10.3389/fmicb.2018.00959&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B17-ijms-26-01934">
<span class="label">17.</span><cite>Chemical Disinfectants and Antiseptics—Quantitative Suspension Test for the Evaluation of Viricidal Activity in the Medical Area—Test Method and Requirements (Phase 2/Step 1) European Committee for Standardization; Brussels, Belgium: 2019.  [(accessed on 15 January 2025)].  Available online:  <a href="https://standards.iteh.ai/catalog/standards/sist/63ff3e17-5d89-42c1-9990-a023a13e1f3c/sist-en-14476-2013a2-2019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://standards.iteh.ai/catalog/standards/sist/63ff3e17-5d89-42c1-9990-a023a13e1f3c/sist-en-14476-2013a2-2019</a>.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Chemical%20Disinfectants%20and%20Antiseptics%E2%80%94Quantitative%20Suspension%20Test%20for%20the%20Evaluation%20of%20Viricidal%20Activity%20in%20the%20Medical%20Area%E2%80%94Test%20Method%20and%20Requirements%20(Phase%202/Step%201)&amp;publication_year=2019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B18-ijms-26-01934">
<span class="label">18.</span><cite>Chemical Disinfectants and Antiseptics—Quantitative Non-Porous Surface Test Without Mechanical Action for the Evaluation of Viricidal Activity of Chemical Disinfectants Used in the Medical Area—Test Method and Requirements (Phase 2/Step 2) European Committee for Standardization; Brussels, Belgium: 2019.  [(accessed on 15 January 2025)].  Available online:  <a href="https://standards.iteh.ai/catalog/standards/sist/2890f2c1-cadd-41dc-bdac-591d4559e2f5/sist-en-16777-2019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://standards.iteh.ai/catalog/standards/sist/2890f2c1-cadd-41dc-bdac-591d4559e2f5/sist-en-16777-2019</a>.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Chemical%20Disinfectants%20and%20Antiseptics%E2%80%94Quantitative%20Non-Porous%20Surface%20Test%20Without%20Mechanical%20Action%20for%20the%20Evaluation%20of%20Viricidal%20Activity%20of%20Chemical%20Disinfectants%20Used%20in%20the%20Medical%20Area%E2%80%94Test%20Method%20and%20Requirements%20(Phase%202/Step%202)&amp;publication_year=2019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B19-ijms-26-01934">
<span class="label">19.</span><cite>Kowalska-Krochmal B., Dudek-Wicher R. The Minimum Inhibitory Concentration of Antibiotics: Methods, Interpretation, Clinical Relevance. Pathogens. 2021;10:165.  doi: 10.3390/pathogens10020165.</cite> [<a href="https://doi.org/10.3390/pathogens10020165" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7913839/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33557078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pathogens&amp;title=The%20Minimum%20Inhibitory%20Concentration%20of%20Antibiotics:%20Methods,%20Interpretation,%20Clinical%20Relevance&amp;author=B.%20Kowalska-Krochmal&amp;author=R.%20Dudek-Wicher&amp;volume=10&amp;publication_year=2021&amp;pages=165&amp;pmid=33557078&amp;doi=10.3390/pathogens10020165&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B20-ijms-26-01934">
<span class="label">20.</span><cite>Muñoz-Villagrán C., Contreras F., Cornejo F., Figueroa M., Valenzuela-Bezanilla D., Luraschi R., Reinoso C., Rivas-Pardo J., Vásquez C., Castro M., et al.  Understanding Gold Toxicity in Aerobically-Grown Escherichia coli. Biol. Res. 2020;53:26.  doi: 10.1186/s40659-020-00292-5.</cite> [<a href="https://doi.org/10.1186/s40659-020-00292-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7278051/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32513271/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biol.%20Res.&amp;title=Understanding%20Gold%20Toxicity%20in%20Aerobically-Grown%20Escherichia%20coli&amp;author=C.%20Mu%C3%B1oz-Villagr%C3%A1n&amp;author=F.%20Contreras&amp;author=F.%20Cornejo&amp;author=M.%20Figueroa&amp;author=D.%20Valenzuela-Bezanilla&amp;volume=53&amp;publication_year=2020&amp;pages=26&amp;pmid=32513271&amp;doi=10.1186/s40659-020-00292-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B21-ijms-26-01934">
<span class="label">21.</span><cite>Sykes J.E., Rankin S.C.  Canine and Feline Infectious Diseases. Elsevier; Amsterdam, The Netherlands: 2014. Isolation and Identification of Aerobic and Anaerobic Bacteria; pp. 17–28.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Canine%20and%20Feline%20Infectious%20Diseases&amp;author=J.E.%20Sykes&amp;author=S.C.%20Rankin&amp;publication_year=2014&amp;isbn=978-1-4377-0795-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B22-ijms-26-01934">
<span class="label">22.</span><cite>Orizola J., Ríos-Silva M., Muñoz-Villagrán C., Vargas E., Vásquez C., Arenas F. In Vitro Biosynthesis of Ag, Au and Te-Containing Nanostructures by Exiguobacterium Cell-Free Extracts. BMC Biotechnol. 2020;20:29.  doi: 10.1186/s12896-020-00625-y.</cite> [<a href="https://doi.org/10.1186/s12896-020-00625-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7260758/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32471409/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Biotechnol.&amp;title=In%20Vitro%20Biosynthesis%20of%20Ag,%20Au%20and%20Te-Containing%20Nanostructures%20by%20Exiguobacterium%20Cell-Free%20Extracts&amp;author=J.%20Orizola&amp;author=M.%20R%C3%ADos-Silva&amp;author=C.%20Mu%C3%B1oz-Villagr%C3%A1n&amp;author=E.%20Vargas&amp;author=C.%20V%C3%A1squez&amp;volume=20&amp;publication_year=2020&amp;pages=29&amp;pmid=32471409&amp;doi=10.1186/s12896-020-00625-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B23-ijms-26-01934">
<span class="label">23.</span><cite>Grabski A.C.  Methods in Enzymology. Volume 463. Elsevier; Amsterdam, The Netherlands: 2009. Chapter 18 Advances in Preparation of Biological Extracts for Protein Purification; pp. 285–303.</cite> [<a href="https://doi.org/10.1016/S0076-6879(09)63018-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19892178/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Methods%20in%20Enzymology&amp;author=A.C.%20Grabski&amp;publication_year=2009&amp;isbn=978-0-12-374536-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B24-ijms-26-01934">
<span class="label">24.</span><cite>Panda T., Deepa K. Biosynthesis of Gold Nanoparticles. J. Nanosci. Nanotechnol. 2011;11:10279–10294. doi: 10.1166/jnn.2011.5021.</cite> [<a href="https://doi.org/10.1166/jnn.2011.5021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22408900/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Nanosci.%20Nanotechnol.&amp;title=Biosynthesis%20of%20Gold%20Nanoparticles&amp;author=T.%20Panda&amp;author=K.%20Deepa&amp;volume=11&amp;publication_year=2011&amp;pages=10279-10294&amp;pmid=22408900&amp;doi=10.1166/jnn.2011.5021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B25-ijms-26-01934">
<span class="label">25.</span><cite>Liu H., Zhang M., Meng F., Su C., Li J. Polysaccharide-Based Gold Nanomaterials: Synthesis Mechanism, Polysaccharide Structure-Effect, and Anticancer Activity. Carbohydr. Polym. 2023;321:121284. doi: 10.1016/j.carbpol.2023.121284.</cite> [<a href="https://doi.org/10.1016/j.carbpol.2023.121284" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37739497/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Carbohydr.%20Polym.&amp;title=Polysaccharide-Based%20Gold%20Nanomaterials:%20Synthesis%20Mechanism,%20Polysaccharide%20Structure-Effect,%20and%20Anticancer%20Activity&amp;author=H.%20Liu&amp;author=M.%20Zhang&amp;author=F.%20Meng&amp;author=C.%20Su&amp;author=J.%20Li&amp;volume=321&amp;publication_year=2023&amp;pages=121284&amp;pmid=37739497&amp;doi=10.1016/j.carbpol.2023.121284&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B26-ijms-26-01934">
<span class="label">26.</span><cite>Rigobello M.P., Scutari G., Folda A., Bindoli A. Mitochondrial Thioredoxin Reductase Inhibition by Gold(I) Compounds and Concurrent Stimulation of Permeability Transition and Release of Cytochrome c. Biochem. Pharmacol. 2004;67:689–696. doi: 10.1016/j.bcp.2003.09.038.</cite> [<a href="https://doi.org/10.1016/j.bcp.2003.09.038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14757168/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20Pharmacol.&amp;title=Mitochondrial%20Thioredoxin%20Reductase%20Inhibition%20by%20Gold(I)%20Compounds%20and%20Concurrent%20Stimulation%20of%20Permeability%20Transition%20and%20Release%20of%20Cytochrome%20c&amp;author=M.P.%20Rigobello&amp;author=G.%20Scutari&amp;author=A.%20Folda&amp;author=A.%20Bindoli&amp;volume=67&amp;publication_year=2004&amp;pages=689-696&amp;pmid=14757168&amp;doi=10.1016/j.bcp.2003.09.038&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B27-ijms-26-01934">
<span class="label">27.</span><cite>Lienkamp A.C., Heine T., Tischler D.  Advances in Applied Microbiology. Volume 110. Elsevier; Amsterdam, The Netherlands: 2020. Glutathione: A Powerful but Rare Cofactor among Actinobacteria; pp. 181–217.</cite> [<a href="https://doi.org/10.1016/bs.aambs.2019.12.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32386605/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Advances%20in%20Applied%20Microbiology&amp;author=A.C.%20Lienkamp&amp;author=T.%20Heine&amp;author=D.%20Tischler&amp;publication_year=2020&amp;isbn=978-0-12-820703-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B28-ijms-26-01934">
<span class="label">28.</span><cite>Urbonavicius J. Identification of a Novel Gene Encoding a Flavin-Dependent tRNA:m5U Methyltransferase in Bacteria—Evolutionary Implications. Nucleic Acids Res. 2005;33:3955–3964. doi: 10.1093/nar/gki703.</cite> [<a href="https://doi.org/10.1093/nar/gki703" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1178002/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16027442/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=Identification%20of%20a%20Novel%20Gene%20Encoding%20a%20Flavin-Dependent%20tRNA:m5U%20Methyltransferase%20in%20Bacteria%E2%80%94Evolutionary%20Implications&amp;author=J.%20Urbonavicius&amp;volume=33&amp;publication_year=2005&amp;pages=3955-3964&amp;pmid=16027442&amp;doi=10.1093/nar/gki703&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B29-ijms-26-01934">
<span class="label">29.</span><cite>Arenas-Salinas M., Vargas-Pérez J.I., Morales W., Pinto C., Muñoz-Díaz P., Cornejo F.A., Pugin B., Sandoval J.M., Díaz-Vásquez W.A., Muñoz-Villagrán C., et al.  Flavoprotein-Mediated Tellurite Reduction: Structural Basis and Applications to the Synthesis of Tellurium-Containing Nanostructures. Front. Microbiol. 2016;7:1160.  doi: 10.3389/fmicb.2016.01160.</cite> [<a href="https://doi.org/10.3389/fmicb.2016.01160" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4960239/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27507969/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&amp;title=Flavoprotein-Mediated%20Tellurite%20Reduction:%20Structural%20Basis%20and%20Applications%20to%20the%20Synthesis%20of%20Tellurium-Containing%20Nanostructures&amp;author=M.%20Arenas-Salinas&amp;author=J.I.%20Vargas-P%C3%A9rez&amp;author=W.%20Morales&amp;author=C.%20Pinto&amp;author=P.%20Mu%C3%B1oz-D%C3%ADaz&amp;volume=7&amp;publication_year=2016&amp;pages=1160&amp;pmid=27507969&amp;doi=10.3389/fmicb.2016.01160&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B30-ijms-26-01934">
<span class="label">30.</span><cite>Raghavan S., Baskin D.S., Sharpe M.A.  Encyclopedia of Biological Chemistry III. Elsevier; Amsterdam, The Netherlands: 2021. Regulation|Monoamine Oxidases; pp. 542–560.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Encyclopedia%20of%20Biological%20Chemistry%20III&amp;author=S.%20Raghavan&amp;author=D.S.%20Baskin&amp;author=M.A.%20Sharpe&amp;publication_year=2021&amp;isbn=978-0-12-822040-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B31-ijms-26-01934">
<span class="label">31.</span><cite>Scott D., Toney M., Muzikár M. Harnessing the Mechanism of Glutathione Reductase for Synthesis of Active Site Bound Metallic Nanoparticles and Electrical Connection to Electrodes. J. Am. Chem. Soc. 2008;130:865–874. doi: 10.1021/ja074660g.</cite> [<a href="https://doi.org/10.1021/ja074660g" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18166048/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=Harnessing%20the%20Mechanism%20of%20Glutathione%20Reductase%20for%20Synthesis%20of%20Active%20Site%20Bound%20Metallic%20Nanoparticles%20and%20Electrical%20Connection%20to%20Electrodes&amp;author=D.%20Scott&amp;author=M.%20Toney&amp;author=M.%20Muzik%C3%A1r&amp;volume=130&amp;publication_year=2008&amp;pages=865-874&amp;pmid=18166048&amp;doi=10.1021/ja074660g&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B32-ijms-26-01934">
<span class="label">32.</span><cite>El-Yazbi A.F., Loppnow G.R. Detecting UV-Induced Nucleic-Acid Damage. TrAC Trends Anal. Chem. 2014;61:83–91. doi: 10.1016/j.trac.2014.05.010.</cite> [<a href="https://doi.org/10.1016/j.trac.2014.05.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=TrAC%20Trends%20Anal.%20Chem.&amp;title=Detecting%20UV-Induced%20Nucleic-Acid%20Damage&amp;author=A.F.%20El-Yazbi&amp;author=G.R.%20Loppnow&amp;volume=61&amp;publication_year=2014&amp;pages=83-91&amp;doi=10.1016/j.trac.2014.05.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B33-ijms-26-01934">
<span class="label">33.</span><cite>Kane J.F. Effects of Rare Codon Clusters on High-Level Expression of Heterologous Proteins in Escherichia coli. Curr. Opin. Biotechnol. 1995;6:494–500. doi: 10.1016/0958-1669(95)80082-4.</cite> [<a href="https://doi.org/10.1016/0958-1669(95)80082-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7579660/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Biotechnol.&amp;title=Effects%20of%20Rare%20Codon%20Clusters%20on%20High-Level%20Expression%20of%20Heterologous%20Proteins%20in%20Escherichia%20coli&amp;author=J.F.%20Kane&amp;volume=6&amp;publication_year=1995&amp;pages=494-500&amp;pmid=7579660&amp;doi=10.1016/0958-1669(95)80082-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B34-ijms-26-01934">
<span class="label">34.</span><cite>Gu W., Zhou T., Wilke C.O. A Universal Trend of Reduced mRNA Stability near the Translation-Initiation Site in Prokaryotes and Eukaryotes. PLoS Comput. Biol. 2010;6:e1000664.  doi: 10.1371/journal.pcbi.1000664.</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1000664" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2816680/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20140241/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput.%20Biol.&amp;title=A%20Universal%20Trend%20of%20Reduced%20mRNA%20Stability%20near%20the%20Translation-Initiation%20Site%20in%20Prokaryotes%20and%20Eukaryotes&amp;author=W.%20Gu&amp;author=T.%20Zhou&amp;author=C.O.%20Wilke&amp;volume=6&amp;publication_year=2010&amp;pages=e1000664&amp;pmid=20140241&amp;doi=10.1371/journal.pcbi.1000664&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B35-ijms-26-01934">
<span class="label">35.</span><cite>Wadhwani S.A., Shedbalkar U.U., Singh R., Chopade B.A. Biosynthesis of Gold and Selenium Nanoparticles by Purified Protein from Acinetobacter Sp. SW 30. Enzym. Microb. Technol. 2018;111:81–86. doi: 10.1016/j.enzmictec.2017.10.007.</cite> [<a href="https://doi.org/10.1016/j.enzmictec.2017.10.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29421042/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Enzym.%20Microb.%20Technol.&amp;title=Biosynthesis%20of%20Gold%20and%20Selenium%20Nanoparticles%20by%20Purified%20Protein%20from%20Acinetobacter%20Sp.%20SW%2030&amp;author=S.A.%20Wadhwani&amp;author=U.U.%20Shedbalkar&amp;author=R.%20Singh&amp;author=B.A.%20Chopade&amp;volume=111&amp;publication_year=2018&amp;pages=81-86&amp;pmid=29421042&amp;doi=10.1016/j.enzmictec.2017.10.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B36-ijms-26-01934">
<span class="label">36.</span><cite>Vlamis-Gardikas A. The Multiple Functions of the Thiol-Based Electron Flow Pathways of Escherichia coli: Eternal Concepts Revisited. Biochim. Et Biophys. Acta (BBA)-Gen. Subj. 2008;1780:1170–1200. doi: 10.1016/j.bbagen.2008.03.013.</cite> [<a href="https://doi.org/10.1016/j.bbagen.2008.03.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18423382/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim.%20Et%20Biophys.%20Acta%20(BBA)-Gen.%20Subj.&amp;title=The%20Multiple%20Functions%20of%20the%20Thiol-Based%20Electron%20Flow%20Pathways%20of%20Escherichia%20coli:%20Eternal%20Concepts%20Revisited&amp;author=A.%20Vlamis-Gardikas&amp;volume=1780&amp;publication_year=2008&amp;pages=1170-1200&amp;pmid=18423382&amp;doi=10.1016/j.bbagen.2008.03.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B37-ijms-26-01934">
<span class="label">37.</span><cite>Luty-Błocho M., Wojnicki M., Grzonka J., Kurzydłowski K.J., Fitzner K. Linking the Gold Nanoparticles Formation Kinetics with Their Morphology. Int. J. Chem. Kinet. 2018;50:204–214. doi: 10.1002/kin.21152.</cite> [<a href="https://doi.org/10.1002/kin.21152" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Chem.%20Kinet.&amp;title=Linking%20the%20Gold%20Nanoparticles%20Formation%20Kinetics%20with%20Their%20Morphology&amp;author=M.%20Luty-B%C5%82ocho&amp;author=M.%20Wojnicki&amp;author=J.%20Grzonka&amp;author=K.J.%20Kurzyd%C5%82owski&amp;author=K.%20Fitzner&amp;volume=50&amp;publication_year=2018&amp;pages=204-214&amp;doi=10.1002/kin.21152&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B38-ijms-26-01934">
<span class="label">38.</span><cite>Wang L., Yan L., Ye L., Chen J., Li Y., Zhang Q., Jing C. Identification and Characterization of a Au(III) Reductase from Erwinia Sp. IMH. JACS Au. 2022;2:1435–1442. doi: 10.1021/jacsau.2c00170.</cite> [<a href="https://doi.org/10.1021/jacsau.2c00170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9241155/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35783184/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=JACS%20Au&amp;title=Identification%20and%20Characterization%20of%20a%20Au(III)%20Reductase%20from%20Erwinia%20Sp.%20IMH&amp;author=L.%20Wang&amp;author=L.%20Yan&amp;author=L.%20Ye&amp;author=J.%20Chen&amp;author=Y.%20Li&amp;volume=2&amp;publication_year=2022&amp;pages=1435-1442&amp;pmid=35783184&amp;doi=10.1021/jacsau.2c00170&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B39-ijms-26-01934">
<span class="label">39.</span><cite>Hill S.J., Fisher A.S.  Encyclopedia of Spectroscopy and Spectrometry. Elsevier; Amsterdam, The Netherlands: 2017. Atomic Absorption, Methods and Instrumentation; pp. 37–43.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Encyclopedia%20of%20Spectroscopy%20and%20Spectrometry&amp;author=S.J.%20Hill&amp;author=A.S.%20Fisher&amp;publication_year=2017&amp;isbn=978-0-12-803224-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B40-ijms-26-01934">
<span class="label">40.</span><cite>Li J., Wu J., Zhang X., Liu Y., Zhou D., Sun H., Zhang H., Yang B. Controllable Synthesis of Stable Urchin-like Gold Nanoparticles Using Hydroquinone to Tune the Reactivity of Gold Chloride. J. Phys. Chem. C. 2011;115:3630–3637. doi: 10.1021/jp1119074.</cite> [<a href="https://doi.org/10.1021/jp1119074" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Phys.%20Chem.%20C&amp;title=Controllable%20Synthesis%20of%20Stable%20Urchin-like%20Gold%20Nanoparticles%20Using%20Hydroquinone%20to%20Tune%20the%20Reactivity%20of%20Gold%20Chloride&amp;author=J.%20Li&amp;author=J.%20Wu&amp;author=X.%20Zhang&amp;author=Y.%20Liu&amp;author=D.%20Zhou&amp;volume=115&amp;publication_year=2011&amp;pages=3630-3637&amp;doi=10.1021/jp1119074&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B41-ijms-26-01934">
<span class="label">41.</span><cite>Sharma D., Kanchi S., Bisetty K. Biogenic Synthesis of Nanoparticles: A Review. Arab. J. Chem. 2019;12:3576–3600. doi: 10.1016/j.arabjc.2015.11.002.</cite> [<a href="https://doi.org/10.1016/j.arabjc.2015.11.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Arab.%20J.%20Chem.&amp;title=Biogenic%20Synthesis%20of%20Nanoparticles:%20A%20Review&amp;author=D.%20Sharma&amp;author=S.%20Kanchi&amp;author=K.%20Bisetty&amp;volume=12&amp;publication_year=2019&amp;pages=3576-3600&amp;doi=10.1016/j.arabjc.2015.11.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B42-ijms-26-01934">
<span class="label">42.</span><cite>Alessio P., Aoki P.H.B., Furini L.N., Aliaga A.E., Leopoldo Constantino C.J.  Nanocharacterization Techniques. Elsevier; Amsterdam, The Netherlands: 2017. Spectroscopic Techniques for Characterization of Nanomaterials; pp. 65–98.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Nanocharacterization%20Techniques&amp;author=P.%20Alessio&amp;author=P.H.B.%20Aoki&amp;author=L.N.%20Furini&amp;author=A.E.%20Aliaga&amp;author=C.J.%20Leopoldo%20Constantino&amp;publication_year=2017&amp;isbn=978-0-323-49778-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B43-ijms-26-01934">
<span class="label">43.</span><cite>Clogston J.D., Patri A.K.  Zeta Potential Measurement. In: McNeil S.E., editor. Characterization of Nanoparticles Intended for Drug Delivery. Volume 697. Humana Press; Totowa, NJ, USA: 2011. pp. 63–70. Methods in Molecular Biology.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20Nanoparticles%20Intended%20for%20Drug%20Delivery&amp;author=J.D.%20Clogston&amp;author=A.K.%20Patri&amp;publication_year=2011&amp;isbn=978-1-60327-197-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B44-ijms-26-01934">
<span class="label">44.</span><cite>Kobayashi Y., Suzuki Y. Compensatory Evolution of Net-Charge in Influenza A Virus Hemagglutinin. PLoS ONE. 2012;7:e40422.  doi: 10.1371/journal.pone.0040422.</cite> [<a href="https://doi.org/10.1371/journal.pone.0040422" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3395715/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22808159/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Compensatory%20Evolution%20of%20Net-Charge%20in%20Influenza%20A%20Virus%20Hemagglutinin&amp;author=Y.%20Kobayashi&amp;author=Y.%20Suzuki&amp;volume=7&amp;publication_year=2012&amp;pages=e40422&amp;pmid=22808159&amp;doi=10.1371/journal.pone.0040422&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B45-ijms-26-01934">
<span class="label">45.</span><cite>Selvamani V.  Characterization and Biology of Nanomaterials for Drug Delivery. Elsevier; Amsterdam, The Netherlands: 2019. Stability Studies on Nanomaterials Used in Drugs; pp. 425–444.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Characterization%20and%20Biology%20of%20Nanomaterials%20for%20Drug%20Delivery&amp;author=V.%20Selvamani&amp;publication_year=2019&amp;isbn=978-0-12-814031-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B46-ijms-26-01934">
<span class="label">46.</span><cite>Zhai W., Zhang D.N., Mai C., Choy J., Jian G., Sra K., Galinski M.S. Comparison of Different Cell Substrates on the Measurement of Human Influenza Virus Neutralizing Antibodies. PLoS ONE. 2012;7:e52327.  doi: 10.1371/journal.pone.0052327.</cite> [<a href="https://doi.org/10.1371/journal.pone.0052327" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3527534/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23284988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Comparison%20of%20Different%20Cell%20Substrates%20on%20the%20Measurement%20of%20Human%20Influenza%20Virus%20Neutralizing%20Antibodies&amp;author=W.%20Zhai&amp;author=D.N.%20Zhang&amp;author=C.%20Mai&amp;author=J.%20Choy&amp;author=G.%20Jian&amp;volume=7&amp;publication_year=2012&amp;pages=e52327&amp;pmid=23284988&amp;doi=10.1371/journal.pone.0052327&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B47-ijms-26-01934">
<span class="label">47.</span><cite>Beyleveld G., White K.M., Ayllon J., Shaw M.L. New-Generation Screening Assays for the Detection of Anti-Influenza Compounds Targeting Viral and Host Functions. Antivir. Res. 2013;100:120–132. doi: 10.1016/j.antiviral.2013.07.018.</cite> [<a href="https://doi.org/10.1016/j.antiviral.2013.07.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3840122/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23933115/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antivir.%20Res.&amp;title=New-Generation%20Screening%20Assays%20for%20the%20Detection%20of%20Anti-Influenza%20Compounds%20Targeting%20Viral%20and%20Host%20Functions&amp;author=G.%20Beyleveld&amp;author=K.M.%20White&amp;author=J.%20Ayllon&amp;author=M.L.%20Shaw&amp;volume=100&amp;publication_year=2013&amp;pages=120-132&amp;pmid=23933115&amp;doi=10.1016/j.antiviral.2013.07.018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B48-ijms-26-01934">
<span class="label">48.</span><cite>Sharma V., Kaushik S. Comparative Analysis of Molecular Methods for Detection of Influenza Viruses. Br. Microbiol. Res. J. 2016;17:1–10. doi: 10.9734/BMRJ/2016/28858.</cite> [<a href="https://doi.org/10.9734/BMRJ/2016/28858" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Br.%20Microbiol.%20Res.%20J.&amp;title=Comparative%20Analysis%20of%20Molecular%20Methods%20for%20Detection%20of%20Influenza%20Viruses&amp;author=V.%20Sharma&amp;author=S.%20Kaushik&amp;volume=17&amp;publication_year=2016&amp;pages=1-10&amp;doi=10.9734/BMRJ/2016/28858&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B49-ijms-26-01934">
<span class="label">49.</span><cite>Killian M.L.  Hemagglutination Assay for Influenza Virus. In: Spackman E., editor. Animal Influenza Virus. Volume 2123. Springer; New York, NY, USA: 2020. pp. 3–10. Methods in Molecular Biology.</cite> [<a href="https://doi.org/10.1007/978-1-0716-0346-8_1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32170676/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Animal%20Influenza%20Virus&amp;author=M.L.%20Killian&amp;publication_year=2020&amp;isbn=978-1-0716-0345-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B50-ijms-26-01934">
<span class="label">50.</span><cite>Spackman E., Senne D.A., Myers T.J., Bulaga L.L., Garber L.P., Perdue M.L., Lohman K., Daum L.T., Suarez D.L. Development of a Real-Time Reverse Transcriptase PCR Assay for Type A Influenza Virus and the Avian H5 and H7 Hemagglutinin Subtypes. J. Clin. Microbiol. 2002;40:3256–3260. doi: 10.1128/JCM.40.9.3256-3260.2002.</cite> [<a href="https://doi.org/10.1128/JCM.40.9.3256-3260.2002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC130722/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12202562/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Microbiol.&amp;title=Development%20of%20a%20Real-Time%20Reverse%20Transcriptase%20PCR%20Assay%20for%20Type%20A%20Influenza%20Virus%20and%20the%20Avian%20H5%20and%20H7%20Hemagglutinin%20Subtypes&amp;author=E.%20Spackman&amp;author=D.A.%20Senne&amp;author=T.J.%20Myers&amp;author=L.L.%20Bulaga&amp;author=L.P.%20Garber&amp;volume=40&amp;publication_year=2002&amp;pages=3256-3260&amp;pmid=12202562&amp;doi=10.1128/JCM.40.9.3256-3260.2002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B51-ijms-26-01934">
<span class="label">51.</span><cite>Furuse Y., Suzuki A., Kamigaki T., Oshitani H. Evolution of the M Gene of the Influenza A Virus in Different Host Species: Large-Scale Sequence Analysis. Virol. J. 2009;6:67. doi: 10.1186/1743-422X-6-67.</cite> [<a href="https://doi.org/10.1186/1743-422X-6-67" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2694789/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19476650/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Virol.%20J.&amp;title=Evolution%20of%20the%20M%20Gene%20of%20the%20Influenza%20A%20Virus%20in%20Different%20Host%20Species:%20Large-Scale%20Sequence%20Analysis&amp;author=Y.%20Furuse&amp;author=A.%20Suzuki&amp;author=T.%20Kamigaki&amp;author=H.%20Oshitani&amp;volume=6&amp;publication_year=2009&amp;pages=67&amp;pmid=19476650&amp;doi=10.1186/1743-422X-6-67&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B52-ijms-26-01934">
<span class="label">52.</span><cite>Rabenau H.F., Schwebke I., Blümel J., Eggers M., Glebe D., Rapp I., Sauerbrei A., Steinmann E., Steinmann J., Willkommen H., et al.  Guideline for Testing Chemical Disinfectants Regarding Their Virucidal Activity within the Field of Human Medicine: As of December 1st, 2014 Prepared by the German Association for the Control of Virus Diseases (DVV) and the Robert Koch Institute (RKI) Bundesgesundheitsbl. 2020;63:645–655. doi: 10.1007/s00103-020-03115-w.</cite> [<a href="https://doi.org/10.1007/s00103-020-03115-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32274539/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bundesgesundheitsbl&amp;title=Guideline%20for%20Testing%20Chemical%20Disinfectants%20Regarding%20Their%20Virucidal%20Activity%20within%20the%20Field%20of%20Human%20Medicine:%20As%20of%20December%201st,%202014%20Prepared%20by%20the%20German%20Association%20for%20the%20Control%20of%20Virus%20Diseases%20(DVV)%20and%20the%20Robert%20Koch%20Institute%20(RKI)&amp;author=H.F.%20Rabenau&amp;author=I.%20Schwebke&amp;author=J.%20Bl%C3%BCmel&amp;author=M.%20Eggers&amp;author=D.%20Glebe&amp;volume=63&amp;publication_year=2020&amp;pages=645-655&amp;pmid=32274539&amp;doi=10.1007/s00103-020-03115-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B53-ijms-26-01934">
<span class="label">53.</span><cite>Forés E., Bofill-Mas S., Itarte M., Martínez-Puchol S., Hundesa A., Calvo M., Borrego C.M., Corominas L.L., Girones R., Rusiñol M. Evaluation of Two Rapid Ultrafiltration-Based Methods for SARS-CoV-2 Concentration from Wastewater. Sci. Total Environ. 2021;768:144786. doi: 10.1016/j.scitotenv.2020.144786.</cite> [<a href="https://doi.org/10.1016/j.scitotenv.2020.144786" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7789912/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33429117/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Total%20Environ.&amp;title=Evaluation%20of%20Two%20Rapid%20Ultrafiltration-Based%20Methods%20for%20SARS-CoV-2%20Concentration%20from%20Wastewater&amp;author=E.%20For%C3%A9s&amp;author=S.%20Bofill-Mas&amp;author=M.%20Itarte&amp;author=S.%20Mart%C3%ADnez-Puchol&amp;author=A.%20Hundesa&amp;volume=768&amp;publication_year=2021&amp;pages=144786&amp;pmid=33429117&amp;doi=10.1016/j.scitotenv.2020.144786&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B54-ijms-26-01934">
<span class="label">54.</span><cite>Lin S.-C., Kappes M.A., Chen M.-C., Lin C.-C., Wang T.T. Distinct Susceptibility and Applicability of MDCK Derivatives for Influenza Virus Research. PLoS ONE. 2017;12:e0172299.  doi: 10.1371/journal.pone.0172299.</cite> [<a href="https://doi.org/10.1371/journal.pone.0172299" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5313193/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28207898/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Distinct%20Susceptibility%20and%20Applicability%20of%20MDCK%20Derivatives%20for%20Influenza%20Virus%20Research&amp;author=S.-C.%20Lin&amp;author=M.A.%20Kappes&amp;author=M.-C.%20Chen&amp;author=C.-C.%20Lin&amp;author=T.T.%20Wang&amp;volume=12&amp;publication_year=2017&amp;pages=e0172299&amp;pmid=28207898&amp;doi=10.1371/journal.pone.0172299&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B55-ijms-26-01934">
<span class="label">55.</span><cite>Tsai H.-C., Lehman C.W., Lin C.-C., Tsai S.-W., Chen C.-M. Functional Evaluation for Adequacy of MDCK-Lineage Cells in Influenza Research. BMC Res. Notes. 2019;12:101.  doi: 10.1186/s13104-019-4134-2.</cite> [<a href="https://doi.org/10.1186/s13104-019-4134-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6390338/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30808400/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Res.%20Notes&amp;title=Functional%20Evaluation%20for%20Adequacy%20of%20MDCK-Lineage%20Cells%20in%20Influenza%20Research&amp;author=H.-C.%20Tsai&amp;author=C.W.%20Lehman&amp;author=C.-C.%20Lin&amp;author=S.-W.%20Tsai&amp;author=C.-M.%20Chen&amp;volume=12&amp;publication_year=2019&amp;pages=101&amp;pmid=30808400&amp;doi=10.1186/s13104-019-4134-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B56-ijms-26-01934">
<span class="label">56.</span><cite>LANXESS  VirkonTM S Disinfection of Avian Influenza Virus at 1:600. 2021.  [(accessed on 15 January 2025)].  Available online:  <a href="https://swineweb.com/wp-content/uploads/2021/01/VirkonTM-S-1.600-AI-US-letter-V3-PRINT.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://swineweb.com/wp-content/uploads/2021/01/VirkonTM-S-1.600-AI-US-letter-V3-PRINT.pdf</a>.</cite>
</li>
<li id="B57-ijms-26-01934">
<span class="label">57.</span><cite>Kim J., Yeom M., Lee T., Kim H.-O., Na W., Kang A., Lim J.-W., Park G., Park C., Song D., et al.  Porous Gold Nanoparticles for Attenuating Infectivity of Influenza A Virus. J. Nanobiotechnol. 2020;18:54.  doi: 10.1186/s12951-020-00611-8.</cite> [<a href="https://doi.org/10.1186/s12951-020-00611-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7092597/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32209114/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Nanobiotechnol.&amp;title=Porous%20Gold%20Nanoparticles%20for%20Attenuating%20Infectivity%20of%20Influenza%20A%20Virus&amp;author=J.%20Kim&amp;author=M.%20Yeom&amp;author=T.%20Lee&amp;author=H.-O.%20Kim&amp;author=W.%20Na&amp;volume=18&amp;publication_year=2020&amp;pages=54&amp;pmid=32209114&amp;doi=10.1186/s12951-020-00611-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B58-ijms-26-01934">
<span class="label">58.</span><cite>Chaika M., Zahorodnya S., Naumenko K., Pankivska Y., Povnitsa O., Mukha I., Vityuk N., Dorovskih A., Lokshyn M., Lysenko V., et al.  Virus Deformation or Destruction: Size-Dependence of Antiviral and Virucidal Activities of Gold Nanoparticles. Adv. Nat. Sci. Nanosci. Nanotechnol. 2022;13:035008. doi: 10.1088/2043-6262/ac879a.</cite> [<a href="https://doi.org/10.1088/2043-6262/ac879a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv.%20Nat.%20Sci.%20Nanosci.%20Nanotechnol.&amp;title=Virus%20Deformation%20or%20Destruction:%20Size-Dependence%20of%20Antiviral%20and%20Virucidal%20Activities%20of%20Gold%20Nanoparticles&amp;author=M.%20Chaika&amp;author=S.%20Zahorodnya&amp;author=K.%20Naumenko&amp;author=Y.%20Pankivska&amp;author=O.%20Povnitsa&amp;volume=13&amp;publication_year=2022&amp;pages=035008&amp;doi=10.1088/2043-6262/ac879a&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B59-ijms-26-01934">
<span class="label">59.</span><cite>Dehghani F., Mosleh-Shirazi S., Shafiee M., Kasaee S.R., Amani A.M. Antiviral and Antioxidant Properties of Green Synthesized Gold Nanoparticles Using Glaucium Flavum Leaf Extract. Appl. Nanosci. 2023;13:4395–4405. doi: 10.1007/s13204-022-02705-1.</cite> [<a href="https://doi.org/10.1007/s13204-022-02705-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9702789/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36466143/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl.%20Nanosci.&amp;title=Antiviral%20and%20Antioxidant%20Properties%20of%20Green%20Synthesized%20Gold%20Nanoparticles%20Using%20Glaucium%20Flavum%20Leaf%20Extract&amp;author=F.%20Dehghani&amp;author=S.%20Mosleh-Shirazi&amp;author=M.%20Shafiee&amp;author=S.R.%20Kasaee&amp;author=A.M.%20Amani&amp;volume=13&amp;publication_year=2023&amp;pages=4395-4405&amp;pmid=36466143&amp;doi=10.1007/s13204-022-02705-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B60-ijms-26-01934">
<span class="label">60.</span><cite>Sani A., Cao C., Cui D. Toxicity of Gold Nanoparticles (AuNPs): A Review. Biochem. Biophys. Rep. 2021;26:100991.  doi: 10.1016/j.bbrep.2021.100991.</cite> [<a href="https://doi.org/10.1016/j.bbrep.2021.100991" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8063742/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33912692/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Rep.&amp;title=Toxicity%20of%20Gold%20Nanoparticles%20(AuNPs):%20A%20Review&amp;author=A.%20Sani&amp;author=C.%20Cao&amp;author=D.%20Cui&amp;volume=26&amp;publication_year=2021&amp;pages=100991&amp;pmid=33912692&amp;doi=10.1016/j.bbrep.2021.100991&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B61-ijms-26-01934">
<span class="label">61.</span><cite>Xuan L., Ju Z., Skonieczna M., Zhou P., Huang R. Nanoparticles-induced Potential Toxicity on Human Health: Applications, Toxicity Mechanisms, and Evaluation Models. MedComm. 2023;4:e327. doi: 10.1002/mco2.327.</cite> [<a href="https://doi.org/10.1002/mco2.327" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10349198/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37457660/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=MedComm&amp;title=Nanoparticles-induced%20Potential%20Toxicity%20on%20Human%20Health:%20Applications,%20Toxicity%20Mechanisms,%20and%20Evaluation%20Models&amp;author=L.%20Xuan&amp;author=Z.%20Ju&amp;author=M.%20Skonieczna&amp;author=P.%20Zhou&amp;author=R.%20Huang&amp;volume=4&amp;publication_year=2023&amp;pages=e327&amp;pmid=37457660&amp;doi=10.1002/mco2.327&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B62-ijms-26-01934">
<span class="label">62.</span><cite>LANXESS  Virkon® S Disinfectant and Virucide. 2018.  [(accessed on 15 January 2025)].  Available online:  <a href="https://syndel.com/wp-content/uploads/2019/01/Virkon-S-Information-Page-Lanxess.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://syndel.com/wp-content/uploads/2019/01/Virkon-S-Information-Page-Lanxess.pdf</a>.</cite>
</li>
<li id="B63-ijms-26-01934">
<span class="label">63.</span><cite>Angelé-Martínez C., Nguyen K.V.T., Ameer F.S., Anker J.N., Brumaghim J.L. Reactive Oxygen Species Generation by Copper(II) Oxide Nanoparticles Determined by DNA Damage Assays and EPR Spectroscopy. Nanotoxicology. 2017;11:278–288. doi: 10.1080/17435390.2017.1293750.</cite> [<a href="https://doi.org/10.1080/17435390.2017.1293750" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5494152/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28248593/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nanotoxicology&amp;title=Reactive%20Oxygen%20Species%20Generation%20by%20Copper(II)%20Oxide%20Nanoparticles%20Determined%20by%20DNA%20Damage%20Assays%20and%20EPR%20Spectroscopy&amp;author=C.%20Angel%C3%A9-Mart%C3%ADnez&amp;author=K.V.T.%20Nguyen&amp;author=F.S.%20Ameer&amp;author=J.N.%20Anker&amp;author=J.L.%20Brumaghim&amp;volume=11&amp;publication_year=2017&amp;pages=278-288&amp;pmid=28248593&amp;doi=10.1080/17435390.2017.1293750&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B64-ijms-26-01934">
<span class="label">64.</span><cite>Lyngsie G., Krumina L., Tunlid A., Persson P. Generation of Hydroxyl Radicals from Reactions between a Dimethoxyhydroquinone and Iron Oxide Nanoparticles. Sci. Rep. 2018;8:10834.  doi: 10.1038/s41598-018-29075-5.</cite> [<a href="https://doi.org/10.1038/s41598-018-29075-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6050337/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30018415/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Generation%20of%20Hydroxyl%20Radicals%20from%20Reactions%20between%20a%20Dimethoxyhydroquinone%20and%20Iron%20Oxide%20Nanoparticles&amp;author=G.%20Lyngsie&amp;author=L.%20Krumina&amp;author=A.%20Tunlid&amp;author=P.%20Persson&amp;volume=8&amp;publication_year=2018&amp;pages=10834&amp;pmid=30018415&amp;doi=10.1038/s41598-018-29075-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B65-ijms-26-01934">
<span class="label">65.</span><cite>Ryu W.S.  Molecular Virology of Human Pathogenic Viruses. Elsevier; Amsterdam, The Netherlands: 2017. Influenza Viruses; p. iv.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Molecular%20Virology%20of%20Human%20Pathogenic%20Viruses&amp;author=W.S.%20Ryu&amp;publication_year=2017&amp;isbn=978-0-12-800838-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B66-ijms-26-01934">
<span class="label">66.</span><cite>Noda T. Native Morphology of Influenza Virions. Front. Microbiol. 2012;2:20301.  doi: 10.3389/fmicb.2011.00269.</cite> [<a href="https://doi.org/10.3389/fmicb.2011.00269" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3249889/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22291683/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&amp;title=Native%20Morphology%20of%20Influenza%20Virions&amp;author=T.%20Noda&amp;volume=2&amp;publication_year=2012&amp;pages=20301&amp;pmid=22291683&amp;doi=10.3389/fmicb.2011.00269&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B67-ijms-26-01934">
<span class="label">67.</span><cite>Korneev D., Kurskaya O., Sharshov K., Eastwood J., Strakhovskaya M. Ultrastructural Aspects of Photodynamic Inactivation of Highly Pathogenic Avian H5N8 Influenza Virus. Viruses. 2019;11:955.  doi: 10.3390/v11100955.</cite> [<a href="https://doi.org/10.3390/v11100955" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6832225/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31623281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Viruses&amp;title=Ultrastructural%20Aspects%20of%20Photodynamic%20Inactivation%20of%20Highly%20Pathogenic%20Avian%20H5N8%20Influenza%20Virus&amp;author=D.%20Korneev&amp;author=O.%20Kurskaya&amp;author=K.%20Sharshov&amp;author=J.%20Eastwood&amp;author=M.%20Strakhovskaya&amp;volume=11&amp;publication_year=2019&amp;pages=955&amp;pmid=31623281&amp;doi=10.3390/v11100955&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B68-ijms-26-01934">
<span class="label">68.</span><cite>Fujimori Y., Sato T., Hayata T., Nagao T., Nakayama M., Nakayama T., Sugamata R., Suzuki K. Novel Antiviral Characteristics of Nanosized Copper(I) Iodide Particles Showing Inactivation Activity against 2009 Pandemic H1N1 Influenza Virus. Appl. Environ. Microbiol. 2012;78:951–955. doi: 10.1128/AEM.06284-11.</cite> [<a href="https://doi.org/10.1128/AEM.06284-11" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3272987/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22156433/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&amp;title=Novel%20Antiviral%20Characteristics%20of%20Nanosized%20Copper(I)%20Iodide%20Particles%20Showing%20Inactivation%20Activity%20against%202009%20Pandemic%20H1N1%20Influenza%20Virus&amp;author=Y.%20Fujimori&amp;author=T.%20Sato&amp;author=T.%20Hayata&amp;author=T.%20Nagao&amp;author=M.%20Nakayama&amp;volume=78&amp;publication_year=2012&amp;pages=951-955&amp;pmid=22156433&amp;doi=10.1128/AEM.06284-11&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B69-ijms-26-01934">
<span class="label">69.</span><cite>Shaw M.L., Stone K.L., Colangelo C.M., Gulcicek E.E., Palese P. Cellular Proteins in Influenza Virus Particles. PLoS Pathog. 2008;4:e1000085.  doi: 10.1371/journal.ppat.1000085.</cite> [<a href="https://doi.org/10.1371/journal.ppat.1000085" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2390764/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18535660/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog.&amp;title=Cellular%20Proteins%20in%20Influenza%20Virus%20Particles&amp;author=M.L.%20Shaw&amp;author=K.L.%20Stone&amp;author=C.M.%20Colangelo&amp;author=E.E.%20Gulcicek&amp;author=P.%20Palese&amp;volume=4&amp;publication_year=2008&amp;pages=e1000085&amp;pmid=18535660&amp;doi=10.1371/journal.ppat.1000085&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B70-ijms-26-01934">
<span class="label">70.</span><cite>Anthis N.J., Clore G.M. Sequence-specific Determination of Protein and Peptide Concentrations by Absorbance at 205 Nm. Protein Sci. 2013;22:851–858. doi: 10.1002/pro.2253.</cite> [<a href="https://doi.org/10.1002/pro.2253" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3690723/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23526461/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci.&amp;title=Sequence-specific%20Determination%20of%20Protein%20and%20Peptide%20Concentrations%20by%20Absorbance%20at%20205%20Nm&amp;author=N.J.%20Anthis&amp;author=G.M.%20Clore&amp;volume=22&amp;publication_year=2013&amp;pages=851-858&amp;pmid=23526461&amp;doi=10.1002/pro.2253&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B71-ijms-26-01934">
<span class="label">71.</span><cite>Chang S.-Y., Huang K.-Y., Chao T.-L., Kao H.-C., Pang Y.-H., Lu L., Chiu C.-L., Huang H.-C., Cheng T.-J.R., Fang J.-M., et al.  Nanoparticle Composite TPNT1 Is Effective against SARS-CoV-2 and Influenza Viruses. Sci. Rep. 2021;11:8692.  doi: 10.1038/s41598-021-87254-3.</cite> [<a href="https://doi.org/10.1038/s41598-021-87254-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8062499/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33888738/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Nanoparticle%20Composite%20TPNT1%20Is%20Effective%20against%20SARS-CoV-2%20and%20Influenza%20Viruses&amp;author=S.-Y.%20Chang&amp;author=K.-Y.%20Huang&amp;author=T.-L.%20Chao&amp;author=H.-C.%20Kao&amp;author=Y.-H.%20Pang&amp;volume=11&amp;publication_year=2021&amp;pages=8692&amp;pmid=33888738&amp;doi=10.1038/s41598-021-87254-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B72-ijms-26-01934">
<span class="label">72.</span><cite>Benhalima L., Amri S., Bensouilah M., Ouzrout R. Antibacterial Effect of Copper Sulfate against Multi-Drug Resistant Nosocomial Pathogens Isolated from Clinical Samples: Effect of Copper Sulfate on Clinical Isolates. Pak. J. Med. Sci. 2019;35:1322–1328. doi: 10.12669/pjms.35.5.336.</cite> [<a href="https://doi.org/10.12669/pjms.35.5.336" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6717487/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31489000/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pak.%20J.%20Med.%20Sci.&amp;title=Antibacterial%20Effect%20of%20Copper%20Sulfate%20against%20Multi-Drug%20Resistant%20Nosocomial%20Pathogens%20Isolated%20from%20Clinical%20Samples:%20Effect%20of%20Copper%20Sulfate%20on%20Clinical%20Isolates&amp;author=L.%20Benhalima&amp;author=S.%20Amri&amp;author=M.%20Bensouilah&amp;author=R.%20Ouzrout&amp;volume=35&amp;publication_year=2019&amp;pages=1322-1328&amp;pmid=31489000&amp;doi=10.12669/pjms.35.5.336&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B73-ijms-26-01934">
<span class="label">73.</span><cite>Bradford M.M. A Rapid and Sensitive Method for the Quantitation of Microgram Quantities of Protein Utilizing the Principle of Protein-Dye Binding. Anal. Biochem. 1976;72:248–254. doi: 10.1016/0003-2697(76)90527-3.</cite> [<a href="https://doi.org/10.1016/0003-2697(76)90527-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/942051/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal.%20Biochem.&amp;title=A%20Rapid%20and%20Sensitive%20Method%20for%20the%20Quantitation%20of%20Microgram%20Quantities%20of%20Protein%20Utilizing%20the%20Principle%20of%20Protein-Dye%20Binding&amp;author=M.M.%20Bradford&amp;volume=72&amp;publication_year=1976&amp;pages=248-254&amp;pmid=942051&amp;doi=10.1016/0003-2697(76)90527-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B74-ijms-26-01934">
<span class="label">74.</span><cite>Thompson J.D., Higgins D.G., Gibson T.J. CLUSTAL W: Improving the Sensitivity of Progressive Multiple Sequence Alignment through Sequence Weighting, Position-Specific Gap Penalties and Weight Matrix Choice. Nucleic Acids Res. 1994;22:4673–4680. doi: 10.1093/nar/22.22.4673.</cite> [<a href="https://doi.org/10.1093/nar/22.22.4673" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC308517/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7984417/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=CLUSTAL%20W:%20Improving%20the%20Sensitivity%20of%20Progressive%20Multiple%20Sequence%20Alignment%20through%20Sequence%20Weighting,%20Position-Specific%20Gap%20Penalties%20and%20Weight%20Matrix%20Choice&amp;author=J.D.%20Thompson&amp;author=D.G.%20Higgins&amp;author=T.J.%20Gibson&amp;volume=22&amp;publication_year=1994&amp;pages=4673-4680&amp;pmid=7984417&amp;doi=10.1093/nar/22.22.4673&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B75-ijms-26-01934">
<span class="label">75.</span><cite>Katoh K. MAFFT: A Novel Method for Rapid Multiple Sequence Alignment Based on Fast Fourier Transform. Nucleic Acids Res. 2002;30:3059–3066. doi: 10.1093/nar/gkf436.</cite> [<a href="https://doi.org/10.1093/nar/gkf436" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC135756/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12136088/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=MAFFT:%20A%20Novel%20Method%20for%20Rapid%20Multiple%20Sequence%20Alignment%20Based%20on%20Fast%20Fourier%20Transform&amp;author=K.%20Katoh&amp;volume=30&amp;publication_year=2002&amp;pages=3059-3066&amp;pmid=12136088&amp;doi=10.1093/nar/gkf436&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B76-ijms-26-01934">
<span class="label">76.</span><cite>Li W., Jaroszewski L., Godzik A. Clustering of Highly Homologous Sequences to Reduce the Size of Large Protein Databases. Bioinformatics. 2001;17:282–283. doi: 10.1093/bioinformatics/17.3.282.</cite> [<a href="https://doi.org/10.1093/bioinformatics/17.3.282" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11294794/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Clustering%20of%20Highly%20Homologous%20Sequences%20to%20Reduce%20the%20Size%20of%20Large%20Protein%20Databases&amp;author=W.%20Li&amp;author=L.%20Jaroszewski&amp;author=A.%20Godzik&amp;volume=17&amp;publication_year=2001&amp;pages=282-283&amp;pmid=11294794&amp;doi=10.1093/bioinformatics/17.3.282&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B77-ijms-26-01934">
<span class="label">77.</span><cite>Russell R.B., Barton G.J. Multiple Protein Sequence Alignment from Tertiary Structure Comparison: Assignment of Global and Residue Confidence Levels. Proteins. 1992;14:309–323. doi: 10.1002/prot.340140216.</cite> [<a href="https://doi.org/10.1002/prot.340140216" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1409577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Multiple%20Protein%20Sequence%20Alignment%20from%20Tertiary%20Structure%20Comparison:%20Assignment%20of%20Global%20and%20Residue%20Confidence%20Levels&amp;author=R.B.%20Russell&amp;author=G.J.%20Barton&amp;volume=14&amp;publication_year=1992&amp;pages=309-323&amp;pmid=1409577&amp;doi=10.1002/prot.340140216&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B78-ijms-26-01934">
<span class="label">78.</span><cite>Orengo C.A., Taylor W.R.  Methods in Enzymology. Volume 266. Elsevier; Amsterdam, The Netherlands: 1996. [36] SSAP: Sequential Structure Alignment Program for Protein Structure Comparison; pp. 617–635.</cite> [<a href="https://doi.org/10.1016/s0076-6879(96)66038-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8743709/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Methods%20in%20Enzymology&amp;author=C.A.%20Orengo&amp;author=W.R.%20Taylor&amp;publication_year=1996&amp;isbn=978-0-12-182167-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B79-ijms-26-01934">
<span class="label">79.</span><cite>Chevallet M., Luche S., Rabilloud T. Silver Staining of Proteins in Polyacrylamide Gels. Nat. Protoc. 2006;1:1852–1858. doi: 10.1038/nprot.2006.288.</cite> [<a href="https://doi.org/10.1038/nprot.2006.288" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1971133/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17487168/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&amp;title=Silver%20Staining%20of%20Proteins%20in%20Polyacrylamide%20Gels&amp;author=M.%20Chevallet&amp;author=S.%20Luche&amp;author=T.%20Rabilloud&amp;volume=1&amp;publication_year=2006&amp;pages=1852-1858&amp;pmid=17487168&amp;doi=10.1038/nprot.2006.288&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B80-ijms-26-01934">
<span class="label">80.</span><cite>Reed L.J., Muench H. A simple method of estimating fifty per cent endpoints. Am. J. Epidemiol. 1938;27:493–497. doi: 10.1093/oxfordjournals.aje.a118408.</cite> [<a href="https://doi.org/10.1093/oxfordjournals.aje.a118408" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Epidemiol.&amp;title=A%20simple%20method%20of%20estimating%20fifty%20per%20cent%20endpoints&amp;author=L.J.%20Reed&amp;author=H.%20Muench&amp;volume=27&amp;publication_year=1938&amp;pages=493-497&amp;doi=10.1093/oxfordjournals.aje.a118408&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B81-ijms-26-01934">
<span class="label">81.</span><cite>Xiang D., Chen Q., Pang L., Zheng C. Inhibitory Effects of Silver Nanoparticles on H1N1 Influenza A Virus in Vitro. J. Virol. Methods. 2011;178:137–142. doi: 10.1016/j.jviromet.2011.09.003.</cite> [<a href="https://doi.org/10.1016/j.jviromet.2011.09.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21945220/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Virol.%20Methods&amp;title=Inhibitory%20Effects%20of%20Silver%20Nanoparticles%20on%20H1N1%20Influenza%20A%20Virus%20in%20Vitro&amp;author=D.%20Xiang&amp;author=Q.%20Chen&amp;author=L.%20Pang&amp;author=C.%20Zheng&amp;volume=178&amp;publication_year=2011&amp;pages=137-142&amp;pmid=21945220&amp;doi=10.1016/j.jviromet.2011.09.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/11899802/bin/ijms-26-01934-s001.zip" data-ga-action="click_feat_suppl" class="usa-link">ijms-26-01934-s001.zip</a><sup> (954.5KB, zip) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data generated or analyzed during this study are included in this published article and the <a href="#app1-ijms-26-01934" class="usa-link">Supplementary Files</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from International Journal of Molecular Sciences are provided here courtesy of <strong>Multidisciplinary Digital Publishing Institute  (MDPI)</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.3390/ijms26051934"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/ijms-26-01934.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.8 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/11899802/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/11899802/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11899802%2F%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11899802/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC11899802/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC11899802/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40076560/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC11899802/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40076560/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC11899802/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/11899802/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="hshCKrkeBj3mO4FZwMy6M9W0TawfKshiG6n9X4ivkxtF1fKNTWmz1ORBTcfpSXkJ">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-B7-tgZZA.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-BLbQGH8u.js"></script>
    
    

    </body>
</html>
