
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/base_style-CPPDhfna.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-BepKMsUl.js"></script>

  <link rel="stylesheet" href="/static/assets/article_style-DUfI1jZp.css" />
<link rel="stylesheet" href="/static/assets/style-BOrsu84o.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-D3wJHomh.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3EPreprint%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Structural basis of substrate specificity of Helix pomatia AMP deaminase and a chimeric ADGF adenosine deaminase - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="E0CA2594930B1EB3002594001FBF54BD.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="biorxiv">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="preprint">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11974800/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="bioRxiv">
<meta name="citation_title" content="Structural basis of substrate specificity of Helix pomatia AMP deaminase and a chimeric ADGF adenosine deaminase">
<meta name="citation_author" content="Gundeep Kaur">
<meta name="citation_author_institution" content="Department of Epigenetics and Molecular Carcinogenesis, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA">
<meta name="citation_author" content="John R Horton">
<meta name="citation_author_institution" content="Department of Epigenetics and Molecular Carcinogenesis, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA">
<meta name="citation_author" content="George Tzertzinis">
<meta name="citation_author_institution" content="New England Biolabs, 240 County Road, Ipswich, MA 01938, USA">
<meta name="citation_author" content="Jujun Zhou">
<meta name="citation_author_institution" content="Department of Epigenetics and Molecular Carcinogenesis, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA">
<meta name="citation_author" content="Ira Schildkraut">
<meta name="citation_author_institution" content="New England Biolabs, 240 County Road, Ipswich, MA 01938, USA">
<meta name="citation_author" content="Xiaodong Cheng">
<meta name="citation_author_institution" content="Department of Epigenetics and Molecular Carcinogenesis, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA">
<meta name="citation_publication_date" content="2025 Mar 27">
<meta name="citation_firstpage" content="2025.03.26.645602">
<meta name="citation_doi" content="10.1101/2025.03.26.645602">
<meta name="citation_pmid" content="40196538">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11974800/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11974800/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11974800/pdf/nihpp-2025.03.26.645602v1.pdf">
<meta name="description" content="HPAMPD, an enzyme enriched in the foot muscle of the mollusk Helix pomatia, exhibits deaminase activity on adenosine-5’-monophosphate (AMP). HPAMPD is the first member of the adenosine deaminase-related growth factor (ADGF) family to prefer the ...">
<meta name="og:title" content="Structural basis of substrate specificity of Helix pomatia AMP deaminase and a chimeric ADGF adenosine deaminase">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="HPAMPD, an enzyme enriched in the foot muscle of the mollusk Helix pomatia, exhibits deaminase activity on adenosine-5’-monophosphate (AMP). HPAMPD is the first member of the adenosine deaminase-related growth factor (ADGF) family to prefer the ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11974800/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://pmc.ncbi.nlm.nih.gov/search/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="11974800">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1101/2025.03.26.645602"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/nihpp-2025.03.26.645602v1.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11974800%2F%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/11974800/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/11974800/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11974800/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <div class="usa-alert usa-alert--info" role="status"><div class="usa-alert__body">
<h4 class="usa-alert__heading">This is a preprint.</h4>
<p class="usa-alert__text"><strong>It has not yet been peer reviewed by a journal.</strong></p>
<p class="usa-alert__text">The National Library of Medicine is <a href="/about/nihpreprints/" class="usa-link">running a pilot</a> to include preprints that result from research funded by NIH in PMC and PubMed.</p>
</div></div><section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-preprints.png" alt="bioRxiv logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to bioRxiv" title="Link to bioRxiv" shape="default" href="https://pmc.ncbi.nlm.nih.gov/about/nihpreprints/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">bioRxiv</button></div> [Preprint]. 2025 Mar 27:2025.03.26.645602. [Version 1] doi: <a href="https://doi.org/10.1101/2025.03.26.645602" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1101/2025.03.26.645602</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://pmc.ncbi.nlm.nih.gov/search/?term=%22bioRxiv%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22bioRxiv%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22bioRxiv%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22bioRxiv%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Structural basis of substrate specificity of <em>Helix pomatia</em> AMP deaminase and a chimeric ADGF adenosine deaminase</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kaur%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Gundeep Kaur</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Gundeep Kaur</span></h3>
<div class="p">
<sup>1</sup>Department of Epigenetics and Molecular Carcinogenesis, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA</div>
<div>performed crystallization and substrate modeling, performed X-ray crystallography experiments</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kaur%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Gundeep Kaur</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Horton%20JR%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">John R Horton</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">John R Horton</span></h3>
<div class="p">
<sup>1</sup>Department of Epigenetics and Molecular Carcinogenesis, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA</div>
<div>performed X-ray crystallography experiments</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Horton%20JR%22%5BAuthor%5D" class="usa-link"><span class="name western">John R Horton</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tzertzinis%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">George Tzertzinis</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">George Tzertzinis</span></h3>
<div class="p">
<sup>2</sup>New England Biolabs, 240 County Road, Ipswich, MA 01938, USA</div>
<div>constructed chimeras</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tzertzinis%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">George Tzertzinis</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Jujun Zhou</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Jujun Zhou</span></h3>
<div class="p">
<sup>1</sup>Department of Epigenetics and Molecular Carcinogenesis, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA</div>
<div>performed the phylogenetic tree analysis, AlphaFold prediction of KomA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jujun Zhou</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schildkraut%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Ira Schildkraut</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Ira Schildkraut</span></h3>
<div class="p">
<sup>2</sup>New England Biolabs, 240 County Road, Ipswich, MA 01938, USA</div>
<div>performed protein purifications and deaminase assays, initiated the study, organized, designed the scope of the study</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schildkraut%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Ira Schildkraut</span></a>
</div>
</div>
<sup>2,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cheng%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Xiaodong Cheng</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Xiaodong Cheng</span></h3>
<div class="p">
<sup>1</sup>Department of Epigenetics and Molecular Carcinogenesis, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA</div>
<div>organized, designed the scope of the study</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cheng%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaodong Cheng</span></a>
</div>
</div>
<sup>1,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="A1">
<sup>1</sup>Department of Epigenetics and Molecular Carcinogenesis, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA</div>
<div id="A2">
<sup>2</sup>New England Biolabs, 240 County Road, Ipswich, MA 01938, USA</div>
<div class="author-notes p"><div class="fn" id="CR1">
<sup>*</sup><p class="display-inline">Correspondence: <span>schildkraut@neb.com</span>; <span>xcheng5@mdanderson.org</span></p>
</div></div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Gundeep Kaur</span></strong>: <span class="role">performed crystallization and substrate modeling, performed X-ray crystallography experiments</span>
</div>
<div>
<strong class="contrib"><span class="name western">John R Horton</span></strong>: <span class="role">performed X-ray crystallography experiments</span>
</div>
<div>
<strong class="contrib"><span class="name western">George Tzertzinis</span></strong>: <span class="role">constructed chimeras</span>
</div>
<div>
<strong class="contrib"><span class="name western">Jujun Zhou</span></strong>: <span class="role">performed the phylogenetic tree analysis, AlphaFold prediction of KomA</span>
</div>
<div>
<strong class="contrib"><span class="name western">Ira Schildkraut</span></strong>: <span class="role">performed protein purifications and deaminase assays, initiated the study, organized, designed the scope of the study</span>
</div>
<div>
<strong class="contrib"><span class="name western">Xiaodong Cheng</span></strong>: <span class="role">organized, designed the scope of the study</span>
</div>
</div>
<div id="clp_a" class="d-panel p" style="display: none">
<p>This work is licensed under a <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License</a>, which allows reusers to copy and distribute the material in any medium or format in unadapted form only, for noncommercial purposes only, and only so long as attribution is given to the creator.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC11974800  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40196538/" class="usa-link">40196538</a>
</div>
<div class="ra xbox p" role="complementary" aria-label="Related or updated information about this article">
<div>
<strong>Updated version available:</strong> A peer-reviewed version of this article, "<a href="/articles/PMC12273558/" class="usa-link">Structural basis for the substrate specificity of <em>Helix pomatia</em> AMP deaminase and a chimeric ADGF adenosine deaminase</a>", has been published in <em>The Journal of Biological Chemistry</em>.</div>
<div>The complete version history of this preprint is available at <a href="https://doi.org/10.1101/2025.03.26.645602" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">bioRxiv</a>.</div>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="ABS1"><h2>Abstract</h2>
<p id="P1">HPAMPD, an enzyme enriched in the foot muscle of the mollusk <em>Helix pomatia</em>, exhibits deaminase activity on adenosine-5’-monophosphate (AMP). HPAMPD is the first member of the <span class="text-underline">a</span>denosine <span class="text-underline">d</span>eaminase-related growth factor (ADGF) family to prefer the nucleotide, AMP, over the nucleoside, adenosine. To investigate the substrate selectivity of HPAMPD, we determined its structure in the apo form and in complex with the adenosine analogs pentostatin (2’-deoxycoformycin) and pentostatin-5’-monophosphate. Structurally, HPAMPD adopts a fold similar to human ADA2, an ADGF family member. HPAMPD has acquired the ability to interact with the 5’-monophosphate group of AMP through polar and charged residues located in three key structural elements: (1) the loop immediately following strand β1, (2) the loop between helices αH and αI, and (3) the end of strand β5 and its adjacent loop. We engineered a chimeric deaminase by integrating these elements from HPAMPD into another related mollusk nucleoside adenosine deaminase, the <em>A. californica</em> ADGF. The chimeric enzyme efficiently deaminates AMP, demonstrating a gained substrate specificity, while retaining the adenosine deamination activity of <em>Aplysia</em> ADGF. The phosphate-binding feature of HPAMPD is a hallmark of nucleotide deaminases, conserved among AMP and N6-methyl-AMP (6mAMP) deaminases. We discuss the human adenosine deaminases each with distinct substrate specificities for the nucleoside, the nucleotide (AMP), and the methylated form, 6mAMP.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> <em>Helix pomatia</em> AMP deaminase (HPAMPD), adenosine deaminase 2 (ADA2), <span class="text-underline">a</span>denosine <span class="text-underline">d</span>eaminase-related <span class="text-underline">g</span>rowth <span class="text-underline">f</span>actor (ADGF) family</p></section></section><section class="abstract" id="ABS2"><h2>Graphical Abstract</h2>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4c53/11974800/7139dc122059/nihpp-2025.03.26.645602v1-f0001.jpg" loading="lazy" height="255" width="600" alt="graphic file with name nihpp-2025.03.26.645602v1-f0001.jpg"></p></section><section id="S1"><h2 class="pmc_sec_title">INTRODUCTION</h2>
<p id="P3">The <span class="text-underline">a</span>denosine <span class="text-underline">d</span>eaminase-related <span class="text-underline">g</span>rowth <span class="text-underline">f</span>actors (ADGF) family of proteins are enzymes which are understood to control the level of extracellular adenosine, an important signaling molecule that effects cellular responses <sup><a href="#R1" class="usa-link" aria-describedby="R1">1</a></sup>. Human ADA2 is a well-studied ADGF. The concept of ADA2 as a growth factor originates from a historical study in which immature flesh fly cells exhibited an activity that stimulated insect cells growth in fresh cultured media <sup><a href="#R2" class="usa-link" aria-describedby="R2">2</a></sup>. This secreted <span class="text-underline">i</span>nsect-<span class="text-underline">d</span>erived <span class="text-underline">g</span>rowth <span class="text-underline">f</span>actor (IDGF) was found to possess deaminase activity, which was essential for its ability to promote cell growth <sup><a href="#R3" class="usa-link" aria-describedby="R3">3</a></sup>. Based on its protein sequence similarity to IDGF, human ADA2 was proposed to function as a growth factor <sup><a href="#R4" class="usa-link" aria-describedby="R4">4</a>,<a href="#R5" class="usa-link" aria-describedby="R5">5</a></sup>. In this study, we focus on the deaminase activity of ADGF family members, including one identified in the sea slug, <em>Aplysia californica</em>
<sup><a href="#R6" class="usa-link" aria-describedby="R6">6</a></sup>. Based on protein sequence conservation, we recently identified a homolog of ADGF in another mollusk, <em>Helix pomatia</em>
<sup><a href="#R7" class="usa-link" aria-describedby="R7">7</a></sup>, commonly known as the Roman snail, Burgundy snail, or escargot. The <em>H. pomatia</em> deaminase shares 57% sequence identity with that of <em>Aplysia californica</em> ADGF and 38% identity to human ADA2 (<a href="#F1" class="usa-link">Figure 1</a>). <em>H. pomatia</em> deaminase was an unexpected member of the ADGF family, as it exhibits deaminase activity with a preference for the <span class="text-underline">nucleotide</span> adenosine 5’-monophosphate (AMP) over the <span class="text-underline">nucleoside</span> adenosine. Consequently, it was named <em><span class="text-underline">H</span>. pomatia</em>
<span class="text-underline">AMP d</span>eaminase (HPAMPD). Before HPAMPD, no ADGFs were known to deaminate AMP.</p>
<figure class="fig xbox font-sm" id="F1"><h3 class="obj_head">Figure 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11974800_nihpp-2025.03.26.645602v1-f0002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4c53/11974800/d7b379a42784/nihpp-2025.03.26.645602v1-f0002.jpg" loading="lazy" height="702" width="720" alt="Figure 1."></a></p>
<div class="p text-right font-secondary"><a href="figure/F1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P36"><strong>(A)</strong> Summary of human enzymes acting on adenosine, AMP, and 6mAMP and the relative activity with HPAMPD on each substrate. <strong>(B)</strong> Sequence alignment of three ADGF deaminases: human ADA2 (UniProt: Q9NZK5-1), <em>Aplysia</em> ADGF (UniProt: <a href="https://www.ncbi.nlm.nih.gov/protein/P15287" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P15287</a>), and HPAMPD (GenBank: <a href="https://ncbi.nlm.nih.gov/nucleotide/XOL57624.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XOL57624.1</a>). The underlined N-terminal sequences represent signal peptides that control protein secretion and translocation <sup><a href="#R65" class="usa-link" aria-describedby="R65">65</a></sup>. Invariant residues among all three enzymes are shown as white letters on a black background, while residues conserved between <em>Aplysia</em> and <em>Helix</em> ADGFs are displayed as white letters on a light green background. Secondary structure elements (α-helices and β-strands) of HPAMPD are depicted below the sequence. Letters above the human ADA2 sequence indicate conserved residues involved in zinc binding (Z) and inhibitor pentostatin binding (I). Letters P below the HPAMPD sequence marks residues uniquely involved in phosphate group binding of pentostatin 5’-monophosphate. Regions swapped between <em>Aplysia</em> ADGF and HPAMPD to generate a chimeric ADGF are indicated by a red line. A disulfide bond (S-S) between C146 and C168 is indicated by a bracket. Glycosylation at asparagine residues N85, N136, N193, and N388 are marked with the letter G (see <a href="#SD1" class="usa-link">Figure S4</a>).</p></figcaption></figure><p id="P4">Human adenosine deaminase 1 (ADA1) deficiency, first described in the 1970s, was recognized as a cause of <span class="text-underline">s</span>evere <span class="text-underline">c</span>ombined <span class="text-underline">i</span>mmuno<span class="text-underline">d</span>eficiency (SCID) <sup><a href="#R8" class="usa-link" aria-describedby="R8">8</a></sup>. Residual adenosine deaminase activity, attributed to adenosine deaminase 2 (ADA2), was initially studied in spleen extracts from patients with SCID and ADA1 deficiency <sup><a href="#R9" class="usa-link" aria-describedby="R9">9</a></sup>. The ADA2 enzyme is encoded by a gene on chromosome 22, within the region affected in patients with cat eye syndrome, which led to its original gene name, <span class="text-underline">c</span>at <span class="text-underline">e</span>ye syndrome <span class="text-underline">c</span>ritical <span class="text-underline">r</span>egion 1 (<em>CECR1</em>)<sup><a href="#R10" class="usa-link" aria-describedby="R10">10</a></sup>. In contrast, ADA2 deficiency, caused by loss-of-function mutations (&gt;100 variants) in the <em>ADA2</em>/<em>CECR1</em> gene – was only described 11 years ago. This deficiency has been linked to a broad spectrum of vascular and inflammatory phenotypes, including early-onset recurrent stroke, systemic vasculopathy, and vasculitis <sup><a href="#R11" class="usa-link" aria-describedby="R11">11</a>,<a href="#R12" class="usa-link" aria-describedby="R12">12</a></sup>. The complexity of the phenotype and pathophysiology of ADA2 deficiency is well-documented (reviewed in <sup><a href="#R13" class="usa-link" aria-describedby="R13">13</a>-<a href="#R18" class="usa-link" aria-describedby="R18">18</a></sup>), with an estimated carrier frequency of over 1 in 236 individuals <sup><a href="#R19" class="usa-link" aria-describedby="R19">19</a></sup> and more than 100 disease-associated mutations were reported <sup><a href="#R20" class="usa-link" aria-describedby="R20">20</a></sup>. [There are 159 mutations of ADA2 in the registry of hereditary auto-inflammatory disorders mutations (Infevers) and 608 mutations of ADA2 registered in Catalogue Of Somatic Mutations In Cancer (COSMIC).]</p>
<p id="P5">Enzymatically, ADA1 and ADA2 catalyze the deamination of adenosine (Ado) and 2′-deoxyadenosine (dAdo), converting them to inosine and deoxyinosine, respectively. In ADA1 deficient SCID patients, plasma levels of Ado and dAdo are elevated, leading to (1) marked accumulation of dATP, derived from dAdo metabolism <sup><a href="#R21" class="usa-link" aria-describedby="R21">21</a></sup>, and (2) increased levels of <em>S</em>-adenosyl-<span class="font-variant-small-caps">l</span>-homocysteine (SAH), due to dAdo-mediated inhibition of SAH hydrolase <sup><a href="#R22" class="usa-link" aria-describedby="R22">22</a></sup>. The excess of dATP inhibits DNA synthesis, while SAH accumulation inhibits most (if not all) transmethylation reactions catalyzed by <em>S</em>-adenosyl-<span class="font-variant-small-caps">l</span>-methionine (SAM)-dependent methyltransferases, affecting multiple cellular processes. These disruptions highlight the critical role of purine metabolism and associated deaminases in cellular functions of purinergic signaling and in anti-cancer therapy resistance <sup><a href="#R23" class="usa-link" aria-describedby="R23">23</a>-<a href="#R25" class="usa-link" aria-describedby="R25">25</a></sup>.</p>
<p id="P6">Human ADA1 enzyme consists of 363 residues (UniProt <a href="https://www.ncbi.nlm.nih.gov/protein/P00813" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P00813</a>) and adopts a single globular domain with an α/β barrel fold <sup><a href="#R26" class="usa-link" aria-describedby="R26">26</a>,<a href="#R27" class="usa-link" aria-describedby="R27">27</a></sup>, as seen in crystal structures (human ADA1: PDB 3IAR and 7RTG; murine Ada1: PDB 2ADA) (<a href="#F1" class="usa-link">Figure 1A</a>). In contrast, human ADA2 (UniProt <a href="https://www.ncbi.nlm.nih.gov/protein/Q9NZK5" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q9NZK5</a>; 511 residues) forms a homodimer <sup><a href="#R5" class="usa-link" aria-describedby="R5">5</a></sup>, and features an additional ~80 N-terminal residues responsible for dimerization as well as an extracellular signal peptide, alongside its ADA1-like catalytic domain <sup><a href="#R28" class="usa-link" aria-describedby="R28">28</a></sup> (PDB 3LGD and 3LGG). The presence of N-linked glycosylation, disulfide bonds, and a signal peptide at the very N-terminus indicates that ADA2 functions in the extracellular environment. In addition, ADA2 is a lysosomal protein <sup><a href="#R29" class="usa-link" aria-describedby="R29">29</a>,<a href="#R30" class="usa-link" aria-describedby="R30">30</a></sup>.</p>
<p id="P7">Humans possess distinct enzymes for the deamination activity of <span class="text-underline">nucleosides</span> and <span class="text-underline">nucleotides</span>, including adenosine, adenosine 5’-monophosphate (AMP), and its methylated form, N6-methyl-AMP (6mAMP) (<a href="#F1" class="usa-link">Figure 1A</a>). HPAMPD exhibits deaminase activity on all three substrates, with a preference order of AMP &gt; adenosine &gt; 6mAMP (<sup><a href="#R7" class="usa-link" aria-describedby="R7">7</a></sup> and this study). In contrast, human ADA2 and <em>Aplysia</em> ADGF do not deaminate AMP and 6mAMP. To investigate the substrate selectivity of HPAMPD, we determined its structures in the apo form and in complexes with pentostatin (2′-deoxycoformycin) and <span class="text-underline">p</span>entostatin 5’-<span class="text-underline">m</span>ono<span class="text-underline">p</span>hosphate (PMP). Additionally, we engineered a chimeric ADGF by integrating elements from <em>A. californica</em> and <em>H. pomatia</em>, resulting in a chimeric enzyme with equivalent activity on AMP and adenosine.</p></section><section id="S2"><h2 class="pmc_sec_title">RESULTS</h2>
<section id="S3"><h3 class="pmc_sec_title">Structural basis of substrate specificity of the ADGFs</h3>
<p id="P8">The human ADA2 is an adenosine deaminase with a relatively high <em>K</em>m for adenosine of 2500 μM and <em>k</em><sub>cat</sub> of 88 s<sup>−1</sup> (ref. <sup><a href="#R5" class="usa-link" aria-describedby="R5">5</a></sup>) (<a href="#F2" class="usa-link">Figure 2A</a>), whereas HPAMPD has a <em>K</em>m of 25 μM and a <em>k</em>cat of 170 s<sup>−1</sup> for AMP <sup><a href="#R7" class="usa-link" aria-describedby="R7">7</a></sup> (<a href="#F2" class="usa-link">Figure 2B</a>). Kinetic analyses compared the deamination rates of adenosine and AMP revealed that human ADA2 and the <em>Aplysia</em> ADGF favor adenosine over AMP by more than 100-fold and 1000-fold, respectively, as expected for ADGFs. In contrast, HPAMPD prefers AMP over adenine by 25-fold <sup><a href="#R7" class="usa-link" aria-describedby="R7">7</a></sup> (<a href="#F2" class="usa-link">Figure 2C</a>).</p>
<figure class="fig xbox font-sm" id="F2"><h4 class="obj_head">Figure 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11974800_nihpp-2025.03.26.645602v1-f0003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4c53/11974800/e87e847991ba/nihpp-2025.03.26.645602v1-f0003.jpg" loading="lazy" height="234" width="780" alt="Figure 2."></a></p>
<div class="p text-right font-secondary"><a href="figure/F2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P37">Substrate specificity of HPAMPD. (<strong>A</strong>) Relative rates of deamination for adenosine and AMP, adapted from <sup><a href="#R7" class="usa-link" aria-describedby="R7">7</a></sup>. (<strong>B</strong>) Deamination reaction of adenosine to inosine. (<strong>C</strong>) Deamination reactions of AMP and 6mAMP to IMP. (<strong>D</strong>) Pentostatin is a transition-state analog of adenosine deamination. (<strong>E</strong>) PMP is a transition-state analog of AMP (and 6mAMP) deamination.</p></figcaption></figure><p id="P9">To investigate the structural basis of HPAMPD substrate selectivity, we crystallized recombinant HPAMPD, which was purified from <em>P. pastoris</em> expression <sup><a href="#R7" class="usa-link" aria-describedby="R7">7</a></sup>. We obtained the apo-structure in the presence of zinc and also determined structures in complexes with pentostatin and pentostatin 5’-monophosphate (PMP). These structures were resolved at high resolutions ranging from 1.5-1.7 Å (<a href="#SD1" class="usa-link">Table S1</a>). Pentostatin and its 5’-monophosphorylated form PMP are anticancer chemotherapeutic drugs that are transition state analogs of adenosine deamination <sup><a href="#R31" class="usa-link" aria-describedby="R31">31</a></sup> (<a href="#F2" class="usa-link">Figure 2D</a>-<a href="#F2" class="usa-link">2E</a>).</p>
<p id="P10">In the crystallographic asymmetric unit of space group <em>P</em>2<sub>1</sub>, there are two dimers of HPAMPD (molecules A and B and molecules C and D) (<a href="#F3" class="usa-link">Figure 3A</a>). The dimer-dimer interface is mediated mainly by water molecules (red dashed line in <a href="#F3" class="usa-link">Figure 3A</a>). These dimers are highly similar to each other and to the human ADA2 dimer, with a root-mean-squared deviation (rmsd) of ~1.2 Å over 837 pairs of Cα atoms (<a href="#F3" class="usa-link">Figure 3B</a>). The corresponding secondary structures, consisting of 19 α-helices and 13 β-strands, are preserved throughout (<a href="#F1" class="usa-link">Figure 1B</a>). In the active site, the Zn ion is in a trigonal bipyramidal geometry <sup><a href="#R32" class="usa-link" aria-describedby="R32">32</a></sup> coordinated by three histidine residues (H121, H123, and H366) and one aspartate (D451) (<a href="#F3" class="usa-link">Figure 3C</a>). Additionally, a water molecule serves as the fifth ligand and acts as the attacking water nucleophile during the catalysis (<a href="#F3" class="usa-link">Figure 3D</a>).</p>
<figure class="fig xbox font-sm" id="F3"><h4 class="obj_head">Figure 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11974800_nihpp-2025.03.26.645602v1-f0004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4c53/11974800/f9b7230955e9/nihpp-2025.03.26.645602v1-f0004.jpg" loading="lazy" height="449" width="780" alt="Figure 3."></a></p>
<div class="p text-right font-secondary"><a href="figure/F3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P38">Structures of HPAMPD. (<strong>A</strong>) Two HPAMPD dimers in the <em>P</em>2<sub>1</sub>space group. (<strong>B</strong>) Superimposition of HPAMPD and human ADA2 dimers. (<strong>C</strong>) Active-site Zn coordination viewed from the Zn-bound water molecule (colored in red). (<strong>D</strong>) Omit electron densities contoured at 5σ for Zn and 5σ for the water. (<strong>I</strong>) Omit electron density contoured at 5σ for bound pentostatin. (<strong>E</strong>) The hydroxyl group of the coformycin ring occupies the position of the attacking water molecule. (<strong>F</strong>) Hydrogen bonds formed by the coformycin ring. (<strong>G</strong>) Aromatic and hydrophobic interactions stabilizing pentostatin binding. (<strong>H</strong>) Omit electron density contoured at 5σ for bound PMP. (<strong>I</strong>) Direct hydrogen bond between the phosphate group of PMP and Y227. (<strong>J</strong>) R232 forms a bridge between D124 and Y227. (<strong>K</strong>) Water-mediated interactions involving the phosphate group of PMP.</p></figcaption></figure><p id="P11">In the complex structure with pentostatin (<a href="#F3" class="usa-link">Figure 3E</a>), the hydroxyl group of the seven-membered coformycin ring occupies the position of the attacking water (<a href="#F3" class="usa-link">Figure 3F</a>). The configuration mimics the transition state generated by the zinc-activated hydroxyl attack on the face of adenine ring. The nitrogen atoms of the coformycin ring establish hydrogen bonds with the carboxylate groups of E369 and D452, as well as the main-chain amide group of A334 (<a href="#F3" class="usa-link">Figure 3G</a>). Additionally, the coformycin ring is stabilized through aromatic interactions, being sandwiched between W212 and F215 from above and H123 from below (<a href="#F3" class="usa-link">Figure 3H</a>). The ribose ring is positioned between F219 and V333. All residues involved in pentostatin binding are either identical or highly conserved among the three enzymes: human ADA2, <em>Aplysia</em> ADGF, and HPAMPD (<a href="#F1" class="usa-link">Figure 1B</a>).</p>
<p id="P12">Next, we examined the structure of HPAMPD bound to pentostatin 5’-monophosphate (PMP) (<a href="#F3" class="usa-link">Figure 3I</a>). All interactions observed with pentostatin are retained in PMP. The additional interaction involving the 5’-monophosphate group includes a direct hydrogen bond with the hydroxyl group of Y227 (<a href="#F3" class="usa-link">Figure 3J</a>), which is stabilized by R232 through a bridge with D124 (<a href="#F3" class="usa-link">Figure 3K</a>). Additionally, a network of water-mediated hydrogen bonds is formed with the side chains of D124, T125, R223 and S256 (<a href="#F3" class="usa-link">Figure 3L</a>). Among them, except for the invariant D124, five polar and positively charged residues are unique to HPAMPD involved in phosphate group binding (via direct H-bond or indirect water-mediated interactions) – T125 after β1, R223 from αH, Y227 located in the loop between αH and αI, R232 from αI and S256 after β5 (<a href="#F1" class="usa-link">Figure 1B</a>). In contrast, the corresponding residues are hydrophobic in <em>Aplysia</em> ADGF (Leu, Ile, Phe, Met and Phe), and in human ADA2 they are Ile, Ser, His, Phe, and Leu (<a href="#F1" class="usa-link">Figure 1B</a>). These hydrophobic and/or smaller residues are less suited for electrostatic interactions with the negatively charged phosphate group, highlighting a structural adaptation unique to HPAMPD.</p>
<p id="P13">The 3’ hydroxyl (OH) group of the ribose moiety is in van der Waals contact with V333 (<a href="#F3" class="usa-link">Figure 3J</a>). The addition of a 3’-monophosphate creates a repulsive force with V333 (<a href="#SD1" class="usa-link">Figure S1A</a>), leading to the loss of HPAMPD activity on 3’-AMP, 3’,5’-adenosine diphosphate, and 3’,5’-cyclic AMP <sup><a href="#R7" class="usa-link" aria-describedby="R7">7</a></sup>. In contrast, the extension of 5’-AMP (100% activity) to 5’-ADP (74-95% activity) preserves function <sup><a href="#R7" class="usa-link" aria-describedby="R7">7</a></sup> by orienting the additional 5’-phosphate group away from the active site and toward the solvent (<a href="#SD1" class="usa-link">Figure S1B</a>). Additionally, HPAMPD maintains moderate activity (25-36%) on 5’-ATP <sup><a href="#R7" class="usa-link" aria-describedby="R7">7</a></sup> (<a href="#SD1" class="usa-link">Figure S1C</a>).</p></section><section id="S4"><h3 class="pmc_sec_title">Generation of chimeric ADGF with altered activity</h3>
<p id="P14">To clarify the origin of substrate selectivity, we hypothesized that residues near the phosphate group were responsible for binding PMP. To test this, we first generated Chimera-1 by replacing two short segments within the backbone of <em>Aplysia</em> ADGF with the corresponding HPAMPD residues: the six-residue sequence immediately after β1: L<span class="text-underline">A</span>I<span class="text-underline">A</span>SL in <em>Aplysia</em> was substituted with T<span class="text-underline">A</span>S<span class="text-underline">A</span>RI (residues 125-130 of HPAMPD), and the eight-residue sequence of strand β5 and its adjacent loop: I<span class="text-underline">E</span>L<span class="text-underline">RS</span>Q<span class="text-underline">L</span>F in <em>Aplysia</em> was replaced with V<span class="text-underline">E</span>V<span class="text-underline">RS</span>S<span class="text-underline">L</span>S (residues 251-258 of HPAMPD). These sequences differ between the two enzymes, with 4 out of 6 and 4 out of 8 residues diverging in each segment, respectively, including residues T125 and S256, which are involved in water-mediated phosphate interactions (<a href="#F3" class="usa-link">Figure 3K</a>). However, the chimeric <em>Aplysia</em> enzyme (Chimera-1) did not recognize AMP as a substrate. With adenosine as substrate, it exhibited activity similar to that of wild type <em>Aplysia</em> ADGF, which was more than 100 times higher than its activity on AMP (<a href="#F4" class="usa-link">Figure 4A</a>).</p>
<figure class="fig xbox font-sm" id="F4"><h4 class="obj_head">Figure 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11974800_nihpp-2025.03.26.645602v1-f0005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4c53/11974800/86eef63a8a5a/nihpp-2025.03.26.645602v1-f0005.jpg" loading="lazy" height="671" width="720" alt="Figure 4."></a></p>
<div class="p text-right font-secondary"><a href="figure/F4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P39">Substrate specificity of engineered chimera ADGF. (<strong>A</strong>) Relative deamination rates of for 100 μM adenosine and AMP for engineered chimera enzymes, along with wild-type <em>Aplysia</em> and <em>Helix</em> ADGF, depicted in cartoon representations to the right. (<strong>B</strong>) Enzymatic activities of the Chimera-2 variant. (<strong>C</strong>) Dimeric structure of Chimera-2. (<strong>D</strong>) Superimposition of Chimera-2 and HPAMPD dimers. (<strong>E</strong>) Superimposition of Chimera-2 and human ADA2 dimers. (<strong>F</strong>) Bound conformation of pentostatin in the active sites of Chimera-2 and HPAMPD. (<strong>G</strong>) Interactions between pentostatin and <em>Aplysia</em> ADGF-specific residues (L296 and G329 colored in blue). (<strong>H</strong>) Bound PMP in the active sites of Chimera-2 and HPAMPD. (<strong>I</strong>) Phosphate interaction within the AMP-bound active site of human AMPD2 (PDB 8HU6). (<strong>J</strong>) Phosphate interaction within the active site of human ADAL1/MAPDA bound with an AMP analog (PDB 8QCH). (<strong>K</strong>) Phosphate interaction within the active site of <em>Arabidopsis</em> MAPDA with a bound 6mAMP (PDB 6IJN)</p></figcaption></figure><p id="P15">Next, we replaced the entire region (from L<span class="text-underline">A</span>I<span class="text-underline">A</span>SL through I<span class="text-underline">E</span>L<span class="text-underline">RS</span>Q<span class="text-underline">L</span>F) with residues 125-258 of HPAMPD, creating Chimera-2. The engineered Chimera-2 efficiently deaminates AMP with a <em>K</em><sub>M</sub> of 135 μM and <em>k</em><sub>cat</sub> of 33 s<sup>−1</sup> and deaminates adenosine with a <em>K</em><sub>M</sub> of 104 μM and <em>k</em><sub>cat</sub> of 22 s<sup>−1</sup> (<a href="#F4" class="usa-link">Figure 4B</a>), demonstrating altered substrate specificity. Notably, Chimera-2 retains the characteristic of adenosine deamination activity of <em>Aplysia</em> ADGF. To explore potential structural differences, we crystallized Chimera-2 in its apo form as well as in complexes with pentostatin and PMP (<a href="#SD1" class="usa-link">Table S1</a>).</p>
<p id="P16">Like human ADA2, Chimera-2 adopts a dimeric structure (<a href="#F4" class="usa-link">Figure 4C</a>). As expected, Chimera-2 closely resembles HPAMPD (rmsd=0.44 Å) and human ADA2 (rmsd=1.15 Å) in overall structures (<a href="#F4" class="usa-link">Figure 4D</a>-<a href="#F4" class="usa-link">4E</a>). The bound pentostatin has a similar conformation in the active sites of both Chimera-2 and HPAMPD, with all conserved residues aligning perfectly (<a href="#F4" class="usa-link">Figure 4F</a>). Additionally, some interactions occur with <em>Aplysia</em> ADGF-specific residues, such as Leu296 (which corresponds to Ala301 in HPAMPD) (<a href="#F4" class="usa-link">Figure 4G</a>). The binding mode of PMP is also identical between Chimera-2 and HPAMPD, including the coordination of the phosphate group by polar residues (<a href="#F4" class="usa-link">Figure 4H</a>).</p>
<p id="P17">In addition to ADA1 and ADA2, humans have three AMP deaminase genes: AMPD1 expressed in muscle (UnitProt: <a href="https://www.ncbi.nlm.nih.gov/protein/P23109" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P23109</a>), AMPD2 in liver (Unit Prot: <a href="https://www.ncbi.nlm.nih.gov/protein/Q01433" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q01433</a>), and AMPD3 in erythrocytes (UnitProt: <a href="https://www.ncbi.nlm.nih.gov/protein/Q01432" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q01432</a>) <sup><a href="#R33" class="usa-link" aria-describedby="R33">33</a></sup> (<a href="#F1" class="usa-link">Figure 1A</a>). The AMPD enzymes are significantly larger in size (comparing 825 residues in AMPD2 and 511 residues in ADA2), yet they share a conserved active site. In a recent structure of AMPD2 bound to AMP <sup><a href="#R34" class="usa-link" aria-describedby="R34">34</a></sup>, the negatively phosphate group of AMP is surrounded by positively charged residues (R503 and K508), polar residues (S422, S542 and Y544), and ordered water molecules (<a href="#F4" class="usa-link">Figure 4I</a>).</p>
<p id="P18">Moreover, X-ray structures are available for 6mAMP deaminases, including human ADAL1 <sup><a href="#R35" class="usa-link" aria-describedby="R35">35</a></sup> and <em>Arabidopsis</em> homolog of MAPDA <sup><a href="#R36" class="usa-link" aria-describedby="R36">36</a>,<a href="#R37" class="usa-link" aria-describedby="R37">37</a></sup>. In both cases, a set of polar residues (Ser, Thr, Asn, His and Tyr), along with a positively charged lysine, interact with the monophosphate group of either the guanosine analog AT8001 (<a href="#F4" class="usa-link">Figure 4J</a>) or the substrate 6mAMP (<a href="#F4" class="usa-link">Figure 4K</a>). Although these phosphate-interacting residues are not perfectly conserved in primary sequences (<a href="#SD1" class="usa-link">Figure S2</a> and <a href="#SD1" class="usa-link">S3</a>), the four enzymes compared - HPAMPD, human AMPD2, human ADAL1, and <em>Arabidopsis</em> MAPDA - utilize similar interactions for AMP binding. These residues are positioned in conserved structural regions, including the loop immediately following strand β1, the loop between helices αH and αI, and the end of strand β5 and its adjacent loop (<a href="#SD1" class="usa-link">Figure S2</a> and <a href="#SD1" class="usa-link">S3</a>). This structural similarity is evident only through structured-based alignments due to large insertions and deletions between secondary structural elements β1 to β5. As demonstrated by our Chimera-2 enzyme, the entire β1 to β5 region of HPAMPD, containing all three structural elements, is essential for AMP binding and activity.</p></section><section id="S5"><h3 class="pmc_sec_title">Discrimination between AMP and 6mAMP</h3>
<p id="P19">While the deamination of AMP to generates IMP and ammonia, the deamination of N6-methyl-AMP (6mAMP) also generates IMP and methylamine (<a href="#F3" class="usa-link">Figure 3B</a>). In human, a specific 6mAMP deaminase (MAPDA), also known as ADA-like protein 1 (ADAL1) (UnitProt: <a href="https://www.ncbi.nlm.nih.gov/protein/Q6DHV7" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q6DHV7</a>), carries out this reaction (<a href="#F1" class="usa-link">Figure 1A</a>) <sup><a href="#R38" class="usa-link" aria-describedby="R38">38</a></sup>. The <em>Arabidopsis</em> homolog of MAPDA is highly specific for 6mAMP and does not act on adenosine, N6-methyladenosine, AMP, N6-methyl-ATP and O6-methyl-guanosine <sup><a href="#R39" class="usa-link" aria-describedby="R39">39</a></sup>. Interestingly, we found that HPAMPD is also capable of deaminating 6mAMP, albeit at a much slower rate. The relative activity for AMP, adenosine, and 6mAMP follows a ratio of 1: 0.04: 0.013 (<a href="#F5" class="usa-link">Figure 5A</a>). In Chimera-2 the residues surrounding the N6 position of adenine ring have not been altered. Despite this, the 10-fold difference in 6mAMP activity between Chimera-2 and HPAMPD is not fully explained by the current structural data. The deamination rate of 6mAMP in Chimera-2 is 1000-fold slower than that of AMP (<a href="#F5" class="usa-link">Figure 5A</a>).</p>
<figure class="fig xbox font-sm" id="F5"><h4 class="obj_head">Figure 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11974800_nihpp-2025.03.26.645602v1-f0006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4c53/11974800/7458846f0c5d/nihpp-2025.03.26.645602v1-f0006.jpg" loading="lazy" height="198" width="720" alt="Figure 5."></a></p>
<div class="p text-right font-secondary"><a href="figure/F5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P40">HPAMPD and Chimera-2 have weak activity on 6mAMP. (<strong>A</strong>) Relative deamination rates of AMP, adenosine, and 6mAMP by HPAMPD and Chimera-2. The graph displays the different deamination rates of Chimera-2 on AMP and 6mAMP at two enzyme concentrations. (<strong>B</strong>) Binding of the N6-methyl group (in yellow sphere) of 6mAMP in the inactive mutant (D295N) of <em>Arabidopsis</em> MAPDA (PDB 6IJN). (<strong>C</strong>) Superimposition of human ADAL1 (PDB 8QCH) and its <em>Arabidopsis</em> homolog (PDB 6IJN). (<strong>D</strong>) HPAMPD, which exhibits low activity on 6mAMP, and human AMPD2, which has no known activity on 6mAMP, share three large aromatic residues and one leucine that form an aromatic cage.</p></figcaption></figure><p id="P20">In the structure of the catalytically inactive mutant (D295N) of the <em>Arabidopsis</em> MAPDA homolog <sup><a href="#R36" class="usa-link" aria-describedby="R36">36</a></sup>, which permits substrate 6mAMP binding without undergoing catalysis, the N6-methyl group is positioned out of plane with the adenine ring, oriented opposite to the attacking nucleophile, and forms van der Waals contacts with residues L55, V57, T269, and F300 (<a href="#F5" class="usa-link">Figure 5B</a>). The corresponding residues in human ADAL1 are L63, C66, S267, and F298 (<a href="#F5" class="usa-link">Figure 5C</a>). Although only two of these four residues (Leu and Phe) are conserved, the variation (Val vs. Cys and Thr vs. Ser) involve residues of similar size. In contrast, three of four corresponding residues in HPAMPD - W212, F215, L425, and W456 - are larger aromatic residues, which may contribute to its low activity on 6mAMP (<a href="#F5" class="usa-link">Figure 5A</a>). These residues form an aromatic pocket surrounding the C9 carbon atom of the non-planar coformycin ring (<a href="#F5" class="usa-link">Figure 5D</a>). Similarly, in human AMPD2, which has no known activity on 6mAMP, the equivalent residues consist of three phenylalanines (F433, F436, F715) and one leucine (L686) (<a href="#SD1" class="usa-link">Figure S2</a>). We propose that the size of these four residues may play a key role in determining deaminase activity on 6mAMP.</p></section></section><section id="S6"><h2 class="pmc_sec_title">DISCUSSION</h2>
<p id="P21">Adenosine deaminase deficiency is a known cause of <span class="text-underline">s</span>evere <span class="text-underline">c</span>ombined <span class="text-underline">i</span>mmuno<span class="text-underline">d</span>eficiency (SCID) <sup><a href="#R40" class="usa-link" aria-describedby="R40">40</a>,<a href="#R41" class="usa-link" aria-describedby="R41">41</a></sup>. Humans possess at least six adenylate deaminases, - ADA1/2, AMPD1-3, and MAPDA/ADAL1 – each with distinct substrate specificities for adenosine, AMP, and 6mAMP, respectively (summarized in <a href="#F1" class="usa-link">Figure 1A</a>). While no known methyltransferase modifies AMP, N6-methylation of adenine in specific RNA sequences is the most common internal mRNA modification in eukaryotes <sup><a href="#R42" class="usa-link" aria-describedby="R42">42</a>,<a href="#R43" class="usa-link" aria-describedby="R43">43</a></sup>. When 6mA-containing RNA is degraded in the cytoplasm, 6mAMP is released. The deamination of 6mAMP may prevent its conversion to 6mADP and eventually 6mATP, which could otherwise be randomly incorporated into newly synthesized RNA molecules by RNA polymerase <sup><a href="#R39" class="usa-link" aria-describedby="R39">39</a>,<a href="#R44" class="usa-link" aria-describedby="R44">44</a></sup>.</p>
<p id="P22">Interestingly, the foot muscle of the mollusk <em>H. pomatia</em> is one of the richest sources of “non-specific” adenylate deaminase(s), exhibiting activities on AMP, ADP, ATP and NADH <sup><a href="#R45" class="usa-link" aria-describedby="R45">45</a>,<a href="#R46" class="usa-link" aria-describedby="R46">46</a></sup>. The recombinant form of HPAMPD shares a similar substrate spectrum to the native enzyme, which has been purified at least twice from <em>H. pomatia</em> foot muscle - first in 1983 <sup><a href="#R45" class="usa-link" aria-describedby="R45">45</a></sup> and again in 2023 <sup><a href="#R7" class="usa-link" aria-describedby="R7">7</a></sup>. Structurally, HPAMPD adopts a fold similar to human ADA2, a nucleoside adenosine deaminase, but has acquired the ability to interact with the 5’-monophosphate group through polar and charged residues located in three key structural elements: (1) the loop immediately following strand β1, (2) the loop between helices αH and αI, and (3) the end of strand β5 and its adjacent loop. The phosphate-binding feature is characteristic of nucleotide deaminases, such as human AMP and 6mAMP deaminases. We note that the <em>Aplysia</em> ADGF discriminates by 2000-fold against adenosine when the N6 position is methylated (6mAMP), while HPAMPD discriminates only about 100-fold against 6mAMP. Interestingly, Chimera-2 while accepting the phosphorylated substrate, which is a property of HPAMPD, discriminated by a 1000-fold against 6mAMP, a property similar to that of the <em>Aplysia</em> ADGF towards deamination of 6mAMP.</p>
<p id="P23">As mentioned above, the deamination of AMP (or its 2’-deoxy form, dAMP) and 6mAMP (or its 2’-deoxy form, 6m(d)AMP) produces the common inosine derivatives IMP (or dIMP). A recent study suggested that dIMP may serve as a precursor for immune messengers in bacterial anti-phage defense <sup><a href="#R47" class="usa-link" aria-describedby="R47">47</a></sup>. In <em>Escherichia coli</em> NCTC13216, dIMP production begins with the deamination of dAMP by a host bacterial enzyme called KomA, which works in conjunction with deoxynucleotide phosphate kinases to generate dIDP and ultimately dITP as signaling molecules<sup><a href="#R47" class="usa-link" aria-describedby="R47">47</a></sup>. KomA, a small protein of 326 amino acids, shares the highest sequence similarity with human ADAL1 (<a href="#SD1" class="usa-link">Figure S5A</a>). The AlphaFold-predicted structure <sup><a href="#R48" class="usa-link" aria-describedby="R48">48</a></sup> of KomA aligns closely with experimentally determined structures of human ADAL1 and <em>Arabidopsis</em> MAPDA, with an RMSD of 1.7-1.8 Å (<a href="#SD1" class="usa-link">Figure S5B</a>-<a href="#SD1" class="usa-link">D</a>). While KomA exhibits <em>in vitro</em> activity on both 2’-hydroxy and 2’-deoxy forms of AMP and ADP <sup><a href="#R47" class="usa-link" aria-describedby="R47">47</a></sup>, its high structural similarity and comparable size to 6mAMP deaminases (ADAL1/MAPDA) suggest that its activity on 6mAMP, 6m(d)AMP as well as potential activity on DNA or RNA, still needs to be investigated. Notably, both bacterial and phage DNA can undergo extensive adenine hypermethylation, primarily as part of the restriction-modification defense system that protects against exogenous DNA <sup><a href="#R49" class="usa-link" aria-describedby="R49">49</a>-<a href="#R52" class="usa-link" aria-describedby="R52">52</a></sup>. This widespread adenine methylation may have functional implications for KomA-mediated immune defense. Recently the deamination of deoxyadenosine in DNA in vitro was described for human ADA2, potentially involved in regulating immune sensing of DNA <sup><a href="#R30" class="usa-link" aria-describedby="R30">30</a></sup>.</p></section><section id="S7"><h2 class="pmc_sec_title">Material and methods</h2>
<section id="S8"><h3 class="pmc_sec_title">Protein purification</h3>
<p id="P24">The HPAMPD and <em>Aplysia</em> ADGF proteins were prepared as described <sup><a href="#R7" class="usa-link" aria-describedby="R7">7</a></sup>. The Chimera-1 and Chimera-2 ADGFs used in this study were all expressed in <em>Pichia pastoris</em> by integrating an expression cassette coding for each and were cultured as described <sup><a href="#R7" class="usa-link" aria-describedby="R7">7</a></sup>. The Chimera-1 protein was directly assayed from the clarified spent culture medium.</p>
<p id="P25"><em>P. pastoris</em> harboring the integrated Chimera-2 expression cassette was cultured in 2400 ml of Tolner media <sup><a href="#R53" class="usa-link" aria-describedby="R53">53</a></sup> with 1% glycerol and 100 μg/ml ampicillin at 30° C. The secreted expression of AMP deaminase activity was measured over time of culturing until culture fluid expressed highest level of secreted AMP deaminase activity. At the time of the highest expression the culture was clarified by centrifugation and the supernatant culture fluid was diluted 2-fold with 20 mM Na-acetate pH 6.0, applied to a Heparin Hyper D column and eluted with a gradient from 50 mM to 1 M NaCl. The peak of AMP deaminase activity was diluted 8-fold into 20 mM Tris-HCl and applied to a HiTrap<sup>™</sup> Q column. Activity was eluted with a gradient from 50 mM to 1 M NaCl. The peak of activity was pooled and was treated with a recombinant protein fusion of Endoglycosidase H and maltose binding protein (Endo H<sub>f</sub>, NEB) by first dialyzing against 20 mM Na acetate pH 6.0, 1 mM EDTA, 50 mM NaCl, 1 mM DTT. Endo H<sub>f</sub> was added to the dialysate and incubated at 25° C overnight. Endo H<sub>f</sub> was removed from the preparation by passing the supernatant through amylose resin (NEB). The flow-through was applied to a HiTrap<sup>™</sup> SP column. The flow-through from the SP column was diluted threefold in 10 mM Tris-HCl (pH 7.5) before being loaded onto a HiTrap<sup>™</sup> Q column. The peak of Chimera-2 activity was pooled and dialyzed against 50% glycerol, 10 mM Tris-HCl pH 7.4, 1 mM DTT, 0.1 mM EDTA. The Chimera-2 preparation is stored at −20° C prior to use.</p></section><section id="S9"><h3 class="pmc_sec_title">Construction of Chimera-1 and Chimera-2</h3>
<p id="P26">The plasmid carrying the HPAMPD is described <sup><a href="#R7" class="usa-link" aria-describedby="R7">7</a></sup>. The plasmid PD912-GAP carrying the <em>Aplysia</em> ADGF was constructed by Genscript (Piscataway, NJ) (<a href="#SD1" class="usa-link">Table S2</a> for plasmid sequence). The chimeras were constructed from two fragments that were generated by PCR. One linear fragment was common to both constructs and was generated by PCR using the <em>Aplysia</em> ADGF plasmid as template with primers TRA and EFA. The other fragment which was specific for Chimera-1 was generated by PCR using the same template with primers TFA and ERA (<a href="#SD1" class="usa-link">Table S2</a>). The two fragments for Chimera-1 were assembled with NEBuilder HiFi DNA Assembly. The fragment specific for Chimera-2 was generated by PCR using the HPAMPD plasmid as template with primers TFH and ERH. Chimera-2 was assembled from the common fragment and Chimera-2 specific fragment with NEBuilder HiFi DNA Assembly. Construction of the expression cassettes and the transformation of <em>P. pastoris</em> followed the strategy described <sup><a href="#R7" class="usa-link" aria-describedby="R7">7</a></sup>.</p></section><section id="S10"><h3 class="pmc_sec_title">Deaminase Activity and binding Assays</h3>
<p id="P27">The assay is based on the classic Kalckar spectrophotometric method <sup><a href="#R54" class="usa-link" aria-describedby="R54">54</a></sup>, which measures the decrease in optical density at 265 nM. The reaction buffer is 20 mM Tris-HCl at pH 7.5, 50 mM NaCl and the substrate concentration is 100 μM (AMP or adenosine or 6mAMP). The reaction volume ranged from 200 to 400 μL. The optical density measurement was determined using a SpectraMax M3 spectrophotometer.</p></section><section id="S11"><h3 class="pmc_sec_title">Crystallography</h3>
<p id="P28">Purified HPAMPD, originally in 50% glycerol, 10 mM Tris pH 7.4, 1 mM DTT and 0.1 mM EDTA was buffer exchanged to 10 mM Tris pH 7.4, 50 mM NaCl and concentrated to ~77 μM. HPAMPD (50 μM) was incubated with and without pentostatin or PMP in 1:3 molar ratio (up to 150 μM pentostatin or PMP) independently in buffer 10 mM Tris pH 7.4, 50 mM NaCl for 30 min at room temperature and crystallization trays were setup using sitting drop vapor diffusion method using commercial screens, Crystals of apo HPAMPD and in complex with pentostatin were observed in 0.1 M BIS-TRIS pH 6.5 and 20% (w/v) polyethylene glycol (PEG) monomethyl ether 5000 whereas crystals of HPAMPD with PMP were observed in 0.1 M HEPES pH 7.5 and 25% (w/v) PEG 3350 within 1 day of setup. All crystals were soaked in cryoprotectant containing 20% ethylene glycol supplemented in the mother liquors followed by flash freezing in liquid nitrogen.</p>
<p id="P29">Purified Chimera-2 was diluted 5 times in buffer [20 mM Tris pH 7.4, 100 mM NaCl, 1 mM DTT, 0.1 mM EDTA] and concentrated to ~10 mg/ml (~85 μM). Crystallization screens of purified Chimera-2 protein were setup at 2.5, 5 and 10 mg/ml using commercial crystallization screens (Hampton research). Needle shaped crystals of apo Chimera-2 were observed at all three concentrations tested in multiple conditions within 24 h, and crystals obtained in 0.1 M sodium acetate trihydrate pH 4.5, 25% PEG3350 resulted in the best dataset.</p>
<p id="P30">Chimera-2 enzymes (50 μM) were mixed with pentostatin or PMP (150 μM each) in 1:3 molar ratio independently in buffer 20 mM Tris pH 7.4, 100 mM NaCl, 1 mM DTT for 30 min at room temperature and crystallization screens were setup using sitting drop vapor diffusion method (0.2 μl protein solution +0.2 μl reservoir solution). Needle shaped crystals of Chimera-2+pentostatin and Chimera-2+PMP were observed in multiple conditions within 24 h. The conditions were further optimized by varying the percentage of PEG 1500 (15-25%) and glycerol (5-25%) and crystallization drops were setup manually. The best crystals of Chimera-2+pentostatin were obtained in 15% glycerol and 25% PEG 1500 after 15 days of setup and for Chimera-2+PMP were obtained in 0.2 M magnesium chloride hexahydrate, 0.1 M BIS-TRIS pH 5.5 and 25% (w/v) PEG 3350 after 2 days.</p>
<p id="P31">Diffraction data were collected at the AMX beamline 17-1D-1 of NSLS-II at Brookhaven National Laboratory and SERCAT beamline 22ID of APS at Argonne National Laboratory. Data were processed with HKL2000 <sup><a href="#R55" class="usa-link" aria-describedby="R55">55</a></sup> and with DIALS <sup><a href="#R56" class="usa-link" aria-describedby="R56">56</a>,<a href="#R57" class="usa-link" aria-describedby="R57">57</a></sup> and AIMLESS <sup><a href="#R58" class="usa-link" aria-describedby="R58">58</a></sup>. All the crystal structures were solved by molecular replacement using either search model of human ADA2 (PDB 3LGG) <sup><a href="#R28" class="usa-link" aria-describedby="R28">28</a></sup> or an AlphaFold <sup><a href="#R48" class="usa-link" aria-describedby="R48">48</a></sup> predicted model from COLAB server <sup><a href="#R59" class="usa-link" aria-describedby="R59">59</a></sup>. Phenix Refine <sup><a href="#R60" class="usa-link" aria-describedby="R60">60</a></sup> was used for all the rounds of refinements and the quality of the structure was improved manual inspection and model (re)building with COOT <sup><a href="#R61" class="usa-link" aria-describedby="R61">61</a>,<a href="#R62" class="usa-link" aria-describedby="R62">62</a></sup> along with iterative rounds of Phenix refinements. All the structures were validated by PDB validation server <sup><a href="#R63" class="usa-link" aria-describedby="R63">63</a></sup> and images were generated using Open-Source PyMOL molecular graphics system, version 3.0.3, Schrodinger, LLC. Symmetry related molecules were identified, and their dimer interface areas were calculated using PISA webserver at the European Bioinformatics Institute <sup><a href="#R64" class="usa-link" aria-describedby="R64">64</a></sup>.</p></section></section><section id="SM1"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="SD1"><div class="caption p"><span>Supplement 1</span></div>
<div class="media p" id="d67e1035"><div class="caption">
<a href="/articles/instance/11974800/bin/media-1.pdf" data-ga-action="click_feat_suppl" class="usa-link">media-1.pdf</a><sup> (3.3MB, pdf) </sup>
</div></div></section></section><section id="S13" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p id="P33">We thank Daniel Kneller, Larry McReynolds, and Zhi-Yi Sun of New England Biolabs (NEB) for comments. We thank M. Wulf (NEB) for providing PMP and L. Liang (NEB) for P. pastoris fermentations. We thank X. Kong (New York University) for assistance of access to 17-ID-1 beamtime. We thank the beamline scientists at the National Synchrotron Light Source II, Brookhaven National Laboratory and the Southeast Regional Collaborative Access Team (SER-CAT) at the Advanced Photon Source (APS), Argonne National Laboratory, USA. The use of SER-CAT is supported by its member institutions and equipment grants (S10_RR25528, S10_RR028976, and S10_OD027000) from the US National Institutes of Health. Use of the APS was supported by the U.S. Department of Energy, Office of Science, Office of Basic Energy Sciences, under contract W-31-109-Eng-38. The work was supported by New England Biolabs (to I.S.), U.S. National Institutes of Health grant R35GM134744 (to X.C.), and Cancer Prevention and Research Institute of Texas grant RR160029 (to X.C., who is a CPRIT Scholar in Cancer Research).</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="FN1">
<p id="P34">Competing Interests</p>
<p id="P35">G.T. and I.S. are employees of New England Biolabs, Inc, a manufacturer and vendor of molecular biology reagents. This affiliation does not affect the authors’ impartiality, adherence to journal standards and policies, or availability of data. The other authors declare no competing interests.</p>
</div></div></section><section id="S12"><h2 class="pmc_sec_title">Data Availability</h2>
<p id="P32">The protein sequence of HPAMPD has been deposited to GenBank with accession number <a href="https://ncbi.nlm.nih.gov/nucleotide/XOL57624.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XOL57624.1</a>. The X-ray structures (coordinates and structure factor files) reported here have been deposited to PDB and are publicly available as of the date of publications. PDB accession numbers are 9NTE and 9NTF (HPAMPD in the apo form), 9NTG (HPAMPD in complex with pentostatin), 9NTH (HPAMPD in complex with PMP), 9NTI (Chimera-2 in the apo-form), 9NTJ (Chimera-2 in complexes with pentostatin), and 9NTK (Chimera-2 in complex with PMP).</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="R1">
<span class="label">1.</span><cite>Maier S.A., Galellis J.R., and McDermid H.E. (2005). Phylogenetic analysis reveals a novel protein family closely related to adenosine deaminase. Journal of molecular evolution
61, 776–794. 10.1007/s00239-005-0046-y.
</cite> [<a href="https://doi.org/10.1007/s00239-005-0046-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16245011/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20molecular%20evolution&amp;title=Phylogenetic%20analysis%20reveals%20a%20novel%20protein%20family%20closely%20related%20to%20adenosine%20deaminase&amp;author=S.A.%20Maier&amp;author=J.R.%20Galellis&amp;author=H.E.%20McDermid&amp;volume=61&amp;publication_year=2005&amp;pages=776-794&amp;pmid=16245011&amp;doi=10.1007/s00239-005-0046-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R2">
<span class="label">2.</span><cite>Homma K., Matsushita T., and Natori S. (1996). Purification, characterization, and cDNA cloning of a novel growth factor from the conditioned medium of NIH-Sape-4, an embryonic cell line of Sarcophaga peregrina (flesh fly). J Biol Chem
271, 13770–13775. 10.1074/jbc.271.23.13770.
</cite> [<a href="https://doi.org/10.1074/jbc.271.23.13770" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8662793/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Purification,%20characterization,%20and%20cDNA%20cloning%20of%20a%20novel%20growth%20factor%20from%20the%20conditioned%20medium%20of%20NIH-Sape-4,%20an%20embryonic%20cell%20line%20of%20Sarcophaga%20peregrina%20(flesh%20fly)&amp;author=K.%20Homma&amp;author=T.%20Matsushita&amp;author=S.%20Natori&amp;volume=271&amp;publication_year=1996&amp;pages=13770-13775&amp;pmid=8662793&amp;doi=10.1074/jbc.271.23.13770&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R3">
<span class="label">3.</span><cite>Homma K.J., Tanaka Y., Matsushita T., Yokoyama K., Matsui H., and Natori S. (2001). Adenosine deaminase activity of insect-derived growth factor is essential for its growth factor activity. J Biol Chem
276, 43761–43766. 10.1074/jbc.M105088200.
</cite> [<a href="https://doi.org/10.1074/jbc.M105088200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11562360/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Adenosine%20deaminase%20activity%20of%20insect-derived%20growth%20factor%20is%20essential%20for%20its%20growth%20factor%20activity&amp;author=K.J.%20Homma&amp;author=Y.%20Tanaka&amp;author=T.%20Matsushita&amp;author=K.%20Yokoyama&amp;author=H.%20Matsui&amp;volume=276&amp;publication_year=2001&amp;pages=43761-43766&amp;pmid=11562360&amp;doi=10.1074/jbc.M105088200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R4">
<span class="label">4.</span><cite>Charlab R., Valenzuela J.G., Andersen J., and Ribeiro J.M. (2001). The invertebrate growth factor/CECR1 subfamily of adenosine deaminase proteins. Gene
267, 13–22. 10.1016/s0378-1119(01)00393-6.
</cite> [<a href="https://doi.org/10.1016/s0378-1119(01)00393-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11311551/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=The%20invertebrate%20growth%20factor/CECR1%20subfamily%20of%20adenosine%20deaminase%20proteins&amp;author=R.%20Charlab&amp;author=J.G.%20Valenzuela&amp;author=J.%20Andersen&amp;author=J.M.%20Ribeiro&amp;volume=267&amp;publication_year=2001&amp;pages=13-22&amp;pmid=11311551&amp;doi=10.1016/s0378-1119(01)00393-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R5">
<span class="label">5.</span><cite>Zavialov A.V., and Engstrom A. (2005). Human ADA2 belongs to a new family of growth factors with adenosine deaminase activity. Biochem J
391, 51–57. 10.1042/BJ20050683.
</cite> [<a href="https://doi.org/10.1042/BJ20050683" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1237138/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15926889/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem%20J&amp;title=Human%20ADA2%20belongs%20to%20a%20new%20family%20of%20growth%20factors%20with%20adenosine%20deaminase%20activity&amp;author=A.V.%20Zavialov&amp;author=A.%20Engstrom&amp;volume=391&amp;publication_year=2005&amp;pages=51-57&amp;pmid=15926889&amp;doi=10.1042/BJ20050683&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R6">
<span class="label">6.</span><cite>Akalal D.B., and Nagle G.T. (2001). Mollusk-derived growth factor: cloning and developmental expression in the central nervous system and reproductive tract of Aplysia. Brain Res Mol Brain Res
91, 163–168. 10.1016/s0169-328x(01)00127-9.
</cite> [<a href="https://doi.org/10.1016/s0169-328x(01)00127-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11457505/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brain%20Res%20Mol%20Brain%20Res&amp;title=Mollusk-derived%20growth%20factor:%20cloning%20and%20developmental%20expression%20in%20the%20central%20nervous%20system%20and%20reproductive%20tract%20of%20Aplysia&amp;author=D.B.%20Akalal&amp;author=G.T.%20Nagle&amp;volume=91&amp;publication_year=2001&amp;pages=163-168&amp;pmid=11457505&amp;doi=10.1016/s0169-328x(01)00127-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R7">
<span class="label">7.</span><cite>Tzertzinis G., Ganatra M.B., Ruse C., Taron C.H., Causey B., Wang L., and Schildkraut I. (2023). The AMP deaminase of the mollusk Helix pomatia is an unexpected member of the adenosine deaminase-related growth factor (ADGF) family. PLoS One
18, e0286435. 10.1371/journal.pone.0286435.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0286435" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10358891/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37471401/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=The%20AMP%20deaminase%20of%20the%20mollusk%20Helix%20pomatia%20is%20an%20unexpected%20member%20of%20the%20adenosine%20deaminase-related%20growth%20factor%20(ADGF)%20family&amp;author=G.%20Tzertzinis&amp;author=M.B.%20Ganatra&amp;author=C.%20Ruse&amp;author=C.H.%20Taron&amp;author=B.%20Causey&amp;volume=18&amp;publication_year=2023&amp;pages=e0286435&amp;pmid=37471401&amp;doi=10.1371/journal.pone.0286435&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R8">
<span class="label">8.</span><cite>Giblett E.R., Anderson J.E., Cohen F., Pollara B., and Meuwissen H.J. (1972). Adenosine-deaminase deficiency in two patients with severely impaired cellular immunity. Lancet
2, 1067–1069. 10.1016/s0140-6736(72)92345-8.
</cite> [<a href="https://doi.org/10.1016/s0140-6736(72)92345-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4117384/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Adenosine-deaminase%20deficiency%20in%20two%20patients%20with%20severely%20impaired%20cellular%20immunity&amp;author=E.R.%20Giblett&amp;author=J.E.%20Anderson&amp;author=F.%20Cohen&amp;author=B.%20Pollara&amp;author=H.J.%20Meuwissen&amp;volume=2&amp;publication_year=1972&amp;pages=1067-1069&amp;pmid=4117384&amp;doi=10.1016/s0140-6736(72)92345-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R9">
<span class="label">9.</span><cite>Schrader W.P., Pollara B., and Meuwissen H.J. (1978). Characterization of the residual adenosine deaminating activity in the spleen of a patient with combined immunodeficiency disease and adenosine deaminase deficiency. Proc Natl Acad Sci U S A
75, 446–450. 10.1073/pnas.75.1.446.
</cite> [<a href="https://doi.org/10.1073/pnas.75.1.446" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC411266/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24216/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Characterization%20of%20the%20residual%20adenosine%20deaminating%20activity%20in%20the%20spleen%20of%20a%20patient%20with%20combined%20immunodeficiency%20disease%20and%20adenosine%20deaminase%20deficiency&amp;author=W.P.%20Schrader&amp;author=B.%20Pollara&amp;author=H.J.%20Meuwissen&amp;volume=75&amp;publication_year=1978&amp;pages=446-450&amp;pmid=24216&amp;doi=10.1073/pnas.75.1.446&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R10">
<span class="label">10.</span><cite>Riazi M.A., Brinkman-Mills P., Nguyen T., Pan H., Phan S., Ying F., Roe B.A., Tochigi J., Shimizu Y., Minoshima S., et al.  (2000). The human homolog of insect-derived growth factor, CECR1, is a candidate gene for features of cat eye syndrome. Genomics
64, 277–285. 10.1006/geno.1999.6099.
</cite> [<a href="https://doi.org/10.1006/geno.1999.6099" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10756095/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genomics&amp;title=The%20human%20homolog%20of%20insect-derived%20growth%20factor,%20CECR1,%20is%20a%20candidate%20gene%20for%20features%20of%20cat%20eye%20syndrome&amp;author=M.A.%20Riazi&amp;author=P.%20Brinkman-Mills&amp;author=T.%20Nguyen&amp;author=H.%20Pan&amp;author=S.%20Phan&amp;volume=64&amp;publication_year=2000&amp;pages=277-285&amp;pmid=10756095&amp;doi=10.1006/geno.1999.6099&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R11">
<span class="label">11.</span><cite>Zhou Q., Yang D., Ombrello A.K., Zavialov A.V., Toro C., Zavialov A.V., Stone D.L., Chae J.J., Rosenzweig S.D., Bishop K., et al.  (2014). Early-onset stroke and vasculopathy associated with mutations in ADA2. The New England journal of medicine
370, 911–920. 10.1056/NEJMoa1307361.
</cite> [<a href="https://doi.org/10.1056/NEJMoa1307361" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4193683/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24552284/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20New%20England%20journal%20of%20medicine&amp;title=Early-onset%20stroke%20and%20vasculopathy%20associated%20with%20mutations%20in%20ADA2&amp;author=Q.%20Zhou&amp;author=D.%20Yang&amp;author=A.K.%20Ombrello&amp;author=A.V.%20Zavialov&amp;author=C.%20Toro&amp;volume=370&amp;publication_year=2014&amp;pages=911-920&amp;pmid=24552284&amp;doi=10.1056/NEJMoa1307361&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R12">
<span class="label">12.</span><cite>Navon Elkan P., Pierce S.B., Segel R., Walsh T., Barash J., Padeh S., Zlotogorski A., Berkun Y., Press J.J., Mukamel M., et al.  (2014). Mutant adenosine deaminase 2 in a polyarteritis nodosa vasculopathy. The New England journal of medicine
370, 921–931. 10.1056/NEJMoa1307362.
</cite> [<a href="https://doi.org/10.1056/NEJMoa1307362" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24552285/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20New%20England%20journal%20of%20medicine&amp;title=Mutant%20adenosine%20deaminase%202%20in%20a%20polyarteritis%20nodosa%20vasculopathy&amp;author=P.%20Navon%20Elkan&amp;author=S.B.%20Pierce&amp;author=R.%20Segel&amp;author=T.%20Walsh&amp;author=J.%20Barash&amp;volume=370&amp;publication_year=2014&amp;pages=921-931&amp;pmid=24552285&amp;doi=10.1056/NEJMoa1307362&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R13">
<span class="label">13.</span><cite>Hashem H., Kelly S.J., Ganson N.J., and Hershfield M.S. (2017). Deficiency of Adenosine Deaminase 2 (DADA2), an Inherited Cause of Polyarteritis Nodosa and a Mimic of Other Systemic Rheumatologic Disorders. Curr Rheumatol Rep
19, 70. 10.1007/s11926-017-0699-8.
</cite> [<a href="https://doi.org/10.1007/s11926-017-0699-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28983775/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Rheumatol%20Rep&amp;title=Deficiency%20of%20Adenosine%20Deaminase%202%20(DADA2),%20an%20Inherited%20Cause%20of%20Polyarteritis%20Nodosa%20and%20a%20Mimic%20of%20Other%20Systemic%20Rheumatologic%20Disorders&amp;author=H.%20Hashem&amp;author=S.J.%20Kelly&amp;author=N.J.%20Ganson&amp;author=M.S.%20Hershfield&amp;volume=19&amp;publication_year=2017&amp;pages=70&amp;pmid=28983775&amp;doi=10.1007/s11926-017-0699-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R14">
<span class="label">14.</span><cite>Meyts I., and Aksentijevich I. (2018). Deficiency of Adenosine Deaminase 2 (DADA2): Updates on the Phenotype, Genetics, Pathogenesis, and Treatment. J Clin Immunol
38, 569–578. 10.1007/s10875-018-0525-8.
</cite> [<a href="https://doi.org/10.1007/s10875-018-0525-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6061100/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29951947/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Immunol&amp;title=Deficiency%20of%20Adenosine%20Deaminase%202%20(DADA2):%20Updates%20on%20the%20Phenotype,%20Genetics,%20Pathogenesis,%20and%20Treatment&amp;author=I.%20Meyts&amp;author=I.%20Aksentijevich&amp;volume=38&amp;publication_year=2018&amp;pages=569-578&amp;pmid=29951947&amp;doi=10.1007/s10875-018-0525-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R15">
<span class="label">15.</span><cite>Moens L., Hershfield M., Arts K., Aksentijevich I., and Meyts I. (2019). Human adenosine deaminase 2 deficiency: A multi-faceted inborn error of immunity. Immunological reviews
287, 62–72. 10.1111/imr.12722.
</cite> [<a href="https://doi.org/10.1111/imr.12722" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30565235/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunological%20reviews&amp;title=Human%20adenosine%20deaminase%202%20deficiency:%20A%20multi-faceted%20inborn%20error%20of%20immunity&amp;author=L.%20Moens&amp;author=M.%20Hershfield&amp;author=K.%20Arts&amp;author=I.%20Aksentijevich&amp;author=I.%20Meyts&amp;volume=287&amp;publication_year=2019&amp;pages=62-72&amp;pmid=30565235&amp;doi=10.1111/imr.12722&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R16">
<span class="label">16.</span><cite>Sahin S., Adrovic A., and Kasapcopur O. (2020). A monogenic autoinflammatory disease with fatal vasculitis: deficiency of adenosine deaminase 2. Curr Opin Rheumatol
32, 3–14. 10.1097/BOR.0000000000000669.
</cite> [<a href="https://doi.org/10.1097/BOR.0000000000000669" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31599797/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Rheumatol&amp;title=A%20monogenic%20autoinflammatory%20disease%20with%20fatal%20vasculitis:%20deficiency%20of%20adenosine%20deaminase%202&amp;author=S.%20Sahin&amp;author=A.%20Adrovic&amp;author=O.%20Kasapcopur&amp;volume=32&amp;publication_year=2020&amp;pages=3-14&amp;pmid=31599797&amp;doi=10.1097/BOR.0000000000000669&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R17">
<span class="label">17.</span><cite>Lee P.Y., Davidson B.A., Abraham R.S., Alter B., Arostegui J.I., Bell K., Belot A., Bergerson J.R.E., Bernard T.J., Brogan P.A., et al.  (2023). Evaluation and Management of Deficiency of Adenosine Deaminase 2: An International Consensus Statement. JAMA Netw Open
6, e2315894. 10.1001/jamanetworkopen.2023.15894.
</cite> [<a href="https://doi.org/10.1001/jamanetworkopen.2023.15894" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37256629/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=JAMA%20Netw%20Open&amp;title=Evaluation%20and%20Management%20of%20Deficiency%20of%20Adenosine%20Deaminase%202:%20An%20International%20Consensus%20Statement&amp;author=P.Y.%20Lee&amp;author=B.A.%20Davidson&amp;author=R.S.%20Abraham&amp;author=B.%20Alter&amp;author=J.I.%20Arostegui&amp;volume=6&amp;publication_year=2023&amp;pages=e2315894&amp;pmid=37256629&amp;doi=10.1001/jamanetworkopen.2023.15894&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R18">
<span class="label">18.</span><cite>Wouters M., Ehlers L., Dzhus M., Kienapfel V., Bucciol G., Delafontaine S., Hombrouck A., Pillay B., Moens L., and Meyts I. (2024). Human ADA2 Deficiency: Ten Years Later. Curr Allergy Asthma Rep
24, 477–484. 10.1007/s11882-024-01163-9.
</cite> [<a href="https://doi.org/10.1007/s11882-024-01163-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11364588/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38970744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Allergy%20Asthma%20Rep&amp;title=Human%20ADA2%20Deficiency:%20Ten%20Years%20Later&amp;author=M.%20Wouters&amp;author=L.%20Ehlers&amp;author=M.%20Dzhus&amp;author=V.%20Kienapfel&amp;author=G.%20Bucciol&amp;volume=24&amp;publication_year=2024&amp;pages=477-484&amp;pmid=38970744&amp;doi=10.1007/s11882-024-01163-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R19">
<span class="label">19.</span><cite>Jee H., Huang Z., Baxter S., Huang Y., Taylor M.L., Henderson L.A., Rosenzweig S., Sharma A., Chambers E.P., Hershfield M.S., et al.  (2022). Comprehensive analysis of ADA2 genetic variants and estimation of carrier frequency driven by a function-based approach. J Allergy Clin Immunol
149, 379–387. 10.1016/j.jaci.2021.04.034.
</cite> [<a href="https://doi.org/10.1016/j.jaci.2021.04.034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8591146/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34004258/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&amp;title=Comprehensive%20analysis%20of%20ADA2%20genetic%20variants%20and%20estimation%20of%20carrier%20frequency%20driven%20by%20a%20function-based%20approach&amp;author=H.%20Jee&amp;author=Z.%20Huang&amp;author=S.%20Baxter&amp;author=Y.%20Huang&amp;author=M.L.%20Taylor&amp;volume=149&amp;publication_year=2022&amp;pages=379-387&amp;pmid=34004258&amp;doi=10.1016/j.jaci.2021.04.034&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R20">
<span class="label">20.</span><cite>Pinto B., Deo P., Sharma S., Syal A., and Sharma A. (2021). Expanding spectrum of DADA2: a review of phenotypes, genetics, pathogenesis and treatment. Clin Rheumatol
40, 3883–3896. 10.1007/s10067-021-05711-w.
</cite> [<a href="https://doi.org/10.1007/s10067-021-05711-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33791889/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Rheumatol&amp;title=Expanding%20spectrum%20of%20DADA2:%20a%20review%20of%20phenotypes,%20genetics,%20pathogenesis%20and%20treatment&amp;author=B.%20Pinto&amp;author=P.%20Deo&amp;author=S.%20Sharma&amp;author=A.%20Syal&amp;author=A.%20Sharma&amp;volume=40&amp;publication_year=2021&amp;pages=3883-3896&amp;pmid=33791889&amp;doi=10.1007/s10067-021-05711-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R21">
<span class="label">21.</span><cite>Cohen A., Hirschhorn R., Horowitz S.D., Rubinstein A., Polmar S.H., Hong R., and Martin D.W. Jr. (1978). Deoxyadenosine triphosphate as a potentially toxic metabolite in adenosine deaminase deficiency. Proc Natl Acad Sci U S A
75, 472–476. 10.1073/pnas.75.1.472.
</cite> [<a href="https://doi.org/10.1073/pnas.75.1.472" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC411272/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/272665/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Deoxyadenosine%20triphosphate%20as%20a%20potentially%20toxic%20metabolite%20in%20adenosine%20deaminase%20deficiency&amp;author=A.%20Cohen&amp;author=R.%20Hirschhorn&amp;author=S.D.%20Horowitz&amp;author=A.%20Rubinstein&amp;author=S.H.%20Polmar&amp;volume=75&amp;publication_year=1978&amp;pages=472-476&amp;pmid=272665&amp;doi=10.1073/pnas.75.1.472&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R22">
<span class="label">22.</span><cite>Hershfield M.S., Kredich N.M., Ownby D.R., Ownby H., and Buckley R. (1979). In vivo inactivation of erythrocyte S-adenosylhomocysteine hydrolase by 2'-deoxyadenosine in adenosine deaminase-deficient patients. J Clin Invest
63, 807–811. 10.1172/JCI109367.
</cite> [<a href="https://doi.org/10.1172/JCI109367" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC372019/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/312296/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&amp;title=In%20vivo%20inactivation%20of%20erythrocyte%20S-adenosylhomocysteine%20hydrolase%20by%202'-deoxyadenosine%20in%20adenosine%20deaminase-deficient%20patients&amp;author=M.S.%20Hershfield&amp;author=N.M.%20Kredich&amp;author=D.R.%20Ownby&amp;author=H.%20Ownby&amp;author=R.%20Buckley&amp;volume=63&amp;publication_year=1979&amp;pages=807-811&amp;pmid=312296&amp;doi=10.1172/JCI109367&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R23">
<span class="label">23.</span><cite>Garcia-Gil M., Camici M., Allegrini S., Pesi R., Petrotto E., and Tozzi M.G. (2018). Emerging Role of Purine Metabolizing Enzymes in Brain Function and Tumors. Int J Mol Sci
19. 10.3390/ijms19113598.</cite> [<a href="https://doi.org/10.3390/ijms19113598" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6274932/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30441833/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&amp;title=Emerging%20Role%20of%20Purine%20Metabolizing%20Enzymes%20in%20Brain%20Function%20and%20Tumors&amp;author=M.%20Garcia-Gil&amp;author=M.%20Camici&amp;author=S.%20Allegrini&amp;author=R.%20Pesi&amp;author=E.%20Petrotto&amp;volume=19&amp;publication_year=2018&amp;pmid=30441833&amp;doi=10.3390/ijms19113598&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R24">
<span class="label">24.</span><cite>Zanoni M., Pegoraro A., Adinolfi E., and De Marchi E. (2022). Emerging roles of purinergic signaling in anti-cancer therapy resistance. Front Cell Dev Biol
10, 1006384. 10.3389/fcell.2022.1006384.
</cite> [<a href="https://doi.org/10.3389/fcell.2022.1006384" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9527280/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36200041/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Cell%20Dev%20Biol&amp;title=Emerging%20roles%20of%20purinergic%20signaling%20in%20anti-cancer%20therapy%20resistance&amp;author=M.%20Zanoni&amp;author=A.%20Pegoraro&amp;author=E.%20Adinolfi&amp;author=E.%20De%20Marchi&amp;volume=10&amp;publication_year=2022&amp;pages=1006384&amp;pmid=36200041&amp;doi=10.3389/fcell.2022.1006384&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R25">
<span class="label">25.</span><cite>Nunez-Rios J.D., Ulrich H., Diaz-Munoz M., Lameu C., and Vazquez-Cuevas F.G. (2023). Purinergic system in cancer stem cells. Purinergic Signal. 10.1007/s11302-023-09976-5.</cite> [<a href="https://doi.org/10.1007/s11302-023-09976-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11904000/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37966629/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Purinergic%20Signal&amp;title=Purinergic%20system%20in%20cancer%20stem%20cells&amp;author=J.D.%20Nunez-Rios&amp;author=H.%20Ulrich&amp;author=M.%20Diaz-Munoz&amp;author=C.%20Lameu&amp;author=F.G.%20Vazquez-Cuevas&amp;publication_year=2023&amp;pmid=37966629&amp;doi=10.1007/s11302-023-09976-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R26">
<span class="label">26.</span><cite>Wilson D.K., Rudolph F.B., and Quiocho F.A. (1991). Atomic structure of adenosine deaminase complexed with a transition-state analog: understanding catalysis and immunodeficiency mutations. Science
252, 1278–1284. 10.1126/science.1925539.
</cite> [<a href="https://doi.org/10.1126/science.1925539" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1925539/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Atomic%20structure%20of%20adenosine%20deaminase%20complexed%20with%20a%20transition-state%20analog:%20understanding%20catalysis%20and%20immunodeficiency%20mutations&amp;author=D.K.%20Wilson&amp;author=F.B.%20Rudolph&amp;author=F.A.%20Quiocho&amp;volume=252&amp;publication_year=1991&amp;pages=1278-1284&amp;pmid=1925539&amp;doi=10.1126/science.1925539&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R27">
<span class="label">27.</span><cite>Ma M.T., Jennings M.R., Blazeck J., and Lieberman R.L. (2022). Catalytically active holo Homo sapiens adenosine deaminase I adopts a closed conformation. Acta Crystallogr D Struct Biol
78, 91–103. 10.1107/S2059798321011785.
</cite> [<a href="https://doi.org/10.1107/S2059798321011785" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8725166/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34981765/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr%20D%20Struct%20Biol&amp;title=Catalytically%20active%20holo%20Homo%20sapiens%20adenosine%20deaminase%20I%20adopts%20a%20closed%20conformation&amp;author=M.T.%20Ma&amp;author=M.R.%20Jennings&amp;author=J.%20Blazeck&amp;author=R.L.%20Lieberman&amp;volume=78&amp;publication_year=2022&amp;pages=91-103&amp;pmid=34981765&amp;doi=10.1107/S2059798321011785&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R28">
<span class="label">28.</span><cite>Zavialov A.V., Yu X., Spillmann D., Lauvau G., and Zavialov A.V. (2010). Structural basis for the growth factor activity of human adenosine deaminase ADA2. J Biol Chem
285, 12367–12377. 10.1074/jbc.M109.083527.
</cite> [<a href="https://doi.org/10.1074/jbc.M109.083527" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2852975/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20147294/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Structural%20basis%20for%20the%20growth%20factor%20activity%20of%20human%20adenosine%20deaminase%20ADA2&amp;author=A.V.%20Zavialov&amp;author=X.%20Yu&amp;author=D.%20Spillmann&amp;author=G.%20Lauvau&amp;author=A.V.%20Zavialov&amp;volume=285&amp;publication_year=2010&amp;pages=12367-12377&amp;pmid=20147294&amp;doi=10.1074/jbc.M109.083527&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R29">
<span class="label">29.</span><cite>Sleat D.E., Wang Y., Sohar I., Lackland H., Li Y., Li H., Zheng H., and Lobel P. (2006). Identification and validation of mannose 6-phosphate glycoproteins in human plasma reveal a wide range of lysosomal and non-lysosomal proteins. Molecular &amp; cellular proteomics : MCP
5, 1942–1956. 10.1074/mcp.M600030-MCP200.
</cite> [<a href="https://doi.org/10.1074/mcp.M600030-MCP200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16709564/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20&amp;%20cellular%20proteomics%20:%20MCP&amp;title=Identification%20and%20validation%20of%20mannose%206-phosphate%20glycoproteins%20in%20human%20plasma%20reveal%20a%20wide%20range%20of%20lysosomal%20and%20non-lysosomal%20proteins&amp;author=D.E.%20Sleat&amp;author=Y.%20Wang&amp;author=I.%20Sohar&amp;author=H.%20Lackland&amp;author=Y.%20Li&amp;volume=5&amp;publication_year=2006&amp;pages=1942-1956&amp;pmid=16709564&amp;doi=10.1074/mcp.M600030-MCP200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R30">
<span class="label">30.</span><cite>Greiner-Tollersrud O.K., Krausz M., Boehler V., Polyzou A., Seidl M., Spahiu A., Abdullah Z., Andryka-Cegielski K., Dominick F.I., Huebscher K., et al.  (2024). ADA2 is a lysosomal deoxyadenosine deaminase acting on DNA involved in regulating TLR9-mediated immune sensing of DNA. Cell reports
43, 114899. 10.1016/j.celrep.2024.114899.
</cite> [<a href="https://doi.org/10.1016/j.celrep.2024.114899" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39441717/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20reports&amp;title=ADA2%20is%20a%20lysosomal%20deoxyadenosine%20deaminase%20acting%20on%20DNA%20involved%20in%20regulating%20TLR9-mediated%20immune%20sensing%20of%20DNA&amp;author=O.K.%20Greiner-Tollersrud&amp;author=M.%20Krausz&amp;author=V.%20Boehler&amp;author=A.%20Polyzou&amp;author=M.%20Seidl&amp;volume=43&amp;publication_year=2024&amp;pages=114899&amp;pmid=39441717&amp;doi=10.1016/j.celrep.2024.114899&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R31">
<span class="label">31.</span><cite>Johnston J.B. (2011). Mechanism of action of pentostatin and cladribine in hairy cell leukemia. Leuk Lymphoma
52
Suppl 2, 43–45. 10.3109/10428194.2011.570394.
</cite> [<a href="https://doi.org/10.3109/10428194.2011.570394" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21463108/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Leuk%20Lymphoma&amp;title=Mechanism%20of%20action%20of%20pentostatin%20and%20cladribine%20in%20hairy%20cell%20leukemia&amp;author=J.B.%20Johnston&amp;volume=52&amp;issue=Suppl%202&amp;publication_year=2011&amp;pages=43-45&amp;pmid=21463108&amp;doi=10.3109/10428194.2011.570394&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R32">
<span class="label">32.</span><cite>Zheng H., Chordia M.D., Cooper D.R., Chruszcz M., Muller P., Sheldrick G.M., and Minor W. (2014). Validation of metal-binding sites in macromolecular structures with the CheckMyMetal web server. Nat Protoc
9, 156–170. 10.1038/nprot.2013.172.
</cite> [<a href="https://doi.org/10.1038/nprot.2013.172" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4410975/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24356774/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Protoc&amp;title=Validation%20of%20metal-binding%20sites%20in%20macromolecular%20structures%20with%20the%20CheckMyMetal%20web%20server&amp;author=H.%20Zheng&amp;author=M.D.%20Chordia&amp;author=D.R.%20Cooper&amp;author=M.%20Chruszcz&amp;author=P.%20Muller&amp;volume=9&amp;publication_year=2014&amp;pages=156-170&amp;pmid=24356774&amp;doi=10.1038/nprot.2013.172&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R33">
<span class="label">33.</span><cite>Gross M. (1994). Molecular biology of AMP deaminase deficiency. Pharm World Sci
16, 55–61. 10.1007/BF01880656.
</cite> [<a href="https://doi.org/10.1007/BF01880656" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8032342/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharm%20World%20Sci&amp;title=Molecular%20biology%20of%20AMP%20deaminase%20deficiency&amp;author=M.%20Gross&amp;volume=16&amp;publication_year=1994&amp;pages=55-61&amp;pmid=8032342&amp;doi=10.1007/BF01880656&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R34">
<span class="label">34.</span><cite>Kitao Y., Saito T., Watanabe S., Ohe Y., Takahashi K., Akaki T., Adachi T., Doi S., Yamanaka K., Murai Y., et al.  (2023). The discovery of 3,3-dimethyl-1,2,3,4-tetrahydroquinoxaline-1-carboxamides as AMPD2 inhibitors with a novel mechanism of action. Bioorganic &amp; medicinal chemistry letters
80, 129110. 10.1016/j.bmcl.2022.129110.
</cite> [<a href="https://doi.org/10.1016/j.bmcl.2022.129110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36563792/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioorganic%20&amp;%20medicinal%20chemistry%20letters&amp;title=The%20discovery%20of%203,3-dimethyl-1,2,3,4-tetrahydroquinoxaline-1-carboxamides%20as%20AMPD2%20inhibitors%20with%20a%20novel%20mechanism%20of%20action&amp;author=Y.%20Kitao&amp;author=T.%20Saito&amp;author=S.%20Watanabe&amp;author=Y.%20Ohe&amp;author=K.%20Takahashi&amp;volume=80&amp;publication_year=2023&amp;pages=129110&amp;pmid=36563792&amp;doi=10.1016/j.bmcl.2022.129110&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R35">
<span class="label">35.</span><cite>Chazot A., Zimberger C., Feracci M., Moussa A., Good S., Sommadossi J.P., Alvarez K., Ferron F., and Canard B. (2024). The activation cascade of the broad-spectrum antiviral bemnifosbuvir characterized at atomic resolution. PLoS Biol
22, e3002743. 10.1371/journal.pbio.3002743.
</cite> [<a href="https://doi.org/10.1371/journal.pbio.3002743" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11349198/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39190717/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol&amp;title=The%20activation%20cascade%20of%20the%20broad-spectrum%20antiviral%20bemnifosbuvir%20characterized%20at%20atomic%20resolution&amp;author=A.%20Chazot&amp;author=C.%20Zimberger&amp;author=M.%20Feracci&amp;author=A.%20Moussa&amp;author=S.%20Good&amp;volume=22&amp;publication_year=2024&amp;pages=e3002743&amp;pmid=39190717&amp;doi=10.1371/journal.pbio.3002743&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R36">
<span class="label">36.</span><cite>Jia Q., and Xie W. (2019). Alternative conformation induced by substrate binding for Arabidopsis thalianaN6-methyl-AMP deaminase. Nucleic Acids Res
47, 3233–3243. 10.1093/nar/gkz070.
</cite> [<a href="https://doi.org/10.1093/nar/gkz070" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6451127/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30721978/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Alternative%20conformation%20induced%20by%20substrate%20binding%20for%20Arabidopsis%20thalianaN6-methyl-AMP%20deaminase&amp;author=Q.%20Jia&amp;author=W.%20Xie&amp;volume=47&amp;publication_year=2019&amp;pages=3233-3243&amp;pmid=30721978&amp;doi=10.1093/nar/gkz070&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R37">
<span class="label">37.</span><cite>Wu B., Zhang D., Nie H., Shen S., Li Y., and Li S. (2019). Structure of Arabidopsis thaliana N(6)-methyl-AMP deaminase ADAL with bound GMP and IMP and implications for N(6)-methyl-AMP recognition and processing. RNA Biol
16, 1504–1512. 10.1080/15476286.2019.1642712.
</cite> [<a href="https://doi.org/10.1080/15476286.2019.1642712" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6779375/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31318636/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=RNA%20Biol&amp;title=Structure%20of%20Arabidopsis%20thaliana%20N(6)-methyl-AMP%20deaminase%20ADAL%20with%20bound%20GMP%20and%20IMP%20and%20implications%20for%20N(6)-methyl-AMP%20recognition%20and%20processing&amp;author=B.%20Wu&amp;author=D.%20Zhang&amp;author=H.%20Nie&amp;author=S.%20Shen&amp;author=Y.%20Li&amp;volume=16&amp;publication_year=2019&amp;pages=1504-1512&amp;pmid=31318636&amp;doi=10.1080/15476286.2019.1642712&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R38">
<span class="label">38.</span><cite>Murakami E., Bao H., Mosley R.T., Du J., Sofia M.J., and Furman P.A. (2011). Adenosine deaminase-like protein 1 (ADAL1): characterization and substrate specificity in the hydrolysis of N(6)- or O(6)-substituted purine or 2-aminopurine nucleoside monophosphates. J Med Chem
54, 5902–5914. 10.1021/jm200650j.
</cite> [<a href="https://doi.org/10.1021/jm200650j" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21755941/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Med%20Chem&amp;title=Adenosine%20deaminase-like%20protein%201%20(ADAL1):%20characterization%20and%20substrate%20specificity%20in%20the%20hydrolysis%20of%20N(6)-%20or%20O(6)-substituted%20purine%20or%202-aminopurine%20nucleoside%20monophosphates&amp;author=E.%20Murakami&amp;author=H.%20Bao&amp;author=R.T.%20Mosley&amp;author=J.%20Du&amp;author=M.J.%20Sofia&amp;volume=54&amp;publication_year=2011&amp;pages=5902-5914&amp;pmid=21755941&amp;doi=10.1021/jm200650j&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R39">
<span class="label">39.</span><cite>Chen M., Urs M.J., Sanchez-Gonzalez I., Olayioye M.A., Herde M., and Witte C.P. (2018). m(6)A RNA Degradation Products Are Catabolized by an Evolutionarily Conserved N(6)-Methyl-AMP Deaminase in Plant and Mammalian Cells. Plant Cell
30, 1511–1522. 10.1105/tpc.18.00236.
</cite> [<a href="https://doi.org/10.1105/tpc.18.00236" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6096584/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29884623/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=m(6)A%20RNA%20Degradation%20Products%20Are%20Catabolized%20by%20an%20Evolutionarily%20Conserved%20N(6)-Methyl-AMP%20Deaminase%20in%20Plant%20and%20Mammalian%20Cells&amp;author=M.%20Chen&amp;author=M.J.%20Urs&amp;author=I.%20Sanchez-Gonzalez&amp;author=M.A.%20Olayioye&amp;author=M.%20Herde&amp;volume=30&amp;publication_year=2018&amp;pages=1511-1522&amp;pmid=29884623&amp;doi=10.1105/tpc.18.00236&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R40">
<span class="label">40.</span><cite>Aranda C.S., Gouveia-Pereira M.P., da Silva C.J.M., Rizzo M., Ishizuka E., de Oliveira E.B., and Condino-Neto A. (2024). Severe combined immunodeficiency diagnosis and genetic defects. Immunological reviews
322, 138–147. 10.1111/imr.13310.
</cite> [<a href="https://doi.org/10.1111/imr.13310" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38287514/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunological%20reviews&amp;title=Severe%20combined%20immunodeficiency%20diagnosis%20and%20genetic%20defects&amp;author=C.S.%20Aranda&amp;author=M.P.%20Gouveia-Pereira&amp;author=C.J.M.%20da%20Silva&amp;author=M.%20Rizzo&amp;author=E.%20Ishizuka&amp;volume=322&amp;publication_year=2024&amp;pages=138-147&amp;pmid=38287514&amp;doi=10.1111/imr.13310&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R41">
<span class="label">41.</span><cite>Secord E., and Hartog N.L. (2022). Review of Treatment for Adenosine Deaminase Deficiency (ADA) Severe Combined Immunodeficiency (SCID). Ther Clin Risk Manag
18, 939–944. 10.2147/TCRM.S350762.
</cite> [<a href="https://doi.org/10.2147/TCRM.S350762" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9512634/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36172599/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ther%20Clin%20Risk%20Manag&amp;title=Review%20of%20Treatment%20for%20Adenosine%20Deaminase%20Deficiency%20(ADA)%20Severe%20Combined%20Immunodeficiency%20(SCID)&amp;author=E.%20Secord&amp;author=N.L.%20Hartog&amp;volume=18&amp;publication_year=2022&amp;pages=939-944&amp;pmid=36172599&amp;doi=10.2147/TCRM.S350762&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R42">
<span class="label">42.</span><cite>Chen K., Zhao B.S., and He C. (2016). Nucleic Acid Modifications in Regulation of Gene Expression. Cell Chem Biol
23, 74–85. 10.1016/j.chembiol.2015.11.007.
</cite> [<a href="https://doi.org/10.1016/j.chembiol.2015.11.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4779186/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26933737/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Chem%20Biol&amp;title=Nucleic%20Acid%20Modifications%20in%20Regulation%20of%20Gene%20Expression&amp;author=K.%20Chen&amp;author=B.S.%20Zhao&amp;author=C.%20He&amp;volume=23&amp;publication_year=2016&amp;pages=74-85&amp;pmid=26933737&amp;doi=10.1016/j.chembiol.2015.11.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R43">
<span class="label">43.</span><cite>Meyer K.D., and Jaffrey S.R. (2017). Rethinking m(6)A Readers, Writers, and Erasers. Annual review of cell and developmental biology
33, 319–342. 10.1146/annurev-cellbio-100616-060758.</cite> [<a href="https://doi.org/10.1146/annurev-cellbio-100616-060758" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5963928/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28759256/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20review%20of%20cell%20and%20developmental%20biology&amp;title=Rethinking%20m(6)A%20Readers,%20Writers,%20and%20Erasers&amp;author=K.D.%20Meyer&amp;author=S.R.%20Jaffrey&amp;volume=33&amp;publication_year=2017&amp;pages=319-342&amp;pmid=28759256&amp;doi=10.1146/annurev-cellbio-100616-060758&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R44">
<span class="label">44.</span><cite>Lockhart J. (2018). The Story Continues: Following the Fate of m(6)A Marks in the Eukaryotic Transcriptome. Plant Cell
30, 1385–1386. 10.1105/tpc.18.00456.
</cite> [<a href="https://doi.org/10.1105/tpc.18.00456" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6096586/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29903727/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=The%20Story%20Continues:%20Following%20the%20Fate%20of%20m(6)A%20Marks%20in%20the%20Eukaryotic%20Transcriptome&amp;author=J%20Lockhart&amp;volume=30&amp;publication_year=2018&amp;pages=1385-1386&amp;pmid=29903727&amp;doi=10.1105/tpc.18.00456&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R45">
<span class="label">45.</span><cite>Stankiewicz A.J. (1983). Direct deamination of AMP, ADP, ATP and NADH by non-specific adenylate deaminase in the foot muscle of the snail Helix pomatia. Biochem J
215, 39–44. 10.1042/bj2150039.
</cite> [<a href="https://doi.org/10.1042/bj2150039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1152361/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6626180/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem%20J&amp;title=Direct%20deamination%20of%20AMP,%20ADP,%20ATP%20and%20NADH%20by%20non-specific%20adenylate%20deaminase%20in%20the%20foot%20muscle%20of%20the%20snail%20Helix%20pomatia&amp;author=A.J.%20Stankiewicz&amp;volume=215&amp;publication_year=1983&amp;pages=39-44&amp;pmid=6626180&amp;doi=10.1042/bj2150039&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R46">
<span class="label">46.</span><cite>Stankiewicz A.J. (1984). Non-specific adenylate deaminase from snail foot muscle. Advances in experimental medicine and biology
165
Pt B, 457–464. 10.1007/978-1-4757-0390-0_87.</cite> [<a href="https://doi.org/10.1007/978-1-4757-0390-0_87" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6720445/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Advances%20in%20experimental%20medicine%20and%20biology&amp;title=Non-specific%20adenylate%20deaminase%20from%20snail%20foot%20muscle&amp;author=A.J.%20Stankiewicz&amp;volume=165&amp;issue=Pt%20B&amp;publication_year=1984&amp;pages=457-464&amp;pmid=6720445&amp;doi=10.1007/978-1-4757-0390-0_87&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R47">
<span class="label">47.</span><cite>Zeng Z., Hu Z., Zhao R., Rao J., Mestre M.R., Liu Y., Liu S., Feng H., Chen Y., He H., et al.  (2025). Base-modified nucleotides mediate immune signaling in bacteria. Science, eads6055. 10.1126/science.ads6055.
</cite> [<a href="https://doi.org/10.1126/science.ads6055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39977546/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Base-modified%20nucleotides%20mediate%20immune%20signaling%20in%20bacteria&amp;author=Z.%20Zeng&amp;author=Z.%20Hu&amp;author=R.%20Zhao&amp;author=J.%20Rao&amp;author=M.R.%20Mestre&amp;publication_year=2025&amp;pages=eads6055&amp;pmid=39977546&amp;doi=10.1126/science.ads6055&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R48">
<span class="label">48.</span><cite>Jumper J., Evans R., Pritzel A., Green T., Figurnov M., Ronneberger O., Tunyasuvunakool K., Bates R., Zidek A., Potapenko A., et al.  (2021). Highly accurate protein structure prediction with AlphaFold. Nature
596, 583–589. 10.1038/s41586-021-03819-2.
</cite> [<a href="https://doi.org/10.1038/s41586-021-03819-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8371605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34265844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Highly%20accurate%20protein%20structure%20prediction%20with%20AlphaFold&amp;author=J.%20Jumper&amp;author=R.%20Evans&amp;author=A.%20Pritzel&amp;author=T.%20Green&amp;author=M.%20Figurnov&amp;volume=596&amp;publication_year=2021&amp;pages=583-589&amp;pmid=34265844&amp;doi=10.1038/s41586-021-03819-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R49">
<span class="label">49.</span><cite>Marinus M.G., and Casadesus J. (2009). Roles of DNA adenine methylation in host-pathogen interactions: mismatch repair, transcriptional regulation, and more. FEMS microbiology reviews
33, 488–503. 10.1111/j.1574-6976.2008.00159.x.
</cite> [<a href="https://doi.org/10.1111/j.1574-6976.2008.00159.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2941194/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19175412/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEMS%20microbiology%20reviews&amp;title=Roles%20of%20DNA%20adenine%20methylation%20in%20host-pathogen%20interactions:%20mismatch%20repair,%20transcriptional%20regulation,%20and%20more&amp;author=M.G.%20Marinus&amp;author=J.%20Casadesus&amp;volume=33&amp;publication_year=2009&amp;pages=488-503&amp;pmid=19175412&amp;doi=10.1111/j.1574-6976.2008.00159.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R50">
<span class="label">50.</span><cite>Casadesus J. (2016). Bacterial DNA Methylation and Methylomes. Advances in experimental medicine and biology
945, 35–61. 10.1007/978-3-319-43624-1_3.
</cite> [<a href="https://doi.org/10.1007/978-3-319-43624-1_3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27826834/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Advances%20in%20experimental%20medicine%20and%20biology&amp;title=Bacterial%20DNA%20Methylation%20and%20Methylomes&amp;author=J.%20Casadesus&amp;volume=945&amp;publication_year=2016&amp;pages=35-61&amp;pmid=27826834&amp;doi=10.1007/978-3-319-43624-1_3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R51">
<span class="label">51.</span><cite>Anton B.P., and Roberts R.J. (2021). Beyond Restriction Modification: Epigenomic Roles of DNA Methylation in Prokaryotes. Annu Rev Microbiol
75, 129–149. 10.1146/annurev-micro-040521-035040.
</cite> [<a href="https://doi.org/10.1146/annurev-micro-040521-035040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34314594/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Microbiol&amp;title=Beyond%20Restriction%20Modification:%20Epigenomic%20Roles%20of%20DNA%20Methylation%20in%20Prokaryotes&amp;author=B.P.%20Anton&amp;author=R.J.%20Roberts&amp;volume=75&amp;publication_year=2021&amp;pages=129-149&amp;pmid=34314594&amp;doi=10.1146/annurev-micro-040521-035040&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R52">
<span class="label">52.</span><cite>Passeri I., Vaccaro F., Mengoni A., and Fagorzi C. (2024). Moving toward the Inclusion of Epigenomics in Bacterial Genome Evolution: Perspectives and Challenges. Int J Mol Sci
25. 10.3390/ijms25084425.</cite> [<a href="https://doi.org/10.3390/ijms25084425" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11050019/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38674013/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&amp;title=Moving%20toward%20the%20Inclusion%20of%20Epigenomics%20in%20Bacterial%20Genome%20Evolution:%20Perspectives%20and%20Challenges&amp;author=I.%20Passeri&amp;author=F.%20Vaccaro&amp;author=A.%20Mengoni&amp;author=C.%20Fagorzi&amp;volume=25&amp;publication_year=2024&amp;pmid=38674013&amp;doi=10.3390/ijms25084425&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R53">
<span class="label">53.</span><cite>Tolner B., Smith L., Begent R.H., and Chester K.A. (2006). Production of recombinant protein in Pichia pastoris by fermentation. Nat Protoc
1, 1006–1021. 10.1038/nprot.2006.126.
</cite> [<a href="https://doi.org/10.1038/nprot.2006.126" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17406338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Protoc&amp;title=Production%20of%20recombinant%20protein%20in%20Pichia%20pastoris%20by%20fermentation&amp;author=B.%20Tolner&amp;author=L.%20Smith&amp;author=R.H.%20Begent&amp;author=K.A.%20Chester&amp;volume=1&amp;publication_year=2006&amp;pages=1006-1021&amp;pmid=17406338&amp;doi=10.1038/nprot.2006.126&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R54">
<span class="label">54.</span><cite>Kalckar H.M. (1947). Differential spectrophotometry of purine compounds by means of specific enzymes; studies of the enzymes of purine metabolism. J Biol Chem
167, 461–475.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/20285041/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Differential%20spectrophotometry%20of%20purine%20compounds%20by%20means%20of%20specific%20enzymes;%20studies%20of%20the%20enzymes%20of%20purine%20metabolism&amp;author=H.M.%20Kalckar&amp;volume=167&amp;publication_year=1947&amp;pages=461-475&amp;pmid=20285041&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R55">
<span class="label">55.</span><cite>Otwinowski Z., Borek D., Majewski W., and Minor W. (2003). Multiparametric scaling of diffraction intensities. Acta Crystallogr A
59, 228–234.
</cite> [<a href="https://doi.org/10.1107/s0108767303005488" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12714773/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr%20A&amp;title=Multiparametric%20scaling%20of%20diffraction%20intensities&amp;author=Z.%20Otwinowski&amp;author=D.%20Borek&amp;author=W.%20Majewski&amp;author=W.%20Minor&amp;volume=59&amp;publication_year=2003&amp;pages=228-234&amp;pmid=12714773&amp;doi=10.1107/s0108767303005488&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R56">
<span class="label">56.</span><cite>Beilsten-Edmands J., Winter G., Gildea R., Parkhurst J., Waterman D., and Evans G. (2020). Scaling diffraction data in the DIALS software package: algorithms and new approaches for multi-crystal scaling. Acta Crystallogr D Struct Biol
76, 385–399. 10.1107/S2059798320003198.
</cite> [<a href="https://doi.org/10.1107/S2059798320003198" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7137103/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32254063/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr%20D%20Struct%20Biol&amp;title=Scaling%20diffraction%20data%20in%20the%20DIALS%20software%20package:%20algorithms%20and%20new%20approaches%20for%20multi-crystal%20scaling&amp;author=J.%20Beilsten-Edmands&amp;author=G.%20Winter&amp;author=R.%20Gildea&amp;author=J.%20Parkhurst&amp;author=D.%20Waterman&amp;volume=76&amp;publication_year=2020&amp;pages=385-399&amp;pmid=32254063&amp;doi=10.1107/S2059798320003198&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R57">
<span class="label">57.</span><cite>Winter G., Waterman D.G., Parkhurst J.M., Brewster A.S., Gildea R.J., Gerstel M., Fuentes-Montero L., Vollmar M., Michels-Clark T., Young I.D., et al.  (2018). DIALS: implementation and evaluation of a new integration package. Acta Crystallogr D Struct Biol
74, 85–97. 10.1107/S2059798317017235.
</cite> [<a href="https://doi.org/10.1107/S2059798317017235" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5947772/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29533234/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr%20D%20Struct%20Biol&amp;title=DIALS:%20implementation%20and%20evaluation%20of%20a%20new%20integration%20package&amp;author=G.%20Winter&amp;author=D.G.%20Waterman&amp;author=J.M.%20Parkhurst&amp;author=A.S.%20Brewster&amp;author=R.J.%20Gildea&amp;volume=74&amp;publication_year=2018&amp;pages=85-97&amp;pmid=29533234&amp;doi=10.1107/S2059798317017235&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R58">
<span class="label">58.</span><cite>Evans P.R., and Murshudov G.N. (2013). How good are my data and what is the resolution?
Acta Crystallogr D Biol Crystallogr
69, 1204–1214. 10.1107/S0907444913000061.
</cite> [<a href="https://doi.org/10.1107/S0907444913000061" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3689523/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23793146/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr%20D%20Biol%20Crystallogr&amp;title=How%20good%20are%20my%20data%20and%20what%20is%20the%20resolution?&amp;author=P.R.%20Evans&amp;author=G.N.%20Murshudov&amp;volume=69&amp;publication_year=2013&amp;pages=1204-1214&amp;pmid=23793146&amp;doi=10.1107/S0907444913000061&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R59">
<span class="label">59.</span><cite>Mirdita M., Schutze K., Moriwaki Y., Heo L., Ovchinnikov S., and Steinegger M. (2022). ColabFold: making protein folding accessible to all. Nat Methods
19, 679–682. 10.1038/s41592-022-01488-1.
</cite> [<a href="https://doi.org/10.1038/s41592-022-01488-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9184281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35637307/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=ColabFold:%20making%20protein%20folding%20accessible%20to%20all&amp;author=M.%20Mirdita&amp;author=K.%20Schutze&amp;author=Y.%20Moriwaki&amp;author=L.%20Heo&amp;author=S.%20Ovchinnikov&amp;volume=19&amp;publication_year=2022&amp;pages=679-682&amp;pmid=35637307&amp;doi=10.1038/s41592-022-01488-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R60">
<span class="label">60.</span><cite>Afonine P.V., Grosse-Kunstleve R.W., Echols N., Headd J.J., Moriarty N.W., Mustyakimov M., Terwilliger T.C., Urzhumtsev A., Zwart P.H., and Adams P.D. (2012). Towards automated crystallographic structure refinement with phenix.refine. Acta Crystallogr D Biol Crystallogr
68, 352–367. 10.1107/S0907444912001308.
</cite> [<a href="https://doi.org/10.1107/S0907444912001308" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3322595/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22505256/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr%20D%20Biol%20Crystallogr&amp;title=Towards%20automated%20crystallographic%20structure%20refinement%20with%20phenix.refine&amp;author=P.V.%20Afonine&amp;author=R.W.%20Grosse-Kunstleve&amp;author=N.%20Echols&amp;author=J.J.%20Headd&amp;author=N.W.%20Moriarty&amp;volume=68&amp;publication_year=2012&amp;pages=352-367&amp;pmid=22505256&amp;doi=10.1107/S0907444912001308&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R61">
<span class="label">61.</span><cite>Emsley P., and Cowtan K. (2004). Coot: model-building tools for molecular graphics. Acta Crystallogr D Biol Crystallogr
60, 2126–2132.
</cite> [<a href="https://doi.org/10.1107/S0907444904019158" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15572765/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr%20D%20Biol%20Crystallogr&amp;title=Coot:%20model-building%20tools%20for%20molecular%20graphics&amp;author=P.%20Emsley&amp;author=K.%20Cowtan&amp;volume=60&amp;publication_year=2004&amp;pages=2126-2132&amp;pmid=15572765&amp;doi=10.1107/S0907444904019158&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R62">
<span class="label">62.</span><cite>Emsley P., Lohkamp B., Scott W.G., and Cowtan K. (2010). Features and development of Coot. Acta Crystallogr D Biol Crystallogr
66, 486–501. 10.1107/S0907444910007493.
</cite> [<a href="https://doi.org/10.1107/S0907444910007493" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2852313/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20383002/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr%20D%20Biol%20Crystallogr&amp;title=Features%20and%20development%20of%20Coot&amp;author=P.%20Emsley&amp;author=B.%20Lohkamp&amp;author=W.G.%20Scott&amp;author=K.%20Cowtan&amp;volume=66&amp;publication_year=2010&amp;pages=486-501&amp;pmid=20383002&amp;doi=10.1107/S0907444910007493&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R63">
<span class="label">63.</span><cite>Read R.J., Adams P.D., Arendall W.B. 3rd, Brunger A.T., Emsley P., Joosten R.P., Kleywegt G.J., Krissinel E.B., Lutteke T., Otwinowski Z., et al.  (2011). A new generation of crystallographic validation tools for the protein data bank. Structure
19, 1395–1412. 10.1016/j.str.2011.08.006.
</cite> [<a href="https://doi.org/10.1016/j.str.2011.08.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3195755/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22000512/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=A%20new%20generation%20of%20crystallographic%20validation%20tools%20for%20the%20protein%20data%20bank&amp;author=R.J.%20Read&amp;author=P.D.%20Adams&amp;author=W.B.%20Arendall&amp;author=A.T.%20Brunger&amp;author=P.%20Emsley&amp;volume=19&amp;publication_year=2011&amp;pages=1395-1412&amp;pmid=22000512&amp;doi=10.1016/j.str.2011.08.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R64">
<span class="label">64.</span><cite>Krissinel E., and Henrick K. (2007). Inference of macromolecular assemblies from crystalline state. J Mol Biol
372, 774–797. 10.1016/j.jmb.2007.05.022.
</cite> [<a href="https://doi.org/10.1016/j.jmb.2007.05.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17681537/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&amp;title=Inference%20of%20macromolecular%20assemblies%20from%20crystalline%20state&amp;author=E.%20Krissinel&amp;author=K.%20Henrick&amp;volume=372&amp;publication_year=2007&amp;pages=774-797&amp;pmid=17681537&amp;doi=10.1016/j.jmb.2007.05.022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R65">
<span class="label">65.</span><cite>Teufel F., Almagro Armenteros J.J., Johansen A.R., Gislason M.H., Pihl S.I., Tsirigos K.D., Winther O., Brunak S., von Heijne G., and Nielsen H. (2022). SignalP 6.0 predicts all five types of signal peptides using protein language models. Nat Biotechnol
40, 1023–1025. 10.1038/s41587-021-01156-3.
</cite> [<a href="https://doi.org/10.1038/s41587-021-01156-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9287161/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34980915/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=SignalP%206.0%20predicts%20all%20five%20types%20of%20signal%20peptides%20using%20protein%20language%20models&amp;author=F.%20Teufel&amp;author=J.J.%20Almagro%20Armenteros&amp;author=A.R.%20Johansen&amp;author=M.H.%20Gislason&amp;author=S.I.%20Pihl&amp;volume=40&amp;publication_year=2022&amp;pages=1023-1025&amp;pmid=34980915&amp;doi=10.1038/s41587-021-01156-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><span>Supplement 1</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11974800/bin/media-1.pdf" data-ga-action="click_feat_suppl" class="usa-link">media-1.pdf</a><sup> (3.3MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The protein sequence of HPAMPD has been deposited to GenBank with accession number <a href="https://ncbi.nlm.nih.gov/nucleotide/XOL57624.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XOL57624.1</a>. The X-ray structures (coordinates and structure factor files) reported here have been deposited to PDB and are publicly available as of the date of publications. PDB accession numbers are 9NTE and 9NTF (HPAMPD in the apo form), 9NTG (HPAMPD in complex with pentostatin), 9NTH (HPAMPD in complex with PMP), 9NTI (Chimera-2 in the apo-form), 9NTJ (Chimera-2 in complexes with pentostatin), and 9NTK (Chimera-2 in complex with PMP).</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from bioRxiv are provided here courtesy of <strong>Cold Spring Harbor Laboratory Preprints</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1101/2025.03.26.645602"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/nihpp-2025.03.26.645602v1.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/11974800/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/11974800/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11974800%2F%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11974800/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC11974800/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC11974800/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40196538/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC11974800/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40196538/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC11974800/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/11974800/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="vWHv1r8tSkYyIqj9K7qwmxVrhkcrQNcnhLbn5uCRZsWYwQTEnv6qxNCQNoeGGQpl">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-B7-tgZZA.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-BLbQGH8u.js"></script>
    
    

    </body>
</html>
