
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/base_style-CPPDhfna.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-BepKMsUl.js"></script>

  <link rel="stylesheet" href="/static/assets/article_style-DUfI1jZp.css" />
<link rel="stylesheet" href="/static/assets/style-BOrsu84o.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-D3wJHomh.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3EAuthor Manuscript%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Error-corrected flow-based sequencing at whole genome scale and its application to circulating cell-free DNA profiling - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="18DE2476930AF47300247600479E545F.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="nihpa">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="manuscript">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12077166/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature methods">
<meta name="citation_title" content="Error-corrected flow-based sequencing at whole genome scale and its application to circulating cell-free DNA profiling">
<meta name="citation_author" content="Alexandre Pellan Cheng">
<meta name="citation_author_institution" content="New York Genome Center, New York, NY, USA">
<meta name="citation_author_institution" content="Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author" content="Adam J Widman">
<meta name="citation_author_institution" content="New York Genome Center, New York, NY, USA">
<meta name="citation_author_institution" content="Memorial Sloan Kettering Cancer Center, New York, NY, USA">
<meta name="citation_author" content="Anushri Arora">
<meta name="citation_author_institution" content="New York Genome Center, New York, NY, USA">
<meta name="citation_author_institution" content="Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author" content="Itai Rusinek">
<meta name="citation_author_institution" content="Ultima Genomics, Fremont, CA, USA">
<meta name="citation_author" content="Aaron Sossin">
<meta name="citation_author_institution" content="New York Genome Center, New York, NY, USA">
<meta name="citation_author_institution" content="Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author" content="Srinivas Rajagopalan">
<meta name="citation_author_institution" content="New York Genome Center, New York, NY, USA">
<meta name="citation_author_institution" content="Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author" content="Nicholas Midler">
<meta name="citation_author_institution" content="New York Genome Center, New York, NY, USA">
<meta name="citation_author_institution" content="Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author" content="William F Hooper">
<meta name="citation_author_institution" content="New York Genome Center, New York, NY, USA">
<meta name="citation_author" content="Rebecca M Murray">
<meta name="citation_author_institution" content="New York Genome Center, New York, NY, USA">
<meta name="citation_author_institution" content="Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author" content="Daniel Halmos">
<meta name="citation_author_institution" content="New York Genome Center, New York, NY, USA">
<meta name="citation_author_institution" content="Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author" content="Theophile Langanay">
<meta name="citation_author_institution" content="New York Genome Center, New York, NY, USA">
<meta name="citation_author_institution" content="Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author" content="Hoyin Chu">
<meta name="citation_author_institution" content="Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author" content="Giorgio Inghirami">
<meta name="citation_author_institution" content="Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author" content="Catherine Potenski">
<meta name="citation_author_institution" content="New York Genome Center, New York, NY, USA">
<meta name="citation_author_institution" content="Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author" content="Soren Germer">
<meta name="citation_author_institution" content="New York Genome Center, New York, NY, USA">
<meta name="citation_author" content="Melissa Marton">
<meta name="citation_author_institution" content="New York Genome Center, New York, NY, USA">
<meta name="citation_author" content="Dina Manaa">
<meta name="citation_author_institution" content="New York Genome Center, New York, NY, USA">
<meta name="citation_author" content="Adrienne Helland">
<meta name="citation_author_institution" content="New York Genome Center, New York, NY, USA">
<meta name="citation_author" content="Rob Furatero">
<meta name="citation_author_institution" content="New York Genome Center, New York, NY, USA">
<meta name="citation_author" content="Jaime McClintock">
<meta name="citation_author_institution" content="New York Genome Center, New York, NY, USA">
<meta name="citation_author" content="Lara Winterkorn">
<meta name="citation_author_institution" content="New York Genome Center, New York, NY, USA">
<meta name="citation_author" content="Zoe Steinsnyder">
<meta name="citation_author_institution" content="New York Genome Center, New York, NY, USA">
<meta name="citation_author" content="Yohyoh Wang">
<meta name="citation_author_institution" content="New York Genome Center, New York, NY, USA">
<meta name="citation_author_institution" content="Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author" content="Asrar I Alimohamed">
<meta name="citation_author_institution" content="Mass General Cancer Center, Massachusetts General Hospital, Boston, MA, USA">
<meta name="citation_author" content="Murtaza S Malbari">
<meta name="citation_author_institution" content="Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author" content="Ashish Saxena">
<meta name="citation_author_institution" content="Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author" content="Margaret K Callahan">
<meta name="citation_author_institution" content="Memorial Sloan Kettering Cancer Center, New York, NY, USA">
<meta name="citation_author" content="Dennie T Frederick">
<meta name="citation_author_institution" content="Mass General Cancer Center, Massachusetts General Hospital, Boston, MA, USA">
<meta name="citation_author" content="Lavinia Spain">
<meta name="citation_author_institution" content="Cancer Dynamics Laboratory, The Francis Crick Institute, London, UK">
<meta name="citation_author_institution" content="Renal and Skin Unit, The Royal Marsden NHS Foundation Trust, London, UK">
<meta name="citation_author" content="Michael Sigouros">
<meta name="citation_author_institution" content="Caryl and Israel Englander Institute for Precision Medicine, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author" content="Jyothi Manohar">
<meta name="citation_author_institution" content="Caryl and Israel Englander Institute for Precision Medicine, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author" content="Abigail King">
<meta name="citation_author_institution" content="Caryl and Israel Englander Institute for Precision Medicine, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author" content="David Wilkes">
<meta name="citation_author_institution" content="Caryl and Israel Englander Institute for Precision Medicine, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author" content="John Otilano">
<meta name="citation_author_institution" content="Caryl and Israel Englander Institute for Precision Medicine, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author" content="Olivier Elemento">
<meta name="citation_author_institution" content="Caryl and Israel Englander Institute for Precision Medicine, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author_institution" content="Department of Physiology and Biophysics, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author" content="Juan Miguel Mosquera">
<meta name="citation_author_institution" content="Caryl and Israel Englander Institute for Precision Medicine, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author_institution" content="Department of Physiology and Biophysics, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author" content="Ariel Jaimovich">
<meta name="citation_author_institution" content="Ultima Genomics, Fremont, CA, USA">
<meta name="citation_author" content="Doron Lipson">
<meta name="citation_author_institution" content="Ultima Genomics, Fremont, CA, USA">
<meta name="citation_author" content="Samra Turajlic">
<meta name="citation_author_institution" content="Cancer Dynamics Laboratory, The Francis Crick Institute, London, UK">
<meta name="citation_author_institution" content="Renal and Skin Unit, The Royal Marsden NHS Foundation Trust, London, UK">
<meta name="citation_author" content="Michael C Zody">
<meta name="citation_author_institution" content="New York Genome Center, New York, NY, USA">
<meta name="citation_author" content="Nasser K Altorki">
<meta name="citation_author_institution" content="Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author" content="Jedd D Wolchok">
<meta name="citation_author_institution" content="Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author_institution" content="Parker Institute for Cancer Immunotherapy, San Francisco, CA, USA">
<meta name="citation_author_institution" content="Ludwig Institute for Cancer Research; Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author" content="Michael A Postow">
<meta name="citation_author_institution" content="Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author_institution" content="Memorial Sloan Kettering Cancer Center, New York, NY, USA">
<meta name="citation_author" content="Nicolas Robine">
<meta name="citation_author_institution" content="New York Genome Center, New York, NY, USA">
<meta name="citation_author" content="Bishoy M Faltas">
<meta name="citation_author_institution" content="Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author_institution" content="Caryl and Israel Englander Institute for Precision Medicine, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author_institution" content="Department of Cell and Developmental Biology, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_author" content="Genevieve Boland">
<meta name="citation_author_institution" content="Mass General Cancer Center, Massachusetts General Hospital, Boston, MA, USA">
<meta name="citation_author" content="Dan A Landau">
<meta name="citation_author_institution" content="New York Genome Center, New York, NY, USA">
<meta name="citation_author_institution" content="Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA">
<meta name="citation_publication_date" content="2025 Apr 11">
<meta name="citation_volume" content="22">
<meta name="citation_issue" content="5">
<meta name="citation_firstpage" content="973">
<meta name="citation_doi" content="10.1038/s41592-025-02648-9">
<meta name="citation_pmid" content="40217113">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12077166/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12077166/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12077166/pdf/nihms-2073968.pdf">
<meta name="description" content="Differentiating sequencing errors from true variants is a central genomics challenge, calling for error suppression strategies that balance costs and sensitivity. For example, circulating cell-free DNA (ccfDNA) sequencing for cancer monitoring is ...">
<meta name="og:title" content="Error-corrected flow-based sequencing at whole genome scale and its application to circulating cell-free DNA profiling">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Differentiating sequencing errors from true variants is a central genomics challenge, calling for error suppression strategies that balance costs and sensitivity. For example, circulating cell-free DNA (ccfDNA) sequencing for cancer monitoring is ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12077166/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://pmc.ncbi.nlm.nih.gov/search/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12077166">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41592-025-02648-9"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/nihms-2073968.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12077166%2F%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12077166/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12077166/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12077166/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-nihpa.png" alt="NIHPA Author Manuscripts logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to NIHPA Author Manuscripts" title="Link to NIHPA Author Manuscripts" shape="default" href="https://www.hhs.gov/open/public-access-guiding-principles/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Methods</button></div>. Author manuscript; available in PMC: 2025 Nov 1.</div>
<div>
<em>Published in final edited form as: </em>Nat Methods. 2025 Apr 11;22(5):973–981. doi: <a href="https://doi.org/10.1038/s41592-025-02648-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41592-025-02648-9</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://pmc.ncbi.nlm.nih.gov/search/?term=%22Nat%20Methods%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Methods%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Methods%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Methods%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Error-corrected flow-based sequencing at whole genome scale and its application to circulating cell-free DNA profiling</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cheng%20AP%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Alexandre Pellan Cheng</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Alexandre Pellan Cheng</span></h3>
<div class="p">
<sup>1</sup>New York Genome Center, New York, NY, USA</div>
<div class="p">
<sup>2</sup>Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cheng%20AP%22%5BAuthor%5D" class="usa-link"><span class="name western">Alexandre Pellan Cheng</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>+,</sup><sup>#,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Widman%20AJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Adam J Widman</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Adam J Widman</span></h3>
<div class="p">
<sup>1</sup>New York Genome Center, New York, NY, USA</div>
<div class="p">
<sup>3</sup>Memorial Sloan Kettering Cancer Center, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Widman%20AJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Adam J Widman</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>+</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Arora%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Anushri Arora</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Anushri Arora</span></h3>
<div class="p">
<sup>1</sup>New York Genome Center, New York, NY, USA</div>
<div class="p">
<sup>2</sup>Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Arora%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Anushri Arora</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>+</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rusinek%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Itai Rusinek</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Itai Rusinek</span></h3>
<div class="p">
<sup>4</sup>Ultima Genomics, Fremont, CA, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rusinek%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Itai Rusinek</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sossin%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Aaron Sossin</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Aaron Sossin</span></h3>
<div class="p">
<sup>1</sup>New York Genome Center, New York, NY, USA</div>
<div class="p">
<sup>2</sup>Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sossin%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Aaron Sossin</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rajagopalan%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Srinivas Rajagopalan</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Srinivas Rajagopalan</span></h3>
<div class="p">
<sup>1</sup>New York Genome Center, New York, NY, USA</div>
<div class="p">
<sup>2</sup>Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rajagopalan%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Srinivas Rajagopalan</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Midler%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Nicholas Midler</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Nicholas Midler</span></h3>
<div class="p">
<sup>1</sup>New York Genome Center, New York, NY, USA</div>
<div class="p">
<sup>2</sup>Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Midler%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nicholas Midler</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hooper%20WF%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">William F Hooper</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">William F Hooper</span></h3>
<div class="p">
<sup>1</sup>New York Genome Center, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hooper%20WF%22%5BAuthor%5D" class="usa-link"><span class="name western">William F Hooper</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Murray%20RM%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Rebecca M Murray</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Rebecca M Murray</span></h3>
<div class="p">
<sup>1</sup>New York Genome Center, New York, NY, USA</div>
<div class="p">
<sup>2</sup>Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Murray%20RM%22%5BAuthor%5D" class="usa-link"><span class="name western">Rebecca M Murray</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Halmos%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Daniel Halmos</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Daniel Halmos</span></h3>
<div class="p">
<sup>1</sup>New York Genome Center, New York, NY, USA</div>
<div class="p">
<sup>2</sup>Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Halmos%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Daniel Halmos</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Langanay%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Theophile Langanay</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Theophile Langanay</span></h3>
<div class="p">
<sup>1</sup>New York Genome Center, New York, NY, USA</div>
<div class="p">
<sup>2</sup>Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Langanay%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Theophile Langanay</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chu%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Hoyin Chu</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Hoyin Chu</span></h3>
<div class="p">
<sup>2</sup>Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chu%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hoyin Chu</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Inghirami%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Giorgio Inghirami</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Giorgio Inghirami</span></h3>
<div class="p">
<sup>2</sup>Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Inghirami%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Giorgio Inghirami</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Potenski%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Catherine Potenski</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Catherine Potenski</span></h3>
<div class="p">
<sup>1</sup>New York Genome Center, New York, NY, USA</div>
<div class="p">
<sup>2</sup>Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Potenski%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Catherine Potenski</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Germer%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Soren Germer</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Soren Germer</span></h3>
<div class="p">
<sup>1</sup>New York Genome Center, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Germer%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Soren Germer</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Marton%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Melissa Marton</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Melissa Marton</span></h3>
<div class="p">
<sup>1</sup>New York Genome Center, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Marton%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Melissa Marton</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Manaa%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Dina Manaa</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Dina Manaa</span></h3>
<div class="p">
<sup>1</sup>New York Genome Center, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Manaa%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dina Manaa</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Helland%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id18"><span class="name western">Adrienne Helland</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Adrienne Helland</span></h3>
<div class="p">
<sup>1</sup>New York Genome Center, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Helland%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Adrienne Helland</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Furatero%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id19"><span class="name western">Rob Furatero</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Rob Furatero</span></h3>
<div class="p">
<sup>1</sup>New York Genome Center, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Furatero%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rob Furatero</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22McClintock%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id20"><span class="name western">Jaime McClintock</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Jaime McClintock</span></h3>
<div class="p">
<sup>1</sup>New York Genome Center, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22McClintock%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jaime McClintock</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Winterkorn%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id21"><span class="name western">Lara Winterkorn</span></a><div hidden="hidden" id="id21">
<h3><span class="name western">Lara Winterkorn</span></h3>
<div class="p">
<sup>1</sup>New York Genome Center, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Winterkorn%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lara Winterkorn</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Steinsnyder%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id22"><span class="name western">Zoe Steinsnyder</span></a><div hidden="hidden" id="id22">
<h3><span class="name western">Zoe Steinsnyder</span></h3>
<div class="p">
<sup>1</sup>New York Genome Center, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Steinsnyder%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zoe Steinsnyder</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id23"><span class="name western">Yohyoh Wang</span></a><div hidden="hidden" id="id23">
<h3><span class="name western">Yohyoh Wang</span></h3>
<div class="p">
<sup>1</sup>New York Genome Center, New York, NY, USA</div>
<div class="p">
<sup>2</sup>Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yohyoh Wang</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alimohamed%20AI%22%5BAuthor%5D" class="usa-link" aria-describedby="id24"><span class="name western">Asrar I Alimohamed</span></a><div hidden="hidden" id="id24">
<h3><span class="name western">Asrar I Alimohamed</span></h3>
<div class="p">
<sup>5</sup>Mass General Cancer Center, Massachusetts General Hospital, Boston, MA, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alimohamed%20AI%22%5BAuthor%5D" class="usa-link"><span class="name western">Asrar I Alimohamed</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Malbari%20MS%22%5BAuthor%5D" class="usa-link" aria-describedby="id25"><span class="name western">Murtaza S Malbari</span></a><div hidden="hidden" id="id25">
<h3><span class="name western">Murtaza S Malbari</span></h3>
<div class="p">
<sup>2</sup>Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Malbari%20MS%22%5BAuthor%5D" class="usa-link"><span class="name western">Murtaza S Malbari</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Saxena%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id26"><span class="name western">Ashish Saxena</span></a><div hidden="hidden" id="id26">
<h3><span class="name western">Ashish Saxena</span></h3>
<div class="p">
<sup>2</sup>Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Saxena%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Ashish Saxena</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Callahan%20MK%22%5BAuthor%5D" class="usa-link" aria-describedby="id27"><span class="name western">Margaret K Callahan</span></a><div hidden="hidden" id="id27">
<h3><span class="name western">Margaret K Callahan</span></h3>
<div class="p">
<sup>3</sup>Memorial Sloan Kettering Cancer Center, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Callahan%20MK%22%5BAuthor%5D" class="usa-link"><span class="name western">Margaret K Callahan</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Frederick%20DT%22%5BAuthor%5D" class="usa-link" aria-describedby="id28"><span class="name western">Dennie T Frederick</span></a><div hidden="hidden" id="id28">
<h3><span class="name western">Dennie T Frederick</span></h3>
<div class="p">
<sup>5</sup>Mass General Cancer Center, Massachusetts General Hospital, Boston, MA, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Frederick%20DT%22%5BAuthor%5D" class="usa-link"><span class="name western">Dennie T Frederick</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Spain%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id29"><span class="name western">Lavinia Spain</span></a><div hidden="hidden" id="id29">
<h3><span class="name western">Lavinia Spain</span></h3>
<div class="p">
<sup>6</sup>Cancer Dynamics Laboratory, The Francis Crick Institute, London, UK</div>
<div class="p">
<sup>7</sup>Renal and Skin Unit, The Royal Marsden NHS Foundation Trust, London, UK</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Spain%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lavinia Spain</span></a>
</div>
</div>
<sup>6,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sigouros%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id30"><span class="name western">Michael Sigouros</span></a><div hidden="hidden" id="id30">
<h3><span class="name western">Michael Sigouros</span></h3>
<div class="p">
<sup>8</sup>Caryl and Israel Englander Institute for Precision Medicine, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sigouros%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Michael Sigouros</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Manohar%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id31"><span class="name western">Jyothi Manohar</span></a><div hidden="hidden" id="id31">
<h3><span class="name western">Jyothi Manohar</span></h3>
<div class="p">
<sup>8</sup>Caryl and Israel Englander Institute for Precision Medicine, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Manohar%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jyothi Manohar</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22King%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id32"><span class="name western">Abigail King</span></a><div hidden="hidden" id="id32">
<h3><span class="name western">Abigail King</span></h3>
<div class="p">
<sup>8</sup>Caryl and Israel Englander Institute for Precision Medicine, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22King%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Abigail King</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wilkes%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id33"><span class="name western">David Wilkes</span></a><div hidden="hidden" id="id33">
<h3><span class="name western">David Wilkes</span></h3>
<div class="p">
<sup>8</sup>Caryl and Israel Englander Institute for Precision Medicine, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wilkes%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">David Wilkes</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Otilano%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id34"><span class="name western">John Otilano</span></a><div hidden="hidden" id="id34">
<h3><span class="name western">John Otilano</span></h3>
<div class="p">
<sup>8</sup>Caryl and Israel Englander Institute for Precision Medicine, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Otilano%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">John Otilano</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Elemento%20O%22%5BAuthor%5D" class="usa-link" aria-describedby="id35"><span class="name western">Olivier Elemento</span></a><div hidden="hidden" id="id35">
<h3><span class="name western">Olivier Elemento</span></h3>
<div class="p">
<sup>8</sup>Caryl and Israel Englander Institute for Precision Medicine, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">
<sup>9</sup>Department of Physiology and Biophysics, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Elemento%20O%22%5BAuthor%5D" class="usa-link"><span class="name western">Olivier Elemento</span></a>
</div>
</div>
<sup>8,</sup><sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mosquera%20JM%22%5BAuthor%5D" class="usa-link" aria-describedby="id36"><span class="name western">Juan Miguel Mosquera</span></a><div hidden="hidden" id="id36">
<h3><span class="name western">Juan Miguel Mosquera</span></h3>
<div class="p">
<sup>8</sup>Caryl and Israel Englander Institute for Precision Medicine, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">
<sup>9</sup>Department of Physiology and Biophysics, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">
<sup>10</sup>Department of Pathology and Laboratory Medicine, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mosquera%20JM%22%5BAuthor%5D" class="usa-link"><span class="name western">Juan Miguel Mosquera</span></a>
</div>
</div>
<sup>8,</sup><sup>9,</sup><sup>10</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jaimovich%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id37"><span class="name western">Ariel Jaimovich</span></a><div hidden="hidden" id="id37">
<h3><span class="name western">Ariel Jaimovich</span></h3>
<div class="p">
<sup>4</sup>Ultima Genomics, Fremont, CA, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jaimovich%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Ariel Jaimovich</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lipson%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id38"><span class="name western">Doron Lipson</span></a><div hidden="hidden" id="id38">
<h3><span class="name western">Doron Lipson</span></h3>
<div class="p">
<sup>4</sup>Ultima Genomics, Fremont, CA, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lipson%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Doron Lipson</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Turajlic%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id39"><span class="name western">Samra Turajlic</span></a><div hidden="hidden" id="id39">
<h3><span class="name western">Samra Turajlic</span></h3>
<div class="p">
<sup>6</sup>Cancer Dynamics Laboratory, The Francis Crick Institute, London, UK</div>
<div class="p">
<sup>7</sup>Renal and Skin Unit, The Royal Marsden NHS Foundation Trust, London, UK</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Turajlic%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Samra Turajlic</span></a>
</div>
</div>
<sup>6,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zody%20MC%22%5BAuthor%5D" class="usa-link" aria-describedby="id40"><span class="name western">Michael C Zody</span></a><div hidden="hidden" id="id40">
<h3><span class="name western">Michael C Zody</span></h3>
<div class="p">
<sup>1</sup>New York Genome Center, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zody%20MC%22%5BAuthor%5D" class="usa-link"><span class="name western">Michael C Zody</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Altorki%20NK%22%5BAuthor%5D" class="usa-link" aria-describedby="id41"><span class="name western">Nasser K Altorki</span></a><div hidden="hidden" id="id41">
<h3><span class="name western">Nasser K Altorki</span></h3>
<div class="p">
<sup>2</sup>Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Altorki%20NK%22%5BAuthor%5D" class="usa-link"><span class="name western">Nasser K Altorki</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wolchok%20JD%22%5BAuthor%5D" class="usa-link" aria-describedby="id42"><span class="name western">Jedd D Wolchok</span></a><div hidden="hidden" id="id42">
<h3><span class="name western">Jedd D Wolchok</span></h3>
<div class="p">
<sup>2</sup>Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">
<sup>11</sup>Parker Institute for Cancer Immunotherapy, San Francisco, CA, USA</div>
<div class="p">
<sup>12</sup>Ludwig Institute for Cancer Research; Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wolchok%20JD%22%5BAuthor%5D" class="usa-link"><span class="name western">Jedd D Wolchok</span></a>
</div>
</div>
<sup>2,</sup><sup>11,</sup><sup>12</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Postow%20MA%22%5BAuthor%5D" class="usa-link" aria-describedby="id43"><span class="name western">Michael A Postow</span></a><div hidden="hidden" id="id43">
<h3><span class="name western">Michael A Postow</span></h3>
<div class="p">
<sup>2</sup>Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">
<sup>3</sup>Memorial Sloan Kettering Cancer Center, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Postow%20MA%22%5BAuthor%5D" class="usa-link"><span class="name western">Michael A Postow</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Robine%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id44"><span class="name western">Nicolas Robine</span></a><div hidden="hidden" id="id44">
<h3><span class="name western">Nicolas Robine</span></h3>
<div class="p">
<sup>1</sup>New York Genome Center, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Robine%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nicolas Robine</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Faltas%20BM%22%5BAuthor%5D" class="usa-link" aria-describedby="id45"><span class="name western">Bishoy M Faltas</span></a><div hidden="hidden" id="id45">
<h3><span class="name western">Bishoy M Faltas</span></h3>
<div class="p">
<sup>2</sup>Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">
<sup>8</sup>Caryl and Israel Englander Institute for Precision Medicine, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">
<sup>13</sup>Department of Cell and Developmental Biology, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Faltas%20BM%22%5BAuthor%5D" class="usa-link"><span class="name western">Bishoy M Faltas</span></a>
</div>
</div>
<sup>2,</sup><sup>8,</sup><sup>13,</sup><sup>^</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Boland%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id46"><span class="name western">Genevieve Boland</span></a><div hidden="hidden" id="id46">
<h3><span class="name western">Genevieve Boland</span></h3>
<div class="p">
<sup>5</sup>Mass General Cancer Center, Massachusetts General Hospital, Boston, MA, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Boland%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Genevieve Boland</span></a>
</div>
</div>
<sup>5,</sup><sup>^</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Landau%20DA%22%5BAuthor%5D" class="usa-link" aria-describedby="id47"><span class="name western">Dan A Landau</span></a><div hidden="hidden" id="id47">
<h3><span class="name western">Dan A Landau</span></h3>
<div class="p">
<sup>1</sup>New York Genome Center, New York, NY, USA</div>
<div class="p">
<sup>2</sup>Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Landau%20DA%22%5BAuthor%5D" class="usa-link"><span class="name western">Dan A Landau</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>^,</sup><sup>#</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="A1">
<sup>1</sup>New York Genome Center, New York, NY, USA</div>
<div id="A2">
<sup>2</sup>Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA</div>
<div id="A3">
<sup>3</sup>Memorial Sloan Kettering Cancer Center, New York, NY, USA</div>
<div id="A4">
<sup>4</sup>Ultima Genomics, Fremont, CA, USA</div>
<div id="A5">
<sup>5</sup>Mass General Cancer Center, Massachusetts General Hospital, Boston, MA, USA</div>
<div id="A6">
<sup>6</sup>Cancer Dynamics Laboratory, The Francis Crick Institute, London, UK</div>
<div id="A7">
<sup>7</sup>Renal and Skin Unit, The Royal Marsden NHS Foundation Trust, London, UK</div>
<div id="A8">
<sup>8</sup>Caryl and Israel Englander Institute for Precision Medicine, Weill Cornell Medicine, New York, NY, USA</div>
<div id="A9">
<sup>9</sup>Department of Physiology and Biophysics, Weill Cornell Medicine, New York, NY, USA</div>
<div id="A10">
<sup>10</sup>Department of Pathology and Laboratory Medicine, Weill Cornell Medicine, New York, NY, USA</div>
<div id="A11">
<sup>11</sup>Parker Institute for Cancer Immunotherapy, San Francisco, CA, USA</div>
<div id="A12">
<sup>12</sup>Ludwig Institute for Cancer Research; Weill Cornell Medicine, New York, NY, USA</div>
<div id="A13">
<sup>13</sup>Department of Cell and Developmental Biology, Weill Cornell Medicine, New York, NY, USA</div>
<div class="author-notes p">
<div class="fn" id="FN1">
<sup>*</sup><p class="display-inline" id="P1">Current affiliations: Département de génie des systèmes, École de technologie supérieure, Montréal, QC, Canada; Axe Cancer, Centre de Recherche du Centre Hospitalier de l’Université de Montréal (CRCHUM), Montréal, QC, Canada</p>
</div>
<div class="fn" id="FN2">
<sup>+</sup><p class="display-inline" id="P2">These authors contributed equally</p>
</div>
<div class="fn" id="FN3">
<sup>^</sup><p class="display-inline" id="P3">Jointly supervised the work</p>
</div>
<div class="fn" id="FN4">
<p id="P4">CONTRIBUTIONS</p>
<p id="P5">D.A.L., A.P.C., A.J.W., G.B. and B.M.F. conceived and designed the project. D.A.L., G.B. and B.M.F. served as lead principal investigators. A.J.W., A.I.A., M.S.M., A.Sa., M.K.C., D.T.F., L.S., M.S., J.M., A.K., S.T., D.W., J.O., O.E., J.M.M., N.K., J.D.W., M.A.P. G.B., B.M.F. performed patient selection, curated patient data and prepared samples for sequencing. G.I. provided mouse PDX samples. M.M., D.M., A.H., R.F., J.M., L.M., Z.S., L.W. performed library preparation and sequencing. A.P.C., A.A., I.R., A.So., S.R., N.M., W.F.H., R.M.M., D.H., T.L., H.C., S.G., M.C.Z., N.R., Y.W., A.J., D.L. performed data analysis. A.P.C., C.P. and D.A.L. wrote the manuscript with comments and contributions from all authors.</p>
</div>
<div class="fn" id="CR1">
<sup>#</sup><p class="display-inline">Correspondence: <span>alexandre.cheng@etsmtl.ca</span> and <span>dlandau@nygenome.org</span></p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Issue date 2025 May.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div>PMCID: PMC12077166  NIHMSID: NIHMS2073968  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40217113/" class="usa-link">40217113</a>
</div>
<div class="ra xbox p" role="complementary" aria-label="Related or updated information about this article"><div>The publisher's version of this article is available at <a href="https://doi.org/10.1038/s41592-025-02648-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Nat Methods</a>
</div></div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="ABS1"><h2>Abstract</h2>
<p id="P6">Differentiating sequencing errors from true variants is a central genomics challenge, calling for error suppression strategies that balance costs and sensitivity. For example, circulating cell-free DNA (ccfDNA) sequencing for cancer monitoring is limited by sparsity of circulating tumor DNA (ctDNA), abundance of genomic material in samples, and pre-analytical error rates. Whole-genome sequencing (WGS) can overcome the low abundance of ccfDNA by integrating signal across the mutation landscape, but higher cost limits its wide adoption. Here we applied deep (~120x) lower-cost WGS (Ultima Genomics) for tumor-informed ctDNA detection within the part-per-million range. We further leveraged lower cost sequencing by developing duplex error-corrected WGS of ccfDNA, achieving 7.7×10<sup>−7</sup> error rates, allowing us to assess disease burden in melanoma and urothelial cancer patients without matched tumor sequencing. This error-corrected WGS approach will have broad applicability across genomics, allowing for accurate calling of low abundance variants at efficient cost, enabling deeper mapping of somatic mosaicism as an emerging central aspect of aging and disease.</p></section><section class="abstract" id="ABS2"><h2>Editorial summary:</h2>
<p id="P7">This work integrates duplex sequencing with cost-effective Ultima sequencing to enhance the accuracy of whole-genome circulating cell-free DNA profiling.</p></section><section id="S1"><h2 class="pmc_sec_title">INTRODUCTION</h2>
<p id="P8">Distinguishing sequencing errors from true variants continues to challenge the genomics field, with tradeoffs between cost and error suppression strategies. These challenges are magnified when identifying ultrarare variants. For example, profiling circulating cell-free DNA (ccfDNA) is a promising clinical tool for non-invasive cancer detection<sup><a href="#R1" class="usa-link" aria-describedby="R1">1</a>–<a href="#R8" class="usa-link" aria-describedby="R8">8</a></sup>. Sequencing of somatic variants in circulating tumor DNA (ctDNA)<sup><a href="#R9" class="usa-link" aria-describedby="R9">9</a>–<a href="#R12" class="usa-link" aria-describedby="R12">12</a></sup> can aid in detection of low burden disease, such as cancer screening, minimal residual disease (MRD)<sup><a href="#R11" class="usa-link" aria-describedby="R11">11</a>,<a href="#R13" class="usa-link" aria-describedby="R13">13</a>–<a href="#R16" class="usa-link" aria-describedby="R16">16</a></sup>, and relapse monitoring<sup><a href="#R17" class="usa-link" aria-describedby="R17">17</a>,<a href="#R18" class="usa-link" aria-describedby="R18">18</a></sup>. However, when disease burden is low, robust detection requires methods with exquisite sensitivity to detect ctDNA signal over the background rate of sequencing or library preparation errors. The technical challenge is typically overcome by increasing sequencing depth at select genomic locations, accompanied by techniques that decrease sequencing error rate, including unique molecular identifier (UMI) error suppression techniques<sup><a href="#R10" class="usa-link" aria-describedby="R10">10</a>,<a href="#R19" class="usa-link" aria-describedby="R19">19</a></sup> or duplex sequencing<sup><a href="#R12" class="usa-link" aria-describedby="R12">12</a>,<a href="#R20" class="usa-link" aria-describedby="R20">20</a>–<a href="#R22" class="usa-link" aria-describedby="R22">22</a></sup>, which enable increased accuracy in differentiating true somatic variants from sequencing artifacts, to optimize detection of low-burden disease.</p>
<p id="P9">Prevailing methods of ctDNA detection use targeted sequencing, which increases the number of genomes sequenced at a targeted location. However, high-throughput targeted sequencing rapidly exhausts available genomes (1,000–10,000 genome equivalents (GEs) per mL of plasma<sup><a href="#R23" class="usa-link" aria-describedby="R23">23</a></sup>), setting a ceiling on ctDNA detection, where further increases in sequencing depth at targeted sites afford no advantage after the limited number of GEs has already been sequenced. Alternatively, whole-genome sequencing (WGS) approaches exploit breadth of coverage to supplant depth, eliminating the reliance on detecting few sites to increase ctDNA characterization in low tumor fraction settings. For example, our recent methods MRDetect<sup><a href="#R14" class="usa-link" aria-describedby="R14">14</a></sup> and MRD-EDGE<sup><a href="#R24" class="usa-link" aria-describedby="R24">24</a></sup> use matched primary tumor mutational profiles to inform genome-wide tumor single nucleotide variant (SNV) detection in ccfDNA, such that the available number of GEs is no longer the limiting factor for ctDNA detection.</p>
<p id="P10">The detection challenges presented by sparsity call for broad, accurate and deep ccfDNA sequencing. Thus, whole-genome, low-error, high-coverage methods are necessary for robust ctDNA analysis. However, the costs associated with these approaches are often prohibitive. Although genome sequencing costs have rapidly dropped, more recently this decrease has stagnated<sup><a href="#R25" class="usa-link" aria-describedby="R25">25</a></sup>, rendering sequencing cost a significant barrier for implementing high-depth WGS for liquid biopsies, where shallow WGS is insufficient for ctDNA detection when tumor fractions are low (e.g., ~10<sup>−5</sup>).</p>
<p id="P11">Recently, low-cost mostly natural sequencing-by-synthesis (mnSBS)<sup><a href="#R26" class="usa-link" aria-describedby="R26">26</a>,<a href="#R27" class="usa-link" aria-describedby="R27">27</a></sup> has been developed by Ultima Genomics, where the flow-based platform produces single-end reads at ~10 billion reads per run for 1USD/Gb, lowering costs compared with current platforms. To date, mnSBS/Ultima sequencing has not been harnessed for clinical ctDNA sequencing in ccfDNA samples and error rate profiles have not been fully characterized or rigorously compared with competing technologies. For potential application to clinical ctDNA monitoring, accurate error rate estimates are critical due to the required high sensitivity of ctDNA detection<sup><a href="#R28" class="usa-link" aria-describedby="R28">28</a></sup>.</p>
<p id="P12">To investigate deep WGS for ctDNA detection, we used the Ultima platform to sequence ccfDNA from plasma samples from healthy controls, cancer patients and patient-derived xenograft (PDX) mouse models. We show that deep plasma WGS (~120x) allows tumor-informed ctDNA detection within the part-per-million range. We further leveraged the cost-effective and high-throughput nature of mnSBS to develop high-coverage WGS duplex error corrected libraries of ccfDNA, achieving error rates as low as 7.7×10<sup>−8</sup>. This allowed us to combine the advantages of genome-wide mutational integration on the one hand, and molecular error correction on the other, to assess disease burden in melanoma and urothelial cancer patients without relying on matched tumor sequencing. Together, our results demonstrate the feasibility and utility of deep WGS for ctDNA detection and duplex sequencing at the scale of whole genome.</p></section><section id="S2"><h2 class="pmc_sec_title">RESULTS</h2>
<section id="S3"><h3 class="pmc_sec_title">Flow-based sequencing enables highly accurate SNV discovery</h3>
<p id="P13">Different sequencing methods have advantages and drawbacks. In flow-based Ultima, sequencing signal intensity translates to the number of consecutive bases of a given nucleotide, increasing susceptibility to homopolymer size estimation errors. However, as each sequencing cycle encompasses a single base, flow-based sequencing systems could be particularly robust to SNV errors.</p>
<p id="P14">To investigate error rates of the flow-based Ultima Genomics platform, we ligated molecular barcodes to the plasma of mouse PDX (n=4; n=1 lung cancer; n=3 diffuse large B cell lymphoma human-mapped fractions of 0.4%, 40%, 73% and 96%) samples. We computationally tracked PCR duplicates in the three PDX samples with high human-mapped read fractions to identify sequencing artifacts and compare error profiles in matched Ultima-Illumina datasets. First, we analyzed homopolymer length discordance rate between PCR duplicates and between a read and the reference genome. We randomly sampled up to 33 million unique ccfDNA molecules (1.5 million molecules for each autosome) containing at least two PCR duplicates from the n=3 PDX datasets (human-mapped reads only). Each read was aligned to its duplicate and to the reference genome, and matching homopolymers were compared by their sequenced size. In Ultima datasets, we observed a strong concordance between PCR duplicates (99.34%; Q-score=21.8) and between reads to the reference genome (99.58%; Q-score=23.8) for homopolymers of size 1 to 12 (<a href="#F4" class="usa-link">Extended Data Figure 1A</a>–<a href="#F4" class="usa-link">B</a>). We also observed artifactual homopolymer sizes near lengths of 12, as Ultima sequencing reports a maximum homopolymer size of 12 bases<sup><a href="#R26" class="usa-link" aria-describedby="R26">26</a></sup>. However, homopolymer size estimations were more accurate in Illumina datasets (99.99%; Q-score=39.9 between PCR duplicates and 99.98%; Q-score=36.3 between a read and the reference genome; <a href="#F4" class="usa-link">Extended Data Figure 1C</a>–<a href="#F4" class="usa-link">D</a>). We also found that accuracy decreased as a function of homopolymer length in both technologies (Spearman’s rho=−0.925, p-value=6.5×10<sup>−16</sup> [Ultima] and −0.996, p-value &lt;2.2×10<sup>−16</sup> [Illumina], <a href="#F4" class="usa-link">Extended Data Figure 1E</a>–<a href="#F4" class="usa-link">F</a>). For example, the homopolymer size estimation error rate was 1.57×10<sup>−3</sup>, 4.46×10<sup>−3</sup>, 8.51×10<sup>−3</sup> for homopolymers of size 1, 2 and 3, respectively in Ultima datasets and 1.02×10<sup>−4</sup>, 1.66×10<sup>−4</sup> and 2.63×10<sup>−4</sup> in Illumina datasets. Expectedly, concordance improved in Ultima datasets after PCR duplicate consensus calling (error rates of 4.08×10<sup>−4</sup>, 5.24×10<sup>−4</sup>, 8.06×10<sup>−4</sup> for homopolymers of size 1, 2 and 3, respectively; family size=2, <a href="#F4" class="usa-link">Extended Data Figure 1E</a>). For single read accuracy, we observed that Ultima homopolymer accuracy falls below 99% at homopolymer lengths of 4 and greater (3.17% of the human genome) (<a href="#F4" class="usa-link">Extended Data Figure 1E</a>,<a href="#F4" class="usa-link">G</a>), whereas matched Illumina datasets fall below 99% at homopolymer lengths of 8 and greater (0.09% of the genome) (<a href="#F4" class="usa-link">Extended Data Figure 1F</a>,<a href="#F4" class="usa-link">G</a>).</p>
<p id="P15">To further compare the performance of in terms of SNV errors, we first identified putative sequencing errors, defined as mismatched PCR duplicates, where only one of the reads contains a mismatch with the reference genome. Overall, single nucleotide differences between PCR duplicates occurred at a rate of 1.58×10<sup>−4</sup> ± 2.65×10<sup>−4</sup> in Ultima datasets, and 9.85×10<sup>−4</sup> ±10.80×10<sup>−4</sup> in Illumina (<a href="#F5" class="usa-link">Extended Data Figure 2A</a>). Interestingly, we found wide variation in sequencing error rates that depended on the trinucleotide context of the variant. In Ultima datasets, errors were most likely to occur in specific motifs: trinucleotides containing a two-mer (for example, C[C&gt;A]T), or where a two-mer would form upon mutation (for example, G[A&gt;G]T), likely reflecting homopolymer size estimation errors that manifest as SNV errors. Conversely, we found that certain trinucleotides contexts were robust to sequencing error, namely when a reported trinucleotide mutation would cause a shift in the number of sequencing cycles when compared to the reference (termed cycle shift motifs, <a href="#F5" class="usa-link">Extended Data Figure 2B</a>). For example, assuming a sequencing cycle of T&gt;G&gt;C&gt;A, T[G&gt;A]C would be considered a cycle shift, while T[G&gt;C]C would not. In Ultima datasets, cycle shift motifs had significantly lower error rates than non-cycle shift motifs (mean of 5.23×10<sup>−5</sup> vs 2.32×10<sup>−4</sup> in cycle shifts and non-cycle shifts, respectively. P-value &lt; 2.2×10<sup>−16</sup>
<a href="#F5" class="usa-link">Extended Data Figure 2C</a>). Cycle shift motifs with the highest error rates were exclusively in C&gt;T mutations, the most common somatic mutation. These may present rare cases where the PCR duplicate containing the mismatch is correct, and the PCR duplicate that matches the reference contains the error. Expectedly, there were no significant differences in error rates between cycle shift and non-cycle shift motifs in Illumina datasets (mean error rate of 9.3×10<sup>−4</sup> vs 1.0×10<sup>−3</sup> in cycle shifts and non-cycle shifts, respectively, p-value=0.16, <a href="#F5" class="usa-link">Extended Data Figure 2C</a>).</p></section><section id="S4"><h3 class="pmc_sec_title">Plasma whole-genome sequencing can detect low tumor burden</h3>
<p id="P16">Having benchmarked Ultima versus Illumina error rates, we sought to test Ultima sequencing for ctDNA detection in clinical samples. Lower limits of ctDNA mutation detection by plasma WGS are dictated by tumor mutational burden, depth of sequencing, and error rates from library preparation and sequencing. To explore these dependencies, we modeled variable tumor fractions (TF), depths of coverages and error rates for a cancer with 10,000 SNVs (~3.7 mutations/Mb) (<a href="#S9" class="usa-link">Methods</a>). Tumors with &gt;10,000 SNVs are seen across ~30% of cancers in the Pan Cancer Analysis of Whole Genomes (PCAWG) consortium dataset<sup><a href="#R29" class="usa-link" aria-describedby="R29">29</a></sup> and are enriched in lung (85%), skin (79%), bladder (83%), and other cancers. This analysis suggests that detection of TFs below 10<sup>−5</sup> requires sequencing depth of ~100x with error rates below 10<sup>−4</sup> (<a href="#F1" class="usa-link">Figure 1A</a>). In these high mutational burden cancers, WGS can provide more opportunities for ctDNA detection than targeted approaches by sequencing a greater number of unique ccfDNA molecules. However, detecting low mutational burden tumors or specific driver variants is better served with targeted approaches (<a href="#F6" class="usa-link">Extended Data Figure 3</a>).</p>
<figure class="fig xbox font-sm" id="F1"><h4 class="obj_head">Figure 1. Ultralow ctDNA detection requires deep sequencing coverage and low error rates.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5a41/12077166/43fd8fbbdd04/nihms-2073968-f0001.jpg" loading="lazy" height="1139" width="1924" alt="Figure 1."></p>
<div class="p text-right font-secondary"><a href="figure/F1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P75"><strong>A</strong> Simulation of ctDNA detection given different error rates (columns), whole genome coverages (rows) and tumor fractions (x-axis). (n=1,000 replicates per set of conditions). <strong>B</strong> Cell-free DNA library preparation pre-analytical workflow. <strong>C</strong> Sequencing depth of matched Illumina and Ultima datasets. <strong>D</strong> Normalized read coverage for Illumina (top) and Ultima sequenced (bottom) matched cell-free DNA sample. <strong>E</strong> Left: Copy number-based tumor fraction estimation measured with Illumina or Ultima sequencing in matched samples using ichorCNA. Matched cancer-free controls were used to create a panel of normals prior to tumor fraction estimation. Right: Single-nucleotide variant-based tumor fraction estimation measured with Illumina or Ultima sequencing. Somatic SNVs were identified through matched tumor-normal sequencing. Two samples without tumor sequencing and with low ctDNA fraction (&lt;5% measured through CNV analysis) were omitted from this analysis. Spearman’s ρ coefficient and corresponding two-sided p-value was calculated using the stats package (v.3.6) function in R (v.3.6). <strong>F</strong> Circulating tumor DNA cost and coverage analysis between Illumina and Ultima sequencing in a matched sample. Areas under the curve (AUCs) are measured by calculating the area under a receiver operating characteristic curve comparing a given group (for example, Illumina 20x at 10<sup>−6</sup> expected tumor fraction) to its platform and coverage-matched cancer-free control (for example, Illumina 20x, expected tumor fraction of 0). (n=20 replicates per set of conditions). All AUCs at expected tumor fractions of 10<sup>−4</sup> and greater were 1.00. Z-scores of a given sample are calculated against their coverage and platform matched cancer-free control (expected tumor fraction of 0). For all boxplots, the lower and upper ends of boxes represent the 25<sup>th</sup> and 75<sup>th</sup> percentiles of the data, respectively, and the horizontal lines represent the median. The whiskers represent at most 1.5 times the IQR.</p></figcaption></figure><p id="P17">Given the need for deeper plasma WGS, sequencing costs impede broad application. We therefore hypothesized that lower cost Ultima sequencing can help overcome this barrier, provided that sequencing error profiles are sufficiently low. As such, we performed Ultima WGS on 15 ccfDNA libraries (n=10 samples from cancer patients [n=7 stage IV melanoma; n=3 III-IV lung adenocarcinoma]; n=5 controls; sequencing depth 115x±34x [including duplicates; mean ± standard deviation]) with matching Illumina sequencing of the same libraries (including duplicates; 33x±10x) (<a href="#F1" class="usa-link">Figure 1B</a>,<a href="#F1" class="usa-link">C</a>, <a href="#SD1" class="usa-link">Supplementary Tables 1</a>–<a href="#SD1" class="usa-link">3</a>). We first measured ctDNA burden by estimating the relative abundance of large copy-number alterations using ichorCNA<sup><a href="#R30" class="usa-link" aria-describedby="R30">30</a></sup> (<a href="#S9" class="usa-link">Methods</a>). Tumor fractions varied (&lt;3%, n=3; 3–10%, n=5; &gt;10%, n=2) and measurements were strongly correlated between matched Illumina and Ultima datasets (spearman’s ρ=0.998, p-value=7.6×10<sup>−11</sup>, <a href="#F1" class="usa-link">Figure 1D</a>,<a href="#F1" class="usa-link">E</a>). Next, estimated ctDNA burden using a tumor-informed SNV approach. We performed WGS of tumor-derived DNA (using standard mutation calling on plasma DNA if tumor DNA was unavailable and ctDNA burden was &gt;5% by ichorCNA<sup><a href="#R30" class="usa-link" aria-describedby="R30">30</a></sup>, <a href="#S9" class="usa-link">Methods</a>, <a href="#SD1" class="usa-link">Supplementary Table 4</a>) and matched normal DNA from peripheral blood mononuclear cells (PBMCs) to identify tumor-specific mutations. To remove sequencing errors, we developed a quality-filtering pipeline informed by Ultima-specific feature cutoffs and blacklisted regions (<a href="#SD2" class="usa-link">Supplementary Figure 1</a>, <a href="#S9" class="usa-link">Methods</a>, <a href="#SD2" class="usa-link">Supplementary Note</a>). We mined the denoised cell-free DNA reads for somatic variants to estimate ctDNA fractions (<a href="#S9" class="usa-link">Methods</a>), finding strong agreement between Illumina and Ultima sequencing sets (spearman’s ρ=0.998, p-value=2.6×10<sup>−7</sup>, <a href="#F1" class="usa-link">Figure 1E</a>), with comparable tumor-specific error rates (<a href="#SD2" class="usa-link">Supplementary Figure 2</a>, <a href="#SD2" class="usa-link">Supplementary Note</a>) and fragment lengths (<a href="#SD2" class="usa-link">Supplementary Figure 3</a>, <a href="#SD2" class="usa-link">Supplementary Note</a>)Together, these results support the utility of Ultima sequencing for detection of low-burden ctDNA.</p>
<p id="P18">To further benchmark the two sequencing technologies, we performed in silico mixing studies<sup><a href="#R14" class="usa-link" aria-describedby="R14">14</a></sup> by computationally mixing sequencing reads from a sample with detectable tumor burden, with sequencing reads from a cancer-free control at known ratios. This method is comparable to molecular mixing studies<sup><a href="#R24" class="usa-link" aria-describedby="R24">24</a></sup> where a known amount of ccfDNA from a sample with detectable tumor burden is spiked into healthy ccfDNA at known concentrations. We generated admixtures from sample MEL-05 (stage IV melanoma patient, TF=7.3%) and CTRL-05 (no known cancer) at different ratios to create admixtures of tumor fractions ranging from 10<sup>−6</sup>-10<sup>−2</sup> (n=50 technical replicates per admixture) at 70x, 40x and 20x sequencing depth for Ultima sequenced datasets, and 20x for Illumina. Performance was evaluated at different simulated tumor fractions using a receiver operating curve (ROC) analysis.</p>
<p id="P19">We determined analytical limits of detection of a plasma sample with matched Illumina and Ultima sequencing in two different contexts: i) at matched coverage (20x each), which shows comparable performance between the two methods (<a href="#F1" class="usa-link">Figure 1F</a>; <a href="#F7" class="usa-link">Extended Data Figure 4</a>); and ii) at matched cost (20x Illumina versus 40x Ultima, assuming 1$/Gb [Ultima] and 2$/Gb [Illumina]). Expectedly, given the similar error rates of the two sequencers without denoising, and slight improvements in Ultima datasets after denoising (<a href="#SD2" class="usa-link">Supplementary Figure 2</a>), the deeper Ultima datasets (40x) demonstrate better limits of detection (AUC=0.98 at tumor fractions of 5×10<sup>−5</sup>) than price-matched Illumina datasets (20x; AUC=0.77 at tumor fractions of 5×10<sup>−5</sup>). Finally, we were able to robustly detect ctDNA in the part-per-million range in deeply sequenced Ultima datasets (AUC=0.79; 70x coverage), indicating that our deep-sequencing framework for ctDNA detection is sensitive enough at low tumor fractions for use in challenging clinical applications such as MRD monitoring.</p></section><section id="S5"><h3 class="pmc_sec_title">Ultra-low error duplex whole-genome sequencing of ccfDNA</h3>
<p id="P20">Advances in molecular error correction have radically enhanced deep-targeted sequencing approaches, for example using unique molecular identifiers (UMIs) that are incorporated during library preparation for sequencing error correction<sup><a href="#R10" class="usa-link" aria-describedby="R10">10</a>,<a href="#R31" class="usa-link" aria-describedby="R31">31</a></sup>. While strand-agnostic UMIs can correct sequencing and PCR errors, UMIs that link forward and reverse DNA strands (duplex sequencing) can correct errors that arise on only one strand (such as G&gt;T transversions from oxidative DNA damage<sup><a href="#R32" class="usa-link" aria-describedby="R32">32</a></sup>) during library preparation<sup><a href="#R22" class="usa-link" aria-describedby="R22">22</a></sup>. At the whole-genome scale, duplex sequencing has been cost prohibitive due to the need for a high rate of duplicate reads. Nonetheless, studies applying duplex sequencing at the genome scale have shown promise for genome-wide rare variant identification<sup><a href="#R20" class="usa-link" aria-describedby="R20">20</a>,<a href="#R21" class="usa-link" aria-describedby="R21">21</a></sup>.</p>
<p id="P21">We reasoned that the lower sequencing cost afforded by mnSBS could open the way for affordable genome-scale duplex sequencing in clinical settings, and decreasing sequencing and library preparation errors could enable tumor-agnostic (<em>de novo)</em> ctDNA detection, where matched tumor tissue, often unavailable for patients<sup><a href="#R33" class="usa-link" aria-describedby="R33">33</a>,<a href="#R34" class="usa-link" aria-describedby="R34">34</a></sup>, cannot be used to reduce background noise. For this important clinical context, we developed duplex WGS for single-end Ultima reads. Here, we created duplex libraries by replacing standard sequencing adapters with adapters containing 3 random nucleotides, thus creating a 6-bp duplex-UMI (using the random nucleotides from both ends of the DNA). Libraries were then sequenced using the Ultima sequencer. While duplex sequencing was developed for paired-end sequencing technologies, we recovered the ends of most ccfDNA molecules (80% of all sequenced molecules) due to their modal length of ~170 bp (<a href="#SD2" class="usa-link">Supplementary Figure 3</a>) to create 6-bp UMIs. Next, we developed a decision-tree classifier that determines the duplex variant based on the variant pileups of individual sequencing reads contributing to the duplex. This allowed us to utilize sequencing reads of the same duplex family that might differ in size due to homopolymer size estimation differences, which would be discarded by current duplex collapsing tools, and maximize duplex yields (<a href="#S9" class="usa-link">Methods</a>, <a href="#SD2" class="usa-link">Supplementary Note</a>, <a href="#SD2" class="usa-link">Supplementary Figure 4</a>), additionally lowering error rates through read end trimming (<a href="#S9" class="usa-link">Methods</a>, <a href="#SD2" class="usa-link">Supplementary Note</a>, <a href="#SD2" class="usa-link">Supplementary Figure 5</a>).</p>
<p id="P22">To test the accuracy of duplex error correction, we utilized the ccfDNA duplex libraries from PDX plasma. Tumor fractions (fraction of reads uniquely mapping to the human genome), were 0.4%, 40%, 73% and 96% (<a href="#SD1" class="usa-link">Supplementary Table 2</a>). For each sample, we denoised sequencing reads in three different ways: <em>i)</em> UMI-agnostic, where PCR duplicates (identified by their mapping positions) are removed from analysis and reads are denoised based on their sequencing and mapping qualities (<a href="#S9" class="usa-link">Methods</a>); <em>ii)</em> using UMIs to identify PCR duplicates for single-strand error correction (reads used to create a consensus are profiled base-by-base and a consensus base pair is determined by computing the likelihood of that base being either an A,T,C or G, using the sequenced nucleotides and their qualities as priors); and <em>iii)</em> using PCR duplicates and forward and reverse strands of a same double-stranded DNA template for error correction (duplex error correction). Similarly to recent studies describing novel error correction methods<sup><a href="#R35" class="usa-link" aria-describedby="R35">35</a>,<a href="#R36" class="usa-link" aria-describedby="R36">36</a></sup>, we defined a residual single nucleotide variant rate as the number of base pairs in denoised reads that were discordant with the reference genome and occurred once, divided by the total number of interrogated (i.e., denoised) bases in a sample (<a href="#S9" class="usa-link">Methods</a>). Here, we limited our analysis to normal ccfDNA (reads mapping to the mouse genome) in the three PDX samples with appreciable mouse-derived cfDNA. Overall, we obtained residual SNV rates of 3.8×10<sup>−4</sup>±3.2×10<sup>−5</sup>, 5.8×10<sup>−5</sup>±2.5×10<sup>−5</sup> and 7.7×10<sup>−7</sup>±5.8×10<sup>−7</sup> for UMI-agnostic, single-strand corrected and duplex corrected reads, respectively (<a href="#F2" class="usa-link">Figure 2A</a>; <a href="#SD1" class="usa-link">Supplementary Table 5</a>), achieving a three orders-of-magnitude reduction in error rate with duplex sequencing. When compared to uncorrected reads (without any quality filtering), we observed a ~3,300x improvement in error rates using duplex WGS (2.5×10<sup>−3</sup>±1.6×10<sup>−3</sup> error rates in uncorrected reads without any denoising), consistent with previous reports employing whole-genome duplex sequencing (2×10<sup>−7</sup> in Abascal <em>et al.</em><sup><a href="#R21" class="usa-link" aria-describedby="R21">21</a></sup>; 4.5×10<sup>−7</sup> in Bae et al.<sup><a href="#R36" class="usa-link" aria-describedby="R36">36</a></sup>), suggesting that ccfDNA errors are driven more by library preparation and DNA degradation than by sequencing errors.</p>
<figure class="fig xbox font-sm" id="F2"><h4 class="obj_head">Figure 2. Duplex correction allows ctDNA identification without tumor sequencing.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5a41/12077166/7d50ff44494b/nihms-2073968-f0002.jpg" loading="lazy" height="1357" width="1924" alt="Figure 2."></p>
<div class="p text-right font-secondary"><a href="figure/F2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P76"><strong>A</strong> Error rates in WGS on mouse PDX samples (n=3). <strong>B</strong> Variant allele frequencies (AFs), calculated using unfiltered sequencing reads) in positions where a variant was found using uncorrected reads (left; inset highlights higher AFs by enforcing a y-axis cutoff of 0.002) and in duplex corrected reads (middle). Removing germline reads reveals somatic mutations with a modal AF of 0.21 (right). <strong>C</strong> Comparison between the modal AF of a patient with progressive disease (samples MEL-12.A-E) in duplex corrected positions (AFs between 5% and 30% only) and copy-number based tumor fraction estimations. <strong>D,E</strong> Trinucleotide frequencies of a cancer-free plasma sample (CTRL-07, <strong>D</strong>) and a stage IVB cancer plasma sample (MEL-12.D, tumor fraction of 23%, <strong>E</strong>) in UMI-agnostic corrected WGS (top row), single-stranded correction (middle row) and duplex correction (bottom row). Cosine similarity with SBS7a/SBS7b (UV damage; Cosmic v3.3) and clonal hematopoiesis<sup><a href="#R42" class="usa-link" aria-describedby="R42">42</a></sup> (CH ) is compared across conditions. <strong>F</strong> Plasma mutational scores due to UV damage in patients with melanoma [stage IV (n=4) and stage III (n=8)] and controls (n=10) at baseline timepoint (prior to treatment or surgery). Plasma signatures were fit to a custom reference of signatures comprising SBS7a, SBS7a (Cosmic v3.3) and CH<sup><a href="#R42" class="usa-link" aria-describedby="R42">42</a></sup> ). <strong>G</strong>
<em>in silico</em> mixing study of metastatic melanoma samples MEL-12.B with control CTRL-06 (40 replicates per tumor fraction, 6.5x coverage per replicate). Tumor scores were estimated by fitting the sample’s trinucleotide frequencies to that of signatures SBS7a, SBS7b (Cosmic v3.3) and CH<sup><a href="#R42" class="usa-link" aria-describedby="R42">42</a></sup>. Areas under the receiving operating characteristic curve (AUCs) were measured by comparing replicates of a given tumor fraction to TF=0 replicates. For boxplots in (<strong>A</strong>) and (<strong>E</strong>), the lower and upper ends of boxes represent the 25<sup>th</sup> and 75<sup>th</sup> percentiles of the data, respectively, and the horizontal lines represent the median. The whiskers represent at most 1.5 times the IQR. P-values were calculated using a two-sided Wilcoxon test. Scans from <a href="#F2" class="usa-link">Figure 2C</a> are adapted from Widman <em>et al</em>., 2024.</p></figcaption></figure><p id="P23">To further analyze duplex error correction for variant calling in ccfDNA, we assessed the variant allele frequency (VAF) distribution in non-cancer control ccfDNA (in humans). While UMI-agnostic WGS showed only 78.8%±2.0% of base substitutions identified at the expected VAF range for germline events (<a href="#S9" class="usa-link">Methods</a>; <a href="#F8" class="usa-link">Extended Data Figure 5</a>), this increased to 97.6%±0.5% after duplex error correction. In high-burden metastatic melanoma ccfDNA samples, duplex error correction and removal of germline mutations allowed for the detection of somatic mutation VAF modes (<a href="#SD1" class="usa-link">Supplementary Table 6</a>) consistent with tumor fraction estimates using ichorCNA (<a href="#F2" class="usa-link">Figure 2B</a>,<a href="#F2" class="usa-link">C</a>), rendering this approach feasible for ctDNA mutation detection in clinical samples without a matched tumor (non tumor-informed).</p></section><section id="S6"><h3 class="pmc_sec_title">Non tumor-informed monitoring of low burden melanoma</h3>
<p id="P24">Current methods for targeted detection of ctDNA include <em>de novo</em> detection of somatic mutations (non tumor-informed) from “off the shelf” sequencing panels that target recurrent mutations of a given cancer type, or <em>tumor-informed</em> deep targeted sequencing, where a tumor is sequenced <em>a priori</em>, and a personalized panel is designed to target cancer-derived mutations. However, these methods have inherent limitations. For example, panel sensitivity is limited by the scarcity of ccfDNA in plasma (only up to 1,000–10,000 genomic equivalents in a mL of plasma<sup><a href="#R23" class="usa-link" aria-describedby="R23">23</a></sup>), incomplete driver detection (~49% driver mutation detected in stage IV patients with non-small cell lung cancer<sup><a href="#R37" class="usa-link" aria-describedby="R37">37</a></sup>) and the inability to distinguish cancer-derived mutations from those arising from alternative biological processes such as clonal hematopoiesis (CH)<sup><a href="#R38" class="usa-link" aria-describedby="R38">38</a>,<a href="#R39" class="usa-link" aria-describedby="R39">39</a></sup>. Likewise, tumor-informed targeted approaches are limited by ccfDNA scarcity, with intra-tumoral heterogeneity and primary tumor sequencing failure further restricting the targeted approach by impacting the accuracy of the mutation landscape used to build the panels<sup><a href="#R33" class="usa-link" aria-describedby="R33">33</a>,<a href="#R34" class="usa-link" aria-describedby="R34">34</a>,<a href="#R40" class="usa-link" aria-describedby="R40">40</a></sup>. Furthermore, the reliance of tumor-informed methods on only mutations from primary tumor results in a lack sensitivity for detecting phylogenetically distant metastasis. For phylogenetically distant metastasis, tumor-informed approaches will offer reduced benefit, while a tumor-free approach considers any cancer mutation, and thus may overcome this challenge.</p>
<p id="P25">WGS-based non tumor-informed mutation detection does not sensitively detect cancer hotspot mutations due to relatively lower coverage per genomic position. Thus, SNV-based WGS methods of ctDNA detection must rely on genome-wide mutational integration, where identifying multiple mutations improves detection power<sup><a href="#R14" class="usa-link" aria-describedby="R14">14</a>,<a href="#R24" class="usa-link" aria-describedby="R24">24</a></sup>. We hypothesized that signatures of somatic mutation accumulation<sup><a href="#R41" class="usa-link" aria-describedby="R41">41</a>–<a href="#R43" class="usa-link" aria-describedby="R43">43</a></sup> could provide a framework for genome-wide mutational integration for non tumor-informed WGS ctDNA detection.</p>
<p id="P26">Specifically, we reasoned that genome-wide mutations could be integrated and summarized as a weighted sum of single-base substitution (SBS) reference mutational signatures originating from the tumor in cancer patients and CH<sup><a href="#R42" class="usa-link" aria-describedby="R42">42</a></sup>. We explored the trinucleotide contexts of ccfDNA variants through mutation signature analysis at denoising levels (UMI-agnostic, UMI single-stranded and duplex), to investigate the potential mutation etiologies. Cosine similarities (measures similarities between two mutational signatures<sup><a href="#R21" class="usa-link" aria-describedby="R21">21</a>,<a href="#R43" class="usa-link" aria-describedby="R43">43</a></sup>) between the UV-associated SBS7a and SBS7b signature (COSMIC<sup><a href="#R44" class="usa-link" aria-describedby="R44">44</a></sup> v3) and high-burden samples (MEL-12A-E, stage IVB melanoma) were highest after duplex correction (mean cosine similarities to SBS7a of 0.21±0.011 (range 0.19–0.22), 0.35±0.051 (0.29–0.43), and 0.94±0.023 (0.89–0.96) between UMI-agnostic denoising, single-strand and duplex corrected datasets, respectively; <a href="#F2" class="usa-link">Figure 2D</a>,<a href="#F2" class="usa-link">E</a>; <a href="#SD2" class="usa-link">Supplementary Figure 6</a>). We found similar improvements when measuring cosine similarities between the CH signature and cancer-free controls, highlighting the importance of duplex correction for accurate signature analysis, and demonstrating that CH is an abundant source of mutations in ccfDNA WGS (<a href="#F2" class="usa-link">Figure 2D</a>,<a href="#F2" class="usa-link">E</a>, <a href="#SD2" class="usa-link">Supplementary Figure 7</a>, <a href="#F9" class="usa-link">Extended Data Figure 6A</a>).</p>
<p id="P27">As <em>de novo</em> mutation identification in error corrected ccfDNA WGS delivers profiles matching SBS7a, SBS7b and CH signatures for identifying melanoma and age-associated circulating DNA fragments, respectively, we developed a tumor-agnostic approach for ctDNA detection based on mutational patterns. First, we tabulated the trinucleotide frequencies of plasma ccfDNA mutations, fitted to reference mutational signatures (SBS7a, SBS7b and CH) using a non-negative likelihood model<sup><a href="#R45" class="usa-link" aria-describedby="R45">45</a></sup>. We obtain relative contributions of the reference mutational signatures, and estimate a tumor score by taking the weight of the tumor-associated SBS7a and SBS7b signatures and multiplying by the number of variants per duplex genomic equivalents sequenced (<a href="#SD1" class="usa-link">Supplementary Table 6</a>).</p>
<p id="P28">We applied our signature-based ctDNA detection platform for pre-operative ctDNA detection (tumor-agnostic ctDNA detection), sequencing plasma samples from eight patients with resectable locoregional stage III melanoma (without tumor or normal DNA) and ten cancer-free controls. In our cohort, tumor fractions measured by ichorCNA were undetectable in samples from patients with stage III disease, suggesting tumor fractions below 3% (ichorCNA<sup><a href="#R30" class="usa-link" aria-describedby="R30">30</a></sup> limit of detection; <a href="#F10" class="usa-link">Extended Data Figure 7</a>). Tumor scores were separable between control, stage III pre-operative melanoma and pre-treatment stage IV melanoma samples (2.4±7.6, 69.9±65.72 and 340.7±238.6, respectively, <a href="#F2" class="usa-link">Figure 2F</a>; trinucleotide frequencies in <a href="#SD2" class="usa-link">Supplementary Figure 8</a>). When applying the same signature-based methodology for single-read variant calling in single-strand correction and UMI-agnostic correction, we obtain 0 detections in our melanoma cohort (<a href="#F9" class="usa-link">Extended Data Figure 6B</a>), owing to the high number of pre-analytical errors in single-strand corrected and UMI-agnostic datasets. Previous studies showed that undetectable ctDNA using targeted panels is associated with favorable prognosis in stage III melanomas<sup><a href="#R46" class="usa-link" aria-describedby="R46">46</a>,<a href="#R47" class="usa-link" aria-describedby="R47">47</a></sup>. However, patients with undetectable ctDNA often still experience disease recurrence, highlighting the need for more sensitive tools for improved stratification. Here, tumor scores for ctDNA detection showed strong separation between cancer-free controls and melanoma patients (<a href="#F2" class="usa-link">Figure 2F</a>), suggesting that deep error corrected sequencing can identify ctDNA in a non tumor-informed approach using mutational signatures, even when ctDNA burden in the plasma is low, such as in resectable melanoma.</p>
<p id="P29">To analytically validate our approach, we performed an <em>in silico</em> mixing study combining duplex-denoised reads from a high-burden ctDNA sample (MEL-12.B, 11% ichorCNA tumor fraction estimate) and a cancer-free control (CTRL-06) at 6.5x duplex sequencing depth at expected tumor fractions of 0, 5×10<sup>−4</sup>-10<sup>−1</sup> (<a href="#F2" class="usa-link">Figure 2G</a>). Tumor scores were detectable (AUC of 0.89) at tumor fractions greater or equal than 10<sup>−3</sup>. Signature scores dropped at tumor fractions at or below 5×10<sup>−4</sup> (AUC of 0.63), given that the number of mutations originating from ctDNA was significantly below the number of mutations arising from cancer-free ccfDNA (mean of 42 and 1,930 variants at 5×10<sup>−4</sup>, from MEL-12.B vs from cancer-free controls, respectively). Tumor scores correlated strongly with expected tumor fractions (spearman’s ρ=0.97, p-value &lt;2.2×10<sup>−16</sup>). Importantly, these results highlight that we can identify ctDNA contributions when the number of tumor-derived variants is below the number of variants originating from background biological processes such as CH (~85 cancer-derived mutations in ~1,920 background mutations, at tumor fraction of 10<sup>−3</sup>), which are expected to dominate rare variant signal in early-stage and MRD contexts.</p>
<p id="P30">We further tested the ability of duplex WGS to track therapeutic response longitudinally. Here, we performed duplex WGS on the plasma of melanoma patients (stage III and stage IV) before and after treatment (<a href="#F11" class="usa-link">Extended Data Figure 8</a>), including patients receiving immunotherapy treatment with plasma collected serially (N=5 patients, 3–5 samples per patient, collected at a pre-treatment timepoint and up to 6 months post-treatment), and individuals receiving surgery, with plasma collected before and after surgery (4 patients, 2–3 samples per patient, post-operative collection up to 6 months). ctDNA levels in patients who had partial response or who were recurrence-free decreased after treatment decreased, while the ctDNA in patients who recurred or who had progressive disease increased from their baseline timepoint (<a href="#F11" class="usa-link">Extended Data Figure 8A</a>). Specifically, patients with progressive disease or recurrence saw a mean increase of 500 melanoma variants per GE after treatment, while patients with partial response or were recurrence-free from disease exhibited an overall decrease in ctDNA of 196 melanoma variants per GE (p=0.0005 between groups,; <a href="#F11" class="usa-link">Extended Data Figure 8B</a>). While clinical studies are necessary to validate these findings, our results suggest a potential for duplex-corrected WGS in tumor-agnostic, plasma only, serial profiling of melanoma patients to assess therapeutic response.</p></section><section id="S7"><h3 class="pmc_sec_title">Duplex WGS resolves complex mutational signatures in ccfDNA</h3>
<p id="P31">Next, we sought to determine whether duplex WGS could deconvolute complex signatures arising from multiple biological processes, such as APOBEC3-derived processes in urothelial cancer. Many urothelial cancers carry mutations indicating APOBEC cytidine deaminase activity, where C&gt;T transitions can occur as a function of uracil generation by cytidine deaminase activity (SBS2), and well as C&gt;G and C&gt;A mutations that arise from polymerase errors following uracil excision (SBS13)<sup><a href="#R48" class="usa-link" aria-describedby="R48">48</a></sup>. Thus, measuring the contributions of SBS2 and SBS13 as an APOBEC-score could be potentially used to detect ctDNA in urothelial cancer patients. Moreover, in neoadjuvant-treated and unresectable or metastatic urothelial carcinomas, platinum-based chemotherapies are often used as first-line treatment, and their use can induce mutagenesis in tumor tissue, which can result in subclonal mutation propagation<sup><a href="#R49" class="usa-link" aria-describedby="R49">49</a>–<a href="#R51" class="usa-link" aria-describedby="R51">51</a></sup>. Platinum-based chemotherapies mainly affect C[C&gt;T]C and C[C&gt;T]T trinucleotides (SBS31 and SBS35)<sup><a href="#R44" class="usa-link" aria-describedby="R44">44</a></sup>, thereby adding an additional layer of complexity to a patient’s cancer mutational compendium. To extend our mutation signature analysis to urothelial cancer, we sequenced n=20 plasma samples from N=20 urothelial cancer patients (stage II-IV). Of these 20 patients, 11 had previously received neoadjuvant platinum chemotherapy. We hypothesized that this complex mutational compendium, where mutations from platinum chemotherapy and varying APOBEC mutagenesis, would benefit from a ccfDNA whole-genome approach, where distinct mutational signatures can be detected. We measured and summed the contributions of SBS2 and SBS13 as an APOBEC-score for urothelial cancer patients, finding that urothelial cancer patient ccfDNA is enriched for mutations containing APOBEC signal compared to cancer-free controls (P=0.0031, <a href="#F3" class="usa-link">Figure 3A</a>), and the number of SBS2-derived mutations correlated positively with the detection of SBS13-derived mutations (Pearson’s R=0.94, p-value=3.2×10<sup>−14</sup>).</p>
<figure class="fig xbox font-sm" id="F3"><h4 class="obj_head">Figure 3. Mutational signature analysis of cell-free DNA from urothelial cancer patients.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5a41/12077166/7e475b9e8ee9/nihms-2073968-f0003.jpg" loading="lazy" height="1014" width="1889" alt="Figure 3."></p>
<div class="p text-right font-secondary"><a href="figure/F3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P77"><strong>A</strong> Plasma mutational scores from APOBEC mutagenesis (SBS2, SBS13) in patients with urothelial cancer (stage II-IV, n=20) and cancer-free controls (n=10). Plasma signatures were fit to a custom reference of signatures comprising SBS2, SBS13, SBS31 and SBS35 (Cosmic v3.3) and clonal hematopoiesis<sup><a href="#R42" class="usa-link" aria-describedby="R42">42</a></sup>. Tumor fractions were measured in a tumor-informed manner (see <a href="#S9" class="usa-link">Methods</a>). <strong>B</strong> Trinucleotide frequency of circulating cell-free DNA (“plasma”, left) and of the tumor (right) for patient BLA-12. For the patient tumor, the three highest SBS contributions are highlighted (SBS31, SBS13, SBS2). Contributions were fit to the entire Cosmic v3.3 catalog. <strong>C</strong> Tumor-informed measurement of APOBEC-derived mutations (SBS2 and SBS13, x-axis) versus tumor-agnostic APOBEC-derived mutations (y-axis) in N=13 samples for which tumor tissue sequencing was available. The tumor-agnostic APOBEC mutations were measured as in (<strong>A</strong>). Plasma signatures were fit to a custom reference of signatures comprising SBS2, SBS13, SBS31 and SBS35 (Cosmic v3.3) and clonal hematopoiesis<sup><a href="#R42" class="usa-link" aria-describedby="R42">42</a></sup>. Tumor signatures were fit to the entire Cosmic v3.3 catalog. The shaded area represents the 95% confidence interval of the data distribution. <strong>D</strong> Plasma mutational scores from platinum therapy mutagenesis (SBS31+SBS35) in patients with urothelial cancer (stage II-IV; n=11 post, n=9 pre) and cancer-free controls (n=10). Plasma signatures were fit to a custom reference of signatures comprising SBS2, SBS13, SBS31 and SBS35 (Cosmic v3.3) and clonal hematopoiesis<sup><a href="#R42" class="usa-link" aria-describedby="R42">42</a></sup>. <strong>E</strong> Barplot of tumor-informed tumor fractions (black dots) and platinum scores (purple; as calculated in <strong>D</strong>) for samples with available tumors for sequencing. For boxplots in (<strong>A</strong>) and (<strong>D</strong>), the lower and upper ends of boxes represent the 25<sup>th</sup> and 75<sup>th</sup> percentiles of the data, respectively, and the horizontal lines represent the median. The whiskers represent at most 1.5 times the IQR. P-values were calculated using a two-sided Wilcoxon test.</p></figcaption></figure><p id="P32">In a subset of urothelial cancer patients (N=13), tumor tissue was available for sequencing (N=29 multiregion samples across 13 patients), where we could compare the mutational profiles of patient plasma to the patient-matched tumors. First, all tumors showed evidence of APOBEC mutagenesis (8–86%, <a href="#F12" class="usa-link">Extended Data Figure 9</a>, <a href="#SD1" class="usa-link">Supplementary Table 7</a>). Second, mutational profiles obtained through de novo (without using a matched tumor; <a href="#SD2" class="usa-link">Supplementary Figure 9</a>) detection of ctDNA strongly resembled those of the matching tumor (cosine similarity &gt; 0.83 at plasma tumor fractions &gt;0.005) and that de novo identification of APOBEC mutations was possible in plasma samples with tumor fractions as low as 1.1×10<sup>−4</sup>. (<a href="#F3" class="usa-link">Figure 3A</a>,<a href="#F3" class="usa-link">B</a>, <a href="#SD2" class="usa-link">Supplementary Figure 10</a>). We measured tumor-informed tumor fractions (<a href="#S9" class="usa-link">Methods</a>; <a href="#SD1" class="usa-link">Supplementary Table 8</a>) and calculated the expected contribution of APOBEC-derived mutations in the plasma by multiplying the tumor-informed tumor fraction by the fraction of tumor variants attributable to APOBEC mutational signatures. We found a strong correlation between the expected, tumor-informed APOBEC contributions in the plasma and our de novo APOBEC score (Pearson’s R=0.95, p-value=1.13×10<sup>−6</sup>, <a href="#F3" class="usa-link">Figure 3C</a>).</p>
<p id="P33">In terms of driver detection, as expected from the lower sequencing depth, duplex WGS detected only 6/59 putative driver mutations (<a href="#SD1" class="usa-link">Supplementary Table 9</a>) across the cohort of patients with matched plasma and tumor sequencing (N=29 multiregion samples across the 13 patients), preferentially in ccfDNA samples with high tumor burden (<a href="#SD1" class="usa-link">Supplementary Table 8</a>–<a href="#SD1" class="usa-link">9</a>, Pearson’s R=0.57, p-value=0.04). While these results demonstrate that our duplex WGS approach can resolve putative driver mutations, deep-targeted sequencing could offer more benefit for driver mutation detection.</p>
<p id="P34">Interestingly, we noted instances of high cosine similarities between patient plasma and matched tumors despite low burden disease (at or below 10<sup>−4</sup>; <a href="#SD2" class="usa-link">Supplementary Figure 10B</a>; for example patient BLA-18 [cosine 0.92]). While stereotypical APOBEC-derived peaks (namely T[C&gt;T]A and T[C&gt;T]T for SBS2 and T[C&gt;G]A and T[C&gt;G]T for SBS13) were not visually obvious in the patient plasma and were undetectable in N=8 patients (<a href="#F3" class="usa-link">Figure 3A</a>), we noted strong contributions of platinum-derived chemotherapy mutations (for example, C[C&gt;T]C and C[C&gt;T]T for SBS31) in patients who were treated with platinum chemotherapy. By fitting the plasma mutational signatures to a custom catalog of APOBEC and platinum chemotherapy-derived mutational signatures (SBS2/SBS13 and SBS31/SBS35, respectively), we found enrichment of platinum chemotherapy mutations only in the plasma of patients who had received treatment (<a href="#F3" class="usa-link">Figure 3D</a>). Interestingly, platinum mutation signatures were detected in the ccfDNA through de novo analysis in some individuals where no ctDNA was detected using a tumor-informed approach (<a href="#F3" class="usa-link">Figure 3E</a>; <a href="#SD2" class="usa-link">Supplementary Figure 10</a>), potentially reflecting signal originating from platinum mutagenesis in non-malignant cells. These results suggest that tumor-agnostic, de novo profiling of patient plasma may offer a more comprehensive overview of cancer patient plasma when compared to tumor-informed approaches that solely focus on the detection of a limited number of clonal mutations.</p></section></section><section id="S8"><h2 class="pmc_sec_title">DISCUSSION</h2>
<p id="P35">Radical decreases in sequencing costs open new opportunities in both clinical genomics as well as basic biology research. Here we harnessed low-cost Ultima sequencing to demonstrate the impact of deeper WGS (~100X) on tumor-informed ctDNA monitoring. Moreover, we also developed methodology to integrate duplex sequencing with the single-end Ultima sequencing, showcasing the feasibility of cost-efficient, highly accurate WGS that can be broadly applied across genomics research. This advance allowed us to apply duplex error correction to clinical plasma samples at the level of the entire genome. We leveraged these highly error-corrected data for non tumor-informed ctDNA detection, based on the similarity of mutational profiles to known cancer mutational signatures. These findings have important clinical implications, as uncoupling ctDNA detection from a tumor-informed mutation profiles radically increases the potential use of ctDNA monitoring in common clinical scenarios where tumor samples cannot be obtained (<a href="#SD2" class="usa-link">Supplementary Note</a>). Excitingly, this demonstration opens up the possibility for the use of WGS in ctDNA cancer screening. In particular, such an approach might be beneficial for specific settings where there is high genetic (e.g. BRCA mutation carriers, Lynch syndrome) or environmental (e.g. tobacco smoke exposure) cancer risk together with distinct mutational signatures. A limitation of whole-genome duplex sequencing is that fact that only a fraction of reads result in a duplex fragment, resulting in an order of magnitude less unique molecules captured compared with standard WGS. However, the ~500 fold decrease in error rates in duplex sequencing compared to bioinformatically-denoised standard WGS may offset this decrease in coverage, depending on the specific application or context. In the tumor-informed setting, where prior knowledge of the somatic mutations can lower the effective error rate of standard WGS, higher inputs are expected to afford lower limits of detection than duplex WGS. In the tumor-agnostic setting, a markedly reduced error rate allows for detection of genome-wide signature and plasma-only MRD even in lower burden disease compared with standard WGS which only allowed detection in high-burden metastatic disease. Finally, we envisage that methods developed herein can provide a general, cost-efficient approach for duplex WGS that can be leveraged across genomics research that requires highly accurate detection of rare variants in somatic tissues.</p></section><section id="S9"><h2 class="pmc_sec_title">METHODS</h2>
<section id="S10"><h3 class="pmc_sec_title">Simulation analysis</h3>
<p id="P36">Simulations for ctDNA detection scores (<a href="#F1" class="usa-link">Figure 1A</a>) were performed assuming a tumor-mutational compendium of 10,000 SNVs with different error rates (10<sup>−3</sup>, 10<sup>−4</sup> and 10<sup>−5</sup>), coverages (1, 10 and 100) and tumor fractions (0, 10<sup>−6</sup>, 10<sup>−5</sup>). For each of the 10,000 SNV mutations, coverage was simulated using a Poisson distribution. Each simulated sequenced base pair was classified as either ctDNA or ccfDNA according to the tumor fraction, and errors misclassified as ctDNA were determined according to the error rate. Estimated tumor fractions were calculated by summing the ctDNA molecules and the errors and dividing by the total base pairs simulated. Z scores were calculated as:
</p>
<table class="disp-formula p" id="FD1"><tr><td class="formula"><math id="M1" display="block"><mi>Z</mi><mspace width="0.5em"></mspace><mi>s</mi><mi>c</mi><mi>o</mi><mi>r</mi><mi>e</mi><mo>=</mo><mfrac><mrow><mi>T</mi><mi>F</mi><mo>-</mo><mi>m</mi><mi>e</mi><mi>a</mi><mi>n</mi><mo>(</mo><msub><mrow><mi>T</mi><mi>F</mi></mrow><mrow><mi>T</mi><mi>F</mi><mo>=</mo><mn>0</mn></mrow></msub><mo>)</mo></mrow><mrow><msub><mrow><mi>s</mi><mi>t</mi><mi>d</mi></mrow><mrow><mi>T</mi><mi>F</mi><mo>=</mo><mn>0</mn></mrow></msub></mrow></mfrac></math></td></tr></table></section><section id="S11"><h3 class="pmc_sec_title">Pan-Cancer Analysis of Whole Genomes (PCAWG) datasets</h3>
<p id="P37">We obtained somatic trinucleotide counts of n=2,780 tumors from the PCAWG database<sup><a href="#R43" class="usa-link" aria-describedby="R43">43</a></sup>. Total somatic mutation counts were used to generate <a href="#F6" class="usa-link">Extended Data Figure 3</a>. To simulate and compare the performance of targeted panels and whole-genome sequencing to detect different cancer types, we defined ctDNA detection opportunities to be a function of panel size, genomic equivalents available to be sequenced in a sample, and sequencing coverage. Specifically:
</p>
<table class="disp-formula p" id="FD2"><tr><td class="formula"><math id="M2" display="block"><mi>c</mi><mi>t</mi><mi>D</mi><mi>N</mi><mi>A</mi><mspace width="0.5em"></mspace><mi>d</mi><mi>e</mi><mi>t</mi><mi>e</mi><mi>c</mi><mi>t</mi><mi>i</mi><mi>o</mi><mi>n</mi><mspace width="0.5em"></mspace><mi>o</mi><mi>p</mi><mi>p</mi><mi>o</mi><mi>r</mi><mi>t</mi><mi>u</mi><mi>n</mi><mi>i</mi><mi>t</mi><mi>i</mi><mi>e</mi><mi>s</mi><mo>=</mo><mo>(</mo><mi>t</mi><mi>a</mi><mi>r</mi><mi>g</mi><mi>e</mi><mi>t</mi><mi>e</mi><mi>d</mi><mspace width="0.5em"></mspace><mi>S</mi><mi>N</mi><mi>V</mi><mi>s</mi><mspace width="0.5em"></mspace><mi>x</mi><mspace width="0.5em"></mspace><mi>G</mi><mi>E</mi><mi>s</mi><mspace width="0.5em"></mspace><mo>|</mo><mspace width="0.5em"></mspace><mi>t</mi><mi>a</mi><mi>r</mi><mi>g</mi><mi>e</mi><mi>t</mi><mi>e</mi><mi>d</mi><mspace width="0.5em"></mspace><mi>S</mi><mi>N</mi><mi>V</mi><mi>s</mi><mspace width="0.5em"></mspace><mi>x</mi><mspace width="0.5em"></mspace><mi>c</mi><mi>o</mi><mi>v</mi><mi>e</mi><mi>r</mi><mi>a</mi><mi>g</mi><mi>e</mi><mo>)</mo></math></td></tr></table>
<p>
For whole-genome sequencing simulations, the “panel size” was set to the median number of somatic mutations of a given cancer type. We assumed a 20-ng cell-free DNA input (corresponding to ~6,700 GEs, assuming the mass of a genome to be 3 pg).</p></section><section id="S12"><h3 class="pmc_sec_title">Human subjects sample processing</h3>
<p id="P38">Blood and tissue samples were obtained from patients after obtaining informed consent and following protocols approved by institutional review boards and in accordance with the Declaration of Helsinki protocol. Samples were obtained from either NewYork-Presbyterian/Weill Cornell Medical Center (Institutional Review Board (IRB) numbers 0201005295 (Tumor Biobanking), 1008011210 (GU Tumor Biobanking), 1011011386 (Urothelial Cancer Sequencing), 100701157 (Genomic and Transcriptomic Profiling), 1305013903 (Precision Medicine), 1708018519 (Cardiac Surgery Biobank), 2014–0024 (approved by the Institutional Animal Care and Use Committee at Weill Cornell Medicine), 1610017682 (Circulating tumor DNA for early detection and management of Non Small Cell Lung Cancer), Memorial Sloan Kettering Cancer Center (IRB number 12–245 (Genomic Profiling in Cancer Patients), Massachusetts General Hospital (IRB number 11–181 (Collection of Tissue and Blood Specimens and Clinical Data from Patients with Melanoma and Other Cutaneous Malignancies)), or the Royal Marsden NHS Foundation Trust in the United Kingdom (<a href="#SD1" class="usa-link">Supplementary Table 1</a>). Tumor, normal and plasma samples from the Royal Marsden NHS Foundation Trust were obtained under an ethically approved protocol (Melanoma TRACERx, Research Ethics Committee Reference 11/LO/0003). Cancer diagnosis was established according to World Health Organization criteria and confirmed in all cases by an independent pathology review. Patients did not receive any compensation.</p></section><section id="S13"><h3 class="pmc_sec_title">Tumor and germline DNA extraction, library preparation and sequencing</h3>
<p id="P39">For melanoma and lung cancer samples, genomic DNA was extracted using the QiAamp DNA Mini Kit (Qiagen, cat#56304) and the QIAamp DNA blood Kit (Qiagen, cat#51104) for tissue and blood samples, respectively, and sheared to 450bp (Covaris cat#500569). Sequencing libraries were prepared on 1μg of DNA using the TruSeq DNA PCR-Free Library Preparation Kit (Illumina, cat#20015963), with one additional bead cleanup performed after end-repair and after adapter ligation. DNA was quantified using a Qubit 3.0 fluorometer and length analysis was performed using an Agilent Bioanalyzer or High Sensitivity Fragment Analyzer. 2×150bp paired-end sequencing was performed on either a HiSeq X or NovaSeq v1.0 Illumina machine. Urothelial tumor/normal sequencing data were obtained from Nguyen <em>et al</em><sup><a href="#R51" class="usa-link" aria-describedby="R51">51</a></sup>.</p></section><section id="S14"><h3 class="pmc_sec_title">Cell-free DNA extraction</h3>
<p id="P40">Cell-free DNA was extracted from plasma using the Magbind ccfDNA extraction kit (Omega Biotek cat#M3298). Manufacturer recommendations for extraction were followed, but elution volume was increased to 35 μL, and elution time was increased to 20 minutes on a thermomixer at 1,600 rpm (room temperature). Extracted ccfDNA was quantified using a Qubit 3.0 fluorometer and length analysis was performed using an Agilent Bioanalyzer or High Sensitivity Fragment Analyzer.</p></section><section id="S15"><h3 class="pmc_sec_title">Cell-free DNA library preparation</h3>
<section id="S16"><h4 class="pmc_sec_title">Whole genome library preparation (without duplex)</h4>
<p id="P41">Next generation sequencing libraries were generated using a double-stranded preparation kit (Kapa Hyper Prep Kit, Roche, cat#KK8502). Full-length adapters (IDT TruSeQ UDI plate, Illumina, cat#20023784) were used for adapter ligation. Six PCR cycles were carried out when input DNA was above 5ng, and 8 cycles were performed when the input was below 5ng. Libraries were quantified using Qubit 3.0 fluorometer and length analysis was performed using an Agilent Bioanalyzer or High Sensitivity Fragment Analyzer. Illumina sequencing libraries were sequenced on a HiSeq X or NovaSeq1.0 using 2×150 paired-end sequencing. Library input amounts can be found in <a href="#SD1" class="usa-link">Supplementary Table 3</a>.</p></section><section id="S17"><h4 class="pmc_sec_title">Whole genome library preparation (with duplex)</h4>
<section id="S18"><h5 class="pmc_sec_title">Version 1:</h5>
<p id="P42">ccfDNA libraries were generated in a similar fashion as described above, although the full-length adapters were replaced with stubby Y-adapters containing 3 UMI bases (IDT Duplex Seq adapters, cat#1080799) and sample indexing was carried out during PCR amplification. To enhance duplicate recovery in human samples, a maximum of 10ng was used as input, and 4ng of prepared libraries was subjected to 6 additional PCR cycles prior to Ultima library conversion. Mouse PDX samples did not undergo additional PCR cycles prior to Ultima library conversion.</p></section><section id="S19"><h5 class="pmc_sec_title">Version 2:</h5>
<p id="P43">Version 2 of the duplex library preparation differs from version 1 as follows: <em>i)</em> 1 ng of ccfDNA was used as input; <em>ii)</em> 8 cycles of PCR are performed; <em>iii)</em> the step of taking 4 ng of prepared libraries and performing 6 additional PCR cycles is omitted. Library input amounts can be found in <a href="#SD1" class="usa-link">Supplementary Table 3</a>.</p></section></section><section id="S20"><h4 class="pmc_sec_title">Ultima sequencing</h4>
<p id="P44">Illumina sequencing libraries underwent Ultima library conversion. Briefly, Illumina libraries were converted to Ultima libraries by PCR, using primers matching Illumina read 1 and read 2 sequences and containing Ultima-specific barcodes (R1 conversion adapter: TCC ATC TCA TCC CTG CGT GTC TCC GAC TGC ACA ATG TGT GCT AGA TCT ACA CGA CGC TCT TCC GAT CT, R2 conversion adapter: CTG TGT GCC TTG GCA GTC TCA GCT CAG ACG TGT GCT CTT CCG ATC T). Samples were then pooled and sequenced on an Ultima sequencer prototype to a target depth of 120x. One sample achieved lower effective coverage (MEL-03.A, 36x), likely due to a sample pooling error.</p></section></section><section id="S21"><h3 class="pmc_sec_title">Whole genome sequencing (without duplex) adapter trimming and alignment</h3>
<p id="P45">Illumina fastQ reads were adapter trimmed using skewer<sup><a href="#R52" class="usa-link" aria-describedby="R52">52</a></sup> (version 0.2.2). Trimmed reads were then aligned to the human genome (version hg38) using bwa mem<sup><a href="#R53" class="usa-link" aria-describedby="R53">53</a></sup>. Duplicate reads were marked in a UMI-unaware fashion using novosort<sup><a href="#R54" class="usa-link" aria-describedby="R54">54</a></sup>. Depth of coverage was estimated using mosdepth<sup><a href="#R55" class="usa-link" aria-describedby="R55">55</a></sup> (version 0.2.9) and duplicate reads were considered. For Ultima reads, adapter trimming was carried out to remove the Illumina conversion adapters. Cutadapt<sup><a href="#R56" class="usa-link" aria-describedby="R56">56</a></sup> (version 2.10; cutadapt --mask-adapter -a CTACACGACGCTCTTCCGATCT;max_error_rate=0.15;min_overlap=10;required... AGATCGGAAGAGCACACGTCTGCTG;max_error_rate=0.2;min_overlap=6) was used to mask adapter sequences, and adapter trimming was then carried out using GATK<sup><a href="#R57" class="usa-link" aria-describedby="R57">57</a></sup> (private Ultima fork, since merged to the latest 4.3.0.0 GATK release) (ClipReads function). Alignment was performed using bwa mem<sup><a href="#R53" class="usa-link" aria-describedby="R53">53</a></sup> (version 0.7.15-r1140) and coverage was estimated using mosdepth counting duplicate reads. Alignment statistics can be found in <a href="#SD1" class="usa-link">Supplementary Table 3</a>. We note that the coverages were calculated as the total number of bases mapping to the human genome divided by the size of the genome. In cell-free DNA, coverages have been shown to be inversely correlated with DNA accessibility, as the open regions of the genome are more susceptible to degradation in the blood. For example, highly expressed genes can see a loss of coverage of nearly 50%. Thus, somatic variants in open chromatin regions may be more difficult to detect via the plasma.</p></section><section id="S22"><h3 class="pmc_sec_title">Copy-number based tumor fraction estimation</h3>
<p id="P46">Genome-wide coverage was calculated over a 1Mbp window and normalized for mappability and GC content biases (using hmmcopy<sup><a href="#R58" class="usa-link" aria-describedby="R58">58</a></sup> version 0.99). Tumor fractions were estimated using ichorCNA<sup><a href="#R30" class="usa-link" aria-describedby="R30">30</a></sup> (version 0.3.2) after correcting for library and sequencing artifacts via a panel of normals (PoN) from cancer-free controls (CTRL-01 to CTRL-05). A separate PoN was created for Illumina and Ultima sequenced samples using libraries sequenced on the respective machines. For plotting purposes (<a href="#F1" class="usa-link">Figure 1D</a>), corrected log2 read counts outputted by ichorCNA were used. Bins marked by ichorCNA as copy gains, amplifications and high-level amplifications were marked and colored as chromosome gains (pink). Bins marked as homozygous deletion states and hemizygous deletions were marked and colored as chromosome losses (blue). Copy neutral regions were marked as neutral (black). Bins with corrected log2 read counts between −0.05 and 0.05 were marked as neutral (black) as well. Given that ichorCNA provides multiple solutions ordered by log likelihood, the CNV-based tumor fraction reported in the manuscript was manually selected amongst all solutions according to ichorCNA guidelines<sup><a href="#R30" class="usa-link" aria-describedby="R30">30</a></sup>. Deviation from the most likely solution is justified in <a href="#SD1" class="usa-link">Supplementary Table 10</a>.</p></section><section id="S23"><h3 class="pmc_sec_title">Whole genome sequencing (without duplex) SNV-based tumor fraction estimation</h3>
<p id="P47">SNV-based tumor fraction estimation was carried out by counting cell-free DNA reads with matching tumor-specific somatic mutations. To limit the effect of problematic regions of the genome, a platform-specific blacklist was built. For Illumina sequencing, regions identified in the ENCODE blacklist<sup><a href="#R59" class="usa-link" aria-describedby="R59">59</a></sup>, centromeres<sup><a href="#R60" class="usa-link" aria-describedby="R60">60</a></sup>, simple repeat regions<sup><a href="#R60" class="usa-link" aria-describedby="R60">60</a></sup> and positions with high mutation rates (GNOMAD<sup><a href="#R61" class="usa-link" aria-describedby="R61">61</a></sup>, AF&gt;0.001) were not considered. For Ultima sequencing, Ultima-specific low-confidence regions composed of homopolymers, AT-rich regions, tandem repeats, and regions with poor mappability and high coverage variability were additionally excluded (<a href="#F6" class="usa-link">Extended Data Figure 3</a>). To limit the effect of sequencing errors, custom scripts were used for platform-specific denoising (<a href="#SD2" class="usa-link">Supplementary Note</a>).</p></section><section id="S24"><h3 class="pmc_sec_title">Whole genome sequencing (without duplex) tumor-informed error rate estimation</h3>
<p id="P48">Tumor-informed error rates were computed by intersecting a given cancer-free control’s cell-free DNA sequencing reads with a cancer patient’s somatic mutations. Reads were then denoised in a platform-specific manner as described above (except for data in <a href="#SD2" class="usa-link">Supplementary Figure 2</a>, which compares denoised data to non-denoised data). An error rate was defined as the total number of single-occurring variants divided by the total number of denoised bases overlapping the somatic mutations.</p></section><section id="S25"><h3 class="pmc_sec_title">PCR duplicates analysis for indel and SNV error rates</h3>
<p id="P49">To measure indel and SNV error rates, alignment files split by chromosome. For each autosomal chromosome, up to 1,500,000 unique DNA molecules were collected and scanned for PCR duplicates (here, defined as two sequencing reads mapping to the same strand of the reference genome, each having a mapping quality of 60, and both containing the same unique molecular identifier. For each unique molecule, 2 PCR duplicates were randomly selected. The PCR duplicates were then aligned to each other and the reference genome (obtained via the alignment file) using global pairwise sequencing alignment (using the pairwise2 module from biopython version 1.79 in python version 3.6). Alignment points and penalties were set to 1, −1.5, −1 and −1 for identical bases, non-identical bases, opening gaps and extending gaps, respectively. Gaps in the alignment were considered to be indels, and differing bases were considered to be SNV errors. For gaps, the size differences between homopolymers of the PCR duplicates were tabulated to create <a href="#F4" class="usa-link">Extended Data Figure 1</a>. For differing bases, the reference trinucleotide, reference base and the PCR duplicate read bases were collected for <a href="#F5" class="usa-link">Extended Data Figure 2</a>. For the SNV analysis, the reference base was assumed to be correct when PCR duplicates differed.</p></section><section id="S26"><h3 class="pmc_sec_title">
<em>In silico</em> mixing study for analytical lower limit of detection estimation (standard WGS)</h3>
<p id="P50">We created <em>in silico</em> mixes at various tumor fractions by computationally combining aligned reads from a high tumor-burden plasma sample (MEL-07, estimated tumor fraction of 7%) with aligned reads from a cancer-free control (CTRL-05). Reads were mixed to create 20x (Illumina and Ultima), 40x (Ultima) and 70x (Ultima) bam files harboring, 10<sup>−6</sup>, 5×10<sup>−5</sup>, 10<sup>−5</sup> 10<sup>−4</sup>, 10<sup>−3</sup> or 10<sup>−2</sup> tumor fraction. The coverage of the high burden sample necessary to obtain a given expected tumor fraction at an expected coverage was defined as:
</p>
<table class="disp-formula p" id="FD3"><tr><td class="formula"><math id="M3" display="block"><msub><mrow><mi>C</mi><mi>o</mi><mi>v</mi><mi>e</mi><mi>r</mi><mi>a</mi><mi>g</mi><mi>e</mi><mspace width="0.5em"></mspace><mi>n</mi><mi>e</mi><mi>e</mi><mi>d</mi><mi>e</mi><mi>d</mi></mrow><mrow><mi>h</mi><mi>i</mi><mi>g</mi><mi>h</mi><mspace width="0.5em"></mspace><mi>b</mi><mi>u</mi><mi>r</mi><mi>d</mi><mi>e</mi><mi>n</mi><mspace width="0.5em"></mspace><mi>s</mi><mi>a</mi><mi>m</mi><mi>p</mi><mi>l</mi><mi>e</mi></mrow></msub><mo>=</mo><mfrac><mrow><mi>e</mi><mi>x</mi><mi>p</mi><mi>e</mi><mi>c</mi><mi>t</mi><mi>e</mi><mi>d</mi><mspace width="0.5em"></mspace><mi>t</mi><mi>u</mi><mi>m</mi><mi>o</mi><mi>r</mi><mspace width="0.5em"></mspace><mi>f</mi><mi>r</mi><mi>a</mi><mi>c</mi><mi>t</mi><mi>i</mi><mi>o</mi><mi>n</mi></mrow><mrow><mi>h</mi><mi>i</mi><mi>g</mi><mi>h</mi><mspace width="0.5em"></mspace><mi>b</mi><mi>u</mi><mi>r</mi><mi>d</mi><mi>e</mi><mi>n</mi><mspace width="0.5em"></mspace><mi>t</mi><mi>u</mi><mi>m</mi><mi>o</mi><mi>r</mi><mspace width="0.5em"></mspace><mi>f</mi><mi>r</mi><mi>a</mi><mi>c</mi><mi>t</mi><mi>i</mi><mi>o</mi><mi>n</mi></mrow></mfrac><mi>x</mi><mspace width="0.5em"></mspace><mi>e</mi><mi>x</mi><mi>p</mi><mi>e</mi><mi>c</mi><mi>t</mi><mi>e</mi><mi>d</mi><mspace width="0.5em"></mspace><mi>c</mi><mi>o</mi><mi>v</mi><mi>e</mi><mi>r</mi><mi>a</mi><mi>g</mi><mi>e</mi></math></td></tr></table>
<p>
The coverage of the cancer-free control was subsequently defined as:
</p>
<table class="disp-formula p" id="FD4"><tr><td class="formula"><math id="M4" display="block"><msub><mrow><mi>C</mi><mi>o</mi><mi>v</mi><mi>e</mi><mi>r</mi><mi>a</mi><mi>g</mi><mi>e</mi><mspace width="0.5em"></mspace><mi>n</mi><mi>e</mi><mi>e</mi><mi>d</mi><mi>e</mi><mi>d</mi></mrow><mrow><mi>c</mi><mi>a</mi><mi>n</mi><mi>c</mi><mi>e</mi><mi>r</mi><mo>-</mo><mi>f</mi><mi>r</mi><mi>e</mi><mi>e</mi><mspace width="0.5em"></mspace><mi>c</mi><mi>o</mi><mi>n</mi><mi>t</mi><mi>r</mi><mi>o</mi><mi>l</mi></mrow></msub><mo>=</mo><mi>e</mi><mi>x</mi><mi>p</mi><mi>e</mi><mi>c</mi><mi>t</mi><mi>e</mi><mi>d</mi><mspace width="0.5em"></mspace><mi>c</mi><mi>o</mi><mi>v</mi><mi>e</mi><mi>r</mi><mi>a</mi><mi>g</mi><mi>e</mi><mo>-</mo><msub><mrow><mi>c</mi><mi>o</mi><mi>v</mi><mi>e</mi><mi>r</mi><mi>a</mi><mi>g</mi><mi>e</mi><mspace width="0.5em"></mspace><mi>n</mi><mi>e</mi><mi>e</mi><mi>d</mi><mi>e</mi><mi>d</mi></mrow><mrow><mi>h</mi><mi>i</mi><mi>g</mi><mi>h</mi><mspace width="0.5em"></mspace><mi>b</mi><mi>u</mi><mi>r</mi><mi>d</mi><mi>e</mi><mi>n</mi><mspace width="0.5em"></mspace><mi>s</mi><mi>a</mi><mi>m</mi><mi>p</mi><mi>l</mi><mi>e</mi></mrow></msub></math></td></tr></table>
<p>
An <em>in silico</em> mixed replicate was then obtained by randomly downsampling MEL-07 and CTRL-05 to obtain the respective coverages (using samtools view -s). Downsampled files were merged and denoised as described above (<em>Whole genome sequencing (without duplex) SNV-based tumor fraction estimation</em>). Tumor fractions were estimated with platform-specific denoising. Z-scores were calculated as:
</p>
<table class="disp-formula p" id="FD5"><tr><td class="formula"><math id="M5" display="block"><mi>Z</mi><mspace width="0.5em"></mspace><mi>s</mi><mi>c</mi><mi>o</mi><mi>r</mi><mi>e</mi><mo>=</mo><mfrac><mrow><mi>T</mi><mi>F</mi><mo>-</mo><mi>m</mi><mi>e</mi><mi>a</mi><mi>n</mi><mo>(</mo><msub><mrow><mi>T</mi><mi>F</mi></mrow><mrow><mi>T</mi><mi>F</mi><mo>=</mo><mn>0</mn></mrow></msub><mo>)</mo></mrow><mrow><msub><mrow><mi>s</mi><mi>t</mi><mi>d</mi></mrow><mrow><mi>T</mi><mi>F</mi><mo>=</mo><mn>0</mn></mrow></msub></mrow></mfrac></math></td></tr></table>
<p>
Specifically, the Z-score of a given replicate was calculated against it’s the mean and standard deviation of coverage and platform matched cancer-free control replicates.</p></section><section id="S27"><h3 class="pmc_sec_title">UMI WGS data processing</h3>
<p id="P51">FastQ reads were adapter and UMI trimmed using cutadapt67 (version 2.10). Trimmed reads were then aligned to the human genome (version hg38) using bwa mem64 (with parameters -K 100000000 -p -v3 -t 16 -Y). Trimmed UMI’s were added to the alignment files as an additional RX tag. Unique molecules were identified by either in single-stranded mode (i.e., collecting PCR duplicates as unique molecules) or in duplex mode (i.e., collecting PCR duplicates and UMI-concordant Watson &amp; Crick strands as unique molecules) using the fgbio suite of tools (version 2.0). Because duplex correction via fgbio requires paired-end reads, we created a synthetic R2 read directly from single-end bam alignment files. R2 reads were built using the same mapping information (such as CIGAR string and mapping quality) and read information (such as sequence and qualities) as the R1 read. The subsequent paired-end alignment file was grouped by UMI (fgbio GroupReadsByUMI with parameters -m0 -s paired -e 1).</p>
<section id="S28"><h4 class="pmc_sec_title">Single-stranded and duplex consensus reads and UMI-agnostic denoising</h4>
<p id="P52">Single-stranded consensus reads were created by following the fgbio workflow. Molecular consensuses were generated using the CallMolecularConsensusReads command (with options -M1; --consensus-call-overlapping-bases false. Duplex analysis included the --paired option). Next, FilterConsensusReads was applied (with options -M 2; -N0; -E 1; -e 1; -n 1). Reads passing the filter were remapped to the human genome (version hg38) for analysis (or a concatenated human and mouse genome for mouse PDX samples (hg38 and mm39, respectively). Single-strand and/or duplex metrics (such as consensus read depth, consensus error rate, number of Ns on the consensus molecule, number of reads with matching UMIs) and mapping information were integrated as additional read tags to the original single-end alignment file. Variant frequencies in the original alignment files (without denoising) were calculated using lofreq (version 2.1.3a, with all filtering modes disabled). The original single-end bam, with the additional single-strand or duplex tags, was processed through the FlowFeatureMapper tool, (described above) which allows for the processing of the additional UMI tags, to obtain putative variants. For single-stranded consensus reads, only consensus variants obtained by the fgbio pipeline were considered. For duplex variants, a decision-tree based classifier was utilized to create duplex-consensus variants (<a href="#SD2" class="usa-link">Supplementary Note</a>). For mouse PDX samples, the following filters were then applied: <em>i)</em> all reads contributing to a consensus read must have a mapping quality of 60; <em>ii)</em> all reads contributing to a consensus must have the 5 flanking bases of a variant match the reference genome; <em>iii)</em> the variant must not be within 25 bases of either end of the cell-free DNA molecule. Finally, UMI-agnostic denoising was performed by filtering by: <em>i)</em> variant position in read (the variant cannot be within 25bp of an end of the read); <em>ii)</em> template length (must be lower than 200bp); <em>iii)</em> mapping quality (cannot be below 60); <em>iv)</em> edit distance (must be below 4) and <em>v)</em> total variants on the read (must be below 11). Duplicate reads were not considered. The same filters were applied for human samples, with the addition of an edit distance filter to further decrease errors (at least one duplex strand (or read for single strand consensuses) must have an edit distance below 2).</p></section></section><section id="S29"><h3 class="pmc_sec_title">Residual SNV rate estimation</h3>
<p id="P53">The residual SNV rate was defined as:
</p>
<table class="disp-formula p" id="FD6"><tr><td class="formula"><math id="M6" display="block"><mi>R</mi><mi>e</mi><mi>s</mi><mi>i</mi><mi>d</mi><mi>u</mi><mi>a</mi><mi>l</mi><mspace width="0.5em"></mspace><mi>S</mi><mi>N</mi><mi>V</mi><mspace width="0.5em"></mspace><mi>r</mi><mi>a</mi><mi>t</mi><mi>e</mi><mo>=</mo><mspace width="0.5em"></mspace><mfrac><mrow><mi>N</mi><mi>u</mi><mi>m</mi><mi>b</mi><mi>e</mi><mi>r</mi><mspace width="0.5em"></mspace><mi>o</mi><mi>f</mi><mspace width="0.5em"></mspace><mi>s</mi><mi>i</mi><mi>n</mi><mi>g</mi><mi>l</mi><mi>e</mi><mspace width="0.5em"></mspace><mi>o</mi><mi>c</mi><mi>c</mi><mi>u</mi><mi>r</mi><mi>i</mi><mi>n</mi><mi>g</mi><mspace width="0.5em"></mspace><mi>m</mi><mi>u</mi><mi>t</mi><mi>a</mi><mi>t</mi><mi>i</mi><mi>o</mi><mi>n</mi><mi>s</mi></mrow><mrow><mi>I</mi><mi>n</mi><mi>t</mi><mi>e</mi><mi>r</mi><mi>r</mi><mi>o</mi><mi>g</mi><mi>a</mi><mi>t</mi><mi>e</mi><mi>d</mi><mspace width="0.5em"></mspace><mi>b</mi><mi>a</mi><mi>s</mi><mi>e</mi><mi>s</mi></mrow></mfrac></math></td></tr></table>
<p>
Here, the number of single occurring mutations refers to the number of variants that only occur in a single denoised duplex, single-strand or read (depending on the analysis performed), and the number of interrogated bases refers to the total number of bases that pass denoising filters. Given that the tools we developed in this manuscript are designed to detect variants and do not report reference bases, the number of interrogated bases was estimated according to:
</p>
<table class="disp-formula p" id="FD7"><tr><td class="formula"><math id="M7" display="block"><mi>I</mi><mi>n</mi><mi>t</mi><mi>e</mi><mi>r</mi><mi>r</mi><mi>o</mi><mi>g</mi><mi>a</mi><mi>t</mi><mi>e</mi><mi>d</mi><mspace width="0.5em"></mspace><mi>b</mi><mi>a</mi><mi>s</mi><mi>e</mi><mi>s</mi><mo>=</mo><mi>t</mi><mi>o</mi><mi>t</mi><mi>a</mi><mi>l</mi><mspace width="0.5em"></mspace><mi>o</mi><mi>b</mi><mi>s</mi><mi>e</mi><mi>r</mi><mi>v</mi><mi>e</mi><mi>d</mi><mspace width="0.5em"></mspace><mi>m</mi><mi>o</mi><mi>l</mi><mi>e</mi><mi>c</mi><mi>u</mi><mi>l</mi><mi>e</mi><mi>s</mi><mspace width="0.5em"></mspace><mi>x</mi><mspace width="0.5em"></mspace><mi>m</mi><mi>o</mi><mi>l</mi><mi>e</mi><mi>c</mi><mi>u</mi><mi>l</mi><mi>e</mi><mspace width="0.5em"></mspace><mi>l</mi><mi>e</mi><mi>n</mi><mi>g</mi><mi>t</mi><mi>h</mi><mspace width="0.5em"></mspace><mi>x</mi><mspace width="0.5em"></mspace><mi>h</mi><mi>o</mi><mi>m</mi><mi>o</mi><mi>z</mi><mi>y</mi><mi>g</mi><mi>o</mi><mi>u</mi><mi>s</mi><mspace width="0.5em"></mspace><mi>v</mi><mi>a</mi><mi>r</mi><mi>i</mi><mi>a</mi><mi>n</mi><mi>t</mi><mspace width="0.5em"></mspace><mi>f</mi><mi>i</mi><mi>l</mi><mi>t</mi><mi>e</mi><mi>r</mi><mi>i</mi><mi>n</mi><mi>g</mi><mspace width="0.5em"></mspace><mi>r</mi><mi>a</mi><mi>t</mi><mi>i</mi><mi>o</mi><mspace width="0.5em"></mspace></math></td></tr></table>
<p>
Here, the total observed molecules refer to either the total number of duplexes sequenced in a given region of the genome, the molecule length was set to 170bp. The product of these two variables estimates the total number of duplex (or single strand) bases sequenced. To account for the effect of filtering, we multiply the total number of bases by the fraction of homozygous variants that pass all filtering criteria.</p></section><section id="S30"><h3 class="pmc_sec_title">Duplex depth estimation</h3>
<p id="P54">The duplex coverage for each sample was estimated by multiplying the number of double-stranded DNA molecules recovered (that is, the number of unique molecules with at least one top and one bottom strand) by 170 (roughly the size of a cell-free DNA molecule) and dividing by the size of the genome (2,875,001,522).</p></section><section id="S31"><h3 class="pmc_sec_title">Trinucleotide frequency tabulation and signature contribution estimation</h3>
<p id="P55">After denoising, the trinucleotide frequencies of duplex-corrected SNVs (or single-stranded or UMI-agnostic-corrected SNVs) were tabulated using deconstructSigs<sup><a href="#R62" class="usa-link" aria-describedby="R62">62</a></sup>. Next, signature contributions were estimated using MuSiCal<sup><a href="#R45" class="usa-link" aria-describedby="R45">45</a></sup>. Here, the previously tabulated trinucleotide frequencies and cancer-specific reference catalogs and specificity thresholds were used. In melanoma studies, MuSiCal was used to refit the sample trinucleotide frequencies to a catalog containing SBS7a, SBS7b (from Cosmic v3.3) and clonal hematopoiesis<sup><a href="#R42" class="usa-link" aria-describedby="R42">42</a></sup>. MuSiCal was run in likelihood-bidirectional mode using an empirically defined threshold of 0.007. This mode performs signature refitting using a likelihood-based sparse non-negative least squares algorithm. Urothelial cancers samples were run similarly, although the catalog was pre-defined to contain signatures from APOBEC mutagenesis (SBS2 and SBS13, Cosmic v3.3), platinum chemotherapy (SBS31 and SBS35, Cosmic v3.3) and clonal hematopoiesis<sup><a href="#R42" class="usa-link" aria-describedby="R42">42</a></sup> with a MuSiCal threshold of 0.025).</p></section><section id="S32"><h3 class="pmc_sec_title">
<em>In silico</em> mixing (duplex WGS)</h3>
<p id="P56">The in silico mixing study was obtained by computationally mixing variants from high burden (HB) sample MEL-12.B (tumor fraction of 11.59%) with variants from cancer-free control (CFC) CTRL-06. First, the duplex coverage for each sample was estimated by multiplying the number of double-stranded DNA molecules recovered (that is, the number of unique molecules with at least one top and one bottom strand) by 170 (roughly the size of a cell-free DNA molecule) and dividing by the size of the genome (2,875,001,522). Here, MEL-12.B and CTRL-06 had coverages of 16x and 8.8x, respectively. A file containing all sequencing reads contributing to an SNV-containing duplex molecule was generated for the HB and CFC sample.</p>
<p id="P57">Next, expected tumor fractions and coverages were set for the in silico mixes. Here, expected tumor fractions ranged from 0.1 to X, and the expected coverage was set to 6.5x. A downsampling ratio of reads from the original HB sample was set as:
</p>
<table class="disp-formula p" id="FD8"><tr><td class="formula"><math id="M8" display="block"><mi>H</mi><mi>B</mi><mspace width="0.5em"></mspace><mi>d</mi><mi>o</mi><mi>w</mi><mi>n</mi><mi>s</mi><mi>a</mi><mi>m</mi><mi>p</mi><mi>l</mi><mi>i</mi><mi>n</mi><mi>g</mi><mspace width="0.5em"></mspace><mi>r</mi><mi>a</mi><mi>t</mi><mi>i</mi><mi>o</mi><mo>=</mo><mfrac><mrow><mi>f</mi><mi>i</mi><mi>n</mi><mi>a</mi><mi>l</mi><mspace width="0.5em"></mspace><mi>c</mi><mi>o</mi><mi>v</mi><mi>e</mi><mi>r</mi><mi>a</mi><mi>g</mi><mi>e</mi></mrow><mrow><mi>H</mi><mi>B</mi><mspace width="0.5em"></mspace><mi>c</mi><mi>o</mi><mi>v</mi><mi>e</mi><mi>r</mi><mi>a</mi><mi>g</mi><mi>e</mi></mrow></mfrac><mi>x</mi><mspace width="0.5em"></mspace><mfrac><mrow><mi>e</mi><mi>x</mi><mi>p</mi><mi>e</mi><mi>c</mi><mi>t</mi><mi>e</mi><mi>d</mi><mspace width="0.5em"></mspace><mi>t</mi><mi>u</mi><mi>m</mi><mi>o</mi><mi>r</mi><mspace width="0.5em"></mspace><mi>f</mi><mi>r</mi><mi>a</mi><mi>c</mi><mi>t</mi><mi>i</mi><mi>o</mi><mi>n</mi></mrow><mrow><mi>H</mi><mi>B</mi><mspace width="0.5em"></mspace><mi>t</mi><mi>u</mi><mi>m</mi><mi>o</mi><mi>r</mi><mspace width="0.5em"></mspace><mi>f</mi><mi>r</mi><mi>a</mi><mi>c</mi><mi>t</mi><mi>i</mi><mi>o</mi><mi>n</mi></mrow></mfrac></math></td></tr></table>
<p>
The CFC downsampling ratio was set as:
</p>
<table class="disp-formula p" id="FD9"><tr><td class="formula"><math id="M9" display="block"><mi>C</mi><mi>F</mi><mi>C</mi><mspace width="0.5em"></mspace><mi>d</mi><mi>o</mi><mi>w</mi><mi>n</mi><mi>s</mi><mi>a</mi><mi>m</mi><mi>p</mi><mi>l</mi><mi>i</mi><mi>n</mi><mi>g</mi><mspace width="0.5em"></mspace><mi>r</mi><mi>a</mi><mi>t</mi><mi>i</mi><mi>o</mi><mo>=</mo><mfrac><mrow><mi>f</mi><mi>i</mi><mi>n</mi><mi>a</mi><mi>l</mi><mspace width="0.5em"></mspace><mi>c</mi><mi>o</mi><mi>v</mi><mi>e</mi><mi>r</mi><mi>a</mi><mi>g</mi><mi>e</mi></mrow><mrow><mi>C</mi><mi>F</mi><mi>C</mi><mspace width="0.5em"></mspace><mi>c</mi><mi>o</mi><mi>v</mi><mi>e</mi><mi>r</mi><mi>a</mi><mi>g</mi><mi>e</mi></mrow></mfrac><mi>x</mi><mspace width="0.5em"></mspace><mfrac><mrow><mn>1</mn><mo>-</mo><mspace width="0.5em"></mspace><mi>e</mi><mi>x</mi><mi>p</mi><mi>e</mi><mi>c</mi><mi>t</mi><mi>e</mi><mi>d</mi><mspace width="0.5em"></mspace><mi>t</mi><mi>u</mi><mi>m</mi><mi>o</mi><mi>r</mi><mspace width="0.5em"></mspace><mi>f</mi><mi>r</mi><mi>a</mi><mi>c</mi><mi>t</mi><mi>i</mi><mi>o</mi><mi>n</mi></mrow><mrow><mi>H</mi><mi>B</mi><mspace width="0.5em"></mspace><mi>t</mi><mi>u</mi><mi>m</mi><mi>o</mi><mi>r</mi><mspace width="0.5em"></mspace><mi>f</mi><mi>r</mi><mi>a</mi><mi>c</mi><mi>t</mi><mi>i</mi><mi>o</mi><mi>n</mi></mrow></mfrac></math></td></tr></table>
<p>
Therefore, the total number of duplexes to sample from a HB or CFC file was equivalent to:
</p>
<table class="disp-formula p" id="FD10"><tr><td class="formula"><math id="M10" display="block"><mi>R</mi><mi>e</mi><mi>a</mi><mi>d</mi><mi>s</mi><mspace width="0.5em"></mspace><mi>t</mi><mi>o</mi><mspace width="0.5em"></mspace><mi>s</mi><mi>a</mi><mi>m</mi><mi>p</mi><mi>l</mi><mi>e</mi><mo>=</mo><mi>d</mi><mi>o</mi><mi>w</mi><mi>n</mi><mi>s</mi><mi>a</mi><mi>m</mi><mi>p</mi><mi>l</mi><mi>i</mi><mi>n</mi><mi>g</mi><mspace width="0.5em"></mspace><mi>r</mi><mi>a</mi><mi>t</mi><mi>i</mi><mi>o</mi><mi mathvariant="normal">*</mi><mi>n</mi><mi>u</mi><mi>m</mi><mi>b</mi><mi>e</mi><mi>r</mi><mspace width="0.5em"></mspace><mi>o</mi><mi>f</mi><mspace width="0.5em"></mspace><mi>S</mi><mi>N</mi><mi>V</mi><mo>-</mo><mi>c</mi><mi>o</mi><mi>n</mi><mi>t</mi><mi>a</mi><mi>i</mi><mi>n</mi><mi>i</mi><mi>n</mi><mi>g</mi><mspace width="0.5em"></mspace><mi>d</mi><mi>u</mi><mi>p</mi><mi>l</mi><mi>e</mi><mi>x</mi><mi>e</mi><mi>s</mi></math></td></tr></table>
<p>
The exact number of reads per seed was sample from a normal distribution with a standard deviation of 2.5 and a mean equivalent to “reads to sample”. Next, the randomly sample SNV-containing duplexes (and the sequencing reads that contribute to this duplex) of the HB and CFC sample were merged and denoised as described above (UMI WGS data processing). Trinucleotide frequencies were tabulated, and melanoma signature scores were measured as described above (Trinucleotide frequency tabulation and signature contribution estimation). Germline variants were removed after in silico mixing, using allele frequencies of the original samples.</p></section><section id="S33"><h3 class="pmc_sec_title">Duplex, single-strand, and UMI-agnostic error rates in mouse PDX plasma samples</h3>
<p id="P58">Denoising was performed as described above. Variants at a given genomic position, for each correction method, were compared to the frequency of that variant in uncorrected datasets. If the variant occurred two or fewer times in an uncorrected dataset, the variant was considered an error. The error rate was defined as the sum of the errors divided by the total number of base pairs for that correction method. For example, the error rate for duplex datasets corresponded to the number of errors divided by the total number of mapped base pairs from consensus duplex reads.</p></section><section id="S34"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="P59">Statistical analysis was performed in R (version 3.6). Boxplots were generated using the ggplot2 (version 3.3.5) R package. The lower and upper ends of the boxes represent the 25<sup>th</sup> and 75<sup>th</sup> percentiles of the data, respectively, and the horizontal line represents the median. The whiskers represent at most 1.5 times the IQR.</p></section></section><section id="S35"><h2 class="pmc_sec_title">Extended Data</h2>
<figure class="fig xbox font-sm" id="F4"><h3 class="obj_head">Extended Data Figure 1: Ultima and Illumina sequencing datasets of human-mapped reads in mouse PDX datasets (n=3).</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5a41/12077166/e3dbe960d026/nihms-2073968-f0004.jpg" loading="lazy" height="2010" width="1926" alt="Extended Data Figure 1:"></p>
<div class="p text-right font-secondary"><a href="figure/F4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P60"><strong>A</strong> Homopolymer size estimation of bases between two PCR duplicates (all samples combined) in Ultima datasets. <strong>B</strong> Homopolymer size estimation of bases between a read and the aligned reference (all samples combined) in Ultima datasets. <strong>C</strong> Homopolymer size estimation of bases between two PCR duplicates (all samples combined) in Illumina datasets. <strong>D</strong> Homopolymer size estimation of bases between a read and the aligned reference (all samples combined) in Illumina datasets. <strong>E</strong> Indel calling accuracy by PCR duplicate family sizes in Ultima datasets. <strong>F</strong> Indel calling accuracy of Illumina sequencing reads (for single family reads). <strong>G</strong> Frequency of homopolymer sizes across the human genome. For boxplots in (<strong>E</strong>) and (<strong>F</strong>), the lower and upper ends of boxes represent the 25<sup>th</sup> and 75<sup>th</sup> percentiles of the data, respectively, and the horizontal lines represent the median. The whiskers represent at most 1.5 times the IQR. Accuracy in (<strong>E</strong>) and (<strong>F</strong>) is defined as the number of correct homopolymer assignments in individual sequencing reads divided by the occurrences of that homopolymer size in the human genome in all sequenced reads.</p></figcaption></figure><figure class="fig xbox font-sm" id="F5"><h3 class="obj_head">Extended Data Figure 2: Flow-based sequencing provides predictable error-robust motifs.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5a41/12077166/94583f8ae174/nihms-2073968-f0005.jpg" loading="lazy" height="1965" width="1875" alt="Extended Data Figure 2:"></p>
<div class="p text-right font-secondary"><a href="figure/F5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P61"><strong>A</strong> Single-nucleotide variant analysis of matched Ultima and Illumina sequencing datasets across 96 trinucleotide contexts. Cycle shift motifs (described in B) are indicated by plus signs. <strong>B</strong> Left: Example sequencing of a TGC trinucleotide in flowspace. Given a flow order of T&gt;G&gt;C&gt;A, one full flow cycle of each nucleotide should provide a 1&gt;1&gt;1&gt;0 signal. <strong>Top, right:</strong> Example of how a T[G&gt;A]C alt disrupts the cycles in flow space basecalling. Two sequencing cycles are required to fully resolve a TAC sequencing motif. We refer to these types of motifs as cycle shift motifs. <strong>Bottom, right:</strong> Example of how a T[G&gt;C]C variant does not affect the cycles of flow space basecalling. <strong>C</strong> Error rates in Ultima and Illumina sequencing datasets for trinucleotide variants that alter the flowspace sequencing cycle. P-values were measured using a two-sided Wilcoxon test. Error bars in (<strong>A</strong>) represent the standard error of the mean. For boxplots in (<strong>C</strong>), the lower and upper ends of boxes represent the 25<sup>th</sup> and 75<sup>th</sup> percentiles of the data, respectively, and the horizontal lines represent the median. The whiskers represent at most 1.5 times the IQR.</p></figcaption></figure><figure class="fig xbox font-sm" id="F6"><h3 class="obj_head">Extended Data Figure 3: Tradeoffs between deep-targeted sequencing and modest whole-genome sequencing for ctDNA detection.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5a41/12077166/a746bd9b0adc/nihms-2073968-f0006.jpg" loading="lazy" height="1064" width="1200" alt="Extended Data Figure 3:"></p>
<div class="p text-right font-secondary"><a href="figure/F6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P62"><strong>A</strong> Mutational burden (number of SNVs) of 22 cancer types retrieved from the Pan Cancer Analysis of Whole Genomes consortium. The numbers along the x-axis represent the number of tumors analyzed per cancer type. <strong>B</strong> Median ctDNA detection opportunities using a whole-genome approach with 10x sequencing coverage, a 10-target panel at 10,000x coverage and a 1-target panel at 10,000x coverage. The pink shaded area represents tumor types for which targeting only a few sites may offer benefit over whole-genome sequencing. The blue shaded area represents tumor types for which a whole-genome approach will offer more opportunities to detect ctDNA over targeted panels. The lower and upper ends of the boxplots in (<strong>A</strong>) represent the 25<sup>th</sup> and 75<sup>th</sup> percentiles of the data, respectively, and the horizontal lines represent the median. The whiskers represent at most 1.5 times the IQR.</p></figcaption></figure><figure class="fig xbox font-sm" id="F7"><h3 class="obj_head">Extended Data Figure 4: Circulating tumor DNA cost and coverage analysis between Illumina and Ultima sequencing in a matched sample.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5a41/12077166/11f58f095bde/nihms-2073968-f0007.jpg" loading="lazy" height="731" width="1365" alt="Extended Data Figure 4:"></p>
<div class="p text-right font-secondary"><a href="figure/F7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P63">Areas under the curve (AUCs) are measured by calculating the area under a receiver operating characteristic curve comparing a given group (for example, Illumina 20x at 10<sup>−6</sup> expected tumor fraction) to its platform and coverage-matched cancer-free control (for example, Illumina 20x, expected tumor fraction of 0). All AUCs at expected tumor fractions of 10<sup>−4</sup> and greater were 1.00. Z-scores of a given sample are calculated against their coverage and platform matched cancer-free control (expected tumor fraction of 0).</p></figcaption></figure><figure class="fig xbox font-sm" id="F8"><h3 class="obj_head">Extended Data Figure 5: Variant allele frequencies for variants across denoising approaches.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5a41/12077166/8f9038c22146/nihms-2073968-f0008.jpg" loading="lazy" height="1585" width="1919" alt="Extended Data Figure 5:"></p>
<div class="p text-right font-secondary"><a href="figure/F8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P64">Variant allele frequencies (calculated using unfiltered sequencing reads) in positions where a variant was found using UMI-agnostic denoised reads, Single strand corrected reads and in duplex corrected reads. Allele frequencies of 0.2 and below are colored in red.</p></figcaption></figure><figure class="fig xbox font-sm" id="F9"><h3 class="obj_head">Extended Data Figure 6: Comparison of detected UV-derived mutations using duplex, single-strand and UMI-agnostic denoising methods.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5a41/12077166/0daefddb0e43/nihms-2073968-f0009.jpg" loading="lazy" height="1131" width="1853" alt="Extended Data Figure 6:"></p>
<div class="p text-right font-secondary"><a href="figure/F9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P65"><strong>A</strong> Cosine similarities by cancer stage at baseline timepoints (pre-treatment or pre-surgery) for UV and CH-associated signatures. <strong>B</strong> Comparison of duplex, single-strand and UMI-agnostic denoising methods to detect melanoma-associated variants using a single-read variant calling pipeline for pre-treatment plasma samples from melanoma patients (top) and cancer-free controls (bottom). P-values were measured using a two-sided Wilcoxon test. For all boxplots, the lower and upper ends of boxes represent the 25<sup>th</sup> and 75<sup>th</sup> percentiles of the data, respectively, and the horizontal lines represent the median. The whiskers represent at most 1.5 times the IQR.</p></figcaption></figure><figure class="fig xbox font-sm" id="F10"><h3 class="obj_head">Extended Data Figure 7: Tumor-agnostic copy-number based tumor fraction estimation in stage III and IV melanoma and cancer-free control samples.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5a41/12077166/d743da03c4d1/nihms-2073968-f0010.jpg" loading="lazy" height="1197" width="1200" alt="Extended Data Figure 7:"></p>
<div class="p text-right font-secondary"><a href="figure/F10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P66">Samples include cancer-free controls (n = 10); stage III melanoma (pre-surgery; n = 10) and stage IV melanoma (pre-treatment; n = 4). Dotted line at 0.03 represents the limit of detection of ichorCNA. For boxplots, the lower and upper ends of boxes represent the 25<sup>th</sup> and 75<sup>th</sup> percentiles of the data, respectively, and the horizontal lines represent the median.</p></figcaption></figure><figure class="fig xbox font-sm" id="F11"><h3 class="obj_head">Extended Data Figure 8: ctDNA dynamics throughout treatment in melanoma patients.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5a41/12077166/bc0aa349cc87/nihms-2073968-f0011.jpg" loading="lazy" height="912" width="1883" alt="Extended Data Figure 8:"></p>
<div class="p text-right font-secondary"><a href="figure/F11/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P67"><strong>A</strong> Changes in circulating tumor DNA (increase or decrease) relative to the earliest sampled timepoint. Solid lines represent patients with recurrence or progressive disease, and dashed lines represent patients with either partial response or who are recurrence-free following treatment. Closed and open circles represent samples with and without detected ctDNA, respectively. <strong>B</strong> Difference in ctDNA relative to the pre-treatment timepoint stratified by clinical outcome. One sample did not have a pre-treatment timepoint available (MEL-15; progressive disease) and so a day 9 post-treatment time point was used as baseline. For boxplots in (<strong>B</strong>), the lower and upper ends of boxes represent the 25<sup>th</sup> and 75<sup>th</sup> percentiles of the data, respectively, and the horizontal lines represent the median. The whiskers represent at most 1.5 times the IQR. P-values were calculated using a two-sided Wilcoxon test.</p></figcaption></figure><figure class="fig xbox font-sm" id="F12"><h3 class="obj_head">Extended Data Figure 9: Major signature contributions from urothelial cancer patients’ tumors measured through whole-genome sequencing.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5a41/12077166/03df13233002/nihms-2073968-f0012.jpg" loading="lazy" height="1064" width="1907" alt="Extended Data Figure 9:"></p>
<div class="p text-right font-secondary"><a href="figure/F12/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P68">Top: total mutation counts per sequenced tumor. Bottom: signature contributions. Trinucleotide frequencies were fit to the entire COSMIC database (version v.3.3). When a patient had two or more tumors (B01, B04, B15, B16, B17, B18, B19), we identified clonal mutations by their presence in at least two tumors.</p></figcaption></figure></section><section id="SM1"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="SD1"><div class="caption p"><span>Supplementary Tables</span></div>
<div class="media p" id="d67e3232"><div class="caption">
<a href="/articles/instance/12077166/bin/NIHMS2073968-supplement-Supplementary_Tables.xlsx" data-ga-action="click_feat_suppl" class="usa-link">NIHMS2073968-supplement-Supplementary_Tables.xlsx</a><sup> (69.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="SD2"><div class="caption p"><span>Supplementary Note and Supplementary Figures</span></div>
<div class="media p" id="d67e3235"><div class="caption">
<a href="/articles/instance/12077166/bin/NIHMS2073968-supplement-Supplementary_Note_and_Supplementary_Figures.pdf" data-ga-action="click_feat_suppl" class="usa-link">NIHMS2073968-supplement-Supplementary_Note_and_Supplementary_Figures.pdf</a><sup> (2.2MB, pdf) </sup>
</div></div></section></section><section id="S36" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGEMENTS</h2>
<p id="P69">We thank the patients and their families for contributing plasma and tissue for this study. We also thank Henry Rui He at Weill Cornell, Jane Park and all members of the Landau laboratory, the New York Genome Center computational biology team, especially Minita Shah, and the New York Genome Center research sequencing laboratory for thoughtful discussions throughout this work. This work was supported by the Mark foundation Aspire Award, the Burroughs Wellcome Fund Career Award for Medical Scientists, a National Cancer Institute R01 grant (R01-CA266619–01) and the Melanoma Research Alliance Established Investigator Award (DAL). A.P.C. received support from the American Cancer Society Postdoctoral Fellowship program. MSKCC investigators are supported by Cancer Center Support Grant P30 CA08748 from the National Institutes of Health/National Cancer Institute. A.J.W. received support from the Conquer Cancer Foundation Young Investigator Award, the Melanoma Research Alliance Young Investigator Award, and the NCI K08 Mentored Career Scientist Award (K08 CA263301–03). DAL is a Scholar of the Leukemia &amp; Lymphoma Society. This work was made possible by the MacMillan Family Foundation and the MacMillan Center for the Study of the Non-Coding Cancer Genome at the New York Genome Center. The opinions, results, and conclusions reported in this paper are those of the authors and are independent from these funding sources.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="FN5">
<p id="P71">COMPETING INTERESTS</p>
<p id="P72">A.P.C. and D.A.L. have filed a provisional patent regarding certain aspects of this manuscript. D.A.L. and A.J.W. have also filed two additional patent applications regarding work presented in this manuscript. A.P.C. is listed as an inventor on submitted patents pertaining to cell-free DNA (US patent applications 63/237,367, 63/056,249, 63/015,095 and 16/500,929), has received consulting fees from Eurofins Viracor and has received conference travel support from Ultima Genomics. D.A.L. is on the Scientific Advisory Board of Mission Bio, Pangea, Alethiomics, and Veracyte, and has received prior research funding from 10x Genomics, Ultima Genomics, Oxford Nanopore Technologies and Illumina. I.R., A.J. and D.L. are employees and shareholders of Ultima Genomics. J.D.W. is a consultant for: Apricity; Ascentage Pharma; Bicara Therapeutics; Bristol Myers Squibb; Daiichi Sankyo; Dragonfly; Imvaq; Larkspur; Psioxus, Takeda; Tizona; Trishula Therapeutics; Immunocore – Data Safety board; Scancell. JDW received Grant/Research Support from: Bristol Myers Squibb; Enterome. JDW has Equity in: Apricity, Arsenal IO/CellCarta; Ascentage; Imvaq; Linneaus; Georgiamune; Takeda; Tizona Therapeutics; Xenimmune. JDW is an inventor on the following patents: Xenogeneic DNA Vaccines; Newcastle Disease viruses for Cancer Therapy; Myeloid-derived suppressor cell (MDSC) assay; Prediction of Responsiveness to Treatment with Immunomodulatory Therapeutics and Method of Monitoring Abscopal Effects during such Treatment; Anti-PD1 Antibody; Anti-CTLA4 antibodies; Anti-GITR antibodies and methods of use thereof; CD40 binding molecules and uses thereof. A.Sa. has received Research Funding from AstraZeneca and has served on Advisory Boards for G1 Therapeutics, Boehringer Ingelheim, Novocure, InxMed, Bristol Myers Squibb, and Galvanize Therapeutics, and as a consultant for Galvanize Therapeutics. M.A.P. has received consulting fees from BMS, Merck, Novartis, Eisai, Pfizer, Chugai and has received institutional support from RGenix, Infinity, BMS, Merck and Novartis. M.K.C. has received consulting fees from BMS, Merck, InCyte, Moderna, ImmunoCore, and AstraZeneca and receives institutional support from BMS. S.T. is funded by Cancer Research UK (grant reference number A29911); the Francis Crick Institute, which receives its core funding from Cancer Research UK (FC10988), the UK Medical Research Council (FC10988), and the Wellcome Trust (FC10988); the National Institute for Health Research (NIHR) Biomedical Research Centre at the Royal Marsden Hospital and Institute of Cancer Research (grant reference number A109), the Royal Marsden Cancer Charity, The Rosetrees Trust (grant reference number A2204), Ventana Medical Systems Inc (grant reference numbers 10467 and 10530), the National Institute of Health (U01 CA247439) and Melanoma Research Alliance (Award Ref no 686061). S.T. has received speaking fees from Roche, Astra Zeneca, Novartis and Ipsen. S.T. has the following patents filed: Indel mutations as a therapeutic target and predictive biomarker PCTGB2018/051892 and PCTGB2018/051893. G.M.B. has sponsored research agreements through her institution with: Olink Proteomics, Teiko Bio, InterVenn Biosciences, Palleon Pharmaceuticals; served on advisory boards for: Iovance, Merck, Nektar Therapeutics, Novartis, and Ankyra Therapeutics; consulted for: Merck, InterVenn Biosciences, and Ankyra Therapeutics; and holds equity in Ankyra Therapeutics. B.M.F. is on the advisory boards for Astrin Bioscience, Inc, Natera, Guardant, Janssen, Gilead, Merck, Immunomedics and QED therapeutics, is a consultant for QED therapeutics, Astra Biosciences and BostonGene, obtains patent royalties from Immunomedics and Gilead, honoraria from Urotoday, Axiom Healthcare Strategies, LLC and research support from Eli-Lilly. B.M.F. reports support from the NIH, DoD-CDMRP, Starr Cancer Consortium and the P-1000 Consortium. The remaining authors declare no competing interests.</p>
</div>
<div class="fn p" id="FN6">
<p id="P73">CODE AVAILABILITY</p>
<p id="P74">Code and custom scripts are available at <a href="https://github.com/alexpcheng/WGSDuplex" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/alexpcheng/WGSDuplex</a>.</p>
</div>
</div></section><section id="S37"><h2 class="pmc_sec_title">DATA AVAILABILITY</h2>
<p id="P70">The raw genomic sequencing data generated is available from the European Genome-Phenome Archive (EGA) under dataset accession code EGAD50000001234. Datasets obtained from the Pan Cancer Analysis of Whole Genomes Consortium (<a href="#SD1" class="usa-link">Supplementary Table 11</a>) are available at <a href="https://www.icgc-argo.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.icgc-argo.org/</a>. Urothelial cancer tumor/normal alignment files were obtained from Nguyen et al.<sup><a href="#R51" class="usa-link" aria-describedby="R51">51</a></sup> and were deposited to dbGap under accession number phs001087.v4.p1.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="R1">
<span class="label">1.</span><cite>Cohen JD
et al. 
Detection and localization of surgically resectable cancers with a multi-analyte blood test. Science
359, 926–930 (2018).
</cite> [<a href="https://doi.org/10.1126/science.aar3247" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6080308/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29348365/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Detection%20and%20localization%20of%20surgically%20resectable%20cancers%20with%20a%20multi-analyte%20blood%20test&amp;author=JD%20Cohen&amp;volume=359&amp;publication_year=2018&amp;pages=926-930&amp;pmid=29348365&amp;doi=10.1126/science.aar3247&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R2">
<span class="label">2.</span><cite>Sanz-Garcia E, Zhao E, Bratman SV &amp; Siu LL
Monitoring and adapting cancer treatment using circulating tumor DNA kinetics: Current research, opportunities, and challenges. Sci. Adv. 8, eabi8618 (2022).
</cite> [<a href="https://doi.org/10.1126/sciadv.abi8618" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8791609/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35080978/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Adv&amp;title=Monitoring%20and%20adapting%20cancer%20treatment%20using%20circulating%20tumor%20DNA%20kinetics:%20Current%20research,%20opportunities,%20and%20challenges&amp;author=E%20Sanz-Garcia&amp;author=E%20Zhao&amp;author=SV%20Bratman&amp;author=LL%20Siu&amp;volume=8&amp;publication_year=2022&amp;pages=eabi8618&amp;pmid=35080978&amp;doi=10.1126/sciadv.abi8618&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R3">
<span class="label">3.</span><cite>Snyder MW, Kircher M, Hill AJ, Daza RM &amp; Shendure J
Cell-free DNA Comprises an In Vivo Nucleosome Footprint that Informs Its Tissues-Of-Origin. Cell
164, 57–68 (2016).
</cite> [<a href="https://doi.org/10.1016/j.cell.2015.11.050" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4715266/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26771485/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Cell-free%20DNA%20Comprises%20an%20In%20Vivo%20Nucleosome%20Footprint%20that%20Informs%20Its%20Tissues-Of-Origin&amp;author=MW%20Snyder&amp;author=M%20Kircher&amp;author=AJ%20Hill&amp;author=RM%20Daza&amp;author=J%20Shendure&amp;volume=164&amp;publication_year=2016&amp;pages=57-68&amp;pmid=26771485&amp;doi=10.1016/j.cell.2015.11.050&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R4">
<span class="label">4.</span><cite>Wan JCM
et al. 
Liquid biopsies come of age: towards implementation of circulating tumour DNA. Nat. Rev. Cancer
17, 223–238 (2017).
</cite> [<a href="https://doi.org/10.1038/nrc.2017.7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28233803/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Cancer&amp;title=Liquid%20biopsies%20come%20of%20age:%20towards%20implementation%20of%20circulating%20tumour%20DNA&amp;author=JCM%20Wan&amp;volume=17&amp;publication_year=2017&amp;pages=223-238&amp;pmid=28233803&amp;doi=10.1038/nrc.2017.7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R5">
<span class="label">5.</span><cite>Wang S
et al. 
Potential clinical significance of a plasma-based KRAS mutation analysis in patients with advanced non-small cell lung cancer. Clin. Cancer Res. Off. J. Am. Assoc. Cancer Res. 16, 1324–1330 (2010).</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-09-2672" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20145159/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.%20Off.%20J.%20Am.%20Assoc.%20Cancer%20Res&amp;title=Potential%20clinical%20significance%20of%20a%20plasma-based%20KRAS%20mutation%20analysis%20in%20patients%20with%20advanced%20non-small%20cell%20lung%20cancer&amp;author=S%20Wang&amp;volume=16&amp;publication_year=2010&amp;pages=1324-1330&amp;pmid=20145159&amp;doi=10.1158/1078-0432.CCR-09-2672&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R6">
<span class="label">6.</span><cite>Murtaza M
et al. 
Non-invasive analysis of acquired resistance to cancer therapy by sequencing of plasma DNA. Nature
497, 108–112 (2013).
</cite> [<a href="https://doi.org/10.1038/nature12065" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23563269/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Non-invasive%20analysis%20of%20acquired%20resistance%20to%20cancer%20therapy%20by%20sequencing%20of%20plasma%20DNA&amp;author=M%20Murtaza&amp;volume=497&amp;publication_year=2013&amp;pages=108-112&amp;pmid=23563269&amp;doi=10.1038/nature12065&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R7">
<span class="label">7.</span><cite>Diehl F
et al. 
Circulating mutant DNA to assess tumor dynamics. Nat. Med. 14, 985–990 (2008).
</cite> [<a href="https://doi.org/10.1038/nm.1789" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2820391/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18670422/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Med&amp;title=Circulating%20mutant%20DNA%20to%20assess%20tumor%20dynamics&amp;author=F%20Diehl&amp;volume=14&amp;publication_year=2008&amp;pages=985-990&amp;pmid=18670422&amp;doi=10.1038/nm.1789&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R8">
<span class="label">8.</span><cite>Agarwal R
et al. 
Dynamic molecular monitoring reveals that SWI–SNF mutations mediate resistance to ibrutinib plus venetoclax in mantle cell lymphoma. Nat. Med. 25, 119–129 (2019).
</cite> [<a href="https://doi.org/10.1038/s41591-018-0243-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30455436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Med&amp;title=Dynamic%20molecular%20monitoring%20reveals%20that%20SWI%E2%80%93SNF%20mutations%20mediate%20resistance%20to%20ibrutinib%20plus%20venetoclax%20in%20mantle%20cell%20lymphoma&amp;author=R%20Agarwal&amp;volume=25&amp;publication_year=2019&amp;pages=119-129&amp;pmid=30455436&amp;doi=10.1038/s41591-018-0243-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R9">
<span class="label">9.</span><cite>Newman AM
et al. 
An ultrasensitive method for quantitating circulating tumor DNA with broad patient coverage. Nat. Med. 20, 548–554 (2014).
</cite> [<a href="https://doi.org/10.1038/nm.3519" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4016134/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24705333/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Med&amp;title=An%20ultrasensitive%20method%20for%20quantitating%20circulating%20tumor%20DNA%20with%20broad%20patient%20coverage&amp;author=AM%20Newman&amp;volume=20&amp;publication_year=2014&amp;pages=548-554&amp;pmid=24705333&amp;doi=10.1038/nm.3519&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R10">
<span class="label">10.</span><cite>Newman AM
et al. 
Integrated digital error suppression for improved detection of circulating tumor DNA. Nat. Biotechnol. 34, 547–555 (2016).
</cite> [<a href="https://doi.org/10.1038/nbt.3520" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4907374/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27018799/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol&amp;title=Integrated%20digital%20error%20suppression%20for%20improved%20detection%20of%20circulating%20tumor%20DNA&amp;author=AM%20Newman&amp;volume=34&amp;publication_year=2016&amp;pages=547-555&amp;pmid=27018799&amp;doi=10.1038/nbt.3520&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R11">
<span class="label">11.</span><cite>Kurtz DM
et al. 
Enhanced detection of minimal residual disease by targeted sequencing of phased variants in circulating tumor DNA. Nat. Biotechnol. 39, 1537–1547 (2021).
</cite> [<a href="https://doi.org/10.1038/s41587-021-00981-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8678141/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34294911/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol&amp;title=Enhanced%20detection%20of%20minimal%20residual%20disease%20by%20targeted%20sequencing%20of%20phased%20variants%20in%20circulating%20tumor%20DNA&amp;author=DM%20Kurtz&amp;volume=39&amp;publication_year=2021&amp;pages=1537-1547&amp;pmid=34294911&amp;doi=10.1038/s41587-021-00981-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R12">
<span class="label">12.</span><cite>Cohen JD
et al. 
Detection of low-frequency DNA variants by targeted sequencing of the Watson and Crick strands. Nat. Biotechnol. 39, 1220–1227 (2021).
</cite> [<a href="https://doi.org/10.1038/s41587-021-00900-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8627329/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33941929/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol&amp;title=Detection%20of%20low-frequency%20DNA%20variants%20by%20targeted%20sequencing%20of%20the%20Watson%20and%20Crick%20strands&amp;author=JD%20Cohen&amp;volume=39&amp;publication_year=2021&amp;pages=1220-1227&amp;pmid=33941929&amp;doi=10.1038/s41587-021-00900-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R13">
<span class="label">13.</span><cite>Chaudhuri AA
et al. 
Early Detection of Molecular Residual Disease in Localized Lung Cancer by Circulating Tumor DNA Profiling. Cancer Discov. 7, 1394–1403 (2017).
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-17-0716" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5895851/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28899864/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Discov&amp;title=Early%20Detection%20of%20Molecular%20Residual%20Disease%20in%20Localized%20Lung%20Cancer%20by%20Circulating%20Tumor%20DNA%20Profiling&amp;author=AA%20Chaudhuri&amp;volume=7&amp;publication_year=2017&amp;pages=1394-1403&amp;pmid=28899864&amp;doi=10.1158/2159-8290.CD-17-0716&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R14">
<span class="label">14.</span><cite>Zviran A
et al. 
Genome-wide cell-free DNA mutational integration enables ultra-sensitive cancer monitoring. Nat. Med. 26, 1114–1124 (2020).
</cite> [<a href="https://doi.org/10.1038/s41591-020-0915-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8108131/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32483360/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Med&amp;title=Genome-wide%20cell-free%20DNA%20mutational%20integration%20enables%20ultra-sensitive%20cancer%20monitoring&amp;author=A%20Zviran&amp;volume=26&amp;publication_year=2020&amp;pages=1114-1124&amp;pmid=32483360&amp;doi=10.1038/s41591-020-0915-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R15">
<span class="label">15.</span><cite>Abbosh C
et al. 
Phylogenetic ctDNA analysis depicts early stage lung cancer evolution. Nature
545, 446–451 (2017).
</cite> [<a href="https://doi.org/10.1038/nature22364" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5812436/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28445469/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Phylogenetic%20ctDNA%20analysis%20depicts%20early%20stage%20lung%20cancer%20evolution&amp;author=C%20Abbosh&amp;volume=545&amp;publication_year=2017&amp;pages=446-451&amp;pmid=28445469&amp;doi=10.1038/nature22364&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R16">
<span class="label">16.</span><cite>Bettegowda C
et al. 
Detection of Circulating Tumor DNA in Early- and Late-Stage Human Malignancies. Sci. Transl. Med. 6, 224ra24 (2014).</cite> [<a href="https://doi.org/10.1126/scitranslmed.3007094" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4017867/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24553385/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Transl.%20Med&amp;title=Detection%20of%20Circulating%20Tumor%20DNA%20in%20Early-%20and%20Late-Stage%20Human%20Malignancies&amp;author=C%20Bettegowda&amp;volume=6&amp;publication_year=2014&amp;pages=224ra24&amp;pmid=24553385&amp;doi=10.1126/scitranslmed.3007094&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R17">
<span class="label">17.</span><cite>Gale D
et al. 
Residual ctDNA after treatment predicts early relapse in patients with early-stage non-small cell lung cancer. Ann. Oncol. 33, 500–510 (2022).
</cite> [<a href="https://doi.org/10.1016/j.annonc.2022.02.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9067454/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35306155/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ann.%20Oncol&amp;title=Residual%20ctDNA%20after%20treatment%20predicts%20early%20relapse%20in%20patients%20with%20early-stage%20non-small%20cell%20lung%20cancer&amp;author=D%20Gale&amp;volume=33&amp;publication_year=2022&amp;pages=500-510&amp;pmid=35306155&amp;doi=10.1016/j.annonc.2022.02.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R18">
<span class="label">18.</span><cite>Tie J
et al. 
Circulating Tumor DNA Analysis Guiding Adjuvant Therapy in Stage II Colon Cancer. N. Engl. J. Med. 386, 2261–2272 (2022).
</cite> [<a href="https://doi.org/10.1056/NEJMoa2200075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9701133/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35657320/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med&amp;title=Circulating%20Tumor%20DNA%20Analysis%20Guiding%20Adjuvant%20Therapy%20in%20Stage%20II%20Colon%20Cancer&amp;author=J%20Tie&amp;volume=386&amp;publication_year=2022&amp;pages=2261-2272&amp;pmid=35657320&amp;doi=10.1056/NEJMoa2200075&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R19">
<span class="label">19.</span><cite>Smith T, Heger A &amp; Sudbery I
UMI-tools: modeling sequencing errors in Unique Molecular Identifiers to improve quantification accuracy. Genome Res. 27, 491–499 (2017).
</cite> [<a href="https://doi.org/10.1101/gr.209601.116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5340976/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28100584/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res&amp;title=UMI-tools:%20modeling%20sequencing%20errors%20in%20Unique%20Molecular%20Identifiers%20to%20improve%20quantification%20accuracy&amp;author=T%20Smith&amp;author=A%20Heger&amp;author=I%20Sudbery&amp;volume=27&amp;publication_year=2017&amp;pages=491-499&amp;pmid=28100584&amp;doi=10.1101/gr.209601.116&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R20">
<span class="label">20.</span><cite>Hoang ML
et al. 
Genome-wide quantification of rare somatic mutations in normal human tissues using massively parallel sequencing. Proc. Natl. Acad. Sci. 113, 9846–9851 (2016).
</cite> [<a href="https://doi.org/10.1073/pnas.1607794113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5024639/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27528664/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci&amp;title=Genome-wide%20quantification%20of%20rare%20somatic%20mutations%20in%20normal%20human%20tissues%20using%20massively%20parallel%20sequencing&amp;author=ML%20Hoang&amp;volume=113&amp;publication_year=2016&amp;pages=9846-9851&amp;pmid=27528664&amp;doi=10.1073/pnas.1607794113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R21">
<span class="label">21.</span><cite>Abascal F
et al. 
Somatic mutation landscapes at single-molecule resolution. Nature
593, 405–410 (2021).
</cite> [<a href="https://doi.org/10.1038/s41586-021-03477-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33911282/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Somatic%20mutation%20landscapes%20at%20single-molecule%20resolution&amp;author=F%20Abascal&amp;volume=593&amp;publication_year=2021&amp;pages=405-410&amp;pmid=33911282&amp;doi=10.1038/s41586-021-03477-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R22">
<span class="label">22.</span><cite>Schmitt MW
et al. 
Detection of ultra-rare mutations by next-generation sequencing. Proc. Natl. Acad. Sci. 109, 14508–14513 (2012).
</cite> [<a href="https://doi.org/10.1073/pnas.1208715109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3437896/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22853953/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci&amp;title=Detection%20of%20ultra-rare%20mutations%20by%20next-generation%20sequencing&amp;author=MW%20Schmitt&amp;volume=109&amp;publication_year=2012&amp;pages=14508-14513&amp;pmid=22853953&amp;doi=10.1073/pnas.1208715109&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R23">
<span class="label">23.</span><cite>Meddeb R
et al. 
Quantifying circulating cell-free DNA in humans. Sci. Rep. 9, 5220 (2019).
</cite> [<a href="https://doi.org/10.1038/s41598-019-41593-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6435718/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30914716/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep&amp;title=Quantifying%20circulating%20cell-free%20DNA%20in%20humans&amp;author=R%20Meddeb&amp;volume=9&amp;publication_year=2019&amp;pages=5220&amp;pmid=30914716&amp;doi=10.1038/s41598-019-41593-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R24">
<span class="label">24.</span><cite>Widman AJ
et al. 
Ultrasensitive plasma-based monitoring of tumor burden using machine-learning-guided signal enrichment. Nat. Med. 30, 1655–1666 (2024).
</cite> [<a href="https://doi.org/10.1038/s41591-024-03040-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7616143/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38877116/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Med&amp;title=Ultrasensitive%20plasma-based%20monitoring%20of%20tumor%20burden%20using%20machine-learning-guided%20signal%20enrichment&amp;author=AJ%20Widman&amp;volume=30&amp;publication_year=2024&amp;pages=1655-1666&amp;pmid=38877116&amp;doi=10.1038/s41591-024-03040-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R25">
<span class="label">25.</span><cite>DNA Sequencing Costs: Data. <a href="https://Genome.gov" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Genome.gov</a> <a href="https://www.genome.gov/about-genomics/fact-sheets/DNA-Sequencing-Costs-Data" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.genome.gov/about-genomics/fact-sheets/DNA-Sequencing-Costs-Data</a>.</cite>
</li>
<li id="R26">
<span class="label">26.</span><cite>Almogy G
et al. 
Cost-efficient whole genome-sequencing using novel mostly natural sequencing-by-synthesis chemistry and open fluidics platform. 2022.05.29.493900 Preprint at 10.1101/2022.05.29.493900 (2022).</cite> [<a href="https://doi.org/10.1101/2022.05.29.493900" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cost-efficient%20whole%20genome-sequencing%20using%20novel%20mostly%20natural%20sequencing-by-synthesis%20chemistry%20and%20open%20fluidics%20platform&amp;author=G%20Almogy&amp;publication_year=2022&amp;pages=2022.05.29.493900&amp;doi=10.1101/2022.05.29.493900&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R27">
<span class="label">27.</span><cite>Simmons SK
et al. 
Mostly natural sequencing-by-synthesis for scRNA-seq using Ultima sequencing. Nat. Biotechnol. 41, 204–211 (2023).
</cite> [<a href="https://doi.org/10.1038/s41587-022-01452-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9931582/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36109685/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol&amp;title=Mostly%20natural%20sequencing-by-synthesis%20for%20scRNA-seq%20using%20Ultima%20sequencing&amp;author=SK%20Simmons&amp;volume=41&amp;publication_year=2023&amp;pages=204-211&amp;pmid=36109685&amp;doi=10.1038/s41587-022-01452-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R28">
<span class="label">28.</span><cite>Hasenleithner SO &amp; Speicher MR
A clinician’s handbook for using ctDNA throughout the patient journey. Mol. Cancer
21, 81 (2022).
</cite> [<a href="https://doi.org/10.1186/s12943-022-01551-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8935823/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35307037/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cancer&amp;title=A%20clinician%E2%80%99s%20handbook%20for%20using%20ctDNA%20throughout%20the%20patient%20journey&amp;author=SO%20Hasenleithner&amp;author=MR%20Speicher&amp;volume=21&amp;publication_year=2022&amp;pages=81&amp;pmid=35307037&amp;doi=10.1186/s12943-022-01551-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R29">
<span class="label">29.</span><cite>Campbell PJ
et al. 
Pan-cancer analysis of whole genomes. Nature
578, 82–93 (2020).
</cite> [<a href="https://doi.org/10.1038/s41586-020-1969-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7025898/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32025007/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Pan-cancer%20analysis%20of%20whole%20genomes&amp;author=PJ%20Campbell&amp;volume=578&amp;publication_year=2020&amp;pages=82-93&amp;pmid=32025007&amp;doi=10.1038/s41586-020-1969-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R30">
<span class="label">30.</span><cite>Adalsteinsson VA
et al. 
Scalable whole-exome sequencing of cell-free DNA reveals high concordance with metastatic tumors. Nat. Commun. 8, 1324 (2017).
</cite> [<a href="https://doi.org/10.1038/s41467-017-00965-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5673918/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29109393/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=Scalable%20whole-exome%20sequencing%20of%20cell-free%20DNA%20reveals%20high%20concordance%20with%20metastatic%20tumors&amp;author=VA%20Adalsteinsson&amp;volume=8&amp;publication_year=2017&amp;pages=1324&amp;pmid=29109393&amp;doi=10.1038/s41467-017-00965-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R31">
<span class="label">31.</span><cite>Rose Brannon A
et al. 
Enhanced specificity of clinical high-sensitivity tumor mutation profiling in cell-free DNA via paired normal sequencing using MSK-ACCESS. Nat. Commun. 12, 3770 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-24109-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8213710/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34145282/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=Enhanced%20specificity%20of%20clinical%20high-sensitivity%20tumor%20mutation%20profiling%20in%20cell-free%20DNA%20via%20paired%20normal%20sequencing%20using%20MSK-ACCESS&amp;author=A%20Rose%20Brannon&amp;volume=12&amp;publication_year=2021&amp;pages=3770&amp;pmid=34145282&amp;doi=10.1038/s41467-021-24109-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R32">
<span class="label">32.</span><cite>Costello M
et al. 
Discovery and characterization of artifactual mutations in deep coverage targeted capture sequencing data due to oxidative DNA damage during sample preparation. Nucleic Acids Res. 41, e67 (2013).
</cite> [<a href="https://doi.org/10.1093/nar/gks1443" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3616734/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23303777/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Discovery%20and%20characterization%20of%20artifactual%20mutations%20in%20deep%20coverage%20targeted%20capture%20sequencing%20data%20due%20to%20oxidative%20DNA%20damage%20during%20sample%20preparation&amp;author=M%20Costello&amp;volume=41&amp;publication_year=2013&amp;pages=e67&amp;pmid=23303777&amp;doi=10.1093/nar/gks1443&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R33">
<span class="label">33.</span><cite>Bratman SV
et al. 
Personalized circulating tumor DNA analysis as a predictive biomarker in solid tumor patients treated with pembrolizumab. Nat. Cancer
1, 873–881 (2020).
</cite> [<a href="https://doi.org/10.1038/s43018-020-0096-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35121950/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Cancer&amp;title=Personalized%20circulating%20tumor%20DNA%20analysis%20as%20a%20predictive%20biomarker%20in%20solid%20tumor%20patients%20treated%20with%20pembrolizumab&amp;author=SV%20Bratman&amp;volume=1&amp;publication_year=2020&amp;pages=873-881&amp;pmid=35121950&amp;doi=10.1038/s43018-020-0096-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R34">
<span class="label">34.</span><cite>Cindy Yang SY
et al. 
Pan-cancer analysis of longitudinal metastatic tumors reveals genomic alterations and immune landscape dynamics associated with pembrolizumab sensitivity. Nat. Commun. 12, 5137 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-25432-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8390680/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34446728/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=Pan-cancer%20analysis%20of%20longitudinal%20metastatic%20tumors%20reveals%20genomic%20alterations%20and%20immune%20landscape%20dynamics%20associated%20with%20pembrolizumab%20sensitivity&amp;author=SY%20Cindy%20Yang&amp;volume=12&amp;publication_year=2021&amp;pages=5137&amp;pmid=34446728&amp;doi=10.1038/s41467-021-25432-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R35">
<span class="label">35.</span><cite>Liu MH
et al. 
DNA mismatch and damage patterns revealed by single-molecule sequencing. Nature
630, 752–761 (2024).
</cite> [<a href="https://doi.org/10.1038/s41586-024-07532-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11216816/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38867045/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=DNA%20mismatch%20and%20damage%20patterns%20revealed%20by%20single-molecule%20sequencing&amp;author=MH%20Liu&amp;volume=630&amp;publication_year=2024&amp;pages=752-761&amp;pmid=38867045&amp;doi=10.1038/s41586-024-07532-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R36">
<span class="label">36.</span><cite>Bae JH
et al. 
Single duplex DNA sequencing with CODEC detects mutations with high sensitivity. Nat. Genet. 55, 871–879 (2023).
</cite> [<a href="https://doi.org/10.1038/s41588-023-01376-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10181940/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37106072/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&amp;title=Single%20duplex%20DNA%20sequencing%20with%20CODEC%20detects%20mutations%20with%20high%20sensitivity&amp;author=JH%20Bae&amp;volume=55&amp;publication_year=2023&amp;pages=871-879&amp;pmid=37106072&amp;doi=10.1038/s41588-023-01376-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R37">
<span class="label">37.</span><cite>Thompson JC
et al. 
Detection of therapeutically targetable driver and resistance mutations in lung cancer patients by next generation sequencing of cell-free circulating tumor DNA. Clin. Cancer Res. Off. J. Am. Assoc. Cancer Res. 22, 5772–5782 (2016).</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-16-1231" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5448134/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27601595/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.%20Off.%20J.%20Am.%20Assoc.%20Cancer%20Res&amp;title=Detection%20of%20therapeutically%20targetable%20driver%20and%20resistance%20mutations%20in%20lung%20cancer%20patients%20by%20next%20generation%20sequencing%20of%20cell-free%20circulating%20tumor%20DNA&amp;author=JC%20Thompson&amp;volume=22&amp;publication_year=2016&amp;pages=5772-5782&amp;pmid=27601595&amp;doi=10.1158/1078-0432.CCR-16-1231&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R38">
<span class="label">38.</span><cite>Hu Y
et al. 
False-Positive Plasma Genotyping Due to Clonal Hematopoiesis. Clin. Cancer Res. 24, 4437–4443 (2018).
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-18-0143" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29567812/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res&amp;title=False-Positive%20Plasma%20Genotyping%20Due%20to%20Clonal%20Hematopoiesis&amp;author=Y%20Hu&amp;volume=24&amp;publication_year=2018&amp;pages=4437-4443&amp;pmid=29567812&amp;doi=10.1158/1078-0432.CCR-18-0143&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R39">
<span class="label">39.</span><cite>Abbosh C, Swanton C &amp; Birkbak NJ
Clonal haematopoiesis: a source of biological noise in cell-free DNA analyses. Ann. Oncol. 30, 358–359 (2019).
</cite> [<a href="https://doi.org/10.1093/annonc/mdy552" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6442654/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30649226/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ann.%20Oncol&amp;title=Clonal%20haematopoiesis:%20a%20source%20of%20biological%20noise%20in%20cell-free%20DNA%20analyses&amp;author=C%20Abbosh&amp;author=C%20Swanton&amp;author=NJ%20Birkbak&amp;volume=30&amp;publication_year=2019&amp;pages=358-359&amp;pmid=30649226&amp;doi=10.1093/annonc/mdy552&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R40">
<span class="label">40.</span><cite>Shaw JA
et al. 
Serial Postoperative Circulating Tumor DNA Assessment Has Strong Prognostic Value During Long-Term Follow-Up in Patients With Breast Cancer. JCO Precis. Oncol. 8, e2300456 (2024).
</cite> [<a href="https://doi.org/10.1200/PO.23.00456" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11161241/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38691816/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=JCO%20Precis.%20Oncol&amp;title=Serial%20Postoperative%20Circulating%20Tumor%20DNA%20Assessment%20Has%20Strong%20Prognostic%20Value%20During%20Long-Term%20Follow-Up%20in%20Patients%20With%20Breast%20Cancer&amp;author=JA%20Shaw&amp;volume=8&amp;publication_year=2024&amp;pages=e2300456&amp;pmid=38691816&amp;doi=10.1200/PO.23.00456&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R41">
<span class="label">41.</span><cite>Alexandrov LB
et al. 
Signatures of mutational processes in human cancer. Nature
500, 415–421 (2013).
</cite> [<a href="https://doi.org/10.1038/nature12477" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3776390/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23945592/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Signatures%20of%20mutational%20processes%20in%20human%20cancer&amp;author=LB%20Alexandrov&amp;volume=500&amp;publication_year=2013&amp;pages=415-421&amp;pmid=23945592&amp;doi=10.1038/nature12477&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R42">
<span class="label">42.</span><cite>Osorio FG
et al. 
Somatic Mutations Reveal Lineage Relationships and Age-Related Mutagenesis in Human Hematopoiesis. Cell Rep. 25, 2308–2316.e4 (2018).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2018.11.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6289083/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30485801/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&amp;title=Somatic%20Mutations%20Reveal%20Lineage%20Relationships%20and%20Age-Related%20Mutagenesis%20in%20Human%20Hematopoiesis&amp;author=FG%20Osorio&amp;volume=25&amp;publication_year=2018&amp;pages=2308-2316.e4&amp;pmid=30485801&amp;doi=10.1016/j.celrep.2018.11.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R43">
<span class="label">43.</span><cite>Alexandrov LB
et al. 
The repertoire of mutational signatures in human cancer. Nature
578, 94–101 (2020).
</cite> [<a href="https://doi.org/10.1038/s41586-020-1943-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7054213/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32025018/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20repertoire%20of%20mutational%20signatures%20in%20human%20cancer&amp;author=LB%20Alexandrov&amp;volume=578&amp;publication_year=2020&amp;pages=94-101&amp;pmid=32025018&amp;doi=10.1038/s41586-020-1943-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R44">
<span class="label">44.</span><cite>Tate JG
et al. 
COSMIC: the Catalogue Of Somatic Mutations In Cancer. Nucleic Acids Res. 47, D941–D947 (2019).
</cite> [<a href="https://doi.org/10.1093/nar/gky1015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6323903/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30371878/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=COSMIC:%20the%20Catalogue%20Of%20Somatic%20Mutations%20In%20Cancer&amp;author=JG%20Tate&amp;volume=47&amp;publication_year=2019&amp;pages=D941-D947&amp;pmid=30371878&amp;doi=10.1093/nar/gky1015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R45">
<span class="label">45.</span><cite>Jin H
et al. 
Accurate and sensitive mutational signature analysis with MuSiCal. Nat. Genet. 56, 541–552 (2024).
</cite> [<a href="https://doi.org/10.1038/s41588-024-01659-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10937379/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38361034/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&amp;title=Accurate%20and%20sensitive%20mutational%20signature%20analysis%20with%20MuSiCal&amp;author=H%20Jin&amp;volume=56&amp;publication_year=2024&amp;pages=541-552&amp;pmid=38361034&amp;doi=10.1038/s41588-024-01659-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R46">
<span class="label">46.</span><cite>Tan L
et al. 
Prediction and monitoring of relapse in stage III melanoma using circulating tumor DNA. Ann. Oncol. 30, 804–814 (2019).
</cite> [<a href="https://doi.org/10.1093/annonc/mdz048" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6551451/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30838379/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ann.%20Oncol&amp;title=Prediction%20and%20monitoring%20of%20relapse%20in%20stage%20III%20melanoma%20using%20circulating%20tumor%20DNA&amp;author=L%20Tan&amp;volume=30&amp;publication_year=2019&amp;pages=804-814&amp;pmid=30838379&amp;doi=10.1093/annonc/mdz048&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R47">
<span class="label">47.</span><cite>Lee JH
et al. 
Pre-operative ctDNA predicts survival in high-risk stage III cutaneous melanoma patients. Ann. Oncol. 30, 815–822 (2019).
</cite> [<a href="https://doi.org/10.1093/annonc/mdz075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6551453/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30860590/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ann.%20Oncol&amp;title=Pre-operative%20ctDNA%20predicts%20survival%20in%20high-risk%20stage%20III%20cutaneous%20melanoma%20patients&amp;author=JH%20Lee&amp;volume=30&amp;publication_year=2019&amp;pages=815-822&amp;pmid=30860590&amp;doi=10.1093/annonc/mdz075&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R48">
<span class="label">48.</span><cite>Petljak M
et al. 
Mechanisms of APOBEC3 mutagenesis in human cancer cells. Nature
607, 799–807 (2022).
</cite> [<a href="https://doi.org/10.1038/s41586-022-04972-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9329121/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35859169/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Mechanisms%20of%20APOBEC3%20mutagenesis%20in%20human%20cancer%20cells&amp;author=M%20Petljak&amp;volume=607&amp;publication_year=2022&amp;pages=799-807&amp;pmid=35859169&amp;doi=10.1038/s41586-022-04972-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R49">
<span class="label">49.</span><cite>Findlay JM
et al. 
Differential clonal evolution in oesophageal cancers in response to neo-adjuvant chemotherapy. Nat. Commun. 7, 11111 (2016).
</cite> [<a href="https://doi.org/10.1038/ncomms11111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4822033/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27045317/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&amp;title=Differential%20clonal%20evolution%20in%20oesophageal%20cancers%20in%20response%20to%20neo-adjuvant%20chemotherapy&amp;author=JM%20Findlay&amp;volume=7&amp;publication_year=2016&amp;pages=11111&amp;pmid=27045317&amp;doi=10.1038/ncomms11111&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R50">
<span class="label">50.</span><cite>Boot A
et al. 
In-depth characterization of the cisplatin mutational signature in human cell lines and in esophageal and liver tumors. Genome Res. 28, 654–665 (2018).
</cite> [<a href="https://doi.org/10.1101/gr.230219.117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5932606/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29632087/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res&amp;title=In-depth%20characterization%20of%20the%20cisplatin%20mutational%20signature%20in%20human%20cell%20lines%20and%20in%20esophageal%20and%20liver%20tumors&amp;author=A%20Boot&amp;volume=28&amp;publication_year=2018&amp;pages=654-665&amp;pmid=29632087&amp;doi=10.1101/gr.230219.117&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R51">
<span class="label">51.</span><cite>Nguyen DD
et al. 
The Interplay between Mutagenesis and Extrachromosomal DNA Shapes Urothelial Cancer Evolution. 2023.05.07.538753 Preprint at 10.1101/2023.05.07.538753 (2023).</cite> [<a href="https://doi.org/10.1101/2023.05.07.538753" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Interplay%20between%20Mutagenesis%20and%20Extrachromosomal%20DNA%20Shapes%20Urothelial%20Cancer%20Evolution&amp;author=DD%20Nguyen&amp;publication_year=2023&amp;pages=2023.05.07.538753&amp;doi=10.1101/2023.05.07.538753&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R52">
<span class="label">52.</span><cite>Jiang H, Lei R, Ding S-W &amp; Zhu S
Skewer: a fast and accurate adapter trimmer for next-generation sequencing paired-end reads. BMC Bioinformatics
15, 182 (2014).
</cite> [<a href="https://doi.org/10.1186/1471-2105-15-182" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4074385/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24925680/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=Skewer:%20a%20fast%20and%20accurate%20adapter%20trimmer%20for%20next-generation%20sequencing%20paired-end%20reads&amp;author=H%20Jiang&amp;author=R%20Lei&amp;author=S-W%20Ding&amp;author=S%20Zhu&amp;volume=15&amp;publication_year=2014&amp;pages=182&amp;pmid=24925680&amp;doi=10.1186/1471-2105-15-182&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R53">
<span class="label">53.</span><cite>Li H &amp; Durbin R
Fast and accurate short read alignment with Burrows-Wheeler transform. Bioinforma. Oxf. Engl. 25, 1754–1760 (2009).</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2705234/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19451168/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinforma.%20Oxf.%20Engl&amp;title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H%20Li&amp;author=R%20Durbin&amp;volume=25&amp;publication_year=2009&amp;pages=1754-1760&amp;pmid=19451168&amp;doi=10.1093/bioinformatics/btp324&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R54">
<span class="label">54.</span><cite>NovoSort | Novocraft. <a href="https://www.novocraft.com/documentation/novosort-2/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.novocraft.com/documentation/novosort-2/</a>.</cite>
</li>
<li id="R55">
<span class="label">55.</span><cite>Bs P &amp; Ar Q
Mosdepth: quick coverage calculation for genomes and exomes. Bioinforma. Oxf. Engl. 34, (2018).</cite> [<a href="https://doi.org/10.1093/bioinformatics/btx699" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6030888/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29096012/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinforma.%20Oxf.%20Engl&amp;title=Mosdepth:%20quick%20coverage%20calculation%20for%20genomes%20and%20exomes&amp;author=P%20Bs&amp;author=Q%20Ar&amp;volume=34&amp;publication_year=2018&amp;pmid=29096012&amp;doi=10.1093/bioinformatics/btx699&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R56">
<span class="label">56.</span><cite>Martin M
Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet.journal
17, 10–12 (2011).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=EMBnet.journal&amp;title=Cutadapt%20removes%20adapter%20sequences%20from%20high-throughput%20sequencing%20reads&amp;author=M%20Martin&amp;volume=17&amp;publication_year=2011&amp;pages=10-12&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R57">
<span class="label">57.</span><cite>DePristo MA
et al. 
A framework for variation discovery and genotyping using next-generation DNA sequencing data. Nat. Genet. 43, 491–498 (2011).
</cite> [<a href="https://doi.org/10.1038/ng.806" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3083463/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21478889/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&amp;title=A%20framework%20for%20variation%20discovery%20and%20genotyping%20using%20next-generation%20DNA%20sequencing%20data&amp;author=MA%20DePristo&amp;volume=43&amp;publication_year=2011&amp;pages=491-498&amp;pmid=21478889&amp;doi=10.1038/ng.806&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R58">
<span class="label">58.</span><cite>Lai D, Ha G &amp; Shah S
HMMcopy: Copy number prediction with correction for GC and mappability bias for HTS data. Bioconductor version: Release (3.15)
10.18129/B9.bioc.HMMcopy (2022).</cite> [<a href="https://doi.org/10.18129/B9.bioc.HMMcopy" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioconductor%20version:%20Release%20(3.15)&amp;title=HMMcopy:%20Copy%20number%20prediction%20with%20correction%20for%20GC%20and%20mappability%20bias%20for%20HTS%20data&amp;author=D%20Lai&amp;author=G%20Ha&amp;author=S%20Shah&amp;publication_year=2022&amp;doi=10.18129/B9.bioc.HMMcopy&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R59">
<span class="label">59.</span><cite>Amemiya HM, Kundaje A &amp; Boyle AP
The ENCODE Blacklist: Identification of Problematic Regions of the Genome. Sci. Rep. 9, 9354 (2019).
</cite> [<a href="https://doi.org/10.1038/s41598-019-45839-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6597582/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31249361/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep&amp;title=The%20ENCODE%20Blacklist:%20Identification%20of%20Problematic%20Regions%20of%20the%20Genome&amp;author=HM%20Amemiya&amp;author=A%20Kundaje&amp;author=AP%20Boyle&amp;volume=9&amp;publication_year=2019&amp;pages=9354&amp;pmid=31249361&amp;doi=10.1038/s41598-019-45839-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R60">
<span class="label">60.</span><cite>Kent WJ
et al. 
The Human Genome Browser at UCSC. Genome Res. 12, 996–1006 (2002).
</cite> [<a href="https://doi.org/10.1101/gr.229102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC186604/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12045153/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res&amp;title=The%20Human%20Genome%20Browser%20at%20UCSC&amp;author=WJ%20Kent&amp;volume=12&amp;publication_year=2002&amp;pages=996-1006&amp;pmid=12045153&amp;doi=10.1101/gr.229102&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R61">
<span class="label">61.</span><cite>Karczewski KJ
et al. 
The mutational constraint spectrum quantified from variation in 141,456 humans. Nature
581, 434–443 (2020).
</cite> [<a href="https://doi.org/10.1038/s41586-020-2308-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7334197/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32461654/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20mutational%20constraint%20spectrum%20quantified%20from%20variation%20in%20141,456%20humans&amp;author=KJ%20Karczewski&amp;volume=581&amp;publication_year=2020&amp;pages=434-443&amp;pmid=32461654&amp;doi=10.1038/s41586-020-2308-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R62">
<span class="label">62.</span><cite>Rosenthal R, McGranahan N, Herrero J, Taylor BS &amp; Swanton C
deconstructSigs: delineating mutational processes in single tumors distinguishes DNA repair deficiencies and patterns of carcinoma evolution. Genome Biol. 17, 31 (2016).
</cite> [<a href="https://doi.org/10.1186/s13059-016-0893-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4762164/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26899170/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=deconstructSigs:%20delineating%20mutational%20processes%20in%20single%20tumors%20distinguishes%20DNA%20repair%20deficiencies%20and%20patterns%20of%20carcinoma%20evolution&amp;author=R%20Rosenthal&amp;author=N%20McGranahan&amp;author=J%20Herrero&amp;author=BS%20Taylor&amp;author=C%20Swanton&amp;volume=17&amp;publication_year=2016&amp;pages=31&amp;pmid=26899170&amp;doi=10.1186/s13059-016-0893-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><span>Supplementary Tables</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12077166/bin/NIHMS2073968-supplement-Supplementary_Tables.xlsx" data-ga-action="click_feat_suppl" class="usa-link">NIHMS2073968-supplement-Supplementary_Tables.xlsx</a><sup> (69.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><span>Supplementary Note and Supplementary Figures</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12077166/bin/NIHMS2073968-supplement-Supplementary_Note_and_Supplementary_Figures.pdf" data-ga-action="click_feat_suppl" class="usa-link">NIHMS2073968-supplement-Supplementary_Note_and_Supplementary_Figures.pdf</a><sup> (2.2MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The raw genomic sequencing data generated is available from the European Genome-Phenome Archive (EGA) under dataset accession code EGAD50000001234. Datasets obtained from the Pan Cancer Analysis of Whole Genomes Consortium (<a href="#SD1" class="usa-link">Supplementary Table 11</a>) are available at <a href="https://www.icgc-argo.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.icgc-argo.org/</a>. Urothelial cancer tumor/normal alignment files were obtained from Nguyen et al.<sup><a href="#R51" class="usa-link" aria-describedby="R51">51</a></sup> and were deposited to dbGap under accession number phs001087.v4.p1.</p></section></section></section></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41592-025-02648-9"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/nihms-2073968.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12077166/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12077166/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12077166%2F%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12077166/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12077166/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12077166/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40217113/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12077166/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40217113/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12077166/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12077166/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="6GdSeLWYNGLqCgHubkYVyheU24s0HU1UMN83KHp1CmK65xpIvjwSCto9ykPFpBd4">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-B7-tgZZA.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-BLbQGH8u.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-DKf1UY8S.js"></script>
    
    

    </body>
</html>
