
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/base_style-CPPDhfna.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-BepKMsUl.js"></script>

  <link rel="stylesheet" href="/static/assets/article_style-DUfI1jZp.css" />
<link rel="stylesheet" href="/static/assets/style-BOrsu84o.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-D3wJHomh.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Prediction of adverse drug reactions based on pharmacogenomics combination features: a preliminary study - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="E0CA2433930AF7E3002433003432BB07.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="frontpharmacol">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11931068/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Frontiers in Pharmacology">
<meta name="citation_title" content="Prediction of adverse drug reactions based on pharmacogenomics combination features: a preliminary study">
<meta name="citation_author" content="Mingxiu He">
<meta name="citation_author_institution" content="College of Medical Information and Engineering, Guangdong Pharmaceutical University, Guangzhou, China">
<meta name="citation_author_institution" content="Department of Information, Guangdong Provincial Key Laboratory of Major Obstetric Diseases, Guangdong Provincial Clinical Research Center for Obstetrics and Gynecology, The Third Affiliated Hospital, Guangzhou Medical University, Guangzhou, China">
<meta name="citation_author" content="Yiyang Shi">
<meta name="citation_author_institution" content="College of Medical Information and Engineering, Guangdong Pharmaceutical University, Guangzhou, China">
<meta name="citation_author" content="Fangfang Han">
<meta name="citation_author_institution" content="College of Medical Information and Engineering, Guangdong Pharmaceutical University, Guangzhou, China">
<meta name="citation_author_institution" content="NMPA Key Laboratory for Technology Research and Evaluation of Pharmacovigilance, Guangzhou, China">
<meta name="citation_author_institution" content="Guangdong Provincial Traditional Chinese Medicine Precision Medicine Big Data Engineering Technology Research Center, Guangzhou, China">
<meta name="citation_author" content="Yongming Cai">
<meta name="citation_author_institution" content="College of Medical Information and Engineering, Guangdong Pharmaceutical University, Guangzhou, China">
<meta name="citation_author_institution" content="NMPA Key Laboratory for Technology Research and Evaluation of Pharmacovigilance, Guangzhou, China">
<meta name="citation_author_institution" content="Guangdong Provincial Traditional Chinese Medicine Precision Medicine Big Data Engineering Technology Research Center, Guangzhou, China">
<meta name="citation_publication_date" content="2025 Mar 10">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="1448106">
<meta name="citation_doi" content="10.3389/fphar.2025.1448106">
<meta name="citation_pmid" content="40129949">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11931068/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11931068/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11931068/pdf/fphar-16-1448106.pdf">
<meta name="description" content="Adverse Drug Reactions (ADRs), a widespread phenomenon in clinical drug treatment, are often associated with a high risk of morbidity and even death. Drugs and changes in gene expression are the two important factors that affect whether and how ...">
<meta name="og:title" content="Prediction of adverse drug reactions based on pharmacogenomics combination features: a preliminary study">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Adverse Drug Reactions (ADRs), a widespread phenomenon in clinical drug treatment, are often associated with a high risk of morbidity and even death. Drugs and changes in gene expression are the two important factors that affect whether and how ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11931068/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://pmc.ncbi.nlm.nih.gov/search/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="11931068">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.3389/fphar.2025.1448106"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/fphar-16-1448106.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11931068%2F%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/11931068/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/11931068/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11931068/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-frontpharmacol.gif" alt="Frontiers in Pharmacology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Frontiers in Pharmacology" title="Link to Frontiers in Pharmacology" shape="default" href="http://www.frontiersin.org/pharmacology" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Front Pharmacol</button></div>. 2025 Mar 10;16:1448106. doi: <a href="https://doi.org/10.3389/fphar.2025.1448106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.3389/fphar.2025.1448106</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://pmc.ncbi.nlm.nih.gov/search/?term=%22Front%20Pharmacol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Front%20Pharmacol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Front%20Pharmacol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Front%20Pharmacol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Prediction of adverse drug reactions based on pharmacogenomics combination features: a preliminary study</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22He%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Mingxiu He</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Mingxiu He</span></h3>
<div class="p">
<sup>1</sup>
College of Medical Information and Engineering, Guangdong Pharmaceutical University, Guangzhou, China
</div>
<div class="p">
<sup>2</sup>
Department of Information, Guangdong Provincial Key Laboratory of Major Obstetric Diseases, Guangdong Provincial Clinical Research Center for Obstetrics and Gynecology, The Third Affiliated Hospital, Guangzhou Medical University, Guangzhou, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22He%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mingxiu He</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shi%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Yiyang Shi</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Yiyang Shi</span></h3>
<div class="p">
<sup>1</sup>
College of Medical Information and Engineering, Guangdong Pharmaceutical University, Guangzhou, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shi%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yiyang Shi</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Han%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Fangfang Han</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Fangfang Han</span></h3>
<div class="p">
<sup>1</sup>
College of Medical Information and Engineering, Guangdong Pharmaceutical University, Guangzhou, China
</div>
<div class="p">
<sup>3</sup>
NMPA Key Laboratory for Technology Research and Evaluation of Pharmacovigilance, Guangzhou, China
</div>
<div class="p">
<sup>4</sup>
Guangdong Provincial Traditional Chinese Medicine Precision Medicine Big Data Engineering Technology Research Center, Guangzhou, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Han%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Fangfang Han</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>4,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cai%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Yongming Cai</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Yongming Cai</span></h3>
<div class="p">
<sup>1</sup>
College of Medical Information and Engineering, Guangdong Pharmaceutical University, Guangzhou, China
</div>
<div class="p">
<sup>3</sup>
NMPA Key Laboratory for Technology Research and Evaluation of Pharmacovigilance, Guangzhou, China
</div>
<div class="p">
<sup>4</sup>
Guangdong Provincial Traditional Chinese Medicine Precision Medicine Big Data Engineering Technology Research Center, Guangzhou, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cai%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yongming Cai</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>4,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>1</sup>
College of Medical Information and Engineering, Guangdong Pharmaceutical University, Guangzhou, China
</div>
<div id="aff2">
<sup>2</sup>
Department of Information, Guangdong Provincial Key Laboratory of Major Obstetric Diseases, Guangdong Provincial Clinical Research Center for Obstetrics and Gynecology, The Third Affiliated Hospital, Guangzhou Medical University, Guangzhou, China
</div>
<div id="aff3">
<sup>3</sup>
NMPA Key Laboratory for Technology Research and Evaluation of Pharmacovigilance, Guangzhou, China
</div>
<div id="aff4">
<sup>4</sup>
Guangdong Provincial Traditional Chinese Medicine Precision Medicine Big Data Engineering Technology Research Center, Guangzhou, China
</div>
<div class="author-notes p">
<div class="fn" id="fn1"><p>
<strong>Edited by:</strong>
<a href="https://loop.frontiersin.org/people/2157329/overview" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Taravat Ghafourian</a>, Nova Southeastern University, United States</p></div>
<div class="fn" id="fn2">
<p>
<strong>Reviewed by:</strong>
<a href="https://loop.frontiersin.org/people/1469343/overview" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Fernando Prieto-Martínez</a>, National Autonomous University of Mexico, Mexico</p>
<p>
<a href="https://loop.frontiersin.org/people/357553/overview" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Patompong Satapornpong</a>, Rangsit University, Thailand</p>
</div>
<div class="fn" id="c001">
<sup>✉</sup><p class="display-inline">*Correspondence: Yongming Cai, <span>cym@gdpu.edu.cn</span>; Fangfang Han, <span>hanff@gdpu.edu.cn</span>
</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Jun 12; Accepted 2025 Feb 24; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Copyright © 2025 He, Shi, Han and Cai.</div>
<p>This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC11931068  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40129949/" class="usa-link">40129949</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Introduction</h3>
<p>Adverse Drug Reactions (ADRs), a widespread phenomenon in clinical drug treatment, are often associated with a high risk of morbidity and even death. Drugs and changes in gene expression are the two important factors that affect whether and how adverse reactions occur. Notably, pharmacogenomics data have recently become more available and could be used to predict ADR occurrence. However, there is a challenge in effectively analyzing the massive data lacking guidance on mutual relationship for ADRs prediction.</p></section><section id="sec2"><h3 class="pmc_sec_title">Methods</h3>
<p>We constructed separate similarity features for drugs and ADRs using pharmacogenomics data from the Comparative Toxicogenomics Database [CTD, including Chemical-Gene Interactions (CGIs) and Gene-Disease Associations (GDAs)]. We proposed a novel deep learning architecture, DGANet, based on the constructed features for ADR prediction. The algorithm uses Convolutional Neural Networks (CNN) and cross-features to learn the latent drug-gene-ADR associations for ADRs prediction.</p></section><section id="sec3"><h3 class="pmc_sec_title">Results and Discussion</h3>
<p>The performance of DGANet was compared to three state-of-the-art algorithms with different genomic features. According to the results, GDANet outperformed the benchmark algorithms (AUROC = 92.76%, AUPRC = 92.49%), demonstrating a 3.36% AUROC and 4.05% accuracy improvement over the cutting-edge algorithms. We further proposed new genomic features that improved DGANet’s predictive capability. Moreover, case studies on top-ranked candidates confirmed DGANet’s ability to predict new ADRs.</p></section><section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> adverse drug reactions, comparative toxicogenomics database, chemical-gene interactions, gene-disease associations, convolutional neural networks</p></section></section><section id="s1"><h2 class="pmc_sec_title">1 Introduction</h2>
<p>Adverse Drug Reactions (ADRs), commonly known as side effects (<a href="#B25" class="usa-link" aria-describedby="B25">Research, 2022</a>), have emerged as a major concern in public health and pharmacotherapy, imposing a substantial socio-economic burden along with severe incidence and mortality rates among patients during drug development. According to their mechanism of occurrence, ADRs can be divided into two main types: dose-related reactions (type A) and non-dose related or idiosynchrasic (type B), among which type A reactions are widely considered predictable (<a href="#B20" class="usa-link" aria-describedby="B20">Micaglio et al., 2021</a>). With the increasing availability of clinical and non-clinical data, computer algorithms have demonstrated the greatest utility in ADR prediction analysis. In the last 2 decades (2004∼2022), computer-based ADR predictions have primarily relied on the structural information of compounds (<a href="#B7" class="usa-link" aria-describedby="B7">Das and Mazumder, 2023</a>). However, the occurrence of Adverse Reactions (ARs) is not solely influenced by the structural information of the compound, but is also affected by the interaction between drugs or their intermediate metabolites and drug-effector gene-encoded proteins such as enzymes, receptors, ion channels, and the genes themselves. Studies have demonstrated that genetic variations in drug-metabolizing enzymes, drug transporters, and drug targets have a significant impact on changes in pharmacokinetics and pharmacodynamics of drugs (<a href="#B39" class="usa-link" aria-describedby="B39">Zhou et al., 2015</a>). Consequently, predicting ADRs solely based on the structural information of compounds may overlook critical information, potentially compromising the predictive performance of the model. According to recent investigations, pharmacogenomics accounts for ∼80% variability in drug pharmacokinetics and pharmacodynamics, as well as over 60% of ADRs (<a href="#B4" class="usa-link" aria-describedby="B4">Cacabelos et al., 2019</a>; <a href="#B22" class="usa-link" aria-describedby="B22">Pirmohamed, 2023</a>). For instance, HLA-pharmacogenomic markers are the main culprits that influence the mechanisms of immunopathogenesis of drug-induced severe cutaneous adverse drug reactions (SCARs) (<a href="#B26" class="usa-link" aria-describedby="B26">Satapornpong et al., 2024</a>). Currently the main clinical areas applying pharmacogenetic testing include hemolitc anaemias, malignant hyperthermia, porphyrias, severe skin disorders, Brugada and long QT syndromes (<a href="#B20" class="usa-link" aria-describedby="B20">Micaglio et al., 2021</a>; <a href="#B26" class="usa-link" aria-describedby="B26">Satapornpong et al., 2024</a>). Furthermore, changes in gene expression can often be detected prior to the emergence of histopathological changes or clinical signs (<a href="#B36" class="usa-link" aria-describedby="B36">Zhang et al., 2020</a>). This suggests that genes can serve as valuable predictive factors, providing early warnings and preventing the occurrence of ADRs, especially type B reactions (<a href="#B20" class="usa-link" aria-describedby="B20">Micaglio et al., 2021</a>). Consequently, integrating pharmacogenomic data with compound structural information into Machine Learning (ML) algorithms, rather than relying solely on compound structural information, could potentially enhance the timeliness and reliability of ADR predictions.</p>
<p>Several large-scale pharmacogenomics databases have recently become publicly available for research purposes, including the Library of Integrated Network-based Cellular Signatures (LINCS) L1000 project (<a href="#B30" class="usa-link" aria-describedby="B30">Subramanian et al., 2017</a>), Search Tool for Interactions of Chemicals (STITCH) (<a href="#B16" class="usa-link" aria-describedby="B16">Kuhn et al., 2008</a>), and the Comparative Toxicogenomics Database (CTD) (<a href="#B8" class="usa-link" aria-describedby="B8">Davis et al., 2023</a>). The LINCS L1000 project (<a href="#B30" class="usa-link" aria-describedby="B30">Subramanian et al., 2017</a>) profiled Gene Expression (GE) in cells treated with different dosages, with expressions assessed at various time points. The LINCS L1000 dataset has been widely used in recent studies to predict ADRs (<a href="#B33" class="usa-link" aria-describedby="B33">Wang et al., 2016</a>; <a href="#B32" class="usa-link" aria-describedby="B32">Üner et al., 2023</a>; <a href="#B18" class="usa-link" aria-describedby="B18">Li et al., 2024</a>) or Drug-Drug Interactions (DDIs) (<a href="#B24" class="usa-link" aria-describedby="B24">Raja et al., 2017</a>; <a href="#B28" class="usa-link" aria-describedby="B28">Shankar et al., 2021</a>). For instance, using the combination of the strongest GEs in LINCS L1000 and chemical structures of drugs, <a href="#B33" class="usa-link" aria-describedby="B33">Wang et al. (2016)</a> predicted ADRs using Extra Trees (ETs) (<a href="#B11" class="usa-link" aria-describedby="B11">Geurts et al., 2006</a>) classifiers (AUROC = 85.4%) and constructed an Adverse Drug Reaction-Gene Ontology (ADR-GO) network to link the most probable ARs predicted by the model to the relevant gene ontologies. Additionally, using the complete set of drug-perturbed Gene Expression Profiles (GEX) and their experimental Metadata (META), <a href="#B32" class="usa-link" aria-describedby="B32">Üner et al. (2023)</a> achieved a better predictive performance among five deep learning architectures, with Macro-AUC and Micro-AUC values of 79.0% and 87.7%, respectively. Given that the metadata is meaningless without GEs and could lead to numerous calculations, we chose the original GE feature for comparison in this study. STITCH is a resource for exploring known and predicted interactions of chemicals and proteins from 1,133 organisms. It integrates experimental, curated, and text-mined evidence, and users can filter their searches by tissue, affinity, and other criteria. <a href="#B3" class="usa-link" aria-describedby="B3">Bongini et al. (2023)</a> develop a model named DruGNN which constructed a graph to predict ADRs based on drug-protein interactions obtained from the STITCH database, and each protein was mapped to the gene from which it was derived. DruGNN achieved an accuracy of 86.3%. Recently, <a href="#B18" class="usa-link" aria-describedby="B18">Li et al. (2024)</a> proposed a novel model named BiMPADR, which integrated drug gene expression data extracted from the LINCS database into drug features and utilized gene–ADR associations extracted from the ADReCS-Target database (<a href="#B12" class="usa-link" aria-describedby="B12">Huang et al., 2018</a>) into ADR features to predict ADRs, achieving an AUC of 89.4%. While previous methods have demonstrated promising predictive outcomes, they exhibit limitations, including low AUROC scores, inability to apply to drugs with limited pre-existing information, and failure to consider both drug and ADR characteristics simultaneously. Moreover, these methods have not fully leveraged the potential of pharmacogenomics data, including the complex and diverse relationships between chemicals, genes, biomarkers, therapeutic targets, etc., rather than solely focusing on changes in gene expression. The CTD database houses a substantial amount of correlation data between chemicals, genes, phenotypes, and diseases. These data were meticulously organized and annotated by professional bioinformatics experts, ensuring data quality and accuracy. Despite the potential of pharmacogenomics data for ADR prediction, research in this area using the CTD database remains limited, possibly due to the challenges associated with data mapping. Consequently, there is still potential areas for advancement in ADR prediction utilizing pharmacogenomics data.</p>
<p>Although the above databases provide rich pharmacogenomics information, the enormous genetic data presents challenges to feature processing and model design for ADR prediction. Therefore, more efficient feature analysis is required to enhance the performance of prediction models. To achieve this goal, we creatively combined chemical structure descriptors, ADR semantic descriptors, and three different genomic descriptors [drug-perturbed GE changes, Chemical-Gene Interactions (CGIs), and Gene-Disease Associations (GDAs)] from various databases to establish drug-genomic-ADR relationships that can be used to train a Convolutional Neural Network (CNN)-based model for predicting drug-ADR associations.</p>
<p>Herein, we first constructed a benchmark validation dataset and five different features and then introduced a new deep learning method for ADR prediction. Subsequently, the ablation experiment validated the effectiveness of our proposed pharmacogenomics features, and additional case studies further demonstrated the practicality of our model as a predictor of novel ADRs.</p></section><section id="s2"><h2 class="pmc_sec_title">2 Materials and methods</h2>
<section id="s2-1"><h3 class="pmc_sec_title">2.1 Benchmark datasets constructed using known drug-ADR relations</h3>
<p>The experimental benchmark datasets used in this study were from five public databases: Side Effect Resource 4.1 (SIDER) (<a href="#B15" class="usa-link" aria-describedby="B15">Kuhn et al., 2016</a>), LINCS L1000 (<a href="#B30" class="usa-link" aria-describedby="B30">Subramanian et al., 2017</a>), CTD (<a href="#B8" class="usa-link" aria-describedby="B8">Davis et al., 2023</a>), PubChem (<a href="#B13" class="usa-link" aria-describedby="B13">Kim et al., 2019</a>), and the US National Library of Medicine’s Medical Subject Headings (MeSH) (<a href="#B10" class="usa-link" aria-describedby="B10">Fernandez-Llimos et al., 2017</a>). In summary, SIDER was used to extract benchmark drug-ADR pairs, LINCS L1000, CTD and PubChem were employed to extract drug characteristics, MESH and CTD were utilized to extract ADR characteristics (<a href="#F1" class="usa-link">Figure 1</a>). In order to generate better results, drugs that are simultaneously recorded in CTD, PubChem, LINCS L1000, and SIDER databases and ADRs that are simultaneously recorded in CTD, MESH, and SIDER databases were included in this study. And all drugs or ADRs without pharmacogenomics data were also excluded. The details of data sets before and after processing in this study are shown in <a href="#T1" class="usa-link">Table 1</a>; <a href="#F1" class="usa-link">Figure 1</a>.</p>
<figure class="fig xbox font-sm" id="F1"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11931068_fphar-16-1448106-g001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/97c2/11931068/2426f9ef5d6b/fphar-16-1448106-g001.jpg" loading="lazy" height="581" width="659" alt="FIGURE 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/F1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Data resources. Figures outside parentheses represent the number of original records downloaded from the databases, whereas figures inside parentheses represent the number of processed records finally used in the experiments.</p></figcaption></figure><section class="tw xbox font-sm" id="T1"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Details of datasets before and after processing.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead valign="top">
<tr>
<th rowspan="2" align="center" colspan="1">DataBase</th>
<th colspan="3" align="center" rowspan="1">Initial</th>
<th colspan="3" align="center" rowspan="1">Processed</th>
</tr>
<tr>
<th align="center" rowspan="1" colspan="1">Drugs</th>
<th align="center" rowspan="1" colspan="1">ADRs</th>
<th align="center" rowspan="1" colspan="1">Drug-ADR pairs</th>
<th align="center" rowspan="1" colspan="1">Drugs</th>
<th align="center" rowspan="1" colspan="1">ADRs</th>
<th align="center" rowspan="1" colspan="1">Drug-ADR pairs</th>
</tr>
</thead>
<tbody valign="top">
<tr>
<td align="left" rowspan="1" colspan="1">CTD</td>
<td align="center" rowspan="1" colspan="1">175,287</td>
<td align="center" rowspan="1" colspan="1">13,183</td>
<td align="center" rowspan="1" colspan="1">—</td>
<td align="center" rowspan="1" colspan="1">453</td>
<td align="center" rowspan="1" colspan="1">1,019</td>
<td align="center" rowspan="1" colspan="1">—</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">LINC L1000</td>
<td align="center" rowspan="1" colspan="1">41,774</td>
<td align="center" rowspan="1" colspan="1">—</td>
<td align="center" rowspan="1" colspan="1">—</td>
<td align="center" rowspan="1" colspan="1">453</td>
<td align="center" rowspan="1" colspan="1">—</td>
<td align="center" rowspan="1" colspan="1">—</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SIDER</td>
<td align="center" rowspan="1" colspan="1">1,430</td>
<td align="center" rowspan="1" colspan="1">5,868</td>
<td align="center" rowspan="1" colspan="1">139,756</td>
<td align="center" rowspan="1" colspan="1">453</td>
<td align="center" rowspan="1" colspan="1">1,019</td>
<td align="center" rowspan="1" colspan="1">23,395</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>The SIDER database is widely used for validating ADRs, and its current version contains 1,430 marketed drugs, 5,868 side effects, and 139,756 Drug-Side Effect Associations (DSAs). In SIDER database, drug terms are coded in STITCH compound IDs, which are also deformations of PubChem compound IDs. The compound IDs can be obtained by removing the prefixes. The Simplified Molecular-Input Line-Entry System (SMILES) strings and synonyms for all the drugs were bulk downloaded from PubChem using these compound IDs. Furthermore, the Chemical-Gene Interaction (CGI) and Gene-Disease Association (GDA) profiles were downloaded from CTD, including 175,287 drugs, 2,418,814 CGIs, 13,183 diseases, and 105,753,621 GDAs. The CGIs and GDAs were downloaded from CTD in March 2024. To reduce the dimensionality of data processing, CGI records were deduplicated using ChemicalID and GeneSymbol, irrespective of the Organism, Interaction, InteractionActions and PubMedIDs within the records. Similarly, the GDA records were deduplicated GeneSymbol and DiseaseID, irrespective of DirectEvidence, InferenceChemicalName, InferenceScore, OmimIDs, PubMedIDs within the records. Additionally, GE signatures for drugs/small molecule compounds in the landmark gene space were downloaded from <a href="http://maayanlab.net" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">maayanlab.net</a>, originally processed from the LINCS L1000 database (<a href="#B30" class="usa-link" aria-describedby="B30">Subramanian et al., 2017</a>).</p>
<p>The five databases use different vocabularies to encode their drugs and ARs, which we mapped in various ways. The medications in CTD, PubChem, and LINCS L1000 which contained one or more SMILES were mapped using a 166-bit MACCS (Molecular ACCess System) fingerprint (<a href="#B9" class="usa-link" aria-describedby="B9">Durant et al., 2002</a>), which can be converted from any format of SMILES using the Python RDKit package (<a href="#B17" class="usa-link" aria-describedby="B17">Landrum, 2024</a>). The few drugs that could not be mapped with MACCS were mapped using names and synonyms retrieved from PubChem and CTD. The ADR terms in SIDER were mapped to Preferred Terms (PTs) coded in MedDRA v16.0. The MEDIC disease vocabulary of CTD is a modified subset of descriptors from the “Diseases” branch of MeSH combined with genetic disorders from the Online Mendelian Inheritance in Man (OMIM) database (<a href="#B1" class="usa-link" aria-describedby="B1">Amberger and Hamosh, 2017</a>). First, we downloaded MedDRA, SNOMED, and MeSH vocabularies from the Observational Medical Outcomes Partnership (OMOP) database (<a href="#B27" class="usa-link" aria-describedby="B27">Sedlmayr et al., 2024</a>). The MedDRA code was then mapped to the MeSH code using a standard concept ID applied in OMOP. Second, we downloaded the disease files from the Human Disease Ontology (HDO) database (<a href="#B14" class="usa-link" aria-describedby="B14">Köhler et al., 2017</a>), which contains the names, synonyms, and IDs linked to other data sources, including SNOMED, MeSH, and the Unified Medical Language System (UMLS) (<a href="#B2" class="usa-link" aria-describedby="B2">Bodenreider, 2004</a>), among others. The MedDRA terms in SIDER were matched to MeSH terms in CTD as long as they shared at least one linked ID. Not all ADRs can be mapped using linked IDs, hence, they were subsequently mapped using names and synonyms herein. Those that could not be mapped to MeSH were excluded from our study.</p>
<p>Among all the five attribute data sources, 453 drugs and 1,091 ADRs were finally used in our experiments, comprising 23,395 known drug-ADR pairs, 101,257 drug-gene interactions with 23,644 genes, and 15,087,041 GDAs with 53,968 genes. We constructed an algorithm for AR prediction using the multi-source data available on these management platforms.</p></section><section id="s2-2"><h3 class="pmc_sec_title">2.2 Construction of drug similarity features</h3>
<section id="s2-2-1"><h4 class="pmc_sec_title">2.2.1 Drug similarity based on chemical structure</h4>
<p>The SMILES string representation for each drug structure was obtained from PubChem and then converted to topological fingerprints using the Python RDKit package (<a href="#B17" class="usa-link" aria-describedby="B17">Landrum, 2024</a>). Topological fingerprints are binary codes based on the topological configuration and rotational angles of quaternary rings within molecular structures. They are used to characterize molecule stereochemistry and potential interactions. Herein, the drug similarity matrices based on the Chemical Structure (CS) were expressed as: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf1">
<math id="m1" overflow="linebreak"><mrow><msubsup><mrow><mi>S</mi><mi>i</mi><mi>m</mi></mrow><mrow><mi>c</mi><mi>s</mi></mrow><mrow><mi>d</mi><mi>r</mi><mi>u</mi><mi>g</mi><mo>−</mo><mi>d</mi><mi>r</mi><mi>u</mi><mi>g</mi></mrow></msubsup></mrow></math>
</span>; and the Tanimoto Coefficient was used to measure the similarity score of each drug pair. The formula for calculating the similarity scores of drugs i and j was as follows:</p>
<table class="disp-formula p" id="equ1"><tr><td class="formula"><math id="m2" display="block" overflow="linebreak"><mrow><msubsup><mrow><mi mathvariant="bold-italic">S</mi><mi mathvariant="bold-italic">i</mi><mi mathvariant="bold-italic">m</mi></mrow><mrow><mi mathvariant="bold-italic">c</mi><mi mathvariant="bold-italic">s</mi><mrow><mfenced open="(" close=")" separators="|"><mrow><mi mathvariant="bold-italic">i</mi><mo>,</mo><mi mathvariant="bold-italic">j</mi></mrow></mfenced></mrow></mrow><mrow><mi mathvariant="bold-italic">d</mi><mi mathvariant="bold-italic">r</mi><mi mathvariant="bold-italic">u</mi><mi mathvariant="bold-italic">g</mi><mo>−</mo><mi mathvariant="bold-italic">d</mi><mi mathvariant="bold-italic">r</mi><mi mathvariant="bold-italic">u</mi><mi mathvariant="bold-italic">g</mi></mrow></msubsup><mo>=</mo><mfrac><mrow><msubsup><mi mathvariant="bold-italic">x</mi><mi mathvariant="bold-italic">i</mi><mrow><mi mathvariant="bold-italic">c</mi><mi mathvariant="bold-italic">s</mi></mrow></msubsup><mo>∙</mo><msubsup><mi mathvariant="bold-italic">x</mi><mi mathvariant="bold-italic">j</mi><mrow><mi mathvariant="bold-italic">c</mi><mi mathvariant="bold-italic">s</mi></mrow></msubsup></mrow><mrow><msup><mrow><mfenced open="‖" close="‖" separators="|"><mrow><msubsup><mi mathvariant="bold-italic">x</mi><mi mathvariant="bold-italic">i</mi><mrow><mi mathvariant="bold-italic">c</mi><mi mathvariant="bold-italic">s</mi></mrow></msubsup></mrow></mfenced></mrow><mn mathvariant="bold">2</mn></msup><mo>+</mo><msup><mrow><mfenced open="‖" close="‖" separators="|"><mrow><msubsup><mi mathvariant="bold-italic">x</mi><mi mathvariant="bold-italic">j</mi><mrow><mi mathvariant="bold-italic">c</mi><mi mathvariant="bold-italic">s</mi></mrow></msubsup></mrow></mfenced></mrow><mn mathvariant="bold">2</mn></msup><mo>−</mo><msubsup><mi mathvariant="bold-italic">x</mi><mi mathvariant="bold-italic">i</mi><mrow><mi mathvariant="bold-italic">c</mi><mi mathvariant="bold-italic">s</mi></mrow></msubsup><mo>∙</mo><msubsup><mi mathvariant="bold-italic">x</mi><mi mathvariant="bold-italic">j</mi><mrow><mi mathvariant="bold-italic">c</mi><mi mathvariant="bold-italic">s</mi></mrow></msubsup></mrow></mfrac></mrow></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf2">
<math id="m3" overflow="linebreak"><mrow><msubsup><mi>x</mi><mi>i</mi><mrow><mi>c</mi><mi>s</mi></mrow></msubsup></mrow></math>
</span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf3">
<math id="m4" overflow="linebreak"><mrow><msubsup><mrow><mtext> </mtext><mi>x</mi></mrow><mi>j</mi><mrow><mi>c</mi><mi>s</mi></mrow></msubsup></mrow></math>
</span> are the topological fingerprint representations of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf4">
<math id="m5" overflow="linebreak"><mrow><msub><mrow><mi>d</mi><mi>r</mi><mi>u</mi><mi>g</mi></mrow><mi>i</mi></msub></mrow></math>
</span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf5">
<math id="m6" overflow="linebreak"><mrow><msub><mrow><mi>d</mi><mi>r</mi><mi>u</mi><mi>g</mi></mrow><mi>j</mi></msub></mrow></math>
</span>, respectively.</p></section><section id="s2-2-2"><h4 class="pmc_sec_title">2.2.2 Drug similarity based on GE changes before and after drug perturbations</h4>
<p>Herein, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf6">
<math id="m7" overflow="linebreak"><mrow><msubsup><mrow><mi>S</mi><mi>i</mi><mi>m</mi></mrow><mrow><mi>G</mi><mi>E</mi></mrow><mrow><mi>d</mi><mi>r</mi><mi>u</mi><mi>g</mi><mo>−</mo><mi>d</mi><mi>r</mi><mi>u</mi><mi>g</mi></mrow></msubsup></mrow></math>
</span> was defined as the drug similarity matrix at the GE level. Based on works of <a href="#B33" class="usa-link" aria-describedby="B33">Wang et al. (2016)</a>, we collected GE signature profiles perturbed by drugs/small molecule compounds from <a href="http://maayanlab.net" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">maayanlab.net</a>. The Phase 1 experiment data of LINCS L1000 (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92742" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE92742</a>), in which a variable named “distil_ss” denotes the signature strength of every experiment, was the source data of the GE features. The larger the “distil_ss” variable, the more differentially expressed the landmark genes are within a signature. This approach quantifies the magnitude of the differential expression of landmark genes when comparing the average drug treatment to the DMSO treatment in the LINCS L1000 dataset. The “distil_ss” values were computed using the Characteristic Direction (CD) method (<a href="#B6" class="usa-link" aria-describedby="B6">Clark et al., 2014</a>), which generates gene expression signatures for drug perturbations in the 978 landmark gene space. The cosine similarity measure was used to compute the drug similarity based on gene expression changes. The formula was as follows:</p>
<table class="disp-formula p" id="equ2"><tr><td class="formula"><math id="m8" display="block" overflow="linebreak"><mrow><msubsup><mrow><mi mathvariant="bold-italic">S</mi><mi mathvariant="bold-italic">i</mi><mi mathvariant="bold-italic">m</mi></mrow><mrow><mi mathvariant="bold-italic">G</mi><mi mathvariant="bold-italic">E</mi><mrow><mfenced open="(" close=")" separators="|"><mrow><mi mathvariant="bold-italic">i</mi><mo>,</mo><mi mathvariant="bold-italic">j</mi></mrow></mfenced></mrow></mrow><mrow><mi mathvariant="bold-italic">d</mi><mi mathvariant="bold-italic">r</mi><mi mathvariant="bold-italic">u</mi><mi mathvariant="bold-italic">g</mi><mo>−</mo><mi mathvariant="bold-italic">d</mi><mi mathvariant="bold-italic">r</mi><mi mathvariant="bold-italic">u</mi><mi mathvariant="bold-italic">g</mi></mrow></msubsup><mo>=</mo><mfrac><mrow><msubsup><mi mathvariant="bold-italic">x</mi><mi mathvariant="bold-italic">i</mi><mrow><mi mathvariant="bold-italic">G</mi><mi mathvariant="bold-italic">E</mi></mrow></msubsup><mo>·</mo><mtext> </mtext><msubsup><mi mathvariant="bold-italic">x</mi><mi mathvariant="bold-italic">j</mi><mrow><mi mathvariant="bold-italic">G</mi><mi mathvariant="bold-italic">E</mi></mrow></msubsup></mrow><mrow><mrow><mfenced open="‖" close="‖" separators="|"><mrow><msubsup><mi mathvariant="bold-italic">x</mi><mi mathvariant="bold-italic">i</mi><mrow><mi mathvariant="bold-italic">G</mi><mi mathvariant="bold-italic">E</mi></mrow></msubsup></mrow></mfenced></mrow><mrow><mfenced open="‖" close="‖" separators="|"><mrow><msubsup><mi mathvariant="bold-italic">x</mi><mi mathvariant="bold-italic">j</mi><mrow><mi mathvariant="bold-italic">G</mi><mi mathvariant="bold-italic">E</mi></mrow></msubsup></mrow></mfenced></mrow></mrow></mfrac></mrow></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf7">
<math id="m9" overflow="linebreak"><mrow><msubsup><mi>x</mi><mi>i</mi><mrow><mi>G</mi><mi>E</mi></mrow></msubsup></mrow></math>
</span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf8">
<math id="m10" overflow="linebreak"><mrow><msubsup><mi>x</mi><mi>j</mi><mrow><mi>G</mi><mi>E</mi></mrow></msubsup></mrow></math>
</span> are the differential gene expressions of perturbations of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf9">
<math id="m11" overflow="linebreak"><mrow><msub><mrow><mi>d</mi><mi>r</mi><mi>u</mi><mi>g</mi></mrow><mrow><mi>i</mi><mtext> </mtext></mrow></msub></mrow></math>
</span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf10">
<math id="m12" overflow="linebreak"><mrow><msub><mrow><mi>d</mi><mi>r</mi><mi>u</mi><mi>g</mi></mrow><mi>j</mi></msub></mrow></math>
</span>, respectively.</p></section><section id="s2-2-3"><h4 class="pmc_sec_title">2.2.3 Drug similarity based on GE changes before and after drug perturbations</h4>
<p>Herein, CGIs were downloaded from CTD [<a href="http://ctdbase.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">ctdbase.org</a> (last updated on 29th November 2023], which curates specific chemical-gene and chemical-protein interactions in vertebrates and invertebrates from published literature. Each CGI was quantified based on four degrees: Increases (e.g., “Chemical X increases the expression of Gene Y mRNA”), decreases, affects, or does not affect. Interactions with the “does not affect” degree were excluded from CTD. There were also numerous indirect CGIs [e.g., “Chemical X inhibits the reaction (protein P results in the increased expression of Gene Y)”]. The variable “InteractionAction” which has 3,431 distinct values, was used to categorize the interactions.</p>
<p>Data dimensionality was first lowered by encoding drug-gene interactions as one-hot vectors without considering cell types and interaction degrees to reduce computational complexity. Briefly, for each drug, all genes that interact or do not interact with it were labeled as 1 and 0, respectively. Subsequently, gene sets <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf11">
<math id="m13" overflow="linebreak"><mrow><msub><mrow><mi>G</mi><mi>T</mi></mrow><mi>i</mi></msub><mo>=</mo><mrow><mfenced open="{" close="}" separators="|"><mrow><msub><mi>g</mi><mrow><mi>i</mi><mn>1</mn></mrow></msub><mo>,</mo><msub><mi>g</mi><mrow><mi>i</mi><mn>2</mn></mrow></msub><mo>,</mo><msub><mi>g</mi><mrow><mi>i</mi><mn>3</mn></mrow></msub><mo>,</mo><mo>…</mo><mo>,</mo><msub><mi>g</mi><mrow><mi>i</mi><mi>n</mi></mrow></msub></mrow></mfenced></mrow></mrow></math>
</span> which contain n genes that interact with drug <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf12">
<math id="m14" overflow="linebreak"><mrow><msub><mi>d</mi><mi>i</mi></msub></mrow></math>
</span> and gene sets <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf13">
<math id="m15" overflow="linebreak"><mrow><msub><mrow><mi>G</mi><mi>T</mi></mrow><mi>j</mi></msub><mo>=</mo><mrow><mfenced open="{" close="}" separators="|"><mrow><msub><mi>g</mi><mrow><mi>j</mi><mn>1</mn></mrow></msub><mo>,</mo><msub><mi>g</mi><mrow><mi>j</mi><mn>2</mn></mrow></msub><mo>,</mo><msub><mi>g</mi><mrow><mi>j</mi><mn>3</mn></mrow></msub><mo>,</mo><mo>…</mo><mo>,</mo><msub><mi>g</mi><mrow><mi>j</mi><mi>m</mi></mrow></msub></mrow></mfenced></mrow></mrow></math>
</span> which contain m genes that interact with drug <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf14">
<math id="m16" overflow="linebreak"><mrow><msub><mi>d</mi><mi>j</mi></msub></mrow></math>
</span> were obtained. The more identical the genes that interact with the two drugs (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf15">
<math id="m17" overflow="linebreak"><mrow><msub><mi>d</mi><mi>i</mi></msub></mrow></math>
</span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf16">
<math id="m18" overflow="linebreak"><mrow><msub><mi>d</mi><mi>j</mi></msub></mrow></math>
</span>), the higher their similarity. In other words, the similarity between the two drugs, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf17">
<math id="m19" overflow="linebreak"><mrow><msub><mi>d</mi><mi>i</mi></msub></mrow></math>
</span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf18">
<math id="m20" overflow="linebreak"><mrow><msub><mi>d</mi><mi>j</mi></msub><mo>,</mo></mrow></math>
</span> can be quantitatively determined using the intersection and union ratio of the two gene sets, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf19">
<math id="m21" overflow="linebreak"><mrow><msub><mrow><mi>G</mi><mi>T</mi></mrow><mi>i</mi></msub></mrow></math>
</span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf20">
<math id="m22" overflow="linebreak"><mrow><msub><mrow><mi>G</mi><mi>T</mi></mrow><mi>j</mi></msub></mrow></math>
</span>. The Jaccard index was used to calculate the drug similarity <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf21">
<math id="m23" overflow="linebreak"><mrow><msubsup><mrow><mi>S</mi><mi>i</mi><mi>m</mi></mrow><mrow><mi>C</mi><mi>G</mi><mi>I</mi></mrow><mrow><mi>d</mi><mi>r</mi><mi>u</mi><mi>g</mi><mo>−</mo><mi>d</mi><mi>r</mi><mi>u</mi><mi>g</mi></mrow></msubsup></mrow></math>
</span> based on CGIs, and the formula was as follows.</p>
<table class="disp-formula p" id="equ3"><tr><td class="formula"><math id="m24" display="block" overflow="linebreak"><mrow><msubsup><mrow><mi mathvariant="bold-italic">S</mi><mi mathvariant="bold-italic">i</mi><mi mathvariant="bold-italic">m</mi></mrow><mrow><mi mathvariant="bold-italic">C</mi><mi mathvariant="bold-italic">G</mi><mi mathvariant="bold-italic">I</mi><mrow><mfenced open="(" close=")" separators="|"><mrow><mi mathvariant="bold-italic">i</mi><mo>,</mo><mi mathvariant="bold-italic">j</mi></mrow></mfenced></mrow></mrow><mrow><mi mathvariant="bold-italic">d</mi><mi mathvariant="bold-italic">r</mi><mi mathvariant="bold-italic">u</mi><mi mathvariant="bold-italic">g</mi><mo>−</mo><mi mathvariant="bold-italic">d</mi><mi mathvariant="bold-italic">r</mi><mi mathvariant="bold-italic">u</mi><mi mathvariant="bold-italic">g</mi></mrow></msubsup><mo>=</mo><mfrac><mrow><mfenced open="|" close="" separators="|"><mrow><mrow><msub><mrow><mi mathvariant="bold-italic">G</mi><mi mathvariant="bold-italic">T</mi></mrow><mi mathvariant="bold-italic">i</mi></msub><mo>∩</mo><mrow><mfenced open="" close="|" separators="|"><mrow><msub><mrow><mi mathvariant="bold-italic">G</mi><mi mathvariant="bold-italic">T</mi></mrow><mi mathvariant="bold-italic">j</mi></msub></mrow></mfenced></mrow></mrow></mrow></mfenced></mrow><mrow><mfenced open="|" close="" separators="|"><mrow><mrow><msub><mrow><mi mathvariant="bold-italic">G</mi><mi mathvariant="bold-italic">T</mi></mrow><mi mathvariant="bold-italic">i</mi></msub><mo>∪</mo><mrow><mfenced open="" close="|" separators="|"><mrow><msub><mrow><mi mathvariant="bold-italic">G</mi><mi mathvariant="bold-italic">T</mi></mrow><mi mathvariant="bold-italic">j</mi></msub></mrow></mfenced></mrow></mrow></mrow></mfenced></mrow></mfrac></mrow></math></td></tr></table></section></section><section id="s2-3"><h3 class="pmc_sec_title">2.3 Construction of ADR similarity features</h3>
<section id="s2-3-1"><h4 class="pmc_sec_title">2.3.1 ADR semantic similarity extracted from the MeSH database</h4>
<p>Drug-induced diseases are a subset of ADRs (<a href="#B21" class="usa-link" aria-describedby="B21">Pathan et al., 2018</a>) and are also commonly referred to as side effects. Herein, we attempted to map all ADRs to a hierarchical clinical terminology vocabulary to describe them more professionally. All the ADRs in the SIDER database were mapped to the MeSH database, as earlier mentioned in <a href="#s2-1" class="usa-link">Section 2.1</a>. The MeSH database is a commonly used standard medical thesaurus published by the National Library of Medicine (NLM) in the United States. It comprises hierarchical sets of descriptors based on their semantic categories and subject attributes. The MeSH database allows for the searching of diseases at various levels of specificity and can be used to examine disease correlations. For instance, the “Angioedema” entry has three possible addresses or codes: C14.907.079, C17.800.862.945.066, and C20.543.480.904.066, which belong to Cardiovascular Disease [C14], Skin and Connective Tissue Disease [C17], and Immune System Disease [C20] categories, respectively. <a href="#F2" class="usa-link">Figure 2</a> shows the hierarchical structure of ' Angioedema’ extracted from the MeSH database.</p>
<figure class="fig xbox font-sm" id="F2"><h5 class="obj_head">FIGURE 2.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11931068_fphar-16-1448106-g002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/97c2/11931068/db61a2d1e8ca/fphar-16-1448106-g002.jpg" loading="lazy" height="414" width="774" alt="FIGURE 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/F2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>An illustration of the hierarchical structure of the disease “Angioedema” extracted from the MeSH database. The numbers next to the connecting line indicate the size of the semantic contribution factor. The closer it is, the more significant the contribution factor.</p></figcaption></figure><p>We constructed a Directed Acyclic Graph (DAG) for each ADR using hierarchical descriptors from MeSH, with nodes representing ADR descriptors and edges representing the relationship between the current ADR and its ancestor ADRs. Each ADR, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf22">
<math id="m25" overflow="linebreak"><mrow><mi>s</mi><mo>,</mo></mrow></math>
</span> can be presented as a graph, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf23">
<math id="m26" overflow="linebreak"><mrow><msub><mrow><mi>D</mi><mi>A</mi><mi>G</mi></mrow><mi>s</mi></msub><mo>=</mo><mrow><mfenced open="(" close=")" separators="|"><mrow><mi>s</mi><mo>,</mo><msub><mi>N</mi><mi>s</mi></msub><mo>,</mo><msub><mi>E</mi><mi>s</mi></msub></mrow></mfenced></mrow></mrow></math>
</span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf24">
<math id="m27" overflow="linebreak"><mrow><msub><mi>N</mi><mi>s</mi></msub></mrow></math>
</span> is the set of all ancestor nodes, including the ADR node (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf25">
<math id="m28" overflow="linebreak"><mrow><mi mathvariant="bold-italic">s</mi></mrow></math>
</span>) itself, and Ed is the set of parent nodes pointing to the child node edges. The semantic contribution value of a node (n) to an ADR (s) in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf26">
<math id="m29" overflow="linebreak"><mrow><msub><mrow><mi>D</mi><mi>A</mi><mi>G</mi></mrow><mi>s</mi></msub></mrow></math>
</span> can be calculated as follows:</p>
<table class="disp-formula p" id="equ4"><tr><td class="formula"><math id="m30" display="block" overflow="linebreak"><mrow><msub><mi mathvariant="bold-italic">C</mi><mi mathvariant="bold-italic">s</mi></msub><mrow><mfenced open="(" close=")" separators="|"><mrow><mi mathvariant="bold-italic">n</mi></mrow></mfenced></mrow><mo>=</mo><mrow><mfenced open="{" close="" separators="|"><mrow><mtable columnalign="center"><mtr><mtd><mrow><mn mathvariant="bold">1</mn><mo>,</mo><mi mathvariant="bold-italic">i</mi><mi mathvariant="bold-italic">f</mi><mtext> </mtext><mi mathvariant="bold-italic">n</mi><mo>=</mo><mi mathvariant="bold-italic">s</mi></mrow></mtd></mtr><mtr><mtd><mrow><mi mathvariant="bold-italic">max</mi><mrow><mfenced open="{" close="}" separators="|"><mrow><mo>Δ</mo><mo>*</mo><msub><mi mathvariant="bold-italic">C</mi><mi mathvariant="bold-italic">s</mi></msub><mrow><mfenced open="" close="|" separators="|"><mrow><mrow><mfenced open="(" close=")" separators="|"><mrow><msup><mi mathvariant="bold-italic">n</mi><mo>′</mo></msup></mrow></mfenced></mrow></mrow></mfenced></mrow><msup><mi mathvariant="bold-italic">n</mi><mo>′</mo></msup><mo>∈</mo><mi mathvariant="bold-italic">c</mi><mi mathvariant="bold-italic">h</mi><mi mathvariant="bold-italic">i</mi><mi mathvariant="bold-italic">l</mi><mi mathvariant="bold-italic">d</mi><mi mathvariant="bold-italic">r</mi><mi mathvariant="bold-italic">e</mi><mi mathvariant="bold-italic">n</mi><mtext> </mtext><mi mathvariant="bold-italic">o</mi><mi mathvariant="bold-italic">f</mi><mtext> </mtext><mi mathvariant="bold-italic">n</mi><mtext> </mtext></mrow></mfenced></mrow><mo>,</mo><mi mathvariant="bold-italic">o</mi><mi mathvariant="bold-italic">t</mi><mi mathvariant="bold-italic">h</mi><mi mathvariant="bold-italic">e</mi><mi mathvariant="bold-italic">r</mi><mi mathvariant="bold-italic">w</mi><mi mathvariant="bold-italic">i</mi><mi mathvariant="bold-italic">s</mi><mi mathvariant="bold-italic">e</mi></mrow></mtd></mtr></mtable></mrow></mfenced></mrow></mrow></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf27">
<math id="m31" overflow="linebreak"><mrow><mo>Δ</mo></mrow></math>
</span> is a semantic contribution factor for the edges linking a node (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf28">
<math id="m32" overflow="linebreak"><mrow><mi mathvariant="bold-italic">n</mi></mrow></math>
</span>) to its child (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf29">
<math id="m33" overflow="linebreak"><mrow><msup><mi mathvariant="bold-italic">n</mi><mo>′</mo></msup></mrow></math>
</span>). Herein, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf30">
<math id="m34" overflow="linebreak"><mrow><mo>Δ</mo></mrow></math>
</span> was set to 0.5. By summarizing all nodes that an ADR (s) has, we can obtain the semantic value of ADR s, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf31">
<math id="m35" overflow="linebreak"><mrow><mtext>DV</mtext><mrow><mfenced open="(" close=")" separators="|"><mrow><mi mathvariant="normal">s</mi></mrow></mfenced></mrow><mo>,</mo></mrow></math>
</span> as follows:</p>
<table class="disp-formula p" id="equ5"><tr><td class="formula"><math id="m36" display="block" overflow="linebreak"><mrow><mi mathvariant="bold-italic">D</mi><mi mathvariant="bold-italic">V</mi><mrow><mfenced open="(" close=")" separators="|"><mrow><mi mathvariant="bold-italic">s</mi></mrow></mfenced></mrow><mo>=</mo><msub><mo>∑</mo><mrow><mi mathvariant="bold-italic">n</mi><mo>∈</mo><msub><mi mathvariant="bold-italic">N</mi><mi mathvariant="bold-italic">s</mi></msub></mrow></msub><msub><mi mathvariant="bold-italic">C</mi><mi mathvariant="bold-italic">s</mi></msub><mrow><mfenced open="(" close=")" separators="|"><mrow><mi mathvariant="bold-italic">n</mi></mrow></mfenced></mrow></mrow></math></td></tr></table>
<p>Where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf32">
<math id="m37" overflow="linebreak"><mrow><msub><mi>N</mi><mi>s</mi></msub></mrow></math>
</span> is the set of the ADR node (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf33">
<math id="m38" overflow="linebreak"><mrow><mi mathvariant="bold-italic">s</mi></mrow></math>
</span>) and its ancestor nodes. Notably, ADRs with more identical ancestors often have greater similarity. Herein, we defined <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf34">
<math id="m39" overflow="linebreak"><mrow><msubsup><mrow><mi>S</mi><mi>i</mi><mi>m</mi></mrow><msub><mrow><mi>M</mi><mi>E</mi><mi>S</mi><mi>H</mi></mrow><mrow><mfenced open="(" close=")" separators="|"><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></mfenced></mrow></msub><mrow><mi>s</mi><mi>i</mi><mi>d</mi><mi>e</mi><mo>−</mo><mi>s</mi><mi>i</mi><mi>d</mi><mi>e</mi></mrow></msubsup></mrow></math>
</span> as the semantic similarity between ADRs, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf35">
<math id="m40" overflow="linebreak"><mrow><msub><mi>s</mi><mi>i</mi></msub></mrow></math>
</span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf36">
<math id="m41" overflow="linebreak"><mrow><msub><mi>s</mi><mi>j</mi></msub></mrow></math>
</span>, and it was calculated as follows:</p>
<table class="disp-formula p" id="equ6"><tr><td class="formula"><math id="m42" display="block" overflow="linebreak"><mrow><msubsup><mrow><mi mathvariant="bold-italic">S</mi><mi mathvariant="bold-italic">i</mi><mi mathvariant="bold-italic">m</mi></mrow><msub><mrow><mi mathvariant="bold-italic">M</mi><mi mathvariant="bold-italic">E</mi><mi mathvariant="bold-italic">S</mi><mi mathvariant="bold-italic">H</mi></mrow><mrow><mfenced open="(" close=")" separators="|"><mrow><mi mathvariant="bold-italic">i</mi><mo>,</mo><mi mathvariant="bold-italic">j</mi></mrow></mfenced></mrow></msub><mrow><mi mathvariant="bold-italic">s</mi><mi mathvariant="bold-italic">i</mi><mi mathvariant="bold-italic">d</mi><mi mathvariant="bold-italic">e</mi><mo>−</mo><mi mathvariant="bold-italic">s</mi><mi mathvariant="bold-italic">i</mi><mi mathvariant="bold-italic">d</mi><mi mathvariant="bold-italic">e</mi></mrow></msubsup><mo>=</mo><mfrac><mrow><msub><mo>∑</mo><mrow><mi mathvariant="bold-italic">n</mi><mo>∈</mo><msub><mi mathvariant="bold-italic">N</mi><msub><mi mathvariant="bold-italic">s</mi><mi mathvariant="bold-italic">i</mi></msub></msub><mo>∩</mo><msub><mi mathvariant="bold-italic">N</mi><msub><mi mathvariant="bold-italic">s</mi><mi mathvariant="bold-italic">j</mi></msub></msub></mrow></msub><mrow><mfenced open="(" close=")" separators="|"><mrow><msub><mi mathvariant="bold-italic">C</mi><msub><mi mathvariant="bold-italic">s</mi><mi mathvariant="bold-italic">i</mi></msub></msub><mrow><mfenced open="(" close=")" separators="|"><mrow><mi mathvariant="bold-italic">n</mi></mrow></mfenced></mrow><mo>+</mo><msub><mi mathvariant="bold-italic">C</mi><msub><mi mathvariant="bold-italic">s</mi><mi mathvariant="bold-italic">j</mi></msub></msub><mrow><mfenced open="(" close=")" separators="|"><mrow><mi mathvariant="bold-italic">n</mi></mrow></mfenced></mrow></mrow></mfenced></mrow></mrow><mrow><mi mathvariant="bold-italic">D</mi><mi mathvariant="bold-italic">V</mi><mrow><mfenced open="(" close=")" separators="|"><mrow><msub><mi mathvariant="bold-italic">s</mi><mi mathvariant="bold-italic">i</mi></msub></mrow></mfenced></mrow><mo>+</mo><mi mathvariant="bold-italic">D</mi><mi mathvariant="bold-italic">V</mi><mrow><mfenced open="(" close=")" separators="|"><mrow><msub><mi mathvariant="bold-italic">s</mi><mi mathvariant="bold-italic">j</mi></msub></mrow></mfenced></mrow></mrow></mfrac></mrow></math></td></tr></table></section><section id="s2-3-2"><h4 class="pmc_sec_title">2.3.2 ADR similarity based on GDA</h4>
<p>In this study, GDAs were obtained from CTD. There are three types of direct evidence for a GDA: M marker, Mechanism, or T therapeutic. As in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf37">
<math id="m43" overflow="linebreak"><mrow><msubsup><mrow><mi>S</mi><mi>i</mi><mi>m</mi></mrow><mrow><mi>C</mi><mi>G</mi><mi>I</mi></mrow><mrow><mi>d</mi><mi>r</mi><mi>u</mi><mi>g</mi><mo>−</mo><mi>d</mi><mi>r</mi><mi>u</mi><mi>g</mi></mrow></msubsup></mrow></math>
</span>, we used gene sets <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf38">
<math id="m44" overflow="linebreak"><mrow><msub><mrow><mi>G</mi><mi>S</mi></mrow><mi>i</mi></msub><mo>=</mo><mrow><mfenced open="{" close="}" separators="|"><mrow><msub><mi>g</mi><mrow><mi>i</mi><mn>1</mn></mrow></msub><mo>,</mo><msub><mi>g</mi><mrow><mi>i</mi><mn>2</mn></mrow></msub><mo>,</mo><msub><mi>g</mi><mrow><mi>i</mi><mn>3</mn></mrow></msub><mo>,</mo><mo>…</mo><mo>,</mo><msub><mi>g</mi><mrow><mi>i</mi><mi>n</mi></mrow></msub></mrow></mfenced></mrow></mrow></math>
</span> to denote n genes with associations with ADR <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf39">
<math id="m45" overflow="linebreak"><mrow><msub><mi>s</mi><mi>i</mi></msub></mrow></math>
</span>, and gene sets <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf40">
<math id="m46" overflow="linebreak"><mrow><msub><mrow><mi>G</mi><mi>S</mi></mrow><mi>j</mi></msub><mo>=</mo><mrow><mfenced open="{" close="}" separators="|"><mrow><msub><mi>g</mi><mrow><mi>j</mi><mn>1</mn></mrow></msub><mo>,</mo><msub><mi>g</mi><mrow><mi>j</mi><mn>2</mn></mrow></msub><mo>,</mo><msub><mi>g</mi><mrow><mi>j</mi><mn>3</mn></mrow></msub><mo>,</mo><mo>…</mo><mo>,</mo><msub><mi>g</mi><mrow><mi>j</mi><mi>m</mi></mrow></msub></mrow></mfenced></mrow></mrow></math>
</span> to denote m genes with associations with ADR <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf41">
<math id="m47" overflow="linebreak"><mrow><msub><mi>s</mi><mi>j</mi></msub></mrow></math>
</span>. Diseases associated with more identical genes tend have greater similarity. In this regard, GDA-based ADR similarity, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf42">
<math id="m48" overflow="linebreak"><mrow><msubsup><mrow><mi>S</mi><mi>i</mi><mi>m</mi></mrow><msub><mrow><mi>G</mi><mi>D</mi><mi>A</mi></mrow><mrow><mfenced open="(" close=")" separators="|"><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></mfenced></mrow></msub><mrow><mi>s</mi><mi>i</mi><mi>d</mi><mi>e</mi><mo>−</mo><mi>s</mi><mi>i</mi><mi>d</mi><mi>e</mi></mrow></msubsup></mrow></math>
</span>, can be measured using the Jaccard index as follows.</p>
<table class="disp-formula p" id="equ7"><tr><td class="formula"><math id="m49" display="block" overflow="linebreak"><mrow><msubsup><mrow><mi mathvariant="bold-italic">S</mi><mi mathvariant="bold-italic">i</mi><mi mathvariant="bold-italic">m</mi></mrow><msub><mrow><mi mathvariant="bold-italic">G</mi><mi mathvariant="bold-italic">D</mi><mi mathvariant="bold-italic">A</mi></mrow><mrow><mfenced open="(" close=")" separators="|"><mrow><mi mathvariant="bold-italic">i</mi><mo>,</mo><mi mathvariant="bold-italic">j</mi></mrow></mfenced></mrow></msub><mrow><mi mathvariant="bold-italic">s</mi><mi mathvariant="bold-italic">i</mi><mi mathvariant="bold-italic">d</mi><mi mathvariant="bold-italic">e</mi><mo>−</mo><mi mathvariant="bold-italic">s</mi><mi mathvariant="bold-italic">i</mi><mi mathvariant="bold-italic">d</mi><mi mathvariant="bold-italic">e</mi></mrow></msubsup><mo>=</mo><mfrac><mrow><mfenced open="|" close="" separators="|"><mrow><mrow><msub><mrow><mi mathvariant="bold-italic">G</mi><mi mathvariant="bold-italic">S</mi></mrow><mi mathvariant="bold-italic">i</mi></msub><mo>∩</mo><mrow><mfenced open="" close="|" separators="|"><mrow><msub><mrow><mi mathvariant="bold-italic">G</mi><mi mathvariant="bold-italic">S</mi></mrow><mi mathvariant="bold-italic">j</mi></msub></mrow></mfenced></mrow></mrow></mrow></mfenced></mrow><mrow><mfenced open="|" close="" separators="|"><mrow><mrow><msub><mrow><mi mathvariant="bold-italic">G</mi><mi mathvariant="bold-italic">S</mi></mrow><mi mathvariant="bold-italic">i</mi></msub><mo>∪</mo><mrow><mfenced open="" close="|" separators="|"><mrow><msub><mrow><mi mathvariant="bold-italic">G</mi><mi mathvariant="bold-italic">S</mi></mrow><mi mathvariant="bold-italic">j</mi></msub></mrow></mfenced></mrow></mrow></mrow></mfenced></mrow></mfrac></mrow></math></td></tr></table></section></section><section id="s2-4"><h3 class="pmc_sec_title">2.4 Drug-gene-ADRs network for ADR prediction</h3>
<p>After data preprocessing in <a href="#s2-2" class="usa-link">Sections 2.2</a>, <a href="#s2-3" class="usa-link">2.3</a>, we obtained three (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf43">
<math id="m50" overflow="linebreak"><mrow><msubsup><mrow><mi>S</mi><mi>i</mi><mi>m</mi></mrow><mrow><mi>c</mi><mi>s</mi></mrow><mrow><mi>d</mi><mi>r</mi><mi>u</mi><mi>g</mi><mo>−</mo><mi>d</mi><mi>r</mi><mi>u</mi><mi>g</mi></mrow></msubsup></mrow></math>
</span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf44">
<math id="m51" overflow="linebreak"><mrow><msubsup><mrow><mi>S</mi><mi>i</mi><mi>m</mi></mrow><mrow><mi>G</mi><mi>E</mi></mrow><mrow><mi>d</mi><mi>r</mi><mi>u</mi><mi>g</mi><mo>−</mo><mi>d</mi><mi>r</mi><mi>u</mi><mi>g</mi></mrow></msubsup></mrow></math>
</span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf45">
<math id="m52" overflow="linebreak"><mrow><msubsup><mrow><mi>S</mi><mi>i</mi><mi>m</mi></mrow><mrow><mi>C</mi><mi>G</mi><mi>I</mi></mrow><mrow><mi>d</mi><mi>r</mi><mi>u</mi><mi>g</mi><mo>−</mo><mi>d</mi><mi>r</mi><mi>u</mi><mi>g</mi></mrow></msubsup></mrow></math>
</span>) and two (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf46">
<math id="m53" overflow="linebreak"><mrow><msubsup><mrow><mi>S</mi><mi>i</mi><mi>m</mi></mrow><mrow><mi>M</mi><mi>E</mi><mi>S</mi><mi>H</mi></mrow><mrow><mi>s</mi><mi>i</mi><mi>d</mi><mi>e</mi><mo>−</mo><mi>s</mi><mi>i</mi><mi>d</mi><mi>e</mi></mrow></msubsup></mrow></math>
</span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf47">
<math id="m54" overflow="linebreak"><mrow><msubsup><mrow><mi>S</mi><mi>i</mi><mi>m</mi></mrow><mrow><mi>G</mi><mi>D</mi><mi>A</mi></mrow><mrow><mi>s</mi><mi>i</mi><mi>d</mi><mi>e</mi><mo>−</mo><mi>s</mi><mi>i</mi><mi>d</mi><mi>e</mi></mrow></msubsup></mrow></math>
</span>) similarity indices for drugs and ADRs, respectively. We then constructed the input vector of drug as Drug and Side as ADR as follows:</p>
<table class="disp-formula p" id="equ8"><tr><td class="formula"><math id="m55" display="block" overflow="linebreak"><mrow><mi mathvariant="bold-italic">D</mi><mi mathvariant="bold-italic">r</mi><mi mathvariant="bold-italic">u</mi><mi mathvariant="bold-italic">g</mi><mo>=</mo><mrow><mfenced open="[" close="]" separators="|"><mrow><msubsup><mrow><mi mathvariant="bold-italic">S</mi><mi mathvariant="bold-italic">i</mi><mi mathvariant="bold-italic">m</mi></mrow><mrow><mi mathvariant="bold-italic">C</mi><mi mathvariant="bold-italic">S</mi></mrow><mrow><mi mathvariant="bold-italic">d</mi><mi mathvariant="bold-italic">r</mi><mi mathvariant="bold-italic">u</mi><mi mathvariant="bold-italic">g</mi><mo>−</mo><mi mathvariant="bold-italic">d</mi><mi mathvariant="bold-italic">r</mi><mi mathvariant="bold-italic">u</mi><mi mathvariant="bold-italic">g</mi></mrow></msubsup><mo>,</mo><msubsup><mrow><mi mathvariant="bold-italic">S</mi><mi mathvariant="bold-italic">i</mi><mi mathvariant="bold-italic">m</mi></mrow><mrow><mi mathvariant="bold-italic">G</mi><mi mathvariant="bold-italic">E</mi></mrow><mrow><mi mathvariant="bold-italic">d</mi><mi mathvariant="bold-italic">r</mi><mi mathvariant="bold-italic">u</mi><mi mathvariant="bold-italic">g</mi><mo>−</mo><mi mathvariant="bold-italic">d</mi><mi mathvariant="bold-italic">r</mi><mi mathvariant="bold-italic">u</mi><mi mathvariant="bold-italic">g</mi></mrow></msubsup><mo>,</mo><msubsup><mrow><mi mathvariant="bold-italic">S</mi><mi mathvariant="bold-italic">i</mi><mi mathvariant="bold-italic">m</mi></mrow><mrow><mi mathvariant="bold-italic">C</mi><mi mathvariant="bold-italic">G</mi><mi mathvariant="bold-italic">I</mi></mrow><mrow><mi mathvariant="bold-italic">d</mi><mi mathvariant="bold-italic">r</mi><mi mathvariant="bold-italic">u</mi><mi mathvariant="bold-italic">g</mi><mo>−</mo><mi mathvariant="bold-italic">d</mi><mi mathvariant="bold-italic">r</mi><mi mathvariant="bold-italic">u</mi><mi mathvariant="bold-italic">g</mi></mrow></msubsup></mrow></mfenced></mrow></mrow></math></td></tr></table>
<table class="disp-formula p" id="equ9"><tr><td class="formula"><math id="m56" display="block" overflow="linebreak"><mrow><mi mathvariant="bold-italic">S</mi><mi mathvariant="bold-italic">i</mi><mi mathvariant="bold-italic">d</mi><mi mathvariant="bold-italic">e</mi><mo>=</mo><mrow><mfenced open="[" close="]" separators="|"><mrow><msubsup><mrow><mi mathvariant="bold-italic">S</mi><mi mathvariant="bold-italic">i</mi><mi mathvariant="bold-italic">m</mi></mrow><mrow><mi mathvariant="bold-italic">M</mi><mi mathvariant="bold-italic">E</mi><mi mathvariant="bold-italic">S</mi><mi mathvariant="bold-italic">H</mi></mrow><mrow><mi mathvariant="bold-italic">s</mi><mi mathvariant="bold-italic">i</mi><mi mathvariant="bold-italic">d</mi><mi mathvariant="bold-italic">e</mi><mo>−</mo><mi mathvariant="bold-italic">s</mi><mi mathvariant="bold-italic">i</mi><mi mathvariant="bold-italic">d</mi><mi mathvariant="bold-italic">e</mi></mrow></msubsup><mo>,</mo><msubsup><mrow><mi mathvariant="bold-italic">S</mi><mi mathvariant="bold-italic">i</mi><mi mathvariant="bold-italic">m</mi></mrow><mrow><mi mathvariant="bold-italic">G</mi><mi mathvariant="bold-italic">D</mi><mi mathvariant="bold-italic">A</mi></mrow><mrow><mi mathvariant="bold-italic">s</mi><mi mathvariant="bold-italic">i</mi><mi mathvariant="bold-italic">d</mi><mi mathvariant="bold-italic">e</mi><mo>−</mo><mi mathvariant="bold-italic">s</mi><mi mathvariant="bold-italic">i</mi><mi mathvariant="bold-italic">d</mi><mi mathvariant="bold-italic">e</mi></mrow></msubsup></mrow></mfenced></mrow></mrow></math></td></tr></table>
<p>We constructed feature crosses using the Cartesian product to better understand the nonlinear relationship between drug features and ADRs. Feature crosses are a useful feature engineering technology that can help the model capture nonlinear relationships in the data. According to research, feature crosses can better capture the interaction between features than fully connected operations (<a href="#B19" class="usa-link" aria-describedby="B19">Lian et al., 2018</a>). Herein, the feature crosses of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf48">
<math id="m57" overflow="linebreak"><mrow><mi>D</mi><mi>r</mi><mi>u</mi><mi>g</mi></mrow></math>
</span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf49">
<math id="m58" overflow="linebreak"><mrow><mi>S</mi><mi>i</mi><mi>d</mi><mi>e</mi></mrow></math>
</span> were defined as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf50">
<math id="m59" overflow="linebreak"><mrow><msub><mi>V</mi><mrow><mi>F</mi><mi>C</mi><mi>s</mi></mrow></msub></mrow></math>
</span> and illustrated as follows:</p>
<table class="disp-formula p" id="equ10"><tr><td class="formula"><math id="m60" display="block" overflow="linebreak"><mrow><msub><mi mathvariant="bold-italic">V</mi><mrow><mi mathvariant="bold-italic">F</mi><mi mathvariant="bold-italic">C</mi><mi mathvariant="bold-italic">s</mi></mrow></msub><mo>=</mo><mi mathvariant="bold-italic">D</mi><mi mathvariant="bold-italic">r</mi><mi mathvariant="bold-italic">u</mi><mi mathvariant="bold-italic">g</mi><mtext> </mtext><mo>⨯</mo><mtext> </mtext><mi mathvariant="bold-italic">S</mi><mi mathvariant="bold-italic">i</mi><mi mathvariant="bold-italic">d</mi><mi mathvariant="bold-italic">e</mi><mo>=</mo><mrow><mfenced open="{" close="}" separators="|"><mrow><mrow><mfenced open="(" close=")" separators="|"><mrow><mi mathvariant="bold-italic">d</mi><mo>,</mo><mi mathvariant="bold-italic">s</mi></mrow></mfenced></mrow><mrow><mfenced open="|" close="" separators="|"><mrow><mrow><mi mathvariant="bold-italic">d</mi><mo>∈</mo><mi mathvariant="bold-italic">D</mi><mi mathvariant="bold-italic">r</mi><mi mathvariant="bold-italic">u</mi><mi mathvariant="bold-italic">g</mi><mo>,</mo><mi mathvariant="bold-italic">s</mi><mo>∈</mo><mi mathvariant="bold-italic">S</mi><mi mathvariant="bold-italic">i</mi><mi mathvariant="bold-italic">d</mi><mi mathvariant="bold-italic">e</mi></mrow></mrow></mfenced></mrow></mrow></mfenced></mrow></mrow></math></td></tr></table>
<p>To effectively integrate similarity information from multiple data sources for ADR prediction, we proposed DGANet, a CNN-based multi-label classification architecture that considers each label as an independent binary problem. <a href="#F3" class="usa-link">Figure 3</a> depicts the architecture of DGANet.</p>
<figure class="fig xbox font-sm" id="F3"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11931068_fphar-16-1448106-g003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/97c2/11931068/c57ee490ed68/fphar-16-1448106-g003.jpg" loading="lazy" height="587" width="708" alt="FIGURE 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/F3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Flowchart of DGANet in ADR prediction. For a drug-side effect pair, DGANet employs two Linear Subnetworks (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf51">
<math id="m61" overflow="linebreak"><mrow><msub><mrow><mi>L</mi><mi>S</mi><mi>N</mi></mrow><mi>d</mi></msub></mrow></math>
</span> for drug <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf52">
<math id="m62" overflow="linebreak"><mrow><mi>d</mi></mrow></math>
</span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf53">
<math id="m63" overflow="linebreak"><mrow><msub><mrow><mi>L</mi><mi>S</mi><mi>N</mi></mrow><mi>s</mi></msub></mrow></math>
</span> for side effect <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf54">
<math id="m64" overflow="linebreak"><mrow><mi>s</mi></mrow></math>
</span>) and one Convolutional Neural Network with a Subnetwork (CSN) for feature crosses <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf55">
<math id="m65" overflow="linebreak"><mrow><msub><mi>V</mi><mrow><mi>F</mi><mi>C</mi><mi>s</mi></mrow></msub></mrow></math>
</span>.</p></figcaption></figure><p>For a drug-ADR pair, DGANet employs two Linear Subnetworks (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf56">
<math id="m66" overflow="linebreak"><mrow><msub><mrow><mi>L</mi><mi>S</mi><mi>N</mi></mrow><mi>d</mi></msub></mrow></math>
</span> for drug <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf57">
<math id="m67" overflow="linebreak"><mrow><mi>d</mi></mrow></math>
</span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf58">
<math id="m68" overflow="linebreak"><mrow><msub><mrow><mi>L</mi><mi>S</mi><mi>N</mi></mrow><mi>s</mi></msub></mrow></math>
</span> for ADR <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf59">
<math id="m69" overflow="linebreak"><mrow><mi>s</mi></mrow></math>
</span>) and one Convolutional Neural Network with a Subnetwork (CSN) for feature crosses <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf60">
<math id="m70" overflow="linebreak"><mrow><msub><mi>V</mi><mrow><mi>F</mi><mi>C</mi><mi>s</mi></mrow></msub></mrow></math>
</span>. The two LSNs, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf61">
<math id="m71" overflow="linebreak"><mrow><msub><mrow><mi>L</mi><mi>S</mi><mi>N</mi></mrow><mi>d</mi></msub></mrow></math>
</span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf62">
<math id="m72" overflow="linebreak"><mrow><msub><mrow><mi>L</mi><mi>S</mi><mi>N</mi></mrow><mi>s</mi></msub></mrow></math>
</span>, share the same architecture. The LSN formula can be summarized as follows:</p>
<table class="disp-formula p" id="equ11"><tr><td class="formula"><math id="m73" display="block" overflow="linebreak"><mrow><msub><mi mathvariant="bold-italic">x</mi><mrow><mi mathvariant="bold-italic">m</mi><mo>,</mo><mi mathvariant="bold-italic">k</mi></mrow></msub><mo>=</mo><mi mathvariant="bold-italic">L</mi><mi mathvariant="bold-italic">i</mi><mi mathvariant="bold-italic">n</mi><mi mathvariant="bold-italic">e</mi><mi mathvariant="bold-italic">a</mi><mi mathvariant="bold-italic">r</mi><mrow><mfenced open="(" close=")" separators="|"><mrow><msup><mrow><mi mathvariant="bold-italic">D</mi><mi mathvariant="bold-italic">r</mi><mi mathvariant="bold-italic">o</mi><mi mathvariant="bold-italic">p</mi><mi mathvariant="bold-italic">o</mi><mi mathvariant="bold-italic">u</mi><mi mathvariant="bold-italic">t</mi></mrow><mrow><mfenced open="(" close=")" separators="|"><mrow><mi mathvariant="bold-italic">p</mi></mrow></mfenced></mrow></msup><mrow><mfenced open="(" close=")" separators="|"><mrow><mi mathvariant="bold-italic">R</mi><mi mathvariant="bold-italic">e</mi><mi mathvariant="bold-italic">l</mi><mi mathvariant="bold-italic">U</mi><mrow><mfenced open="(" close=")" separators="|"><mrow><msup><mrow><mi mathvariant="bold-italic">F</mi><mi mathvariant="bold-italic">C</mi></mrow><mrow><mi mathvariant="bold-italic">n</mi><mn mathvariant="bold">1</mn></mrow></msup><mrow><mfenced open="(" close=")" separators="|"><mrow><mi mathvariant="bold-italic">C</mi><mi mathvariant="bold-italic">A</mi><mi mathvariant="bold-italic">T</mi><mrow><mfenced open="(" close=")" separators="|"><mrow><msub><mi mathvariant="bold-italic">x</mi><mi mathvariant="bold-italic">m</mi></msub><mo>,</mo><msub><mi mathvariant="bold-italic">c</mi><mi mathvariant="bold-italic">k</mi></msub></mrow></mfenced></mrow></mrow></mfenced></mrow></mrow></mfenced></mrow></mrow></mfenced></mrow></mrow></mfenced></mrow></mrow></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf63">
<math id="m74" overflow="linebreak"><mrow><msub><mi>x</mi><mrow><mi>m</mi><mo>,</mo><mi>k</mi></mrow></msub></mrow></math>
</span> is the latent representation of m drugs or m ADRs with k different similarity features, which is the output of the LSN, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf64">
<math id="m75" overflow="linebreak"><mrow><msup><mrow><mi>F</mi><mi>C</mi></mrow><mrow><mfenced open="(" close=")" separators="|"><mrow><mi>n</mi></mrow></mfenced></mrow></msup></mrow></math>
</span> is a fully connected layer with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf65">
<math id="m76" overflow="linebreak"><mrow><mi>n</mi></mrow></math>
</span> neurons, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf66">
<math id="m77" overflow="linebreak"><mrow><msup><mrow><mi>D</mi><mi>r</mi><mi>o</mi><mi>p</mi><mi>o</mi><mi>u</mi><mi>t</mi></mrow><mrow><mfenced open="(" close=")" separators="|"><mrow><mi>p</mi></mrow></mfenced></mrow></msup></mrow></math>
</span> is a dropout layer with probability <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf67">
<math id="m78" overflow="linebreak"><mrow><mi>p</mi></mrow></math>
</span>. On the other hand, Linear and RelU represent linear and rectified linear unit activation functions, respectively, and CAT concatenates given feature vectors. We then obtained the vector embeddings of drugs, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf68">
<math id="m79" overflow="linebreak"><mrow><msub><mrow><mi>e</mi><mi>m</mi><mi>b</mi></mrow><mrow><mi>d</mi><mi>r</mi><mi>u</mi><mi>g</mi><mi>s</mi></mrow></msub></mrow></math>
</span>, and ADRs, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf69">
<math id="m80" overflow="linebreak"><mrow><msub><mrow><mi>e</mi><mi>m</mi><mi>b</mi></mrow><mrow><mi>a</mi><mi>d</mi><mi>r</mi><mi>s</mi></mrow></msub></mrow></math>
</span>. A six-layered CNN was used. Previously constructed feature crosses <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf70">
<math id="m81" overflow="linebreak"><mrow><msub><mi>V</mi><mrow><mi>F</mi><mi>C</mi><mi>s</mi></mrow></msub></mrow></math>
</span> were fed into the CSN to learn the representation of feature crosses, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf71">
<math id="m82" overflow="linebreak"><mrow><msub><mrow><mi>e</mi><mi>m</mi><mi>b</mi></mrow><mrow><mi>c</mi><mi>r</mi><mi>o</mi><mi>s</mi><mi>s</mi></mrow></msub></mrow></math>
</span>. Finally, the vector embeddings, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf72">
<math id="m83" overflow="linebreak"><mrow><msub><mrow><mi>e</mi><mi>m</mi><mi>b</mi></mrow><mrow><mi>d</mi><mi>r</mi><mi>u</mi><mi>g</mi><mi>s</mi></mrow></msub></mrow></math>
</span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf73">
<math id="m84" overflow="linebreak"><mrow><msub><mrow><mi>e</mi><mi>m</mi><mi>b</mi></mrow><mrow><mi>a</mi><mi>d</mi><mi>r</mi><mi>s</mi></mrow></msub></mrow></math>
</span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf74">
<math id="m85" overflow="linebreak"><mrow><msub><mrow><mi>e</mi><mi>m</mi><mi>b</mi></mrow><mrow><mi>c</mi><mi>r</mi><mi>o</mi><mi>s</mi><mi>s</mi></mrow></msub></mrow></math>
</span> constructed with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf75">
<math id="m86" overflow="linebreak"><mrow><msub><mrow><mi>L</mi><mi>S</mi><mi>N</mi></mrow><mi>d</mi></msub></mrow></math>
</span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf76">
<math id="m87" overflow="linebreak"><mrow><msub><mrow><mi>L</mi><mi>S</mi><mi>N</mi></mrow><mi>s</mi></msub><mo>,</mo></mrow></math>
</span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf77">
<math id="m88" overflow="linebreak"><mrow><mi>C</mi><mi>S</mi><mi>N</mi></mrow></math>
</span>, were concatenated and fed into a multi-label classifier, using two fully connected layers, activation functions, and a dropout. Finally, our model generated a vector, and values &gt;0 indicated a correlation between the drug and ADR. The output vector can be represented as follows:</p>
<table class="disp-formula p" id="equ12"><tr><td class="formula"><math id="m89" display="block" overflow="linebreak"><mrow><msub><mi mathvariant="bold-italic">y</mi><mrow><mi mathvariant="bold-italic">d</mi><mo>,</mo><mi mathvariant="bold-italic">s</mi></mrow></msub><mo>=</mo><mi mathvariant="bold-italic">R</mi><mi mathvariant="bold-italic">e</mi><mi mathvariant="bold-italic">l</mi><mi mathvariant="bold-italic">u</mi><mrow><mfenced open="(" close=")" separators="|"><mrow><msup><mrow><mi mathvariant="bold-italic">F</mi><mi mathvariant="bold-italic">C</mi></mrow><mrow><mi mathvariant="bold-italic">n</mi><mn mathvariant="bold">2</mn></mrow></msup><mrow><mfenced open="(" close=")" separators="|"><mrow><msup><mrow><mi mathvariant="bold-italic">D</mi><mi mathvariant="bold-italic">r</mi><mi mathvariant="bold-italic">o</mi><mi mathvariant="bold-italic">p</mi><mi mathvariant="bold-italic">o</mi><mi mathvariant="bold-italic">u</mi><mi mathvariant="bold-italic">t</mi></mrow><mrow><mfenced open="(" close=")" separators="|"><mrow><mi mathvariant="bold-italic">p</mi></mrow></mfenced></mrow></msup><mrow><mfenced open="(" close=")" separators="|"><mrow><mi mathvariant="bold-italic">R</mi><mi mathvariant="bold-italic">e</mi><mi mathvariant="bold-italic">l</mi><mi mathvariant="bold-italic">u</mi><mrow><mfenced open="(" close=")" separators="|"><mrow><msup><mrow><mi mathvariant="bold-italic">F</mi><mi mathvariant="bold-italic">C</mi></mrow><mrow><mi mathvariant="bold-italic">n</mi><mn mathvariant="bold">1</mn></mrow></msup><mrow><mfenced open="(" close=")" separators="|"><mrow><mi mathvariant="bold-italic">C</mi><mi mathvariant="bold-italic">A</mi><mi mathvariant="bold-italic">T</mi><mrow><mfenced open="(" close=")" separators="|"><mrow><msubsup><mrow><mi mathvariant="bold-italic">e</mi><mi mathvariant="bold-italic">m</mi><mi mathvariant="bold-italic">b</mi></mrow><mrow><mi mathvariant="bold-italic">d</mi><mi mathvariant="bold-italic">r</mi><mi mathvariant="bold-italic">u</mi><mi mathvariant="bold-italic">g</mi><mi mathvariant="bold-italic">s</mi></mrow><mi mathvariant="bold-italic">d</mi></msubsup><mo>,</mo><msubsup><mrow><mi mathvariant="bold-italic">e</mi><mi mathvariant="bold-italic">m</mi><mi mathvariant="bold-italic">b</mi></mrow><mrow><mi mathvariant="bold-italic">c</mi><mi mathvariant="bold-italic">r</mi><mi mathvariant="bold-italic">o</mi><mi mathvariant="bold-italic">s</mi><mi mathvariant="bold-italic">s</mi></mrow><mrow><mi mathvariant="bold-italic">d</mi><mo>,</mo><mi mathvariant="bold-italic">s</mi></mrow></msubsup><mo>,</mo><msubsup><mrow><mi mathvariant="bold-italic">e</mi><mi mathvariant="bold-italic">m</mi><mi mathvariant="bold-italic">b</mi></mrow><mrow><mi mathvariant="bold-italic">a</mi><mi mathvariant="bold-italic">d</mi><mi mathvariant="bold-italic">r</mi><mi mathvariant="bold-italic">s</mi></mrow><mi mathvariant="bold-italic">s</mi></msubsup></mrow></mfenced></mrow></mrow></mfenced></mrow></mrow></mfenced></mrow></mrow></mfenced></mrow></mrow></mfenced></mrow></mrow></mfenced></mrow></mrow></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf78">
<math id="m90" overflow="linebreak"><mrow><msub><mi>y</mi><mrow><mi>d</mi><mo>,</mo><mi>s</mi></mrow></msub></mrow></math>
</span> is the output association of drug d and ADR s, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf79">
<math id="m91" overflow="linebreak"><mrow><msup><mrow><mi>F</mi><mi>C</mi></mrow><mrow><mfenced open="(" close=")" separators="|"><mrow><mi>n</mi></mrow></mfenced></mrow></msup></mrow></math>
</span> is a fully connected layer with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf80">
<math id="m92" overflow="linebreak"><mrow><mi>n</mi></mrow></math>
</span> neurons, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf81">
<math id="m93" overflow="linebreak"><mrow><msup><mrow><mi>D</mi><mi>r</mi><mi>o</mi><mi>p</mi><mi>o</mi><mi>u</mi><mi>t</mi></mrow><mrow><mfenced open="(" close=")" separators="|"><mrow><mi>p</mi></mrow></mfenced></mrow></msup></mrow></math>
</span> is a dropout layer with probability <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf82">
<math id="m94" overflow="linebreak"><mrow><mi>p</mi></mrow></math>
</span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf83">
<math id="m95" overflow="linebreak"><mrow><msubsup><mrow><mi>e</mi><mi>m</mi><mi>b</mi></mrow><mrow><mi>d</mi><mi>r</mi><mi>u</mi><mi>g</mi><mi>s</mi></mrow><mi>d</mi></msubsup></mrow></math>
</span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf84">
<math id="m96" overflow="linebreak"><mrow><msubsup><mrow><mi>e</mi><mi>m</mi><mi>b</mi></mrow><mrow><mi>c</mi><mi>r</mi><mi>o</mi><mi>s</mi><mi>s</mi></mrow><mrow><mi>d</mi><mo>,</mo><mi>s</mi></mrow></msubsup></mrow></math>
</span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf85">
<math id="m97" overflow="linebreak"><mrow><msubsup><mrow><mi>e</mi><mi>m</mi><mi>b</mi></mrow><mrow><mi>a</mi><mi>d</mi><mi>r</mi><mi>s</mi></mrow><mi>s</mi></msubsup></mrow></math>
</span> are the embedding vectors of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf86">
<math id="m98" overflow="linebreak"><mrow><msub><mrow><mi>L</mi><mi>S</mi><mi>N</mi></mrow><mi>d</mi></msub></mrow></math>
</span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf87">
<math id="m99" overflow="linebreak"><mrow><msubsup><mrow><mi>e</mi><mi>m</mi><mi>b</mi></mrow><mrow><mi>c</mi><mi>r</mi><mi>o</mi><mi>s</mi><mi>s</mi></mrow><mrow><mi>d</mi><mo>,</mo><mi>s</mi></mrow></msubsup></mrow></math>
</span>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf88">
<math id="m100" overflow="linebreak"><mrow><mi>C</mi><mi>S</mi><mi>N</mi></mrow></math>
</span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf89">
<math id="m101" overflow="linebreak"><mrow><msub><mrow><mi>L</mi><mi>S</mi><mi>N</mi></mrow><mi>s</mi></msub></mrow></math>
</span>, respectively.</p>
<p>For model optimization, we adopted the ZLPR function (<a href="#B29" class="usa-link" aria-describedby="B29">Su et al., 2022</a>) to calculate the errors between the predicted and true values. The concept considers the correlation between labels, yielding more comprehensive outcomes than binary relevance methods. The loss function formula was as follows:</p>
<table class="disp-formula p" id="equ13"><tr><td class="formula"><math id="m102" display="block" overflow="linebreak"><mrow><msub><mi mathvariant="bold-script">L</mi><mrow><mi mathvariant="bold-italic">t</mi><mi mathvariant="bold-italic">l</mi><mi mathvariant="bold-italic">p</mi><mi mathvariant="bold-italic">r</mi></mrow></msub><mo>=</mo><mi mathvariant="bold-italic">log</mi><mrow><mfenced open="(" close=")" separators="|"><mrow><msup><mi mathvariant="bold-italic">e</mi><msub><mi mathvariant="bold-italic">s</mi><mn mathvariant="bold">0</mn></msub></msup><mo>+</mo><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mi mathvariant="bold-italic">i</mi><mo>∈</mo><msub><mi mathvariant="bold-italic">Ω</mi><mrow><mi mathvariant="bold-italic">n</mi><mi mathvariant="bold-italic">e</mi><mi mathvariant="bold-italic">g</mi></mrow></msub></mrow></munder></mstyle><msup><mi mathvariant="bold-italic">e</mi><msub><mi mathvariant="bold-italic">s</mi><mi mathvariant="bold-italic">i</mi></msub></msup></mrow></mfenced></mrow><mo>+</mo><mi mathvariant="bold-italic">log</mi><mrow><mfenced open="(" close=")" separators="|"><mrow><msup><mi mathvariant="bold-italic">e</mi><msub><mrow><mo>−</mo><mi mathvariant="bold-italic">s</mi></mrow><mn mathvariant="bold">0</mn></msub></msup><mo>+</mo><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mi mathvariant="bold-italic">j</mi><mo>∈</mo><msub><mi mathvariant="bold-italic">Ω</mi><mrow><mi mathvariant="bold-italic">p</mi><mi mathvariant="bold-italic">o</mi><mi mathvariant="bold-italic">s</mi></mrow></msub></mrow></munder></mstyle><msup><mi mathvariant="bold-italic">e</mi><mrow><mo>−</mo><msub><mi mathvariant="bold-italic">s</mi><mi mathvariant="bold-italic">j</mi></msub></mrow></msup></mrow></mfenced></mrow></mrow></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf90">
<math id="m103" overflow="linebreak"><mrow><msub><mi mathvariant="italic">Ω</mi><mrow><mi mathvariant="italic">p</mi><mi mathvariant="italic">o</mi><mi mathvariant="italic">s</mi></mrow></msub></mrow></math>
</span> is the set of positive labels, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf91">
<math id="m104" overflow="linebreak"><mrow><msub><mi mathvariant="italic">Ω</mi><mrow><mi mathvariant="italic">n</mi><mi mathvariant="italic">e</mi><mi mathvariant="italic">g</mi></mrow></msub></mrow></math>
</span> is the set of negative labels, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf92">
<math id="m105" overflow="linebreak"><mrow><msub><mi mathvariant="italic">s</mi><mi mathvariant="italic">i</mi></msub></mrow></math>
</span> is the model output score of the <em>i</em>th category. And <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf93">
<math id="m106" overflow="linebreak"><mrow><msub><mi mathvariant="italic">s</mi><mn mathvariant="bold">0</mn></msub></mrow></math>
</span> was set to 0 in our experiment.</p>
<p>Finally, the loss function <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf94">
<math id="m107" overflow="linebreak"><mrow><msub><mi mathvariant="script">L</mi><mrow><mi>t</mi><mi>l</mi><mi>p</mi><mi>r</mi></mrow></msub></mrow></math>
</span> was optimized using the Adam algorithm, and the learning rate was set to 0.005.</p></section></section><section id="s3"><h2 class="pmc_sec_title">3 Results</h2>
<section id="s3-1"><h3 class="pmc_sec_title">3.1 Statistical analysis</h3>
<p>As mentioned in <a href="#s2-1" class="usa-link">Section 2.1</a>, our benchmark dataset comprised 453 drugs, 1,091 ADRs, 101,257 CGIs with 23,644 genes, 15,087,041 GDAs with 53,968 genes, and 23,395 known drug-ADR pairs. The Drug-ADR, Drug-Gene, and ADR-Gene statistics exhibited a long-tail distribution (<a href="#F4" class="usa-link">Figure 4</a>). Specifically, few drugs accounted for a large proportion of all drug-ADR pairs and drug-gene interactions, whereas a large number of drugs were associated with only a small proportion of drug-ADR pairs and drug-gene interactions. Additionally, among these three statistics, the number of positive samples was far less than that of negative samples. A class-balanced sampling method was adopted to address the imbalance in the Drug-ADR dataset. The Jaccard Index was used to calculate the similarity between the Drug-Gene and ADR-Gene datasets.</p>
<figure class="fig xbox font-sm" id="F4"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11931068_fphar-16-1448106-g004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/97c2/11931068/542f68fc7ca2/fphar-16-1448106-g004.jpg" loading="lazy" height="340" width="708" alt="FIGURE 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/F4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Frequency-rank distribution of experimental data. <strong>(A)</strong> presents the frequency-rank distribution of drug and related ADRs; <strong>(B)</strong> presents the frequency-rank distribution of drug and related genes; <strong>(C)</strong> presents the frequency-rank distribution of ADR and related genes.</p></figcaption></figure></section><section id="s3-2"><h3 class="pmc_sec_title">3.2 Cross-validation test of ADR prediction for different combination features</h3>
<p>The ADR prediction model was evaluated through 5-fold cross-validation. The Area Under the Receiver Operating Characteristic Curve (AUROC), Area Under the Precision-Recall Curve (AUPRC), Average Accuracy Percentage (Acc) and Matthews correlation coefficient (MCC) (<a href="#B5" class="usa-link" aria-describedby="B5">Chicco and Jurman, 2023</a>) metrics were used to assess our model’s performance in assigning the correct ADRs to individual drugs. Higher values in both metrics indicated better performance. In each fold, the model was trained using a randomly selected subset of 80% known drug-ADR associations and a matching number of randomly sampled non-associating pairs, with the remaining 20% utilized for testing. The AUROC and AUPRC values of the five folds were then averaged and used as a benchmark for comparison to the other algorithms and optimizing hyperparameters. Herein, seven different data settings for drugs [(i) CS + GE + CGI, (ii) CS + CGI, (iii) CS + GE, (iv) GE + CGI, (v) CS, (vi) CGI, and (vii) GE] and three different data settings for ADRs [(a) MESH + GDA, (b) MESH, and (c) GDA] were used to accomplish a fair comparison.</p>
<section id="s3-2-1"><h4 class="pmc_sec_title">3.2.1 Comparative analysis of genomic descriptors of drugs (GE and CGI)</h4>
<p>A comparable result of GE and CGI was attained when the input feature of ADRs was set to MeSH + GDA (see <a href="#T2" class="usa-link">Table 2</a>). <a href="#F5" class="usa-link">Figure 5</a> shows the AUROC and AUPRC values. According to the results, setting (vi; CGI) had the best performance (AUROC = 92.30 ± 0.35%, AUPRC = 91.80 ± 0.38%, MCC = 71.22 ± 0.66%, Acc = 85.57 ± 0.31), even better than that of setting (i) (CS + GE + CGI; AUROC = 91.75 ± 1.14%, AUPRC = 91.62 ± 0.30%, MCC = 70.49 ± 1.04%, Acc = 85.21 ± 0.54). This finding contradicts the widely held belief that adding more features to a model improves training accuracy. Compared to single feature settings [(v) CS and (vii) GE], the AUROC values of setting (vi; CGI) were higher by 0.42% and 1.41%, respectively. Furthermore, setting (ii; CS + CGI) performed better than setting (v; CS) and setting (iv; GE + CGI) performed better than setting (vii; GE). After adding CGI to CS and GE, the AUROC values increased by 0.34% and 0.48% respectively. Moreover, the AUROC and AUPRC values of setting (i; CS + GE + CGI) were 91.75% and 91.62%, respectively. These values were 0.43% and 0.48% higher than those of setting (iii; CS + GE). On the other hand, the AUROC scores of settings (iii), (iv), and (i) were lower than those of settings (v), (vi), and (ii) (by 0.56%, 0.93%, and 0.47%) respectively). These findings indicate that adding CGI improved model performance significantly in various situations, while adding GE decreased the AUROC and AUPRC values. In this regard, CGI is more informative than GE for ADR prediction, potentially because it contains interactions curated from various resources, providing more comprehensive drug information than GE, which is based solely on the experimental results of the LINC L1000 project.</p>
<section class="tw xbox font-sm" id="T2"><h5 class="obj_head">TABLE 2.</h5>
<div class="caption p"><p>Evaluation results of different drug feature settings with ADR setting (a).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead valign="top"><tr>
<th align="left" rowspan="1" colspan="1">Settings</th>
<th align="left" rowspan="1" colspan="1">Drug feature</th>
<th align="left" rowspan="1" colspan="1">ADR feature</th>
<th align="left" rowspan="1" colspan="1">AUROC(%)</th>
<th align="left" rowspan="1" colspan="1">AUPRC (%)</th>
<th align="left" rowspan="1" colspan="1">Acc(%)</th>
<th align="left" rowspan="1" colspan="1">MCC(%)</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" rowspan="1" colspan="1">i</td>
<td align="left" rowspan="1" colspan="1">CS + GE + CGI</td>
<td align="left" rowspan="1" colspan="1">MESH + GDA</td>
<td align="left" rowspan="1" colspan="1">91.75 ± 1.14</td>
<td align="left" rowspan="1" colspan="1">91.62 ± 0.30</td>
<td align="left" rowspan="1" colspan="1">85.21 ± 0.54</td>
<td align="left" rowspan="1" colspan="1">70.49 ± 1.04</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">ii</td>
<td align="left" rowspan="1" colspan="1">CS + CGI</td>
<td align="left" rowspan="1" colspan="1">MESH + GDA</td>
<td align="left" rowspan="1" colspan="1">92.22 ± 0.46</td>
<td align="left" rowspan="1" colspan="1">91.73 ± 0.49</td>
<td align="left" rowspan="1" colspan="1">85.01 ± 0.60</td>
<td align="left" rowspan="1" colspan="1">70.10 ± 1.20</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">iii</td>
<td align="left" rowspan="1" colspan="1">CS + GE</td>
<td align="left" rowspan="1" colspan="1">MESH + GDA</td>
<td align="left" rowspan="1" colspan="1">91.32 ± 0.66</td>
<td align="left" rowspan="1" colspan="1">91.14 ± 0.22</td>
<td align="left" rowspan="1" colspan="1">84.24 ± 0.41</td>
<td align="left" rowspan="1" colspan="1">68.58 ± 0.84</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">iv</td>
<td align="left" rowspan="1" colspan="1">GE + CGI</td>
<td align="left" rowspan="1" colspan="1">MESH + GDA</td>
<td align="left" rowspan="1" colspan="1">91.37 ± 0.43</td>
<td align="left" rowspan="1" colspan="1">91.11 ± 0.55</td>
<td align="left" rowspan="1" colspan="1">84.10 ± 0.37</td>
<td align="left" rowspan="1" colspan="1">68.26 ± 0.73</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">v</td>
<td align="left" rowspan="1" colspan="1">CS</td>
<td align="left" rowspan="1" colspan="1">MESH + GDA</td>
<td align="left" rowspan="1" colspan="1">91.88 ± 0.29</td>
<td align="left" rowspan="1" colspan="1">
<strong>91.83 ± 0.38</strong>
</td>
<td align="left" rowspan="1" colspan="1">84.82 ± 0.36</td>
<td align="left" rowspan="1" colspan="1">69.67 ± 0.72</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">vi</td>
<td align="left" rowspan="1" colspan="1">CGI</td>
<td align="left" rowspan="1" colspan="1">MESH + GDA</td>
<td align="left" rowspan="1" colspan="1">
<strong>92.30 ± 0.35</strong>
</td>
<td align="left" rowspan="1" colspan="1">91.80 ± 0.38</td>
<td align="left" rowspan="1" colspan="1">
<strong>85.57 ± 0.31</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>71.22 ± 0.66</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">vii</td>
<td align="left" rowspan="1" colspan="1">GE</td>
<td align="left" rowspan="1" colspan="1">MESH + GDA</td>
<td align="left" rowspan="1" colspan="1">90.89 ± 0.65</td>
<td align="left" rowspan="1" colspan="1">90.78 ± 0.25</td>
<td align="left" rowspan="1" colspan="1">83.85 ± 0.46</td>
<td align="left" rowspan="1" colspan="1">67.88 ± 0.86</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="fn3"><p>The best performance is highlighted in bold.</p></div></div></section><figure class="fig xbox font-sm" id="F5"><h5 class="obj_head">FIGURE 5.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11931068_fphar-16-1448106-g005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/97c2/11931068/aaa87dcf3d8c/fphar-16-1448106-g005.jpg" loading="lazy" height="385" width="708" alt="FIGURE 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/F5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>AUROC and AUPRC of DGANet with the fixed ADR feature setting (a) and seven different drug feature settings. <strong>(A)</strong> presents the AUROC curves of the comparison; <strong>(B)</strong> presents the AUPRC curves of the comparison.</p></figcaption></figure></section><section id="s3-2-2"><h4 class="pmc_sec_title">3.2.2 Evaluation of the effect of GDA-based ADR similarity on DGANet</h4>
<p>Similar evaluation experiments were performed with ADR settings (b) and (c). The results are shown in <a href="#T3" class="usa-link">Tables 3</a>, <a href="#T4" class="usa-link">4</a>. <a href="#F6" class="usa-link">Figures 6</a>, <a href="#F7" class="usa-link">7</a> show the AUROC and AUPRC, respectively. A comparison of <a href="#T2" class="usa-link">Tables 2</a>–<a href="#T4" class="usa-link">4</a> revealed that ADR setting (a) had the highest scores, followed by setting (b), and setting (c) scored lowest. The gap was largest when the drug characteristics were set as CS + CGI. The AUROC values of setting (c) were 4.28% and 4.0% lower than those of settings (a) and (b), respectively. The AUROC and AUPRC values of <a href="#T2" class="usa-link">Tables 2</a>, <a href="#T3" class="usa-link">3</a> were both higher than 89.19%, whereas the highest AUROC and AUPRC scores in <a href="#T4" class="usa-link">Table 4</a> were 88.17% and 86.62%, respectively. In addition, the MCC values in <a href="#T2" class="usa-link">Tables 2</a>, <a href="#T3" class="usa-link">3</a> exceeded 67.88%, whereas the MCC values in <a href="#T4" class="usa-link">Table 4</a> were lower than 60%. Furthermore, MeSH was superior to GDA, with the integration of MeSH and GDA achieving the best outcome. In conclusion, GDA could improve the ADR prediction performance but had less information than the semantic similarity of ARs themselves.</p>
<section class="tw xbox font-sm" id="T3"><h5 class="obj_head">TABLE 3.</h5>
<div class="caption p"><p>Evaluation results of different drug feature settings with ADR setting (b).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead valign="top"><tr>
<th align="left" rowspan="1" colspan="1">Settings</th>
<th align="left" rowspan="1" colspan="1">Drug feature</th>
<th align="left" rowspan="1" colspan="1">ADR feature</th>
<th align="left" rowspan="1" colspan="1">AUROC(%)</th>
<th align="left" rowspan="1" colspan="1">AUPRC(%)</th>
<th align="left" rowspan="1" colspan="1">Acc(%)</th>
<th align="left" rowspan="1" colspan="1">MCC(%)</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" rowspan="1" colspan="1">i</td>
<td align="left" rowspan="1" colspan="1">CS + GE + CGI</td>
<td align="left" rowspan="1" colspan="1">MESH</td>
<td align="left" rowspan="1" colspan="1">90.19 ± 2.25</td>
<td align="left" rowspan="1" colspan="1">90.87 ± 0.88</td>
<td align="left" rowspan="1" colspan="1">84.50 ± 0.80</td>
<td align="left" rowspan="1" colspan="1">69.08 ± 1.63</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">ii</td>
<td align="left" rowspan="1" colspan="1">CS + CGI</td>
<td align="left" rowspan="1" colspan="1">MESH</td>
<td align="left" rowspan="1" colspan="1">
<strong>91.94 ± 0.42</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>91.74 ± 0.39</strong>
</td>
<td align="left" rowspan="1" colspan="1">84.90 ± 0.40</td>
<td align="left" rowspan="1" colspan="1">69.86 ± 0.82</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">iii</td>
<td align="left" rowspan="1" colspan="1">CS + GE</td>
<td align="left" rowspan="1" colspan="1">MESH</td>
<td align="left" rowspan="1" colspan="1">90.00 ± 1.88</td>
<td align="left" rowspan="1" colspan="1">91.01 ± 0.55</td>
<td align="left" rowspan="1" colspan="1">84.39 ± 0.46</td>
<td align="left" rowspan="1" colspan="1">68.83 ± 0.92</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">iv</td>
<td align="left" rowspan="1" colspan="1">GE + CGI</td>
<td align="left" rowspan="1" colspan="1">MESH</td>
<td align="left" rowspan="1" colspan="1">90.41 ± 1.07</td>
<td align="left" rowspan="1" colspan="1">91.03 ± 0.34</td>
<td align="left" rowspan="1" colspan="1">84.15 ± 0.25</td>
<td align="left" rowspan="1" colspan="1">68.38 ± 0.49</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">v</td>
<td align="left" rowspan="1" colspan="1">CS</td>
<td align="left" rowspan="1" colspan="1">MESH</td>
<td align="left" rowspan="1" colspan="1">91.50 ± 0.30</td>
<td align="left" rowspan="1" colspan="1">91.43 ± 0.34</td>
<td align="left" rowspan="1" colspan="1">84.48 ± 0.35</td>
<td align="left" rowspan="1" colspan="1">69.06 ± 0.72</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">vi</td>
<td align="left" rowspan="1" colspan="1">CGI</td>
<td align="left" rowspan="1" colspan="1">MESH</td>
<td align="left" rowspan="1" colspan="1">91.70 ± 0.50</td>
<td align="left" rowspan="1" colspan="1">91.63 ± 0.22</td>
<td align="left" rowspan="1" colspan="1">
<strong>84.93 ± 0.29</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>69.93 ± 0.58</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">vii</td>
<td align="left" rowspan="1" colspan="1">GE</td>
<td align="left" rowspan="1" colspan="1">MESH</td>
<td align="left" rowspan="1" colspan="1">89.19 ± 2.35</td>
<td align="left" rowspan="1" colspan="1">90.58 ± 1.04</td>
<td align="left" rowspan="1" colspan="1">84.11 ± 0.79</td>
<td align="left" rowspan="1" colspan="1">68.29 ± 1.55</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="fn4"><p>The best performance is highlighted in bold.</p></div></div></section><section class="tw xbox font-sm" id="T4"><h5 class="obj_head">TABLE 4.</h5>
<div class="caption p"><p>Evaluation results of different drug feature settings with ADR setting (c).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead valign="top"><tr>
<th align="left" rowspan="1" colspan="1">Settings</th>
<th align="left" rowspan="1" colspan="1">Drug feature</th>
<th align="left" rowspan="1" colspan="1">ADR feature</th>
<th align="left" rowspan="1" colspan="1">AUROC(%)</th>
<th align="left" rowspan="1" colspan="1">AUPRC(%)</th>
<th align="left" rowspan="1" colspan="1">Acc(%)</th>
<th align="left" rowspan="1" colspan="1">MCC(%)</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" rowspan="1" colspan="1">i</td>
<td align="left" rowspan="1" colspan="1">CS + GE + CGI</td>
<td align="left" rowspan="1" colspan="1">GDA</td>
<td align="left" rowspan="1" colspan="1">
<strong>88.17 ± 0.43</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>87.66 ± 0.56</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>79.90 ± 0.43</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>59.98 ± 0.92</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">ii</td>
<td align="left" rowspan="1" colspan="1">CS + CGI</td>
<td align="left" rowspan="1" colspan="1">GDA</td>
<td align="left" rowspan="1" colspan="1">87.94 ± 0.32</td>
<td align="left" rowspan="1" colspan="1">87.52 ± 0.49</td>
<td align="left" rowspan="1" colspan="1">79.80 ± 0.34</td>
<td align="left" rowspan="1" colspan="1">59.83 ± 0.73</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">iii</td>
<td align="left" rowspan="1" colspan="1">CS + GE</td>
<td align="left" rowspan="1" colspan="1">GDA</td>
<td align="left" rowspan="1" colspan="1">87.69 ± 0.47</td>
<td align="left" rowspan="1" colspan="1">87.26 ± 0.54</td>
<td align="left" rowspan="1" colspan="1">79.34 ± 0.45</td>
<td align="left" rowspan="1" colspan="1">58.74 ± 0.88</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">iv</td>
<td align="left" rowspan="1" colspan="1">GE + CGI</td>
<td align="left" rowspan="1" colspan="1">GDA</td>
<td align="left" rowspan="1" colspan="1">87.82 ± 0.60</td>
<td align="left" rowspan="1" colspan="1">87.29 ± 0.69</td>
<td align="left" rowspan="1" colspan="1">79.40 ± 0.72</td>
<td align="left" rowspan="1" colspan="1">58.94 ± 1.38</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">v</td>
<td align="left" rowspan="1" colspan="1">CS</td>
<td align="left" rowspan="1" colspan="1">GDA</td>
<td align="left" rowspan="1" colspan="1">87.40 ± 0.81</td>
<td align="left" rowspan="1" colspan="1">87.02 ± 0.91</td>
<td align="left" rowspan="1" colspan="1">79.32 ± 0.98</td>
<td align="left" rowspan="1" colspan="1">58.68 ± 1.95</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">vi</td>
<td align="left" rowspan="1" colspan="1">CGI</td>
<td align="left" rowspan="1" colspan="1">GDA</td>
<td align="left" rowspan="1" colspan="1">87.74 ± 0.56</td>
<td align="left" rowspan="1" colspan="1">87.15 ± 0.66</td>
<td align="left" rowspan="1" colspan="1">79.46 ± 0.68</td>
<td align="left" rowspan="1" colspan="1">59.09 ± 1.30</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">vii</td>
<td align="left" rowspan="1" colspan="1">GE</td>
<td align="left" rowspan="1" colspan="1">GDA</td>
<td align="left" rowspan="1" colspan="1">87.09 ± 0.31</td>
<td align="left" rowspan="1" colspan="1">86.62 ± 0.46</td>
<td align="left" rowspan="1" colspan="1">78.77 ± 0.25</td>
<td align="left" rowspan="1" colspan="1">57.57 ± 0.50</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="fn5"><p>The best performance is highlighted in bold.</p></div></div></section><figure class="fig xbox font-sm" id="F6"><h5 class="obj_head">FIGURE 6.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11931068_fphar-16-1448106-g006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/97c2/11931068/2d5d126960ce/fphar-16-1448106-g006.jpg" loading="lazy" height="385" width="708" alt="FIGURE 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/F6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>AUROC and AUPRC of DGANet with the fixed ADR feature setting (b) and seven different drug feature settings. <strong>(A)</strong> presents the AUROC curves of the comparison; <strong>(B)</strong> presents the AUPRC curves of the comparison.</p></figcaption></figure><figure class="fig xbox font-sm" id="F7"><h5 class="obj_head">FIGURE 7.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11931068_fphar-16-1448106-g007.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/97c2/11931068/8790c9a7346a/fphar-16-1448106-g007.jpg" loading="lazy" height="385" width="708" alt="FIGURE 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/F7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>AUROC and AUPRC of DGANet with the fixed ADR feature setting (c) and seven different drug feature settings. <strong>(A)</strong> presents the AUROC curves of the comparison; <strong>(B)</strong> presents the AUPRC curves of the comparison.</p></figcaption></figure></section></section><section id="s3-3"><h3 class="pmc_sec_title">3.3 Performance improvement in ADR prediction by adding similarity based on existing DSAs</h3>
<p>Multiple studies (<a href="#B23" class="usa-link" aria-describedby="B23">Poleksic and Xie, 2018</a>; <a href="#B37" class="usa-link" aria-describedby="B37">Zhao et al., 2022</a>; <a href="#B38" class="usa-link" aria-describedby="B38">Zhao et al., 2021</a>) have recently shown that embedding neighborhood similarity of known DSAs can improve ADR prediction accuracy. However, this approach could easily lead to train-test contamination. Herein, the samples in the test set were set to zero before feature construction in each fold to avoid train-test contamination. In this regard, the model could learn nothing about the test set in the training phase. The results are shown in <a href="#T5" class="usa-link">Tables 5</a>. Compared to the results in <a href="#T2" class="usa-link">Table 2</a>, the top 3 settings (ii), (v), and (vi) exhibited increased AUROC (by 0.77%, 0.54%, and 0.44%, respectively), AUPRC (by 0.85%, 0.75%, and 0.85%, respectively), Acc (by 0.77%, 0.54%, and 0.44%, respectively) and MCC (by 0.77%, 0.54%, and 0.44%, respectively) values after adding similarity based on existing DSAs. The AUROC and AUPRC are presented in the <a href="#s11" class="usa-link">Supplementary Figures S1–S3</a> illustrates the learning rate curves of our model. This finding indicates that integrating neighborhood similarity associated with known DSAs can improve the model’s performance. Notably, this method relies significantly on existing drug-ADR associations, and the information on new developed drugs and ADRs may be incomplete. Nonetheless, it still demonstrates high practicality and accuracy in predicting drug reuse ADRs and severe rare ADRs (<a href="#B23" class="usa-link" aria-describedby="B23">Poleksic and Xie, 2018</a>).</p>
<section class="tw xbox font-sm" id="T5"><h4 class="obj_head">TABLE 5.</h4>
<div class="caption p"><p>Evaluation results of DGANet after adding existing drug-ADR associations to predict new ADRs.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead valign="top"><tr>
<th align="center" rowspan="1" colspan="1">Settings</th>
<th align="center" rowspan="1" colspan="1">Drug feature</th>
<th align="center" rowspan="1" colspan="1">ADR feature</th>
<th align="center" rowspan="1" colspan="1">AUROC(%)</th>
<th align="center" rowspan="1" colspan="1">AUPRC (%)</th>
<th align="center" rowspan="1" colspan="1">Acc(%)</th>
<th align="center" rowspan="1" colspan="1">MCC(%)</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" rowspan="1" colspan="1">i</td>
<td align="left" rowspan="1" colspan="1">CS + GE + CGI + DSAs</td>
<td align="left" rowspan="1" colspan="1">MESH + GDA + DSAs</td>
<td align="left" rowspan="1" colspan="1">92.66 ± 0.50</td>
<td align="left" rowspan="1" colspan="1">92.39 ± 1.03</td>
<td align="left" rowspan="1" colspan="1">
<strong>85.95 ± 0.56</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>71.95 ± 1.16</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">ii</td>
<td align="left" rowspan="1" colspan="1">CS + CGI + DSAs</td>
<td align="left" rowspan="1" colspan="1">MESH + GDA + DSAs</td>
<td align="left" rowspan="1" colspan="1">
<strong>92.76 ± 0.37</strong>
</td>
<td align="left" rowspan="1" colspan="1">92.49 ± 0.52</td>
<td align="left" rowspan="1" colspan="1">85.89 ± 0.33</td>
<td align="left" rowspan="1" colspan="1">71.84 ± 0.64</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">iii</td>
<td align="left" rowspan="1" colspan="1">CS + GE + DSAs</td>
<td align="left" rowspan="1" colspan="1">MESH + GDA + DSAs</td>
<td align="left" rowspan="1" colspan="1">92.49 ± 0.64</td>
<td align="left" rowspan="1" colspan="1">92.37 ± 0.72</td>
<td align="left" rowspan="1" colspan="1">85.43 ± 0.84</td>
<td align="left" rowspan="1" colspan="1">70.92 ± 1.69</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">iv</td>
<td align="left" rowspan="1" colspan="1">GE + CGI + DSAs</td>
<td align="left" rowspan="1" colspan="1">MESH + GDA + DSAs</td>
<td align="left" rowspan="1" colspan="1">92.55 ± 0.28</td>
<td align="left" rowspan="1" colspan="1">92.26 ± 0.49</td>
<td align="left" rowspan="1" colspan="1">85.65 ± 0.28</td>
<td align="left" rowspan="1" colspan="1">71.38 ± 0.57</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">v</td>
<td align="left" rowspan="1" colspan="1">CS + DSAs</td>
<td align="left" rowspan="1" colspan="1">MESH + GDA + DSAs</td>
<td align="left" rowspan="1" colspan="1">92.68 ± 0.31</td>
<td align="left" rowspan="1" colspan="1">92.58 ± 0.45</td>
<td align="left" rowspan="1" colspan="1">85.77 ± 0.49</td>
<td align="left" rowspan="1" colspan="1">71.64 ± 0.99</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">vi</td>
<td align="left" rowspan="1" colspan="1">CGI + DSAs</td>
<td align="left" rowspan="1" colspan="1">MESH + GDA + DSAs</td>
<td align="left" rowspan="1" colspan="1">92.74 ± 0.38</td>
<td align="left" rowspan="1" colspan="1">
<strong>92.65 ± 0.50</strong>
</td>
<td align="left" rowspan="1" colspan="1">85.78 ± 0.47</td>
<td align="left" rowspan="1" colspan="1">71.60 ± 0.94</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">vii</td>
<td align="left" rowspan="1" colspan="1">GE + DSAs</td>
<td align="left" rowspan="1" colspan="1">MESH + GDA + DSAs</td>
<td align="left" rowspan="1" colspan="1">92.26 ± 1.42</td>
<td align="left" rowspan="1" colspan="1">92.30 ± 1.03</td>
<td align="left" rowspan="1" colspan="1">85.63 ± 1.06</td>
<td align="left" rowspan="1" colspan="1">71.31 ± 2.13</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="fn6"><p>The best performance is highlighted in bold.</p></div></div></section></section><section id="s3-4"><h3 class="pmc_sec_title">3.4 Performance comparison between DGANet and the state-of-the-art ADR prediction models with pharmacogenoimic features</h3>
<p>To evaluate the performance of the DGANet model, we compared it with three state-of-the-art methods with pharmacogenomic features, including Wang’s method (<a href="#B33" class="usa-link" aria-describedby="B33">Wang et al., 2016</a>), MMNN. Sum (<a href="#B32" class="usa-link" aria-describedby="B32">Üner et al., 2023</a>), DruGNN (<a href="#B3" class="usa-link" aria-describedby="B3">Bongini et al., 2023</a>) and BiMPADR (<a href="#B18" class="usa-link" aria-describedby="B18">Li et al., 2024</a>). <a href="#T6" class="usa-link">Table 6</a> shows the comparison results. In Wang’s method and MMNN. Sum, ADRs associated with fewer than ten drugs were excluded, and the remaining drug-ADR datasets used in experiments remained imbalanced, leading to significantly lower AUPRC scores compared to AUROC scores. Our method (DGANet) addressed this imbalance by employing class-balanced sampling to achieve a balanced train and test dataset. Despite utilizing a smaller number of drugs and ADRs, our method still demonstrated notable improvements of 4.6% and 5.06% in AUROC score, respectively, with and without the incorporation of neighborhood similarity based on existing DSAs. While DGANet’s overall accuracy (85.95%) was slightly lower than DruGNN (86.3%), which was achieved among common ADRs (each drug may cause over 360 ADRs), DGANet’s accuracy surpassed the other Acc scores achieved by DruGNN when dealing with less common ADRs. Compared to BiMPADR, DGANet exhibited significant improvements of 3.36% and 4.05% in AUROC score and overall accuracy, respectively.</p>
<section class="tw xbox font-sm" id="T6"><h4 class="obj_head">TABLE 6.</h4>
<div class="caption p"><p>Performance comparison of different models in the ADR prediction task with pharmacogenomics data.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead valign="top"><tr>
<th align="center" rowspan="1" colspan="1">Dataset</th>
<th align="center" rowspan="1" colspan="1">Model</th>
<th align="center" rowspan="1" colspan="1">Drug<br>Features</th>
<th align="center" rowspan="1" colspan="1">ADR<br>Features</th>
<th align="center" rowspan="1" colspan="1">#Drug<br>/ADR</th>
<th align="center" rowspan="1" colspan="1">AUROC (%)</th>
<th align="center" rowspan="1" colspan="1">AUPRC (%)</th>
<th align="center" rowspan="1" colspan="1">ACC (%)</th>
</tr></thead>
<tbody valign="top">
<tr>
<td rowspan="2" align="left" colspan="1">LINCS L1000 &amp;SIDER</td>
<td align="left" rowspan="1" colspan="1">Wang et al.’s Method</td>
<td align="left" rowspan="1" colspan="1">GO + CS</td>
<td align="left" rowspan="1" colspan="1">—</td>
<td align="left" rowspan="1" colspan="1">791/1,053</td>
<td align="left" rowspan="1" colspan="1">85.40</td>
<td align="left" rowspan="1" colspan="1">—</td>
<td align="left" rowspan="1" colspan="1">—</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">MMNN.Sum</td>
<td align="left" rowspan="1" colspan="1">CS+[GEX, META]</td>
<td align="left" rowspan="1" colspan="1">—</td>
<td align="left" rowspan="1" colspan="1">791/1,053</td>
<td align="left" rowspan="1" colspan="1">87.70</td>
<td align="left" rowspan="1" colspan="1">59.20</td>
<td align="left" rowspan="1" colspan="1">—</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">STITCH&amp; SIDER</td>
<td align="left" rowspan="1" colspan="1">DruGNN</td>
<td align="left" rowspan="1" colspan="1">Drug-Gene graph</td>
<td align="left" rowspan="1" colspan="1">—</td>
<td align="left" rowspan="1" colspan="1">1,341/360</td>
<td align="left" rowspan="1" colspan="1">—</td>
<td align="left" rowspan="1" colspan="1">—</td>
<td align="left" rowspan="1" colspan="1">
<strong>86.30</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">LINCS L1000 &amp;SIDER&amp; ADReCS</td>
<td align="left" rowspan="1" colspan="1">BiMPADR</td>
<td align="left" rowspan="1" colspan="1">Drug fingerprints<br>+GE</td>
<td align="left" rowspan="1" colspan="1">ADR-Gene association</td>
<td align="left" rowspan="1" colspan="1">656/751</td>
<td align="left" rowspan="1" colspan="1">89.4</td>
<td align="left" rowspan="1" colspan="1">—</td>
<td align="left" rowspan="1" colspan="1">81.9</td>
</tr>
<tr>
<td rowspan="2" align="left" colspan="1">LINCS L1000 &amp;SIDER&amp;CTD</td>
<td rowspan="2" align="left" colspan="1">DGANet</td>
<td align="left" rowspan="1" colspan="1">CGI</td>
<td align="left" rowspan="1" colspan="1">MESH + GDA</td>
<td align="left" rowspan="1" colspan="1">453/1,019</td>
<td align="left" rowspan="1" colspan="1">92.30</td>
<td align="left" rowspan="1" colspan="1">91.87</td>
<td align="left" rowspan="1" colspan="1">85.57</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">CS + GE<br>+DSAs</td>
<td align="left" rowspan="1" colspan="1">MESH + GDA + DSAs</td>
<td align="left" rowspan="1" colspan="1">453/1,019</td>
<td align="left" rowspan="1" colspan="1">
<strong>92.76</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>92.49</strong>
</td>
<td align="left" rowspan="1" colspan="1">85.95</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="fn7"><p>The best performance is highlighted in bold.</p></div></div></section></section><section id="s3-5"><h3 class="pmc_sec_title">3.5 Literature evidence supports high ranked drug-induced ADRs</h3>
<p>To further assess the performance of DGANet in identifying potential drug-ADR associations, case studies on the top 20 candidate drug-ADR associations unrecorded in SIDER were collected for validation and analysis (<a href="#s11" class="usa-link">Supplementary Table S3</a>). Moreover, we found that several indications were mistakenly predicted as ADR, such as Misoprostol and Pruritus. Among the 20 drug-ADR associations, 19 were included in MetaADEDB (<a href="#B34" class="usa-link" aria-describedby="B34">Yu et al., 2021a</a>) and OFFSIDES (<a href="#B31" class="usa-link" aria-describedby="B31">Tatonetti et al., 2012</a>), suggesting that the drug-induced ADRs were indeed associated with the corresponding drugs. And we collected the related genes from CTD database for 10 drug-ADR pairs, which might contribute to the occurrence of ADRs. The drug-induced ADRs labeled “Literature” were reported by published literature, indicating that they were not recorded in the adverse reaction databases, but their association has been reported previously (<a href="#B35" class="usa-link" aria-describedby="B35">Yu et al., 2021b</a>).</p></section></section><section id="s4"><h2 class="pmc_sec_title">4 Discussion</h2>
<p>Pharmacogenomics incorporates genomic profiling to identify biomarkers based on relevant genotype–phenotype interactions that can predict drug response and risk of ADRs. Using pharmacogenomics to predict adverse reactions can help improve the safety and effectiveness of medical care. To address the growing volume of complex, open-source pharmacogenomics data, artificial intelligence (AI) algorithms capable of large-scale computation and high-performance statistical analysis are essential. This paper introduces deep learning methods into the exploratory research of pharmacogenomics data analysis, leveraging the renowned CTD database, and proposes two characteristics for drugs and ADRs (CGI and GDA) and an intelligent prediction model (DGANet) for ADRs prediction based on the characteristics of pharmacogenomics data.</p>
<p>Initially, we curated a benchmark dataset comprising 453 drugs, 1,091 ADRs, 101,257 CGIs with 23,644 genes, 15,087,041 GDAs with 53,968 genes, and 23,395 known drug-ADR pairs from SIDER, LINCS L1000, CTD, PubChem, and MeSH source databases. Notably, CGIs and GDAs are the characteristic expression patterns we initially proposed to represent the relationships between drugs and genes, as well as adverse reactions and genes. In the evaluation experiments of ADR prediction, we compared these two pharmacogenomics features with traditional drug features (such as GE and CS). The five-fold cross-validation experimental results demonstrated that both of these pharmacogenomics features exhibit a significant tendency to enhance the predictive performance of ADRs. Particularly, CGI consistently outperformed the widely used drug features (GE and CS) in our experiments, and CGI could enhance model performance when combined with other drug features (<a href="#T2" class="usa-link">Tables 2</a>–<a href="#T5" class="usa-link">5</a>). Generally, the combination of CS and CGI achieved the highest AUROC and AUPRC values. However, we also observed some variations. In <a href="#T2" class="usa-link">Table 2</a>, the scores of CGI even surpassed those of the combined CS and CGI. We posit that this discrepancy might be attributed to the presence of coincidental information extracted from the same research between CGI and GDA.</p>
<p>Secondly, we proposed an intelligent model (DGANet) for predicting ADRs which achieved pharmacogenomics information fusion across all of drug, genomic and ADR features automatically. Compared with several state-of-the-art models based on different fusion methods of pharmacogenomics features and classifiers, the DGANet exhibited the highest performance in AUROC and AUPRC (<a href="#T6" class="usa-link">Table 6</a>). Case studies provide specific examples that further demonstrate the validity and practicality of the GDANet (<a href="#s11" class="usa-link">Supplementary Table S3</a>).</p>
<p>Despite being a preliminary study, our proposed DGANet model still has room for improvement in terms of accuracy and effectiveness. One limitation is DGANet’s inability to fully elucidate the diverse nature of drug-gene interactions and gene-disease associations, which is critical as the complex biological processes underlying drug responses are heavily influenced by specific gene expression variations. Another limitation is the lack of a comprehensive description of biological diversity within the currently utilized pharmacogenomics data, which poses a significant obstacle for all ADR prediction research methods.</p></section><section id="s5"><h2 class="pmc_sec_title">5 Conclusion</h2>
<p>In this study, we proposed DGANet, a new CNN-based model that integrates CGI and drug-perturbed GE changes (GE) into drug feature, GDA into ADR feature, and has achieved compelling results. The result showed that the two noval characteristics (CGI and GDA) we proposed both have an enhancing effect on the model. However, this study represents a preliminary investigation. By leveraging pharmacogenomics information and predicting adverse drug reactions, DGANet contributes to understanding how drugs influence gene expression and biological pathways that may lead to adverse reactions, offering valuable insights for drug safety research. Nevertheless, its mechanism is still unclear and further research is needed. More factors related to ADRs, such as gender, phenotype, dosage, etc., should be taken into account. With the latest advances in genomics and precision medicine, as well as regulatory guidance in pharmacogenomics, we believe that pharmacogenomics biomarkers will become increasingly common in all therapeutic fields. We anticipate that with ongoing data updates and the expansion of available databases, a richer pool of pharmacogenomics information will become accessible for future research into ADR prediction methods. In future studies, we plan to integrate more pharmacogenomics information into our model, such as Chemical–GO enriched associations, Chemical–pathway enriched associations and so on. The aim will be to find a more effective algorithm and corresponding feature construction methods to predict ADR more effectively and accurately.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>The author(s) declare that financial support was received for the research, authorship, and/or publication of this article. This paper was funded by the Guangzhou Science and Technology Bureau 2025 Municipal School (Institute) Enterprise Joint Funding Project “Research and Development of Precision Medication Platform Based on Pharmacogenomics Data“ (No. 2025A03J3712).</p></section><section id="s6"><h2 class="pmc_sec_title">Data availability statement</h2>
<p>Publicly available datasets were analyzed in this study. This data can be found here: <a href="https://github.com/hemingxiu/DGANet" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/hemingxiu/DGANet</a>.</p></section><section id="s7"><h2 class="pmc_sec_title">Author contributions</h2>
<p>MH: Data curation, Methodology, Software, Validation, Writing–original draft, Writing–review and editing. YS: Methodology, Software, Visualization, Writing–review and editing. FH: Conceptualization, Writing–review and editing. YC: Conceptualization, Supervision, Writing–review and editing.</p></section><section id="s9"><h2 class="pmc_sec_title">Conflict of interest</h2>
<p>The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p></section><section id="s10"><h2 class="pmc_sec_title">Publisher’s note</h2>
<p>All claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.</p></section><section id="s11"><h2 class="pmc_sec_title">Supplementary material</h2>
<p>The Supplementary Material for this article can be found online at: <a href="https://www.frontiersin.org/articles/10.3389/fphar.2025.1448106/full#supplementary-material" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.frontiersin.org/articles/10.3389/fphar.2025.1448106/full#supplementary-material</a>
</p>
<section class="sm xbox font-sm" id="SM1"><div class="media p"><div class="caption">
<a href="/articles/instance/11931068/bin/DataSheet1.pdf" data-ga-action="click_feat_suppl" class="usa-link">DataSheet1.pdf</a><sup> (160.6KB, pdf) </sup>
</div></div></section></section><section id="s12"><h2 class="pmc_sec_title">Abbreviations</h2>
<p>ADR, Adverse Drug Reactions; DDI, Drug‒Drug Interaction; CS, Chemical Structure; GE, Gene Expression; GDA, Gene-Disease Association; CGI, Chemical-Gene Interactions; DSA, Drug-Side Effect Association; LSN, Linear Subnetworks; CSN, Convolutional Neural Network with a Subnetwork; AUROC, Area under the receiver operating characteristic curve; AUPRC, Area under the precision-recall curve.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="B1">
<cite>
Amberger J. S., Hamosh A. (2017). Searching online mendelian inheritance in man (OMIM): a knowledgebase of human genes and genetic phenotypes. Curr. Protoc. Bioinforma.
58, 1. 10.1002/cpbi.27
</cite> [<a href="https://doi.org/10.1002/cpbi.27" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5662200/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28654725/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Protoc.%20Bioinforma.&amp;title=Searching%20online%20mendelian%20inheritance%20in%20man%20(OMIM):%20a%20knowledgebase%20of%20human%20genes%20and%20genetic%20phenotypes&amp;author=J.%20S.%20Amberger&amp;author=A.%20Hamosh&amp;volume=58&amp;publication_year=2017&amp;pages=1&amp;pmid=28654725&amp;doi=10.1002/cpbi.27&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B2">
<cite>
Bodenreider O. (2004). The unified Medical Language System (UMLS): integrating biomedical terminology. Nucleic Acids Res.
32, D267–D270. 10.1093/nar/gkh061

</cite> [<a href="https://doi.org/10.1093/nar/gkh061" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC308795/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14681409/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=The%20unified%20Medical%20Language%20System%20(UMLS):%20integrating%20biomedical%20terminology&amp;author=O.%20Bodenreider&amp;volume=32&amp;publication_year=2004&amp;pages=D267-D270&amp;pmid=14681409&amp;doi=10.1093/nar/gkh061&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B3">
<cite>
Bongini P., Scarselli F., Bianchini M., Dimitri G. M., Pancino N., Lió P. (2023). Modular multi–source prediction of drug side–effects with DruGNN. IEEE/ACM Trans. Comput. Biol. Bioinforma.
20, 1211–1220. 10.1109/TCBB.2022.3175362
</cite> [<a href="https://doi.org/10.1109/TCBB.2022.3175362" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35576419/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE/ACM%20Trans.%20Comput.%20Biol.%20Bioinforma.&amp;title=Modular%20multi%E2%80%93source%20prediction%20of%20drug%20side%E2%80%93effects%20with%20DruGNN&amp;author=P.%20Bongini&amp;author=F.%20Scarselli&amp;author=M.%20Bianchini&amp;author=G.%20M.%20Dimitri&amp;author=N.%20Pancino&amp;volume=20&amp;publication_year=2023&amp;pages=1211-1220&amp;pmid=35576419&amp;doi=10.1109/TCBB.2022.3175362&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B4">
<cite>
Cacabelos R., Cacabelos N., Carril J. C. (2019). The role of pharmacogenomics in adverse drug reactions. Expert Rev. Clin. Pharmacol.
12, 407–442. 10.1080/17512433.2019.1597706

</cite> [<a href="https://doi.org/10.1080/17512433.2019.1597706" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30916581/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Expert%20Rev.%20Clin.%20Pharmacol.&amp;title=The%20role%20of%20pharmacogenomics%20in%20adverse%20drug%20reactions&amp;author=R.%20Cacabelos&amp;author=N.%20Cacabelos&amp;author=J.%20C.%20Carril&amp;volume=12&amp;publication_year=2019&amp;pages=407-442&amp;pmid=30916581&amp;doi=10.1080/17512433.2019.1597706&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B5">
<cite>
Chicco D., Jurman G. (2023). The Matthews correlation coefficient (MCC) should replace the ROC AUC as the standard metric for assessing binary classification. BioData Min.
16 (1), 4. 10.1186/s13040-023-00322-4

</cite> [<a href="https://doi.org/10.1186/s13040-023-00322-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9938573/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36800973/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BioData%20Min.&amp;title=The%20Matthews%20correlation%20coefficient%20(MCC)%20should%20replace%20the%20ROC%20AUC%20as%20the%20standard%20metric%20for%20assessing%20binary%20classification&amp;author=D.%20Chicco&amp;author=G.%20Jurman&amp;volume=16&amp;issue=1&amp;publication_year=2023&amp;pages=4&amp;pmid=36800973&amp;doi=10.1186/s13040-023-00322-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B6">
<cite>
Clark N. R., Hu K. S., Feldmann A. S., Kou Y., Chen E. Y., Duan Q., et al.  (2014). The characteristic direction: a geometrical approach to identify differentially expressed genes. BMC Bioinforma.
15, 79–16. 10.1186/1471-2105-15-79
</cite> [<a href="https://doi.org/10.1186/1471-2105-15-79" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4000056/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24650281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinforma.&amp;title=The%20characteristic%20direction:%20a%20geometrical%20approach%20to%20identify%20differentially%20expressed%20genes&amp;author=N.%20R.%20Clark&amp;author=K.%20S.%20Hu&amp;author=A.%20S.%20Feldmann&amp;author=Y.%20Kou&amp;author=E.%20Y.%20Chen&amp;volume=15&amp;publication_year=2014&amp;pages=79-16&amp;pmid=24650281&amp;doi=10.1186/1471-2105-15-79&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B7">
<cite>
Das P., Mazumder D. H. (2023). An extensive survey on the use of supervised machine learning techniques in the past two decades for prediction of drug side effects. Artif. Intell. Rev.
56, 9809–9836. 10.1007/s10462-023-10413-7
</cite> [<a href="https://doi.org/10.1007/s10462-023-10413-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9930028/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36819660/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Artif.%20Intell.%20Rev.&amp;title=An%20extensive%20survey%20on%20the%20use%20of%20supervised%20machine%20learning%20techniques%20in%20the%20past%20two%20decades%20for%20prediction%20of%20drug%20side%20effects&amp;author=P.%20Das&amp;author=D.%20H.%20Mazumder&amp;volume=56&amp;publication_year=2023&amp;pages=9809-9836&amp;pmid=36819660&amp;doi=10.1007/s10462-023-10413-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B8">
<cite>
Davis A. P., Wiegers T. C., Johnson R. J., Sciaky D., Wiegers J., Mattingly C. J. (2023). Comparative Toxicogenomics database (CTD): update 2023. Nucleic Acids Res.
51, D1257–D1262. 10.1093/nar/gkac833

</cite> [<a href="https://doi.org/10.1093/nar/gkac833" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36169237/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=Comparative%20Toxicogenomics%20database%20(CTD):%20update%202023&amp;author=A.%20P.%20Davis&amp;author=T.%20C.%20Wiegers&amp;author=R.%20J.%20Johnson&amp;author=D.%20Sciaky&amp;author=J.%20Wiegers&amp;volume=51&amp;publication_year=2023&amp;pages=D1257-D1262&amp;pmid=36169237&amp;doi=10.1093/nar/gkac833&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B9">
<cite>
Durant J. L., Leland B. A., Henry D. R., Nourse J. G. (2002). Reoptimization of MDL keys for use in drug discovery. Am. Chem. Soc.
42, 1273–1280. 10.1021/ci010132r
</cite> [<a href="https://doi.org/10.1021/ci010132r" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12444722/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Am.%20Chem.%20Soc.&amp;title=Reoptimization%20of%20MDL%20keys%20for%20use%20in%20drug%20discovery&amp;author=J.%20L.%20Durant&amp;author=B.%20A.%20Leland&amp;author=D.%20R.%20Henry&amp;author=J.%20G.%20Nourse&amp;volume=42&amp;publication_year=2002&amp;pages=1273-1280&amp;pmid=12444722&amp;doi=10.1021/ci010132r&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B10">
<cite>
Fernandez-Llimos F., Minguet F., Salgado T. M. (2017). New pharmacy-specific medical subject Headings included in the 2017 database. Am. J. Health-System Pharm.
74, 1128–1129. 10.2146/ajhp170046
</cite> [<a href="https://doi.org/10.2146/ajhp170046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28743776/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Health-System%20Pharm.&amp;title=New%20pharmacy-specific%20medical%20subject%20Headings%20included%20in%20the%202017%20database&amp;author=F.%20Fernandez-Llimos&amp;author=F.%20Minguet&amp;author=T.%20M.%20Salgado&amp;volume=74&amp;publication_year=2017&amp;pages=1128-1129&amp;pmid=28743776&amp;doi=10.2146/ajhp170046&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B11">
<cite>
Geurts P., Ernst D., Wehenkel L. (2006). Extremely randomized trees. Mach. Learn.
63 (1), 3–42. 10.1007/s10994-006-6226-1
</cite> [<a href="https://doi.org/10.1007/s10994-006-6226-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mach.%20Learn.&amp;title=Extremely%20randomized%20trees&amp;author=P.%20Geurts&amp;author=D.%20Ernst&amp;author=L.%20Wehenkel&amp;volume=63&amp;issue=1&amp;publication_year=2006&amp;pages=3-42&amp;doi=10.1007/s10994-006-6226-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B12">
<cite>
Huang L.-H., He Q. S., Liu K., Cheng J., Zhong M. D., Chen L. S., et al.  (2018). ADReCS-Target: target profiles for aiding drug safety research and application. Nucleic Acids Res.
46 (Database issue), D911-D917–D917. 10.1093/nar/gkx899

</cite> [<a href="https://doi.org/10.1093/nar/gkx899" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5753178/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30053268/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=ADReCS-Target:%20target%20profiles%20for%20aiding%20drug%20safety%20research%20and%20application&amp;author=L.-H.%20Huang&amp;author=Q.%20S.%20He&amp;author=K.%20Liu&amp;author=J.%20Cheng&amp;author=M.%20D.%20Zhong&amp;volume=46&amp;issue=Database%20issue&amp;publication_year=2018&amp;pages=D911-D917-D917&amp;pmid=30053268&amp;doi=10.1093/nar/gkx899&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B13">
<cite>
Kim S., Chen J., Cheng T., Gindulyte A., He J., He S., et al.  (2019). PubChem 2019 update: improved access to chemical data. Nucleic Acids Res.
47, D1102-D1109–D1109. 10.1093/nar/gky1033

</cite> [<a href="https://doi.org/10.1093/nar/gky1033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6324075/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30371825/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=PubChem%202019%20update:%20improved%20access%20to%20chemical%20data&amp;author=S.%20Kim&amp;author=J.%20Chen&amp;author=T.%20Cheng&amp;author=A.%20Gindulyte&amp;author=J.%20He&amp;volume=47&amp;publication_year=2019&amp;pages=D1102-D1109-D1109&amp;pmid=30371825&amp;doi=10.1093/nar/gky1033&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B14">
<cite>
Köhler S., Vasilevsky N. A., Engelstad M., Foster E., McMurry J., Aymé S., et al.  (2017). The human phenotype Ontology in 2017. Nucleic Acids Res.
45, D865-D876–D876. 10.1093/nar/gkw1039

</cite> [<a href="https://doi.org/10.1093/nar/gkw1039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5210535/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27899602/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=The%20human%20phenotype%20Ontology%20in%202017&amp;author=S.%20K%C3%B6hler&amp;author=N.%20A.%20Vasilevsky&amp;author=M.%20Engelstad&amp;author=E.%20Foster&amp;author=J.%20McMurry&amp;volume=45&amp;publication_year=2017&amp;pages=D865-D876-D876&amp;pmid=27899602&amp;doi=10.1093/nar/gkw1039&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B15">
<cite>
Kuhn M., Letunic I., Jensen L. J., Bork P. (2016). The SIDER database of drugs and side effects. Nucleic Acids Res.
44, D1075–D1079. 10.1093/nar/gkv1075

</cite> [<a href="https://doi.org/10.1093/nar/gkv1075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4702794/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26481350/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=The%20SIDER%20database%20of%20drugs%20and%20side%20effects&amp;author=M.%20Kuhn&amp;author=I.%20Letunic&amp;author=L.%20J.%20Jensen&amp;author=P.%20Bork&amp;volume=44&amp;publication_year=2016&amp;pages=D1075-D1079&amp;pmid=26481350&amp;doi=10.1093/nar/gkv1075&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B16">
<cite>
Kuhn M., von Mering C., Campillos M., Jensen L. J., Bork P. (2008). STITCH: interaction networks of chemicals and proteins. Nucleic Acids Res.
36, D684–D688. 10.1093/nar/gkm795

</cite> [<a href="https://doi.org/10.1093/nar/gkm795" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2238848/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18084021/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=STITCH:%20interaction%20networks%20of%20chemicals%20and%20proteins&amp;author=M.%20Kuhn&amp;author=C.%20von%20Mering&amp;author=M.%20Campillos&amp;author=L.%20J.%20Jensen&amp;author=P.%20Bork&amp;volume=36&amp;publication_year=2008&amp;pages=D684-D688&amp;pmid=18084021&amp;doi=10.1093/nar/gkm795&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B17">
<cite>
Landrum G. (2024). rdkit/rdkit: 2024_09_1 (Q3 2024) Release. Zenodo. 10.5281/zenodo.13848108
</cite> [<a href="https://doi.org/10.5281/zenodo.13848108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=rdkit/rdkit:%202024_09_1%20(Q3%202024)%20Release&amp;author=G.%20Landrum&amp;publication_year=2024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B18">
<cite>
Li S., Zhang L., Wang L., Ji J., He J., Zheng X., et al.  (2024). BiMPADR: a deep learning framework for predicting adverse drug reactions in new drugs. Mol. Basel, Switz.
29 (8), 1784. 10.3390/molecules29081784
</cite> [<a href="https://doi.org/10.3390/molecules29081784" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11051887/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38675604/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Basel,%20Switz.&amp;title=BiMPADR:%20a%20deep%20learning%20framework%20for%20predicting%20adverse%20drug%20reactions%20in%20new%20drugs&amp;author=S.%20Li&amp;author=L.%20Zhang&amp;author=L.%20Wang&amp;author=J.%20Ji&amp;author=J.%20He&amp;volume=29&amp;issue=8&amp;publication_year=2024&amp;pages=1784&amp;pmid=38675604&amp;doi=10.3390/molecules29081784&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B19">
<cite>
Lian J., Zhou X., Zhang F., Chen Z., Xie X., Sun G. (2018). “xDeepFM: combining explicit and implicit feature interactions for recommender systems,” in Proceedings of the 24th ACM SIGKDD international conference on knowledge discovery and data mining (New York, NY, USA: Association for Computing Machinery; ), 1754–1763. 10.1145/3219819.3220023
</cite> [<a href="https://doi.org/10.1145/3219819.3220023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Proceedings%20of%20the%2024th%20ACM%20SIGKDD%20international%20conference%20on%20knowledge%20discovery%20and%20data%20mining&amp;author=J.%20Lian&amp;author=X.%20Zhou&amp;author=F.%20Zhang&amp;author=Z.%20Chen&amp;author=X.%20Xie&amp;publication_year=2018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B20">
<cite>
Micaglio E., Locati E. T., Monasky M. M., Romani F., Heilbron F., Pappone C. (2021). Role of pharmacogenetics in adverse drug reactions: an update towards personalized medicine. Front. Pharmacol.
12, 651720. 10.3389/fphar.2021.651720

</cite> [<a href="https://doi.org/10.3389/fphar.2021.651720" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8120428/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33995067/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&amp;title=Role%20of%20pharmacogenetics%20in%20adverse%20drug%20reactions:%20an%20update%20towards%20personalized%20medicine&amp;author=E.%20Micaglio&amp;author=E.%20T.%20Locati&amp;author=M.%20M.%20Monasky&amp;author=F.%20Romani&amp;author=F.%20Heilbron&amp;volume=12&amp;publication_year=2021&amp;pages=651720&amp;pmid=33995067&amp;doi=10.3389/fphar.2021.651720&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B21">
<cite>
Pathan M., Londhe M., Jadhav D. (2018). “Drug-induced diseases: prevention, detection, and management,” in 
<em>Drug-induced diseases</em>, (American society of health-system pharmacists), 49–50. Available at: <a href="https://publications.ashp.org/display/book/9781585285310/9781585285310.xml" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://publications.ashp.org/display/book/9781585285310/9781585285310.xml</a> (Accessed May 8, 2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Drug-induced%20diseases,%20(American%20society%20of%20health-system%20pharmacists)&amp;author=M.%20Pathan&amp;author=M.%20Londhe&amp;author=D.%20Jadhav&amp;publication_year=2018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B22">
<cite>
Pirmohamed M. (2023). Pharmacogenomics: current status and future perspectives. Nat. Rev. Genet.
24, 350–362. 10.1038/s41576-022-00572-8

</cite> [<a href="https://doi.org/10.1038/s41576-022-00572-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36707729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&amp;title=Pharmacogenomics:%20current%20status%20and%20future%20perspectives&amp;author=M.%20Pirmohamed&amp;volume=24&amp;publication_year=2023&amp;pages=350-362&amp;pmid=36707729&amp;doi=10.1038/s41576-022-00572-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B23">
<cite>
Poleksic A., Xie L. (2018). Predicting serious rare adverse reactions of novel chemicals. Bioinformatics
34, 2835–2842. 10.1093/bioinformatics/bty193

</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty193" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6084596/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29617731/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Predicting%20serious%20rare%20adverse%20reactions%20of%20novel%20chemicals&amp;author=A.%20Poleksic&amp;author=L.%20Xie&amp;volume=34&amp;publication_year=2018&amp;pages=2835-2842&amp;pmid=29617731&amp;doi=10.1093/bioinformatics/bty193&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B24">
<cite>
Raja K., Patrick M., Elder J. T., Tsoi L. C. (2017). Machine learning workflow to enhance predictions of Adverse Drug Reactions (ADRs) through drug-gene interactions: application to drugs for cutaneous diseases. Sci. Rep.
7, 3690. 10.1038/s41598-017-03914-3

</cite> [<a href="https://doi.org/10.1038/s41598-017-03914-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5473874/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28623363/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Machine%20learning%20workflow%20to%20enhance%20predictions%20of%20Adverse%20Drug%20Reactions%20(ADRs)%20through%20drug-gene%20interactions:%20application%20to%20drugs%20for%20cutaneous%20diseases&amp;author=K.%20Raja&amp;author=M.%20Patrick&amp;author=J.%20T.%20Elder&amp;author=L.%20C.%20Tsoi&amp;volume=7&amp;publication_year=2017&amp;pages=3690&amp;pmid=28623363&amp;doi=10.1038/s41598-017-03914-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B25">
<cite>
Research C. for D. E. (2022). Finding and learning about side effects (adverse reactions). FDA. Available at: <a href="https://www.fda.gov/drugs/information-consumers-and-patients-drugs/finding-and-learning-about-side-effects-adverse-reactions" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.fda.gov/drugs/information-consumers-and-patients-drugs/finding-and-learning-about-side-effects-adverse-reactions</a>.
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Finding%20and%20learning%20about%20side%20effects%20(adverse%20reactions)&amp;author=C.%20for%20D.%20E.%20Research&amp;publication_year=2022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B26">
<cite>
Satapornpong P., Vorasatit L., John S. (2024). Advances in clinical pharmacogenomics and prevention of severe cutaneous adverse drug reactions in the era of precision medicine. IntechOpen. 10.5772/intechopen.1003691
</cite> [<a href="https://doi.org/10.5772/intechopen.1003691" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=IntechOpen&amp;title=Advances%20in%20clinical%20pharmacogenomics%20and%20prevention%20of%20severe%20cutaneous%20adverse%20drug%20reactions%20in%20the%20era%20of%20precision%20medicine&amp;author=P.%20Satapornpong&amp;author=L.%20Vorasatit&amp;author=S.%20John&amp;publication_year=2024&amp;doi=10.5772/intechopen.1003691&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B27">
<cite>
Sedlmayr M., Zoch M., Wolfien M., Peng Y., Ahmadi N. (2024). OMOP CDM can facilitate data-driven studies for cancer prediction. Int. J. Mol. Sci.
23, 1–11. 10.3390/ijms231911834
</cite> [<a href="https://doi.org/10.3390/ijms231911834" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9569469/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36233137/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=OMOP%20CDM%20can%20facilitate%20data-driven%20studies%20for%20cancer%20prediction&amp;author=M.%20Sedlmayr&amp;author=M.%20Zoch&amp;author=M.%20Wolfien&amp;author=Y.%20Peng&amp;author=N.%20Ahmadi&amp;volume=23&amp;publication_year=2024&amp;pages=1-11&amp;pmid=36233137&amp;doi=10.3390/ijms231911834&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B28">
<cite>
Shankar S., Bhandari I., Okou D. T., Srinivasa G., Athri P. (2021). Predicting adverse drug reactions of two-drug combinations using structural and transcriptomic drug representations to train an artificial neural network. Chem. Biol. and Drug Des.
97, 665–673. 10.1111/cbdd.13802

</cite> [<a href="https://doi.org/10.1111/cbdd.13802" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33006799/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Biol.%20and%20Drug%20Des.&amp;title=Predicting%20adverse%20drug%20reactions%20of%20two-drug%20combinations%20using%20structural%20and%20transcriptomic%20drug%20representations%20to%20train%20an%20artificial%20neural%20network&amp;author=S.%20Shankar&amp;author=I.%20Bhandari&amp;author=D.%20T.%20Okou&amp;author=G.%20Srinivasa&amp;author=P.%20Athri&amp;volume=97&amp;publication_year=2021&amp;pages=665-673&amp;pmid=33006799&amp;doi=10.1111/cbdd.13802&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B29"><cite>
Su J., Zhu M., Murtadha A., Pan S., Wen B., Liu Y. (2022). ZLPR: a novel loss for multi-label classification. Available at: <a href="https://arxiv.longhoe.net/abs/2208.02955v1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://arxiv.longhoe.net/abs/2208.02955v1</a> (Accessed May 8, 2024).</cite></li>
<li id="B30">
<cite>
Subramanian A., Narayan R., Corsello S. M., Peck D. D., Natoli T. E., Lu X., et al.  (2017). A next generation connectivity map: L1000 platform and the first 1,000,000 profiles. Cell
171, 1437–1452. 10.1016/j.cell.2017.10.049

</cite> [<a href="https://doi.org/10.1016/j.cell.2017.10.049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5990023/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29195078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A%20next%20generation%20connectivity%20map:%20L1000%20platform%20and%20the%20first%201,000,000%20profiles&amp;author=A.%20Subramanian&amp;author=R.%20Narayan&amp;author=S.%20M.%20Corsello&amp;author=D.%20D.%20Peck&amp;author=T.%20E.%20Natoli&amp;volume=171&amp;publication_year=2017&amp;pages=1437-1452&amp;pmid=29195078&amp;doi=10.1016/j.cell.2017.10.049&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B31">
<cite>
Tatonetti N. P., Ye P. P., Daneshjou R., Altman R. B. (2012). Data-driven prediction of drug effects and interactions. Sci. Transl. Med.
4, 125ra31. 10.1126/scitranslmed.3003377
</cite> [<a href="https://doi.org/10.1126/scitranslmed.3003377" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3382018/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22422992/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Transl.%20Med.&amp;title=Data-driven%20prediction%20of%20drug%20effects%20and%20interactions&amp;author=N.%20P.%20Tatonetti&amp;author=P.%20P.%20Ye&amp;author=R.%20Daneshjou&amp;author=R.%20B.%20Altman&amp;volume=4&amp;publication_year=2012&amp;pages=125ra31&amp;pmid=22422992&amp;doi=10.1126/scitranslmed.3003377&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B32">
<cite>
Üner O. C., Kuru H. I., Cinbis R. G., Tastan O., Cicek A. E. (2023). DeepSide: a deep learning approach for drug side effect prediction. IEEE/ACM Trans. Comput. Biol. Bioinforma.
20, 330–339. 10.1109/TCBB.2022.3141103
</cite> [<a href="https://doi.org/10.1109/TCBB.2022.3141103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34995191/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE/ACM%20Trans.%20Comput.%20Biol.%20Bioinforma.&amp;title=DeepSide:%20a%20deep%20learning%20approach%20for%20drug%20side%20effect%20prediction&amp;author=O.%20C.%20%C3%9Cner&amp;author=H.%20I.%20Kuru&amp;author=R.%20G.%20Cinbis&amp;author=O.%20Tastan&amp;author=A.%20E.%20Cicek&amp;volume=20&amp;publication_year=2023&amp;pages=330-339&amp;pmid=34995191&amp;doi=10.1109/TCBB.2022.3141103&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B33">
<cite>
Wang Z., Clark N. R., Ma’ayan A. (2016). Drug-induced adverse events prediction with the LINCS L1000 data. Bioinformatics
32, 2338–2345. 10.1093/bioinformatics/btw168

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btw168" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4965635/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27153606/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Drug-induced%20adverse%20events%20prediction%20with%20the%20LINCS%20L1000%20data&amp;author=Z.%20Wang&amp;author=N.%20R.%20Clark&amp;author=A.%20Ma%E2%80%99ayan&amp;volume=32&amp;publication_year=2016&amp;pages=2338-2345&amp;pmid=27153606&amp;doi=10.1093/bioinformatics/btw168&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B34">
<cite>
Yu R. J., Krantz M. S., Phillips E. J., Stone C. A. (2021a). Emerging causes of drug-induced anaphylaxis: a review of anaphylaxis-associated reports in the fda adverse event reporting system (faers). J. Allergy Clin. Immunol. Pract.
9, 819–829.e2. 10.1016/j.jaip.2020.09.021

</cite> [<a href="https://doi.org/10.1016/j.jaip.2020.09.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7870524/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32992044/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Allergy%20Clin.%20Immunol.%20Pract.&amp;title=Emerging%20causes%20of%20drug-induced%20anaphylaxis:%20a%20review%20of%20anaphylaxis-associated%20reports%20in%20the%20fda%20adverse%20event%20reporting%20system%20(faers)&amp;author=R.%20J.%20Yu&amp;author=M.%20S.%20Krantz&amp;author=E.%20J.%20Phillips&amp;author=C.%20A.%20Stone&amp;volume=9&amp;publication_year=2021a&amp;pages=819-829.e2&amp;pmid=32992044&amp;doi=10.1016/j.jaip.2020.09.021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B35">
<cite>
Yu Z., Wu Z., Li W., Liu G., Tang Y. (2021b). MetaADEDB 2.0: a comprehensive database on adverse drug events. Bioinformatics
37, 2221–2222. 10.1093/bioinformatics/btaa973

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btaa973" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33306787/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=MetaADEDB%202.0:%20a%20comprehensive%20database%20on%20adverse%20drug%20events&amp;author=Z.%20Yu&amp;author=Z.%20Wu&amp;author=W.%20Li&amp;author=G.%20Liu&amp;author=Y.%20Tang&amp;volume=37&amp;publication_year=2021b&amp;pages=2221-2222&amp;pmid=33306787&amp;doi=10.1093/bioinformatics/btaa973&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B36">
<cite>
Zhang J. D., Sach-Peltason L., Kramer C., Wang K., Ebeling M. (2020). Multiscale modelling of drug mechanism and safety. Drug Discov. Today
25 (3), 519–534. 10.1016/j.drudis.2019.12.009

</cite> [<a href="https://doi.org/10.1016/j.drudis.2019.12.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31899257/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Drug%20Discov.%20Today&amp;title=Multiscale%20modelling%20of%20drug%20mechanism%20and%20safety&amp;author=J.%20D.%20Zhang&amp;author=L.%20Sach-Peltason&amp;author=C.%20Kramer&amp;author=K.%20Wang&amp;author=M.%20Ebeling&amp;volume=25&amp;issue=3&amp;publication_year=2020&amp;pages=519-534&amp;pmid=31899257&amp;doi=10.1016/j.drudis.2019.12.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B37">
<cite>
Zhao H., Wang S., Zheng K., Zhao Q., Zhu F., Wang J. (2022). A similarity-based deep learning approach for determining the frequencies of drug side effects. Briefings Bioinforma.
23, bbab449. 10.1093/bib/bbab449
</cite> [<a href="https://doi.org/10.1093/bib/bbab449" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34718402/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Briefings%20Bioinforma.&amp;title=A%20similarity-based%20deep%20learning%20approach%20for%20determining%20the%20frequencies%20of%20drug%20side%20effects&amp;author=H.%20Zhao&amp;author=S.%20Wang&amp;author=K.%20Zheng&amp;author=Q.%20Zhao&amp;author=F.%20Zhu&amp;volume=23&amp;publication_year=2022&amp;pages=bbab449&amp;pmid=34718402&amp;doi=10.1093/bib/bbab449&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B38">
<cite>
Zhao H., Zheng K., Li Y., Wang J. (2021). A novel graph attention model for predicting frequencies of drug–side effects from multi-view data. Briefings Bioinforma.
22, bbab239. 10.1093/bib/bbab239
</cite> [<a href="https://doi.org/10.1093/bib/bbab239" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34213525/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Briefings%20Bioinforma.&amp;title=A%20novel%20graph%20attention%20model%20for%20predicting%20frequencies%20of%20drug%E2%80%93side%20effects%20from%20multi-view%20data&amp;author=H.%20Zhao&amp;author=K.%20Zheng&amp;author=Y.%20Li&amp;author=J.%20Wang&amp;volume=22&amp;publication_year=2021&amp;pages=bbab239&amp;pmid=34213525&amp;doi=10.1093/bib/bbab239&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B39">
<cite>
Zhou Z.-W., Chen X. W., Sneed K. B., Yang Y. X., Zhang X., He Z. X., et al.  (2015). Clinical association between pharmacogenomics and adverse drug reactions. Drugs
75 (6), 589–631. 10.1007/s40265-015-0375-0

</cite> [<a href="https://doi.org/10.1007/s40265-015-0375-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25895462/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Drugs&amp;title=Clinical%20association%20between%20pharmacogenomics%20and%20adverse%20drug%20reactions&amp;author=Z.-W.%20Zhou&amp;author=X.%20W.%20Chen&amp;author=K.%20B.%20Sneed&amp;author=Y.%20X.%20Yang&amp;author=X.%20Zhang&amp;volume=75&amp;issue=6&amp;publication_year=2015&amp;pages=589-631&amp;pmid=25895462&amp;doi=10.1007/s40265-015-0375-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/11931068/bin/DataSheet1.pdf" data-ga-action="click_feat_suppl" class="usa-link">DataSheet1.pdf</a><sup> (160.6KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Publicly available datasets were analyzed in this study. This data can be found here: <a href="https://github.com/hemingxiu/DGANet" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/hemingxiu/DGANet</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Frontiers in Pharmacology are provided here courtesy of <strong>Frontiers Media SA</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.3389/fphar.2025.1448106"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/fphar-16-1448106.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.5 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/11931068/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/11931068/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11931068%2F%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11931068/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC11931068/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC11931068/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40129949/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC11931068/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40129949/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC11931068/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/11931068/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="WUYYl5a26GAJQ79n2x4moqeXFFioSbbaAoO7ZhJr4NaBDkUNSqGEKGwUFc4h4hgk">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-B7-tgZZA.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-BLbQGH8u.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-DKf1UY8S.js"></script>
    
    

    </body>
</html>
