
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/base_style-CPPDhfna.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-BepKMsUl.js"></script>

  <link rel="stylesheet" href="/static/assets/article_style-DUfI1jZp.css" />
<link rel="stylesheet" href="/static/assets/style-BOrsu84o.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-D3wJHomh.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Impacts of prenatal nutrition on metabolic pathways in beef cattle: an integrative approach using metabolomics and metagenomics - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="18DE2622930B3353002622000826C1EE.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcgeno">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11983759/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Genomics">
<meta name="citation_title" content="Impacts of prenatal nutrition on metabolic pathways in beef cattle: an integrative approach using metabolomics and metagenomics">
<meta name="citation_author" content="Guilherme Henrique Gebim Polizel">
<meta name="citation_author_institution" content="Department of Animal Science, Faculty of Animal Science and Food Engineering, University of São Paulo, Av. Duque de Caxias Norte, 225, Pirassununga, SP 13635-900 Brazil">
<meta name="citation_author" content="Wellison J S Diniz">
<meta name="citation_author_institution" content="Department of Animal Sciences, College of Agriculture, Auburn University, Auburn, AL 36849 USA">
<meta name="citation_author" content="Aline Silva Mello Cesar">
<meta name="citation_author_institution" content="Department of Food Science and Technology, Luiz de Queiroz College of Agriculture, University of São Paulo, Av. Pádua Dias 11, Piracicaba, SP 13418-900 Brazil">
<meta name="citation_author" content="German D Ramírez-Zamudio">
<meta name="citation_author_institution" content="Department of Animal Science, Faculty of Animal Science and Food Engineering, University of São Paulo, Av. Duque de Caxias Norte, 225, Pirassununga, SP 13635-900 Brazil">
<meta name="citation_author" content="Angela Cánovas">
<meta name="citation_author_institution" content="Centre for Genetic Improvement of Livestock, Department of Animal Biosciences, University of Guelph, 50 Stone Road East, Guelph, ON Canada">
<meta name="citation_author" content="Evandro Fernando Ferreira Dias">
<meta name="citation_author_institution" content="Department of Animal Science, Faculty of Animal Science and Food Engineering, University of São Paulo, Av. Duque de Caxias Norte, 225, Pirassununga, SP 13635-900 Brazil">
<meta name="citation_author" content="Arícia Christofaro Fernandes">
<meta name="citation_author_institution" content="Department of Animal Science, Faculty of Animal Science and Food Engineering, University of São Paulo, Av. Duque de Caxias Norte, 225, Pirassununga, SP 13635-900 Brazil">
<meta name="citation_author" content="Barbara Carolina Teixeira Prati">
<meta name="citation_author_institution" content="Department of Animal Science, Faculty of Animal Science and Food Engineering, University of São Paulo, Av. Duque de Caxias Norte, 225, Pirassununga, SP 13635-900 Brazil">
<meta name="citation_author" content="Édison Furlan">
<meta name="citation_author_institution" content="Department of Animal Science, Faculty of Animal Science and Food Engineering, University of São Paulo, Av. Duque de Caxias Norte, 225, Pirassununga, SP 13635-900 Brazil">
<meta name="citation_author" content="Gabriela do Vale Pombo">
<meta name="citation_author_institution" content="Department of Animal Science, Faculty of Animal Science and Food Engineering, University of São Paulo, Av. Duque de Caxias Norte, 225, Pirassununga, SP 13635-900 Brazil">
<meta name="citation_author" content="Miguel Henrique de Almeida Santana">
<meta name="citation_author_institution" content="Department of Animal Science, Faculty of Animal Science and Food Engineering, University of São Paulo, Av. Duque de Caxias Norte, 225, Pirassununga, SP 13635-900 Brazil">
<meta name="citation_publication_date" content="2025 Apr 10">
<meta name="citation_volume" content="26">
<meta name="citation_firstpage" content="359">
<meta name="citation_doi" content="10.1186/s12864-025-11545-6">
<meta name="citation_pmid" content="40211121">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11983759/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11983759/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11983759/pdf/12864_2025_Article_11545.pdf">
<meta name="description" content="This study assessed the long-term metabolic effects of prenatal nutrition in Nelore bulls through an integrated analysis of metabolome and microbiome data to elucidate the interconnected host-microbe metabolic pathways. To this end, a total of 126 ...">
<meta name="og:title" content="Impacts of prenatal nutrition on metabolic pathways in beef cattle: an integrative approach using metabolomics and metagenomics">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="This study assessed the long-term metabolic effects of prenatal nutrition in Nelore bulls through an integrated analysis of metabolome and microbiome data to elucidate the interconnected host-microbe metabolic pathways. To this end, a total of 126 ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11983759/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://pmc.ncbi.nlm.nih.gov/search/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="11983759">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12864-025-11545-6"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12864_2025_Article_11545.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11983759%2F%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/11983759/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/11983759/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11983759/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcgeno.png" alt="BMC Genomics logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Genomics" title="Link to BMC Genomics" shape="default" href="https://bmcgenomics.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Genomics</button></div>. 2025 Apr 10;26:359. doi: <a href="https://doi.org/10.1186/s12864-025-11545-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12864-025-11545-6</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://pmc.ncbi.nlm.nih.gov/search/?term=%22BMC%20Genomics%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Genomics%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Genomics%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Genomics%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Impacts of prenatal nutrition on metabolic pathways in beef cattle: an integrative approach using metabolomics and metagenomics</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Polizel%20GHG%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Guilherme Henrique Gebim Polizel</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Guilherme Henrique Gebim Polizel</span></h3>
<div class="p">
<sup>1</sup>Department of Animal Science, Faculty of Animal Science and Food Engineering, University of São Paulo, Av. Duque de Caxias Norte, 225, Pirassununga, SP 13635-900 Brazil </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Polizel%20GHG%22%5BAuthor%5D" class="usa-link"><span class="name western">Guilherme Henrique Gebim Polizel</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Diniz%20WJS%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Wellison J S Diniz</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Wellison J S Diniz</span></h3>
<div class="p">
<sup>2</sup>Department of Animal Sciences, College of Agriculture, Auburn University, Auburn, AL 36849 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Diniz%20WJS%22%5BAuthor%5D" class="usa-link"><span class="name western">Wellison J S Diniz</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cesar%20ASM%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Aline Silva Mello Cesar</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Aline Silva Mello Cesar</span></h3>
<div class="p">
<sup>3</sup>Department of Food Science and Technology, Luiz de Queiroz College of Agriculture, University of São Paulo, Av. Pádua Dias 11, Piracicaba, SP 13418-900 Brazil </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cesar%20ASM%22%5BAuthor%5D" class="usa-link"><span class="name western">Aline Silva Mello Cesar</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ram%C3%ADrez-Zamudio%20GD%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">German D Ramírez-Zamudio</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">German D Ramírez-Zamudio</span></h3>
<div class="p">
<sup>1</sup>Department of Animal Science, Faculty of Animal Science and Food Engineering, University of São Paulo, Av. Duque de Caxias Norte, 225, Pirassununga, SP 13635-900 Brazil </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ram%C3%ADrez-Zamudio%20GD%22%5BAuthor%5D" class="usa-link"><span class="name western">German D Ramírez-Zamudio</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22C%C3%A1novas%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Angela Cánovas</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Angela Cánovas</span></h3>
<div class="p">
<sup>4</sup>Centre for Genetic Improvement of Livestock, Department of Animal Biosciences, University of Guelph, 50 Stone Road East, Guelph, ON Canada </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22C%C3%A1novas%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Angela Cánovas</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dias%20EFF%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Evandro Fernando Ferreira Dias</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Evandro Fernando Ferreira Dias</span></h3>
<div class="p">
<sup>1</sup>Department of Animal Science, Faculty of Animal Science and Food Engineering, University of São Paulo, Av. Duque de Caxias Norte, 225, Pirassununga, SP 13635-900 Brazil </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dias%20EFF%22%5BAuthor%5D" class="usa-link"><span class="name western">Evandro Fernando Ferreira Dias</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fernandes%20AC%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Arícia Christofaro Fernandes</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Arícia Christofaro Fernandes</span></h3>
<div class="p">
<sup>1</sup>Department of Animal Science, Faculty of Animal Science and Food Engineering, University of São Paulo, Av. Duque de Caxias Norte, 225, Pirassununga, SP 13635-900 Brazil </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fernandes%20AC%22%5BAuthor%5D" class="usa-link"><span class="name western">Arícia Christofaro Fernandes</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Prati%20BCT%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Barbara Carolina Teixeira Prati</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Barbara Carolina Teixeira Prati</span></h3>
<div class="p">
<sup>1</sup>Department of Animal Science, Faculty of Animal Science and Food Engineering, University of São Paulo, Av. Duque de Caxias Norte, 225, Pirassununga, SP 13635-900 Brazil </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Prati%20BCT%22%5BAuthor%5D" class="usa-link"><span class="name western">Barbara Carolina Teixeira Prati</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Furlan%20%C3%89%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Édison Furlan</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Édison Furlan</span></h3>
<div class="p">
<sup>1</sup>Department of Animal Science, Faculty of Animal Science and Food Engineering, University of São Paulo, Av. Duque de Caxias Norte, 225, Pirassununga, SP 13635-900 Brazil </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Furlan%20%C3%89%22%5BAuthor%5D" class="usa-link"><span class="name western">Édison Furlan</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pombo%20GDV%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Gabriela do Vale Pombo</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Gabriela do Vale Pombo</span></h3>
<div class="p">
<sup>1</sup>Department of Animal Science, Faculty of Animal Science and Food Engineering, University of São Paulo, Av. Duque de Caxias Norte, 225, Pirassununga, SP 13635-900 Brazil </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pombo%20GDV%22%5BAuthor%5D" class="usa-link"><span class="name western">Gabriela do Vale Pombo</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Santana%20MHDA%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Miguel Henrique de Almeida Santana</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Miguel Henrique de Almeida Santana</span></h3>
<div class="p">
<sup>1</sup>Department of Animal Science, Faculty of Animal Science and Food Engineering, University of São Paulo, Av. Duque de Caxias Norte, 225, Pirassununga, SP 13635-900 Brazil </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Santana%20MHDA%22%5BAuthor%5D" class="usa-link"><span class="name western">Miguel Henrique de Almeida Santana</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Animal Science, Faculty of Animal Science and Food Engineering, University of São Paulo, Av. Duque de Caxias Norte, 225, Pirassununga, SP 13635-900 Brazil </div>
<div id="Aff2">
<sup>2</sup>Department of Animal Sciences, College of Agriculture, Auburn University, Auburn, AL 36849 USA </div>
<div id="Aff3">
<sup>3</sup>Department of Food Science and Technology, Luiz de Queiroz College of Agriculture, University of São Paulo, Av. Pádua Dias 11, Piracicaba, SP 13418-900 Brazil </div>
<div id="Aff4">
<sup>4</sup>Centre for Genetic Improvement of Livestock, Department of Animal Biosciences, University of Guelph, 50 Stone Road East, Guelph, ON Canada </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Dec 12; Accepted 2025 Mar 28; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC11983759  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40211121/" class="usa-link">40211121</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">This study assessed the long-term metabolic effects of prenatal nutrition in Nelore bulls through an integrated analysis of metabolome and microbiome data to elucidate the interconnected host-microbe metabolic pathways. To this end, a total of 126 cows were assigned to three supplementation strategies during pregnancy: NP (control)– only mineral supplementation; PP– protein-energy supplementation during the last trimester; and FP– protein-energy supplementation throughout pregnancy. At the end of the finishing phase, blood, fecal, and ruminal fluid samples were collected from 63 male offspring. The plasma underwent targeted metabolomics analysis, and fecal and ruminal fluid samples were used to perform 16 S rRNA gene sequencing. Metabolite and ASV (amplicon sequence variant) co-abundance networks were constructed for each treatment using the weighted gene correlation network analysis (WGCNA) framework. Significant modules (<em>p</em> ≤ 0.1) were selected for over-representation analyses to assess the metabolic pathways underlying the metabolome (MetaboAnalyst 6.0) and the microbiome (MicrobiomeProfiler). To explore the metabolome-metagenome interplay, correlation analyses between host metabolome and microbiome were performed. Additionally, a holistic integration of metabolic pathways was performed (MicrobiomeAnalyst 2.0).</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par2">A total of one and two metabolite modules associated with the NP and FP were identified, respectively. Regarding fecal microbiome, three, one, and two modules for the NP, PP, and FP were identified, respectively. The rumen microbiome demonstrated two modules correlated with each of the groups under study. Metabolite and microbiome enrichment analyses revealed the main metabolic pathways associated with lipid and protein metabolism, and regulatory mechanisms. The correlation analyses performed between the host metabolome and fecal ASVs revealed 13 and 12 significant correlations for NP and FP, respectively. Regarding the rumen, 16 and 17 significant correlations were found for NP and FP, respectively. The NP holistic analysis was mainly associated with amino acid and methane metabolism. Glycerophospholipid and polyunsaturated fatty acid metabolism were over-represented in the FP group.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par3">Prenatal nutrition significantly affected the plasma metabolome, fecal microbiome, and ruminal fluid microbiome of Nelore bulls, providing insights into key pathways in protein, lipid, and methane metabolism. These findings offer novel discoveries about the molecular mechanisms underlying the effects of prenatal nutrition.</p></section><section id="sec4"><h3 class="pmc_sec_title">Clinical trial number</h3>
<p id="Par4">Not applicable.</p></section><section id="sec5"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12864-025-11545-6.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Maternal nutrition, Metabolites, Methane, Microbiome, PUFAs, Systems biology, WGCNA</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par5">Pregnancy is a physiological status that demands substantial nutrients to meet the fetus’s needs for growth and development and the dam’s maintenance requirements [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. When divided into three trimesters, bovine pregnancy is characterized by distinct fetal developmental milestones: placentation, organ differentiation, and rapid growth [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>, <a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. The extent to which maternal nutritional requirements are met during these periods can significantly impact fetal development [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. In cow-calf operations in the tropics, the physiological and metabolic conditions of cows, as well as the strategy of more technical production systems, coincide in that the breeding season occurs during the rainy period when forage supply and quality are adequate for reproductive success [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>, <a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. However, mid to late pregnancy overlaps with the year’s dry season when forage supply is limited and of low quality [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>–<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. Several studies have demonstrated that maternal nutrition during pregnancy has significant phenotypic, molecular, and metabolic effects, with both short and long-term consequences for the offspring [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>, <a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>–<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. To address these challenges and mitigate the potential negative impacts of undernutrition, protein and energy supplementation have been employed as a strategy during critical pregnancy stages in beef cows raised in extensive pasture systems [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>].</p>
<p id="Par6">The influence of maternal nutrition on the offspring gastrointestinal microbiome still needs to be fully elucidated despite well-established evidence that various host factors, including age, breed, genetics, and especially diet, shape the rumen and gut microbiome [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>, <a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. While some studies have demonstrated a relationship between the microbiome of cows and their offspring in fecal and ruminal samples [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>, <a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>], the understanding of microbial colonization timing continues to evolve. Traditionally, it was widely accepted that rumen colonization by microorganisms begins at birth [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>–<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. However, this notion has been challenged by a recent study on bovine fetuses, which supports the theory of in-utero gastrointestinal tract colonization by bacteria and archaea. The study suggests that colonization may commence as early as the first trimester of gestation [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>], significantly altering the perspective on early microbial establishment in ruminants.</p>
<p id="Par7">Despite the advances in prenatal nutrition research in beef cattle, divergent findings persist in the observed effects on the offspring. Many of these contradictory results are often associated with the variation of nutritional planes applied, including differences in the duration, composition, period and level [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. Host nutrition broadly impacts the gut and rumen microbiome, health, and many individual microbial taxa. These changes in microbiome composition and diversity are associated with traits such as feed efficiency [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>], disease [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>], methane emissions [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>], and metabolism [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. To address these complexities and better understand the relationship between host nutrition and microbiota, -OMIC approaches, such as plasma metabolomics [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>–<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>], have been integrated with microbiome analysis. This integration enables more in-depth characterization and insights into the phenotype under study.</p>
<p id="Par8">Moreover, the lack of holistic studies employing comprehensive and integrative -OMIC approaches has limited our ability to fully understand the causes of these differences. Implementing such studies may be a potential approach to minimize the impact of these differences and advance the knowledge in the fetal programming field. The association among maternal nutrition, offspring metabolome, and microbiome is complex. Key questions remain, such as how prenatal nutrition shapes the metabolome and microbiome, how these changes influence metabolic pathways, and what relationships exist between metabolites and microbial communities. Addressing these questions through integrative approaches will yield valuable insights into the molecular interactions and regulatory mechanisms affected by prenatal nutrition.</p>
<p id="Par9">The hypothesis of this study is that different maternal nutritional strategies have long-term effects on offspring, impacting metabolic pathways through the integration of both plasma metabolome, and fecal and ruminal microbiome. The objectives were fourfold: (1) To evaluate the impact of prenatal nutritional strategies on the plasma metabolome and microbiome co-abundance networks in the finishing phase; (2) To investigate associations between significant co-abundant metabolites and microbes and their involvement in metabolic pathways; (3) To examine relationships between metabolites and microbial taxa (fecal and ruminal fluid); and (4) To conduct a holistic integration analysis of plasma metabolome and microbiome to assess how prenatal nutrition influences metabolic pathways in beef cattle.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Experimental design</h3>
<p id="Par10">The animals used in this experiment were provided by the Faculty of Animal Science and Food Engineering (FZEA-USP) campus. All animal procedures were approved by FZEA-USP Institutional Animal Care and Use Committee (Protocol #1843241117). Thirty days after the cows’ artificial insemination, which was randomly assigned to semen from four sires, pregnancy was confirmed. The use of semen from multiple sires aimed to provide genetic variability and reduce potential sire-related biases in the study. Based on age, body weight (BW), and body condition score, the dams were divided into three groups of 42 animals each to provide a balanced experimental design. These groups were kept on <em>Urochloa brizantha</em> cv. Marandu grazing paddocks that included troughs for water and feed supplementation. The following prenatal treatments were provided to the three groups of cows: Not Programmed (NP (control)), Partial Programming (PP), and Full Programming (FP). The NP cows received only mineral supplements throughout pregnancy (0.03% of their BW per day). The PP cows received protein-energy supplementation during the last trimester of pregnancy equivalent to 0.3% of their BW per day. On the other hand, the FP cows received the same protein-energy supplementation for a longer period, starting from pregnancy confirmation until calving.</p>
<p id="Par11">The protein-energy supplementation groups (PP and FP) also received mineral supplementation, totaling 0.03% of their BW per day. The composition of the protein-energy supplement already accounted for this mineral addition (as shown in Table <a href="#Tab1" class="usa-link">1</a>). During pregnancy, the nutritional values of the grazing paddocks were equivalent across all groups (Table <a href="#Tab2" class="usa-link">2</a>) [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. Detailed information on the pasture conditions, along with the phenotypic and metabolic effects of the treatments (NP, PP, and FP) on dams, was described previously by Schalch Junior et al. [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>].</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Composition of prenatal supplement, including the mineral and protein-energy supplement with their respective ingredient and nutrient levels</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Ingredients</th>
<th align="left" colspan="1" rowspan="1">Mineral Supplement</th>
<th align="left" colspan="1" rowspan="1">Protein-Energy Supplement</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Corn (%)</td>
<td align="left" colspan="1" rowspan="1">35.00</td>
<td align="left" colspan="1" rowspan="1">60.00</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Soybean meal (%)</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">30.00</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Dicalcium phosphate (%)</td>
<td align="left" colspan="1" rowspan="1">10.00</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Urea 45% (%)</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">2.50</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Salt (%)</td>
<td align="left" colspan="1" rowspan="1">30.00</td>
<td align="left" colspan="1" rowspan="1">5.00</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Minerthal 160 MD (%)*</td>
<td align="left" colspan="1" rowspan="1">25.00</td>
<td align="left" colspan="1" rowspan="1">2.50</td>
</tr>
</tbody>
</table></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Nutrients</th>
<th align="left" colspan="1" rowspan="1">Mineral Supplement</th>
<th align="left" colspan="1" rowspan="1">Protein-Energy Supplement</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Total digestible nutrients (%)</td>
<td align="left" colspan="1" rowspan="1">26.76</td>
<td align="left" colspan="1" rowspan="1">67.55</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Crude protein (%)</td>
<td align="left" colspan="1" rowspan="1">2.79</td>
<td align="left" colspan="1" rowspan="1">24.78</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Non-protein nitrogen (%)</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">7.03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Acid detergent fiber (%)</td>
<td align="left" colspan="1" rowspan="1">1.25</td>
<td align="left" colspan="1" rowspan="1">4.76</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Neutral detergent fiber (%)</td>
<td align="left" colspan="1" rowspan="1">4.29</td>
<td align="left" colspan="1" rowspan="1">11.24</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Fat (%)</td>
<td align="left" colspan="1" rowspan="1">1.26</td>
<td align="left" colspan="1" rowspan="1">2.61</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Calcium (g/kg)</td>
<td align="left" colspan="1" rowspan="1">74.11</td>
<td align="left" colspan="1" rowspan="1">6.20</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Phosphorus (g/kg)</td>
<td align="left" colspan="1" rowspan="1">59.38</td>
<td align="left" colspan="1" rowspan="1">7.24</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p15"><p>*Mineral premix composition (Minerthal company): Calcium = 8.6 g/kg; Cobalt = 6.4 mg/kg; Copper = 108 mg/kg; Sulfur = 2.4 g/kg; Fluorine = 64 mg/kg; Phosphorus = 6.4 g/kg; Iodine = 5.4 mg/kg; Manganese = 108 mg/kg; Selenium = 3.2 mg/kg; Zinc = 324 mg/kg; Sodium monensin = 160 mg/kg [<a href="#CR90" class="usa-link" aria-describedby="CR90">90</a>]</p></div></div></section><section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Nutrient composition of the pastures consumed by cows in the different groups</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Pasture nutrients</th>
<th align="left" colspan="1" rowspan="1">NP</th>
<th align="left" colspan="1" rowspan="1">PP</th>
<th align="left" colspan="1" rowspan="1">FP</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">CP % (crude protein)</td>
<td align="center" colspan="1" rowspan="1">7.38 ± 1.72</td>
<td align="center" colspan="1" rowspan="1">7.82 ± 2.28</td>
<td align="center" colspan="1" rowspan="1">7.40 ± 2.30</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TDN % (total digestible nutrients)</td>
<td align="center" colspan="1" rowspan="1">63.1 ± 1.45</td>
<td align="center" colspan="1" rowspan="1">64.1 ± 2.33</td>
<td align="center" colspan="1" rowspan="1">61.4 ± 2.12</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NDF % (neutral detergent fiber)</td>
<td align="center" colspan="1" rowspan="1">59.0 ± 3.67</td>
<td align="center" colspan="1" rowspan="1">61.4 ± 5.05</td>
<td align="center" colspan="1" rowspan="1">58.4 ± 4.11</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ca % (calcium)</td>
<td align="center" colspan="1" rowspan="1">0.38 ± 0.11</td>
<td align="center" colspan="1" rowspan="1">0.35 ± 0.05</td>
<td align="center" colspan="1" rowspan="1">0.39 ± 0.08</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">P % (phosphorus)</td>
<td align="center" colspan="1" rowspan="1">0.19 ± 0.03</td>
<td align="center" colspan="1" rowspan="1">0.19 ± 0.03</td>
<td align="center" colspan="1" rowspan="1">0.17 ± 0.03</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p18"><p>(NP, PP, and FP), with values presented as mean ± standard deviation [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]</p></div></div></section><p id="Par14">After calving, protein-energy supplementation was stopped. All offspring, regardless of their prenatal nutritional treatment, followed the same health procedures and dietary regimens until weaning at 240 ± 28 days. During this period, cows had the same mineral supplementation (0.03% of BW) as during the pregnancy phase and were kept on an extensive grazing system of <em>Urochloa brizantha</em> cv. Marandu paddocks.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Rearing and finishing phase</h3>
<p id="Par15">Post-weaning, animals were separated by sex, regardless of their prior treatment, and reared until the end of the developmental phase at 570 ± 28 days under the same nutritional management. During this phase, young bulls received two types of supplements: an energy supplement (TDN = 67.55%; CP = 24.78%; fat = 2.61%; 0.3% of BW) during the dry season (winter), and a protein supplement (TDN = 53.15%; CP = 30.03%; fat = 1.65%; 0.1% of BW) during the wet season (summer). From calving until 570 ± 28 days of age, the young bulls grazed on <em>Urochloa brizantha</em> cv. Marandu pastures with free access to water.</p>
<p id="Par16">The finishing phase for the 63 bulls started at 570 ± 28 days of age and ended with slaughter at 676 ± 28 days. During this phase, the bulls were fed three different diets: an initial adaptation diet for 15 days, followed by a second diet for 35 days, and a final diet for 56 days. These diets were formulated to gradually adjust the bulls’ nutrition throughout the finishing phase. Additional information about the finishing diet was reported elsewhere [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>].</p>
<p id="Par17">After completing the finishing phase, the bulls were slaughtered at the FZEA/USP school slaughterhouse, which is located approximately 500 m from the feedlot facilities. The average weights and ages at slaughter were as follows: NP = 591.2 ± 40.05 kg (678 ± 29 days), PP = 602.6 ± 49.65 kg (676 ± 29 days), and 597.4 ± 51.06 kg (675 ± 28 days). The slaughter process and subsequent carcass processing were carried out in accordance with the guidelines set by the Ministry of Agriculture, Livestock, and Supply of Brazil (MAPA), as specified in Normative Instruction No. 9 of 2004. Detailed phenotypic information about slaughter, gastrointestinal and meat parameters are available in [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>, <a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>].</p></section><section id="Sec5"><h3 class="pmc_sec_title">Plasma, fecal and ruminal sample collection</h3>
<p id="Par18">Blood and fecal samples from 63 Nelore bulls were collected at 660 ± 28 days of age, 16 days prior to slaughter. The rumen fluid samples were collected in the slaughterhouse, within 15 min after slaughter (676 ± 28 days), during the processing of the animals. For this study, 5 experimental units were randomly selected from each treatment group (Plasma, <em>n</em> = 15; Feces, <em>n</em> = 15; Ruminal fluid, <em>n</em> = 15), with the same animals included in all analyses.</p>
<p id="Par19">Blood samples were aseptically collected from the jugular vein of each animal into EDTA-coated tubes (BD Vacutainer, São Paulo, Brazil) to prevent coagulation. Immediately after collection, the tubes were placed on ice to maintain sample integrity and minimize metabolic activity. All blood samples were processed within one-hour post-collection to ensure consistency and reliability of downstream analyses. To separate plasma, the blood samples were centrifuged at 3,000×g for 10 min at 4 °C using a refrigerated centrifuge. This step allowed for the effective separation of plasma from cellular components. Following centrifugation, the plasma supernatants were carefully aspirated to avoid contamination with the buffy coat or red blood cells and transferred to pre-labeled, sterile microcentrifuge tubes. The plasma aliquots were then immediately stored at -80 °C to preserve their biochemical properties and prevent degradation until further analysis.</p>
<p id="Par20">Stringent protocols were used to obtain fecal and ruminal fluid samples for metagenomics analysis, ensuring sample integrity and minimizing contamination. Fecal samples were collected directly from the rectum of each animal to retrieve fresh material. Disposable gloves were used throughout collection to minimize cross-contamination, and sterile, pre-labeled microtubes were used to collect around 10–20 g of feces from each animal. Following the slaughtering and evisceration, ruminal fluid samples were filtered with 4 layers of sterilized cheesecloth and placed in conical tubes (20 mL per animal). The tubes (feces and ruminal fluid) were initially kept in ice packs and later stored in an ultrafreezer at -80 °C until DNA extraction.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Plasma targeted metabolomics</h3>
<p id="Par21">The AbsoluteIDQ<sup>®</sup> p180 Kit (Biocrates Life Sciences, Innsbruck, Austria) was utilized to conduct targeted metabolomics analyses on plasma samples. This kit measures 188 metabolites (21 amino acids, 21 biogenic amines, 40 acylcarnitines (Cx), 14 lysophosphatidylcholines (lysoPC), 76 phosphatidylcholines (PC), 15 sphingolipids (SMx), and one monosaccharide). The analyses were conducted by Apex Science Company (Campinas, São Paulo, Brazil) using the SCIEX 4000 series<sup>®</sup> system. Amino acids and biogenic amines were quantified using high-performance liquid chromatography tandem mass spectrometry (HPLC-MS/MS) with electrospray ionization. Lysophosphatidylcholines, phosphatidylcholines, acylcarnitines, and hexose were assessed via flow injection analysis-tandem mass spectrometry (FIA-MS/MS). The quantification and identification of metabolites were based on Biocrates MetIDQ™ software and its proprietary library, which contains prevalidated reference standards and optimized methods tailored for the p180 Kit. The data were processed using MetIDQ<sup>®</sup> software, and metabolite concentrations were determined using isotopically labeled internal standards, ensuring high precision and reproducibility. Biocrates employs rigorous quality control measures, including experimental methods to determine metabolite-specific limits of detection (LOD) for each analyte. Three distinct levels of quality control samples were included on the analysis plate as described in the user manual. These controls consist of lyophilized human plasma samples spiked with low, medium, and high concentrations of target metabolites. The performance of these quality control samples was assessed using the MetVAL module of the MetIDQ<sup>®</sup> software, and the results were displayed in both the MetVAL module and the MetSTAT results table. The inclusion of these quality control measures ensures that the analytical process adheres to the highest standards of accuracy and reliability.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Metabolomics data processing</h3>
<p id="Par22">The identified metabolites were filtered based on the LOD established by the Biocrates company. Initially, values exceeding the LOD (observed in nine of the 188 metabolites, representing 4.78%) were treated as missing data. Metabolites with more than 70% of their values falling below or above the LOD threshold were subsequently removed from the dataset (six metabolites were removed; 3.19%) ensuring that at least 5 animals had values within the LOD range. Compounds with uniform values across the samples were also excluded (five metabolites were removed; 2.66%), resulting in a final dataset of 177 metabolites. For the metabolites retained after filtering, values below the LOD were replaced with the minimum detected value, while values above the LOD were replaced with the maximum detected value for each respective variable. This adjustment was applied to three metabolites, accounting for 1.69% of the dataset. This approach ensured that the dataset was suitable for subsequent analyses. The dataset was autoscaled using the “scale” function in the R statistical environment v.4.4.0 to meet normalization requirements and prepare it for further analyses. The raw metabolomics dataset is available on the Additional file <a href="#MOESM1" class="usa-link">1</a>.</p></section><section id="Sec8"><h3 class="pmc_sec_title">DNA extraction, PCR amplification and 16 S rRNA sequencing from fecal and ruminal fluid samples</h3>
<p id="Par23">The Macherey Nagel NucleoSpin Tissue<sup>®</sup> commercial kit was used to extract the DNA, and 16 S rRNA gene sequencing was performed on the MiSeq platform (2 × 250 bp; Illumina, San Diego, CA, USA) using primers for the V3 to V4 region, as recommended by the manufacturer. The methodology involved two PCR steps. The first step amplified the targeted region of the 16 S rRNA gene from the template DNA, including the Illumina adapter sequences. The PCR products were then purified using AMPure XP beads, and fragment sizes were evaluated via agarose gel electrophoresis. In the second step, barcodes from the Nextera XT kit were attached. This was followed by additional PCR purification and library validation steps. The libraries were quantified to ensure uniform sample pooling into a single library.</p>
<p id="Par24">The Illumina forward overhang adapter sequence [5ʹ-TCGTCGGCAGCGTCAGATGTGTATAAGAGACAG-(locus-specific sequence)-3´] and reverse overhang adapter sequence [5ʹ-GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAG‐(locus-specific sequence)-3´] were completed with the locus-specific sequence using S-D-Bact-0341-b-S-17 (5′-CCTACGGGNGGCWGCAG-3′) as forward primer, and S-D-Bact-0785-a-A-21 (5′-GACTACHVGGGTATCTAATCC-3´) as reverse primer [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>].</p>
<p id="Par25">A heterogeneous control, the phi-X phage, was combined with the pool of amplicons. Finally, the libraries and phi-X were denatured to allow sequencing.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Metagenomics data processing</h3>
<p id="Par26">The processing of amplicon metagenomic data was carried out on the statistical environment R v.4.4.0. The DADA2 v.1.32.0 workflow [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>] was used to infer the Amplicon Sequence Variants (ASVs) and for the taxonomic assignments. Briefly, we first filtered and trimmed the raw sequencing reads to remove poor-quality bases (quality score ≥ 30) and adapter contamination. Identical reads were then combined (dereplication). The reads were then denoised, merged, and filtered to remove artifacts related to PCR and PhiX-related chimeras. The ASVs were quantified and taxonomically annotated using the SILVA database of non-redundant sequences (version v.138.1, nr99) [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]. The data were structured in objects including the ASVs quantifications, the taxonomy annotations and the sample treatments groups (NP, PP and FP) through the phyloseq package v.1.48.0 [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. The phyloseq was used to remove phyla with just one feature and to filter for prevalence (≥ 20%; at least three samples with counts). After all filtering steps, the fecal (Additional file <a href="#MOESM2" class="usa-link">2</a>) and ruminal fluid (Additional file <a href="#MOESM3" class="usa-link">3</a>) dataset resulted in 258 and 302 microbial taxa (genus level), respectively. Fecal and ruminal fluid microbiome taxonomy table can be accessed in Additional files <a href="#MOESM4" class="usa-link">4</a> and <a href="#MOESM5" class="usa-link">5</a>, respectively. The final datasets were then log2 transformed for further analyses. Results regarding taxa abundance and diversity can be found in [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>].</p></section><section id="Sec10"><h3 class="pmc_sec_title">WGCNA analysis</h3>
<p id="Par27">The Weighted Gene Co-expression Network Analysis (WGCNA) R-package v. 1.72-5 [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>] was used to explore the co-expression pattern of metabolites and ASVs in response to the prenatal nutrition effects. In addition, the WGCNA approach was used to reduce the data dimensionality, and the results were used for further analyses.</p>
<p id="Par28">The first stage of the WGCNA analysis required turning our nutritional treatment groups (NP, PP, and FP) into binary variables using dummy transformation to match the WGCNA requirements. An adjacency matrix was generated using the WGCNA framework, which estimated the Spearman’s correlation coefficients between metabolite pairs and ASVs pairs datasets separately. To create co-abundance networks following the scale-free assumptions (R<sup>2</sup> ≥ 0.80) [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>], soft thresholds were established for the plasma metabolome (power = 12, R<sup>2</sup> = 0.92), fecal microbiome (power = 11, R<sup>2</sup> = 0.85), and rumen fluid microbiome (power = 18, R<sup>2</sup> = 0.93).</p>
<p id="Par29">Following this, the adjacency matrix was converted into a topological overlap matrix. The cluster analysis was then carried out to detect modules, ensuring a minimum of five components per module for both -OMICs (metabolomics and metagenomics) [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>, <a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. Modules having a correlation value (r) of 0.75 or higher were merged within each dataset. Using hierarchical clustering (Additional file <a href="#MOESM6" class="usa-link">6</a>), metabolites or microbial taxa with a similar abundance pattern (individually in each -OMIC dataset) across samples were placed into the same module and randomly designated by color. The modules were then summarized based on the eigengene concept, and the module eigengene values were correlated (Spearman’s correlation) with the groups (NP, PP, and FP). Modules were considered significant when the p-value was ≤ 0.1, according to WGCNA manual guidelines. Heatmaps were then created to show the correlations (with corresponding p-values) between the groups and the metabolite and microbial taxa modules.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Functional enrichment analysis</h3>
<p id="Par30">Functional enrichment analyses to uncover metabolic pathways underlying each significant module were conducted for both ASVs and metabolites. To perform ASVs functional enrichment analysis, we first used the EMBOSS Transeq tool (<a href="https://www.ebi.ac.uk/jdispatcher/st/emboss_transeq" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ebi.ac.uk/jdispatcher/st/emboss_transeq</a>) to translate the nucleotide FASTA sequences into amino acid FASTA sequences. These sequences were then annotated using the database genus_prokaryotes + viruses of the GhostKOALA [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]. This web service maps genes to KOs, which represent groups of orthologous genes associated with specific molecular functions. By analyzing the KO content, organisms and samples can be clustered, revealing distinct groups with unique characteristics. These groups are identified through various data mining techniques and primarily, though not exclusively, reflect differences in metabolic potential [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>]. Ultimately, these insights facilitate the formulation of targeted hypotheses about the physiological traits of individual ASVs and the broader community within specific samples and the prenatal nutritional treatments applied. Over representation analysis was carried out using the MicrobiomeProfiler v.1.10.0 R package to identify the metabolic pathways using KEGG (Kyoto Encyclopedia of Genes and Genomes) database involved in each significant module generated by the WGCNA analysis. Metabolic pathways with FDR (False Discovery Rate) ≤ 0.1 were considered significant.</p>
<p id="Par31">Metabolomic data enrichment analysis was conducted by inputting the list of metabolites identified in each significant module into the MetaboAnalyst v.6.0 platform [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>] The “Over Representation Analysis” function was used to identify enriched metabolic pathways. This approach evaluates whether certain metabolites are overrepresented within specific pathways, providing insights into the biological relevance of the metabolite modules. Pathway analysis was based on the KEGG database, ensuring comprehensive coverage of known metabolic pathways. Additionally, chemical structures (sub-classes) of the metabolites within each module were examined to further characterize the modules. Significance was determined using an adjusted p-value threshold of ≤ 0.1.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Metagenomics-metabolomics data integration</h3>
<p id="Par32">Two approaches were used to integrate the metagenomics and metabolomics data: (1) Spearman’s correlation analysis between the plasma metabolites and ASVs for each treatment, and (2) integration of all significant components from each treatment, regardless the sample tissue, to fully understand the impact of prenatal nutrition on metabolic pathways. Our inputs comprised all ASVs and metabolites from significantly associated modules. It is worth noting that the PP treatment was not included in the integration analyses because it did not show any significant modules in the plasma WGCNA analysis.</p>
<p id="Par33">The Spearman’s correlation analysis was used to identify host metabolites and ASVs significantly correlated, and consequently associated with the prenatal nutritional treatments (NP and FP). This method was selected due to its robustness to non-normal distributions and non-linear relationships, particularly relevant as metabolites were autoscaled and ASVs log2-transformed. Significant correlations were visualized as heatmaps, highlighting associations influenced by prenatal nutrition.</p>
<p id="Par34">Additionally, the NP and FP treatments were also individually analyzed (regardless of the -OMIC technology or approach or tissue type used) to identify the differential metabolic pathways affected by prenatal nutrition. The MicrobiomeAnalyst v.2.0 [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>] was employed to obtain functional insights, carrying out an integrated over representation analysis across the whole metabolic map involving the three tissues simultaneously (plasma metabolome, fecal and ruminal microbiome). For this analysis, significant components identified in each prenatal treatment (NP and FP) were imputed. Metabolic pathways with FDR ≤ 0.1 were considered significant, and common metabolic pathways between the treatments were removed. Figure <a href="#Fig1" class="usa-link">1</a> illustrates the workflow of the analyses conducted, along with the experimental design, providing a comprehensive overview of each step performed.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11983759_12864_2025_11545_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6303/11983759/5c4dd4e4276c/12864_2025_11545_Fig1_HTML.jpg" loading="lazy" id="d33e811" height="1116" width="795" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Experimental design and workflow illustrating sample collection, treatments, and analyses performed</p></figcaption></figure></section></section><section id="Sec13"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec14"><h3 class="pmc_sec_title">WGCNA analysis</h3>
<p id="Par36">The metabolome co-abundance analysis identified one significant module for the NP group (yellow), with a correlation coefficient of 0.49, and two significant modules for the FP group (yellow and green), with correlation coefficients of -0.52 (yellow) and 0.49 (green). No significant modules were found for the PP group. Notably, while the yellow module was shared between the NP and FP groups, it exhibited opposite correlation signs, suggesting different metabolic behaviors between these groups (Fig. <a href="#Fig2" class="usa-link">2</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11983759_12864_2025_11545_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6303/11983759/61c231738284/12864_2025_11545_Fig2_HTML.jpg" loading="lazy" id="d33e830" height="797" width="779" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Plasma metabolome module–treatment correlation heatmap. Each row corresponds to a metabolite module, and each column corresponds to a prenatal nutritional treatment group (NP, PP, and FP). Each cell contains the corresponding correlation and p-value. The table is colour-coded by correlation, according to the colour legend</p></figcaption></figure><p id="Par38">The significant correlations (<em>p</em> ≤ 0.1) regarding fecal microbiome ranged from|0.47| to|0.49| in the NP group modules, and|0.56| to|0.59| in the FP group modules. In the PP group, only one significant module was correlated with the fecal microbiome (purple module [<em>r</em> = -0.49]). As demonstrated in the Fig. <a href="#Fig3" class="usa-link">3</a>, we also identified shared modules between the groups in the fecal microbiome. The NP and PP groups shared the purple module, showing opposite correlation (0.49 and − 0.49, respectively). In addition, the NP group showed two more significant modules (pink [<em>r</em> = -0.49] and cyan [<em>r</em> = -0.47]). The FP group showed significant negative correlations with the brown (<em>r</em> = -0.59) and salmon (<em>r</em> = -0.56) modules.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11983759_12864_2025_11545_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6303/11983759/30e69093fea5/12864_2025_11545_Fig3_HTML.jpg" loading="lazy" id="d33e864" height="905" width="785" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Fecal microbiome module–treatment correlations heatmap. Each row corresponds to an ASV module, and each column corresponds to a prenatal nutritional treatment group (NP, PP, and FP). Each cell contains the corresponding correlation and p-value. The table is colour-coded by correlation, according to the colour legend</p></figcaption></figure><p id="Par40">Regarding the ruminal fluid microbiome, the NP group was correlated (<em>p</em> ≤ 0.1) with pink and purple modules (<em>r</em> = 0.52; <em>r</em> = 0.69, respectively). The pink module was likewise shared with PP group (<em>r</em> = -0.59), while the purple module was shared with the FP group (<em>r</em> = -0.54), both indicating opposite correlations between the groups. Thus, the metabolic pathways associated with each module act differently among the prenatal groups. Furthermore, the PP group was exclusively correlated with the blue module (<em>r</em> = 0.46), and the FP group uniquely correlated with the magenta module (<em>r</em> = -0.53). Table <a href="#Tab3" class="usa-link">3</a> presents all the components of each significant module identified in the WGCNA analyses.</p>
<section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Components of the significant modules from WGCNA analysis. The table lists the corresponding -OMIC dataset, tissue type, module color, and the components associated with each significant module</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">-OMICS</th>
<th align="left" colspan="1" rowspan="1">Tissue</th>
<th align="left" colspan="1" rowspan="1">Modules</th>
<th align="left" colspan="1" rowspan="1">Components</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Metabolome</td>
<td align="left" colspan="1" rowspan="1">Plasma</td>
<td align="left" colspan="1" rowspan="1">Yellow</td>
<td align="left" colspan="1" rowspan="1">Ala; Arg; Asn; Asp; Gln; Glu; Gly; His; Ile; Leu; Lys; Met; Orn; Phe; Pro; Ser; Thr; Tyr; Val; Ac.Orn; Carnosine; t4.OH.Pro; C3-DC.C4(OH)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Metabolome</td>
<td align="left" colspan="1" rowspan="1">Plasma</td>
<td align="left" colspan="1" rowspan="1">Green</td>
<td align="left" colspan="1" rowspan="1">Alpha.aaa; ADMA; Creatinine; Serotonin; C10.1; C10.2; C12.1; C14.1; lysoPC.a.C16.1; lysoPC.a.C17.0; lysoPC.a.C18.1; lysoPC.a.C20.3; lysoPC.a.C24.0; lysoPC.a.C26.0; lysoPC.a.C28.0; lysoPC.a.C28.1; PC.aa.C26.0; PC.aa.C30.0; PC.aa.C32.0; PC.aa.C34.1; PC.aa.C36.0; PC.aa.C36.1; PC.aa.C36.3; PC.aa.C36.5; PC.aa.C36.6; PC.aa.C38.3; PC.aa.C38.4; PC.aa.C38.5; PC.aa.C40.1; PC.aa.C40.2; PC.aa.C40.3; PC.aa.C40.4; PC.aa.C40.5; PC.aa.C42.2; PC.aa.C42.4; PC.ae.C30.0; PC.ae.C30.1; PC.ae.C30.2; PC.ae.C32.2; PC.ae.C34.0; PC.ae.C34.3; PC.ae.C36.0; PC.ae.C36.1; PC.ae.C38.0; PC.ae.C38.1; PC.ae.C38.2; PC.ae.C38.3; PC.ae.C38.4; PC.ae.C40.1; PC.ae.C40.2; PC.ae.C40.3; PC.ae.C40.4; PC.ae.C40.5; PC.ae.C42.0; PC.ae.C42.1; PC.ae.C42.2; PC.ae.C42.3; PC.ae.C42.4; PC.ae.C42.5; SM.OH.C22.2; SM.OH.C24.1; SM.C16.0; SM.C16.1; SM.C24.0; SM.C26.0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Metagenome</td>
<td align="left" colspan="1" rowspan="1">Feces</td>
<td align="left" colspan="1" rowspan="1">Pink</td>
<td align="left" colspan="1" rowspan="1">ASV9; ASV16; ASV17; ASV19; ASV29; ASV38; ASV60; ASV65; ASV82; ASV104; ASV111; ASV112; ASV119; ASV180; ASV463</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Metagenome</td>
<td align="left" colspan="1" rowspan="1">Feces</td>
<td align="left" colspan="1" rowspan="1">Cyan</td>
<td align="left" colspan="1" rowspan="1">ASV76; ASV169; ASV174; ASV189; ASV217; ASV268; ASV270; ASV552; ASV567</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Metagenome</td>
<td align="left" colspan="1" rowspan="1">Feces</td>
<td align="left" colspan="1" rowspan="1">Purple</td>
<td align="left" colspan="1" rowspan="1">ASV69; ASV81; ASV89; ASV113; ASV117; ASV181; ASV234; ASV245; ASV309; ASV330; ASV437</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Metagenome</td>
<td align="left" colspan="1" rowspan="1">Feces</td>
<td align="left" colspan="1" rowspan="1">Brown</td>
<td align="left" colspan="1" rowspan="1">ASV7; ASV8; ASV15; ASV21; ASV24; ASV25; ASV26; ASV28; ASV30; ASV31; ASV32; ASV33; ASV34; ASV37; ASV40; ASV46; ASV50; ASV54; ASV58; ASV72; ASV73; ASV86; ASV92; ASV95; ASV328</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Metagenome</td>
<td align="left" colspan="1" rowspan="1">Feces</td>
<td align="left" colspan="1" rowspan="1">Salmon</td>
<td align="left" colspan="1" rowspan="1">ASV97; ASV98; ASV116; ASV149; ASV182; ASV205; ASV213; ASV238; ASV275; ASV751</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Metagenome</td>
<td align="left" colspan="1" rowspan="1">Rumen fluid</td>
<td align="left" colspan="1" rowspan="1">Pink</td>
<td align="left" colspan="1" rowspan="1">ASV39; ASV51; ASV53; ASV63; ASV65; ASV68; ASV87; ASV95; ASV109; ASV131; ASV142; ASV215; ASV220; ASV290; ASV387</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Metagenome</td>
<td align="left" colspan="1" rowspan="1">Rumen fluid</td>
<td align="left" colspan="1" rowspan="1">Purple</td>
<td align="left" colspan="1" rowspan="1">ASV12; ASV34; ASV57; ASV76; ASV100; ASV133; ASV186; ASV198; ASV231; ASV587; ASV648; ASV861; ASV1035</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Metagenome</td>
<td align="left" colspan="1" rowspan="1">Rumen fluid</td>
<td align="left" colspan="1" rowspan="1">Blue</td>
<td align="left" colspan="1" rowspan="1">ASV21; ASV78; ASV89; ASV91; ASV110; ASV158; ASV192; ASV197; ASV200; ASV230; ASV233; ASV245; ASV276; ASV347; ASV402; ASV458; ASV520; ASV547; ASV1042; ASV1139</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Metagenome</td>
<td align="left" colspan="1" rowspan="1">Rumen fluid</td>
<td align="left" colspan="1" rowspan="1">Magenta</td>
<td align="left" colspan="1" rowspan="1">ASV9; ASV27; ASV30; ASV45; ASV67; ASV83; ASV85; ASV96; ASV101; ASV116; ASV268; ASV309; ASV322; ASV443; ASV1045</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11983759_12864_2025_11545_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6303/11983759/bb41a019dd6b/12864_2025_11545_Fig4_HTML.jpg" loading="lazy" id="d33e1024" height="1159" width="785" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Rumen fluid microbiome module–treatment correlations heatmap. Each row corresponds to an ASV module, and each column corresponds to a prenatal nutritional treatment group (NP, PP, and FP). Each cell contains the corresponding correlation and p-value. The table is colour-coded by correlation, according to the colour legend</p></figcaption></figure></section><section id="Sec15"><h3 class="pmc_sec_title">Metabolome functional enrichment analysis</h3>
<p id="Par43">Based on the metabolome functional enrichment analysis of each significant module, we found several important metabolic pathways and sub-classes (adjusted p-value ≤ 0.1) related to the prenatal treatments.</p>
<p id="Par44">The yellow module showed that all metabolic pathways were associated with amino acid metabolic processes (arginine biosynthesis [<em>p</em> = 9.13e-07]; valine, leucine and isoleucine biosynthesis [<em>p</em> = 8.47e-05]; alanine, aspartate and glutamate metabolism [<em>p</em> = 7.7e-04]; histidine metabolism [<em>p</em> = 0.001]; arginine and proline metabolism [<em>p</em> = 0.001]; glyoxylate and dicarboxylate metabolism [<em>p</em> = 0.010]; phenylalanine, tyrosine and tryptophan biosynthesis [<em>p</em> = 0.013]; nitrogen metabolism [<em>p</em> = 0.025]; beta-alanine metabolism [<em>p</em> = 0.025]; phenylalanine metabolism [<em>p</em> = 0.041]; glutathione metabolism [<em>p</em> = 0.048]; and glycine, serine and threonine metabolism [<em>p</em> = 0.069]).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11983759_12864_2025_11545_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6303/11983759/e6a6438906da/12864_2025_11545_Fig5_HTML.jpg" loading="lazy" id="d33e1078" height="602" width="798" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Bubble plot illustrating the over-representation analysis of metabolites associated with the yellow module from plasma WGCNA analysis</p></figcaption></figure><p id="Par46">Regarding the functional enrichment of the green module (correlated exclusively with the FP group; Fig. <a href="#Fig6" class="usa-link">6</a>), two metabolic sub-classes involved in lipid metabolism (glycerophosphocholines [<em>p</em> = 8.7e-52]; and phosphosphingolipids [<em>p</em> = 0.066]) were identified.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11983759_12864_2025_11545_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6303/11983759/64d2e956bad3/12864_2025_11545_Fig6_HTML.jpg" loading="lazy" id="d33e1099" height="593" width="787" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Bubble plot illustrating the over-representation analysis of metabolites associated with the green module from plasma WGCNA analysis</p></figcaption></figure></section><section id="Sec16"><h3 class="pmc_sec_title">Microbiome functional enrichment analysis</h3>
<p id="Par48">The microbiome functional enrichment analysis of each significant module revealed some relevant metabolic pathways (adjusted p-value ≤ 0.1) associated with the prenatal groups.</p>
<p id="Par49">In the fecal microbiome (Fig. <a href="#Fig7" class="usa-link">7</a>), we identified four functionally enriched modules (cyan, purple, brown and salmon). In the cyan module (Fig. <a href="#Fig7" class="usa-link">7</a>A), three metabolic pathways were significantly enriched (nitrogen metabolism [<em>p</em> = 0.024]; phenylalanine, tyrosine and tryptophan biosynthesis [<em>p</em> = 0.024]; and biosynthesis of amino acids [<em>p</em> = 0.051]). Regarding the purple module (Fig. <a href="#Fig7" class="usa-link">7</a>B), five metabolic pathways were significantly enriched (nitrogen metabolism [<em>p</em> = 0.062]; Phenylalanine, tyrosine and tryptophan biosynthesis [<em>p</em> = 0.062]; Bacterial secretion system [<em>p</em> = 0.062]; cysteine and methionine metabolism [<em>p</em> = 0.078]; and biosynthesis of amino acids [<em>p</em> = 0.099). The brown module (Fig. <a href="#Fig7" class="usa-link">7</a>C) presented four metabolic pathways significantly enriched (nitrogen metabolism [<em>p</em> = 0.031]; RNA degradation [<em>p</em> = 0.031]; pertussis [<em>p</em> = 0.031]; and glycerophospholipid metabolism [<em>p</em> = 0.033]). A total of five metabolic pathways were identified associated with the salmon module (Fig. <a href="#Fig7" class="usa-link">7</a>D) including RNA degradation [<em>p</em> = 0.053], nitrogen metabolism [<em>p</em> = 0.062], lipopolysaccharide biosynthesis [<em>p</em> = 0.067], Galactose metabolism [<em>p</em> = 0.067], and biosynthesis of siderophore group nonribosomal peptides [<em>p</em> = 0.075]). No significant metabolic pathways were identified in the pink module.</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11983759_12864_2025_11545_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6303/11983759/6329594b7316/12864_2025_11545_Fig7_HTML.jpg" loading="lazy" id="d33e1197" height="511" width="798" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Bar plots illustrating the over-representation analysis of fecal microbiome from the significant modules correlated in the WGCNA. <strong>A</strong> Significant metabolic pathways associated with ASVs in the cyan module. <strong>B</strong> Significant metabolic pathways associated with ASVs in the purple module. <strong>C</strong> Significant metabolic pathways associated with ASVs in the brown module. <strong>D</strong> Significant metabolic pathways associated with ASVs in the salmon module</p></figcaption></figure><p id="Par51">In the rumen fluid microbiome (Fig. <a href="#Fig8" class="usa-link">8</a>), we also found four significant enriched modules (pink, purple, blue and magenta). The pink module (Fig. <a href="#Fig8" class="usa-link">8</a>A) presented five significant metabolic pathways enriched (nitrogen metabolism [<em>p</em> = 0.012]; biotin metabolism [<em>p</em> = 0.071]; pinene, camphor and geraniol degradation [<em>p</em> = 0.071]; fatty acid biosynthesis [<em>p</em> = 0.071]; and protein export [<em>p</em> = 0.071]). The purple module (Fig. <a href="#Fig8" class="usa-link">8</a>B) showed five significant metabolic pathways enriched (lipoarabinomannan (LAM) biosynthesis [<em>p</em> = 0.061]; biotin metabolism [<em>p</em> = 0.062]; fatty acid biosynthesis [<em>p</em> = 0.062]; protein export [<em>p</em> = 0.062]; biosynthesis of cofactors [<em>p</em> = 0.063]). In the blue module (Fig. <a href="#Fig8" class="usa-link">8</a>C), four metabolic pathways were significantly enriched (RNA degradation [<em>p</em> = 0.091]; nitrogen metabolism [<em>p</em> = 0.094]; ribosome [<em>p</em> = 0.095]; and nucleotide metabolism [<em>p</em> = 0.098]. Lastly, the magenta module (Fig. <a href="#Fig8" class="usa-link">8</a>D) showed five metabolic pathways enriched (pinene, camphor and geraniol degradation [<em>p</em> = 0.063]; lysine biosynthesis [<em>p</em> = 0.063]; protein export [<em>p</em> = 0.064]; mismatch repair [<em>p</em> = 0.066]; and peptidoglycan biosynthesis [<em>p</em> = 0.071].</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11983759_12864_2025_11545_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6303/11983759/299f9ace1816/12864_2025_11545_Fig8_HTML.jpg" loading="lazy" id="d33e1298" height="502" width="798" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Bar plots illustrating the over-representation analysis of rumen fluid microbiome from the significant modules correlated in the WGCNA. <strong>A</strong> Significant metabolic pathways associated with ASVs in the pink module. <strong>B</strong> Significant metabolic pathways associated with ASVs in the purple module. <strong>C</strong> Significant metabolic pathways associated with ASVs in the blue module. <strong>D</strong> Significant metabolic pathways associated with ASVs in the magenta module</p></figcaption></figure></section><section id="Sec17"><h3 class="pmc_sec_title">Metagenomics-metabolomics correlation analysis</h3>
<p id="Par53">Figure <a href="#Fig9" class="usa-link">9</a> shows the significant correlations observed based on the Spearman’s correlation analysis between significant plasma metabolites and ASVs for each treatment (NP and FP).</p>
<figure class="fig xbox font-sm" id="Fig9"><h4 class="obj_head">Fig. 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11983759_12864_2025_11545_Fig9_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6303/11983759/a333b684464c/12864_2025_11545_Fig9_HTML.jpg" loading="lazy" id="d33e1328" height="588" width="798" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Heatmaps demonstrating the significant correlations between the host plasma metabolome and its fecal and rumen fluid microbiome. <strong>A</strong> NP group correlation between its significant metabolites and its significant fecal ASVs. <strong>B</strong> NP group correlation between its significant metabolites and its significant rumen fluid ASVs. <strong>C</strong> FP group correlation between its significant metabolites and its significant fecal ASVs. <strong>D</strong> FP group correlation between its significant metabolites and its significant rumen fluid ASVs. The heatmaps were reduced to just display components with at least one significant correlation. Significant correlations are numerically represented in the heatmap, and the significant levels are demonstrated according to the p-values threshold (*p-value &lt; 0.01; **p-value &lt; 0.001)</p></figcaption></figure><p id="Par55">In the NP treatment, we identified 13 significant correlations between fecal ASVs and plasma metabolites (Fig. <a href="#Fig9" class="usa-link">9</a>A). The correlation coefficients ranged from|0.65| to|0.81|, with threonine and tyrosine accounting for eight of these significant correlations. Tyrosine exhibited the strongest correlation (<em>r</em> = -0.81) with ASV65 (<em>Romboutsia</em> genus), followed by threonine, which showed the second highest correlation with the same ASV (<em>r</em> = -0.74).</p>
<p id="Par56">The correlations between rumen fluid ASVs and plasma metabolites in the NP treatment (Fig. <a href="#Fig9" class="usa-link">9</a>B) showed 16 significant correlations ranging from|0.65| to|0.77|. Two metabolites, malonylcarnitine (C3-DC C4(OH)) and carnosine, accounted for 11 out of the 16 significant correlations identified. The top two correlations were associated with malonylcarnitine and ASV76 (<em>Saccharofermentans</em> genus; <em>r</em> = 0.77), and carnosine and the same ASV (<em>r</em> = 0.77).</p>
<p id="Par57">Regarding the FP treatment, the correlation between plasma metabolites and fecal microbial taxa revealed 12 significant correlations ranging from|0.64| to|0.75| (Fig. <a href="#Fig9" class="usa-link">9</a>C). This assessment did not show significant correlations concentrated in specific metabolites, unlike previous findings. The top two highest correlation values were between proline and ASV205 (<em>Agathobacter</em> genus; <em>r</em> = 0.75), and between aspartate and ASV149 (<em>Clostridium sensu stricto 1</em> genus; <em>r</em> = 0.73).</p>
<p id="Par58">The correlations between rumen fluid microbial taxa and plasma metabolites in the FP treatment (Fig. <a href="#Fig9" class="usa-link">9</a>D) showed 17 significant correlations ranging from|0.65| to|0.77|. Most of the correlations (15 out of 17), including the highest ones, were associated mainly with malonylcarnitine and carnosine. Malonylcarnitine is involved in five of the top six correlations. These include its correlation with <em>Saccharofermentans</em> genus (ASV76 and ASV186; <em>r</em> = 0.77 and <em>r</em> = 0.73, respectively), <em>Acetitomaculum</em> genus (ASV45 and ASV67; <em>r</em> = 0.76 and <em>r</em> = 0.75, respectively) and <em>Christensenellaceae R-7 group</em> genus (ASV57; <em>r</em> = 0.74).</p></section><section id="Sec18"><h3 class="pmc_sec_title">Metagenome-metabolome metabolic pathways analysis</h3>
<p id="Par59">We refined the first findings (32 significant pathways for the NP group and 26 significant pathways for the FP group) by focusing on the distinct metabolic pathways related with each nutritional treatment group (NP and FP). A total of 10 unique metabolic pathways were found for NP treatment including methane metabolism [FDR = 0.029], pantothenate and CoA biosynthesis [0.037], thiamine metabolism [FDR = 0.029], carbapenem biosynthesis [FDR = 0.039], novobiocin biosynthesis [FDR = 0.039], phenylalanine, tyrosine and tryptophan biosynthesis [FDR = 0.043], staurosporine biosynthesis [FDR = 0.073], phenylalanine metabolism [FDR = 0.076], porphyrin metabolism [FDR = 0.088] and phenylpropanoid biosynthesis [FDR = 0.096]), which are involved with processes such as amino acid and methane metabolism, as well as antibiotics biosynthesis.</p>
<p id="Par60">In the FP group, five unique metabolic pathways were identified, predominantly related to lipid metabolism (i.e., glycerophospholipid metabolism [FDR = 8.24e-25], linoleic acid metabolism [4.0e-25], alpha-linolenic acid metabolism [FDR = 3.44e-17], arachidonic acid metabolism [FDR = 8.92e-14]), and antibiotic biosynthesis (Penicillin and cephalosporin biosynthesis [FDR = 0.047]). Figure <a href="#Fig10" class="usa-link">10</a> shows the unique metabolic pathways underlying each prenatal nutritional approach.</p>
<figure class="fig xbox font-sm" id="Fig10"><h4 class="obj_head">Fig. 10.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11983759_12864_2025_11545_Fig10_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6303/11983759/9e81514bdd31/12864_2025_11545_Fig10_HTML.jpg" loading="lazy" id="d33e1422" height="588" width="798" alt="Fig. 10"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Diamond plot showing the holistic integration of significant metabolites and ASVs (fecal and rumen fluid) in each prenatal nutrition group (NP and FP). The figure illustrates all the significant exclusive metabolic pathways associated with each group</p></figcaption></figure></section></section><section id="Sec19"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par62">Prenatal nutrition can affect fetal growth and development through epigenetic alterations, which may consequently impact the short and long-term metabolism of the offspring [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>–<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]. Alongside this, the -OMICs integration studies are becoming increasingly essential for improving knowledge about the functional and biological processes in all organisms [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>].</p>
<section id="Sec20"><h3 class="pmc_sec_title">Prenatal nutrition effects on plasma metabolic pathways</h3>
<p id="Par63">From the two significant co-abundance modules of plasma metabolome, we identified several pathways impacted by prenatal nutrition. The yellow module showed that all 13 significant metabolic pathways were highly associated with amino acid metabolic processes. Although the yellow module showed significant correlation with both NP and FP groups, the correlation coefficients were inverses, indicating different modulation of these pathways.</p>
<p id="Par64">Amino acids play important roles, such as energy sources, primary producers of C1 carbon compounds, anaplerotic metabolites providing intermediates for the tricarboxylic acid (TCA) cycle and gluconeogenesis, precursors for numerous hormones, neurotransmitters, and other specialized metabolites [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>].</p>
<p id="Par65">Biosynthesis of amino acids, as well as glycine, serine and threonine metabolism have already shown an association with prenatal nutrition on fetal skeletal muscle in beef cattle [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]. According to Elolimy et al. [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>], the prenatal methionine supplementation during late pregnancy affected metabolic pathways including alanine metabolism, phenylalanine and tyrosine metabolism, glutamate metabolism, arginine and proline metabolism, and tryptophan metabolism during the preweaning period of offspring. Although, valine, leucine and isoleucine degradation pathways have been identified by the same authors, our study revealed that prenatal nutrition affected the biosynthesis instead of the degradation of this metabolic pathway. In general, these metabolic pathways are similar with our present findings in the yellow module. Additionally, our previous studies [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>, <a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>] have identified the effect of prenatal nutrition on similar amino acid metabolic pathways as the present study (arginine biosynthesis; histidine metabolism; beta-alanine metabolism). These findings indicate that the metabolic alterations caused by prenatal nutrition persist until the animal’s slaughter.</p>
<p id="Par66">Arginine and proline play important roles in proper cellular function and development by participating in DNA and RNA synthesis, protein glycosylation, and detoxification [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>, <a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]. Phenylalanine, tryptophan, and tyrosine act as precursors to neurotransmitters including dopamine and serotonin [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]. The branched-chain amino acids (BCAA) are composed by leucine, isoleucine and valine, playing roles such as protein metabolism, energy homeostasis (lipid and glucose), gut health and immunity [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>]. Therefore, we speculate that prenatal nutrition plays a pivotal role in modulating the intricate crosstalk between the blood, brain, and gastrointestinal tract. This modulation has the potential to influence key physiological processes, including nutrient metabolism, neurotransmitter synthesis, and immune responses, thereby shaping developmental and functional outcomes.</p>
<p id="Par67">Regarding the green module, which was significant exclusively for the FP group, the metabolites were over-represented in the glycerophosphocholine (GPC) and phosphosphingolipid sub-classes. The GPC metabolite subclass, also known as choline alfoscerate, functions as a choline precursor and is closely related to phosphatidylcholine. It plays essential roles in cell membrane integrity, osmoregulation, signaling, and lipid transport [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>, <a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>]. Phosphosphingolipids, an important subclass of lipids within the sphingolipid class, are involved in key biological processes such as plasma membrane composition, cell signaling, proliferation and differentiation, stress responses, apoptosis, insulin resistance, aging, cancer, and lipid signaling pathways [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>, <a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>]. In beef cattle, Menezes et al. [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>] reported that maternal nutrition, particularly maternal body weight gain, changed the hepatic fetal lipid composition in the first trimester of pregnancy. These findings suggest that the over-representation of GPC and phosphosphingolipids in the FP group may reflect changes in lipid metabolism and processes related to cell signaling and osmoregulation, which are crucial for energy balance and cellular function during development.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Prenatal nutrition effects on microbiome metabolic pathways</h3>
<p id="Par68">The cyan and purple fecal microbiome modules were correlated with the NP group (<em>r</em> = -0.47 and <em>r</em> = 0.49, respectively). Despite showing moderate correlations [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>], these modules were biologically relevant because four of the five significant metabolic pathways identified in the NP fecal microbiome were closely related to amino acid metabolism, including nitrogen metabolism, phenylalanine, tyrosine and tryptophan biosynthesis, cysteine and methionine metabolism, and biosynthesis of amino acids. Notably, these pathways are similar to those observed in the NP plasma metabolome. Given the moderate correlations and the strong functional relevance of these pathways, we focused on these modules as they highlight important metabolic processes potentially influenced by prenatal nutrition. The PP group was also correlated with the purple module; however, showing an inverse correlation regarding the NP group. This indicates different modulation of the metabolic pathways involved in the purple module. Furthermore, the NP metabolic levels of tyrosine were negatively correlated with four ASVs (ASV29 [<em>Paeniclostridium</em>], ASV60 [<em>Romboutsia</em>], ASV65 [<em>Romboutsia</em>], ASV112 [<em>Romboutsia</em>]). This finding, alongside the tyrosine biosynthesis pathway identified, highlights the strong association among <em>Romboutsia</em> genera, tyrosine and NP group.</p>
<p id="Par69">The gut microbiota is essential for amino acid production, particularly de novo biosynthesis. Several in vitro studies have revealed that ruminal bacterial species engage in the de novo biosynthesis of amino acids when physiological peptide amounts are present [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>]. Furthermore, the gut microbiota profile is determinant for the tryptophan catabolite levels in the systemic circulation [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>, <a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>]. These findings are consistent with our plasma metabolic pathway results, which highlight the interplay between prenatal nutrition, host metabolism, and the gut microbiome. Our correlation results align with these findings, as the NP fecal microbiome demonstrated significant associations with metabolites, five of which belong to the amino acid class. While this suggests a potential link between the NP group and amino acid metabolism, further investigation is required to establish causation or fully substantiate this connection.</p>
<p id="Par70">The FP group was exclusively correlated with the brown (<em>r</em> = -0.59) and salmon (<em>r</em> = -0.56) modules. While nitrogen metabolism was also over-represented in the FP group, most of the important metabolic pathways found were not related to amino acid metabolism. Pathways including RNA degradation, glycerophospholipid metabolism, and lipopolysaccharide (LPS) biosynthesis were identified. This is an intriguing discovery, given that GPC and phosphosphingolipid were over-represented in the FP plasma metabolome. Nevertheless, the correlation results demonstrated that only two metabolites from the phospholipid class (PC aa C36:6 and PC ae C36:0) were significantly correlated with a fecal microbial taxon (ASV32 [<em>Faecalibacterium</em>]).</p>
<p id="Par71">Although our study did not specifically investigate the epigenome or related molecules, the RNA degradation pathway identified in the FP group is associated with these factors; thus, we discuss their potential relevance in this context. Epigenetic alterations and regulatory mechanisms, like RNA degradation, represent main processes via which gut bacteria can affect the host health [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>, <a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>]. Cortese et al. [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>] has demonstrated that bacteria may directly cause epigenetic alterations in the host, as indicated by distinct patterns of DNA modification in immature intestinal epithelial cells after exposure to commensal or pathogenic pathogens. Notably, the same study discovered that prenatal glucocorticoid-induced epigenetic programming leads to changes in gut microbiota composition in mice, demonstrating intricate connections between the microbiome and epigenome. In addition, alterations in the LPS biosynthetic pathways have been reported in cesarean delivered neonates in comparison with vaginally delivered neonates [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>]. The LPS is a surface membrane component of gram-negative bacteria and is recognized by toll-like receptor 4 (TLR4) on the membranes of intestinal epithelial cells, which stimulates components of the immune system [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>]. The epigenetic alterations caused by prenatal nutrition may be reflected in the long-term gut microbiome, potentially leading to immunomodulatory effects. In summary, considering the prenatal nutrition effects on fecal microbiome pathways, we can clearly identify a close association among the prenatal nutrition group, the over-represented pathways in the fecal microbiome, and those in the plasma (discussed in the Sect. “<a href="#Sec20" class="usa-link">Prenatal nutrition effects on plasma metabolic pathways</a>”).</p>
<p id="Par72">Regarding the ruminal fluid microbiome, the metabolic pathways detected in the NP group consisted mainly of protein metabolism (nitrogen metabolism, protein export), followed by vitamin metabolism (biotin metabolism), and lipid metabolism (fatty acid biosynthesis). Biotin (vitamin B7) is a water-soluble vitamin that plays a significant role in mammal health and illness as a coenzyme for carboxylases involved in a variety of metabolic pathways, including the cellular stress response, gene regulation, and immunological responses [<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>–<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>]. In the rumen, the metabolism of proteins and fatty acids is closely linked to microbial activity and fermentation processes. The anaerobic nature of the rumen involves partial breakdown of substrates (including proteins), and the end fermentation products involve volatile fatty acids (VFAs; mainly acetate, propionate, and butyrate), in addition to CO<sub>2</sub> [<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>]. As ruminants consume dietary protein, it is quickly digested in the rumen into peptides and amino acids, resulting in ammonia generation and nitrogen loss. When nitrogen retention is inefficient, financial expenses increase due to the demand for more dietary protein. In severe cases, high amounts of rumen ammonia can induce metabolic stress in the animal, whereas excessive nitrogen excretion in manure might harm the ecosystem [<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>]. Despite the fact that three of the five significant metabolites in the NP rumen fluid ASVs correlation analysis belonged to the amino acid class, malonylcarnitine and carnosine exhibited a higher number and stronger degree of correlations. Given that the correlation analysis of NP fecal ASVs included more than 80% of the significant correlated metabolites belonging to amino acids, these findings may indicate a reduced relationship between the rumen fluid microbiota and amino acid metabolites. The FP rumen fluid correlations were similar to NP rumen fluid correlations, with malonylcarnitine and carnosine demonstrating several significant correlations with different ASVs. More research combining host metabolites and microbiota is needed to fully understand these relationships.</p>
<p id="Par73">The PP group, in addition to the previously mentioned pathways, also exhibited regulatory mechanisms related to RNA degradation and nucleotide metabolism, while the FP group was associated with processes like mismatch repair. These findings suggest potential regulatory roles of prenatal nutrition in microbiome-mediated pathways. However, further research is needed to address the gaps in the literature regarding the complex interactions between epigenetic modifications, regulatory mechanisms, and the microbiome, particularly in the context of prenatal nutrition and its long-term effects on offspring development. In summary, our results reveal significant divergences in the ruminal and fecal microbiomes of offspring from different prenatal nutritional treatments despite uniform management throughout their productive cycle. These microbial differences are interconnected with the host metabolome, influencing multiple metabolic pathways including protein, lipid, and vitamin metabolism, as well as regulatory mechanisms in the ruminal fluid microbiome. This interconnection underscores the lasting impact of prenatal nutrition on offspring development and metabolic function.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Metabolomics-metagenomics integration approach reveals lipid and methane metabolism as main pathways impacted by prenatal nutrition</h3>
<p id="Par74">The results found in the holistic integration analysis revealed the most important metabolic pathways involved in the long-term impacts of prenatal nutrition in beef cattle offspring. The NP group demonstrated close associations with amino acid metabolism and methane metabolism, while the FP group showed a strong relationship with lipid metabolism.</p>
<p id="Par75">Regarding the NP group, we found some pathways similar with our previous analyses (phenylalanine, tyrosine and tryptophan biosynthesis; phenylalanine metabolism) related to amino acid metabolic processes. However, the most significant metabolic pathway identified was associated with methane metabolism. Ruminants emit 16% of worldwide methane (CH<sub>4</sub>), with the beef cattle and dairy sectors accounting for 35% and 30%, respectively [<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>]. Methane is an unnecessary by-product of microbial fermentation of mostly complex carbohydrates in the rumen, produced by methanogenic archaea and expelled into the environment via the animal’s mouth and nose. Eructated methane from ruminal microbial fermentation not only contributes to global warming but also wastes energy, reducing feed efficiency [<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>]. Given the microbiome’s crucial role in methane production, strategies to mitigate emissions in ruminants primarily target dietary interventions that alter rumen conditions and microbial ecology [<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>–<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a>]. Our findings suggest a novel association between prenatal nutrition and methane metabolism pathways in the offspring’s ruminal microbiome. This association could have implications for feed efficiency, as previous research has shown that less feed-efficient steers have a higher relative abundance of methanogens in their ruminal microbial populations compared to high feed-efficient steers [<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>]. Our results raise the intriguing possibility that prenatal nutrition throughout gestation might influence the animal’s long-term feed efficiency and methane production capacity. Further research is needed to elucidate the mechanisms underlying this association and its potential applications in sustainable livestock management and methane mitigation strategies [<a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>].</p>
<p id="Par76">Interestingly, the FP group continues to show a strong association with lipid metabolism, as demonstrated in our previous analyses performed in this study. Glycerophospholipid metabolism (also identified in the brown module of the fecal microbiome), as well as linoleic acid, alpha-linolenic acid, and arachidonic acid metabolism pathways, were shown as the main pathways affected by the FP group in a holistic perspective. Lipid metabolism is a pivotal focus in livestock industry research, whether for milk or meat production [<a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>]. In addition to the lipid roles already discussed (available in the discussion Sect. 4.1 and 4.2), the PUFAs (polyunsaturated fatty acids) play several other important roles. The intramuscular fat content in beef typically includes only 5% PUFAs [<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a>], which can vary depending on nutritional factors [<a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>]. A high ratio of omega-6 (n-6; such as linoleic acid and arachidonic acid) in relation to n-3 fatty acids promotes several illnesses, including cardiovascular disease, arthritis, and cancer, whereas lower levels have suppressive effects in humans [<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a>], pigs [<a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>–<a href="#CR85" class="usa-link" aria-describedby="CR85">85</a>] and mice [<a href="#CR86" class="usa-link" aria-describedby="CR86">86</a>]. Association between maternal low-protein diet and disordered regulation of lipid metabolism in rats was reported [<a href="#CR87" class="usa-link" aria-describedby="CR87">87</a>]. In ruminants, some studies have identified prenatal nutrition effects on lipid metabolism [<a href="#CR88" class="usa-link" aria-describedby="CR88">88</a>, <a href="#CR89" class="usa-link" aria-describedby="CR89">89</a>]. Supporting our findings, Menezes et al. [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>] demonstrated that maternal nutrition significantly influenced fetal hepatic lipid composition in cattle during early gestation. Their study revealed that appropriate maternal nutrition leading to moderate weight gain optimized fetal lipid profiles, resulting in higher omega-3 and lower omega-6 fatty acid abundances in fetal livers. This aligns with our observed alterations in PUFA metabolism pathways. Although Menezes et al. [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>] focused on fetal liver tissue, in contrast to our analysis of blood metabolome and fecal microbiome, both studies highlight the broad impact of prenatal nutrition on offspring lipid metabolism. This concordance across different experimental approaches suggests that prenatal nutrition may have lasting effects on various aspects of lipid metabolism in offspring.</p>
<p id="Par77">In general, our findings corroborate to novel discoveries, particularly related to the association among prenatal nutrition, methane metabolism and PUFA metabolism in beef cattle. Further studies are needed to assess more in-depth the linkage between beef prenatal nutrition and these pathways.</p></section></section><section id="Sec23"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par78">Prenatal nutrition modulated several metabolic pathways in bulls, mainly involved with protein, lipid and methane metabolism, as well as some regulatory mechanisms. Based on all the analyses performed, we conclude that the NP group had a high association with amino acid metabolic processes and methane metabolism, while the FP group showed a strong relationship with lipid metabolism, mainly PUFA metabolism. The PP group demonstrated metabolic pathways associated with regulatory mechanisms and amino acid metabolism. It is important to note that, in addition to exclusive modules within each treatment, modules shared between groups revealed in all contrasts opposing correlations, presumably indicating epigenetic modifications caused by prenatal nutrition. Briefly, the integration of plasma host metabolome and metagenomics (fecal and ruminal fluid) has provided intriguing insights into prenatal nutrition in beef cattle, revealing novel discoveries.</p></section><section id="Sec24"><h2 class="pmc_sec_title">Electronic supplementary material</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/11983759/bin/12864_2025_11545_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (30.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/11983759/bin/12864_2025_11545_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2</a><sup> (28.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/11983759/bin/12864_2025_11545_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3</a><sup> (30.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/11983759/bin/12864_2025_11545_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 4</a><sup> (20.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/11983759/bin/12864_2025_11545_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 5</a><sup> (22.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/11983759/bin/12864_2025_11545_MOESM6_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 6</a><sup> (166.9KB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The authors acknowledge the São Paulo Research Foundation (FAPESP), grant number 17/12105-2,21/03265-1 and 25/05539-2, and the Faculty of Animal Science and Food Engineering (FZEA-USP).</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>ASVs</dt>
<dd><p id="Par82">Amplicon sequence variants</p></dd>
<dt>BCAA</dt>
<dd><p id="Par83">Branched-chain amino acids</p></dd>
<dt>BW</dt>
<dd><p id="Par84">Body weight</p></dd>
<dt>C3-DC C4(OH)</dt>
<dd><p id="Par85">Malonylcarnitine</p></dd>
<dt>CH<sub>4</sub>
</dt>
<dd><p id="Par86">Methane</p></dd>
<dt>CP</dt>
<dd><p id="Par87">Crude protein</p></dd>
<dt>DHA</dt>
<dd><p id="Par88">Docosahexaenoic acid</p></dd>
<dt>DMI</dt>
<dd><p id="Par89">Dry matter intake</p></dd>
<dt>DPA</dt>
<dd><p id="Par90">Docosapentaenoic acid</p></dd>
<dt>EPA</dt>
<dd><p id="Par91">Eicosapentaenoic acid</p></dd>
<dt>FDR</dt>
<dd><p id="Par92">False discovery rate</p></dd>
<dt>FIA-MS/MS</dt>
<dd><p id="Par93">Flow injection analysis-tandem mass spectrometry</p></dd>
<dt>FP</dt>
<dd><p id="Par94">Full programming</p></dd>
<dt>GPC</dt>
<dd><p id="Par95">Glycerophosphocholine</p></dd>
<dt>HPLC</dt>
<dd><p id="Par96">High-performance liquid chromatography</p></dd>
<dt>HPLC-MS/MS</dt>
<dd><p id="Par97">Liquid chromatography tandem-mass spectrometry</p></dd>
<dt>KEGG</dt>
<dd><p id="Par98">Kyoto encyclopedia of genes and genomes</p></dd>
<dt>KO</dt>
<dd><p id="Par99">KEGG ortholog</p></dd>
<dt>LAM</dt>
<dd><p id="Par100">Lipoarabinomannan</p></dd>
<dt>LOD</dt>
<dd><p id="Par101">Limits of detection</p></dd>
<dt>LPS</dt>
<dd><p id="Par102">Lipopolysaccharide</p></dd>
<dt>MAPA</dt>
<dd><p id="Par103">Ministry of Agriculture, Livestock, and Supply of Brazil</p></dd>
<dt>NP</dt>
<dd><p id="Par104">Not programmed</p></dd>
<dt>PP</dt>
<dd><p id="Par105">Partial programming</p></dd>
<dt>PUFAs</dt>
<dd><p id="Par106">Polyunsaturated fatty acids</p></dd>
<dt>r</dt>
<dd><p id="Par107">correlation</p></dd>
<dt>TCA</dt>
<dd><p id="Par108">Tricarboxylic acid</p></dd>
<dt>TDN</dt>
<dd><p id="Par109">Total digestible nutrients</p></dd>
<dt>TLR4</dt>
<dd><p id="Par110">Toll-like receptor 4</p></dd>
<dt>VFAs</dt>
<dd><p id="Par111">Volatile fatty acids</p></dd>
<dt>WGCNA</dt>
<dd><p id="Par112">Weighted gene co-expression network analysis</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Conceptualization, M.H.A.S.; methodology, W.J.S.D., A.S.M.C., G.H.G.P.; formal analysis, G.H.G.P.; investigation, G.H.G.P., A.C.F., É.F., B.C.T.P., G.V.P., E.F.F.D., M.H.A.S.; writing—original draft preparation, G.H.G.P.; writing—review and editing, W.J.S.D., A.S.M.C., A.C., G.D.R.Z., M.H.A.S.; supervision, M.H.A.S.; project administration, M.H.A.S.; funding acquisition, M.H.A.S. and G.H.G.P. All authors have read and agreed to the published version of the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This research was funded by São Paulo Research Foundation (FAPESP), grant number (17/12105-2, 21/03265-1 and 25/05539-2), National Council for Scientific and Technological Development (CNPq) [grant number 307593/2021-5]..</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The metabolome dataset analysed during the current study are available in the supplementary material (Additional file <a href="#MOESM1" class="usa-link">1</a>), as well as the microbiome datasets (fecal microbiome [Additional file <a href="#MOESM2" class="usa-link">2</a>]; rumen fluid microbiome [Additional file <a href="#MOESM3" class="usa-link">3</a>]). The corresponding taxonomy annotation can be found in Additional file <a href="#MOESM4" class="usa-link">4</a> (fecal microbiome) and Additional file <a href="#MOESM5" class="usa-link">5</a> (rumen fluid microbiome). The hierarchical clustering of WGCNA modules for the -OMIC datasets is shown in Additional file <a href="#MOESM6" class="usa-link">6</a>.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par79">This study was approved by the Research Ethics Committee of the Faculty of Animal Science and Food Engineering, University of São Paulo, under protocol No. 1843241117, in compliance with the National Council for the Control of Animal Experimentation recommendations. In addition, the Faculty of Animal Science and Food Engineering provided the animals to carry out this study.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par80">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par81">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Bauman DE, Bruce Currie W. Partitioning of nutrients during pregnancy and lactation: A review of mechanisms involving homeostasis and homeorhesis. J Dairy Sci. 1980;63:1514–29.
</cite> [<a href="https://doi.org/10.3168/jds.s0022-0302(80)83111-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7000867/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bauman%20DE,%20Bruce%20Currie%20W.%20Partitioning%20of%20nutrients%20during%20pregnancy%20and%20lactation:%20A%20review%20of%20mechanisms%20involving%20homeostasis%20and%20homeorhesis.%20J%20Dairy%20Sci.%201980;63:1514%E2%80%9329." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Lemley CO, Camacho LE, Vonnahme KA. Maternal recognition and physiology of pregnancy. Bovine Reprod. 2021;324–38. 10.1002/9781119602484.ch26</cite>
</li>
<li id="CR3">
<span class="label">3.</span><cite>Reynolds LP, Dahlen CR, Ward AK, Crouse MS, Borowicz PP, Davila-Ruiz BJ, et al. Role of the placenta in developmental programming: observations from models using large animals. Anim Reprod Sci. 2023;257:107322.
</cite> [<a href="https://doi.org/10.1016/j.anireprosci.2023.107322" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10591737/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37696224/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reynolds%20LP,%20Dahlen%20CR,%20Ward%20AK,%20Crouse%20MS,%20Borowicz%20PP,%20Davila-Ruiz%20BJ,%20et%20al.%20Role%20of%20the%20placenta%20in%20developmental%20programming:%20observations%20from%20models%20using%20large%20animals.%20Anim%20Reprod%20Sci.%202023;257:107322." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Barcelos S, de Nascimento S, Silva KB, da Mezzomo TE, Alves R, de Souza Duarte KS. The effects of prenatal diet on calf performance and perspectives for fetal programming studies: A Meta-Analytical investigation. Animals. 2022;12:2145.
</cite> [<a href="https://doi.org/10.3390/ani12162145" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9404886/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36009734/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barcelos%20S,%20de%20Nascimento%20S,%20Silva%20KB,%20da%20Mezzomo%20TE,%20Alves%20R,%20de%20Souza%20Duarte%20KS.%20The%20effects%20of%20prenatal%20diet%20on%20calf%20performance%20and%20perspectives%20for%20fetal%20programming%20studies:%20A%20Meta-Analytical%20investigation.%20Animals.%202022;12:2145." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Nascimento KB, Galvão MC, Meneses JAM, Ramírez-Zamudio GD, Pereira DG, Paulino PVR et al. Maternal protein supplementation during mid-gestation improves offspring performance and metabolism in beef cows. J Anim Sci. 2024;102. 10.1093/jas/skae058</cite> [<a href="https://doi.org/10.1093/jas/skae058" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10998463/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38437631/" class="usa-link">PubMed</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Meneses JAM, Nascimento KB, Galvão MC, Ramírez-Zamudio GD, Gionbelli TRS, Ladeira MM, et al. Protein supplementation during mid-gestation alters the amino acid patterns, hepatic metabolism, and maternal skeletal muscle turnover of pregnant Zebu beef cows. Animals. 2022;12:3567.</cite> [<a href="https://doi.org/10.3390/ani12243567" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9774392/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36552487/" class="usa-link">PubMed</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Nascimento KB, Galvão MC, Meneses JAM, Moreira GM, Ramírez-Zamudio GD, de Souza SP, et al. Effects of Maternal Protein Supplementation at Mid-Gestation of Cows on Intake, Digestibility, and Feeding Behavior of the Offspring. Animals. 2022;12:2865.</cite> [<a href="https://doi.org/10.3390/ani12202865" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9597828/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36290250/" class="usa-link">PubMed</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Rodrigues LM, Schoonmaker JP, Resende FD, Siqueira GR, Neto ORM, Gionbelli MP, et al. Effects of protein supplementation on Nellore cows’ reproductive performance, growth, myogenesis, lipogenesis and intestine development of the progeny. Anim Prod Sci. 2020;61:371–80.</cite> [<a href="https://scholar.google.com/scholar_lookup?Rodrigues%20LM,%20Schoonmaker%20JP,%20Resende%20FD,%20Siqueira%20GR,%20Neto%20ORM,%20Gionbelli%20MP,%20et%20al.%20Effects%20of%20protein%20supplementation%20on%20Nellore%20cows%E2%80%99%20reproductive%20performance,%20growth,%20myogenesis,%20lipogenesis%20and%20intestine%20development%20of%20the%20progeny.%20Anim%20Prod%20Sci.%202020;61:371%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Daniel JB, Friggens NC, Van Laar H, Ingvartsen KL, Sauvant D. Modeling homeorhetic trajectories of milk component yields, body composition and dry-matter intake in dairy cows: influence of parity, milk production potential and breed. Animal. 2018;12:1182–95.
</cite> [<a href="https://doi.org/10.1017/S1751731117002828" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29098979/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Daniel%20JB,%20Friggens%20NC,%20Van%20Laar%20H,%20Ingvartsen%20KL,%20Sauvant%20D.%20Modeling%20homeorhetic%20trajectories%20of%20milk%20component%20yields,%20body%20composition%20and%20dry-matter%20intake%20in%20dairy%20cows:%20influence%20of%20parity,%20milk%20production%20potential%20and%20breed.%20Animal.%202018;12:1182%E2%80%9395." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Redifer CA, Wichman LG, Davies-Jenkins SL, Rathert-Williams AR, Freetly HC, Meyer AM. Late gestational nutrient restriction in primiparous beef females: performance and metabolic status of lactating dams and pre-weaning calves. J Anim Sci. 2024;102. 10.1093/jas/skae015</cite> [<a href="https://doi.org/10.1093/jas/skae015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10894509/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38243834/" class="usa-link">PubMed</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Fernandes AC, Henrique G, Polizel G, Cracco RC, Correia FA, Cançado Q, et al. Metabolomics changes in meat and subcutaneous fat of male cattle submitted to fetal programming. Metabolites. 2023;14:9.</cite> [<a href="https://doi.org/10.3390/metabo14010009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10819762/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38248812/" class="usa-link">PubMed</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Cracco RC, Alexandre PA, Polizel GHG, Fernandes AC, de Almeida Santana MH. Evaluation of muscle long Non-Coding RNA profile during rearing and finishing phase of bulls subjected to different prenatal nutritional strategies. Animals. 2024;14:652.
</cite> [<a href="https://doi.org/10.3390/ani14040652" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10886332/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38396620/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cracco%20RC,%20Alexandre%20PA,%20Polizel%20GHG,%20Fernandes%20AC,%20de%20Almeida%20Santana%20MH.%20Evaluation%20of%20muscle%20long%20Non-Coding%20RNA%20profile%20during%20rearing%20and%20finishing%20phase%20of%20bulls%20subjected%20to%20different%20prenatal%20nutritional%20strategies.%20Animals.%202024;14:652." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Marquez DC, Paulino MF, Rennó LN, Villadiego FC, Ortega RM, Moreno DS, et al. Supplementation of grazing beef cows during gestation as a strategy to improve skeletal muscle development of the offspring. Animal. 2017;11:2184–92.
</cite> [<a href="https://doi.org/10.1017/S1751731117000982" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28571587/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Marquez%20DC,%20Paulino%20MF,%20Renn%C3%B3%20LN,%20Villadiego%20FC,%20Ortega%20RM,%20Moreno%20DS,%20et%20al.%20Supplementation%20of%20grazing%20beef%20cows%20during%20gestation%20as%20a%20strategy%20to%20improve%20skeletal%20muscle%20development%20of%20the%20offspring.%20Animal.%202017;11:2184%E2%80%9392." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Jiang B, Qin C, Xu Y, Song X, Fu Y, Li R, et al. Multi-omics reveals the mechanism of rumen Microbiome and its metabolome together with host metabolome participating in the regulation of milk production traits in dairy buffaloes. Front Microbiol. 2024;15:1301292.
</cite> [<a href="https://doi.org/10.3389/fmicb.2024.1301292" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10959287/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38525073/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiang%20B,%20Qin%20C,%20Xu%20Y,%20Song%20X,%20Fu%20Y,%20Li%20R,%20et%20al.%20Multi-omics%20reveals%20the%20mechanism%20of%20rumen%20Microbiome%20and%20its%20metabolome%20together%20with%20host%20metabolome%20participating%20in%20the%20regulation%20of%20milk%20production%20traits%20in%20dairy%20buffaloes.%20Front%20Microbiol.%202024;15:1301292." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Kim M. Assessment of the Gastrointestinal microbiota using 16S ribosomal RNA gene amplicon sequencing in ruminant nutrition. Anim Biosci. 2023;36:364–73.
</cite> [<a href="https://doi.org/10.5713/ab.22.0382" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9899581/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36701925/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20M.%20Assessment%20of%20the%20Gastrointestinal%20microbiota%20using%2016S%20ribosomal%20RNA%20gene%20amplicon%20sequencing%20in%20ruminant%20nutrition.%20Anim%20Biosci.%202023;36:364%E2%80%9373." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Klein-Jöbstl D, Quijada NM, Dzieciol M, Feldbacher B, Wagner M, Drillich M, et al. Microbiota of newborn calves and their mothers reveals possible transfer routes for newborn calves’ Gastrointestinal microbiota. PLoS ONE. 2019;14:e0220554.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0220554" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6675284/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31369600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Klein-J%C3%B6bstl%20D,%20Quijada%20NM,%20Dzieciol%20M,%20Feldbacher%20B,%20Wagner%20M,%20Drillich%20M,%20et%20al.%20Microbiota%20of%20newborn%20calves%20and%20their%20mothers%20reveals%20possible%20transfer%20routes%20for%20newborn%20calves%E2%80%99%20Gastrointestinal%20microbiota.%20PLoS%20ONE.%202019;14:e0220554." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Jin S, Zhang Z, Zhang G, He B, Qin Y, Yang B, et al. Maternal rumen bacteriota shapes the offspring rumen bacteriota, affecting the development of young ruminants. Microbiol Spectr. 2023;11.</cite> [<a href="https://doi.org/10.1128/spectrum.03590-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10100811/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36809041/" class="usa-link">PubMed</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Yáñez-Ruiz DR, Abecia L, Newbold CJ. Manipulating rumen microbiome and fermentation through interventions during early life: A review. Front Microbiol. 2015;6:162946.</cite> [<a href="https://doi.org/10.3389/fmicb.2015.01133" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4604304/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26528276/" class="usa-link">PubMed</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Belanche A, Palma-Hidalgo JM, Nejjam I, Jiménez E, Martín-García AI, Yáñez-Ruiz DR. Inoculation with rumen fluid in early life as a strategy to optimize the weaning process in intensive dairy goat systems. J Dairy Sci. 2020;103:5047–60.
</cite> [<a href="https://doi.org/10.3168/jds.2019-18002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32278566/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Belanche%20A,%20Palma-Hidalgo%20JM,%20Nejjam%20I,%20Jim%C3%A9nez%20E,%20Mart%C3%ADn-Garc%C3%ADa%20AI,%20Y%C3%A1%C3%B1ez-Ruiz%20DR.%20Inoculation%20with%20rumen%20fluid%20in%20early%20life%20as%20a%20strategy%20to%20optimize%20the%20weaning%20process%20in%20intensive%20dairy%20goat%20systems.%20J%20Dairy%20Sci.%202020;103:5047%E2%80%9360." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Palma-Hidalgo JM, Jiménez E, Popova M, Morgavi DP, Martín-García AI, Yáñez-Ruiz DR, et al. Inoculation with rumen fluid in early life accelerates the rumen microbial development and favours the weaning process in goats. Anim Microbiome. 2021;3:1–21.
</cite> [<a href="https://doi.org/10.1186/s42523-021-00073-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7814744/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33499992/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Palma-Hidalgo%20JM,%20Jim%C3%A9nez%20E,%20Popova%20M,%20Morgavi%20DP,%20Mart%C3%ADn-Garc%C3%ADa%20AI,%20Y%C3%A1%C3%B1ez-Ruiz%20DR,%20et%20al.%20Inoculation%20with%20rumen%20fluid%20in%20early%20life%20accelerates%20the%20rumen%20microbial%20development%20and%20favours%20the%20weaning%20process%20in%20goats.%20Anim%20Microbiome.%202021;3:1%E2%80%9321." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Amat S, Holman DB, Schmidt K, Menezes ACB, Baumgaertner F, Winders T, et al. The nasopharyngeal, ruminal, and vaginal microbiota and the core taxa shared across these microbiomes in Virgin yearling heifers exposed to divergent in utero nutrition during their first trimester of gestation and in pregnant beef heifers in response to mineral supplementation. Microorganisms. 2021;9:2011.
</cite> [<a href="https://doi.org/10.3390/microorganisms9102011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8537542/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34683332/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Amat%20S,%20Holman%20DB,%20Schmidt%20K,%20Menezes%20ACB,%20Baumgaertner%20F,%20Winders%20T,%20et%20al.%20The%20nasopharyngeal,%20ruminal,%20and%20vaginal%20microbiota%20and%20the%20core%20taxa%20shared%20across%20these%20microbiomes%20in%20Virgin%20yearling%20heifers%20exposed%20to%20divergent%20in%20utero%20nutrition%20during%20their%20first%20trimester%20of%20gestation%20and%20in%20pregnant%20beef%20heifers%20in%20response%20to%20mineral%20supplementation.%20Microorganisms.%202021;9:2011." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Bickhart DM, Weimer PJ. Symposium review: Host–rumen microbe interactions may be leveraged to improve the productivity of dairy cows. J Dairy Sci. 2018;101:7680–9.</cite> [<a href="https://doi.org/10.3168/jds.2017-13328" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29102146/" class="usa-link">PubMed</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Wang Y, Nan X, Zhao Y, Jiang L, Wang M, Wang H, et al. Rumen Microbiome structure and metabolites activity in dairy cows with clinical and subclinical mastitis. J Anim Sci Biotechnol. 2021;12:1–21.
</cite> [<a href="https://doi.org/10.1186/s40104-020-00543-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7869221/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33557959/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20Y,%20Nan%20X,%20Zhao%20Y,%20Jiang%20L,%20Wang%20M,%20Wang%20H,%20et%20al.%20Rumen%20Microbiome%20structure%20and%20metabolites%20activity%20in%20dairy%20cows%20with%20clinical%20and%20subclinical%20mastitis.%20J%20Anim%20Sci%20Biotechnol.%202021;12:1%E2%80%9321." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Danielsson R, Dicksved J, Sun L, Gonda H, Müller B, Schnürer A, et al. Methane production in dairy cows correlates with rumen methanogenic and bacterial community structure. Front Microbiol. 2017;8(FEB):226104.</cite> [<a href="https://doi.org/10.3389/fmicb.2017.00226" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5313486/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28261182/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Danielsson%20R,%20Dicksved%20J,%20Sun%20L,%20Gonda%20H,%20M%C3%BCller%20B,%20Schn%C3%BCrer%20A,%20et%20al.%20Methane%20production%20in%20dairy%20cows%20correlates%20with%20rumen%20methanogenic%20and%20bacterial%20community%20structure.%20Front%20Microbiol.%202017;8(FEB):226104." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Xue MY, Sun HZ, Wu XH, Liu JX, Guan LL. Multi-omics reveals that the rumen Microbiome and its metabolome together with the host metabolome contribute to individualized dairy cow performance. Microbiome. 2020;8:1–19.
</cite> [<a href="https://doi.org/10.1186/s40168-020-00819-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7218573/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32398126/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xue%20MY,%20Sun%20HZ,%20Wu%20XH,%20Liu%20JX,%20Guan%20LL.%20Multi-omics%20reveals%20that%20the%20rumen%20Microbiome%20and%20its%20metabolome%20together%20with%20the%20host%20metabolome%20contribute%20to%20individualized%20dairy%20cow%20performance.%20Microbiome.%202020;8:1%E2%80%9319." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Asnicar F, Berry SE, Valdes AM, Nguyen LH, Piccinno G, Drew DA, et al. Microbiome connections with host metabolism and habitual diet from 1,098 deeply phenotyped individuals. Nat Med. 2021;27:2.</cite> [<a href="https://doi.org/10.1038/s41591-020-01183-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8353542/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33432175/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Asnicar%20F,%20Berry%20SE,%20Valdes%20AM,%20Nguyen%20LH,%20Piccinno%20G,%20Drew%20DA,%20et%20al.%20Microbiome%20connections%20with%20host%20metabolism%20and%20habitual%20diet%20from%201,098%20deeply%20phenotyped%20individuals.%20Nat%20Med.%202021;27:2." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Li H, Liu S, Zhang K, Zhu X, Dai J, Lu Y. Gut Microbiome and plasma metabolome alterations in myopic mice. Front Microbiol. 2023;14:1251243.
</cite> [<a href="https://doi.org/10.3389/fmicb.2023.1251243" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10764480/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38179454/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20H,%20Liu%20S,%20Zhang%20K,%20Zhu%20X,%20Dai%20J,%20Lu%20Y.%20Gut%20Microbiome%20and%20plasma%20metabolome%20alterations%20in%20myopic%20mice.%20Front%20Microbiol.%202023;14:1251243." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Luo Z, Yong K, Luo Q, Du Z, Ma L, Huang Y, et al. Altered fecal Microbiome and correlations of the metabolome with plasma metabolites in dairy cows with left displaced abomasum. Microbiol Spectr. 2022;10.</cite> [<a href="https://doi.org/10.1128/spectrum.01972-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9769586/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36222683/" class="usa-link">PubMed</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Schalch Junior FJ, Polizel GHG, Cançado FACQ, Fernandes AC, Mortari I, Pires PRL, et al. Prenatal supplementation in beef cattle and its effects on plasma metabolome of dams and calves. Metabolites. 2022;12:347.
</cite> [<a href="https://doi.org/10.3390/metabo12040347" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9028846/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35448533/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schalch%20Junior%20FJ,%20Polizel%20GHG,%20Can%C3%A7ado%20FACQ,%20Fernandes%20AC,%20Mortari%20I,%20Pires%20PRL,%20et%20al.%20Prenatal%20supplementation%20in%20beef%20cattle%20and%20its%20effects%20on%20plasma%20metabolome%20of%20dams%20and%20calves.%20Metabolites.%202022;12:347." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Dias EFF, de Carvalho FE, Polizel GHG, Cançado FACQ, Furlan É, Fernandes AC, et al. Fetal programming influence on Microbiome diversity and ruminal and cecal epithelium in beef cattle. Animals. 2024;14:870.
</cite> [<a href="https://doi.org/10.3390/ani14060870" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10967552/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38539968/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dias%20EFF,%20de%20Carvalho%20FE,%20Polizel%20GHG,%20Can%C3%A7ado%20FACQ,%20Furlan%20%C3%89,%20Fernandes%20AC,%20et%20al.%20Fetal%20programming%20influence%20on%20Microbiome%20diversity%20and%20ruminal%20and%20cecal%20epithelium%20in%20beef%20cattle.%20Animals.%202024;14:870." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Fernandes AC, Beline M, Polizel GHG, Cracco RC, Dias EFF, Furlan É, et al. Fetal programming and its effects on meat quality of nellore bulls. Vet Sci. 2023;10:672.</cite> [<a href="https://doi.org/10.3390/vetsci10120672" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10748345/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38133223/" class="usa-link">PubMed</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Klindworth A, Pruesse E, Schweer T, Peplies J, Quast C, Horn M, et al. Evaluation of general 16S ribosomal RNA gene PCR primers for classical and next-generation sequencing-based diversity studies. Nucleic Acids Res. 2013;41:e1–1.
</cite> [<a href="https://doi.org/10.1093/nar/gks808" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3592464/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22933715/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Klindworth%20A,%20Pruesse%20E,%20Schweer%20T,%20Peplies%20J,%20Quast%20C,%20Horn%20M,%20et%20al.%20Evaluation%20of%20general%2016S%20ribosomal%20RNA%20gene%20PCR%20primers%20for%20classical%20and%20next-generation%20sequencing-based%20diversity%20studies.%20Nucleic%20Acids%20Res.%202013;41:e1%E2%80%931." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Callahan BJ, McMurdie PJ, Rosen MJ, Han AW, Johnson AJA, Holmes SP. DADA2: High-resolution sample inference from illumina amplicon data. Nat Methods 2016;13:7.</cite> [<a href="https://doi.org/10.1038/nmeth.3869" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4927377/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27214047/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Callahan%20BJ,%20McMurdie%20PJ,%20Rosen%20MJ,%20Han%20AW,%20Johnson%20AJA,%20Holmes%20SP.%20DADA2:%20High-resolution%20sample%20inference%20from%20illumina%20amplicon%20data.%20Nat%20Methods%202016;13:7." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Pruesse E, Quast C, Knittel K, Fuchs BM, Ludwig W, Peplies J, et al. SILVA: a comprehensive online resource for quality checked and aligned ribosomal RNA sequence data compatible with ARB. Nucleic Acids Res. 2007;35:7188–96.
</cite> [<a href="https://doi.org/10.1093/nar/gkm864" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2175337/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17947321/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pruesse%20E,%20Quast%20C,%20Knittel%20K,%20Fuchs%20BM,%20Ludwig%20W,%20Peplies%20J,%20et%20al.%20SILVA:%20a%20comprehensive%20online%20resource%20for%20quality%20checked%20and%20aligned%20ribosomal%20RNA%20sequence%20data%20compatible%20with%20ARB.%20Nucleic%20Acids%20Res.%202007;35:7188%E2%80%9396." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>McMurdie PJ, Holmes S. Phyloseq: an R package for reproducible interactive analysis and graphics of Microbiome census data. PLoS ONE. 2013;8:e61217.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0061217" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3632530/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23630581/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McMurdie%20PJ,%20Holmes%20S.%20Phyloseq:%20an%20R%20package%20for%20reproducible%20interactive%20analysis%20and%20graphics%20of%20Microbiome%20census%20data.%20PLoS%20ONE.%202013;8:e61217." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Langfelder P, Horvath S. WGCNA: an R package for weighted correlation network analysis. BMC Bioinformatics. 2008;9:1–13.
</cite> [<a href="https://doi.org/10.1186/1471-2105-9-559" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2631488/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19114008/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Langfelder%20P,%20Horvath%20S.%20WGCNA:%20an%20R%20package%20for%20weighted%20correlation%20network%20analysis.%20BMC%20Bioinformatics.%202008;9:1%E2%80%9313." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Zhang B, Horvath S. A general framework for weighted gene co-expression network analysis. Stat Appl Genet Mol Biol. 2005;4.</cite> [<a href="https://doi.org/10.2202/1544-6115.1128" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16646834/" class="usa-link">PubMed</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Fonseca PAS, Suárez-Vega A, Cánovas A. Weighted gene correlation network meta-analysis reveals functional candidate genes associated with high- and sub-fertile reproductive performance in beef cattle. Genes. 2020;11:543.</cite> [<a href="https://doi.org/10.3390/genes11050543" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7290847/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32408659/" class="usa-link">PubMed</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>de Souza Fonseca PA, Suárez-Vega A, Cánovas A. Unrevealing functional candidate genes for bovine fertility through RNA sequencing meta-analysis and regulatory elements networks of co-expressed genes and LncRNAs. Funct Integr Genomics. 2022;22:1361–76.
</cite> [<a href="https://doi.org/10.1007/s10142-022-00893-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36001276/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?de%20Souza%20Fonseca%20PA,%20Su%C3%A1rez-Vega%20A,%20C%C3%A1novas%20A.%20Unrevealing%20functional%20candidate%20genes%20for%20bovine%20fertility%20through%20RNA%20sequencing%20meta-analysis%20and%20regulatory%20elements%20networks%20of%20co-expressed%20genes%20and%20LncRNAs.%20Funct%20Integr%20Genomics.%202022;22:1361%E2%80%9376." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Kanehisa M, Sato Y, Morishima K. BlastKOALA and GhostKOALA: KEGG tools for functional characterization of genome and metagenome sequences. J Mol Biol. 2016;428:726–31.
</cite> [<a href="https://doi.org/10.1016/j.jmb.2015.11.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26585406/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kanehisa%20M,%20Sato%20Y,%20Morishima%20K.%20BlastKOALA%20and%20GhostKOALA:%20KEGG%20tools%20for%20functional%20characterization%20of%20genome%20and%20metagenome%20sequences.%20J%20Mol%20Biol.%202016;428:726%E2%80%9331." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Melkonian C, Gottstein W, Blasche S, Kim Y, Abel-Kistrup M, Swiegers H, et al. Finding functional differences between species in a microbial community: Case studies in wine fermentation and Kefir culture. Front Microbiol. 2019;10 JUN:448903.</cite> [<a href="https://doi.org/10.3389/fmicb.2019.01347" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6603220/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31293529/" class="usa-link">PubMed</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Pang Z, Lu Y, an Zhou G, Hui F, Xu L, Viau C, et al. MetaboAnalyst 6.0: towards a unified platform for metabolomics data processing, analysis and interpretation. Nucleic Acids Res. 2024;52:W398–406.
</cite> [<a href="https://doi.org/10.1093/nar/gkae253" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11223798/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38587201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pang%20Z,%20Lu%20Y,%20an%20Zhou%20G,%20Hui%20F,%20Xu%20L,%20Viau%20C,%20et%20al.%20MetaboAnalyst%206.0:%20towards%20a%20unified%20platform%20for%20metabolomics%20data%20processing,%20analysis%20and%20interpretation.%20Nucleic%20Acids%20Res.%202024;52:W398%E2%80%93406." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Lu Y, Zhou G, Ewald J, Pang Z, Shiri T, Xia J. MicrobiomeAnalyst 2.0: comprehensive statistical, functional and integrative analysis of Microbiome data. Nucleic Acids Res. 2023;51:W310–8.
</cite> [<a href="https://doi.org/10.1093/nar/gkad407" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10320150/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37166960/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu%20Y,%20Zhou%20G,%20Ewald%20J,%20Pang%20Z,%20Shiri%20T,%20Xia%20J.%20MicrobiomeAnalyst%202.0:%20comprehensive%20statistical,%20functional%20and%20integrative%20analysis%20of%20Microbiome%20data.%20Nucleic%20Acids%20Res.%202023;51:W310%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Greenwood PL, Cafe LM. Prenatal and pre-weaning growth and nutrition of cattle: Long-term consequences for beef production. Animal. 2007;1:1283–96.
</cite> [<a href="https://doi.org/10.1017/S175173110700050X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22444884/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Greenwood%20PL,%20Cafe%20LM.%20Prenatal%20and%20pre-weaning%20growth%20and%20nutrition%20of%20cattle:%20Long-term%20consequences%20for%20beef%20production.%20Animal.%202007;1:1283%E2%80%9396." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Greenwood PL, Bell AW. Developmental programming and growth of livestock tissues for meat production. Veterinary Clin North Am - Food Anim Pract. 2019;35:303–19.</cite> [<a href="https://doi.org/10.1016/j.cvfa.2019.02.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31103183/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Greenwood%20PL,%20Bell%20AW.%20Developmental%20programming%20and%20growth%20of%20livestock%20tissues%20for%20meat%20production.%20Veterinary%20Clin%20North%20Am%20-%20Food%20Anim%20Pract.%202019;35:303%E2%80%9319." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Wu G, Bazer FW, Wallace JM, Spencer TE. Board-invited review: intrauterine growth retardation: implications for the animal sciences. J Anim Sci. 2006;84:2316–37.
</cite> [<a href="https://doi.org/10.2527/jas.2006-156" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16908634/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu%20G,%20Bazer%20FW,%20Wallace%20JM,%20Spencer%20TE.%20Board-invited%20review:%20intrauterine%20growth%20retardation:%20implications%20for%20the%20animal%20sciences.%20J%20Anim%20Sci.%202006;84:2316%E2%80%9337." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Reynolds LP, Borowicz PP, Caton JS, Crouse MS, Dahlen CR, Ward AK. Developmental programming of fetal growth and development. Veterinary clinics of North America. Food Anim Pract. 2019;35:229–47.</cite> [<a href="https://doi.org/10.1016/j.cvfa.2019.02.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31103178/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reynolds%20LP,%20Borowicz%20PP,%20Caton%20JS,%20Crouse%20MS,%20Dahlen%20CR,%20Ward%20AK.%20Developmental%20programming%20of%20fetal%20growth%20and%20development.%20Veterinary%20clinics%20of%20North%20America.%20Food%20Anim%20Pract.%202019;35:229%E2%80%9347." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Maghsoudi Z, Nguyen H, Tavakkoli A, Nguyen T. A comprehensive survey of the approaches for pathway analysis using multi-omics data integration. Brief Bioinform. 2022;23.</cite> [<a href="https://doi.org/10.1093/bib/bbac435" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9677478/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36252928/" class="usa-link">PubMed</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Bröer S, Bröer A. Amino acid homeostasis and signalling in mammalian cells and organisms. Biochem J. 2017;474:1935–63.
</cite> [<a href="https://doi.org/10.1042/BCJ20160822" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5444488/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28546457/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Br%C3%B6er%20S,%20Br%C3%B6er%20A.%20Amino%20acid%20homeostasis%20and%20signalling%20in%20mammalian%20cells%20and%20organisms.%20Biochem%20J.%202017;474:1935%E2%80%9363." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Muroya S, Zhang Y, Kinoshita A, Otomaru K, Oshima K, Gotoh Y, et al. Maternal undernutrition during pregnancy alters amino acid metabolism and gene expression associated with energy metabolism and angiogenesis in fetal calf muscle. Metabolites. 2021;11:582.
</cite> [<a href="https://doi.org/10.3390/metabo11090582" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8465837/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34564398/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Muroya%20S,%20Zhang%20Y,%20Kinoshita%20A,%20Otomaru%20K,%20Oshima%20K,%20Gotoh%20Y,%20et%20al.%20Maternal%20undernutrition%20during%20pregnancy%20alters%20amino%20acid%20metabolism%20and%20gene%20expression%20associated%20with%20energy%20metabolism%20and%20angiogenesis%20in%20fetal%20calf%20muscle.%20Metabolites.%202021;11:582." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Elolimy A, Alharthi A, Zeineldin M, Parys C, Helmbrecht A, Loor JJ. Supply of methionine during Late-Pregnancy alters fecal microbiota and metabolome in neonatal dairy calves without changes in daily feed intake. Front Microbiol. 2019;10:442493.</cite> [<a href="https://doi.org/10.3389/fmicb.2019.02159" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6761860/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31608024/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Elolimy%20A,%20Alharthi%20A,%20Zeineldin%20M,%20Parys%20C,%20Helmbrecht%20A,%20Loor%20JJ.%20Supply%20of%20methionine%20during%20Late-Pregnancy%20alters%20fecal%20microbiota%20and%20metabolome%20in%20neonatal%20dairy%20calves%20without%20changes%20in%20daily%20feed%20intake.%20Front%20Microbiol.%202019;10:442493." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Polizel GHG, Fernandes AC, Furlan É, Prati BCT, Ferraz JBS, de Santana MH. Impacts of different prenatal supplementation strategies on the plasma metabolome of bulls in the rearing and finishing phase. Metabolites. 2023;13:259.
</cite> [<a href="https://doi.org/10.3390/metabo13020259" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9960736/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36837878/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Polizel%20GHG,%20Fernandes%20AC,%20Furlan%20%C3%89,%20Prati%20BCT,%20Ferraz%20JBS,%20de%20Santana%20MH.%20Impacts%20of%20different%20prenatal%20supplementation%20strategies%20on%20the%20plasma%20metabolome%20of%20bulls%20in%20the%20rearing%20and%20finishing%20phase.%20Metabolites.%202023;13:259." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Sparks DL, Doelle H, Chatterjee C. Circulating nucleotides in health and disease. Receptors Clin Investig. 2014;1:344.</cite> [<a href="https://scholar.google.com/scholar_lookup?Sparks%20DL,%20Doelle%20H,%20Chatterjee%20C.%20Circulating%20nucleotides%20in%20health%20and%20disease.%20Receptors%20Clin%20Investig.%202014;1:344." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Li HH, Pan JL, Hui S, Ma XW, Wang ZL, Yao HX, et al. Retracted Article: High-throughput metabolomics identifies serum metabolic signatures in acute kidney injury using LC-MS combined with pattern recognition approach. RSC Adv. 2018;8:14838–47.
</cite> [<a href="https://doi.org/10.1039/c8ra01749b" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9079920/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35541357/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20HH,%20Pan%20JL,%20Hui%20S,%20Ma%20XW,%20Wang%20ZL,%20Yao%20HX,%20et%20al.%20Retracted%20Article:%20High-throughput%20metabolomics%20identifies%20serum%20metabolic%20signatures%20in%20acute%20kidney%20injury%20using%20LC-MS%20combined%20with%20pattern%20recognition%20approach.%20RSC%20Adv.%202018;8:14838%E2%80%9347." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="/articles/PMC8698548/" class="text-red">Retracted</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Bergwerff CE, Luman M, Blom HJ, Oosterlaan J. No tryptophan, tyrosine and phenylalanine abnormalities in children with attention-deficit/hyperactivity disorder. PLoS ONE. 2016;11:e0151100.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0151100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4777504/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26938936/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bergwerff%20CE,%20Luman%20M,%20Blom%20HJ,%20Oosterlaan%20J.%20No%20tryptophan,%20tyrosine%20and%20phenylalanine%20abnormalities%20in%20children%20with%20attention-deficit/hyperactivity%20disorder.%20PLoS%20ONE.%202016;11:e0151100." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Nie C, He T, Zhang W, Zhang G, Ma X. Branched chain amino acids: beyond nutrition metabolism. Int J Mol Sci. 2018;19:954.</cite> [<a href="https://doi.org/10.3390/ijms19040954" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5979320/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29570613/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nie%20C,%20He%20T,%20Zhang%20W,%20Zhang%20G,%20Ma%20X.%20Branched%20chain%20amino%20acids:%20beyond%20nutrition%20metabolism.%20Int%20J%20Mol%20Sci.%202018;19:954." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Kim GW, Chung SH. The beneficial effect of Glycerophosphocholine to local fat accumulation: a comparative study with phosphatidylcholine and aminophylline. Korean J Physiol Pharmacol. 2021;25:333–9.
</cite> [<a href="https://doi.org/10.4196/kjpp.2021.25.4.333" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8255124/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34187950/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20GW,%20Chung%20SH.%20The%20beneficial%20effect%20of%20Glycerophosphocholine%20to%20local%20fat%20accumulation:%20a%20comparative%20study%20with%20phosphatidylcholine%20and%20aminophylline.%20Korean%20J%20Physiol%20Pharmacol.%202021;25:333%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Gallazzini M, Burg MB. What’s new about osmotic regulation of Glycerophosphocholine. Physiology. 2009;24:245–9.
</cite> [<a href="https://doi.org/10.1152/physiol.00009.2009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2943332/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19675355/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gallazzini%20M,%20Burg%20MB.%20What%E2%80%99s%20new%20about%20osmotic%20regulation%20of%20Glycerophosphocholine.%20Physiology.%202009;24:245%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Merrill AH. Sphingolipids. Biochemistry of lipids. Lipoproteins Membr. 2008;363–97. 10.1016/B978-044453219-0.50015-5</cite>
</li>
<li id="CR60">
<span class="label">60.</span><cite>Wang J, Chen YL, Li YK, Chen DK, He JF, Yao N. Functions of sphingolipids in pathogenesis during host–pathogen interactions. Front Microbiol. 2021;12:701041.
</cite> [<a href="https://doi.org/10.3389/fmicb.2021.701041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8366399/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34408731/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20J,%20Chen%20YL,%20Li%20YK,%20Chen%20DK,%20He%20JF,%20Yao%20N.%20Functions%20of%20sphingolipids%20in%20pathogenesis%20during%20host%E2%80%93pathogen%20interactions.%20Front%20Microbiol.%202021;12:701041." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Menezes ACB, Dahlen CR, McCarthy KL, Kassetas CJ, Baumgaertner F, Kirsch JD, et al. Fetal hepatic lipidome is more greatly affected by maternal rate of gain compared with vitamin and mineral supplementation at day 83 of gestation. Metabolites. 2023;13:175.
</cite> [<a href="https://doi.org/10.3390/metabo13020175" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9961797/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36837794/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Menezes%20ACB,%20Dahlen%20CR,%20McCarthy%20KL,%20Kassetas%20CJ,%20Baumgaertner%20F,%20Kirsch%20JD,%20et%20al.%20Fetal%20hepatic%20lipidome%20is%20more%20greatly%20affected%20by%20maternal%20rate%20of%20gain%20compared%20with%20vitamin%20and%20mineral%20supplementation%20at%20day%2083%20of%20gestation.%20Metabolites.%202023;13:175." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Schober P, Schwarte LA. Correlation coefficients: appropriate use and interpretation. Anesth Analg. 2018;126:1763–8.
</cite> [<a href="https://doi.org/10.1213/ANE.0000000000002864" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29481436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schober%20P,%20Schwarte%20LA.%20Correlation%20coefficients:%20appropriate%20use%20and%20interpretation.%20Anesth%20Analg.%202018;126:1763%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Hullar MAJ, Fu BC. Diet, the gut microbiome, and epigenetics. Cancer J (United States). 2014;20:170–5.</cite> [<a href="https://doi.org/10.1097/PPO.0000000000000053" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4267719/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24855003/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hullar%20MAJ,%20Fu%20BC.%20Diet,%20the%20gut%20microbiome,%20and%20epigenetics.%20Cancer%20J%20(United%20States).%202014;20:170%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Takahashi K, Sugi Y, Nakano K, Tsuda M, Kurihara K, Hosono A, et al. Epigenetic control of the host gene by commensal bacteria in large intestinal epithelial cells. J Biol Chem. 2011;286:35755–62.
</cite> [<a href="https://doi.org/10.1074/jbc.M111.271007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3195625/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21862578/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Takahashi%20K,%20Sugi%20Y,%20Nakano%20K,%20Tsuda%20M,%20Kurihara%20K,%20Hosono%20A,%20et%20al.%20Epigenetic%20control%20of%20the%20host%20gene%20by%20commensal%20bacteria%20in%20large%20intestinal%20epithelial%20cells.%20J%20Biol%20Chem.%202011;286:35755%E2%80%9362." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Takahashi K, Sugi Y, Hosono A, Kaminogawa S. Epigenetic regulation of TLR4 gene expression in intestinal epithelial cells for the maintenance of intestinal homeostasis. J Immunol. 2009;183:6522–9.
</cite> [<a href="https://doi.org/10.4049/jimmunol.0901271" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19846881/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Takahashi%20K,%20Sugi%20Y,%20Hosono%20A,%20Kaminogawa%20S.%20Epigenetic%20regulation%20of%20TLR4%20gene%20expression%20in%20intestinal%20epithelial%20cells%20for%20the%20maintenance%20of%20intestinal%20homeostasis.%20J%20Immunol.%202009;183:6522%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Cortese R, Lu L, Yu Y, Ruden D, Claud EC. Epigenome-Microbiome crosstalk: A potential new paradigm influencing neonatal susceptibility to disease. Epigenetics. 2016;11:205–15.
</cite> [<a href="https://doi.org/10.1080/15592294.2016.1155011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4854540/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26909656/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cortese%20R,%20Lu%20L,%20Yu%20Y,%20Ruden%20D,%20Claud%20EC.%20Epigenome-Microbiome%20crosstalk:%20A%20potential%20new%20paradigm%20influencing%20neonatal%20susceptibility%20to%20disease.%20Epigenetics.%202016;11:205%E2%80%9315." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Wampach L, Heintz-Buschart A, Fritz JV, Ramiro-Garcia J, Habier J, Herold M, et al. Birth mode is associated with earliest strain-conferred gut Microbiome functions and immunostimulatory potential. Nat Commun. 2018;9:1–14.</cite> [<a href="https://doi.org/10.1038/s41467-018-07631-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6269548/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30504906/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wampach%20L,%20Heintz-Buschart%20A,%20Fritz%20JV,%20Ramiro-Garcia%20J,%20Habier%20J,%20Herold%20M,%20et%20al.%20Birth%20mode%20is%20associated%20with%20earliest%20strain-conferred%20gut%20Microbiome%20functions%20and%20immunostimulatory%20potential.%20Nat%20Commun.%202018;9:1%E2%80%9314." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Kuroishi T. Regulation of immunological and inflammatory functions by biotin1. 2015;93:1091–6. 10.1139/cjpp-2014-0460.</cite> [<a href="https://doi.org/10.1139/cjpp-2014-0460" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26168302/" class="usa-link">PubMed</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Elahi A, Sabui S, Narasappa NN, Agrawal S, Lambrecht NW, Agrawal A, et al. Biotin deficiency induces Th1- and Th17-Mediated Proinflammatory responses in human CD4 + T lymphocytes via activation of the mTOR signaling pathway. J Immunol. 2018;200:2563–70.
</cite> [<a href="https://doi.org/10.4049/jimmunol.1701200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5893381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29531163/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Elahi%20A,%20Sabui%20S,%20Narasappa%20NN,%20Agrawal%20S,%20Lambrecht%20NW,%20Agrawal%20A,%20et%20al.%20Biotin%20deficiency%20induces%20Th1-%20and%20Th17-Mediated%20Proinflammatory%20responses%20in%20human%20CD4%E2%80%89+%E2%80%89T%20lymphocytes%20via%20activation%20of%20the%20mTOR%20signaling%20pathway.%20J%20Immunol.%202018;200:2563%E2%80%9370." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Madsen CT, Sylvestersen KB, Young C, Larsen SC, Poulsen JW, Andersen MA, et al. Biotin starvation causes mitochondrial protein hyperacetylation and partial rescue by the SIRT3-like deacetylase Hst4p. Nature Communications. 2015;6:1–12.</cite> [<a href="https://doi.org/10.1038/ncomms8726" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4510963/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26158509/" class="usa-link">PubMed</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Newbold CJ, Ramos-Morales E, Review. Ruminal Microbiome and microbial metabolome: effects of diet and ruminant host. Animal. 2020;14:s78–86.
</cite> [<a href="https://doi.org/10.1017/S1751731119003252" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32024572/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Newbold%20CJ,%20Ramos-Morales%20E,%20Review.%20Ruminal%20Microbiome%20and%20microbial%20metabolome:%20effects%20of%20diet%20and%20ruminant%20host.%20Animal.%202020;14:s78%E2%80%9386." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Walker ND, Newbold CJ, Wallace RJ. Nitrogen metabolism in the rumen. Nitrogen and phosphorus nutrition of cattle: reducing the environmental impact of cattle operations. 2005:71–115.</cite>
</li>
<li id="CR73">
<span class="label">73.</span><cite>Tseten T, Sanjorjo RA, Kwon M, Kim SW. Strategies to mitigate enteric methane emissions from ruminant animals. J Microbiol Biotechnol. 2022;32:269.
</cite> [<a href="https://doi.org/10.4014/jmb.2202.02019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9628856/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35283433/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tseten%20T,%20Sanjorjo%20RA,%20Kwon%20M,%20Kim%20SW.%20Strategies%20to%20mitigate%20enteric%20methane%20emissions%20from%20ruminant%20animals.%20J%20Microbiol%20Biotechnol.%202022;32:269." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Manzanilla-Pech CIV, Stephansen RB, Difford GF, Løvendahl P, Lassen J. Selecting for feed efficient cows will help to reduce methane gas emissions. Front Genet. 2022;13:885932.
</cite> [<a href="https://doi.org/10.3389/fgene.2022.885932" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9178123/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35692829/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Manzanilla-Pech%20CIV,%20Stephansen%20RB,%20Difford%20GF,%20L%C3%B8vendahl%20P,%20Lassen%20J.%20Selecting%20for%20feed%20efficient%20cows%20will%20help%20to%20reduce%20methane%20gas%20emissions.%20Front%20Genet.%202022;13:885932." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Meo-Filho P, Hood J, Lee MRF, Fleming H, Meethal ME, Misselbrook T. Performance and enteric methane emissions from housed beef cattle fed silage produced on pastures with different forage profiles. Animal. 2023;17:100726.
</cite> [<a href="https://doi.org/10.1016/j.animal.2023.100726" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36921381/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meo-Filho%20P,%20Hood%20J,%20Lee%20MRF,%20Fleming%20H,%20Meethal%20ME,%20Misselbrook%20T.%20Performance%20and%20enteric%20methane%20emissions%20from%20housed%20beef%20cattle%20fed%20silage%20produced%20on%20pastures%20with%20different%20forage%20profiles.%20Animal.%202023;17:100726." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Simanungkalit G, Bhuiyan M, Bell R, Sweeting A, Morton CL, Cowley F, et al. The effects of antibiotic-free supplementation on the ruminal pH variability and methane emissions of beef cattle under the challenge of subacute ruminal acidosis (SARA). Res Vet Sci. 2023;160:30–8.
</cite> [<a href="https://doi.org/10.1016/j.rvsc.2023.05.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37263098/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Simanungkalit%20G,%20Bhuiyan%20M,%20Bell%20R,%20Sweeting%20A,%20Morton%20CL,%20Cowley%20F,%20et%20al.%20The%20effects%20of%20antibiotic-free%20supplementation%20on%20the%20ruminal%20pH%20variability%20and%20methane%20emissions%20of%20beef%20cattle%20under%20the%20challenge%20of%20subacute%20ruminal%20acidosis%20(SARA).%20Res%20Vet%20Sci.%202023;160:30%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Pittaluga AM, Yang F, Gaffney JR, Embree M, Relling AE. Effect of supplementation with ruminal probiotics on growth performance, carcass characteristics, plasma metabolites, methane emissions, and the associated rumen Microbiome changes in beef cattle. J Anim Sci. 2023;101. 10.1093/jas/skac308</cite> [<a href="https://doi.org/10.1093/jas/skac308" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9831096/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36592753/" class="usa-link">PubMed</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Carmichael MN, Dycus MM, Lourenco JM, Welch CB, Davis DB, Krause TR, et al. Ruminal microbiome differences in angus steers with differing feed efficiencies during the feedlot finishing phase. Microorganisms. 2024;12:536.</cite> [<a href="https://doi.org/10.3390/microorganisms12030536" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10972478/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38543587/" class="usa-link">PubMed</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>He W, Fang X, Lu X, Liu Y, Li G, Zhao Z, et al. Function identification of bovine ACSF3 gene and its association with lipid metabolism traits in beef cattle. Front Vet Sci. 2022;8:766765.
</cite> [<a href="https://doi.org/10.3389/fvets.2021.766765" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8770830/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35071379/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?He%20W,%20Fang%20X,%20Lu%20X,%20Liu%20Y,%20Li%20G,%20Zhao%20Z,%20et%20al.%20Function%20identification%20of%20bovine%20ACSF3%20gene%20and%20its%20association%20with%20lipid%20metabolism%20traits%20in%20beef%20cattle.%20Front%20Vet%20Sci.%202022;8:766765." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Scollan N, Hocquette JF, Nuernberg K, Dannenberger D, Richardson I, Moloney A. Innovations in beef production systems that enhance the nutritional and health value of beef lipids and their relationship with meat quality. Meat Sci. 2006;74:17–33.
</cite> [<a href="https://doi.org/10.1016/j.meatsci.2006.05.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22062713/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Scollan%20N,%20Hocquette%20JF,%20Nuernberg%20K,%20Dannenberger%20D,%20Richardson%20I,%20Moloney%20A.%20Innovations%20in%20beef%20production%20systems%20that%20enhance%20the%20nutritional%20and%20health%20value%20of%20beef%20lipids%20and%20their%20relationship%20with%20meat%20quality.%20Meat%20Sci.%202006;74:17%E2%80%9333." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Nogoy KMC, Sun B, Shin S, Lee Y, Li XZ, Choi SH, et al. Fatty acid composition of Grain- and Grass-Fed beef and their nutritional value and health implication. Food Sci Anim Resour. 2022;42:18.
</cite> [<a href="https://doi.org/10.5851/kosfa.2021.e73" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8728510/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35028571/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nogoy%20KMC,%20Sun%20B,%20Shin%20S,%20Lee%20Y,%20Li%20XZ,%20Choi%20SH,%20et%20al.%20Fatty%20acid%20composition%20of%20Grain-%20and%20Grass-Fed%20beef%20and%20their%20nutritional%20value%20and%20health%20implication.%20Food%20Sci%20Anim%20Resour.%202022;42:18." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Simopoulos AP. Evolutionary aspects of diet, the omega-6/omega-3 ratio and genetic variation: nutritional implications for chronic diseases. Biomed Pharmacother. 2006;60:502–7.
</cite> [<a href="https://doi.org/10.1016/j.biopha.2006.07.080" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17045449/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Simopoulos%20AP.%20Evolutionary%20aspects%20of%20diet,%20the%20omega-6/omega-3%20ratio%20and%20genetic%20variation:%20nutritional%20implications%20for%20chronic%20diseases.%20Biomed%20Pharmacother.%202006;60:502%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Cánovas A, Pena RN, Gallardo D, Ramírez O, Amills M, Quintanilla R. Segregation of regulatory polymorphisms with effects on the gluteus medius transcriptome in a purebred pig population. PLoS ONE. 2012;7:e35583.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0035583" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3335821/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22545120/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?C%C3%A1novas%20A,%20Pena%20RN,%20Gallardo%20D,%20Ram%C3%ADrez%20O,%20Amills%20M,%20Quintanilla%20R.%20Segregation%20of%20regulatory%20polymorphisms%20with%20effects%20on%20the%20gluteus%20medius%20transcriptome%20in%20a%20purebred%20pig%20population.%20PLoS%20ONE.%202012;7:e35583." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Cánovas A, Estany J, Tor M, Pena RN, Doran O. Acetyl-CoA carboxylase and stearoyl-CoA desaturase protein expression in subcutaneous adipose tissue is reduced in pigs selected for decreased backfat thickness at constant intramuscular fat content. J Anim Sci. 2009;87:3905–14.
</cite> [<a href="https://doi.org/10.2527/jas.2009-2091" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19684259/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?C%C3%A1novas%20A,%20Estany%20J,%20Tor%20M,%20Pena%20RN,%20Doran%20O.%20Acetyl-CoA%20carboxylase%20and%20stearoyl-CoA%20desaturase%20protein%20expression%20in%20subcutaneous%20adipose%20tissue%20is%20reduced%20in%20pigs%20selected%20for%20decreased%20backfat%20thickness%20at%20constant%20intramuscular%20fat%20content.%20J%20Anim%20Sci.%202009;87:3905%E2%80%9314." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Cánovas A, Quintanilla R, Amills M, Pena RN. Muscle transcriptomic profiles in pigs with divergent phenotypes for fatness traits. BMC Genomics. 2010;11:1–15.
</cite> [<a href="https://doi.org/10.1186/1471-2164-11-372" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2894043/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20540717/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?C%C3%A1novas%20A,%20Quintanilla%20R,%20Amills%20M,%20Pena%20RN.%20Muscle%20transcriptomic%20profiles%20in%20pigs%20with%20divergent%20phenotypes%20for%20fatness%20traits.%20BMC%20Genomics.%202010;11:1%E2%80%9315." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>López-Domínguez JA, Cánovas Á, Medrano JF, Islas-Trejo A, Kim K, Taylor SL, et al. Omega-3 fatty acids partially revert the metabolic gene expression profile induced by long-term calorie restriction. Exp Gerontol. 2016;77:29–37.
</cite> [<a href="https://doi.org/10.1016/j.exger.2016.02.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26875793/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?L%C3%B3pez-Dom%C3%ADnguez%20JA,%20C%C3%A1novas%20%C3%81,%20Medrano%20JF,%20Islas-Trejo%20A,%20Kim%20K,%20Taylor%20SL,%20et%20al.%20Omega-3%20fatty%20acids%20partially%20revert%20the%20metabolic%20gene%20expression%20profile%20induced%20by%20long-term%20calorie%20restriction.%20Exp%20Gerontol.%202016;77:29%E2%80%9337." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Erhuma A, Salter AM, Sculley DV, Langley-Evans SC, Bennett AJ. Prenatal exposure to a low-protein diet programs disordered regulation of lipid metabolism in the aging rat. Am J Physiol Endocrinol Metab. 2007;292:1702–14.</cite> [<a href="https://doi.org/10.1152/ajpendo.00605.2006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1890310/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17299084/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Erhuma%20A,%20Salter%20AM,%20Sculley%20DV,%20Langley-Evans%20SC,%20Bennett%20AJ.%20Prenatal%20exposure%20to%20a%20low-protein%20diet%20programs%20disordered%20regulation%20of%20lipid%20metabolism%20in%20the%20aging%20rat.%20Am%20J%20Physiol%20Endocrinol%20Metab.%202007;292:1702%E2%80%9314." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Wallace JM, Milne JS, Aitken BW, Aitken RP, Adam CL. Ovine prenatal growth-restriction and sex influence fetal adipose tissue phenotype and impact postnatal lipid metabolism and adiposity in vivo from birth until adulthood. PLoS ONE. 2020;15:e0228732.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0228732" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7021317/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32059008/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wallace%20JM,%20Milne%20JS,%20Aitken%20BW,%20Aitken%20RP,%20Adam%20CL.%20Ovine%20prenatal%20growth-restriction%20and%20sex%20influence%20fetal%20adipose%20tissue%20phenotype%20and%20impact%20postnatal%20lipid%20metabolism%20and%20adiposity%20in%20vivo%20from%20birth%20until%20adulthood.%20PLoS%20ONE.%202020;15:e0228732." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Diniz WJS, Ward AK, McCarthy KL, Kassetas CJ, Baumgaertner F, Reynolds LP, et al. Periconceptual maternal nutrition affects fetal liver programming of energy- and lipid-related genes. Animals. 2023;13:600.
</cite> [<a href="https://doi.org/10.3390/ani13040600" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9951695/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36830387/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Diniz%20WJS,%20Ward%20AK,%20McCarthy%20KL,%20Kassetas%20CJ,%20Baumgaertner%20F,%20Reynolds%20LP,%20et%20al.%20Periconceptual%20maternal%20nutrition%20affects%20fetal%20liver%20programming%20of%20energy-%20and%20lipid-related%20genes.%20Animals.%202023;13:600." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Polizel GHG, Fantinato-Neto P, Rangel RB, Grigoletto L, Bussiman F, de Cracco O. Evaluation of reproductive traits and the effect of nutrigenetics on bulls submitted to fetal programming. Livest Sci. 2021;247:104487.</cite> [<a href="https://scholar.google.com/scholar_lookup?Polizel%20GHG,%20Fantinato-Neto%20P,%20Rangel%20RB,%20Grigoletto%20L,%20Bussiman%20F,%20de%20Cracco%20O.%20Evaluation%20of%20reproductive%20traits%20and%20the%20effect%20of%20nutrigenetics%20on%20bulls%20submitted%20to%20fetal%20programming.%20Livest%20Sci.%202021;247:104487." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/11983759/bin/12864_2025_11545_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (30.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/11983759/bin/12864_2025_11545_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2</a><sup> (28.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/11983759/bin/12864_2025_11545_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3</a><sup> (30.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/11983759/bin/12864_2025_11545_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 4</a><sup> (20.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/11983759/bin/12864_2025_11545_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 5</a><sup> (22.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/11983759/bin/12864_2025_11545_MOESM6_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 6</a><sup> (166.9KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The metabolome dataset analysed during the current study are available in the supplementary material (Additional file <a href="#MOESM1" class="usa-link">1</a>), as well as the microbiome datasets (fecal microbiome [Additional file <a href="#MOESM2" class="usa-link">2</a>]; rumen fluid microbiome [Additional file <a href="#MOESM3" class="usa-link">3</a>]). The corresponding taxonomy annotation can be found in Additional file <a href="#MOESM4" class="usa-link">4</a> (fecal microbiome) and Additional file <a href="#MOESM5" class="usa-link">5</a> (rumen fluid microbiome). The hierarchical clustering of WGCNA modules for the -OMIC datasets is shown in Additional file <a href="#MOESM6" class="usa-link">6</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Genomics are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12864-025-11545-6"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12864_2025_Article_11545.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.8 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/11983759/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/11983759/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11983759%2F%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11983759/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC11983759/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC11983759/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40211121/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC11983759/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40211121/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC11983759/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/11983759/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="AbpxUn32qP7CFe1iuqXzXqqjb4eBTd6lLNaqJALemcthy503E89LV16UM5xKxyN4">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-B7-tgZZA.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-BLbQGH8u.js"></script>
    
    

    </body>
</html>
