
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/base_style-CPPDhfna.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-BepKMsUl.js"></script>

  <link rel="stylesheet" href="/static/assets/article_style-DUfI1jZp.css" />
<link rel="stylesheet" href="/static/assets/style-BOrsu84o.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-D3wJHomh.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3EAuthor Manuscript%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Comprehensive structural glycomic characterization of the glycocalyx of cells and tissues - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="CFBE248A930B0DF300248A0028693792.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="nihpa">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="manuscript">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11790333/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature protocols">
<meta name="citation_title" content="Comprehensive structural glycomic characterization of the glycocalyx of cells and tissues">
<meta name="citation_author" content="Qiongyu Li">
<meta name="citation_author_institution" content="Department of Chemistry, University of California, Davis, California, USA.">
<meta name="citation_author" content="Yixuan Xie">
<meta name="citation_author_institution" content="Department of Chemistry, University of California, Davis, California, USA.">
<meta name="citation_author" content="Maurice Wong">
<meta name="citation_author_institution" content="Department of Chemistry, University of California, Davis, California, USA.">
<meta name="citation_author" content="Mariana Barboza">
<meta name="citation_author_institution" content="Department of Anatomy, Physiology and Cell Biology, School of Veterinary Medicine, University of California, Davis, California, USA.">
<meta name="citation_author" content="Carlito B Lebrilla">
<meta name="citation_author_institution" content="Department of Chemistry, University of California, Davis, California, USA.">
<meta name="citation_author_institution" content="Department of Biochemistry, University of California, Davis, California, USA.">
<meta name="citation_publication_date" content="2020 Jul 17">
<meta name="citation_volume" content="15">
<meta name="citation_issue" content="8">
<meta name="citation_firstpage" content="2668">
<meta name="citation_doi" content="10.1038/s41596-020-0350-4">
<meta name="citation_pmid" content="32681150">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11790333/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11790333/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11790333/pdf/nihms-2039916.pdf">
<meta name="description" content="The glycocalyx is comprised of glycosylated proteins and lipids and forms the outermost layer of cells. It is involved in a paramount of fundamental inter- and intracellular processes, including non-self and self-cell recognition, cell signaling, ...">
<meta name="og:title" content="Comprehensive structural glycomic characterization of the glycocalyx of cells and tissues">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The glycocalyx is comprised of glycosylated proteins and lipids and forms the outermost layer of cells. It is involved in a paramount of fundamental inter- and intracellular processes, including non-self and self-cell recognition, cell signaling, ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11790333/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://pmc.ncbi.nlm.nih.gov/search/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="11790333">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41596-020-0350-4"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/nihms-2039916.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11790333%2F%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/11790333/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/11790333/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11790333/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-nihpa.png" alt="NIHPA Author Manuscripts logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to NIHPA Author Manuscripts" title="Link to NIHPA Author Manuscripts" shape="default" href="https://www.hhs.gov/open/public-access-guiding-principles/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Protoc</button></div>. Author manuscript; available in PMC: 2025 Feb 3.</div>
<div>
<em>Published in final edited form as: </em>Nat Protoc. 2020 Jul 17;15(8):2668–2704. doi: <a href="https://doi.org/10.1038/s41596-020-0350-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41596-020-0350-4</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://pmc.ncbi.nlm.nih.gov/search/?term=%22Nat%20Protoc%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Protoc%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Protoc%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Protoc%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Comprehensive structural glycomic characterization of the glycocalyx of cells and tissues</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Qiongyu Li</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Qiongyu Li</span></h3>
<div class="p">
<sup>1</sup>Department of Chemistry, University of California, Davis, California, USA.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qiongyu Li</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xie%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Yixuan Xie</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Yixuan Xie</span></h3>
<div class="p">
<sup>1</sup>Department of Chemistry, University of California, Davis, California, USA.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xie%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yixuan Xie</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wong%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Maurice Wong</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Maurice Wong</span></h3>
<div class="p">
<sup>1</sup>Department of Chemistry, University of California, Davis, California, USA.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wong%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Maurice Wong</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Barboza%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Mariana Barboza</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Mariana Barboza</span></h3>
<div class="p">
<sup>2</sup>Department of Anatomy, Physiology and Cell Biology, School of Veterinary Medicine, University of California, Davis, California, USA.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Barboza%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mariana Barboza</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lebrilla%20CB%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Carlito B Lebrilla</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Carlito B Lebrilla</span></h3>
<div class="p">
<sup>1</sup>Department of Chemistry, University of California, Davis, California, USA.</div>
<div class="p">
<sup>3</sup>Department of Biochemistry, University of California, Davis, California, USA.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lebrilla%20CB%22%5BAuthor%5D" class="usa-link"><span class="name western">Carlito B Lebrilla</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="A1">
<sup>1</sup>Department of Chemistry, University of California, Davis, California, USA.</div>
<div id="A2">
<sup>2</sup>Department of Anatomy, Physiology and Cell Biology, School of Veterinary Medicine, University of California, Davis, California, USA.</div>
<div id="A3">
<sup>3</sup>Department of Biochemistry, University of California, Davis, California, USA.</div>
<div class="author-notes p">
<div class="fn" id="FN1">
<p id="P1">Author contributions</p>
<p id="P2">Q.L., Y.X., M.W., M.B., and C.B.L. contributed to the development of this protocol and wrote and edited the manuscript.</p>
</div>
<div class="fn" id="CR1">
<sup>✉</sup><p class="display-inline">Correspondence and requests for materials should be addressed to C.B.L. <span>cblebrilla@ucdavis.edu</span></p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Issue date 2020 Aug.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div>PMCID: PMC11790333  NIHMSID: NIHMS2039916  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/32681150/" class="usa-link">32681150</a>
</div>
<div class="ra xbox p" role="complementary" aria-label="Related or updated information about this article"><div>The publisher's version of this article is available at <a href="https://doi.org/10.1038/s41596-020-0350-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Nat Protoc</a>
</div></div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="ABS1"><h2>Abstract</h2>
<p id="P3">The glycocalyx is comprised of glycosylated proteins and lipids and forms the outermost layer of cells. It is involved in a paramount of fundamental inter- and intracellular processes, including non-self and self-cell recognition, cell signaling, cellular structure maintenance, and immune protection. Characterization of the glycocalyx is thus essential for understanding cell physiology and elucidating its role in promoting health and disease. This protocol describes how to comprehensively characterize the glycocalyx N-glycans and O-glycans of glycoproteins as well as intact glycolipids in parallel, using the same enriched membrane fraction. Profiling of the glycans and the glycolipids is performed using nanoflow liquid chromatography-mass spectrometry (nanoLC-MS). Sample preparation, quantitative LC-MS/MS analysis, and data processing methods are provided. Additionally, glycoproteomic analysis is described that yields the site-specific glycosylation of membrane proteins. To reduced the amount of sample needed, N-glycan, O-glycan, and glycolipid analyses are performed on the same enriched fraction, while glycoproteomic analysis is performed on a separate enriched fraction. The sample preparation process takes 2–3 days, while the time spent on instrumental and data analyses could vary from 1 to 5 days for different sample sizes. This workflow is applicable to both cell and tissue samples. Systematic changes in the glycocalyx associated with specific glycoforms and glycoconjugates can be monitored with quantitation using this protocol. The ability to quantitate individual glycoforms and glycoconjugates will find utility in a broad range of fundamental and applied clinical studies, including glycan-based biomarker discovery and therapeutics.</p></section><section id="S1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="P5">A layer of glycan chains known as the glycocalyx covers the cell surface of every living cell. The glycans are conjugated to proteins or lipids, and are organized in complicated and highly interactive networks <sup><a href="#R1" class="usa-link" aria-describedby="R1">1</a>–<a href="#R3" class="usa-link" aria-describedby="R3">3</a></sup>. The glycocalyx is a critical mediator of cell-cell communication events that occur in a variety of biological and physiological processes<sup><a href="#R4" class="usa-link" aria-describedby="R4">4</a>–<a href="#R6" class="usa-link" aria-describedby="R6">6</a></sup>. Its proximity necessarily involves it in a host of common diseases including infection<sup><a href="#R7" class="usa-link" aria-describedby="R7">7</a>–<a href="#R10" class="usa-link" aria-describedby="R10">10</a></sup>, cancer<sup><a href="#R11" class="usa-link" aria-describedby="R11">11</a>–<a href="#R14" class="usa-link" aria-describedby="R14">14</a></sup>, and autoimmune diseases<sup><a href="#R15" class="usa-link" aria-describedby="R15">15</a>–<a href="#R17" class="usa-link" aria-describedby="R17">17</a></sup>. Despite the importance of glycans and the glycocalyx, their structural analysis has trailed that of other critical biopolymers such as DNA, RNA, and proteins. The inherent complexity and significant heterogeneity of glycans have made their structural elucidation uniquely challenging. The development of plasma membrane enrichment techniques, coupled to highly efficient chromatographic separation methods and ultrasensitive mass spectrometry instruments now allow the analyses of glycans and glycoconjugates in large complicated mixtures at a previously unattainable range of femtomolar to attomolar amounts. Over the last decade, great progress has been made toward the characterization of individual glycocalyx components namely N-glycans and O-glycans on glycoproteins, glycolipids, and glycosaminoglycans (GAGs). These advances have led to the identification of critical glycan structures in normal and pathological conditions<sup><a href="#R18" class="usa-link" aria-describedby="R18">18</a>–<a href="#R22" class="usa-link" aria-describedby="R22">22</a></sup>. Despite many extensive efforts, individual approaches to characterize glycocalyx glycans have yielded fragmented or incomplete pictures of the glycocalyx composition in most cells and tissues studied thus far. To circumvent this limitation, we developed an integrated multiglycomic method based on mass spectrometry (MS) workflows that enables deep and extensive characterization of the glycocalyx, including the identification of the N-glycome<sup><a href="#R23" class="usa-link" aria-describedby="R23">23</a></sup>, O-glycome<sup><a href="#R24" class="usa-link" aria-describedby="R24">24</a></sup> and site-specific occupancy of cell surface glycoproteins<sup><a href="#R24" class="usa-link" aria-describedby="R24">24</a>,<a href="#R25" class="usa-link" aria-describedby="R25">25</a></sup>, as well as intact cell surface glycolipids<sup><a href="#R26" class="usa-link" aria-describedby="R26">26</a></sup>.</p>
<section id="S2"><h3 class="pmc_sec_title">Development of the protocol</h3>
<p id="P6">Specific chromatographic separation and sensitive mass spectrometry have been a key factor in the development of this protocol. Our laboratory has been involved in developing nanoflow liquid chromatography mass spectrometry-based analytical methods for the analysis of naturally occurring free glycans in body fluids such as human milk oligosaccharides (HMO)<sup><a href="#R27" class="usa-link" aria-describedby="R27">27</a>,<a href="#R28" class="usa-link" aria-describedby="R28">28</a></sup>. We have further developed methods to analyze glycans released from glycoproteins<sup><a href="#R29" class="usa-link" aria-describedby="R29">29</a>,<a href="#R30" class="usa-link" aria-describedby="R30">30</a></sup> and glycans conjugated to corresponding glycopeptides<sup><a href="#R24" class="usa-link" aria-describedby="R24">24</a>,<a href="#R31" class="usa-link" aria-describedby="R31">31</a></sup> and glycolipids<sup><a href="#R26" class="usa-link" aria-describedby="R26">26</a></sup>. Since 2010, we have focused these efforts on the development of methodologies for the characterization of the cell surface glycocalyx in cell lines<sup><a href="#R23" class="usa-link" aria-describedby="R23">23</a>,<a href="#R24" class="usa-link" aria-describedby="R24">24</a>,<a href="#R26" class="usa-link" aria-describedby="R26">26</a>,<a href="#R32" class="usa-link" aria-describedby="R32">32</a>,<a href="#R33" class="usa-link" aria-describedby="R33">33</a></sup> and tissues<sup><a href="#R34" class="usa-link" aria-describedby="R34">34</a></sup>. These methods have generally involved glycan profiling with nanoflow liquid chromatography-quadrupole time-of-flight mass spectrometry (nanoLC-QTOF MS) and nanoLC-TOF MS. Incorporated later in the workflow is the use of nanoLC-orbitrap MS/MS for glycoproteomic analysis with site-specific characterization.</p>
<p id="P7">In the development of these methods, every effort was made to minimize the amounts of samples, whether it be cell lines or tissues. For this reason, nanoflow liquid chromatography is used throughout the analysis because of its high sensitivity. Additionally, efforts were made to perform several analyses on the same sample. Thus, from the same starting plasma membrane material all the N-glycans, O-glycans, and glycolipids were characterized by first releasing the N-glycans, then extracting the glycolipids, and finally releasing the and characterizing the O-glycans. A separate portion of the plasma membrane was analyzed for protein and glycoprotein analysis, because the glycan-depleted fractions could not be used for glycoproteomic analysis.</p></section><section id="S3"><h3 class="pmc_sec_title">Advantages and comparison to alternative methods</h3>
<p id="P8">The most commonly used method for characterizing the glycocalyx employs lectins, which are proteins that recognize glycans through specific structural motifs. Lectins are convenient, rapid, and require no additional complicated or large instrumentation. Fluorescently-labeled lectins are readily available to characterize the cell surface glycocalyx using microscopy. However, this approach has severe limitations. Lectins provide information about specific structural motifs but are unable to determine complete glycan structures, glycan composition, or differentiate main classes of glycans. For example, <em>sambucus nigra</em> (SNA) lectin recognizes α(2,6)-linked sialic acid on galactose but it cannot distinguish whether the sialic acid is on N-glycan, O-glycan or glycolipid. Similarly, Concanavalin A (Con A) may detect a-mannose, which is presented in nearly all N-glycans, and therefore cannot distinguish between high mannose-, complex-, or hybrid-type glycans. Lectins also provide little quantitative information. For example, an SNA lectin may be used to determine changes in the presence of α(2,6)-linked sialic acid. However, it cannot quantify the types of glycan that are varied. Responses of lectins with different specificities also cannot be quantitively compared.</p>
<p id="P9">The glycomic analysis in this protocol is conducted on native glycans, i.e., without labeling. This method minimizes sample losses due to sample manipulation and incomplete reactions associated with labeling. Various labeling methods have been employed for glycomic analysis, including reactions with 1-phenyl-3-methyl-5-pyrazolone (PMP), 2-aminobenzamide (2-AB), and 2-aminobenzoic acid (2-AA)<sup><a href="#R35" class="usa-link" aria-describedby="R35">35</a></sup>. Although the derivatization improves the ionization efficiency of glycans, the complicated sample preparation and clean-up lead to potential losses, particularly those with relatively low abundances.</p>
<p id="P10">For the glycoproteomic analysis, glycopeptides are enriched with solid phase extraction hydrophilic interaction liquid chromatography (SPE HILIC) cartridges. The process yields 90% glycopeptides and some remaining nonglycosylated peptides. This enrichment method is based on hydrophilic interactions between the glycans and the stationary phase. The procedure is relatively simple and applicable to large sample sets for rapid throughput analysis. Other enrichment methods with different binding interactions can also be incorporated into this protocol. For example, lectin chromatography may be used to enrich N-glycopeptides, through the binding of specific structural glycan features<sup><a href="#R36" class="usa-link" aria-describedby="R36">36</a></sup>. However, the buffers for enrichment that use high concentrations of NaCl and Tris-HCl are not compatible with MS analysis, and further clean-up steps may be required before MS<sup><a href="#R37" class="usa-link" aria-describedby="R37">37</a></sup>. The enrichment of glycopeptides through click chemistry is an effective approach for glycoproteomic analysis. The method involves the metabolic labeling of glycoproteins with unnatural monosaccharides such as N-azidoacetylmannosamine (ManNAz) and N-azidoacetylgalactosamine (GalNAz) followed by enrichment through copper-free click chemistry<sup><a href="#R38" class="usa-link" aria-describedby="R38">38</a></sup>. This method has been applied to a host of biological systems such as membrane glycoproteins of primary neurons<sup><a href="#R39" class="usa-link" aria-describedby="R39">39</a></sup> and pancreatic cancer cell lines<sup><a href="#R40" class="usa-link" aria-describedby="R40">40</a></sup>. However, the utility may be complicated by the reaction efficiencies of the click chemistry and the incorporation of monosaccharides analogues, which varies among cell lines.</p>
<p id="P11">Liquid chromatography (LC) is used for compound separation and is currently the most effective technique for separation. In this protocol, different stationary phases for reverse phase including porous graphite carbon (PGC) and C18 are used for the separation of glycans, glycolipids and glycopeptides. Normal phase such as HILIC can also be applied<sup><a href="#R41" class="usa-link" aria-describedby="R41">41</a>,<a href="#R42" class="usa-link" aria-describedby="R42">42</a></sup>. The LC separation is applicable to both native and labeled glycans. Standard, ultrahigh-pressure, micro, and nanoLC can all be employed with their respective flow rates, pressure, detection sensitivities, and separation efficiencies. In this protocol, the application of nanoLC enables the analysis with high sensitivity. Other separation techniques can also be employed for the analysis of glycoconjugates. For example, capillary electrophoresis (CE) has been shown to have high efficiency for glycan and glycoprotein separation. Previously, the coupling of CE with MS had been challenging due to low flow rates of CE; however, this issue has been addressed more recently with techniques such as a sheath-flow interface<sup><a href="#R43" class="usa-link" aria-describedby="R43">43</a></sup>. Commercial CE systems are available and have been applied to native and digested glycoproteins<sup><a href="#R44" class="usa-link" aria-describedby="R44">44</a></sup>. However, the application of these systems toward glycoconjugates from complicated biological samples is still limited<sup><a href="#R45" class="usa-link" aria-describedby="R45">45</a></sup>. Ion mobility (IM) has held great promise for rapid separation of glycans and glycoconjugates. Compounds are separated based on their collision cross sections and charges thereby enabling the separation of glycan isomers<sup><a href="#R45" class="usa-link" aria-describedby="R45">45</a>,<a href="#R46" class="usa-link" aria-describedby="R46">46</a></sup>. IM-MS of glycans released from different biological samples such as human serum<sup><a href="#R47" class="usa-link" aria-describedby="R47">47</a></sup> and the parotid gland<sup><a href="#R48" class="usa-link" aria-describedby="R48">48</a></sup> have been reported. However, the separation resolution of glycans with IM is still severely limited. The employment of IM-MS to glycopeptide analysis is even more complicated because of the high numbers of isomers, gas-phase conformers due to the large similarities in structures<sup><a href="#R45" class="usa-link" aria-describedby="R45">45</a></sup>.</p>
<p id="P12">An alternative to the LC-MS method is matrix-assisted laser desorption/ionization (MALDI). MALDI is more tolerant of high concentrations of salts and contaminates.<sup><a href="#R49" class="usa-link" aria-describedby="R49">49</a></sup> It has been coupled to mass detectors with high resolution and mass accuracy such as Fourier-transform ion cyclotron resonance (FTICR) and time-of-flight (TOF). MALDI MS is significantly more rapid and can provide comprehensive glycomic profiles of released glycans. However, MALDI MS is not as sensitive as nanoLC-MS. Also, MALDI does not provide separation of isomers, although it can be coupled to LC if fractions are collected and individually probed. Without pre-separation, the glycan profile obtained from MALDI MS is essentially a mass profile and is not as extensive as those obtained from LC-MS.</p>
<p id="P13">The most commonly used MS fragmentation technique used for glycan and glycolipid analyses is collision induced dissociation (CID). The dissociation of glycans using infrared multiphoton dissociation (IRMPD) yields similar fragments to those from CID<sup><a href="#R50" class="usa-link" aria-describedby="R50">50</a></sup>. It has been extensively used but is not readily available<sup><a href="#R51" class="usa-link" aria-describedby="R51">51</a></sup>. For glycopeptides, stepped high-energy collisional dissociation (HCD), a marketing term for a higher energy form of CID, is used. The fragmentation of glycans with CID cleaves the glycosidic bonds. For glycopeptides, CID generates mainly fragments for glycans, which have more labile bonds, but also peptide bond cleavages with lower abundances. With the stepped HCD, both glycans and peptides are fragmented with higher efficiency<sup><a href="#R52" class="usa-link" aria-describedby="R52">52</a></sup>. Another fragmentation technique applied to glycopeptides is electrotransfer/higher-energy collision dissociation (EThcD), which yields abundant fragments corresponding to both glycans and peptides<sup><a href="#R53" class="usa-link" aria-describedby="R53">53</a></sup>. However, EThcD requires longer duty cycles, which may lead to lower identification rates for glycopeptides.</p></section><section id="S4"><h3 class="pmc_sec_title">Experimental design</h3>
<section id="S5"><h4 class="pmc_sec_title">Overview of the protocol.</h4>
<p id="P14">The workflow for the analysis of the glycocalyx of cells consists of the steps outlined in <a href="#F1" class="usa-link">Figure 1</a> and <a href="#F2" class="usa-link">Figure 2</a>. Cells are first harvested, and the membrane disrupted. Enrichment of the plasma membrane is performed by a series of centrifugation and ultracentrifugation steps (<a href="#F1" class="usa-link">Figure 1</a>). The membrane pellet is divided into two fractions to perform the glycomic analysis and the glycoproteomic analysis. One fraction of the membrane is used for the release and extraction of glycans (N- and O-) and glycolipids, followed by their separate LC-MS analyses, while a second fraction is used for proteomic and glycoproteomic analyses. Glycomic profiles of N- and O-glycans are performed on released compounds using nanoLC-TOF and QTOF MS employing porous graphitized carbon chromatographic chip. Glycolipid profiling is performed on intact compounds using nanoLC-QTOF MS with a C18 chip column. Agilent Corp. (Santa Clara, Ca) MassHunter software is employed for data processing and in-house libraries are used for the identification of N-glycans, O-glycans and glycolipids. Peptide and glycopeptide analyses are performed using nanoLC-Orbitrap MS with C18 chromatographic columns. Identifications of peptides and glycopeptides are performed using the Protein Metrics package (Cupertino, CA), which includes Byonic for glycoproteomic analysis and Byologic for glycopeptide quantitation. While the description is primarily for cultured cells, mouse tissue samples such as brain, liver, spleen and lung can also be processed and analyzed in this manner<sup><a href="#R34" class="usa-link" aria-describedby="R34">34</a></sup>. Representative data acquired from N-, O-glycomic, and glycolipidomic analyses of mouse brain samples are illustrated in <a href="#F3" class="usa-link">Figure 3</a>. Additionally, the methods can be adapted in parts to include, for example, only N- or only O-glycan profiling.</p>
<figure class="fig xbox font-sm" id="F1"><h5 class="obj_head">Figure 1. Summary of the workflow for cell sample collection and membrane extraction.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2ce2/11790333/65a1c46b6832/nihms-2039916-f0001.jpg" loading="lazy" height="1050" width="718" alt="Figure 1."></p>
<div class="p text-right font-secondary"><a href="figure/F1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P385">More than ten million cells are harvested, followed by buffer exchange and homogenization via sonication at 4 °C. Nucleus and cell debris are firstly separated via centrifugation. Then the crude plasma membrane fraction is pelletized via ultracentrifugation, followed by aqueous sodium carbonate and water washings.</p></figcaption></figure><figure class="fig xbox font-sm" id="F2"><h5 class="obj_head">Figure 2. Summary of the workflow for comprehensive LC-MS/MS analysis of the cell membrane glycocalyx.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2ce2/11790333/491a89760e01/nihms-2039916-f0002.jpg" loading="lazy" height="592" width="1050" alt="Figure 2."></p>
<div class="p text-right font-secondary"><a href="figure/F2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P386">The steps of the experiment are outlined, including the time requirements for the corresponding steps. The route for N-glycomic analysis is labeled with red, the route for glycolipidomic analysis is with green, the route for O-glycomic analysis is pink, and the route of site-specific glycoproteomic analysis is orange.</p></figcaption></figure><figure class="fig xbox font-sm" id="F3"><h5 class="obj_head">Figure 3. Examples of tissues with cell surface glycocalyxes analyzed by LC-MS/MS.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2ce2/11790333/13a72108f25a/nihms-2039916-f0003.jpg" loading="lazy" height="928" width="1050" alt="Figure 3."></p>
<div class="p text-right font-secondary"><a href="figure/F3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P387"><strong>a</strong>. Brain N-glycome was obtained by PGC-chip-QTOF MS/MS. More than 600 N-glycans were observed with individual ion counts around 3 ×10<sup>5</sup>. <strong>b</strong>. O-Glycan profile obtained by nanoLC-QTOF MS/MS using a PGC LC chip. More than 70 O-glycans were observed with ion abundances at 5 × 10<sup>9</sup> total ion. <strong>c.</strong> Glycolipid profile obtained by nanoLC-QTOF MS/MS using a C18 chip. More than 100 glycolipids were observed with abundances of 2 × 10<sup>8</sup>. The peaks were color coded to show the glycan subtypes.</p></figcaption></figure><p id="P15">The protocol enables the quantitation of glycans through ion counts obtained directly from the nanoLC-MS system as peak areas (or volumes). While absolute abundances are not achievable due mainly to the lack of standards, ion counts allow measurements of absolute changes and relative abundances. These values can be further normalized to the amount of starting material (either cell counts or protein amounts). With the normalized abundances, changes in glycocalyx compositions under different conditions can be monitored and compared accurately. To obtain high reproducibility, serially diluted quality control samples were used to determine the variations in the quantitation of compounds. The linearity of selected compounds were also determined in this way. When standards do become available in the future, they can be readily incorporated either through standard addition or directly by using stable isotope labeling.</p></section><section id="S6"><h4 class="pmc_sec_title">Sample preparation and cell membrane extraction (Steps i)-11).</h4>
<p id="P16">Cell lines should be cultured following the protocols provided by the American Type Culture Collection (ATCC) or from literature advice. We have successfully used human cell lines including normal prostate immortalized cell line PNT2, colorectal adenocarcinoma cell line Caco-2, lung carcinoma cell line A549 and pluripotent cell line NTERA-2. The morphology of cells should be checked regularly, at least once every other day. The method can also be applied to other cell lines. Each cell line will have special care guidance that should be noted. For example, some cell lines are sensitive to the amount of glucose in the media,such as HT-29<sup><a href="#R54" class="usa-link" aria-describedby="R54">54</a></sup> cells, which is used as illustration in this protocol, and differentiates if there is a decrease in the glucose concentration in the media. To prevent differentiation, we recommend changing media for HT-29 cells every day and splitting cells at near 80% confluency.</p>
<p id="P17">For a comprehensive analysis, a total of at least ten million cells are required, and these need to be separated into two (Fraction I and II). Fraction I, for glycomic (N- and O-glycans) and glycolipid analyses require fewer cells (approximately 1 × 10<sup>6</sup> cells). Fraction II, for site-specific glycoproteomic analysis requires the larger fraction of cells (at least 9 × 10<sup>6</sup> cells). Immediately after collecting cells, it is important to undertake buffer exchange and homogenization at 4 °C to prevent the untargeted modifications of proteins and to minimize the activity of endogenous proteases<sup><a href="#R55" class="usa-link" aria-describedby="R55">55</a></sup>. Analysis of flash-frozen tissue samples can be performed following the same procedure. Upon collection, tissues can be immediately processed or flash-frozen at −80 °C until further processing. For direct analysis, rinse tissues with PBS, add the homogenization buffer, keep samples at 4 °C, and continue with the homogenization step. The ultrasonic homogenization protocol we describe is optimized for mammalian cell lines<sup><a href="#R32" class="usa-link" aria-describedby="R32">32</a></sup> and soft-tissues. For more fiborous or harder tissues such as cartilage, more intense homogenization conditions should be employed<sup><a href="#R34" class="usa-link" aria-describedby="R34">34</a>,<a href="#R56" class="usa-link" aria-describedby="R56">56</a></sup>.</p>
<p id="P18">The membrane extraction method we describe was adapted from a previously described protocol<sup><a href="#R32" class="usa-link" aria-describedby="R32">32</a></sup>. The cell membrane is extracted after harvesting and lysing cells and tissues. The nucleus and mitochondria are removed by centrifugation at low speed, followed by pelleting of the cell membrane fraction using ultracentrifugation. By introducing sodium carbonate solution, membrane-associated proteins that are noncovalently bound can be separated and dissolved away while the membrane fraction is pelleted to obtain a cleaner membrane fraction <sup><a href="#R57" class="usa-link" aria-describedby="R57">57</a>,<a href="#R58" class="usa-link" aria-describedby="R58">58</a></sup>. The extraction of the cell membrane can be achieved with minimal sample preparation and is compatible with mass spectrometry analysis. The resulting membrane pellet is then ready for further processing and analysis. <em>Fraction I</em> is subjected to comprehensive glycomic and glycolipidomic analyses. Each sample derived from this fraction has sufficient amounts for injections (3–4 times). <em>Fraction II</em> is subjected to glycoproteomic analysis, with the final sample sufficient for 1–2 injections. The workflow for cell sample collection and membrane fractionation is illustrated in <a href="#F1" class="usa-link">Figure 1</a>.</p></section><section id="S7"><h4 class="pmc_sec_title">N-Glycan analysis: Release, purification, and mass spectrometry (MS) measurements (Steps 12–33).</h4>
<p id="P19">After cell membrane extraction, N-glycans are released from the glycoproteins using the enzyme peptide-N-glycosidase F (PNGase F). Membrane proteins should be denatured by heating for up to 2 minutes before N-glycan release, in order to achieve higher digestion efficiency. The complete release of N-glycans is required because the remaining N-glycans can interfere with O-glycan analysis. Microwaves can be used to efficiently and rapidly remove the N-glycans from the proteins<sup><a href="#R59" class="usa-link" aria-describedby="R59">59</a></sup>. To minimize the content of amine-containing N-glycans, incubation at 37 °C in a water bath should be conducted after the N-glycan release to complete the hydrolysis of the glycan-amine to the glycan-aldehyde. Alternatively, an amine-free buffer such as PBS or HEPES can be used to avoid amine groups<sup><a href="#R60" class="usa-link" aria-describedby="R60">60</a></sup>. Glycans containing α(1,3)-linked fucose (primarily from plant proteins) cannot be cleaved by PNGase F. For these compounds, the enzyme PNGase A is employed<sup><a href="#R61" class="usa-link" aria-describedby="R61">61</a></sup>. Upon digestion, released N-glycans are collected in supernatants after ultracentrifugation. Ethanol precipitation can also be used by adding 4 volumes of ethanol to the digestions solution and incubating at −20 °C overnight or −80 °C for 2 hours (h), followed by centrifugation to precipitate proteins completely. The pelletized membrane without N-glycans should be saved for subsequent O-glycomic and glycolipidomic analyses. The supernatant containing N-glycans is subjected to SPE clean-up with PGC in cartridges or 96-well plates for large-batch preparation.</p>
<p id="P20">Desalted native N-glycans are injected into a nanoLC-TOF-MS/MS equipped with a PGC nanochip for analysis directly. Direct analysis of native samples avoids loss from incomplete modification associated with all derivatization methods<sup><a href="#R62" class="usa-link" aria-describedby="R62">62</a></sup>. The use of nano-LC provides high sensitivity and large dynamic range, enabling the identification of low-abundance species. Normal-flow or micro-flow LC systems can also be employed for analysis of samples but require significantly higher amounts of material. Separation of N-glycans in PGC columns split the anomers for most but not all N-glycans. Thus, a single structure will yield two retention times. To avoid splitting of the anomeric species, the N-glycans can be reduced using the same method as O-glycans to remove anomers if desired (Steps 75–78). In addition, the PGC chip yields the separation of isomeric glycans and has been demonstrated in several previous publications<sup><a href="#R33" class="usa-link" aria-describedby="R33">33</a>,<a href="#R63" class="usa-link" aria-describedby="R63">63</a>,<a href="#R64" class="usa-link" aria-describedby="R64">64</a></sup>. Furthermore, for the instrument quality control (QC), the pooled released N-glycans from several cell lines under the normal condition (Caco-2, PNT2, and A549) can be used. We also suggest the use of a commercial pooled serum and RNaseB N-glycan sample to perform quality control (QC) of the method and associated instrumentation if no N-glycans from cell lines are collected. A typical example of an extracted compound chromatogram (ECC) for N-glycans is shown in <a href="#F4" class="usa-link">Figure 4a</a>. The major peaks are annotated; however, over 500 compounds (including isomers) are observed. The separation of isomers is readily illustrated in the ECC. An ECC for QC N-glycans is shown in <a href="#SD6" class="usa-link">Supplementary Figure 1a</a>.</p>
<figure class="fig xbox font-sm" id="F4"><h5 class="obj_head">Figure 4. Examples of LC-MS profiles from the Caco-2 cell line.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2ce2/11790333/ea5aae874b2f/nihms-2039916-f0004.jpg" loading="lazy" height="923" width="1050" alt="Figure 4."></p>
<div class="p text-right font-secondary"><a href="figure/F4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P388"><strong>a</strong>. N-Glycan profile obtained by nanoLC-QTOF MS/MS using a PGC chip. More than 500 N-glycans were observed with ion abundances of 4 × 10<sup>9</sup>. <strong>b</strong>. O-Glycan profile obtained by nanoLC-QTOF MS/MS with a PGC chip. More than 50 O-glycans were observed with abundances of 5 × 10<sup>9</sup>. <strong>c.</strong> Glycolipid profile obtained by nanoLC-QTOF MS/MS with a C18 chip. More than 200 glycolipids were observed with abundances of 2 × 10<sup>8</sup> ion. The peaks are color coded to show the glycan subtypes. N-Glycan profile reprinted with permission from ref. <a href="#R23" class="usa-link" aria-describedby="R23">23</a>. Copyright 2017 by Oxford University Press. O-Glycan profile reprinted with permission from ref. <a href="#R24" class="usa-link" aria-describedby="R24">24</a>. Creative Commons licence (CC BY 3.0) (<a href="http://creativecommons.org/licenses/by/3.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/3.0/</a>). Glycolipid profile reprinted with permission from ref. <a href="#R26" class="usa-link" aria-describedby="R26">26</a>. Creative Commons licence (CC BY 4.0) (<a href="http://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>).</p></figcaption></figure></section><section id="S8"><h4 class="pmc_sec_title">Glycosphingolipid analysis: Extraction, purification, and MS measurements (Steps 44–56)).</h4>
<p id="P21">Glycosphingolipids are amphipathic compounds, and care should be taken to keep the purified glycosphingolipids dissolved to prevent loss through surface adsorption. A solution containing at least 50% of an organic solvent such as methanol should be used on purified glycosphingolipids. Initial testing of pipets and vials should also be considered as surface adsorption can lead to poor recovery. The method we present here avoids the use of detergents and surfactants as they are incompatible with MS analysis.</p>
<p id="P22">Folch<sup><a href="#R65" class="usa-link" aria-describedby="R65">65</a></sup> or Bligh-Dyer<sup><a href="#R66" class="usa-link" aria-describedby="R66">66</a></sup> extraction is applied to the membrane pellet from the previous step to separate glycolipids from other lipid species such as phospholipids and cholesterol. The latter lipids are more abundant than glycosphingolipids by 1 or more orders of magnitude, and they can mask the analyte signal during MS analysis if not removed. Other extraction systems can also be applied, although Folch/B&amp;D extraction is advantageous for having the methanolic/aqueous phase as the top layer in the biphasic solvent extraction system. It should be noted that the recoveries of different glycosphingolipid species depend on the characteristics of both the glycan and the lipid. Glycosphingolipids with charged groups such as sialic acids have very good recovery (&gt;90%) even when they have longer lipids. Those with three or fewer neutral monosaccharides and longer lipids have poorer recoveries of less than 50%. Cerebrosides, which have only one monosaccharide residue, are not effectively recovered; about 90% of cerebrosides partition to the chloroform layer.</p>
<p id="P23">Internal standards such as deuterated glycolipids can be added to the sample prior to extraction for absolute quantitation. Internal standards of several glycolipid species are now commercially available from lipid-focused companies such as Avanti Inc. (AL, US) and Matreya Inc. (PA, US), including sialylated gangliosides, sulfatides, and neutral glycolipids. These companies also supply purified glycolipids and glycolipid reference mixtures that can be used for quality control and method optimization.</p>
<p id="P24">The samples are enriched with C8 SPE and analyzed with nanoLC-TOF-MS/MS equipped with a C18 nano-chip. The instrument method applies data-dependent acquisition (DDA) to obtain MS/MS spectra, which is necessary for the confident identification of glycosphingolipid species. Optimization of instrument parameters is critical for the effective fragmentation of glycosphingolipids, which will allow identification of both the ceramide (lipid) and glycan composition. Quality control samples can be obtained commercially from the aforementioned companies, or prepared from cultured cells such as Caco-2 or PNT2. Each cell line can express a limited variety of lipid compositions and glycan motifs. Glycolipids from several cell lines can be pooled together to include a larger variety for quality control. For example, the pluripotent cell line NTERA-2 can provide Stage Specific Embryonic Antigens (SSEA) 3 and 4, which are globo-type glycolipids that are expressed only by stem cells. An ECC of glycolipids from the glycocalyx of a cell line is shown in <a href="#F4" class="usa-link">Figure 4c</a>. In a typical chromatogram, over 200 compounds (including isomers) are observed. An ECC for QC glycolipids is shown in <a href="#SD6" class="usa-link">Supplementary Figure 1c</a>.</p></section><section id="S9"><h4 class="pmc_sec_title">O-Glycan analysis: Release, purification, and MS analysis (Steps 73–89).</h4>
<p id="P25">The pellets collected from Folch extraction are subjected to O-glycan release through β-elimination. Organic solvent left from Folch extraction should be removed before the release procedure. We recommend using larger volume (4–5 mL) screw-cap tubes for the reaction because large amounts of gas may be released and cause sample loss if the smaller 1.5 mL snap-cap eppendorf tubes are used. After overnight incubation, the basic mixture is neutralized with acetic acid. The whole process should be conducted on ice to avoid overheating. Care should also be taken when adding acetic acids, adding small aliquots is recommended to prevent more intense reactions that can cause the liquid to overflow. After centrifugation at high speed, O-glycans are collected from the supernatant, and the pellet can be discarded. At this point, the desired N- and O-glycans, as well as glycolipids, are all obtained from Fraction I. Chemical release of O-glycans using mild conditions, such as ammonium carbamate, can be employed but this is not recommended due to the low efficiency of the cleavage resulting in the potential loss of low-abundance compounds<sup><a href="#R67" class="usa-link" aria-describedby="R67">67</a></sup>. Cleaved O-glycans should be enriched on PGC SPE plates for desalting, and the glycans are further cleaned from small background peptides using iSPE HILIC cartridges/plates. At least five washes are recommended after sample loading because there is a large amount of chemically produced peptides due to the beta-elimination process. The same LC-MS instrument used for N-glycan analysis should be used for O-glycans. Some parameters need to be adjusted such as the mass range, which should be changed from 600 to 2,000 <em>m/z</em> to 300–2,000 <em>m/z</em> due to the relatively smaller sizes of O-glycans. For the instrument quality control (QC), the pooled released O-glycans from several cell lines under the normal condition (Caco-2, PNT2, and A549) can be used. A typical ECC of O-glycans from the glycocalyx of a cell line is shown in <a href="#F4" class="usa-link">Figure 4b</a>. The abundant species are annotated with typically over 50 compounds (including isomers). An ECC for QC O-glycans is shown in <a href="#SD6" class="usa-link">Supplementary Figure 1b</a>.</p></section><section id="S10"><h4 class="pmc_sec_title">Glycoproteomic analysis: Protein digestion, enrichment, and MS analysis (Steps 100–117).</h4>
<p id="P26">A comprehensive glycoproteomic analysis is conducted on extracted cell membranes. First, membrane proteins should be dissolved and denatured with urea for better digestion<sup><a href="#R68" class="usa-link" aria-describedby="R68">68</a></sup>, followed by the addition of DTT and alkylation with IAA to break S-S bonds. Samples are then digested with trypsin at 37 °C for 18 h. To obtain more comprehensive glycosylation site-specific mapping results, other enzymes with different specificities may be used in combination with trypsin such as Glu-C<sup><a href="#R69" class="usa-link" aria-describedby="R69">69</a></sup> and Lys-C<sup><a href="#R70" class="usa-link" aria-describedby="R70">70</a></sup>. Due to the lower ionization efficiencies of glycopeptides and possible ion suppression effects from coeluting peptides, the enrichment of glycopeptides is necessary before MS analysis. We suggest using iSPE HILIC cartridges, which give high enrichment efficiency. We have also tried the iSPE HILIC plates for large sample sets. However, the plate should not be centrifuged because this can cause the solid phase to dry out, leading to the loss of glycopeptides. With enrichment, the percentage of glycopeptides can be increased to 90% of all detected peptide species.</p>
<p id="P27">Glycopeptides are subjected to nanoLC-Orbitrap Fusion Lumos for MS/MS analysis. It is important to determine the sample concentration before the injection. The concentration should be adjusted to 1 μg/μL, with 1 μg used for injection. Oversampling could speed up column aging and may even cause clogging. We suggest that a digested serum glycopeptide sample be used for quality control (QC) of LC-MS and overall sample preparation. Typical glycoproteomic data from serum standards are illustrated in <a href="#F5" class="usa-link">Figure 5</a>. It is critical to use appropriate orbitrap parameters to obtain high-quality spectra, including the automatic gain control (AGC) and the maximum ion injection time (MIIT). Lower AGC and MIIT values will result in poor MS2 spectra, due to the low abundances of collected precursors. However, large AGC and MIIT values require longer accumulation times resulting in a decrease in identified species<sup><a href="#R71" class="usa-link" aria-describedby="R71">71</a></sup>. In addition, higher AGC values can result in space charge effects leading to poor mass accuracy and decreased identification rates<sup><a href="#R72" class="usa-link" aria-describedby="R72">72</a></sup>. For the fragmentation, higher-energy collision dissociation (HCD) with stepped-collision energy is employed, providing high-quality MS2 spectra for glycopeptides<sup><a href="#R73" class="usa-link" aria-describedby="R73">73</a></sup>. The spectra are obtained with the combination of fragmentations using three different collision energies. With the stepped-collision energy HCD, both glycans and peptide backbones are well fragmented. The glycan fragments can have higher intensities than those of peptide fragments in HCD spectra. Other fragmentation methods such as electron-transfer/higher-energy collision dissociation (EThcD)<sup><a href="#R74" class="usa-link" aria-describedby="R74">74</a></sup>, HCD triggered ETD<sup><a href="#R75" class="usa-link" aria-describedby="R75">75</a></sup>, and activated ion ETD (AI-ETD)<sup><a href="#R76" class="usa-link" aria-describedby="R76">76</a></sup> can also be used.</p>
<figure class="fig xbox font-sm" id="F5"><h5 class="obj_head">Figure 5. Glycoproteomic data of serum standard.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2ce2/11790333/c62466e70a21/nihms-2039916-f0005.jpg" loading="lazy" height="702" width="1050" alt="Figure 5."></p>
<div class="p text-right font-secondary"><a href="figure/F5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P389"><strong>a.</strong> The TIC of a serum glycoproteomic LC-MS/MS. For the analysis, approximately 1μg of digested serum glycopeptides were injected for analysis. The intensity of the base peak in the MS scan was 2.12E10. <strong>b.</strong> The summary of serum glycosylation types. The glycans was classified into five subtypes: complex/hybrid without fucoses or sialic acids (CH), fucosylated (F), sialofucosylated (SF), high-mannose (HM), and sialylated. <strong>c.</strong> The summary of the identified glycopeptides, glycoproteins, and glycosites. The data were collected from three technical replicates.</p></figcaption></figure></section><section id="S11"><h4 class="pmc_sec_title">Data processing.</h4>
<p id="P28">To process the glycomic data (Steps 34–43, 90–99), our in-house library based on the putative biosynthetic pathways<sup><a href="#R77" class="usa-link" aria-describedby="R77">77</a></sup> and previously discovered glycan masses is used with the compositions confirmed by tandem MS (MS/MS). There is no single curated database of possible glycan masses. However, several compilations are published and can be used for compound searches. A neural network derived glycan library derived from putative structures is published<sup><a href="#R78" class="usa-link" aria-describedby="R78">78</a></sup>. Glycan libraries of different species, including those with varied monosaccharides, for example xylose for plants<sup><a href="#R79" class="usa-link" aria-describedby="R79">79</a></sup>, N-glycolylneuraminic acid (Neu5Gc) for non-<em>Homo sapiens</em> mammals<sup><a href="#R80" class="usa-link" aria-describedby="R80">80</a>,<a href="#R81" class="usa-link" aria-describedby="R81">81</a></sup>, and N-acetylmuramic acid (MurNAc) for bacteria<sup><a href="#R82" class="usa-link" aria-describedby="R82">82</a></sup> exist. Other specialized glycans may also be added. For example, the 2-keto-3-deoxy-D-glycero-D-galacto-nononic acid (KDN) incorporated glycoconjugates could be included in the library for zebrafish<sup><a href="#R83" class="usa-link" aria-describedby="R83">83</a></sup>. Even glycans with larger monosaccharide modifications such as those found with 2,4-diacetamido-2,4,6-trideoxyhexose (DATDH) expressed in <em>Neisseria meningitides</em><sup><a href="#R84" class="usa-link" aria-describedby="R84">84</a></sup> may be included. Additionally, compositions of glycans containing unnatural monosaccharides such as N-azidoacetylgalactosamine (GalNAz) and N-azidoacetyl sialic acid (SiaNAz) can also be included. In general, care should be taken to ensure that the glycan compositions are confirmed by tandem MS. If only MS1 is used, naturally occurring glycans with combinations of, for example, Sia<sub>1</sub>HexNAc<sub>x</sub>Hex<sub>y</sub>, could be misassigned to SiaNAz<sub>1</sub>HexNAc<sub>(x-1)</sub>Hex<sub>(y+1)</sub>. Glycans with unnatural monosaccharides can also complicate the analysis but benefit from further MS/MS characterization. An example of the MS/MS spectra of unnatural monosaccharides incorporated N-glycans and assigned fragmentations are illustrated in <a href="#F6" class="usa-link">Figure 6</a>. Glycans containing SiaNAz yield fragment ions with <em>m/z</em> values of 315.09 [SiaNAz-H<sub>2</sub>O+H]<sup>+</sup> and 333.10 [SiaNAz+H]<sup>+</sup> in the MS/MS spectra.</p>
<figure class="fig xbox font-sm" id="F6"><h5 class="obj_head">Figure 6. Tandem MS spectra of selected compounds.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2ce2/11790333/501e145caf5e/nihms-2039916-f0006.jpg" loading="lazy" height="693" width="1050" alt="Figure 6."></p>
<div class="p text-right font-secondary"><a href="figure/F6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P390"><strong>a.</strong> N-Glycan Hex<sub>5</sub>HexNAc<sub>4</sub>Fuc<sub>1</sub>Sia<sub>1</sub> containing three mannose, two galactose, four N-acetylglucosamine, one fucose, and one sialic acid residue with a precursor <em>m/z</em> value of 1039.991. The <em>m/z</em> values of 292.097 and 274.092 correspond to the the sialic acid residue, and 657.237 and 819.273 correspond to the Gal<sub>1</sub>GlcNAc<sub>1</sub>Sia<sub>1</sub> and Man<sub>1</sub>Gal<sub>1</sub>GlcNAc<sub>1</sub>Sia<sub>1</sub> fragments, respectively. <strong>b</strong>. N-Glycan Hex<sub>5</sub>HexNAc<sub>4</sub>Fuc<sub>1</sub>SiaNAz<sub>1</sub> containing three mannose, two galactose, four N-acetylglucosamine, one fucose, and one azido-sialic acid residue with a precursor <em>m/z</em> value of 1061.374. The <em>m/z</em> 315.093 and 333.104 correspond to the azido-sialic acid residue, and 698.238 and 893.319 correspond to the Gal<sub>1</sub>GlcNAc<sub>1</sub>SiaNAz<sub>1</sub> and Man<sub>1</sub>Gal<sub>1</sub>GlcNAc<sub>1</sub>SiaNAz<sub>1</sub> fragments, respectively. <strong>c</strong>. N-Glycan Hex<sub>5</sub>HexNAc<sub>4</sub>Fuc<sub>1</sub>(SiaNAz-DBCO-NH<sub>2</sub>)<sub>1</sub> containing three mannose, two galactose, four N-acetylglucosamine, one fucose, and one dibenzocyclooctyne-amine (DBCO-NH<sub>2</sub>) conjugated azido-sialic acid residue with a precursor <em>m/z</em> value of 799.296. The <em>m/z</em> value of 591.223 and 609.233 correspond to the (DBCO-NH<sub>2</sub>) conjugated azido-sialic acid residue, and 974.352 and 1136.403 correspond to the Gal<sub>1</sub>GlcNAc<sub>1</sub>(SiaNAz-DBCO-NH<sub>2</sub>)<sub>1</sub> and Man<sub>1</sub>Gal<sub>1</sub>GlcNAc<sub>1</sub>(SiaNAz-DBCO-NH<sub>2</sub>)<sub>1</sub> fragments, respectively. The glycan annotations and legend of monosaccharides are provided. Figures reprinted with permission from ref. <a href="#R25" class="usa-link" aria-describedby="R25">25</a>. Creative Commons licence (CC BY 3.0) (<a href="http://creativecommons.org/licenses/by/3.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/3.0/</a>).</p></figcaption></figure><p id="P29">A library containing reduced O-glycan masses is useful for the data analysis of O-glycans. O-Glycomic profiles yield masses that are larger than the native structure by 2.016 Da, corresponding to the addition of two hydrogens. This modification stems from the reduction of the aldehyde after β-elimination. Thus, the entries in the glycan library should not only consider the possible compositions based on synthetic pathways but also potential modifications introduced during sample preparation. With the positive mode sulfated O-glycans can also be identified (<a href="#SD6" class="usa-link">Supplementary Figure 2</a>). The N-glycans need to be rigorously removed because they may interfere with the O-glycan analysis. N-Glycans can dissociate under basic conditions to yield fragments corresponding to the peeling reaction. However, the unique O-glycan core structures and reduced compositions can be used to distinguish them from fragmented N-glycans.</p>
<p id="P30">The data analysis of intact glycosphingolipids (Steps 57–72) has the added complexity of combining possible glycan compositions with ceramides of different hydrocarbon lengths, degrees of unsaturation, and numbers of hydroxyl groups. The identification of ceramides and glycans is based on MS/MS fragmentation data. There is currently no software that automates this process. In this protocol, we offer some tools that can assist with the correct identification of glycosphingolipid species and for generating a subset library from a database that contains the neutral monoisotopic mass and chemical formulae of intact glycosphingolipids. We built upon work previously done by Merrill (<a href="http://www.sphingomap.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.sphingomap.org</a>)<sup><a href="#R85" class="usa-link" aria-describedby="R85">85</a></sup> in collating possible glycan structures and incorporated their compositions into intact glycosphingolipids. It is important to form a subset library because the combination of so many glycan and ceramide moieties contains many isobaric compounds, and an uncurated list can lead to false identifications or misassignments. Additionally, it is also important to create a new library for each kind of sample analyzed as the expression of glycosphingolipids can differ based on cell type, species source, and even among individuals of the same species.</p>
<p id="P31">For glycoproteomic data processing (Steps 118–125), Byonic software is used to identify the glycopeptides, and the results can be directly applied for label-free quantitation using Byologic software. Other software, such as pGlyco, can also be used to identify glycopeptides<sup><a href="#R86" class="usa-link" aria-describedby="R86">86</a></sup>. The data analysis with Byonic<sup>™</sup> software provides information, including glycopeptide amino acid sequences, glycosylation sites, and tentative glycan compositions. Although a general glycan library from an open source can be used for glycopeptide identification, before conducting glycoproteomic analysis on an unknown cell or tissue sample we suggest doing the glycomic analysis first to calibrate the specific glycan library for the cell type and to use the calibrated glycan library for glycopeptide data analysis. This not only reduces the search time (especially for O-glycopeptide mapping) but also minimizes false-positive identification. Several parameters including DeltaMod, Byonic scores, and |Log Prob| are recommended for screening the results to control the FDR further and eliminate misassignments<sup><a href="#R87" class="usa-link" aria-describedby="R87">87</a></sup>. The DeltaMod of 10.0 is the minimum threshold for confident modifications. Byonic scores indicate the quality of the peptide-spectrum match and, as illustrated by the software developers, a score of over 300 is a good score for the match. |Log Prob| reflects the posterior error probability, and results with |Log Prob|&lt;2 (error probabilities &gt; 0.01) are usually removed. Examples of site-specific mapping of N- and O-glycoproteins, as well as quantitation of glycopeptides are shown in <a href="#F7" class="usa-link">Figure 7</a>.</p>
<figure class="fig xbox font-sm" id="F7"><h5 class="obj_head">Figure 7. Examples of site-specific glycopeptide mapping of N-linked glycoprotein and O-linked glycoprotein.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2ce2/11790333/cdacd04e47aa/nihms-2039916-f0007.jpg" loading="lazy" height="813" width="1050" alt="Figure 7."></p>
<div class="p text-right font-secondary"><a href="figure/F7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P391"><strong>a</strong>. Human Basigin, also known as CD147, was identified from Caco-2 cells, and its N-linked glycans were identified with the site heterogeneity at Asn160 and Asn268. b. Human trans-Golgi network integral membrane protein 2 was obtained from HT-29 cells, and its O-linked glycans were identified with the site heterogeneity at Thr62, Thr132, Thr175, Ser182, Thr188, Thr202, and Ser270. <strong>c</strong>. Basigin extracellular/intracellular domains were analyzed by transmembrane helix prediction software, and both of Asn160 and Asn268 were expected to be extracellular. The relative abundances of glycans at two different sites were quantified with the Byologic software. The heat map showed the degrees of glycosylation at two different glycosites thereby revealing quantitatively the glycosylation heterogeneity at different sites.</p></figcaption></figure></section></section><section id="S12"><h3 class="pmc_sec_title">Limitations of the protocol</h3>
<p id="P32">A limitation of the glycomic analysis method is the requirement for specialized instruments that are large, centralized, and expensive. While LC-MS methods provide comprehensive information, users require specialized training. However, LC-MS instruments are now more widely available in most departments and institutions. Nearly every researcher has access to an LC-MS instrument even when it is not directly available in their respective laboratory. Moreover, by clearly describing the methods for glycomic analysis, we hope to make the method more readily available so that the lack of expertise will be less of a barrier. In this protocol, we use LC with nano-flow coupled to high-resolution MS QTOF and orbitrap to achieve better sensitivity and higher mass accuracy. Better sensitivity facilitates the identification of a higher number of compounds, and the higher mass accuracy contributes to more confident identifications. This protocol can also be used with standard LC flow rates and lower resolution mass spectrometers; however, it would require larger sample sizes with less certainty in the identification. More certainty can be obtained with tandem MS (CID), which yields distinct fragment masses even with low mass accuracy instruments.</p>
<p id="P33">Another potential limitation is that interpreting the LC-MS data is extremely time-consuming. Specialized software are available but wedded to specific instrument manufacturers. For example, the software used in this protocol for glycomic and glycolipidomic data analyses is MassHunter, which was developed specifically for Agilent Inc. instruments. However, software that are used for other applications such as proteomic can be adapted to glycans. They include Skyline and mzMine 2 for processing the data acquired from other instruments. For glycoproteomic analysis, we include the details regarding the use of Byonic, which is a commercial software appropriate for analyzing peptides with glycosylation and other post-translational modifications. Other software such as MaxQuant and pGlyco can also be used for glycomic and glycoproteomic analysis. In addition, using ion counts for quantitation is the best we have, but it does have intrinsic limitations, with the most significant being that not all compounds have the same ionization and detection responses. Nonetheless, it is accurate for measuring fold changes of individual compounds, but perhaps not as accurate for comparing abundances of different compounds in the same run.</p>
<p id="P34">Other limitations are associated with the intrinsic nature of glycans and their structures. For example, although LC-MS can provide extensive separation of isomers, it yields little structural information. Determining the structures requires extensive use of exoglycosidases, which have been performed to determine the glycome of serum<sup><a href="#R88" class="usa-link" aria-describedby="R88">88</a>,<a href="#R89" class="usa-link" aria-describedby="R89">89</a></sup>. Similarly, glycoproteomic analysis gives the best results when the glycopeptides are enriched. This means that to obtain an extensive (glyco)proteomic analysis, two analyses need to be performed: the first one is the peptide analysis (proteomic), followed by a glycopeptide analysis (glycoproteomic). Within the glycoproteomic analysis, relative abudances of each glycoform can be obtained; however the site-specific occupancy can only be inferred if the total abundances of all glycoforms are compared. Accurate quantitation of site occupancy requires the deglycosylation and analysis of both aglycosylated and non-glycosylated peptides with LC-MS<sup><a href="#R90" class="usa-link" aria-describedby="R90">90</a></sup>. Other issues are encountered in the glycoproteomic analysis including misassignment of glycan structures and ambiguous localization of glycosylation sites, especially for N-glycopeptides with two potential glycosites within one tryptic peptide. Thus, it is worthwhile checking the MS/MS spectra of those suspected compounds. O-Glycopeptides may also be difficult to assign as the sites do not have a consensus sequence. For the enrichment of O-glycans using iSPE-HILIC cartridges, some short O-glycans may be lost due to the potential bias of the HILIC enrichment toward more elongated glycans.</p>
<p id="P35">GAGs including heparin/heparan sulfate, chondroitin sulfate/dermatan sulfate, keratan sulfate, and hyaluronan are linear polysaccharides consisting of multiple disaccharide repeating units (mostly an amino sugar and a uronic sugar or galactose) and play significant roles in a variety of biological functions<sup><a href="#R91" class="usa-link" aria-describedby="R91">91</a></sup>. However, due to the structural and chemical nature of GAGs, their analysis require completely different methods with different MS techniques that are beyond the scope of this protocol.</p></section><section id="S13"><h3 class="pmc_sec_title">Future applications</h3>
<p id="P36">Glycans and glycosylation impact nearly every aspect of biology. Estimates are that 50% of all proteins, and nearly 80% of human proteins are glycosylated. As this protocol offers a nearly complete rendering of all glycans(glycosaminoglycans notwithstanding) in any biological sample, its utility will cover nearly every aspect of biology. In addition, this workflow could be readily incorporated with other techniques, including proteomic analysis and metabolic labeling. The more specific and immediate applications include i) The broad characterization of cell lines from different species. ii) The glycomic characterization and quantitation of cell membranes subjected to enzyme inhibitors and gene modifications. iii) The discovery and development of glycan, glycoprotein, and glycolipid biomarkers for diseases. iv) Studies of host-microbe interactions and the role of glycocalyx in adhesion and invasion. v) Studies of protein functions and structures, including cell surface receptors such as Siglecs, channels, enzymes, and pumps. vi) Studies of cancer cell physiology and the role of glycans in cell adhesion and signaling.</p>
<p id="P37">These methods will also have broader impacts in diverse areas of research where glycoconjugates (lipids and proteins) are involved. These research areas include cancer diagnostic, progression and therapy, infection, immunology, and diet and nutrition. In the study of cancer progression and therapy, it is known that the glycosylation of cells differs at different stages in cancer progression. Glycan changes are also seen during tumor growth and metastasis. All approved cancer protein biomarkers are glycosylated. Additionally, many new cancer therapeutics are glycosylated, and their targets on the cell membrane are glycoproteins. Secondly, the glycocalyx is also important in understanding host-microbe interactions. The binding sites on host cells are often glycans on proteins and lipids, which mediate microbe adhesion and invasion. Bacteria and viruses also have glycan-binding proteins and infect host cells with associated glycan structures. In immunology, immune cells and antibodies directly recognize glycans on glycoproteins and glycolipids. Activated immune cells are affected by glycosylation. A better understanding of the role of glycosylation in immunity and autoimmunity could provide better therapies. Finally, commensal bacteria consume oligosaccharides provided by the host, such as those in human milk, and mucins on the cell membrane. Oligosaccharides and glycans are currently revolutionizing our understanding of how the microbiome functions in diet and nutrition studies.</p></section></section><section id="S14"><h2 class="pmc_sec_title">Materials</h2>
<section id="S15"><h3 class="pmc_sec_title">Biological materials</h3>
<ul id="L1" class="list" style="list-style-type:disc">
<li><p id="P38">PNT2 cells (Sigma, cat. no. 95012613, RRID: CVCL-2164)</p></li>
<li><p id="P39">Caco-2 cells (ATCC, cat. no. HTB-37, RRID: CVCL-0025). ! CAUTION This cell line will differentiate when it reaches 100% confluency without any chemical triggering.</p></li>
<li><p id="P40">A549 cells (ATCC, cat. no. CCL-185, RRID: CVCL-0023)</p></li>
<li><p id="P41">NTERA-2 cl.D1 cells (ATCC, cat. no. CRL-1973, RRID: CVCL-3407)</p></li>
</ul>
<p id="P42">CAUTION The cell lines used in the research should be regularly checked to ensure they are authentic and are not infected with mycoplasma.</p>
<ul id="L2" class="list" style="list-style-type:disc"><li><p id="P43">Tissue sample of interest (i.e., brain, liver, spleen, etc.) should be collected in a clean tube of adequate volume (1.5, 2, 5ml), snap-frozen in liquid N<sub>2</sub>, and stored at −80°C until further analysis</p></li></ul></section><section id="S16"><h3 class="pmc_sec_title">Reagents</h3>
<ul id="L3" class="list" style="list-style-type:disc">
<li><p id="P44">Fetal Bovine Serum (FBS, Thermo Fisher Scientific, cat. no. 16000–069)</p></li>
<li><p id="P45">Penicillin Streptomycin (P/S, Thermo Fisher Scientific, cat. no. 15140–122)</p></li>
<li><p id="P46">Trypsin-EDTA (1X, Thermo Fisher Scientific, cat. no. 25300–054)</p></li>
<li><p id="P47">Dulbecco’s Phosphate Buffered Saline (D-PBS, 1X, ATCC, cat. no. 30–2200)</p></li>
<li><p id="P48">Sucrose (BioXtra, Milliporesigma, cat. no. S7903)</p></li>
<li><p id="P49">4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid (HEPES buffer, 1M, Thermo Fisher Scientific, cat. no. 15630080)</p></li>
<li><p id="P50">Potassium hydroxide (KOH, BioXtra, Milliporesigma, cat. no. P5958) ! CAUTION Potassium hydroxide causes severe skin burns and eye damage.</p></li>
<li><p id="P51">Sodium carbonate (Na<sub>2</sub>CO<sub>3</sub>, BioXtra, Milliporesigma, cat. no. S7795) ! CAUTION Sodium carbonate causes serious eye irritation.</p></li>
<li><p id="P52">Ammonium bicarbonate (NH<sub>4</sub>HCO<sub>3</sub>, BioUltra, Milliporesigma, cat. no. 09830) ! CAUTION NH<sub>4</sub>HCO<sub>3</sub> irrates skin, eyes and respiratory system.</p></li>
<li><p id="P53">Dithiothreitol (DTT, molecular grade, Promega, cat. no. V3151) ! CAUTION DTT causes skin and eye irritation and may cause respiratory irritation.</p></li>
<li><p id="P54">Iodoacetamide (IAA, BioUltra, Milliporesigma, cat. no. 144–48-9) ! CAUTION Iodoacetamide may cause an allergic skin reaction.</p></li>
<li><p id="P55">Sodium hydroxide (NaOH, anhydrous, Milliporesigma, cat. no. 795429) ! CAUTION Sodium hydroxide causes severe skin burns and eye damage.</p></li>
<li><p id="P56">Sodium borohydride (NaBH<sub>4</sub>, Supelco<sup>™</sup>, Milliporesigma, cat. no. 1.06371) ! CAUTION Sodium borohydride causes skin corrosion, serious eye damage, and reproductive toxicity.</p></li>
<li><p id="P57">Formic acid (FA, Optima<sup>™</sup>; Fisher Scientific, cat. no. A117–50) ! CAUTION Formic acid is flammable liquid and vapor.</p></li>
<li><p id="P58">Trifluoroacetic acid (TFA, Pierce<sup>™</sup>, ThermoFisher Scientific, cat. no. 28902) ! CAUTION Trifluoroacetic acid causes severe skin burns and eye damage</p></li>
<li><p id="P59">Acetonitrile (ACN, CHROMASOLV<sup>™</sup>, Honeywell, cat. no. 34967) ! CAUTION Acetonitrile is flammable liquid and vapor.</p></li>
<li>
<p id="P60">Water (H<sub>2</sub>O, Pierce<sup>™</sup>, ThermoFisher Scientific, cat. no. 51140)</p>
<p id="P61">⮙ CRITICAL This water is used only for the nanoLC system in the glycoproteomic analysis.</p>
</li>
<li><p id="P62">Methanol (MeOH, Milliporesigma, cat. no. 34860) ! CAUTION Methanol is flammable liquid and vapor.</p></li>
<li><p id="P63">2-Propanol (IPA, Milliporesigma, cat. no. 34863) ! CAUTION Isopropyl alcohol is flammable liquid and vapor.</p></li>
<li><p id="P64">Chloroform (CHCl<sub>3</sub>, Milliporesigma, cat. no. 34854) ! CAUTION Chloroform is flammable liquid and vapor. It is also acutely toxic.</p></li>
<li><p id="P65">Ammonium acetate (Supelco<sup>™</sup>, Milliporesigma, cat. no. AX1222)</p></li>
<li><p id="P66">Acetic acid (glacial, Alfa Aesar<sup>™</sup>, ThermoFisher Scientific, cat. no. AA39745AE) ! CAUTION Acetic acid causes severe eye and skin burns, and acetic acid is flammable liquid and vapor.</p></li>
<li>
<p id="P67">PNGase F (Glycerol-free, New England BioLabs, cat. no. P0705L)</p>
<p id="P68">⮙ CRITICAL PNGase F cannot cleave off N-glycans with α(1,3)-linked core fucose.</p>
</li>
<li><p id="P69">Trypsin (Sequencing Grade Modified, Promega, cat. no. V5111)</p></li>
<li><p id="P70">Protease Inhibitor Cocktail Set V, EDTA-Free (Calbiochem<sup>™</sup>, cat. no. 539137)</p></li>
<li><p id="P71">N-Omega-CD3-Octadecanoyl monosialoganglioside GM1 (NH4<sup>+</sup> salt) (Matreya LLC, cat. no. 2050)</p></li>
<li><p id="P72">N-Omega-CD3-Octadecanoyl disialoganglioside GD3 (Matreya LLC, cat. no. 2054)</p></li>
<li><p id="P73">Quantitative Colorimetric Peptide Assay (Pierce<sup>™</sup>, ThermoFisher Scientific, cat. no. 23275)</p></li>
<li><p id="P74">BCA Protein Assay Kit (Pierce<sup>™</sup>, ThermoFisher Scientific, cat. no. 23225)</p></li>
</ul></section><section id="S17"><h3 class="pmc_sec_title">Equipment</h3>
<ul id="L6" class="list" style="list-style-type:disc">
<li><p id="P75">1.5 mL Microcentrifuge Tubes Natural (Beckman Coulter, cat. no. 357448)</p></li>
<li><p id="P76">1.5 mL Eppendorf<sup>™</sup> Snap-Cap Microcentrifuge Flex-Tube<sup>™</sup> Tubes (ThermoFisher Scientific, cat. no. 022364111)</p></li>
<li><p id="P77">Sorvall<sup>™</sup> WX 100+ Ultracentrifuge (ThermoFisher Scientific, cat. no. 75000100 )</p></li>
<li><p id="P78">Fiberlite<sup>™</sup> F50L-24 × 1.5 Fixed-Angle Rotor (ThermoFisher Scientific, cat. no. 096–247028)</p></li>
<li><p id="P79">Q700 Sonicator system with standard probe (QSonica, cat. no. Q700–110)</p></li>
<li><p id="P80">Forma<sup>™</sup> Series II 3110 Water-Jacketed CO<sub>2</sub> Incubators (ThermoFisher Scientific, cat. no. 3110)</p></li>
<li><p id="P81">Fisherbrand<sup>™</sup> Isotemp<sup>™</sup> Digital-Control Water Baths (ThermoFisher Scientific, cat. no. 15–462-S3S)</p></li>
<li><p id="P82">Discover Proteomics – Microwave Protein Sample Preparation (CEM Corporation)</p></li>
<li><p id="P83">Lab Armor<sup>™</sup> Bead Bath (ThermoFisher Scientific, cat. no. 10–876-006)</p></li>
<li><p id="P84">iSPE-HILIC SPE Cartridges (HILICON, cat. no. 200.001.0100)</p></li>
<li><p id="P85">C8 plate (Glygen, cat. no. FNSC08.800)</p></li>
<li><p id="P86">Graphitized Carbon plate (Glygen, cat. no. FNSCAR800)</p></li>
<li><p id="P87">Acclaim<sup>™</sup> PepMap<sup>™</sup> 100 C18 LC Column (3 μm, 0.075 mm x 250 mm, ThermoFisher Scientific, cat. no. 164569)</p></li>
<li><p id="P88">Eppendorf<sup>™</sup> 96-Well twin.tec<sup>™</sup> PCR Plates (ThermoFisher Scientific, cat. no. E951020401)</p></li>
<li><p id="P89">Adhesive Sealing Film ( ThermoFisher Scientific, NC0525923)</p></li>
<li><p id="P90">250 μL Polypropylene Vial (Agilent, cat. no. 5188–2788)</p></li>
<li><p id="P91">2 mL Crimp/Snap Top Vials &amp; Cap (Agilent, cat. no. 5182–0541)</p></li>
<li><p id="P92">E-Pure<sup>™</sup> water purification systems (ThermoFisher Scientific, cat. no. D4631)</p></li>
<li><p id="P93">Leica DMI3000 B microscope (Lecia Microsystems)</p></li>
<li><p id="P94">Agilent 1200 Series Liquid Chromatograph-Chip System (Agilent Technologies)</p></li>
<li><p id="P95">Agilent 6520 Accurate Mass Q-TOF LC/MS (Agilent Technologies)</p></li>
<li><p id="P96">Agilent C18 Chip (40 nL enrichment column, 5 μm 75 μm x 150 mm separation column, Agilent Technologies, cat. no. G4240–62006)</p></li>
<li><p id="P97">Agilent Graphitized Carbon Chip (40 nL enrichment column, 5 μm 75 μm x 43 mm separation column, Agilent Technologies, cat. no. G4240–64010)</p></li>
<li><p id="P98">Orbitrap Fusion Lumos Tribrid Mass Spectrometer with EASY-ETD (ThermoFisher Scientific)</p></li>
<li><p id="P99">UltiMate<sup>™</sup> WPS-3000RS nanoLC system (ThermoFisher Scientific)</p></li>
<li><p id="P100">Nanospray Flex ion source (ThermoFisher Scientific)</p></li>
</ul></section><section id="S18"><h3 class="pmc_sec_title">Software</h3>
<p id="P101">⮙ CRITICAL All required software can be run on a standard personal computer equipped with a Windows operating system.</p>
<ul id="L9" class="list" style="list-style-type:disc">
<li><p id="P102">Agilent Data Acquisition for QTOF 6500</p></li>
<li><p id="P103">Agilent MassHunter Qualitative Analysis (B.08.00)</p></li>
<li><p id="P104">Thermo Fisher Scientific Xcalibur<sup>™</sup> (4.0)</p></li>
<li><p id="P105">Protein Metrics Byonic<sup>™</sup> (v 3.5.0)</p></li>
<li><p id="P106">Protein Metrics Byonic<sup>™</sup>-Viewer (v 3.5.0)</p></li>
<li><p id="P107">Protein Metrics Byologic<sup>™</sup> (v 3.5.0) (optional)</p></li>
</ul></section><section id="S19"><h3 class="pmc_sec_title">Reagent set-up</h3>
<section id="S20"><h4 class="pmc_sec_title">Homogenization buffer (HB buffer)</h4>
<p id="P108">This solution is made up of 0.25 M sucrose, 20mM HEPESKOH at pH 7.4 and 1:100 protease inhibitor mixture in water. Prepare 20 mM HEPES and adjust pH to around 7.4 by adding solid KOH. Weigh out 4.28 g of sucrose and dissolve in 39 mL of H<sub>2</sub>O. Dissolve one bottle of protease inhibitor with 1 mL of H<sub>2</sub>O. Add 10 mL of HEPES·KOH solution and protease inhibitor to the sucrose solution. Vortex the solution to mix and aliquot 1.2 mL of solution to 1.5 mL Eppendorf tubes. This solution can be stored at −20 °C for several months.</p></section><section id="S21"><h4 class="pmc_sec_title">Cell membrane pellet wash solution</h4>
<p id="P110">This solution is 0.2 M Na<sub>2</sub>CO<sub>3</sub> in water. Weigh out 10.60 g of Na<sub>2</sub>CO<sub>3</sub> and dissolve in 500 mL of H<sub>2</sub>O. The volume of the stock solution can be adjusted accordingly. This solution can be prepared as stock solution and stored at 4 °C for several months.</p></section><section id="S22"><h4 class="pmc_sec_title">N-Glycan release solution</h4>
<p id="P111">This solution is 100 mM NH<sub>4</sub>HCO<sub>3</sub> and 5 mM DTT in water. Weight out 79.06 mg of NH<sub>4</sub>HCO<sub>3</sub> and 7.71 mg of DTT and dissolve in 10 mL of H<sub>2</sub>O. The solution can be prepared as stock solution and stored at 4 °C for several months.</p></section><section id="S23"><h4 class="pmc_sec_title">Folch extraction solution</h4>
<p id="P112">This solvent is the mixture of H<sub>2</sub>O/MeOH/CHCl<sub>3</sub> with volume ratio of 3:8:4. Mix 9 mL of H<sub>2</sub>O, 24 mL of MeOH, and 12 mL of CHCl<sub>3</sub> to prepare 45 mL of Folch solvent. This solution can be stored at 4 °C for several months.</p></section><section id="S24"><h4 class="pmc_sec_title">O-Glycan beta-elimination solution 1</h4>
<p id="P113">This solution is 2 M NaBH<sub>4</sub> in water. Weigh out 378.30 mg of NaBH<sub>4</sub> and dissolve in 5 mL of H<sub>2</sub>O. This solution should be prepared freshly every time right before the experiment. The volume of solution can be adjusted according to the number of samples.</p></section><section id="S25"><h4 class="pmc_sec_title">O-Glycan beta-elimination solution 2</h4>
<p id="P114">This solution is 2 M NaOH in water. Weight out 160.00 mg of NaOH and dissolve in 2 mL of H<sub>2</sub>O. This solution should be prepared freshly every time right before the experiment. The volume of solution can be adjusted according to the number of samples.</p></section><section id="S26"><h4 class="pmc_sec_title">Protein digestion buffer</h4>
<p id="P115">This solution is 50 mM NH<sub>4</sub>HCO<sub>3</sub> in water. Weight out 79.06 mg of NH<sub>4</sub>HCO<sub>3</sub> and dissolve in 20 mL of H<sub>2</sub>O. The solution should be prepared freshly every time right before the experiment.</p></section><section id="S27"><h4 class="pmc_sec_title">Protein denature solution</h4>
<p id="P116">This solution is 550 mM DTT in 50 mM NH<sub>4</sub>HCO<sub>3</sub> solution. Weigh out 424.19 mg of DTT and dissolve in 5 mL of 50 mM NH<sub>4</sub>HCO<sub>3</sub> solution. This solution can be aliquoted and stored at −20 °C for several months.</p></section><section id="S28"><h4 class="pmc_sec_title">Protein alkylation solution</h4>
<p id="P117">This solution is 450 mM IAA in 50 mM NH<sub>4</sub>HCO<sub>3</sub> solution. Weigh out 416.16 mg of IAA and dissolve in 5 mL of 50 mM NH<sub>4</sub>HCO<sub>3</sub> solution. This solution can be aliquoted and stored at −20 °C for several months.</p></section><section id="S29"><h4 class="pmc_sec_title">PGC SPE cartridge condition solution</h4>
<p id="P118">This solution is 80% (vol/vol) ACN in water with 0.1% (vol/vol) TFA. Add 99.5 mL of H<sub>2</sub>O and 0.5 mL of TFA to 400 mL of ACN. This solution can be prepared as stock solution and stored at 4 °C for several months.</p></section><section id="S30"><h4 class="pmc_sec_title">PGC SPE cartridge elution solution</h4>
<p id="P119">This solution is 40% (vol/vol) ACN in water with 0.05% (vol/vol) TFA. Add 200 mL of ACN and 0.25 mL of TFA to 299.75 mL of H<sub>2</sub>O. This solution can be prepared as stock solution and stored at 4 °C for several months.</p></section><section id="S31"><h4 class="pmc_sec_title">C8 SPE plate condition solution</h4>
<p id="P120">This solution is 1:1 methanol/water (vol/vol). Add 50 mL of methanol and 50 mL of water, and mix thoroughly. This solution can be prepared as stock solution and stored at 4 °C for several months.</p></section><section id="S32"><h4 class="pmc_sec_title">C8 SPE plate elution solution</h4>
<p id="P121">This solution is 1:1 methanol/IPA (vol/vol). Add 50 mL of methanol and 50 mL of IPA, and mix thoroughly. This solution can be prepared as stock solution and stored at 4 °C for several months.</p></section><section id="S33"><h4 class="pmc_sec_title">HILIC cartridge condition solution (O-glycan)</h4>
<p id="P122">This solution is 90% (vol/vol) ACN in water with 1% (vol/vol) TFA. Add 45 mL of H<sub>2</sub>O and 5 mL of TFA to 450 mL of ACN. This solution can be prepared as stock solution and stored at 4 °C for several months.</p></section><section id="S34"><h4 class="pmc_sec_title">HILIC cartridge condition solution (glycopeptide)</h4>
<p id="P123">This solution is 80% (vol/vol) ACN in water with 1% (vol/vol) TFA. Add 95 mL of H<sub>2</sub>O and 5 mL of TFA to 400 mL of ACN. This solution can be prepared as stock solution and stored at 4 °C for several months.</p></section><section id="S35"><h4 class="pmc_sec_title">HILIC cartridge elution solution</h4>
<p id="P124">This solution is Milli-Q water with 0.1% (vol/vol) TFA. Add 0.5 mL of TFA to 499.5 mL of H<sub>2</sub>O. This solution can be prepared as stock solution and stored at 4 °C for several months.</p></section><section id="S36"><h4 class="pmc_sec_title">N- and O-Glycomics LC solvent A</h4>
<p id="P125">This solvent is 3% (vol/vol) ACN in water with 0.1% (vol/vol) FA. Add 30 mL of ACN and 1 mL of FA to H<sub>2</sub>O, degas and transfer to mobile phase container to make a total of 1 L of glycomics LC solvent A. This solution should be prepared freshly right before running each batch of samples.</p></section><section id="S37"><h4 class="pmc_sec_title">N- and O-Glycomics LC solvent B</h4>
<p id="P126">This solvent is 90% (vol/vol) ACN in water with 1% (vol/vol) FA. Add 90 mL of H<sub>2</sub>O and 10 mL of FA to ACN, degas and transfer to mobile phase container to make a total of 1 L of glycomics LC solvent B. This solution should be prepared freshly right before running each batch of samples.</p></section><section id="S38"><h4 class="pmc_sec_title">Glycolipidomics LC solvent A</h4>
<p id="P127">This solvent is water with 0.1% (vol/vol) acetic acid and 20 mM ammonium acetate. Add 1.54 g of ammonium acetate to 400 mL of H<sub>2</sub>O, mix thoroughly until all salts are dissolved. Add 1 mL of acetic acid and adjust water to final volume of 1 L, degas and transfer to mobile phase container to make a total of 1 L of glycolipidomics solvent A. This solution should be prepared freshly right before running each batch of samples.</p></section><section id="S39"><h4 class="pmc_sec_title">Glycolipidomics LC solvent B</h4>
<p id="P128">This solvent is 15% (vol/vol) IPA in methanol with 0.1% (vol/vol) acetic acid. Add 150 mL of IPA and 1 mL of acetic acid to 400 mL of methanol. Add methanol to final volume of 1 L, degas and transfer to mobile phase container to make a total of 1 L of glycolipidomics solvent B. This solution should be prepared freshly right before running each batch of samples.</p></section><section id="S40"><h4 class="pmc_sec_title">Glycopeptide LC solvent A</h4>
<p id="P129">This solvent is 99.9% (vol/vol) LC-MS grade water with 0.1% (vol/vol) FA. Add 1 mL of FA to LC-MS grade water to make a total volume of 1 L of solvent A. This solution can be stored at room temperature and used for two weeks.</p></section><section id="S41"><h4 class="pmc_sec_title">Glycopeptide LC solvent B</h4>
<p id="P130">This solvent is 90% (vol/vol) LC-MS grade ACN, and 9.9% LC-MS grade water with 0.1% (vol/vol) FA. Add 99 mL of LC-MS grade water and 1 mL of FA to LC-MS grade ACN to make a total volume of 1 L of solvent B. This solution can be stored at room temperature and used for two weeks.</p></section></section></section><section id="S42"><h2 class="pmc_sec_title">Procedures</h2>
<section id="S43"><h3 class="pmc_sec_title">
<em>Sample preparation</em> ●Timing depends on sample type</h3>
<ul id="L10" class="list" style="list-style-type:none"><li>
<span class="label">1</span><div class="display-inline p" id="P131">To prepare tissue follow option A. To prepare mammalian cell lines follow option B.
<ol id="L11" class="list" style="list-style-type:upper-alpha">
<li><div class="p" id="P132">
<strong><em>Tissue sample preparation</em> ●Timing 1–3 h depends on the size of sample set</strong>
<ol id="L12" class="list" style="list-style-type:lower-roman">
<li><div class="p" id="P133">Collect tissue samples from dissection. Cut the frozen tissue samples into small pieces if necessary (for muscle).</div></li>
<li><div class="p" id="P134">Wash around 50–100 mg of tissue sample with 1 mL of PBS buffer.</div></li>
<li><div class="p" id="P135">Centrifuge the sample at 1,000 x g for 3 min. Discard the supernatant. Suspend the pellet with 1.2 mL of HB buffer.</div></li>
</ol>
</div></li>
<li><div class="p" id="P136">
<strong><em>Mammalian cell growth</em> ●Timing 5 d</strong>
<ol id="L13" class="list" style="list-style-type:lower-roman">
<li>
<div class="p" id="P137">Culture cells using appropriate growth medium supplemented with 10% (vol/vol) FBS and 1% (vol/vol) Penicillin–Streptomycin. Grow cells at 37 °C in a cell culture incubator in a humidified atmosphere containing 5% CO<sub>2</sub>.</div>
<div class="p" id="P138">⮙ CRITICAL STEP We have successfully cultured PNT2, Caco-2, and A549 cell lines. Growth media and conditions may need to be altered for different cell lines.</div>
</li>
<li><div class="p" id="P139">Grow cells until 80% confluency is reached. Split cells using techniques appropriate for your cell line into 10 cm petri dish or T75 flask. Briefly, aspirate media and wash cells with 10 mL of PBS for two times, followed by adding 2.5 mL of Trypsin-EDTA and incubating for 5 min. Then add 5 mL of media to stop the digestion and homogenize the mixture by pipetting for several times. Add a desired volume of cell mixture (usually in the range of 200 μL to 1 mL) to a new dish or flask containing 10 mL of media.</div></li>
<li><div class="p" id="P140">Monitor cell proliferation using a microscope and change media every other day.</div></li>
<li>
<div class="p" id="P141">Cells may be harvested at around 90% confluency. Wash cells with 10 mL of PBS twice before harvesting to thoroughly remove detached cells and the growth medium.</div>
<div class="p" id="P142">⮙ CRITICAL STEP Some cells should be harvested earlier to prevent further maturation, such as the Caco-2 cell line, if you prefer to analyze undifferentiated cells.</div>
</li>
<li>
<div class="p" id="P143">Add 10 mL of PBS and scrape cells with a scraper. Scrape cells in one direction and make sure all cells are collected. Then transfer the mixture to a 15 mL centrifuge tube.</div>
<div class="p" id="P144">⮙ CRITICAL STEP For the Caco-2 cell line, continue culturing after 100% confluency for another 14 days if you wish to obtain differentiated Caco-2 cells.</div>
</li>
<li><div class="p" id="P145">Wash the dish or the flask with 5 mL of PBS and combine it with the previous mixture.</div></li>
<li><div class="p" id="P146">Centrifuge the suspension at 1,000 x g for 3 min at 4 °C. Remove the supernatant and resuspend the cells in 1.2 mL of HB buffer. Count the cell number if needed to make sure at least 1 × 10<sup>6</sup> cells are collected.</div></li>
</ol>
</div></li>
</ol>
</div>
</li></ul></section><section id="S44"><h3 class="pmc_sec_title">
<em>Cell and tissue lysis and membrane extraction</em> ●Timing 5–8 h depends on the size of sample set</h3>
<ul id="L17" class="list" style="list-style-type:none">
<li>
<span class="label">2</span><p class="display-inline" id="P147">Pipette up and down to mix the cell suspension before lysis.</p>
</li>
<li>
<span class="label">3</span><p class="display-inline" id="P148">Place the 15mL falcon tube on an ice bucket or a 9-well cooling rack and keep the sample at 4 °C</p>
</li>
<li>
<span class="label">4</span><p class="display-inline" id="P149">Place the probe of the sonicator into the sample.</p>
<p id="P150">⮙ CRITICAL STEP make sure the tip of the tube is in the center of the sample.</p>
</li>
<li>
<span class="label">5</span><p class="display-inline" id="P151">Sonicate the sample using the following conditions: 25 amplitude (Amp); 5 seconds (s) on; 10 s off for 25 s of total sonication time.</p>
<p id="P152">⮙ CRITICAL STEP Monitor the power during the entire sonicating process. Make sure the power is not exceeding 60 watts (W) to prevent over-heating.</p>
</li>
<li>
<span class="label">6</span><p class="display-inline" id="P153">Transfer the homogenate to a 1.5 mL Eppendorf tube and centrifuge at 2,000 x g for 10 minutes (min) at 4 °C to remove nucleus.</p>
</li>
<li>
<span class="label">7</span><p class="display-inline" id="P154">Take the clear supernatant and transfer to a 1.5 mL ultracentrifuge tube.</p>
<p id="P155">⮙ CRITICAL STEP Be careful when transferring the supernatant to avoid taking the layer of fat or lipid on the surface of the sample. The surface layer will be obvious when large amounts of cells are processed.</p>
</li>
<li>
<span class="label">8</span><p class="display-inline" id="P156">Ultracentrifuge samples at 200,000 x g for 45 min at 4 °C to separate the plasma membrane from the other cell compartments. Remove the supernatant.</p>
<p id="P157">⮙ CRITICAL STEP Tubes should be placed in the rotor in a consistent orientation. When removing supernatant, use a normal pipette tip to remove most of the supernatant and switch to a gel tip to remove the rest. The narrow and flexible tube of a gel tip will allow us to remove as much of the supernatant as possible without disturbing the pellet. Make sure that the tip does not disturb the pellet.</p>
</li>
<li>
<span class="label">9</span><p class="display-inline" id="P158">Resuspend the pellet in 0.2 M sodium carbonate solution to remove cell membrane-associated proteins. Pipette mix the pellet with a normal pipette tip to homogenize the mixture. Repeat the ultracentrifugation in step 8 using the same parameters. Remove the supernatant.</p>
</li>
<li>
<span class="label">10</span><p class="display-inline" id="P159">Resuspend the pellet in Milli-Q water to wash the pellet. Pipette mix the pellet to make the mixture homogenous. Repeat the ultracentrifugation in step 8 using the same parameters. Remove the supernatant.</p>
</li>
<li>
<span class="label">11</span><p class="display-inline" id="P160">Retain the cell membrane pellet for further analysis. Cell membrane pellet can be divided into two fractions for either N- and O-glycomics and glycolipidomics, or glycoproteomic analysis.</p>
</li>
</ul>
<p id="P161">▪ PAUSE POINT The pellet can be stored at −20 °C for several months until the glycomic or glycoproteomic analyses.</p></section><section id="S45"><h3 class="pmc_sec_title">Comprehensive N-glycomics: <em>N-Glycan release</em> ●Timing 20 h</h3>
<ul id="L22" class="list" style="list-style-type:none">
<li>
<span class="label">12</span><p class="display-inline" id="P162">Dissolve the Fraction I pellet in 100–200 μL of buffer (100 mM ammonium bicarbonate, 5 mM DTT) in an Eppendorf tube and break the pellet gently by pipette mixing.</p>
</li>
<li>
<span class="label">13</span><p class="display-inline" id="P163">Secure the lid with parafilm and denature the pellet by heating up to 100 °C for 10 s, followed by 10 s of cooling-down, with a total duration of 2 min..</p>
</li>
<li>
<span class="label">14</span><p class="display-inline" id="P164">Leave the sample to cool down to room temperature (which usually takes 2 min) and then add 2 μL of PNGase F enzyme and pipette mix.</p>
</li>
<li>
<span class="label">15</span><p class="display-inline" id="P165">Conduct N-glycan release by using the microwave to heat to 60 °C for 10 min. This is usually achieved by heating at 20 W.</p>
<p id="P166">⮙ CRITICAL STEP By the end of the incubation, the final temperature usually is 37 °C. If the desired temperature is not reached, continue the incubation until the temperature shown is 37 °C.</p>
</li>
<li>
<span class="label">16</span><p class="display-inline" id="P167">Keep the mixture in a 37 °C water bath overnight to completely hydrolyze primary amines to hydroxyl groups.</p>
</li>
<li>
<span class="label">17</span><p class="display-inline" id="P168">After the incubation, add 350 μL of Milli-Q water, and ultracentrifuge at 200,000 x g for 45 min at 4 °C, to separate the released N-glycans from the pellet. Transfer the supernatant to a new 1.5 mL Eppendorf tube.</p>
<p id="P169">⮙ CRITICAL STEP Use the gel tip to transfer the supernatant. Be careful when transferring to prevent the disturbance of the pellet.</p>
</li>
<li>
<span class="label">18</span><p class="display-inline" id="P170">Save the pellet for O-glycomics and glycolipidomics.</p>
</li>
</ul>
<p id="P171">▪ PAUSE POINT The pellet can be stored at −20 °C for several months until the glycolipid extraction.</p></section><section id="S46"><h3 class="pmc_sec_title">Comprehensive N-glycomics: <em>Desalting of N-glycans with PGC SPE plates</em> ●Timing 1–2 h</h3>
<ul id="L25" class="list" style="list-style-type:none">
<li>
<span class="label">19</span><p class="display-inline" id="P172">Add 200 μL of 80% (vol/vol) ACN, 0.1% (vol/vol) TFA solution to each well and centrifuge the plate at 150 x g for 1 min. Discard the flow-through. Repeat the step twice for conditioning.</p>
</li>
<li>
<span class="label">20</span><p class="display-inline" id="P173">Add 200 μL of pure Milli-Q water to each well and centrifuge the plate using the conditions given in step 19. Discard the flow-through. Repeat the step twice for equilibration.</p>
<p id="P174">? TROUBLESHOOTING</p>
</li>
<li>
<span class="label">21</span><p class="display-inline" id="P175">Load 200 μL of sample to each well of the plate until all the sample has been transfered. Centrifuge as described in step 19 and discard the flow-through.</p>
</li>
<li>
<span class="label">22</span><p class="display-inline" id="P176">After sample loading, desalt samples by adding 200 μL of pure Milli-Q water per well and centrifuging as described in step 19 three times. If the starting amount of cells is larger, samples can be washed for more than three times. Centrifuge and discard the flow-through in between each centrifugation.</p>
</li>
<li>
<span class="label">23</span><p class="display-inline" id="P177">For sample elution, add 200 μL of 40% (vol/vol) ACN, 0.05% (vol/vol) TFA to each well. Centrifuge as described in step 19 and collect the flow-through. Elute samples for two times (in total 400 μL). If the starting amount of cells is larger, elute sample with 200 μL of solvent.</p>
</li>
<li>
<span class="label">24</span><p class="display-inline" id="P178">Vacuum dry the eluted sample completely. This usually takes around 2–4 h.</p>
</li>
</ul>
<p id="P179">▪ PAUSE POINT The vacuum dried sample can be stored at −20 °C for several months until analysis.</p></section><section id="S48"><h3 class="pmc_sec_title">Comprehensive N-glycomics: <em>Nano-chip-QTOF-MS/MS analysis of N-glycans</em> ●Timing 1 h per sample</h3>
<ul id="L27" class="list" style="list-style-type:none">
<li>
<span class="label">25</span><p class="display-inline" id="P180">Reconstitute the sample with Milli-Q water. Start by reconstituting with 30 μL of water and adjust the amount accordingly. If the final TIC signal in the mass spectrometry is less than 1 × 10<sup>5</sup>, dry down the sample further to get higher concentration. Test the amount of water for reconstitution in this way for each cell line.</p>
</li>
<li>
<span class="label">26</span><p class="display-inline" id="P181">Transfer the sample into an injection vial and inject 5–10 μL of sample into the MS for analysis.</p>
<p id="P182">⮙ CRITICAL STEP To ensure sufficient sample is available to be injected, the minimum volume of reconstituted sample per well should be 10 μL.</p>
<p id="P183">⮙ CRITICAL STEP If a large set of samples are to be analyzed, a 96-well injection plate can be used.</p>
</li>
<li>
<span class="label">27</span><p class="display-inline" id="P184">Place injection vials or plates in the autosampler compartment. Close the door of the compartment properly.</p>
<p id="P185">⮙ CRITICAL STEP At the beginning of the batch, run 2–3 blank samples and one pooled N-glycan standard. For quality control (QC), run the pooled N-glycan standard and the blank after every 10 samples. At the end of the batch, flush the chip 2–3 times using the established flush method.</p>
<p id="P186">? TROUBLESHOOTING</p>
</li>
<li>
<span class="label">28</span><p class="display-inline" id="P187">Make a worklist in the “Worklist” tab or load the established worklist from an Excel file.</p>
</li>
<li>
<span class="label">29</span><p class="display-inline" id="P188">Load the freshly prepared glycan analysis solvents A and B to the pump. Change the solvent level from the “Nano Pump &gt; Bottle Fillings” window to make sure the software will not error out.</p>
<p id="P189">⮙ CRITICAL STEP The detailed N-glycan LC-MS analysis method is included in <a href="#T1" class="usa-link">Table 1</a>.</p>
</li>
<li>
<span class="label">30</span><p class="display-inline" id="P190">Load the Agilent PGC-Chip II into the chip cube. Switch the chip position to analysis position to make solvents flow from the nano pumps through both enrichment and analysis columns. Start the flow at 300 nL/min with 100% (vol/vol) solvent B to flush the chip for several min.</p>
<p id="P191">⮙ CRITICAL STEP Check the capillary pump and nano pump pressure. Make sure the capillary pump pressure is in the range of 60–90 bar, and the nano pump pressure is in the range of 30–50 bar.</p>
<p id="P192">? TROUBLESHOOTING</p>
</li>
<li>
<span class="label">31</span><p class="display-inline" id="P193">Observe the spray of the chip from the camera. If dripping or scattered spray is seen, adjust the horizontal position of the chip by screwing the wheel beneath the chip cub. The vertical position of the chip can be accessed from the acquisition software in the “ChipCub &gt; Adjust Tip” window. The spray can also be improved by changing the capillary voltage in the “Method” tab.</p>
<p id="P194">? TROUBLESHOOTING</p>
</li>
<li>
<span class="label">32</span><p class="display-inline" id="P195">Start the worklist after checking all the above steps.</p>
<p id="P196">? TROUBLESHOOTING</p>
</li>
<li>
<span class="label">33</span><p class="display-inline" id="P197">After finishing the batch, change the lids of any injection vials and save the vials directly at −20 °C. Seal the injection plate with the seal sheet. Wrap the plate with aluminum foil and store at −20 °C.</p>
<p id="P199">PAUSE POINT Data analysis can be conducted afterwards when needed.</p>
</li>
</ul>
<section class="tw xbox font-sm" id="T1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p id="P394">LC-MS/MS Data acquisition parameters for N-glycan, glycolipid, and O-glycan analyses.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="box" rules="all">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="center" valign="top" rowspan="1" colspan="1"></th>
<th align="center" valign="top" rowspan="1" colspan="1">N-glycomics</th>
<th align="center" valign="top" rowspan="1" colspan="1">O-glycomics</th>
<th align="center" valign="top" rowspan="1" colspan="1">Glycolipidomics</th>
</tr></thead>
<tbody>
<tr><td colspan="4" align="left" valign="top" rowspan="1">
<strong>LC Parameters</strong>
</td></tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Column Packing Material</td>
<td align="center" valign="middle" rowspan="1" colspan="1">PGC</td>
<td align="center" valign="middle" rowspan="1" colspan="1">PGC</td>
<td align="center" valign="middle" rowspan="1" colspan="1">C18</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Typical Injection volume</td>
<td align="center" valign="middle" rowspan="1" colspan="1">2 μL</td>
<td align="center" valign="middle" rowspan="1" colspan="1">5 μL</td>
<td align="center" valign="middle" rowspan="1" colspan="1">5 μL</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Solvent A (vol/vol)</td>
<td align="center" valign="middle" rowspan="1" colspan="1">0.1% FA, 3% ACN, and 96.9% H<sub>2</sub>O</td>
<td align="center" valign="middle" rowspan="1" colspan="1">0.1% FA, 3% ACN, and 96.9% H<sub>2</sub>O</td>
<td align="center" valign="middle" rowspan="1" colspan="1">0.1% acetic acid, 99.9% 20 mM ammonium acetate in H<sub>2</sub>O</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Solvent B (vol/vol)</td>
<td align="center" valign="middle" rowspan="1" colspan="1">1% FA, 89% ACN, and 10% H<sub>2</sub>O</td>
<td align="center" valign="middle" rowspan="1" colspan="1">1% FA, 89% ACN, and 10% H<sub>2</sub>O</td>
<td align="center" valign="middle" rowspan="1" colspan="1">0.1% acetic acid, 99.9% 20 mM ammonium acetate in 85:15 methanol/isopropanol (vol/vol)</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Flow rate (cap pump)</td>
<td colspan="3" align="center" valign="middle" rowspan="1">3 μL/min</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Flow rate (nano pump)</td>
<td colspan="3" align="center" valign="middle" rowspan="1">0.3 μL/min</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Gradient (%B (vol/vol))</td>
<td align="center" valign="middle" rowspan="1" colspan="1">0 min-2 min: 0%−0% <br>2 min-20 min: 0%−16% <br>20 min-40 min: 16%−72% <br>40 min-42 min: 72%−100% <br>42 min-52 min: 100%−100% <br>52 min-54 min: 100%−0% <br>54 min-65 min: 0%−0%</td>
<td align="center" valign="middle" rowspan="1" colspan="1">0 min-2 min: 0%−0% <br>2 min-20 min: 0%−16% <br>20 min-40 min: 16%−72% <br>40 min-42 min: 72%−100% <br>42 min-52 min: 100%−100% <br>52 min-54 min: 100%−0% <br>54 min-65 min: 0%−0%</td>
<td align="center" valign="middle" rowspan="1" colspan="1">0 min-4 min: 70%−85% <br>4 min-40 min: 85%−100% <br>40 min-50 min: 100%−100% <br>50 min-52 min: 100%−70% <br>52 min-65 min: 70%−70% <br>
</td>
</tr>
<tr><td colspan="4" align="left" valign="middle" rowspan="1">
<strong>MS Parameters</strong>
</td></tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">
<em>m/z</em> mass range</td>
<td align="center" valign="middle" rowspan="1" colspan="1">600–2000</td>
<td align="center" valign="middle" rowspan="1" colspan="1">300–2000</td>
<td align="center" valign="middle" rowspan="1" colspan="1">600–2000</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Polarity</td>
<td colspan="3" align="center" valign="middle" rowspan="1">Positive</td>
</tr>
<tr>
<td align="left" valign="middle" rowspan="1" colspan="1">Gas Temperature</td>
<td colspan="3" align="center" valign="middle" rowspan="1">325 °C</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Capillary Voltage</td>
<td colspan="3" align="center" valign="middle" rowspan="1">1,700–2,000 V (N-glycans, O-glycans)<br>1,400–1,700 (glycolipids)</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Cycle Time</td>
<td colspan="3" align="center" valign="middle" rowspan="1">5.25 s</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">MS Scan Rate</td>
<td colspan="3" align="center" valign="middle" rowspan="1">0.8 spectra/s (1250 ms/spectrum)</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">MS/MS Scan Rate</td>
<td colspan="3" align="center" valign="middle" rowspan="1">1.0 spectra/s (1000 ms/spectrum)</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">MS threshold</td>
<td colspan="3" align="center" valign="middle" rowspan="1">Abs. threshold 200; Rel. threshold 0.01%</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">MS/MS <br>threshold</td>
<td colspan="3" align="center" valign="middle" rowspan="1">Abs. threshold 200; Rel. threshold 0.01%</td>
</tr>
<tr>
<td align="left" valign="middle" rowspan="1" colspan="1">Calibrated Mass</td>
<td colspan="3" align="center" valign="middle" rowspan="1">1221.990637</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Acquisition Mode</td>
<td colspan="3" align="center" valign="middle" rowspan="1">Auto MS/MS (Seg)</td>
</tr>
<tr>
<td align="left" valign="middle" rowspan="1" colspan="1">Activation Type</td>
<td colspan="3" align="center" valign="middle" rowspan="1">CID</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Activation Energy</td>
<td colspan="3" align="center" valign="middle" rowspan="1">V<sub>collision</sub>=1.8 * (<em>m/z</em>)/100 – 2.4 (N-glycans, O-glycans)<br>V<sub>collision</sub>=1.2 * (<em>m/z</em>) /100 + 12 (Glycolipids)</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Max Precursors per Cycle</td>
<td colspan="3" align="center" valign="middle" rowspan="1">4</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Precursor Selection Threshold (Abs)</td>
<td colspan="3" align="center" valign="middle" rowspan="1">1,000</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Precursor Target Counts/Spectrum</td>
<td colspan="3" align="center" valign="middle" rowspan="1">25,000 counts/spectrum</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Dynamic Exclusion</td>
<td colspan="3" align="center" valign="middle" rowspan="1">Excluded after 1 spectrum, released after 0.35 min</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Precursor charge state preference</td>
<td colspan="3" align="center" valign="middle" rowspan="1">2 &gt; 1 &gt; 3 &gt; unknown charge &gt; larger than 3 charges</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Isotope Model</td>
<td colspan="3" align="center" valign="middle" rowspan="1">Common organic molecules</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="S53"><h3 class="pmc_sec_title">Comprehensive N-glycomics: <em>N-Glycomics data analysis</em> ●Timing 1–2 h per batch</h3>
<ul id="L34" class="list" style="list-style-type:none">
<li>
<span class="label">34</span><p class="display-inline" id="P200">Start the Agilent MassHunter Qualitative Analysis B.08.00 software.</p>
</li>
<li>
<span class="label">35</span><p class="display-inline" id="P201">Open the data file in the software.</p>
</li>
<li>
<span class="label">36</span><p class="display-inline" id="P202">Use the “Find Compounds by Molecular Feature” function to survey the N-glycans present in the sample. Set up the extraction parameters in the window. Use the underivatized N-glycan library for extraction. An underivatized N-glycan library can be established referring to a previous study<sup><a href="#R89" class="usa-link" aria-describedby="R89">89</a></sup>. The template of an N-glycan library is included as <a href="#SD1" class="usa-link">Supplementary Data I_N-glycan_lib_template.csv</a>. More N-glycans can be added to the template based on ref. <a href="#R89" class="usa-link" aria-describedby="R89">89</a> and used directly for database search.</p>
<p id="P203">⮙ CRITICAL STEP The N-glycan library is a csv file. It contains columns of Mass, Cpd (glycan compositions), and Comments (types of N-glycans). The formulas of N-glycans can also be added to the Formula column in the library. The detailed data analysis method is summarized in <a href="#T2" class="usa-link">Table 2</a>.</p>
<p id="P204">⮙ CRITICAL STEP The composition of an N-glycan compound is noted as the number of each monosaccharide [ABCD] in the order of Hex_HexNAc_Fuc_NeuAc. More numbers can be included if other groups are contained in the composition, such as NeuGc and sulfate groups.</p>
</li>
<li>
<span class="label">37</span><p class="display-inline" id="P205">Start the extraction of compounds by clicking the green Play/Run button and choose the files to run the Molecular Feature Extraction. The program will return a list of compounds that match the set criteria. The extracted results include the total compound chromatogram (TCC) and extracted compound chromatogram (ECC).</p>
</li>
<li>
<span class="label">38</span><p class="display-inline" id="P206">Reset colors of extracted compound peaks based on the following criteria: Change all the fucosylated compound peaks to green, sialylated compound peaks to pink, sialylfucosylated compound peaks to blue, complex or hybrid compound peaks to orange, and high-mannose compound peaks to red.</p>
</li>
<li>
<span class="label">39</span><p class="display-inline" id="P207">Add the compound labels to the ECC.</p>
</li>
<li>
<span class="label">40</span><p class="display-inline" id="P208">Save ECC, both versions with compound labels and without compound labels, as figures.</p>
</li>
<li>
<span class="label">41</span><p class="display-inline" id="P209">Check the MS/MS spectra to identify fragmentations of N-glycans in the “MS Spectrum Results” window. Their fragmentation spectra will have the characteristic masses for monosaccharides and oligosaccharides.</p>
<p id="P210">⮙ CRITICAL STEP Fragmentations should be checked when analyzing glycans containing unnatural monosaccharides, such as SiaNAz and GalNAz. Care should also be taken when analyzing non-human glycan samples, where the existence of NeuGc is possible. The glycan with the composition of 1 NeuGc + x Hex + y Fuc has the same mass as the glycan with the composition of 1 NeuAc + (x+1) Hex + (y-1) Fuc.</p>
</li>
<li>
<span class="label">42</span><p class="display-inline" id="P211">Export extracted N-glycan compounds as csv files, and curate the extracted data as needed.</p>
</li>
<li>
<span class="label">43</span><p class="display-inline" id="P212">Calculate the relative abundances of each compound by summarizing the abundances of isomers or compounds with different charge states of one N-glycan composition.</p>
</li>
</ul>
<section class="tw xbox font-sm" id="T2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p id="P395">Data analysis parameters using Agilent MassHunter Qualitative Analysis B.08.00 software.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="box" rules="all">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="center" valign="top" rowspan="1" colspan="1">Tabs</th>
<th align="center" valign="top" rowspan="1" colspan="1">Checked Parameters</th>
<th align="center" valign="top" rowspan="1" colspan="1">Values</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="3" align="center" valign="middle" colspan="1">Extraction</td>
<td align="center" valign="middle" rowspan="1" colspan="1">Use peaks with height</td>
<td align="center" valign="middle" rowspan="1" colspan="1">≥ 100 counts</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Extraction algorithm Target data type</td>
<td align="center" valign="middle" rowspan="1" colspan="1">Small molecules (chromatographic)</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Restrict retention time</td>
<td align="center" valign="middle" rowspan="1" colspan="1">0–40 min</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Ion Species</td>
<td align="center" valign="middle" rowspan="1" colspan="1">Positive ions</td>
<td align="center" valign="middle" rowspan="1" colspan="1">+H (N-glycans, O-glycans) <br>+H, +NH<sub>4</sub> (glycolipids)</td>
</tr>
<tr>
<td rowspan="3" align="center" valign="middle" colspan="1">Charge State</td>
<td align="center" valign="middle" rowspan="1" colspan="1">Peak spacing tolerance</td>
<td align="center" valign="middle" rowspan="1" colspan="1">0.0025 <em>m/z</em>, 7.0 ppm</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Isotope model</td>
<td align="center" valign="middle" rowspan="1" colspan="1">Common organic molecules, no halogens</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Limit assigned charge states to a maximum</td>
<td align="center" valign="middle" rowspan="1" colspan="1">3</td>
</tr>
<tr>
<td rowspan="2" align="center" valign="middle" colspan="1">Compound filters</td>
<td align="center" valign="middle" rowspan="1" colspan="1">Absolute Height</td>
<td align="center" valign="middle" rowspan="1" colspan="1">≥ 1,000</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Compound quality score</td>
<td align="center" valign="middle" rowspan="1" colspan="1">≥ 30</td>
</tr>
<tr>
<td rowspan="2" align="center" valign="middle" colspan="1">Mass Filters</td>
<td align="center" valign="middle" rowspan="1" colspan="1">Filter mass list</td>
<td align="center" valign="middle" rowspan="1" colspan="1">20 ppm</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Database</td>
<td align="center" valign="middle" rowspan="1" colspan="1">corresponding libraries</td>
</tr>
<tr>
<td rowspan="4" align="center" valign="middle" colspan="1">Results</td>
<td colspan="2" align="center" valign="middle" rowspan="1">Delete previous compounds</td>
</tr>
<tr><td colspan="2" align="center" valign="middle" rowspan="1">Highlight first compound</td></tr>
<tr><td colspan="2" align="center" valign="middle" rowspan="1">Extract ECC</td></tr>
<tr><td colspan="2" align="center" valign="middle" rowspan="1">Extract MS/MS spectrum</td></tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="S54"><h3 class="pmc_sec_title">Comprehensive glycolipidomics: <em>Glycolipid extraction</em> ●Timing 1 h</h3>
<ul id="L37" class="list" style="list-style-type:none">
<li>
<span class="label">44</span><p class="display-inline" id="P213">Resuspend the pellet after N-glycan releasing (from step 17) with 500 μL of Folch solvent and pipette mix the solution thoroughly to fully dissolve the lipids.</p>
<p id="P214">⮙ CRITICAL STEP Sonicate the samples for 15 min followed by 1-minute vortexing if necessary.</p>
</li>
<li>
<span class="label">45</span><p class="display-inline" id="P215">Centrifuge at 8,800 x g for 5 min to separate the pelletized proteins from the dissolved lipids in the supernatant. Collect the supernatant and transfer it to a new Eppendorf tube.</p>
<p id="P216">? TROUBLESHOOTING</p>
<p id="P217">⮙ CRITICAL STEP The pellets, which contain mostly precipitated proteins, can be set aside for O-glycan release (step 73).</p>
</li>
<li>
<span class="label">46</span><p class="display-inline" id="P218">Add internal standards to the supernatant if desired. About 1 nmol of each labeled glycolipid is sufficient for detection and quantitation. For example, add 20 μL of a solution of 50 μM N-omega-CD3-Octadecanoyl monosialoganglioside GM1 and 50 μM N-omega-CD3-Octadecanoyl disialoganglioside GD3 dissolved in methanol.</p>
</li>
<li>
<span class="label">47</span><p class="display-inline" id="P219">Add 100 μL of 0.1 M potassium chloride to the supernatant and vortex mix for 1 min. Turbidity may be observed due to phase separation and can be ignored.</p>
</li>
<li>
<span class="label">48</span><p class="display-inline" id="P220">Separate the aqueous and organic phases by centrifuging at 8,800 x g for 5 min. The upper methanol and aqueous layer will contain glycosphingolipids with 3 or more monosaccharide residues. The bottom chloroform-rich layer will contain mostly phospholipids, cholesterol, and other less polar lipids. A middle disk-shaped layer of phospholipids may form during this step.</p>
<p id="P221">? TROUBLESHOOTING</p>
</li>
<li>
<span class="label">49</span><p class="display-inline" id="P222">Carefully collect the upper layer and dry the sample in vacuum, which takes 2–3 h.</p>
</li>
</ul></section><section id="S57"><h3 class="pmc_sec_title">Comprehensive glycolipidomics: <em>Desalting of glycolipids with C8 SPE plates</em> ●Timing 1–2 h</h3>
<ul id="L40" class="list" style="list-style-type:none">
<li>
<span class="label">50</span><p class="display-inline" id="P223">Wash a C8 SPE plate twice with 600 μL of 1:1 methanol/isopropanol (vol/vol) by centrifuging at 150 x g.</p>
</li>
<li>
<span class="label">51</span><p class="display-inline" id="P224">Condition the plate by washing twice with 600 μL of 1:1 methanol/water (vol/vol).</p>
</li>
<li>
<span class="label">52</span><p class="display-inline" id="P225">Reconstitute the dried samples from step 49 with 600 μL of 1:1 methanol/water and load the sample on the plate.</p>
</li>
<li>
<span class="label">53</span><p class="display-inline" id="P226">Wash the plate at least 3 times with 600 μL of 1:1 methanol/water (vol/vol).</p>
<p id="P227">⮙ CRITICAL STEP Washing volume can be varied between samples. If a high salt content is expected, use a greater volume for washing.</p>
</li>
<li>
<span class="label">54</span><p class="display-inline" id="P228">Elute the glycolipids from the plate using 600 μL of 1:1 methanol/isopropanol (vol/vol).</p>
</li>
<li>
<span class="label">55</span><p class="display-inline" id="P229">Dry the samples completely in vacuum, taking around 2–3 h.</p>
</li>
</ul>
<p id="P230">▪ PAUSE POINT The vacuum dried sample can be stored at −20 °C for several months until analysis.</p></section><section id="S58"><h3 class="pmc_sec_title">Comprehensive glycolipidomics: <em>Nano-chip-QTOF-MS/MS analysis of glycolipids</em> ●Timing 1 h per sample</h3>
<ul id="L42" class="list" style="list-style-type:none"><li>
<span class="label">56</span><p class="display-inline" id="P231">For glycolipidomic analysis using nano-chip-QTOF, reconstitute the sample with 50 μL of 1:1 (vol/vol) methanol/water and inject around 5 μL of sample into the instrument.</p>
<p id="P232">⮙ CRITICAL STEP More detailed instrument parameters for glycolipidomics are included in <a href="#T1" class="usa-link">Table 1</a>.</p>
<p id="P233">⮙ CRITICAL STEP For quality control (QC), the pooled glycolipid samples from the pluripotent cell line NTERA-2 is used.</p>
</li></ul>
<p id="P234">▪ PAUSE POINT Data analysis can be conducted afterwards when needed.</p></section><section id="S59"><h3 class="pmc_sec_title">Comprehensive glycolipidomics: <em>Glycolipidomics - data analysis</em> ●Timing 1–2 h per batch</h3>
<ul id="L44" class="list" style="list-style-type:none">
<li>
<span class="label">57</span><p class="display-inline" id="P235">Start the Agilent MassHunter Qualitative Analysis B.08.00 software.</p>
</li>
<li>
<span class="label">58</span><p class="display-inline" id="P236">Open the data file in the software.</p>
</li>
<li>
<span class="label">59</span><p class="display-inline" id="P237">Manually identify the glycans and ceramides from the MS/MS data to generate a library of glycosphingolipids. If the glycan compositions and the associated lipids in the sample are already known, skip ahead to generate a glycosphingolipid library for data extraction.</p>
<p id="P238">⮙ CRITICAL STEP Using the full library directly can lead to higher rates of false identification because the sheer number of possible glycolipid combinations result in many compounds having similar masses.</p>
</li>
<li>
<span class="label">60</span><p class="display-inline" id="P239">Use the “Find Compounds by Molecular Feature” function to survey the glycolipids present in the sample. Set up the parameters as <a href="#T2" class="usa-link">Table 2</a> to filter through the identified compound features.</p>
</li>
<li>
<span class="label">61</span><p class="display-inline" id="P240">Start the extraction of compounds by clicking the green Play/Run button and choose the files to run the Molecular Feature Extraction. The program will return a list of compounds that match the set criteria. (If list does not appear, go to View &gt; Compound List. To customize the data shown in the list, right-click on the column header and selecting Add/Remove Columns.)</p>
</li>
<li>
<span class="label">62</span><p class="display-inline" id="P241">Check the MS/MS spectra to identify glycosphingolipids in the “MS Spectrum Results” window. Their fragmentation spectra will have the characteristic masses for monosaccharides, oligosaccharides, and ceramides. Other observable fragments include the sphingoid fragment and combinations of a ceramide with oligosaccharides.</p>
<p id="P242">⮙ CRITICAL STEP MS/MS spectra that have abundant 184.07 <em>m/z</em> fragments are likely to be phospholipids. This <em>m/z</em> signal is from the phosphocholine head group.</p>
</li>
<li>
<span class="label">63</span><p class="display-inline" id="P243">From spectra of possible glycosphingolipids, take note of the precursor <em>m/z</em> from the MS1 spectrum, the charge state of the precursor, and the ceramide <em>m/z</em> in the MS/MS spectrum. A table of possible ceramide <em>m/z</em> is provided in the file <a href="#SD5" class="usa-link">Supplementary Data II_Ceramide_mz.csv</a>. Some commonly observed ceramides in mammalian cells include d34:1, d36:1, d42:1, and d42:2. Critical: MS parameters must be optimized for glycolipid fragmentation.</p>
</li>
<li>
<span class="label">64</span><p class="display-inline" id="P244">Input the precursor <em>m/z</em>, precursor charge state, and the ceramide <em>m/z</em> in the appropriate fields (highlighted light blue) in the file <a href="#SD2" class="usa-link">Supplementary Data III_GSL_glycan_finder.xlsx</a>. Include at least 2 decimal places for better matching. The Excel file calculations take into account possible proton, ammonium, and sodium adducts to match the closest glycan composition that fits the inputs. A good match should have a PPM Error of less than 10. Cross-check the given glycan composition with the fragmentation data to confirm the identification. Some common saccharide and lipid fragments are described in the file <a href="#SD4" class="usa-link">Supplementary Data IV_Saccharide_mz.csv</a>.</p>
</li>
<li>
<span class="label">65</span><p class="display-inline" id="P245">Generate a list of the identified ceramides and glycan compositions found through this process.</p>
</li>
<li>
<span class="label">66</span><p class="display-inline" id="P246">Generate a database of glycosphingolipids by using the Pivot table in the file <a href="#SD3" class="usa-link">Supplementary Data V_GSL_lib_generator.xlsx</a>. Select the glycan compositions and identified lipids in the appropriate drop-down menus. Alternatively, the ceramides can also be filtered based on the number of carbons, number of hydroxyl groups, and the degree of unsaturation.</p>
</li>
<li>
<span class="label">67</span><p class="display-inline" id="P247">Copy the information into a new csv file, taking care to follow the format that is readable by the software.</p>
</li>
<li>
<span class="label">68</span><p class="display-inline" id="P248">Extract the glycosphingolipid data with the newly-generated database using the “Find Compounds by Molecular Feature”.</p>
</li>
<li>
<span class="label">69</span><p class="display-inline" id="P249">Reset colors of extracted glycolipid peaks based on the following criteria: Change all the fucosylated compound peaks to green, sialylated compound peaks to pink, sialofucosylated compound peaks to blue, undecorated compound peaks to orange, sulfated compound peaks to red, and globo compound peaks to black.</p>
</li>
<li>
<span class="label">70</span><p class="display-inline" id="P250">Add the compound labels to the ECC.</p>
</li>
<li>
<span class="label">71</span><p class="display-inline" id="P251">Save the ECC, both of versions with compound labels and without compound labels, as figures.</p>
</li>
<li>
<span class="label">72</span><p class="display-inline" id="P252">Export extracted glycolipid compounds as csv files, and curate the extracted data as needed.</p>
</li>
</ul></section><section id="S60"><h3 class="pmc_sec_title">Comprehensive O-glycomics:</h3>
<section id="S61"><h4 class="pmc_sec_title">Comprehensive O-glycomics: <em>O-Glycan release</em> ●Timing 1 d</h4>
<ul id="L47" class="list" style="list-style-type:none">
<li>
<span class="label">73</span><p class="display-inline" id="P253">Take the pellet saved from Folch extraction (step 44) and transfer it to a 4 mL cryovial tube.</p>
</li>
<li>
<span class="label">74</span><p class="display-inline" id="P254">Reconstitute the pellet with 90 μL of Milli-Q water and sonicate for 10 min to break up the piece.</p>
</li>
<li>
<span class="label">75</span><p class="display-inline" id="P255">Add 10 μL of 2 M NaOH and 100 μL of 2 M NaBH<sub>4</sub> sequencially and vortex the mixture for 10 s.</p>
<p id="P256">? TROUBLESHOOTING</p>
</li>
<li>
<span class="label">76</span><p class="display-inline" id="P257">Incubate samples at 45 °C in a water bath for 18 h.</p>
<p id="P258">⮙ CRITICAL STEP During the first hour of incubation, vortex the mixture and release the generated gas twice to avoid over pressure in the tube.</p>
</li>
<li>
<span class="label">77</span><p class="display-inline" id="P259">Add 90–110 μL of acetic acid to the mixture to neutralize the solution.</p>
<p id="P260">⮙ CRITICAL STEP While adding the acid, place samples on ice to prevent over-heating of the mixture.</p>
<p id="P261">? TROUBLESHOOTING</p>
</li>
<li>
<span class="label">78</span><p class="display-inline" id="P262">Check the pH values of samples with pH paper. The final pH values of samples are 4–6.</p>
</li>
<li>
<span class="label">79</span><p class="display-inline" id="P263">Centrifuge samples at 21,000 g for 30 min. Retain the supernatant and discard the pellet.</p>
</li>
</ul></section><section id="S64"><h4 class="pmc_sec_title">Comprehensive O-glycomics: <em>Desalting of O-glycans with PGC SPE plates</em> ●Timing 1–2 h</h4>
<ul id="L50" class="list" style="list-style-type:none"><li>
<span class="label">80</span><p class="display-inline" id="P264">Conduct SPE clean-up by following steps 19–24 to desalt. Dry samples completely in a vacuum.</p>
</li></ul></section><section id="S65"><h4 class="pmc_sec_title">Comprehensive O-glycomics: <em>Enrichment of O-glycans with HILIC cartridges</em> ●Timing 1–2 h</h4>
<ul id="L51" class="list" style="list-style-type:none">
<li>
<span class="label">81</span><p class="display-inline" id="P265">Reconstitute the dried O-glycan samples with 1mL of 90% (vol/vol) ACN, 1% (vol/vol) TFA solution.</p>
</li>
<li>
<span class="label">82</span><p class="display-inline" id="P266">Add 1 mL of ACN to the iSPE HILIC cartridge to wet the material. Discard the flow-through. To speed up the procedure, a pump or vacuum manifold can be used.</p>
</li>
<li>
<span class="label">83</span><p class="display-inline" id="P267">Add 1 mL of water with 0.1% (vol/vol) TFA to the cartridge for the conditioning. Discard the flow-though. Repeat the procedure one more time. Pump the liquid to speed up the procedure.</p>
</li>
<li>
<span class="label">84</span><p class="display-inline" id="P268">Equilibrate the cartridge with 1 mL of 90% (vol/vol) ACN, 1% (vol/vol) TFA solution and discard the flow-through. Repeat the procedure twice.</p>
</li>
<li>
<span class="label">85</span><p class="display-inline" id="P269">Load the sample to the cartridge and let the sample flow through the cartridge slowly (2 drops per sec). Collect the flow-through and reload it to the cartridge.</p>
</li>
<li>
<span class="label">86</span><p class="display-inline" id="P270">Reload the sample as described in previous step 5 times.</p>
</li>
<li>
<span class="label">87</span><p class="display-inline" id="P271">Elute the sample by adding 1 mL of water with 0.1% (vol/vol) TFA. Repeat 3 times.</p>
</li>
<li>
<span class="label">88</span><p class="display-inline" id="P272">Dry samples completely in vacuum.</p>
</li>
</ul>
<p id="P273">▪ PAUSE POINT The vacuum dried sample can be stored at −20 °C for several months until analysis.</p></section><section id="S66"><h4 class="pmc_sec_title">Comprehensive O-glycomics: <em>Nano-chip-QTOF-MS/MS analysis of O-glycans</em> ●Timing 1 h per sample</h4>
<ul id="L52" class="list" style="list-style-type:none"><li>
<span class="label">89</span><p class="display-inline" id="P274">For nano-chip-QTOF-MS/MS analysis, reconstitute the sample with around 90 μL of water and inject 5 μL of sample into the instrument. The volume of water for reconstitution and injection can be adjusted accordingly. Repeat steps 25–33 on this sample.</p>
<p id="P275">⮙ CRITICAL STEP The detailed O-glycan analysis LC-MS method is included in <a href="#T1" class="usa-link">Table 1</a>.</p>
<p id="P276">⮙ CRITICAL STEP For quality control, the pooled released O-glycans from several cell lines under the normal condition (Caco-2, PNT2, and A549) can be used.</p>
</li></ul>
<p id="P277">▪ PAUSE POINT Data analysis can be conducted afterwards when needed.</p></section><section id="S67"><h4 class="pmc_sec_title">Comprehensive O-glycomics: <em>O-Glycomics data analysis</em> ●Timing 1–2 h per batch</h4>
<ul id="L54" class="list" style="list-style-type:none">
<li>
<span class="label">90</span><p class="display-inline" id="P278">Start the Agilent Qualitative Analysis B.08.00 software.</p>
</li>
<li>
<span class="label">91</span><p class="display-inline" id="P279">Open the data file in the software.</p>
</li>
<li>
<span class="label">92</span><p class="display-inline" id="P280">Use the “Find Compounds by Molecular Feature” function to survey the O-glycans present in the sample. Set up the parameters as <a href="#T2" class="usa-link">Table 2</a> to filter through the identified compound features.</p>
<p id="P281">⮙ CRITICAL STEP The O-glycan library is a csv file. It contains the masses, compositions, and types of reduced O-glycans. The formulas of these O-glycans can also be added to the library.</p>
</li>
<li>
<span class="label">93</span><p class="display-inline" id="P282">Start the extraction of compounds by clicking the green Play/Run button and choose the files to run the Molecular Feature Extraction. The program will return a list of compounds that match the set criteria. The extracted results include the total compound chromatogram (TCC) and extracted compound chromatogram (ECC).</p>
</li>
<li>
<span class="label">94</span><p class="display-inline" id="P283">Reset colors of extracted O-glycan peaks based on the following criteria: Change all the fucosylated compound peaks to green, sialylated compound peaks to pink, sialofucosylated compound peaks to blue, and undecorated compound peaks to orange.</p>
</li>
<li>
<span class="label">95</span><p class="display-inline" id="P284">Add the compound labels to the ECC.</p>
</li>
<li>
<span class="label">96</span><p class="display-inline" id="P285">Check the MS/MS spectra to identify fragmentations of O-glycans in the “MS Spectrum Results” window. Their fragmentation spectra will have the characteristic masses for monosaccharides and oligosaccharides.</p>
<p id="P286">⮙ CRITICAL STEP Fragmentations should always be checked when analyzing glycans containing unnatural monosaccharides, such as SiaNAz and GalNAz. Care should also be taken when analyzing non-human glycan samples, where the existence of NeuGc is possible. The glycan with the composition of 1 NeuGc + x Hex + y Fuc has the same mass as the glycan with the composition of 1 NeuAc + (x+1) Hex + (y-1) Fuc.</p>
</li>
<li>
<span class="label">97</span><p class="display-inline" id="P287">Save the ECC, both as versions with compound labels and without compound labels, as figures.</p>
</li>
<li>
<span class="label">98</span><p class="display-inline" id="P288">Export extracted O-glycan compounds as csv files, and curate the extracted data as needed.</p>
</li>
<li>
<span class="label">99</span><p class="display-inline" id="P289">Calculate the relative abundances of each compound by summarizing the abundances of isomers or compounds with different charge states of one O-glycan composition.</p>
</li>
</ul></section><section id="S68"><h4 class="pmc_sec_title">
<em>Site-specific glycoproteomic analysis: Protein digestion</em> ●Timing 20 h</h4>
<ul id="L57" class="list" style="list-style-type:none">
<li>
<span class="label">100</span><p class="display-inline" id="P290">Resuspend cell membrane pellet from Fraction II (step 11) using 60 μL of freshly made 8 M urea.</p>
<p id="P291">⮙ CRITICAL STEP Pipette mix the solution to fully resuspend the cell pellet.</p>
</li>
<li>
<span class="label">101</span><p class="display-inline" id="P292">Sonicate the solution for 10–15 min. Measure the protein concentration by BCA assay.</p>
<p id="P293">⮙ CRITICAL STEP The amount of DTT, IAA, and trypsin to be used in the following steps (102–104), depends on the initial sample concentration determined by BCA assay. 1 mg/mL protein sample requires 2 μL of DTT, 4 μL of IAA, and 2 μg of trypsin.</p>
</li>
<li>
<span class="label">102</span><p class="display-inline" id="P294">Add 2 μL of 450 mM DTT and pipette well to mix the samples. Use parafilm to secure the lids and incubate at 55 °C in a water bath for 50 min.</p>
<p id="P295">⮙ CRITICAL STEP Higher temperatures (&gt;60 °C) will cause urea-based carbamylation at the protein N terminals and at the side chain amine groups of lysine and arginine residues, which further blocks the protease digestion and affects protein identification and quantification.</p>
<p id="P296">⮙ CRITICAL STEP Incubation with DTT &amp; IAA, i.e. this and the next step, is not needed if the protein does not contain disulfide bonds.</p>
</li>
<li>
<span class="label">103</span><p class="display-inline" id="P297">Add 4 μL of 550 mM IAA, pipette mix well the samples, and incubate at room temperature for 20 min.</p>
<p id="P298">⮙ CRITICAL STEP IAA is light sensitive, so samples should be placed in a dark drawer/area.</p>
</li>
<li>
<span class="label">104</span><p class="display-inline" id="P299">Add 420 μL of protein digestion buffer to dilute the urea concentration to less than 8 M. Reconstitute 20 μg of sequencing-grade modified trypsin in freshly made 200 μl of protein digestion buffer and then add 20 μL of trypsin to the sample. Pippetting to mix samples well.</p>
<p id="P300">⮙ CRITICAL STEP Do not vortex mix the trypsin which may reduce the activity of the enzyme.</p>
</li>
<li>
<span class="label">105</span><p class="display-inline" id="P301">Seal tubes with parafilm, and place samples in a 37 °C water bath and incubate for 18 h.</p>
</li>
<li>
<span class="label">106</span><p class="display-inline" id="P302">After the incubation, quench the reaction with 10 μL of 18% (vol/vol) formic acid.</p>
</li>
<li>
<span class="label">107</span><p class="display-inline" id="P303">Dry the samples in vacuum, which usually takes 2–3 h. After complete drying, the samples are ready for glycopeptide enrichment.</p>
<p id="P304">⮙ CRITICAL STEP Do not use a high-temperature for drying glycopeptide samples to avoid the denaturation or undesired modifications of glycopeptides.</p>
</li>
</ul>
<p id="P305">▪ PAUSE POINT The completely dried sample can be stored at −20 °C for several days until the HILIC enrichment.</p></section><section id="S69"><h4 class="pmc_sec_title">
<em>Site-specific glycoproteomic analysis: Enrichment of glycopeptides with HILIC SPE cartridges</em> ●Timing 2–3 h</h4>
<ul id="L64" class="list" style="list-style-type:none">
<li>
<span class="label">108</span><p class="display-inline" id="P306">Wash iSPE-HILIC SPE cartridges with 1 mL of 99.9% (vol/vol) ACN, 0.1% (vol/vol) TFA. To speed up the procedure, a pump or vacuum manifold can be used.</p>
</li>
<li>
<span class="label">109</span><p class="display-inline" id="P307">Wash the cartridges with 2 mL of 99.9% (vol/vol) H<sub>2</sub>O, 0.1% (vol/vol) TFA.</p>
</li>
<li>
<span class="label">110</span><p class="display-inline" id="P308">Equilibrate the cartridges with 3 mL of 80% (vol/vol) ACN, 19% (vol/vol) H<sub>2</sub>O, 1% (vol/vol) TFA.</p>
</li>
<li>
<span class="label">111</span><p class="display-inline" id="P309">Reconstitute the samples with 1 mL of 80% (vol/vol) ACN, 19% (vol/vol) H<sub>2</sub>O, 1% (vol/vol) TFA, and load the samples to the cartridges.</p>
<p id="P310">? TROUBLESHOOTING</p>
</li>
<li>
<span class="label">112</span><p class="display-inline" id="P311">Reload the flow-through from step 111 5 times.</p>
</li>
<li>
<span class="label">113</span><p class="display-inline" id="P312">Wash the cartridges with 5 mL of 80% (vol/vol) ACN, 19% (vol/vol) H<sub>2</sub>O, 1% (vol/vol) TFA.</p>
<p id="P313">⮙ CRITICAL STEP Collect and save the flow-through from this step for proteomic analysis if necessary.</p>
</li>
<li>
<span class="label">114</span><p class="display-inline" id="P314">Elute the glycopeptides with 2.5 mL of 99.9% (vol/vol) H<sub>2</sub>O, 0.1% (vol/vol) TFA.</p>
</li>
<li>
<span class="label">115</span><p class="display-inline" id="P315">Dry the samples in vacuum completely for 3–5 h for glycopeptide analysis.</p>
</li>
</ul></section><section id="S71"><h4 class="pmc_sec_title">
<em>Site-specific glycoproteomic analysis: Nano-LC-MS/MS analysis of Glycopeptides</em> ●Timing 4 h per sample with one blank injection</h4>
<ul id="L66" class="list" style="list-style-type:none">
<li>
<span class="label">116</span><p class="display-inline" id="P316">Reconstitute the samples with 20 μL of H<sub>2</sub>O, use Pierce<sup>™</sup> Quantitative Colorimetric Peptide Assay following the manufacturer’s instructions to measure the glycopeptide concentration, and adjust the concentration to 1 μg/μL.</p>
<p id="P317">⮙ CRITICAL STEP If the reconstituted sample is opaque or contains some visible undissolvable precipitates, the sample should be spin down briefly for 15 s and only the supernatant can be injected to the instrument.</p>
</li>
<li>
<span class="label">117</span><p class="display-inline" id="P318">Run glycoproteomic anaylsis. We use an UltiMate<sup>™</sup> WPS-3000RS nanoLC system coupled to the Nanospray Flex ion source of an Orbitrap Fusion Lumos. We use a C18 column (Acclaim<sup>™</sup> PepMap<sup>™</sup> 100 C18 LC Column, 3 μm, 0.075 mm x 150 mm) for glycopeptide separation. Details of the method are summarized in <a href="#T3" class="usa-link">Table 3</a>.</p>
<p id="P319">⮙ CRITICAL STEP After every sample run, the column should be cleaned by running a blank injection to avoid sample carry-over.</p>
<p id="P320">⮙ CRITICAL STEP A digested commercial serum N-glycopeptide sample can be used for quality control (QC) of LC-MS.</p>
</li>
</ul>
<section class="tw xbox font-sm" id="T3"><h5 class="obj_head">Table 3.</h5>
<div class="caption p"><p id="P396">LC-MS/MS data acquisition parameters for glycopeotide analysis.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="box" rules="cols">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<tbody>
<tr><td colspan="2" align="left" valign="top" rowspan="1">
<strong>LC Parameters</strong>
</td></tr>
<tr><td colspan="2" align="left" valign="top" rowspan="1">
<hr>
</td></tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Injection volume</td>
<td align="left" valign="top" rowspan="1" colspan="1">1 μL</td>
</tr>
<tr><td colspan="2" align="left" valign="top" rowspan="1">
<hr>
</td></tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Flow rate</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.3 μL/min</td>
</tr>
<tr><td colspan="2" align="left" valign="top" rowspan="1">
<hr>
</td></tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<span class="text-underline">Time interval (min)</span>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">nanoLC gradient (% B (vol/vol))</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">0–5</td>
<td align="left" valign="top" rowspan="1" colspan="1">4%−4%</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">5–133</td>
<td align="left" valign="top" rowspan="1" colspan="1">4%−32%</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">133–152</td>
<td align="left" valign="top" rowspan="1" colspan="1">32%−48%</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">152–155</td>
<td align="left" valign="top" rowspan="1" colspan="1">48%−100%</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">155–170.5</td>
<td align="left" valign="top" rowspan="1" colspan="1">100%−100%</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">170.5–170.7</td>
<td align="left" valign="top" rowspan="1" colspan="1">100%−4%</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">170.7–180</td>
<td align="left" valign="top" rowspan="1" colspan="1">4%−4%</td>
</tr>
<tr><td colspan="2" align="left" valign="top" rowspan="1">
<hr>
</td></tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<strong>MS Parameters</strong>
</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr><td colspan="2" align="left" valign="top" rowspan="1">
<hr>
</td></tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Polarity</td>
<td align="left" valign="top" rowspan="1" colspan="1">Positive</td>
</tr>
<tr><td colspan="2" align="left" valign="top" rowspan="1">
<hr>
</td></tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Internal lock-mass calibrant</td>
<td align="left" valign="top" rowspan="1" colspan="1">455.12002</td>
</tr>
<tr><td colspan="2" align="left" valign="top" rowspan="1">
<hr>
</td></tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<span class="text-underline">MS1 OT</span>
</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">OT resolution</td>
<td align="left" valign="top" rowspan="1" colspan="1">60,000</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Scan Range, <em>m/z</em>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">350–2,000</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">RF lens (%)</td>
<td align="left" valign="top" rowspan="1" colspan="1">20</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">AGC target</td>
<td align="left" valign="top" rowspan="1" colspan="1">1.0e6</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Maximum Injection Time (ms)</td>
<td align="left" valign="top" rowspan="1" colspan="1">50</td>
</tr>
<tr><td colspan="2" align="left" valign="top" rowspan="1">
<hr>
</td></tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<span class="text-underline">Filters</span>
</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Charge state</td>
<td align="left" valign="top" rowspan="1" colspan="1">2–6</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Dynamic exclusion</td>
<td align="left" valign="top" rowspan="1" colspan="1">Exclude 1 time within 35 s</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Intensity</td>
<td align="left" valign="top" rowspan="1" colspan="1">5.0e4</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">MIPS</td>
<td align="left" valign="top" rowspan="1" colspan="1">Peptide</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Precursor selection range</td>
<td align="left" valign="top" rowspan="1" colspan="1">700–2,000</td>
</tr>
<tr><td colspan="2" align="left" valign="top" rowspan="1">
<hr>
</td></tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<span class="text-underline">MS2 OT</span>
</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Isolation window (<em>m/z</em>)</td>
<td align="left" valign="top" rowspan="1" colspan="1">4</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Activation type</td>
<td align="left" valign="top" rowspan="1" colspan="1">HCD</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">HCD collision energy (%)</td>
<td align="left" valign="top" rowspan="1" colspan="1">30±10</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">OT resolution</td>
<td align="left" valign="top" rowspan="1" colspan="1">15,000</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">First mass</td>
<td align="left" valign="top" rowspan="1" colspan="1">120</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">AGC target</td>
<td align="left" valign="top" rowspan="1" colspan="1">8.0e4</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Maximum Injection Time (ms)</td>
<td align="left" valign="top" rowspan="1" colspan="1">300</td>
</tr>
<tr><td colspan="2" align="left" valign="top" rowspan="1">
<hr>
</td></tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Cycle time (s)</td>
<td align="left" valign="top" rowspan="1" colspan="1">3</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="S72"><h4 class="pmc_sec_title">
<em>Site-specific glycoproteomic analysis:Glycoproteomics data analysis</em> ●Timing 2–8 h per file</h4>
<ul id="L69" class="list" style="list-style-type:none">
<li>
<span class="label">118</span><p class="display-inline" id="P321">Download and install the most recent version of Byonic<sup>™</sup> from <a href="https://www.proteinmetrics.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.proteinmetrics.com/</a></p>
</li>
<li>
<span class="label">119</span><p class="display-inline" id="P322">Open the Byonic<sup>™</sup> software, the .raw file is used for the MS/MS data file, and the <em>Homo sapiens</em> (Human) protein database from UniProt (UP000005640) is recommended for the protein database file. Set up searching parameters under each tab following <a href="#T4" class="usa-link">Table 4</a>.</p>
</li>
<li>
<span class="label">120</span><p class="display-inline" id="P324">Select the enzyme (trypsin) for digestion. Also setthe fragmentation type used for collecting MS/MS spectra, for example, for the stepped-collision energy HCD, choose the fragmentation type CID&amp; HCD.</p>
<p id="P325">⮙ CRITICAL STEP Detailed parameters for trypsin digested glycopeptides are included in <a href="#T4" class="usa-link">Table 4</a>.</p>
<p id="P326">⮙ CRITICAL STEP Other enzymes such as Glu-C and Lys-C can be selected, together with their corresponding cleavage sites.</p>
</li>
<li>
<span class="label">121</span><p class="display-inline" id="P327">Choose carbamidomethyl modification at cysteine residues (+57.021464) as a fixed modification. Details of other modifications are summarized in <a href="#T5" class="usa-link">Table 5</a>.</p>
<p id="P328">⮙ CRITICAL STEP Other specific modifications such as phosphorylations can be added based on the protein features.</p>
<p id="P329">⮙ CRITICAL STEP Search time will be dramatically increased with total common max numbers. It is usual to choose 1 for total common maximum number and total rare maximum number.</p>
</li>
<li>
<span class="label">122</span><p class="display-inline" id="P330">Load the N- or O- glycan libraries established from glycomics analysis (steps 36 and 92) as the glycans modification databases.</p>
<p id="P331">⮙ CRITICAL STEP To obtain confident identification, it is recommended that you use the databases from previous N-glycomics and O-glycomics, especially for unnatural monosaccharide incorporated glycopeptides.</p>
<p id="P332">⮙ CRITICAL STEP Identification of the proteins with O-glycosylation will tremendously increase the software searching time.</p>
</li>
<li>
<span class="label">123</span><p class="display-inline" id="P333">Use default settings for advanced parameters. If other peptide quantification software such as Skyline is desired, enable the mzIdentML file export.</p>
</li>
<li>
<span class="label">124</span><p class="display-inline" id="P334">Enable Excel report for curating the data as needed. Start searching by clicking “Run” button. The status of searching can be checked in the “Progress” tab.</p>
</li>
<li>
<span class="label">125</span><p class="display-inline" id="P335">In the exported Excel result file, use Score (&gt;300), |LogPro| (&gt;2.0), and DeltaMod (&gt;10, only for peptides with modifications) to filter confident identification. Peptide-spectrum matches can be validated by checking the peptide fragmentations using Byonic-Viewer.</p>
</li>
</ul>
<section class="tw xbox font-sm" id="T4"><h5 class="obj_head">Table 4.</h5>
<div class="caption p"><p id="P397">Data analysis parameters of glycopeptides using Byonic v 3.5.0 software</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="box" rules="all">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="center" valign="middle" rowspan="1" colspan="1">Tab</th>
<th align="center" valign="middle" rowspan="1" colspan="1">Section</th>
<th align="center" valign="middle" rowspan="1" colspan="1">Parameters</th>
<th align="center" valign="middle" rowspan="1" colspan="1">Values</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="9" align="center" valign="middle" colspan="1">Digestion and Instrument Parameters</td>
<td rowspan="4" align="left" valign="middle" colspan="1">Sample digestion</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Cleavage site (s)</td>
<td align="left" valign="middle" rowspan="1" colspan="1">KR</td>
</tr>
<tr>
<td align="left" valign="middle" rowspan="1" colspan="1">Cleavage side</td>
<td align="left" valign="middle" rowspan="1" colspan="1">C-terminal</td>
</tr>
<tr>
<td align="left" valign="middle" rowspan="1" colspan="1">Digestion specificity</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Fully specific (fastest)</td>
</tr>
<tr>
<td align="left" valign="middle" rowspan="1" colspan="1">Missed cleavages</td>
<td align="left" valign="middle" rowspan="1" colspan="1">2</td>
</tr>
<tr>
<td rowspan="5" align="left" valign="middle" colspan="1">Instrument parameters</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Precursor mass tolerance</td>
<td align="left" valign="middle" rowspan="1" colspan="1">10 ppm</td>
</tr>
<tr>
<td align="left" valign="middle" rowspan="1" colspan="1">Fragmentation type</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Both CID &amp; HCD</td>
</tr>
<tr>
<td align="left" valign="middle" rowspan="1" colspan="1">Fragment mass tolerance (CID)</td>
<td align="left" valign="middle" rowspan="1" colspan="1">20 ppm</td>
</tr>
<tr>
<td align="left" valign="middle" rowspan="1" colspan="1">Fragment mass tolerance (HCD)</td>
<td align="left" valign="middle" rowspan="1" colspan="1">20 ppm</td>
</tr>
<tr>
<td align="left" valign="middle" rowspan="1" colspan="1">Recalibration (lock mass)</td>
<td align="left" valign="middle" rowspan="1" colspan="1">None</td>
</tr>
<tr>
<td rowspan="2" align="center" valign="middle" colspan="1">Modifications</td>
<td rowspan="2" align="left" valign="middle" colspan="1">Fixed and variable modifications</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Total common max</td>
<td align="left" valign="middle" rowspan="1" colspan="1">1</td>
</tr>
<tr>
<td align="left" valign="middle" rowspan="1" colspan="1">Fixed modification</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Carbamidomethyl modification at cysteine residues (+57.021464)</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1"></td>
<td align="left" valign="middle" rowspan="1" colspan="1"></td>
<td align="left" valign="middle" rowspan="1" colspan="1">Variable modifications</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Details included in <a href="#T5" class="usa-link">Table 5</a> (Edit more from <br>“Enter/edit”)</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Glycans</td>
<td align="left" valign="middle" rowspan="1" colspan="1">/</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Enter/edit</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Load the glycan library</td>
</tr>
<tr>
<td rowspan="4" align="center" valign="middle" colspan="1">Advanced</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Spectrum input options</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Default</td>
<td align="left" valign="middle" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="middle" rowspan="1" colspan="1">Peptide output options</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Show all N-glycopeptides</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Uncheck</td>
</tr>
<tr>
<td rowspan="2" align="left" valign="middle" colspan="1">Proteins output options</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Protein FDR</td>
<td align="left" valign="middle" rowspan="1" colspan="1">1% FDR (or 20 reverse count)</td>
</tr>
<tr>
<td align="left" valign="middle" rowspan="1" colspan="1">Export mzIdentML</td>
<td align="left" valign="middle" rowspan="1" colspan="1">check</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="tw xbox font-sm" id="T5"><h5 class="obj_head">Table 5.</h5>
<div class="caption p"><p id="P398">Fixed and variable modifications</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="box" rules="all">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="center" valign="middle" rowspan="1" colspan="1">Modifications</th>
<th align="center" valign="middle" rowspan="1" colspan="1">Targets</th>
<th align="center" valign="middle" rowspan="1" colspan="1">Fine Control</th>
</tr></thead>
<tbody>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Carbamidomethyl / +57.021464</td>
<td align="center" valign="middle" rowspan="1" colspan="1">C</td>
<td align="center" valign="middle" rowspan="1" colspan="1">Fixed</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Oxidation / +15.994915</td>
<td align="center" valign="middle" rowspan="1" colspan="1">M</td>
<td align="center" valign="middle" rowspan="1" colspan="1">Variable – common 1</td>
</tr>
<tr>
<td rowspan="2" align="center" valign="middle" colspan="1">Deamidated / +0.984016</td>
<td align="center" valign="middle" rowspan="1" colspan="1">N</td>
<td align="center" valign="middle" rowspan="1" colspan="1">Variable – common 1</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Q</td>
<td align="center" valign="middle" rowspan="1" colspan="1">Variable – common 1</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Acetyl / +42.010565</td>
<td align="center" valign="middle" rowspan="1" colspan="1">Protein N-term</td>
<td align="center" valign="middle" rowspan="1" colspan="1">Variable – rare 1</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Gln&gt;pyro-Glu / −17.206549</td>
<td align="center" valign="middle" rowspan="1" colspan="1">Protein N-term Q</td>
<td align="center" valign="middle" rowspan="1" colspan="1">Variable – rare 1</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">Glu&gt;pyro-Glu / −18.010565</td>
<td align="center" valign="middle" rowspan="1" colspan="1">Protein N-term E</td>
<td align="center" valign="middle" rowspan="1" colspan="1">Variable – rare 1</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="S73"><h4 class="pmc_sec_title">
<em>Site-specific glycoproteomic analysis: Label-free glycopeptide quantification (optional)</em> ●Timing 2–8 h per file</h4>
<ul id="L73" class="list" style="list-style-type:none">
<li>
<span class="label">126</span><p class="display-inline" id="P336">Download and install the most recent version of and Byologic<sup>™</sup> from <a href="https://www.proteinmetrics.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.proteinmetrics.com/</a></p>
</li>
<li>
<span class="label">127</span><p class="display-inline" id="P337">Drag .byrslt file from the Byonic search file to Byologic input for MS1 and MS2 search.</p>
<p id="P338">⮙ CRITICAL STEP .raw file should be relocated if the file folder has been changed.</p>
</li>
<li>
<span class="label">128</span><p class="display-inline" id="P339">Load the filtered protein lists from Byonic results.</p>
<p id="P340">⮙ CRITICAL STEP Byonic import can be directly used if no filter is applied for the protein identification.</p>
</li>
<li>
<span class="label">129</span><p class="display-inline" id="P341">Choose default settings for MS extract options.</p>
<p id="P342">⮙ CRITICAL STEP The default parameters in MS extract options should be re-evaluated if any peak cut off has been noticed.</p>
</li>
<li>
<span class="label">130</span><p class="display-inline" id="P343">Create the file to start analyzing and export the report.</p>
</li>
</ul></section></section></section><section id="S74"><h2 class="pmc_sec_title">Troubleshooting</h2>
<p id="P344">For troubleshooting guidance, see <a href="#T6" class="usa-link">Table 6</a>.</p>
<section class="tw xbox font-sm" id="T6"><h3 class="obj_head">Table 6:</h3>
<div class="caption p"><p id="P399">Troubleshooting</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="box" rules="all">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="center" valign="top" rowspan="1" colspan="1">Step</th>
<th align="center" valign="top" rowspan="1" colspan="1">Problem</th>
<th align="center" valign="top" rowspan="1" colspan="1">Possible reason</th>
<th align="center" valign="top" rowspan="1" colspan="1">Solution</th>
</tr></thead>
<tbody>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">20</td>
<td align="left" valign="middle" rowspan="1" colspan="1">There is leftover water in some wells.</td>
<td align="left" valign="middle" rowspan="1" colspan="1">It is difficult for hydrophilic water to go through hydrophobic PGC materials.</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Use a higher speed (180 x g) to centrifuge for another round.</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">27</td>
<td align="left" valign="middle" rowspan="1" colspan="1">The autosampler errors out at the beginning of the worklist.</td>
<td align="left" valign="middle" rowspan="1" colspan="1">The correct type of well-plate is not assigned.</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Check the “assign well-plate” tab. Select the “Eppendorf 96-well plate” for the 96-well injection plate and select the “54 vial plate” for the injection vials.</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">30</td>
<td align="left" valign="middle" rowspan="1" colspan="1">The pressure of pumps is higher than normal.</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Chips might not be properly conditioned. It is also possible that the chip or other compartments such as <br>inlet filters or some capillaries are clogged.</td>
<td align="left" valign="middle" rowspan="1" colspan="1">To condition the chip completely, decrease the composition of solvent B with an increment of 20% until 2% of solvent B. If the pressure is still high, back flush the chip or change an inlet filter or a new capillary.</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">31</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Dripping or scattered spray is observed.</td>
<td align="left" valign="middle" rowspan="1" colspan="1">The capillary voltage is not optimized.</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Increase the source voltage while observing the spray changes. Increase the capillary voltage with an increment of 50 volts (V) until a consistent spray is seen. The highest voltage cannot exceed 2,400 V. Make sure that the capillary current does not exceed 100 nano ampere (nA).</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">32</td>
<td align="left" valign="middle" rowspan="1" colspan="1">There is no signal during the run or the LC pumps errors out.</td>
<td align="left" valign="middle" rowspan="1" colspan="1">The spray disappears during one batch or the solvent level goes down to 0.005 L.</td>
<td align="left" valign="middle" rowspan="1" colspan="1">During the analysis of one batch of sample, check the chip spray and the solvent level regularly to make sure the instrument is running properly.</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">45</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Phase separation is observed.</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Water and/or salt content in sample is high.</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Adjust the solvent composition by adding 2:1 Methanol/Chloroform in 50 μL increments until solvents become miscible and a single liquid phase is observed. Vortex mix the sample in between additions. If phases are still immiscible after adding 200 μL, skip step 47 (addition of KCl) and continue as normal.</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">48</td>
<td align="left" valign="middle" rowspan="1" colspan="1">No phase separation is observed.</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Not enough H<sub>2</sub>O or KCl solution</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Add an extra small amount of water or 0.1 M KCl solution and mix carefully.</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">75</td>
<td align="left" valign="middle" rowspan="1" colspan="1">The volume of added NaBH<sub>4</sub> is not consistent for every sample.</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Large amounts of bubbles are generated in the NaBH<sub>4</sub> solution.</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Vortex mix NaBH<sub>4</sub> solution shortly to remove bubbles every time just before adding to the sample to make sure the volume added is accurate. <br>
</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">77</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Samples are overflowed from the tube.</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Large amounts of gas are generated in the neutralization reaction.</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Add the acetic acid drop by drop to avoid generating bubbles vigorously.</td>
</tr>
<tr>
<td align="center" valign="middle" rowspan="1" colspan="1">111</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Samples cannot be dissolved completely.</td>
<td align="left" valign="middle" rowspan="1" colspan="1">The dried pellet is formed, and they are more soluble in hydrophilic solvent rather than the solvent with 80% ACN.</td>
<td align="left" valign="middle" rowspan="1" colspan="1">Sonicate the mixture for several minutes. When drying samples, dry them down to around 100 μL. Estimate the volume of the sample and add H<sub>2</sub>O, 80% (vol/vol) ACN, and 1% (vol/vol) TFA to make the final volume 1 mL.</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="S75"><h2 class="pmc_sec_title">Timing</h2>
<p id="P345">Steps 1.A)i)-1.A)Step 1A, tissue sample preparation: 1–3 h depends on the size of sample set</p>
<p id="P346">Steps 1.B)i)-1.B)vii), mammalian cell growth: 5 d</p>
<p id="P347">Steps 2–11, cell lysis and membrane extraction: 5–8 h depends on the size of sample set</p>
<p id="P348">Steps 12–18, N-glycan release: 20 h</p>
<p id="P349">Steps 19–24, desalting of N-glycans with PGC SPE plates: 1–2 h</p>
<p id="P350">Steps 25–33, nano-chip-QTOF-MS/MS analysis of N-glycans: 1 h per sample</p>
<p id="P351">Steps 34–43, N-glycomics data analysis: 1–2 h per batch</p>
<p id="P352">Steps 44–49, glycolipids extraction: 1 h</p>
<p id="P353">Steps 50–55, desalting of glycolipids with C8 SPE plates: 1–2 h</p>
<p id="P354">Steps 56, nano-chip-QTOF-MS/MS analysis of glycolipids: 1 h per sample</p>
<p id="P355">Steps 57–72, glycolipidomics data analysis: 1–2 h per batch</p>
<p id="P356">Steps 73–79, O-glycan release: 1d</p>
<p id="P357">Steps 80, desalting of O-glycans with PGC SPE plates: 1–2 h</p>
<p id="P358">Steps 81–88, enrichment of O-glycans with HILIC cartridges: 1–2 h</p>
<p id="P359">Steps 89, nano-chip-QTOF-MS/MS analysis of O-glycans: 1 h per sample</p>
<p id="P360">Steps 90–99, O-glycomics data analysis: 1–2 h per batch</p>
<p id="P361">Steps 100–107, Protein digestion: 20 h</p>
<p id="P362">Steps 108–115, enrichment of glycopeptides with HILIC cartridges: 2–3 h</p>
<p id="P363">Steps 116–117, nanoLC-MS/MS analysis of glycopeptide: 4 h per sample with one blank injection</p>
<p id="P364">Steps 118–125, glycoproteomics data analysis: 2–8 h per file</p>
<p id="P365">Steps 126–130, label-free glycopeptide quantification (optional): 2–8 h per file</p></section><section id="S76"><h2 class="pmc_sec_title">Anticipated results</h2>
<p id="P366">This protocol is used for the comprehensive characterization of cell membrane glycocalyx of both cell and tissue samples. Successful application of this protocol will lead to the identification and label-free-quantitation of hundreds of non-redundant glycosylated species.The glycocalyx of the Caco-2 cell line before and after differentiation has been successfully characterized using this workflow. Anticipated results should be in line with past performance of the overall method. Below we describe anticipated results based on past performance within specific studies.</p>
<section id="S77"><h3 class="pmc_sec_title">Glycomic variations during cell differentiation.</h3>
<p id="P367">By monitoring the morphology with microscopy, it was observed that the differentiation of Caco-2 cells started from the fifth to the seventh day and completed on day 21. The biological progressive changes of cell surface glycocalyx during cell differentiation were illustrated following the first part of the workflow. The N-glycomic analysis showed that the level of high-mannose type N-glycans decreased, while the levels of fucosylated and sialylated N-glycans were elevated during differentiation (<a href="#F8" class="usa-link">Figure 8</a>)<sup><a href="#R33" class="usa-link" aria-describedby="R33">33</a></sup>. The glycolipidomic analysis identified more than 200 intact glycosphingolipids (GSL) on differentiated Caco-2 cells (<a href="#F9" class="usa-link">Figure 9</a>)<sup><a href="#R24" class="usa-link" aria-describedby="R24">24</a>,<a href="#R26" class="usa-link" aria-describedby="R26">26</a></sup>. During the maturation process the globo-type GSLs decreased, while GSLs with sialylated and sulfated head groups increased. Additionally, the relative abundances of longer ceramides with 40–42 carbons decreased whereas shorter ceramides with 32 to 34 carbons showed higher relative abundances during maturation. Following the second part of the workflow, the characterization of intact glycopeptides of differentiated Caco-2 cells generated 2553 glycopeptides and 165 glycoproteins with 444 glycosites<sup><a href="#R24" class="usa-link" aria-describedby="R24">24</a></sup>. Approximately 62% of identified glycoproteins were single-pass transmembrane proteins, with half of them having their N-terminus exposed to the extracellular space. Another 20% of identified glycoproteins were multi-pass transmembrane proteins containing several protein regions extending into the extracellular region. The rest of identified glycoproteins were secretory or peripheral proteins.</p>
<figure class="fig xbox font-sm" id="F8"><h4 class="obj_head">Figure 8. The N-glycomic analysis of Caco-2 cells cultured grown at different days.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2ce2/11790333/6c083002be75/nihms-2039916-f0008.jpg" loading="lazy" height="1050" width="450" alt="Figure 8."></p>
<div class="p text-right font-secondary"><a href="figure/F8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P392">The profile illustrated the increased levels of sialylated and fucosylated C/H N-glycans, and the decreased level of high-mannose N-glycans. Reprinted with permission from ref. <a href="#R33" class="usa-link" aria-describedby="R33">33</a>. Copyright 2015 by American society for biochemistry and molecular biology.</p></figcaption></figure><figure class="fig xbox font-sm" id="F9"><h4 class="obj_head">Figure 9. The identification and quantitation of glycosphingolipids (GSL) from Caco-2 and PNT2 cells lines.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2ce2/11790333/2c44ecf5d9cc/nihms-2039916-f0009.jpg" loading="lazy" height="943" width="1050" alt="Figure 9."></p>
<div class="p text-right font-secondary"><a href="figure/F9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="P393"><strong>a</strong>. Sialylated GSLs with theire tentative structures were ordered by their abundances. <strong>b</strong>. Sialylated GSLs were quantified and compared based on the head groups and ceramide types. Reprinted with permission from ref. <a href="#R24" class="usa-link" aria-describedby="R24">24</a>. Creative Commons licence (CC BY 3.0) (<a href="http://creativecommons.org/licenses/by/3.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/3.0/</a>)</p></figcaption></figure></section><section id="S78"><h3 class="pmc_sec_title">Incorporation of non-native monosaccharide.</h3>
<p id="P368">The workflow has also been applied to determine the incorporation of unnatural SiaNAz into the glycocalyx, with differentiated Caco-2 cells as an example<sup><a href="#R24" class="usa-link" aria-describedby="R24">24</a></sup>. The SiaNAz-incorporated N-glycans accounted for nearly 30% of total sialylated N-glycans. In contrast, SiaNAz groups were more highly incorporated into O-glycans, where more than 70% of sialylated O-glycans contained SiaNAz. The rate of incorporation of SiaNAz into all sialylated glycosphingolipids was around 8%.</p></section><section id="S79"><h3 class="pmc_sec_title">The glycocalyx under variable environmental conditions.</h3>
<p id="P369">The protocol has also been employed to monitor the glycan variations seen when cells are subjected to environmental changes. This includes the use of different cell culture supplementations such as the addition of exogenous dietary monosaccharides and short-chain fatty acids, and differing pH in the culturing conditions<sup><a href="#R23" class="usa-link" aria-describedby="R23">23</a></sup>. The supplementation of different exogenous monosaccharides generated varied N-glycan types. In addition, fucosylated N-glycans showed significant changes for cells treated with short-chain fatty acids (SCFAs), including acetate, lactate, and butyrate. The levels of sialylated N-glycans showed A drastic increase for cells cultured at lower pHs. These findings shed light on the roles that the cell surface glycocalyx plays in the interaction of cells in differing environments.</p></section><section id="S80"><h3 class="pmc_sec_title">Isotopic and oxidative labeling in the glycocalyx.</h3>
<p id="P370">This workflow is also compatible with other techniques, including proteomic analysis and metabolic labeling. For example, by combining the glycomics and glycoproteomics workflows with standard proteomics workflow, we identified potential sialic acid-associating proteins on the cell surface using proximity labeling<sup><a href="#R92" class="usa-link" aria-describedby="R92">92</a></sup>. The glycomic results demonstrated the incorporation of unnatural monosaccharides and the conjugation of the labeling probe. The glycoproteomic analysis mapped sialylated glycoproteins on the cell surface with site-specific information, and the oxidative proteomic analysis provided information about the proteins that were localized around the sialic acid. As a result, it revealed the structural environment of sialylated proteins in the glycocalyx. We have also successfully combined this method with metabolic labeling to investigate the metabolic fate of monosaccharides in cell membranes<sup><a href="#R93" class="usa-link" aria-describedby="R93">93</a></sup>. Caco-2 and hepatic KKU-M213 cells were treated with <sup>13</sup>C-labeled dietary saccharides, and the <sup>13</sup>C-incorporated glycans and glycoproteins on the cell membrane were characterized quantitatively using the described glycomics and glycoproteomics workflows.</p></section></section><section id="SM1"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="SD1"><div class="caption p">
<span>Supplementary Data 1</span><p id="P371">Supplementary Data I_N-glycan_lib_template.csv contains examples of N-glycans with their masses, compositions, and types.</p>
</div>
<div class="media p" id="d67e2093"><div class="caption">
<a href="/articles/instance/11790333/bin/NIHMS2039916-supplement-Supplementary_Data_1.csv" data-ga-action="click_feat_suppl" class="usa-link">NIHMS2039916-supplement-Supplementary_Data_1.csv</a><sup> (1.5KB, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="SD2"><div class="caption p">
<span>Supplementary Data III</span><p id="P372">Supplementary Data III_GSL_glycan_finder.xlsx matches the closest glycan composition to the glycosphingolipid with the known precursor <em>m/z</em>, precursor charge state, and the ceramide <em>m/z</em>.</p>
</div>
<div class="media p" id="d67e2105"><div class="caption">
<a href="/articles/instance/11790333/bin/NIHMS2039916-supplement-Supplementary_Data_III.xlsx" data-ga-action="click_feat_suppl" class="usa-link">NIHMS2039916-supplement-Supplementary_Data_III.xlsx</a><sup> (29.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="SD3"><div class="caption p">
<span>Supplementary Data V</span><p id="P373">Supplementary Data V_GSL_lib_generator.xlsx contains a Pivot table for generating a database of glycosphingolipids.</p>
</div>
<div class="media p" id="d67e2111"><div class="caption">
<a href="/articles/instance/11790333/bin/NIHMS2039916-supplement-Supplementary_Data_V.xlsx" data-ga-action="click_feat_suppl" class="usa-link">NIHMS2039916-supplement-Supplementary_Data_V.xlsx</a><sup> (2.6MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="SD4"><div class="caption p">
<span>Supplementary Data IV</span><p id="P374">Supplementary Data IV_Saccharide_mz.csv contains some common saccharide and lipid fragments.</p>
</div>
<div class="media p" id="d67e2117"><div class="caption">
<a href="/articles/instance/11790333/bin/NIHMS2039916-supplement-Supplementary_Data_IV.csv" data-ga-action="click_feat_suppl" class="usa-link">NIHMS2039916-supplement-Supplementary_Data_IV.csv</a><sup> (863B, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="SD5"><div class="caption p">
<span>Supplementary Data II</span><p id="P375">Supplementary Data II_Ceramide_mz.csv contains possible ceramide <em>m/z</em> values.</p>
</div>
<div class="media p" id="d67e2126"><div class="caption">
<a href="/articles/instance/11790333/bin/NIHMS2039916-supplement-Supplementary_Data_II.csv" data-ga-action="click_feat_suppl" class="usa-link">NIHMS2039916-supplement-Supplementary_Data_II.csv</a><sup> (3.8KB, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="SD6"><div class="caption p"><span>Supplementary Figures</span></div>
<div class="media p" id="d67e2129"><div class="caption">
<a href="/articles/instance/11790333/bin/NIHMS2039916-supplement-Supplementary_Figures.docx" data-ga-action="click_feat_suppl" class="usa-link">NIHMS2039916-supplement-Supplementary_Figures.docx</a><sup> (518.2KB, docx) </sup>
</div></div></section></section><section id="floats-group1" class="floats-group"><section class="bt xbox font-sm" id="BX1"><ol id="L78" class="list" style="list-style-type:decimal">
<li><p id="P400">For each data set displayed in a figure or table please explain whether it has included in a previously published paper or public repository, and indicate where.</p></li>
<li><p id="P401">Submit actual figures individually in one of our accepted file formats (preferably eps or tif; see Guide to Authors for details).</p></li>
<li><p id="P402">Ensure that data is fully explained (n values given, error bars defined, etc.). Note that graphs showing individual data points are often preferable to those showing means and error bars. If there are fewer than 6 data points, individual data points must be provided. Please ensure any statistics are applied appropriately. This article gives further guidance, should it be needed: <a href="http://www.nature.com/nature/journal/v492/n7428/full/492180a.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.nature.com/nature/journal/v492/n7428/full/492180a.html</a>. If it is a problem to provide graphs showing each data point, an alternative (less desirable) is to include the actual raw data as a <a href="#SD6" class="usa-link">Supplementary File</a> associated with the figure.</p></li>
<li><p id="P403">The colour yellow can be difficult to distinguish. Please consider replacing the yellow in your figures with another colour.</p></li>
<li><p id="P404">Please avoid the use of coloured backgrounds in figures/flowcharts. These make the overlying text difficult to read, particularly when printed in black and white.</p></li>
<li><p id="P405">Please remove the boxes around your graphs/histograms/plots. Our house style is to use these only when x, y and z axes are used.</p></li>
<li><p id="P406">In order to address the issue of cell line misidentification and cross-contamination, for any results shown using cell lines in the manuscript please provide source of the cell lines and indicate whether the cell lines have been correctly identified/authenticated (and if so, by what methods). Also, please state whether cell lines have been tested for mycoplasma contamination. Further information about this policy and useful resources regarding identification and authentication of cell lines can be found at <a href="http://www.nature.com/authors/policies/availability.html#further" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.nature.com/authors/policies/availability.html#further</a>.</p></li>
<li><p id="P407">Please add Procedure step numbers to different parts of the flow chart figures so that readers can link the <a href="#S42" class="usa-link">Procedure sections</a> with the overall scheme. Where these are already present they will need updating to reflect the final procedure step numbers.</p></li>
</ol></section></section><section id="S83" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p id="P376">This work was supported by the National Institutes of Health (GMRO1R01, GM049077).</p></section><section id="APP1" class="app"><h2 class="pmc_sec_title">Related links</h2>
<section id="S84"><h3 class="pmc_sec_title">Key references using this protocol</h3>
<ol id="L77" class="list" style="list-style-type:decimal">
<li><p id="P378">Park, D. <em>et al</em>. Membrane glycomics reveal heterogeneity and quantitative distribution of cell surface sialylation. <em>Chemical science</em>
<strong>9</strong>, 6271–6285 (2018), DOI: <a href="https://doi.org/10.1039/c8sc01875h" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1039/c8sc01875h</a>.</p></li>
<li><p id="P379">Wong, M., Xu, G., Park, D., Barboza, M. &amp; Lebrilla, C. B. Intact glycosphingolipidomic analysis of the cell membrane during differentiation yields extensive glycan and lipid changes. <em>Scientific reports</em>
<strong>8</strong>, 10993 (2018), DOI: <a href="https://doi.org/10.1038/s41598-018-29324-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-018-29324-7</a>.</p></li>
<li><p id="P380">Li, Q., Xie, Y., Xu, G. &amp; Lebrilla, C. B. Identification of potential sialic acid binding proteins on cell membrane by proximity chemical labeling. <em>Chemical Science</em> (2019), DOI: <a href="https://doi.org/10.1039/c9sc01360a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1039/c9sc01360a</a>.</p></li>
</ol></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="FN2">
<p id="P381">Reporting Summary</p>
<p id="P382">Further information on research design is available in the Nature Research Reporting Summary linked to this article.</p>
</div>
<div class="fn p" id="FN3">
<p id="P383">Competing interests</p>
<p id="P384">The authors declare no competing interests.</p>
</div>
</div></section><section id="S82"><h2 class="pmc_sec_title">Data availability</h2>
<p id="P377">Data are all available online. Data from ref. <a href="#R23" class="usa-link" aria-describedby="R23">23</a> Park, D. <em>et al</em>. <em>Glycobiology</em>
<strong>27</strong>, 847–860 (2017), DOI:<a href="https://doi.org/10.1093/glycob/cwx041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1093/glycob/cwx041</a> (used for <a href="#F4" class="usa-link">Fig. 4a</a>) are available at Glycobiology online. Data from ref. <a href="#R24" class="usa-link" aria-describedby="R24">24</a> Park, D. <em>et al</em>. <em>Chemical science</em>
<strong>9</strong>, 6271–6285 (2018), DOI: <a href="https://doi.org/10.1039/c8sc01875h" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1039/c8sc01875h</a> (used for <a href="#F4" class="usa-link">Fig. 4b</a> and <a href="#F9" class="usa-link">9</a>) are available at Chemical Science online. Data from ref. <a href="#R26" class="usa-link" aria-describedby="R26">26</a> Wong, M. <em>et al</em>. <em>Scientific reports</em>
<strong>8</strong>, 10993 (2018), DOI: <a href="https://doi.org/10.1038/s41598-018-29324-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-018-29324-7</a> (used for <a href="#F4" class="usa-link">Fig. 4c</a>) are available at Scientific Reports online. Data from ref. <a href="#R25" class="usa-link" aria-describedby="R25">25</a> Li, Q. <em>et al</em>. <em>Chemical science</em>
<strong>10</strong>, 6199–6209 (2019), DOI:<a href="https://doi.org/10.1039/c9sc01360a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1039/c9sc01360a</a> (used for <a href="#F6" class="usa-link">Fig. 6</a>) are available at Chemical Science online. Data from ref. <a href="#R33" class="usa-link" aria-describedby="R33">33</a> Park, D. <em>et al</em>. <em>Molecular &amp; cellular proteomics</em> : MCP <strong>14</strong>, 2910–2921 (2015), DOI:<a href="https://doi.org/10.1074/mcp.M115.053983" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1074/mcp.M115.053983</a> (used for <a href="#F8" class="usa-link">Fig. 8</a>) are available at MCP online.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="R1">
<span class="label">1.</span><cite>Martinez-Seara Monne H, Danne R, Róg T, Ilpo V. &amp; Gurtovenko A. Structure of Glycocalyx. Biophysical Journal
104, 251a, doi: 10.1016/j.bpj.2012.11.1412 (2013).</cite> [<a href="https://doi.org/10.1016/j.bpj.2012.11.1412" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophysical%20Journal&amp;title=Structure%20of%20Glycocalyx&amp;author=H%20Martinez-Seara%20Monne&amp;author=R%20Danne&amp;author=T%20R%C3%B3g&amp;author=V%20Ilpo&amp;author=A%20Gurtovenko&amp;volume=104&amp;publication_year=2013&amp;pages=251a&amp;doi=10.1016/j.bpj.2012.11.1412&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R2">
<span class="label">2.</span><cite>Reitsma S, Slaaf DW, Vink H, Van Zandvoort MA &amp; Oude Egbrink MG
The endothelial glycocalyx: composition, functions, and visualization. Pflügers Archiv-European Journal of Physiology
454, 345–359 (2007).
</cite> [<a href="https://doi.org/10.1007/s00424-007-0212-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1915585/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17256154/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pfl%C3%BCgers%20Archiv-European%20Journal%20of%20Physiology&amp;title=The%20endothelial%20glycocalyx:%20composition,%20functions,%20and%20visualization&amp;author=S%20Reitsma&amp;author=DW%20Slaaf&amp;author=H%20Vink&amp;author=MA%20Van%20Zandvoort&amp;author=MG%20Oude%20Egbrink&amp;volume=454&amp;publication_year=2007&amp;pages=345-359&amp;pmid=17256154&amp;doi=10.1007/s00424-007-0212-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R3">
<span class="label">3.</span><cite>Flessner MF
Endothelial glycocalyx and the peritoneal barrier. Peritoneal dialysis international
28, 6–12 (2008).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/18178940/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Peritoneal%20dialysis%20international&amp;title=Endothelial%20glycocalyx%20and%20the%20peritoneal%20barrier&amp;author=MF%20Flessner&amp;volume=28&amp;publication_year=2008&amp;pages=6-12&amp;pmid=18178940&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R4">
<span class="label">4.</span><cite>Mensah SA
et al. 
Regeneration of glycocalyx by heparan sulfate and sphingosine 1-phosphate restores inter-endothelial communication. PloS one
12, e0186116 (2017).</cite> [<a href="https://doi.org/10.1371/journal.pone.0186116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5638341/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29023478/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PloS%20one&amp;title=Regeneration%20of%20glycocalyx%20by%20heparan%20sulfate%20and%20sphingosine%201-phosphate%20restores%20inter-endothelial%20communication&amp;author=SA%20Mensah&amp;volume=12&amp;publication_year=2017&amp;pmid=29023478&amp;doi=10.1371/journal.pone.0186116&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R5">
<span class="label">5.</span><cite>Kuo JC-H, Gandhi JG, Zia RN &amp; Paszek MJ
Physical biology of the cancer cell glycocalyx. Nature Physics
14, 658 (2018).
</cite> [<a href="https://doi.org/10.1038/s41567-018-0186-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8046174/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33859716/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Physics&amp;title=Physical%20biology%20of%20the%20cancer%20cell%20glycocalyx&amp;author=JC-H%20Kuo&amp;author=JG%20Gandhi&amp;author=RN%20Zia&amp;author=MJ%20Paszek&amp;volume=14&amp;publication_year=2018&amp;pages=658&amp;pmid=33859716&amp;doi=10.1038/s41567-018-0186-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R6">
<span class="label">6.</span><cite>Yao Y, Rabodzey A. &amp; Dewey CF Jr
Glycocalyx modulates the motility and proliferative response of vascular endothelium to fluid shear stress. American Journal of Physiology-Heart and Circulatory Physiology
293, H1023–H1030 (2007).
</cite> [<a href="https://doi.org/10.1152/ajpheart.00162.2007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17468337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Physiology-Heart%20and%20Circulatory%20Physiology&amp;title=Glycocalyx%20modulates%20the%20motility%20and%20proliferative%20response%20of%20vascular%20endothelium%20to%20fluid%20shear%20stress&amp;author=Y%20Yao&amp;author=A%20Rabodzey&amp;author=CF%20Dewey&amp;volume=293&amp;publication_year=2007&amp;pages=H1023-H1030&amp;pmid=17468337&amp;doi=10.1152/ajpheart.00162.2007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R7">
<span class="label">7.</span><cite>Gristina AG &amp; Costerton J. Bacterial adherence and the glycocalyx and their role in musculoskeletal infection. Orthopedic Trauma Directions
9, 23–25 (2011).</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/6472832/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Orthopedic%20Trauma%20Directions&amp;title=Bacterial%20adherence%20and%20the%20glycocalyx%20and%20their%20role%20in%20musculoskeletal%20infection&amp;author=AG%20Gristina&amp;author=J%20Costerton&amp;volume=9&amp;publication_year=2011&amp;pages=23-25&amp;pmid=6472832&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R8">
<span class="label">8.</span><cite>Martin L, Koczera P, Zechendorf E. &amp; Schuerholz T. The endothelial glycocalyx: new diagnostic and therapeutic approaches in sepsis. BioMed research international
2016, 1–8, doi: 10.1155/2016/3758278 (2016).</cite> [<a href="https://doi.org/10.1155/2016/3758278" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5028820/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27699168/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BioMed%20research%20international&amp;title=The%20endothelial%20glycocalyx:%20new%20diagnostic%20and%20therapeutic%20approaches%20in%20sepsis&amp;author=L%20Martin&amp;author=P%20Koczera&amp;author=E%20Zechendorf&amp;author=T%20Schuerholz&amp;volume=2016&amp;publication_year=2016&amp;pages=1-8&amp;pmid=27699168&amp;doi=10.1155/2016/3758278&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R9">
<span class="label">9.</span><cite>Arabyan N. et al. 
Salmonella degrades the host glycocalyx leading to altered infection and glycan remodeling. Scientific reports
6, 29525 (2016).
</cite> [<a href="https://doi.org/10.1038/srep29525" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4937416/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27389966/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20reports&amp;title=Salmonella%20degrades%20the%20host%20glycocalyx%20leading%20to%20altered%20infection%20and%20glycan%20remodeling&amp;author=N%20Arabyan&amp;volume=6&amp;publication_year=2016&amp;pages=29525&amp;pmid=27389966&amp;doi=10.1038/srep29525&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R10">
<span class="label">10.</span><cite>Yeo TW
et al. 
Glycocalyx Breakdown Is Associated With Severe Disease and Fatal Outcome in Plasmodium falciparum Malaria. Clinical Infectious Diseases
69, 1712–1720, doi: 10.1093/cid/ciz038 (2019).
</cite> [<a href="https://doi.org/10.1093/cid/ciz038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6821254/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30753363/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clinical%20Infectious%20Diseases&amp;title=Glycocalyx%20Breakdown%20Is%20Associated%20With%20Severe%20Disease%20and%20Fatal%20Outcome%20in%20Plasmodium%20falciparum%20Malaria&amp;author=TW%20Yeo&amp;volume=69&amp;publication_year=2019&amp;pages=1712-1720&amp;pmid=30753363&amp;doi=10.1093/cid/ciz038&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R11">
<span class="label">11.</span><cite>Hakomori S.-i. in Advances in Cancer Research Vol. Volume 52 (eds Vande Woude George F&amp; Klein George) 257–331 (Academic Press, 1989).2662714
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Advances%20in%20Cancer%20Research&amp;author=S.-i.%20Hakomori&amp;author=F%20Vande%20Woude%20George&amp;author=George%20Klein&amp;publication_year=1989&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R12">
<span class="label">12.</span><cite>Noda K. et al. 
Relationship between elevated FX expression and increased production of GDP-L-fucose, a common donor substrate for fucosylation in human hepatocellular carcinoma and hepatoma cell lines. Cancer research
63, 6282–6289 (2003).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/14559815/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20research&amp;title=Relationship%20between%20elevated%20FX%20expression%20and%20increased%20production%20of%20GDP-L-fucose,%20a%20common%20donor%20substrate%20for%20fucosylation%20in%20human%20hepatocellular%20carcinoma%20and%20hepatoma%20cell%20lines&amp;author=K%20Noda&amp;volume=63&amp;publication_year=2003&amp;pages=6282-6289&amp;pmid=14559815&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R13">
<span class="label">13.</span><cite>Nie H. et al. 
Specific N-glycans of hepatocellular carcinoma cell surface and the abnormal increase of core-α−1, 6-fucosylated triantennary glycan via N-acetylglucosaminyltransferases-IVa regulation. Scientific reports
5, 16007 (2015).
</cite> [<a href="https://doi.org/10.1038/srep16007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4633583/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26537865/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20reports&amp;title=Specific%20N-glycans%20of%20hepatocellular%20carcinoma%20cell%20surface%20and%20the%20abnormal%20increase%20of%20core-%CE%B1%E2%88%921,%206-fucosylated%20triantennary%20glycan%20via%20N-acetylglucosaminyltransferases-IVa%20regulation&amp;author=H%20Nie&amp;volume=5&amp;publication_year=2015&amp;pages=16007&amp;pmid=26537865&amp;doi=10.1038/srep16007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R14">
<span class="label">14.</span><cite>Paszek MJ
et al. 
The cancer glycocalyx mechanically primes integrin-mediated growth and survival. Nature
511, 319 (2014).
</cite> [<a href="https://doi.org/10.1038/nature13535" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4487551/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25030168/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20cancer%20glycocalyx%20mechanically%20primes%20integrin-mediated%20growth%20and%20survival&amp;author=MJ%20Paszek&amp;volume=511&amp;publication_year=2014&amp;pages=319&amp;pmid=25030168&amp;doi=10.1038/nature13535&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R15">
<span class="label">15.</span><cite>Maverakis E. et al. 
Glycans in the immune system and The Altered Glycan Theory of Autoimmunity: a critical review. Journal of autoimmunity
57, 1–13 (2015).
</cite> [<a href="https://doi.org/10.1016/j.jaut.2014.12.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4340844/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25578468/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20autoimmunity&amp;title=Glycans%20in%20the%20immune%20system%20and%20The%20Altered%20Glycan%20Theory%20of%20Autoimmunity:%20a%20critical%20review&amp;author=E%20Maverakis&amp;volume=57&amp;publication_year=2015&amp;pages=1-13&amp;pmid=25578468&amp;doi=10.1016/j.jaut.2014.12.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R16">
<span class="label">16.</span><cite>Dixon J. et al. 
Electron microscopic investigation of the bladder urothelium and glycocalyx in patients with interstitial cystitis. The Journal of urology
135, 621–625 (1986).
</cite> [<a href="https://doi.org/10.1016/s0022-5347(17)45763-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3944920/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20urology&amp;title=Electron%20microscopic%20investigation%20of%20the%20bladder%20urothelium%20and%20glycocalyx%20in%20patients%20with%20interstitial%20cystitis&amp;author=J%20Dixon&amp;volume=135&amp;publication_year=1986&amp;pages=621-625&amp;pmid=3944920&amp;doi=10.1016/s0022-5347(17)45763-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R17">
<span class="label">17.</span><cite>de Buy Wenniger LJM
et al. 
The Cholangiocyte Glycocalyx Stabilizes the ‘Biliary HCO3-Umbrella’: An Integrated Line of Defense against Toxic Bile Acids. Digestive Diseases
33, 397–407 (2015).
</cite> [<a href="https://doi.org/10.1159/000371864" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26045275/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Digestive%20Diseases&amp;title=The%20Cholangiocyte%20Glycocalyx%20Stabilizes%20the%20%E2%80%98Biliary%20HCO3-Umbrella%E2%80%99:%20An%20Integrated%20Line%20of%20Defense%20against%20Toxic%20Bile%20Acids&amp;author=LJM%20de%20Buy%20Wenniger&amp;volume=33&amp;publication_year=2015&amp;pages=397-407&amp;pmid=26045275&amp;doi=10.1159/000371864&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R18">
<span class="label">18.</span><cite>An HJ, Kronewitter SR, de Leoz MLA &amp; Lebrilla CB
Glycomics and disease markers. Current Opinion in Chemical Biology
13, 601–607, doi: 10.1016/j.cbpa.2009.08.015 (2009).
</cite> [<a href="https://doi.org/10.1016/j.cbpa.2009.08.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2788081/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19775929/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Opinion%20in%20Chemical%20Biology&amp;title=Glycomics%20and%20disease%20markers&amp;author=HJ%20An&amp;author=SR%20Kronewitter&amp;author=MLA%20de%20Leoz&amp;author=CB%20Lebrilla&amp;volume=13&amp;publication_year=2009&amp;pages=601-607&amp;pmid=19775929&amp;doi=10.1016/j.cbpa.2009.08.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R19">
<span class="label">19.</span><cite>Blomme B, Van Steenkiste C, Callewaert N. &amp; Van Vlierberghe H. Alteration of protein glycosylation in liver diseases. Journal of Hepatology
50, 592–603, doi: 10.1016/j.jhep.2008.12.010 (2009).
</cite> [<a href="https://doi.org/10.1016/j.jhep.2008.12.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19157620/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Hepatology&amp;title=Alteration%20of%20protein%20glycosylation%20in%20liver%20diseases&amp;author=B%20Blomme&amp;author=C%20Van%20Steenkiste&amp;author=N%20Callewaert&amp;author=H%20Van%20Vlierberghe&amp;volume=50&amp;publication_year=2009&amp;pages=592-603&amp;pmid=19157620&amp;doi=10.1016/j.jhep.2008.12.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R20">
<span class="label">20.</span><cite>Dall’Olio F. et al. 
N-glycomic biomarkers of biological aging and longevity: A link with inflammaging. Ageing Research Reviews
12, 685–698, doi: 10.1016/j.arr.2012.02.002 (2013).
</cite> [<a href="https://doi.org/10.1016/j.arr.2012.02.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22353383/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ageing%20Research%20Reviews&amp;title=N-glycomic%20biomarkers%20of%20biological%20aging%20and%20longevity:%20A%20link%20with%20inflammaging&amp;author=F%20Dall%E2%80%99Olio&amp;volume=12&amp;publication_year=2013&amp;pages=685-698&amp;pmid=22353383&amp;doi=10.1016/j.arr.2012.02.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R21">
<span class="label">21.</span><cite>Lee S-M
et al. 
N-Glycosylation of asparagine 130 in the extracellular domain of the human calcitonin receptor significantly increases peptide hormone affinity. Biochemistry
56, 3380–3393 (2017).
</cite> [<a href="https://doi.org/10.1021/acs.biochem.7b00256" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5551398/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28614667/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=N-Glycosylation%20of%20asparagine%20130%20in%20the%20extracellular%20domain%20of%20the%20human%20calcitonin%20receptor%20significantly%20increases%20peptide%20hormone%20affinity&amp;author=S-M%20Lee&amp;volume=56&amp;publication_year=2017&amp;pages=3380-3393&amp;pmid=28614667&amp;doi=10.1021/acs.biochem.7b00256&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R22">
<span class="label">22.</span><cite>Veillon L, Fakih C, Abou-El-Hassan H, Kobeissy F. &amp; Mechref Y. Glycosylation changes in brain cancer. ACS chemical neuroscience
9, 51–72 (2017).
</cite> [<a href="https://doi.org/10.1021/acschemneuro.7b00271" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5771830/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28982002/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20chemical%20neuroscience&amp;title=Glycosylation%20changes%20in%20brain%20cancer&amp;author=L%20Veillon&amp;author=C%20Fakih&amp;author=H%20Abou-El-Hassan&amp;author=F%20Kobeissy&amp;author=Y%20Mechref&amp;volume=9&amp;publication_year=2017&amp;pages=51-72&amp;pmid=28982002&amp;doi=10.1021/acschemneuro.7b00271&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R23">
<span class="label">23.</span><cite>Park D. et al. 
Enterocyte glycosylation is responsive to changes in extracellular conditions: implications for membrane functions. Glycobiology
27, 847–860, doi: 10.1093/glycob/cwx041 (2017).
</cite> [<a href="https://doi.org/10.1093/glycob/cwx041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5881715/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28486580/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Glycobiology&amp;title=Enterocyte%20glycosylation%20is%20responsive%20to%20changes%20in%20extracellular%20conditions:%20implications%20for%20membrane%20functions&amp;author=D%20Park&amp;volume=27&amp;publication_year=2017&amp;pages=847-860&amp;pmid=28486580&amp;doi=10.1093/glycob/cwx041&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R24">
<span class="label">24.</span><cite>Park DD
et al. 
Membrane glycomics reveal heterogeneity and quantitative distribution of cell surface sialylation. Chemical science
9, 6271–6285 (2018).
</cite> [<a href="https://doi.org/10.1039/c8sc01875h" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6063140/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30123482/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chemical%20science&amp;title=Membrane%20glycomics%20reveal%20heterogeneity%20and%20quantitative%20distribution%20of%20cell%20surface%20sialylation&amp;author=DD%20Park&amp;volume=9&amp;publication_year=2018&amp;pages=6271-6285&amp;pmid=30123482&amp;doi=10.1039/c8sc01875h&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R25">
<span class="label">25.</span><cite>Li Q, Xie Y, Xu G. &amp; Lebrilla CB
Identification of potential sialic acid binding proteins on cell membranes by proximity chemical labeling. Chemical science
10, 6199–6209 (2019).
</cite> [<a href="https://doi.org/10.1039/c9sc01360a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6585875/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31360427/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chemical%20science&amp;title=Identification%20of%20potential%20sialic%20acid%20binding%20proteins%20on%20cell%20membranes%20by%20proximity%20chemical%20labeling&amp;author=Q%20Li&amp;author=Y%20Xie&amp;author=G%20Xu&amp;author=CB%20Lebrilla&amp;volume=10&amp;publication_year=2019&amp;pages=6199-6209&amp;pmid=31360427&amp;doi=10.1039/c9sc01360a&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R26">
<span class="label">26.</span><cite>Wong M, Xu G, Park D, Barboza M. &amp; Lebrilla CB
Intact glycosphingolipidomic analysis of the cell membrane during differentiation yields extensive glycan and lipid changes. Scientific Reports
8, 10993, doi: 10.1038/s41598-018-29324-7 (2018).
</cite> [<a href="https://doi.org/10.1038/s41598-018-29324-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6054638/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30030471/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Intact%20glycosphingolipidomic%20analysis%20of%20the%20cell%20membrane%20during%20differentiation%20yields%20extensive%20glycan%20and%20lipid%20changes&amp;author=M%20Wong&amp;author=G%20Xu&amp;author=D%20Park&amp;author=M%20Barboza&amp;author=CB%20Lebrilla&amp;volume=8&amp;publication_year=2018&amp;pages=10993&amp;pmid=30030471&amp;doi=10.1038/s41598-018-29324-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R27">
<span class="label">27.</span><cite>Wu S, Tao N, German JB, Grimm R. &amp; Lebrilla CB
Development of an annotated library of neutral human milk oligosaccharides. Journal of proteome research
9, 4138–4151 (2010).
</cite> [<a href="https://doi.org/10.1021/pr100362f" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2919513/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20578730/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20proteome%20research&amp;title=Development%20of%20an%20annotated%20library%20of%20neutral%20human%20milk%20oligosaccharides&amp;author=S%20Wu&amp;author=N%20Tao&amp;author=JB%20German&amp;author=R%20Grimm&amp;author=CB%20Lebrilla&amp;volume=9&amp;publication_year=2010&amp;pages=4138-4151&amp;pmid=20578730&amp;doi=10.1021/pr100362f&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R28">
<span class="label">28.</span><cite>Ninonuevo MR
et al. 
A strategy for annotating the human milk glycome. Journal of agricultural and food chemistry
54, 7471–7480 (2006).
</cite> [<a href="https://doi.org/10.1021/jf0615810" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17002410/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20agricultural%20and%20food%20chemistry&amp;title=A%20strategy%20for%20annotating%20the%20human%20milk%20glycome&amp;author=MR%20Ninonuevo&amp;volume=54&amp;publication_year=2006&amp;pages=7471-7480&amp;pmid=17002410&amp;doi=10.1021/jf0615810&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R29">
<span class="label">29.</span><cite>Barboza M. et al. 
Glycosylation of human milk lactoferrin exhibits dynamic changes during early lactation enhancing its role in pathogenic bacteria-host interactions. Molecular &amp; Cellular Proteomics
11, M111. 015248 (2012).</cite> [<a href="https://doi.org/10.1074/mcp.M111.015248" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3433903/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22261723/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20&amp;%20Cellular%20Proteomics&amp;title=Glycosylation%20of%20human%20milk%20lactoferrin%20exhibits%20dynamic%20changes%20during%20early%20lactation%20enhancing%20its%20role%20in%20pathogenic%20bacteria-host%20interactions&amp;author=M%20Barboza&amp;volume=11&amp;publication_year=2012&amp;pages=M111&amp;pmid=22261723&amp;doi=10.1074/mcp.M111.015248&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R30">
<span class="label">30.</span><cite>Chu CS
et al. 
Profile of native N-linked glycan structures from human serum using high performance liquid chromatography on a microfluidic chip and time - of - flight mass spectrometry. Proteomics
9, 1939–1951 (2009).
</cite> [<a href="https://doi.org/10.1002/pmic.200800249" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2765869/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19288519/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=Profile%20of%20native%20N-linked%20glycan%20structures%20from%20human%20serum%20using%20high%20performance%20liquid%20chromatography%20on%20a%20microfluidic%20chip%20and%20time%20-%20of%20-%20flight%20mass%20spectrometry&amp;author=CS%20Chu&amp;volume=9&amp;publication_year=2009&amp;pages=1939-1951&amp;pmid=19288519&amp;doi=10.1002/pmic.200800249&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R31">
<span class="label">31.</span><cite>Lee H. et al. 
Multiple precursor ion scanning of gangliosides and sulfatides with a reversed-phase microfluidic chip and quadrupole time-of-flight mass spectrometry. Analytical chemistry
84, 5905–5912 (2012).
</cite> [<a href="https://doi.org/10.1021/ac300254d" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3402638/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22697387/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20chemistry&amp;title=Multiple%20precursor%20ion%20scanning%20of%20gangliosides%20and%20sulfatides%20with%20a%20reversed-phase%20microfluidic%20chip%20and%20quadrupole%20time-of-flight%20mass%20spectrometry&amp;author=H%20Lee&amp;volume=84&amp;publication_year=2012&amp;pages=5905-5912&amp;pmid=22697387&amp;doi=10.1021/ac300254d&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R32">
<span class="label">32.</span><cite>An HJ
et al. 
Extensive determination of glycan heterogeneity reveals an unusual abundance of high mannose glycans in enriched plasma membranes of human embryonic stem cells. Molecular &amp; Cellular Proteomics
11, M111. 010660 (2012).</cite> [<a href="https://doi.org/10.1074/mcp.M111.010660" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3322563/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22147732/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20&amp;%20Cellular%20Proteomics&amp;title=Extensive%20determination%20of%20glycan%20heterogeneity%20reveals%20an%20unusual%20abundance%20of%20high%20mannose%20glycans%20in%20enriched%20plasma%20membranes%20of%20human%20embryonic%20stem%20cells&amp;author=HJ%20An&amp;volume=11&amp;publication_year=2012&amp;pages=M111&amp;pmid=22147732&amp;doi=10.1074/mcp.M111.010660&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R33">
<span class="label">33.</span><cite>Park D. et al. 
Characteristic Changes in Cell Surface Glycosylation Accompany Intestinal Epithelial Cell (IEC) Differentiation: High Mannose Structures Dominate the Cell Surface Glycome of Undifferentiated Enterocytes. Molecular &amp; cellular proteomics : MCP
14, 2910–2921, doi: 10.1074/mcp.M115.053983 (2015).
</cite> [<a href="https://doi.org/10.1074/mcp.M115.053983" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4638035/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26355101/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20&amp;%20cellular%20proteomics%20:%20MCP&amp;title=Characteristic%20Changes%20in%20Cell%20Surface%20Glycosylation%20Accompany%20Intestinal%20Epithelial%20Cell%20(IEC)%20Differentiation:%20High%20Mannose%20Structures%20Dominate%20the%20Cell%20Surface%20Glycome%20of%20Undifferentiated%20Enterocytes&amp;author=D%20Park&amp;volume=14&amp;publication_year=2015&amp;pages=2910-2921&amp;pmid=26355101&amp;doi=10.1074/mcp.M115.053983&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R34">
<span class="label">34.</span><cite>Ruhaak LR
et al. 
Differential N-Glycosylation Patterns in Lung Adenocarcinoma Tissue. Journal of Proteome Research
14, 4538–4549, doi: 10.1021/acs.jproteome.5b00255 (2015).
</cite> [<a href="https://doi.org/10.1021/acs.jproteome.5b00255" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4644186/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26322380/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Proteome%20Research&amp;title=Differential%20N-Glycosylation%20Patterns%20in%20Lung%20Adenocarcinoma%20Tissue&amp;author=LR%20Ruhaak&amp;volume=14&amp;publication_year=2015&amp;pages=4538-4549&amp;pmid=26322380&amp;doi=10.1021/acs.jproteome.5b00255&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R35">
<span class="label">35.</span><cite>Ruhaak L. et al. 
Glycan labeling strategies and their use in identification and quantification. Analytical and bioanalytical chemistry
397, 3457–3481 (2010).
</cite> [<a href="https://doi.org/10.1007/s00216-010-3532-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2911528/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20225063/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20and%20bioanalytical%20chemistry&amp;title=Glycan%20labeling%20strategies%20and%20their%20use%20in%20identification%20and%20quantification&amp;author=L%20Ruhaak&amp;volume=397&amp;publication_year=2010&amp;pages=3457-3481&amp;pmid=20225063&amp;doi=10.1007/s00216-010-3532-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R36">
<span class="label">36.</span><cite>Goldstein IJ, Winter HC &amp; Poretz RD in New Comprehensive Biochemistry Vol. 29 (eds Montreuil J, Vliegenthart JFG, &amp; Schachter H) 403–474 (Elsevier, 1997).</cite> [<a href="https://scholar.google.com/scholar_lookup?title=New%20Comprehensive%20Biochemistry&amp;author=IJ%20Goldstein&amp;author=HC%20Winter&amp;author=RD%20Poretz&amp;author=J%20Montreuil&amp;author=JFG%20Vliegenthart&amp;publication_year=1997&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R37">
<span class="label">37.</span><cite>Lee LY
et al. 
An optimized approach for enrichment of glycoproteins from cell culture lysates using native multi-lectin affinity chromatography. Journal of separation science
35, 2445–2452 (2012).
</cite> [<a href="https://doi.org/10.1002/jssc.201200049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22997032/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20separation%20science&amp;title=An%20optimized%20approach%20for%20enrichment%20of%20glycoproteins%20from%20cell%20culture%20lysates%20using%20native%20multi-lectin%20affinity%20chromatography&amp;author=LY%20Lee&amp;volume=35&amp;publication_year=2012&amp;pages=2445-2452&amp;pmid=22997032&amp;doi=10.1002/jssc.201200049&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R38">
<span class="label">38.</span><cite>Palaniappan KK &amp; Bertozzi CR
Chemical Glycoproteomics. Chemical Reviews
116, 14277–14306, doi: 10.1021/acs.chemrev.6b00023 (2016).
</cite> [<a href="https://doi.org/10.1021/acs.chemrev.6b00023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5327817/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27960262/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chemical%20Reviews&amp;title=Chemical%20Glycoproteomics&amp;author=KK%20Palaniappan&amp;author=CR%20Bertozzi&amp;volume=116&amp;publication_year=2016&amp;pages=14277-14306&amp;pmid=27960262&amp;doi=10.1021/acs.chemrev.6b00023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R39">
<span class="label">39.</span><cite>Kuhn PH
et al. 
Secretome protein enrichment identifies physiological BACE1 protease substrates in neurons. The EMBO journal
31, 3157–3168, doi: 10.1038/emboj.2012.173 (2012).
</cite> [<a href="https://doi.org/10.1038/emboj.2012.173" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3400020/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22728825/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20EMBO%20journal&amp;title=Secretome%20protein%20enrichment%20identifies%20physiological%20BACE1%20protease%20substrates%20in%20neurons&amp;author=PH%20Kuhn&amp;volume=31&amp;publication_year=2012&amp;pages=3157-3168&amp;pmid=22728825&amp;doi=10.1038/emboj.2012.173&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R40">
<span class="label">40.</span><cite>Haun RS
et al. 
Bioorthogonal labeling cell-surface proteins expressed in pancreatic cancer cells to identify potential diagnostic/therapeutic biomarkers. Cancer biology &amp; therapy
16, 1557–1565 (2015).
</cite> [<a href="https://doi.org/10.1080/15384047.2015.1071740" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4846140/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26176765/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20biology%20&amp;%20therapy&amp;title=Bioorthogonal%20labeling%20cell-surface%20proteins%20expressed%20in%20pancreatic%20cancer%20cells%20to%20identify%20potential%20diagnostic/therapeutic%20biomarkers&amp;author=RS%20Haun&amp;volume=16&amp;publication_year=2015&amp;pages=1557-1565&amp;pmid=26176765&amp;doi=10.1080/15384047.2015.1071740&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R41">
<span class="label">41.</span><cite>Zacharias LG
et al. 
HILIC and ERLIC Enrichment of Glycopeptides Derived from Breast and Brain Cancer Cells. Journal of Proteome Research
15, 3624–3634, doi: 10.1021/acs.jproteome.6b00429 (2016).
</cite> [<a href="https://doi.org/10.1021/acs.jproteome.6b00429" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5748158/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27533485/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Proteome%20Research&amp;title=HILIC%20and%20ERLIC%20Enrichment%20of%20Glycopeptides%20Derived%20from%20Breast%20and%20Brain%20Cancer%20Cells&amp;author=LG%20Zacharias&amp;volume=15&amp;publication_year=2016&amp;pages=3624-3634&amp;pmid=27533485&amp;doi=10.1021/acs.jproteome.6b00429&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R42">
<span class="label">42.</span><cite>Sibille E. et al. 
Ganglioside profiling of the human retina: comparison with other ocular structures, brain and plasma reveals tissue specificities. PloS one
11, e0168794 (2016).</cite> [<a href="https://doi.org/10.1371/journal.pone.0168794" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5173345/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27997589/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PloS%20one&amp;title=Ganglioside%20profiling%20of%20the%20human%20retina:%20comparison%20with%20other%20ocular%20structures,%20brain%20and%20plasma%20reveals%20tissue%20specificities&amp;author=E%20Sibille&amp;volume=11&amp;publication_year=2016&amp;pmid=27997589&amp;doi=10.1371/journal.pone.0168794&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R43">
<span class="label">43.</span><cite>Lapainis T, Rubakhin SS &amp; Sweedler JV
Capillary electrophoresis with electrospray ionization mass spectrometric detection for single-cell metabolomics. Analytical chemistry
81, 5858–5864 (2009).
</cite> [<a href="https://doi.org/10.1021/ac900936g" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2810313/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19518091/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20chemistry&amp;title=Capillary%20electrophoresis%20with%20electrospray%20ionization%20mass%20spectrometric%20detection%20for%20single-cell%20metabolomics&amp;author=T%20Lapainis&amp;author=SS%20Rubakhin&amp;author=JV%20Sweedler&amp;volume=81&amp;publication_year=2009&amp;pages=5858-5864&amp;pmid=19518091&amp;doi=10.1021/ac900936g&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R44">
<span class="label">44.</span><cite>Mellors J, Gorbounov V, Ramsey R. &amp; Ramsey J. Fully integrated glass microfluidic device for performing high-efficiency capillary electrophoresis and electrospray ionization mass spectrometry. Analytical chemistry
80, 6881–6887 (2008).
</cite> [<a href="https://doi.org/10.1021/ac800428w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3125599/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18698800/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20chemistry&amp;title=Fully%20integrated%20glass%20microfluidic%20device%20for%20performing%20high-efficiency%20capillary%20electrophoresis%20and%20electrospray%20ionization%20mass%20spectrometry&amp;author=J%20Mellors&amp;author=V%20Gorbounov&amp;author=R%20Ramsey&amp;author=J%20Ramsey&amp;volume=80&amp;publication_year=2008&amp;pages=6881-6887&amp;pmid=18698800&amp;doi=10.1021/ac800428w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R45">
<span class="label">45.</span><cite>Ruhaak LR, Xu G, Li Q, Goonatilleke E. &amp; Lebrilla CB
Mass spectrometry approaches to glycomic and glycoproteomic analyses. Chemical reviews
118, 7886–7930 (2018).
</cite> [<a href="https://doi.org/10.1021/acs.chemrev.7b00732" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7757723/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29553244/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chemical%20reviews&amp;title=Mass%20spectrometry%20approaches%20to%20glycomic%20and%20glycoproteomic%20analyses&amp;author=LR%20Ruhaak&amp;author=G%20Xu&amp;author=Q%20Li&amp;author=E%20Goonatilleke&amp;author=CB%20Lebrilla&amp;volume=118&amp;publication_year=2018&amp;pages=7886-7930&amp;pmid=29553244&amp;doi=10.1021/acs.chemrev.7b00732&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R46">
<span class="label">46.</span><cite>Both P. et al. 
Discrimination of epimeric glycans and glycopeptides using IM-MS and its potential for carbohydrate sequencing. Nature chemistry
6, 65 (2014).</cite> [<a href="https://doi.org/10.1038/nchem.1817" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24345949/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20chemistry&amp;title=Discrimination%20of%20epimeric%20glycans%20and%20glycopeptides%20using%20IM-MS%20and%20its%20potential%20for%20carbohydrate%20sequencing&amp;author=P%20Both&amp;volume=6&amp;publication_year=2014&amp;pages=65&amp;pmid=24345949&amp;doi=10.1038/nchem.1817&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R47">
<span class="label">47.</span><cite>Plasencia MD, Isailovic D, Merenbloom SI, Mechref Y. &amp; Clemmer DE
Resolving and assigning N-linked glycan structural isomers from ovalbumin by IMS-MS. Journal of the American Society for Mass Spectrometry
19, 1706–1715 (2008).
</cite> [<a href="https://doi.org/10.1016/j.jasms.2008.07.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2626179/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18760624/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Society%20for%20Mass%20Spectrometry&amp;title=Resolving%20and%20assigning%20N-linked%20glycan%20structural%20isomers%20from%20ovalbumin%20by%20IMS-MS&amp;author=MD%20Plasencia&amp;author=D%20Isailovic&amp;author=SI%20Merenbloom&amp;author=Y%20Mechref&amp;author=DE%20Clemmer&amp;volume=19&amp;publication_year=2008&amp;pages=1706-1715&amp;pmid=18760624&amp;doi=10.1016/j.jasms.2008.07.020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R48">
<span class="label">48.</span><cite>Hofmann J. et al. 
Identification of Lewis and Blood Group Carbohydrate Epitopes by Ion Mobility-Tandem-Mass Spectrometry Fingerprinting. Analytical Chemistry
89, 2318–2325, doi: 10.1021/acs.analchem.6b03853 (2017).
</cite> [<a href="https://doi.org/10.1021/acs.analchem.6b03853" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28192913/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=Identification%20of%20Lewis%20and%20Blood%20Group%20Carbohydrate%20Epitopes%20by%20Ion%20Mobility-Tandem-Mass%20Spectrometry%20Fingerprinting&amp;author=J%20Hofmann&amp;volume=89&amp;publication_year=2017&amp;pages=2318-2325&amp;pmid=28192913&amp;doi=10.1021/acs.analchem.6b03853&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R49">
<span class="label">49.</span><cite>Leopold J, Popkova Y, Engel KM &amp; Schiller J. Recent developments of useful MALDI matrices for the mass spectrometric characterization of lipids. Biomolecules
8, 173 (2018).
</cite> [<a href="https://doi.org/10.3390/biom8040173" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6316665/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30551655/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomolecules&amp;title=Recent%20developments%20of%20useful%20MALDI%20matrices%20for%20the%20mass%20spectrometric%20characterization%20of%20lipids&amp;author=J%20Leopold&amp;author=Y%20Popkova&amp;author=KM%20Engel&amp;author=J%20Schiller&amp;volume=8&amp;publication_year=2018&amp;pages=173&amp;pmid=30551655&amp;doi=10.3390/biom8040173&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R50">
<span class="label">50.</span><cite>Seipert RR
et al. 
Factors that influence fragmentation behavior of N-linked glycopeptide ions. Analytical chemistry
80, 3684–3692, doi: 10.1021/ac800067y (2008).
</cite> [<a href="https://doi.org/10.1021/ac800067y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6551518/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18363335/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20chemistry&amp;title=Factors%20that%20influence%20fragmentation%20behavior%20of%20N-linked%20glycopeptide%20ions&amp;author=RR%20Seipert&amp;volume=80&amp;publication_year=2008&amp;pages=3684-3692&amp;pmid=18363335&amp;doi=10.1021/ac800067y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R51">
<span class="label">51.</span><cite>Ruhaak LR, Xu G, Li Q, Goonatilleke E. &amp; Lebrilla CB
Mass Spectrometry Approaches to Glycomic and Glycoproteomic Analyses. Chemical Reviews, doi: 10.1021/acs.chemrev.7b00732 (2018).</cite> [<a href="https://doi.org/10.1021/acs.chemrev.7b00732" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7757723/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29553244/" class="usa-link">PubMed</a>]</li>
<li id="R52">
<span class="label">52.</span><cite>Cao L. et al. 
Intact glycopeptide characterization using mass spectrometry. Expert Review of Proteomics
13, 513–522, doi: 10.1586/14789450.2016.1172965 (2016).
</cite> [<a href="https://doi.org/10.1586/14789450.2016.1172965" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4955807/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27140194/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Expert%20Review%20of%20Proteomics&amp;title=Intact%20glycopeptide%20characterization%20using%20mass%20spectrometry&amp;author=L%20Cao&amp;volume=13&amp;publication_year=2016&amp;pages=513-522&amp;pmid=27140194&amp;doi=10.1586/14789450.2016.1172965&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R53">
<span class="label">53.</span><cite>Yu Q. et al. 
Electron-Transfer/Higher-Energy Collision Dissociation (EThcD)-Enabled Intact Glycopeptide/Glycoproteome Characterization. Journal of The American Society for Mass Spectrometry
28, 1751–1764, doi: 10.1007/s13361-017-1701-4 (2017).
</cite> [<a href="https://doi.org/10.1007/s13361-017-1701-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5711575/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28695533/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20The%20American%20Society%20for%20Mass%20Spectrometry&amp;title=Electron-Transfer/Higher-Energy%20Collision%20Dissociation%20(EThcD)-Enabled%20Intact%20Glycopeptide/Glycoproteome%20Characterization&amp;author=Q%20Yu&amp;volume=28&amp;publication_year=2017&amp;pages=1751-1764&amp;pmid=28695533&amp;doi=10.1007/s13361-017-1701-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R54">
<span class="label">54.</span><cite>Forgue-Lafitte M-E, Coudray A-M, Bréant B. &amp; Mešter J. Proliferation of the human colon carcinoma cell line HT29: autocrine growth and deregulated expression of the c-myc oncogene. Cancer research
49, 6566–6571 (1989).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/2684395/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20research&amp;title=Proliferation%20of%20the%20human%20colon%20carcinoma%20cell%20line%20HT29:%20autocrine%20growth%20and%20deregulated%20expression%20of%20the%20c-myc%20oncogene&amp;author=M-E%20Forgue-Lafitte&amp;author=A-M%20Coudray&amp;author=B%20Br%C3%A9ant&amp;author=J%20Me%C5%A1ter&amp;volume=49&amp;publication_year=1989&amp;pages=6566-6571&amp;pmid=2684395&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R55">
<span class="label">55.</span><cite>Dey PM, Brownleader MD &amp; Harborne JB in Plant Biochemistry (eds Dey PM &amp; Harborne JB) 1–47 (Academic Press, 1997).</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Plant%20Biochemistry&amp;author=PM%20Dey&amp;author=MD%20Brownleader&amp;author=JB%20Harborne&amp;author=PM%20Dey&amp;author=JB%20Harborne&amp;publication_year=1997&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R56">
<span class="label">56.</span><cite>Banneau G, Ayadi M, Armenoult L. &amp; Carvalho E. Homogenization of cartilage tumors to extract total RNA to microarray and sequencing analysis using Precellys bead-beating technology. Biotechniques
52, 196–197 (2012).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Biotechniques&amp;title=Homogenization%20of%20cartilage%20tumors%20to%20extract%20total%20RNA%20to%20microarray%20and%20sequencing%20analysis%20using%20Precellys%20bead-beating%20technology&amp;author=G%20Banneau&amp;author=M%20Ayadi&amp;author=L%20Armenoult&amp;author=E%20Carvalho&amp;volume=52&amp;publication_year=2012&amp;pages=196-197&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R57">
<span class="label">57.</span><cite>Fujiki Y, Hubbard AL, Fowler S. &amp; Lazarow PB
Isolation of intracellular membranes by means of sodium carbonate treatment: application to endoplasmic reticulum. The Journal of cell biology
93, 97–102 (1982).
</cite> [<a href="https://doi.org/10.1083/jcb.93.1.97" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2112113/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7068762/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20cell%20biology&amp;title=Isolation%20of%20intracellular%20membranes%20by%20means%20of%20sodium%20carbonate%20treatment:%20application%20to%20endoplasmic%20reticulum&amp;author=Y%20Fujiki&amp;author=AL%20Hubbard&amp;author=S%20Fowler&amp;author=PB%20Lazarow&amp;volume=93&amp;publication_year=1982&amp;pages=97-102&amp;pmid=7068762&amp;doi=10.1083/jcb.93.1.97&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R58">
<span class="label">58.</span><cite>Li Q, Xie Y, Wong M. &amp; Lebrilla BC
Characterization of Cell Glycocalyx with Mass Spectrometry Methods. Cells
8, doi: 10.3390/cells8080882 (2019).</cite> [<a href="https://doi.org/10.3390/cells8080882" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6721671/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31412618/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cells&amp;title=Characterization%20of%20Cell%20Glycocalyx%20with%20Mass%20Spectrometry%20Methods&amp;author=Q%20Li&amp;author=Y%20Xie&amp;author=M%20Wong&amp;author=BC%20Lebrilla&amp;volume=8&amp;publication_year=2019&amp;pmid=31412618&amp;doi=10.3390/cells8080882&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R59">
<span class="label">59.</span><cite>Sandoval WN
et al. 
Rapid removal of N-linked oligosaccharides using microwave assisted enzyme catalyzed deglycosylation. International Journal of Mass Spectrometry
259, 117–123, doi: 10.1016/j.ijms.2006.09.003 (2007).</cite> [<a href="https://doi.org/10.1016/j.ijms.2006.09.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Mass%20Spectrometry&amp;title=Rapid%20removal%20of%20N-linked%20oligosaccharides%20using%20microwave%20assisted%20enzyme%20catalyzed%20deglycosylation&amp;author=WN%20Sandoval&amp;volume=259&amp;publication_year=2007&amp;pages=117-123&amp;doi=10.1016/j.ijms.2006.09.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R60">
<span class="label">60.</span><cite>Lauber MA
et al. 
Rapid Preparation of Released N-Glycans for HILIC Analysis Using a Labeling Reagent that Facilitates Sensitive Fluorescence and ESI-MS Detection. Analytical Chemistry
87, 5401–5409, doi: 10.1021/acs.analchem.5b00758 (2015).
</cite> [<a href="https://doi.org/10.1021/acs.analchem.5b00758" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25927596/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=Rapid%20Preparation%20of%20Released%20N-Glycans%20for%20HILIC%20Analysis%20Using%20a%20Labeling%20Reagent%20that%20Facilitates%20Sensitive%20Fluorescence%20and%20ESI-MS%20Detection&amp;author=MA%20Lauber&amp;volume=87&amp;publication_year=2015&amp;pages=5401-5409&amp;pmid=25927596&amp;doi=10.1021/acs.analchem.5b00758&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R61">
<span class="label">61.</span><cite>Van Ree R. et al. 
β (1, 2)-xylose and α (1, 3)-fucose residues have a strong contribution in IgE binding to plant glycoallergens. Journal of Biological Chemistry
275, 11451–11458 (2000).
</cite> [<a href="https://doi.org/10.1074/jbc.275.15.11451" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10753962/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=%CE%B2%20(1,%202)-xylose%20and%20%CE%B1%20(1,%203)-fucose%20residues%20have%20a%20strong%20contribution%20in%20IgE%20binding%20to%20plant%20glycoallergens&amp;author=R%20Van%20Ree&amp;volume=275&amp;publication_year=2000&amp;pages=11451-11458&amp;pmid=10753962&amp;doi=10.1074/jbc.275.15.11451&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R62">
<span class="label">62.</span><cite>Ruhaak LR
et al. 
Glycan labeling strategies and their use in identification and quantification. Analytical and Bioanalytical Chemistry
397, 3457–3481, doi: 10.1007/s00216-010-3532-z (2010).
</cite> [<a href="https://doi.org/10.1007/s00216-010-3532-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2911528/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20225063/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20and%20Bioanalytical%20Chemistry&amp;title=Glycan%20labeling%20strategies%20and%20their%20use%20in%20identification%20and%20quantification&amp;author=LR%20Ruhaak&amp;volume=397&amp;publication_year=2010&amp;pages=3457-3481&amp;pmid=20225063&amp;doi=10.1007/s00216-010-3532-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R63">
<span class="label">63.</span><cite>Hua S. et al. 
Comprehensive native glycan profiling with isomer separation and quantitation for the discovery of cancer biomarkers. Analyst
136, 3663–3671 (2011).
</cite> [<a href="https://doi.org/10.1039/c1an15093f" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3331797/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21776491/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analyst&amp;title=Comprehensive%20native%20glycan%20profiling%20with%20isomer%20separation%20and%20quantitation%20for%20the%20discovery%20of%20cancer%20biomarkers&amp;author=S%20Hua&amp;volume=136&amp;publication_year=2011&amp;pages=3663-3671&amp;pmid=21776491&amp;doi=10.1039/c1an15093f&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R64">
<span class="label">64.</span><cite>Park D. et al. 
Salmonella Typhimurium Enzymatically Landscapes the Host Intestinal Epithelial Cell (IEC) Surface Glycome to Increase Invasion. Molecular &amp; Cellular Proteomics
15, 3653–3664, doi: 10.1074/mcp.M116.063206 (2016).
</cite> [<a href="https://doi.org/10.1074/mcp.M116.063206" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5141278/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27754876/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20&amp;%20Cellular%20Proteomics&amp;title=Salmonella%20Typhimurium%20Enzymatically%20Landscapes%20the%20Host%20Intestinal%20Epithelial%20Cell%20(IEC)%20Surface%20Glycome%20to%20Increase%20Invasion&amp;author=D%20Park&amp;volume=15&amp;publication_year=2016&amp;pages=3653-3664&amp;pmid=27754876&amp;doi=10.1074/mcp.M116.063206&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R65">
<span class="label">65.</span><cite>Folch J, Lees M. &amp; Sloane-Stanley G. A simple method for the isolation and purification of total lipids from animal tissues. The Journal of biological chemistry
226, 497–509 (1957).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/13428781/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20biological%20chemistry&amp;title=A%20simple%20method%20for%20the%20isolation%20and%20purification%20of%20total%20lipids%20from%20animal%20tissues&amp;author=J%20Folch&amp;author=M%20Lees&amp;author=G%20Sloane-Stanley&amp;volume=226&amp;publication_year=1957&amp;pages=497-509&amp;pmid=13428781&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R66">
<span class="label">66.</span><cite>Bligh EG &amp; Dyer WJ
A Rapid Method of Total Lipid Extraction and Purification. Canadian Journal of Biochemistry and Physiology
37, 911–917, doi: 10.1139/o59-099 (1959).
</cite> [<a href="https://doi.org/10.1139/o59-099" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/13671378/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Canadian%20Journal%20of%20Biochemistry%20and%20Physiology&amp;title=A%20Rapid%20Method%20of%20Total%20Lipid%20Extraction%20and%20Purification&amp;author=EG%20Bligh&amp;author=WJ%20Dyer&amp;volume=37&amp;publication_year=1959&amp;pages=911-917&amp;pmid=13671378&amp;doi=10.1139/o59-099&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R67">
<span class="label">67.</span><cite>Miura Y. et al. 
Glycoblotting-assisted O-glycomics: ammonium carbamate allows for highly efficient O-glycan release from glycoproteins. Analytical chemistry
82, 10021–10029 (2010).
</cite> [<a href="https://doi.org/10.1021/ac101599p" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21077635/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20chemistry&amp;title=Glycoblotting-assisted%20O-glycomics:%20ammonium%20carbamate%20allows%20for%20highly%20efficient%20O-glycan%20release%20from%20glycoproteins&amp;author=Y%20Miura&amp;volume=82&amp;publication_year=2010&amp;pages=10021-10029&amp;pmid=21077635&amp;doi=10.1021/ac101599p&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R68">
<span class="label">68.</span><cite>Sun S, Zhou J-Y, Yang W. &amp; Zhang H. Inhibition of protein carbamylation in urea solution using ammonium-containing buffers. Analytical biochemistry
446, 76–81 (2014).
</cite> [<a href="https://doi.org/10.1016/j.ab.2013.10.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4072244/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24161613/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20biochemistry&amp;title=Inhibition%20of%20protein%20carbamylation%20in%20urea%20solution%20using%20ammonium-containing%20buffers&amp;author=S%20Sun&amp;author=J-Y%20Zhou&amp;author=W%20Yang&amp;author=H%20Zhang&amp;volume=446&amp;publication_year=2014&amp;pages=76-81&amp;pmid=24161613&amp;doi=10.1016/j.ab.2013.10.024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R69">
<span class="label">69.</span><cite>Li Q. et al. 
Site-specific glycosylation quantitation of 50 serum glycoproteins enhanced by predictive glycopeptidomics for improved disease biomarker discovery. Analytical chemistry
91, 5433–5445 (2019).
</cite> [<a href="https://doi.org/10.1021/acs.analchem.9b00776" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30882205/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20chemistry&amp;title=Site-specific%20glycosylation%20quantitation%20of%2050%20serum%20glycoproteins%20enhanced%20by%20predictive%20glycopeptidomics%20for%20improved%20disease%20biomarker%20discovery&amp;author=Q%20Li&amp;volume=91&amp;publication_year=2019&amp;pages=5433-5445&amp;pmid=30882205&amp;doi=10.1021/acs.analchem.9b00776&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R70">
<span class="label">70.</span><cite>Du Y, Wang F, May K, Xu W. &amp; Liu H. LC–MS analysis of glycopeptides of recombinant monoclonal antibodies by a rapid digestion procedure. Journal of Chromatography B
907, 87–93 (2012).</cite> [<a href="https://doi.org/10.1016/j.jchromb.2012.09.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23036907/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Chromatography%20B&amp;title=LC%E2%80%93MS%20analysis%20of%20glycopeptides%20of%20recombinant%20monoclonal%20antibodies%20by%20a%20rapid%20digestion%20procedure&amp;author=Y%20Du&amp;author=F%20Wang&amp;author=K%20May&amp;author=W%20Xu&amp;author=H%20Liu&amp;volume=907&amp;publication_year=2012&amp;pages=87-93&amp;pmid=23036907&amp;doi=10.1016/j.jchromb.2012.09.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R71">
<span class="label">71.</span><cite>Kalli A, Smith GT, Sweredoski MJ &amp; Hess S. Evaluation and optimization of mass spectrometric settings during data-dependent acquisition mode: focus on LTQ-Orbitrap mass analyzers. Journal of proteome research
12, 3071–3086 (2013).
</cite> [<a href="https://doi.org/10.1021/pr3011588" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3748959/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23642296/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20proteome%20research&amp;title=Evaluation%20and%20optimization%20of%20mass%20spectrometric%20settings%20during%20data-dependent%20acquisition%20mode:%20focus%20on%20LTQ-Orbitrap%20mass%20analyzers&amp;author=A%20Kalli&amp;author=GT%20Smith&amp;author=MJ%20Sweredoski&amp;author=S%20Hess&amp;volume=12&amp;publication_year=2013&amp;pages=3071-3086&amp;pmid=23642296&amp;doi=10.1021/pr3011588&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R72">
<span class="label">72.</span><cite>Kalli A. &amp; Hess S. Effect of mass spectrometric parameters on peptide and protein identification rates for shotgun proteomic experiments on an LTQ-orbitrap mass analyzer. Proteomics
12, 21–31 (2012).
</cite> [<a href="https://doi.org/10.1002/pmic.201100464" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22065615/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=Effect%20of%20mass%20spectrometric%20parameters%20on%20peptide%20and%20protein%20identification%20rates%20for%20shotgun%20proteomic%20experiments%20on%20an%20LTQ-orbitrap%20mass%20analyzer&amp;author=A%20Kalli&amp;author=S%20Hess&amp;volume=12&amp;publication_year=2012&amp;pages=21-31&amp;pmid=22065615&amp;doi=10.1002/pmic.201100464&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R73">
<span class="label">73.</span><cite>Yang H, Yang C. &amp; Sun T. Characterization of glycopeptides using a stepped higher-energy C-trap dissociation approach on a hybrid quadrupole orbitrap. Rapid Communications in Mass Spectrometry
32, 1353–1362 (2018).
</cite> [<a href="https://doi.org/10.1002/rcm.8191" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29873418/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Rapid%20Communications%20in%20Mass%20Spectrometry&amp;title=Characterization%20of%20glycopeptides%20using%20a%20stepped%20higher-energy%20C-trap%20dissociation%20approach%20on%20a%20hybrid%20quadrupole%20orbitrap&amp;author=H%20Yang&amp;author=C%20Yang&amp;author=T%20Sun&amp;volume=32&amp;publication_year=2018&amp;pages=1353-1362&amp;pmid=29873418&amp;doi=10.1002/rcm.8191&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R74">
<span class="label">74.</span><cite>Chen Z. et al. 
Site-specific characterization and quantitation of N-glycopeptides in PKM2 knockout breast cancer cells using DiLeu isobaric tags enabled by electron-transfer/higher-energy collision dissociation (EThcD). Analyst
143, 2508–2519 (2018).
</cite> [<a href="https://doi.org/10.1039/c8an00216a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5975206/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29687791/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analyst&amp;title=Site-specific%20characterization%20and%20quantitation%20of%20N-glycopeptides%20in%20PKM2%20knockout%20breast%20cancer%20cells%20using%20DiLeu%20isobaric%20tags%20enabled%20by%20electron-transfer/higher-energy%20collision%20dissociation%20(EThcD)&amp;author=Z%20Chen&amp;volume=143&amp;publication_year=2018&amp;pages=2508-2519&amp;pmid=29687791&amp;doi=10.1039/c8an00216a&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R75">
<span class="label">75.</span><cite>Singh C, Zampronio CG, Creese AJ &amp; Cooper HJ
Higher energy collision dissociation (HCD) product ion-triggered electron transfer dissociation (ETD) mass spectrometry for the analysis of N-linked glycoproteins. Journal of proteome research
11, 4517–4525 (2012).
</cite> [<a href="https://doi.org/10.1021/pr300257c" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22800195/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20proteome%20research&amp;title=Higher%20energy%20collision%20dissociation%20(HCD)%20product%20ion-triggered%20electron%20transfer%20dissociation%20(ETD)%20mass%20spectrometry%20for%20the%20analysis%20of%20N-linked%20glycoproteins&amp;author=C%20Singh&amp;author=CG%20Zampronio&amp;author=AJ%20Creese&amp;author=HJ%20Cooper&amp;volume=11&amp;publication_year=2012&amp;pages=4517-4525&amp;pmid=22800195&amp;doi=10.1021/pr300257c&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R76">
<span class="label">76.</span><cite>Riley NM, Hebert AS, Westphall MS &amp; Coon JJ
Capturing site-specific heterogeneity with large-scale N-glycoproteome analysis. Nature Communications
10, 1311, doi: 10.1038/s41467-019-09222-w (2019).</cite> [<a href="https://doi.org/10.1038/s41467-019-09222-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6428843/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30899004/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Capturing%20site-specific%20heterogeneity%20with%20large-scale%20N-glycoproteome%20analysis&amp;author=NM%20Riley&amp;author=AS%20Hebert&amp;author=MS%20Westphall&amp;author=JJ%20Coon&amp;volume=10&amp;publication_year=2019&amp;pages=1311&amp;pmid=30899004&amp;doi=10.1038/s41467-019-09222-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R77">
<span class="label">77.</span><cite>Kronewitter SR
et al. 
The development of retrosynthetic glycan libraries to profile and classify the human serum N-linked glycome. PROTEOMICS
9, 2986–2994, doi: 10.1002/pmic.200800760 (2009).
</cite> [<a href="https://doi.org/10.1002/pmic.200800760" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2758495/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19452454/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PROTEOMICS&amp;title=The%20development%20of%20retrosynthetic%20glycan%20libraries%20to%20profile%20and%20classify%20the%20human%20serum%20N-linked%20glycome&amp;author=SR%20Kronewitter&amp;volume=9&amp;publication_year=2009&amp;pages=2986-2994&amp;pmid=19452454&amp;doi=10.1002/pmic.200800760&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R78">
<span class="label">78.</span><cite>Senger RS &amp; Karim MN
Prediction of N-linked glycan branching patterns using artificial neural networks. Mathematical Biosciences
211, 89–104, doi: 10.1016/j.mbs.2007.10.005 (2008).
</cite> [<a href="https://doi.org/10.1016/j.mbs.2007.10.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18054050/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mathematical%20Biosciences&amp;title=Prediction%20of%20N-linked%20glycan%20branching%20patterns%20using%20artificial%20neural%20networks&amp;author=RS%20Senger&amp;author=MN%20Karim&amp;volume=211&amp;publication_year=2008&amp;pages=89-104&amp;pmid=18054050&amp;doi=10.1016/j.mbs.2007.10.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R79">
<span class="label">79.</span><cite>Jackson S. &amp; Nicolson SW
Xylose as a nectar sugar: from biochemistry to ecology. Comparative Biochemistry and Physiology Part B: Biochemistry and Molecular Biology
131, 613–620 (2002).</cite> [<a href="https://doi.org/10.1016/s1096-4959(02)00028-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11923077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comparative%20Biochemistry%20and%20Physiology%20Part%20B:%20Biochemistry%20and%20Molecular%20Biology&amp;title=Xylose%20as%20a%20nectar%20sugar:%20from%20biochemistry%20to%20ecology&amp;author=S%20Jackson&amp;author=SW%20Nicolson&amp;volume=131&amp;publication_year=2002&amp;pages=613-620&amp;pmid=11923077&amp;doi=10.1016/s1096-4959(02)00028-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R80">
<span class="label">80.</span><cite>Samraj AN
et al. 
A red meat-derived glycan promotes inflammation and cancer progression. Proceedings of the National Academy of Sciences
112, 542–547 (2015).</cite> [<a href="https://doi.org/10.1073/pnas.1417508112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4299224/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25548184/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=A%20red%20meat-derived%20glycan%20promotes%20inflammation%20and%20cancer%20progression&amp;author=AN%20Samraj&amp;volume=112&amp;publication_year=2015&amp;pages=542-547&amp;pmid=25548184&amp;doi=10.1073/pnas.1417508112&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R81">
<span class="label">81.</span><cite>Chou H-H
et al. 
A mutation in human CMP-sialic acid hydroxylase occurred after the Homo-Pan divergence. Proceedings of the National Academy of Sciences
95, 11751–11756 (1998).</cite> [<a href="https://doi.org/10.1073/pnas.95.20.11751" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC21712/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9751737/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=A%20mutation%20in%20human%20CMP-sialic%20acid%20hydroxylase%20occurred%20after%20the%20Homo-Pan%20divergence&amp;author=H-H%20Chou&amp;volume=95&amp;publication_year=1998&amp;pages=11751-11756&amp;pmid=9751737&amp;doi=10.1073/pnas.95.20.11751&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R82">
<span class="label">82.</span><cite>Tra VN &amp; Dube DH
Glycans in pathogenic bacteria–potential for targeted covalent therapeutics and imaging agents. Chemical Communications
50, 4659–4673 (2014).
</cite> [<a href="https://doi.org/10.1039/c4cc00660g" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4049282/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24647371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chemical%20Communications&amp;title=Glycans%20in%20pathogenic%20bacteria%E2%80%93potential%20for%20targeted%20covalent%20therapeutics%20and%20imaging%20agents&amp;author=VN%20Tra&amp;author=DH%20Dube&amp;volume=50&amp;publication_year=2014&amp;pages=4659-4673&amp;pmid=24647371&amp;doi=10.1039/c4cc00660g&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R83">
<span class="label">83.</span><cite>Yamakawa N. et al. 
Systems glycomics of adult zebrafish identifies organ-specific sialylation and glycosylation patterns. Nature communications
9, 4647 (2018).</cite> [<a href="https://doi.org/10.1038/s41467-018-06950-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6220181/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30405127/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20communications&amp;title=Systems%20glycomics%20of%20adult%20zebrafish%20identifies%20organ-specific%20sialylation%20and%20glycosylation%20patterns&amp;author=N%20Yamakawa&amp;volume=9&amp;publication_year=2018&amp;pages=4647&amp;pmid=30405127&amp;doi=10.1038/s41467-018-06950-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R84">
<span class="label">84.</span><cite>Hartley MD
et al. 
Biochemical characterization of the O-linked glycosylation pathway in Neisseria gonorrhoeae responsible for biosynthesis of protein glycans containing N, - N′diacetylbacillosamine. Biochemistry
50, 4936–4948 (2011).
</cite> [<a href="https://doi.org/10.1021/bi2003372" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3108506/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21542610/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Biochemical%20characterization%20of%20the%20O-linked%20glycosylation%20pathway%20in%20Neisseria%20gonorrhoeae%20responsible%20for%20biosynthesis%20of%20protein%20glycans%20containing%20N,%20-%20N%E2%80%B2diacetylbacillosamine&amp;author=MD%20Hartley&amp;volume=50&amp;publication_year=2011&amp;pages=4936-4948&amp;pmid=21542610&amp;doi=10.1021/bi2003372&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R85">
<span class="label">85.</span><cite>Merrill AH
SphinGOMAP: Lipidomic analysis of [glyco]sphingolipid metabolism. The FASEB Journal
20, A1472 (2006).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=The%20FASEB%20Journal&amp;title=SphinGOMAP:%20Lipidomic%20analysis%20of%20%5Bglyco%5Dsphingolipid%20metabolism&amp;author=AH%20Merrill&amp;volume=20&amp;publication_year=2006&amp;pages=A1472&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R86">
<span class="label">86.</span><cite>Zeng W-F
et al. 
pGlyco: a pipeline for the identification of intact N-glycopeptides by using HCD- and CID-MS/MS and MS3. Scientific reports
6 (2016).</cite> [<a href="https://doi.org/10.1038/srep25102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4853738/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27139140/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20reports&amp;title=pGlyco:%20a%20pipeline%20for%20the%20identification%20of%20intact%20N-glycopeptides%20by%20using%20HCD-%20and%20CID-MS/MS%20and%20MS3&amp;author=W-F%20Zeng&amp;volume=6&amp;publication_year=2016&amp;pmid=27139140&amp;doi=10.1038/srep25102&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R87">
<span class="label">87.</span><cite>Bern M, Kil YJ &amp; Becker C. in Current Protocols in Bioinformatics (John Wiley &amp; Sons, Inc., 2002).</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Current%20Protocols%20in%20Bioinformatics&amp;author=M%20Bern&amp;author=YJ%20Kil&amp;author=C%20Becker&amp;publication_year=2002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R88">
<span class="label">88.</span><cite>Aldredge D, An HJ, Tang N, Waddell K. &amp; Lebrilla CB
Annotation of a Serum N-Glycan Library for Rapid Identification of Structures. Journal of Proteome Research
11, 1958–1968, doi: 10.1021/pr2011439 (2012).
</cite> [<a href="https://doi.org/10.1021/pr2011439" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3292799/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22320385/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Proteome%20Research&amp;title=Annotation%20of%20a%20Serum%20N-Glycan%20Library%20for%20Rapid%20Identification%20of%20Structures&amp;author=D%20Aldredge&amp;author=HJ%20An&amp;author=N%20Tang&amp;author=K%20Waddell&amp;author=CB%20Lebrilla&amp;volume=11&amp;publication_year=2012&amp;pages=1958-1968&amp;pmid=22320385&amp;doi=10.1021/pr2011439&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R89">
<span class="label">89.</span><cite>Song T, Aldredge D. &amp; Lebrilla CB
A method for in-depth structural annotation of human serum glycans that yields biological variations. Analytical chemistry
87, 7754–7762 (2015).
</cite> [<a href="https://doi.org/10.1021/acs.analchem.5b01340" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5444872/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26086522/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20chemistry&amp;title=A%20method%20for%20in-depth%20structural%20annotation%20of%20human%20serum%20glycans%20that%20yields%20biological%20variations&amp;author=T%20Song&amp;author=D%20Aldredge&amp;author=CB%20Lebrilla&amp;volume=87&amp;publication_year=2015&amp;pages=7754-7762&amp;pmid=26086522&amp;doi=10.1021/acs.analchem.5b01340&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R90">
<span class="label">90.</span><cite>Hülsmeier AJ, Paesold-Burda P. &amp; Hennet T. N-glycosylation site occupancy in serum glycoproteins using multiple reaction monitoring liquid chromatography-mass spectrometry. Molecular &amp; Cellular Proteomics
6, 2132–2138 (2007).
</cite> [<a href="https://doi.org/10.1074/mcp.M700361-MCP200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17823199/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20&amp;%20Cellular%20Proteomics&amp;title=N-glycosylation%20site%20occupancy%20in%20serum%20glycoproteins%20using%20multiple%20reaction%20monitoring%20liquid%20chromatography-mass%20spectrometry&amp;author=AJ%20H%C3%BClsmeier&amp;author=P%20Paesold-Burda&amp;author=T%20Hennet&amp;volume=6&amp;publication_year=2007&amp;pages=2132-2138&amp;pmid=17823199&amp;doi=10.1074/mcp.M700361-MCP200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R91">
<span class="label">91.</span><cite>Zhang F, Zhang Z. &amp; Linhardt RJ in Handbook of Glycomics (eds Cummings Richard D. &amp; Michael Pierce J) 59–80 (Academic Press, 2010).</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Handbook%20of%20Glycomics&amp;author=F%20Zhang&amp;author=Z%20Zhang&amp;author=RJ%20Linhardt&amp;author=Richard%20D.%20Cummings&amp;author=J%20Michael%20Pierce&amp;publication_year=2010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R92">
<span class="label">92.</span><cite>Li Q, Xie Y, Xu G. &amp; Lebrilla CB
Identification of potential sialic acid binding proteins on cell membrane by proximity chemical labeling. Chemical Science, doi: 10.1039/C9SC01360A (2019).</cite> [<a href="https://doi.org/10.1039/C9SC01360A" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6585875/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31360427/" class="usa-link">PubMed</a>]</li>
<li id="R93">
<span class="label">93.</span><cite>Xu G. et al. 
Unveiling the metabolic fate of monosaccharides in cell membranes with glycomic 1477 and glycoproteomic analyses. Chemical Science
10, 6992–7002, doi: 10.1039/C9SC01653H (2019).
</cite> [<a href="https://doi.org/10.1039/C9SC01653H" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6676465/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31588266/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chemical%20Science&amp;title=Unveiling%20the%20metabolic%20fate%20of%20monosaccharides%20in%20cell%20membranes%20with%20glycomic%201477%20and%20glycoproteomic%20analyses&amp;author=G%20Xu&amp;volume=10&amp;publication_year=2019&amp;pages=6992-7002&amp;pmid=31588266&amp;doi=10.1039/C9SC01653H&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<span>Supplementary Data 1</span><p>Supplementary Data I_N-glycan_lib_template.csv contains examples of N-glycans with their masses, compositions, and types.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11790333/bin/NIHMS2039916-supplement-Supplementary_Data_1.csv" data-ga-action="click_feat_suppl" class="usa-link">NIHMS2039916-supplement-Supplementary_Data_1.csv</a><sup> (1.5KB, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<span>Supplementary Data III</span><p>Supplementary Data III_GSL_glycan_finder.xlsx matches the closest glycan composition to the glycosphingolipid with the known precursor <em>m/z</em>, precursor charge state, and the ceramide <em>m/z</em>.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11790333/bin/NIHMS2039916-supplement-Supplementary_Data_III.xlsx" data-ga-action="click_feat_suppl" class="usa-link">NIHMS2039916-supplement-Supplementary_Data_III.xlsx</a><sup> (29.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<span>Supplementary Data V</span><p>Supplementary Data V_GSL_lib_generator.xlsx contains a Pivot table for generating a database of glycosphingolipids.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11790333/bin/NIHMS2039916-supplement-Supplementary_Data_V.xlsx" data-ga-action="click_feat_suppl" class="usa-link">NIHMS2039916-supplement-Supplementary_Data_V.xlsx</a><sup> (2.6MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p">
<span>Supplementary Data IV</span><p>Supplementary Data IV_Saccharide_mz.csv contains some common saccharide and lipid fragments.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11790333/bin/NIHMS2039916-supplement-Supplementary_Data_IV.csv" data-ga-action="click_feat_suppl" class="usa-link">NIHMS2039916-supplement-Supplementary_Data_IV.csv</a><sup> (863B, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p">
<span>Supplementary Data II</span><p>Supplementary Data II_Ceramide_mz.csv contains possible ceramide <em>m/z</em> values.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11790333/bin/NIHMS2039916-supplement-Supplementary_Data_II.csv" data-ga-action="click_feat_suppl" class="usa-link">NIHMS2039916-supplement-Supplementary_Data_II.csv</a><sup> (3.8KB, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="caption p"><span>Supplementary Figures</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11790333/bin/NIHMS2039916-supplement-Supplementary_Figures.docx" data-ga-action="click_feat_suppl" class="usa-link">NIHMS2039916-supplement-Supplementary_Figures.docx</a><sup> (518.2KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Data are all available online. Data from ref. <a href="#R23" class="usa-link" aria-describedby="R23">23</a> Park, D. <em>et al</em>. <em>Glycobiology</em>
<strong>27</strong>, 847–860 (2017), DOI:<a href="https://doi.org/10.1093/glycob/cwx041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1093/glycob/cwx041</a> (used for <a href="#F4" class="usa-link">Fig. 4a</a>) are available at Glycobiology online. Data from ref. <a href="#R24" class="usa-link" aria-describedby="R24">24</a> Park, D. <em>et al</em>. <em>Chemical science</em>
<strong>9</strong>, 6271–6285 (2018), DOI: <a href="https://doi.org/10.1039/c8sc01875h" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1039/c8sc01875h</a> (used for <a href="#F4" class="usa-link">Fig. 4b</a> and <a href="#F9" class="usa-link">9</a>) are available at Chemical Science online. Data from ref. <a href="#R26" class="usa-link" aria-describedby="R26">26</a> Wong, M. <em>et al</em>. <em>Scientific reports</em>
<strong>8</strong>, 10993 (2018), DOI: <a href="https://doi.org/10.1038/s41598-018-29324-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-018-29324-7</a> (used for <a href="#F4" class="usa-link">Fig. 4c</a>) are available at Scientific Reports online. Data from ref. <a href="#R25" class="usa-link" aria-describedby="R25">25</a> Li, Q. <em>et al</em>. <em>Chemical science</em>
<strong>10</strong>, 6199–6209 (2019), DOI:<a href="https://doi.org/10.1039/c9sc01360a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1039/c9sc01360a</a> (used for <a href="#F6" class="usa-link">Fig. 6</a>) are available at Chemical Science online. Data from ref. <a href="#R33" class="usa-link" aria-describedby="R33">33</a> Park, D. <em>et al</em>. <em>Molecular &amp; cellular proteomics</em> : MCP <strong>14</strong>, 2910–2921 (2015), DOI:<a href="https://doi.org/10.1074/mcp.M115.053983" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1074/mcp.M115.053983</a> (used for <a href="#F8" class="usa-link">Fig. 8</a>) are available at MCP online.</p></section></section></section></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41596-020-0350-4"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/nihms-2039916.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/11790333/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/11790333/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11790333%2F%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11790333/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC11790333/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC11790333/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/32681150/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC11790333/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/32681150/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC11790333/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/11790333/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="7W23CgIZEbftlWH0xlTdWu2n3AEL3dkaCgPJjIhaGg8eK7NyHxDOCrTfJFzzxmuO">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-B7-tgZZA.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-BLbQGH8u.js"></script>
    
    

    </body>
</html>
