
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/base_style-CPPDhfna.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-BepKMsUl.js"></script>

  <link rel="stylesheet" href="/static/assets/article_style-DUfI1jZp.css" />
<link rel="stylesheet" href="/static/assets/style-BOrsu84o.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-D3wJHomh.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Plasmacytoid dendritic cell sensing of hepatitis E virus is shaped by both viral and host factors - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="18DE25A4930B28130025A400143A7F91.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="lifesciall">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11966012/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Life Science Alliance">
<meta name="citation_title" content="Plasmacytoid dendritic cell sensing of hepatitis E virus is shaped by both viral and host factors">
<meta name="citation_author" content="Garima Joshi">
<meta name="citation_author_institution" content="CIRI, INSERM, , U1111, Université Claude Bernard Lyon 1, , CNRS, UMR5308, École Normale Supérieure de Lyon, , University Lyon, Lyon, France">
<meta name="citation_author" content="Elodie Décembre">
<meta name="citation_author_institution" content="CIRI, INSERM, , U1111, Université Claude Bernard Lyon 1, , CNRS, UMR5308, École Normale Supérieure de Lyon, , University Lyon, Lyon, France">
<meta name="citation_author" content="Jacques Brocard">
<meta name="citation_author_institution" content="Université Claude Bernard Lyon 1, , CNRS UAR3444, INSERM, US8, ENS de Lyon, SFR Biosciences, Lyon, France">
<meta name="citation_author" content="Claire Montpellier">
<meta name="citation_author_institution" content="University Lille, CNRS, INSERM, , CHU Lille, Institut Pasteur de Lille, , U1019-UMR 9017-CIIL-Center for Infection and Immunity of Lille, Lille, France">
<meta name="citation_author" content="Martin Ferrié">
<meta name="citation_author_institution" content="University Lille, CNRS, INSERM, , CHU Lille, Institut Pasteur de Lille, , U1019-UMR 9017-CIIL-Center for Infection and Immunity of Lille, Lille, France">
<meta name="citation_author" content="Omran Allatif">
<meta name="citation_author_institution" content="CIRI, INSERM, , U1111, Université Claude Bernard Lyon 1, , CNRS, UMR5308, École Normale Supérieure de Lyon, , University Lyon, Lyon, France">
<meta name="citation_author" content="Ann-Kathrin Mehnert">
<meta name="citation_author_institution" content="Department of Infectious Diseases, Virology, Heidelberg University, Medical Faculty Heidelberg, Heidelberg, Germany and German Centre for Infection Research (DZIF), Partner Site Heidelberg, Heidelberg, Germany">
<meta name="citation_author" content="Johann Pons">
<meta name="citation_author_institution" content="Sup’biotech, École Des Ingénieurs En Biotechnologies, Villejuif, Paris">
<meta name="citation_author" content="Delphine Galiana">
<meta name="citation_author_institution" content="CIRI, INSERM, , U1111, Université Claude Bernard Lyon 1, , CNRS, UMR5308, École Normale Supérieure de Lyon, , University Lyon, Lyon, France">
<meta name="citation_author" content="Viet Loan Dao Thi">
<meta name="citation_author_institution" content="Department of Infectious Diseases, Virology, Heidelberg University, Medical Faculty Heidelberg, Heidelberg, Germany and German Centre for Infection Research (DZIF), Partner Site Heidelberg, Heidelberg, Germany">
<meta name="citation_author" content="Nolwenn Jouvenet">
<meta name="citation_author_institution" content="Institut Pasteur, Université de Paris, CNRS UMR 3569, Virus sensing and signaling Unit, Paris, France">
<meta name="citation_author" content="Laurence Cocquerel">
<meta name="citation_author_institution" content="University Lille, CNRS, INSERM, , CHU Lille, Institut Pasteur de Lille, , U1019-UMR 9017-CIIL-Center for Infection and Immunity of Lille, Lille, France">
<meta name="citation_author" content="Marlène Dreux">
<meta name="citation_author_institution" content="CIRI, INSERM, , U1111, Université Claude Bernard Lyon 1, , CNRS, UMR5308, École Normale Supérieure de Lyon, , University Lyon, Lyon, France">
<meta name="citation_publication_date" content="2025 Apr 2">
<meta name="citation_volume" content="8">
<meta name="citation_issue" content="6">
<meta name="citation_firstpage" content="e202503256">
<meta name="citation_doi" content="10.26508/lsa.202503256">
<meta name="citation_pmid" content="40175091">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11966012/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11966012/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11966012/pdf/LSA-2025-03256.pdf">
<meta name="description" content="Close contacts between HEV-infected cells and plasmacytoid dendritic cells trigger interferon production in a TLR7- and ORF2-dependent manner, leading to the inhibition of viral replication.">
<meta name="og:title" content="Plasmacytoid dendritic cell sensing of hepatitis E virus is shaped by both viral and host factors">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Close contacts between HEV-infected cells and plasmacytoid dendritic cells trigger interferon production in a TLR7- and ORF2-dependent manner, leading to the inhibition of viral replication.">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11966012/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://pmc.ncbi.nlm.nih.gov/search/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="11966012">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.26508/lsa.202503256"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/LSA-2025-03256.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11966012%2F%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/11966012/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/11966012/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11966012/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-lifesciall.png" alt="Life Science Alliance logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Life Science Alliance" title="Link to Life Science Alliance" shape="default" href="http://www.life-science-alliance.org/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Life Sci Alliance</button></div>. 2025 Apr 2;8(6):e202503256. doi: <a href="https://doi.org/10.26508/lsa.202503256" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.26508/lsa.202503256</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://pmc.ncbi.nlm.nih.gov/search/?term=%22Life%20Sci%20Alliance%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Life%20Sci%20Alliance%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Life%20Sci%20Alliance%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Life%20Sci%20Alliance%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Plasmacytoid dendritic cell sensing of hepatitis E virus is shaped by both viral and host factors</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Joshi%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Garima Joshi</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Garima Joshi</span></h3>
<div class="p">
<sup>1</sup>CIRI, INSERM, , U1111, Université Claude Bernard Lyon 1, , CNRS, UMR5308, École Normale Supérieure de Lyon, , University Lyon, Lyon, France</div>
<div>Conceptualization, Data curation, Formal analysis, Funding acquisition, Investigation, Methodology, Writing—original draft, Writing—review and editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Joshi%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Garima Joshi</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22D%C3%A9cembre%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Elodie Décembre</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Elodie Décembre</span></h3>
<div class="p">
<sup>1</sup>CIRI, INSERM, , U1111, Université Claude Bernard Lyon 1, , CNRS, UMR5308, École Normale Supérieure de Lyon, , University Lyon, Lyon, France</div>
<div>Investigation, Methodology</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22D%C3%A9cembre%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Elodie Décembre</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brocard%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Jacques Brocard</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Jacques Brocard</span></h3>
<div class="p">
<sup>2</sup>Université Claude Bernard Lyon 1, , CNRS UAR3444, INSERM, US8, ENS de Lyon, SFR Biosciences, Lyon, France</div>
<div>Formal analysis, Validation, Methodology</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brocard%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jacques Brocard</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Montpellier%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Claire Montpellier</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Claire Montpellier</span></h3>
<div class="p">
<sup>3</sup>University Lille, CNRS, INSERM, , CHU Lille, Institut Pasteur de Lille, , U1019-UMR 9017-CIIL-Center for Infection and Immunity of Lille, Lille, France</div>
<div>Resources, Investigation, Methodology</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Montpellier%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Claire Montpellier</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ferri%C3%A9%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Martin Ferrié</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Martin Ferrié</span></h3>
<div class="p">
<sup>3</sup>University Lille, CNRS, INSERM, , CHU Lille, Institut Pasteur de Lille, , U1019-UMR 9017-CIIL-Center for Infection and Immunity of Lille, Lille, France</div>
<div>Resources, Investigation, Methodology</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ferri%C3%A9%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Martin Ferrié</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Allatif%20O%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Omran Allatif</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Omran Allatif</span></h3>
<div class="p">
<sup>1</sup>CIRI, INSERM, , U1111, Université Claude Bernard Lyon 1, , CNRS, UMR5308, École Normale Supérieure de Lyon, , University Lyon, Lyon, France</div>
<div>Formal analysis, Validation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Allatif%20O%22%5BAuthor%5D" class="usa-link"><span class="name western">Omran Allatif</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mehnert%20AK%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Ann-Kathrin Mehnert</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Ann-Kathrin Mehnert</span></h3>
<div class="p">
<sup>5</sup>Department of Infectious Diseases, Virology, Heidelberg University, Medical Faculty Heidelberg, Heidelberg, Germany and German Centre for Infection Research (DZIF), Partner Site Heidelberg, Heidelberg, Germany</div>
<div>Investigation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mehnert%20AK%22%5BAuthor%5D" class="usa-link"><span class="name western">Ann-Kathrin Mehnert</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pons%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Johann Pons</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Johann Pons</span></h3>
<div class="p">
<sup>6</sup>Sup’biotech, École Des Ingénieurs En Biotechnologies, Villejuif, Paris</div>
<div>Investigation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pons%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Johann Pons</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Galiana%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Delphine Galiana</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Delphine Galiana</span></h3>
<div class="p">
<sup>1</sup>CIRI, INSERM, , U1111, Université Claude Bernard Lyon 1, , CNRS, UMR5308, École Normale Supérieure de Lyon, , University Lyon, Lyon, France</div>
<div>Formal analysis, Supervision, Investigation, Writing—review and editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Galiana%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Delphine Galiana</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dao%20Thi%20VL%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Viet Loan Dao Thi</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Viet Loan Dao Thi</span></h3>
<div class="p">
<sup>5</sup>Department of Infectious Diseases, Virology, Heidelberg University, Medical Faculty Heidelberg, Heidelberg, Germany and German Centre for Infection Research (DZIF), Partner Site Heidelberg, Heidelberg, Germany</div>
<div>Resources, Supervision, Methodology, Writing—review and editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dao%20Thi%20VL%22%5BAuthor%5D" class="usa-link"><span class="name western">Viet Loan Dao Thi</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jouvenet%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Nolwenn Jouvenet</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Nolwenn Jouvenet</span></h3>
<div class="p">
<sup>4</sup>Institut Pasteur, Université de Paris, CNRS UMR 3569, Virus sensing and signaling Unit, Paris, France</div>
<div>Formal analysis, Supervision, Validation, Writing—review and editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jouvenet%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nolwenn Jouvenet</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cocquerel%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Laurence Cocquerel</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Laurence Cocquerel</span></h3>
<div class="p">
<sup>3</sup>University Lille, CNRS, INSERM, , CHU Lille, Institut Pasteur de Lille, , U1019-UMR 9017-CIIL-Center for Infection and Immunity of Lille, Lille, France</div>
<div>Conceptualization, Resources, Formal analysis, Supervision, Validation, Methodology, Writing—review and editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cocquerel%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Laurence Cocquerel</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dreux%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Marlène Dreux</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Marlène Dreux</span></h3>
<div class="p">
<sup>1</sup>CIRI, INSERM, , U1111, Université Claude Bernard Lyon 1, , CNRS, UMR5308, École Normale Supérieure de Lyon, , University Lyon, Lyon, France</div>
<div>Conceptualization, Supervision, Funding acquisition, Writing—original draft, Writing—review and editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dreux%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marlène Dreux</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>1</sup>CIRI, INSERM, , U1111, Université Claude Bernard Lyon 1, , CNRS, UMR5308, École Normale Supérieure de Lyon, , University Lyon, Lyon, France</div>
<div id="aff2">
<sup>2</sup>Université Claude Bernard Lyon 1, , CNRS UAR3444, INSERM, US8, ENS de Lyon, SFR Biosciences, Lyon, France</div>
<div id="aff3">
<sup>3</sup>University Lille, CNRS, INSERM, , CHU Lille, Institut Pasteur de Lille, , U1019-UMR 9017-CIIL-Center for Infection and Immunity of Lille, Lille, France</div>
<div id="aff4">
<sup>4</sup>Institut Pasteur, Université de Paris, CNRS UMR 3569, Virus sensing and signaling Unit, Paris, France</div>
<div id="aff5">
<sup>5</sup>Department of Infectious Diseases, Virology, Heidelberg University, Medical Faculty Heidelberg, Heidelberg, Germany and German Centre for Infection Research (DZIF), Partner Site Heidelberg, Heidelberg, Germany</div>
<div id="aff6">
<sup>6</sup>Sup’biotech, École Des Ingénieurs En Biotechnologies, Villejuif, Paris</div>
<div class="author-notes p">
<div class="fn" id="cor1">
<sup>✉</sup><p class="display-inline">Correspondence: <span>garima.joshi@ens-lyon.fr</span></p>
</div>
<div class="fn" id="cor2">
<sup>✉</sup><p class="display-inline">Email: <span>marlene.dreux@ens-lyon.fr</span></p>
</div>
<div class="fn" id="fn1"><p>Marlène Dreux died on March 18, 2025</p></div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Garima Joshi</span></strong>: <span class="role">Conceptualization, Data curation, Formal analysis, Funding acquisition, Investigation, Methodology, Writing—original draft, Writing—review and editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Elodie Décembre</span></strong>: <span class="role">Investigation, Methodology</span>
</div>
<div>
<strong class="contrib"><span class="name western">Jacques Brocard</span></strong>: <span class="role">Formal analysis, Validation, Methodology</span>
</div>
<div>
<strong class="contrib"><span class="name western">Claire Montpellier</span></strong>: <span class="role">Resources, Investigation, Methodology</span>
</div>
<div>
<strong class="contrib"><span class="name western">Martin Ferrié</span></strong>: <span class="role">Resources, Investigation, Methodology</span>
</div>
<div>
<strong class="contrib"><span class="name western">Omran Allatif</span></strong>: <span class="role">Formal analysis, Validation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Ann-Kathrin Mehnert</span></strong>: <span class="role">Investigation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Johann Pons</span></strong>: <span class="role">Investigation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Delphine Galiana</span></strong>: <span class="role">Formal analysis, Supervision, Investigation, Writing—review and editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Viet Loan Dao Thi</span></strong>: <span class="role">Resources, Supervision, Methodology, Writing—review and editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Nolwenn Jouvenet</span></strong>: <span class="role">Formal analysis, Supervision, Validation, Writing—review and editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Laurence Cocquerel</span></strong>: <span class="role">Conceptualization, Resources, Formal analysis, Supervision, Validation, Methodology, Writing—review and editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Marlène Dreux</span></strong>: <span class="role">Conceptualization, Supervision, Funding acquisition, Writing—original draft, Writing—review and editing</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 6; Revised 2025 Mar 14; Accepted 2025 Mar 14; Collection date 2025 Jun.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 Joshi et al.</div>
<p>This article is available under a Creative Commons License (Attribution 4.0 International, as described at <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://creativecommons.org/licenses/by/4.0/</a>).</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC11966012  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40175091/" class="usa-link">40175091</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><hr class="headless">
<p>Close contacts between HEV-infected cells and plasmacytoid dendritic cells trigger interferon production in a TLR7- and ORF2-dependent manner, leading to the inhibition of viral replication.</p></section><section class="abstract" id="abstract2"><h2>Abstract</h2>
<p>Type I and III interferons critically protect the host against viral infection. Previous studies showed that IFN responses are suppressed in cells infected by hepatitis E virus (HEV). Here, we studied the anti-HEV function of IFN secreted by plasmacytoid dendritic cells (pDCs), specialized producers of IFNs. We showed that pDCs co-cultured with HEV-replicating cells secreted IFN in a cell contact–dependent manner. This pDC response required the endosomal nucleic acid sensor TLR7 and adhesion molecules. IFNs secreted by pDCs reduced viral spread. Intriguingly, ORF2, the capsid protein of HEV, can be produced in various forms by the infected cells, and we wanted to study their role in anti-HEV immune response. During infection, a fraction of ORF2 localizes into the nucleus, and glycosylated forms of ORF2 are massively secreted by infected cells. We showed that glycosylated ORF2 potentiates the recognition of infected cells by pDCs, by regulating cell contacts. On the other hand, nuclear ORF2 triggers immune response by IRF3 activation. Together, our results suggest that pDCs may be essential to control HEV replication.</p></section><section id="s1"><h2 class="pmc_sec_title">Introduction</h2>
<p>Hepatitis E virus (HEV) is the most common cause of acute viral hepatitis worldwide. It has been estimated that this virus infects ∼100 million people every year and is responsible for 14 million symptomatic cases and 300,000 deaths, mainly in regions of the world with poor sanitary conditions (<a href="#bib38" class="usa-link" aria-describedby="bib38">Li et al, 2020</a>). HEV infection is often asymptomatic and resolves on its own in case of healthy subjects. However, severe cases have primarily been reported in pregnant women, whereas chronic infections are more common in immunocompromised patients (<a href="#bib37" class="usa-link" aria-describedby="bib37">Lhomme et al, 2020</a>). This makes host immunity a crucial factor in influencing the outcome of the disease. In addition, HEV infection is associated with a broad range of extrahepatic manifestations, including renal and neurological disorders (<a href="#bib65" class="usa-link" aria-describedby="bib65">Songtanin et al, 2023</a>). The <em>Paslahepevirus balayani</em> species of the Paslahepevirus genus contains five HEV genotypes (gt) that are pathogenic in humans. HEV gt1 and gt2 are primarily transmitted through contaminated water, exclusively infect humans, and are responsible for waterborne hepatitis outbreaks in developing countries. In contrast, industrialized countries usually fall victim to HEV gt3 and gt4, which have a zoonotic origin (<a href="#bib18" class="usa-link" aria-describedby="bib18">Doceul et al, 2016</a>). Chronic human infection with camel-associated HEV gt7 has also been reported (<a href="#bib34" class="usa-link" aria-describedby="bib34">Lee et al, 2016</a>). Alarmingly, a rat HEV from the Rocahepevirus species was recently reported to be also transmitted to humans (<a href="#bib66" class="usa-link" aria-describedby="bib66">Sridhar et al, 2018</a>; <a href="#bib1" class="usa-link" aria-describedby="bib1">Andonov et al, 2019</a>).</p>
<p>The most common genotype causing chronic HEV infection in the developed world is gt3 (<a href="#bib41" class="usa-link" aria-describedby="bib41">Ma et al, 2022b</a>). HEV infection has been recognized as a burgeoning issue in industrialized countries because of its chronicity in immunocompromised gt3-infected patients, the transmission of HEV through blood transfusion, a growing number of diagnosed HEV cases, and complications in patients with preexisting liver disease (<a href="#bib58" class="usa-link" aria-describedby="bib58">Sayed et al, 2017</a>). Importantly, HEV was recently ranked sixth among the top 10 zoonotic viruses presenting the greatest risk of transmission to humans (<a href="#bib27" class="usa-link" aria-describedby="bib27">Grange et al, 2021</a>).</p>
<p>HEV has a single-stranded, positive-sense RNA genome that contains three ORFs: ORF1, ORF2, and ORF3 (<a href="#bib69" class="usa-link" aria-describedby="bib69">Tam et al, 1991</a>). ORF1 encodes the nonstructural ORF1 polyprotein that displays domains essential for viral replication including the RNA-dependent RNA polymerase (RdRp) (<a href="#bib32" class="usa-link" aria-describedby="bib32">Koonin et al, 1992</a>). The RdRp produces a negative-sense RNA replicative intermediate, which serves as a template for the subgenomic RNA. The subgenomic RNA encodes both ORF2 and ORF3 proteins (<a href="#bib26" class="usa-link" aria-describedby="bib26">Graff et al, 2006</a>). ORF2 protein is the viral capsid protein, and ORF3 is a small multifunctional phosphoprotein, involved in particle egress (<a href="#bib50" class="usa-link" aria-describedby="bib50">Nimgaonkar et al, 2018</a>).</p>
<p>ORF2, which is composed of 660 amino acids, is produced in three forms: infectious ORF2 (ORF2i), glycosylated ORF2 (ORF2g), and cleaved ORF2 (ORF2c) (<a href="#bib46" class="usa-link" aria-describedby="bib46">Montpellier et al, 2018</a>). The precise sequences of ORF2i, ORF2g, and ORF2c proteins have been proposed by two groups (<a href="#bib46" class="usa-link" aria-describedby="bib46">Montpellier et al, 2018</a>; <a href="#bib79" class="usa-link" aria-describedby="bib79">Yin et al, 2018</a>; <a href="#bib2" class="usa-link" aria-describedby="bib2">Ankavay et al, 2019</a>; <a href="#bib29" class="usa-link" aria-describedby="bib29">Hervouet et al, 2022</a>). The ORF2i protein is not glycosylated and forms the structural component of infectious particles. In contrast, ORF2g and ORF2c proteins (herein referred to as ORF2g/c) are highly glycosylated, secreted in large amounts in the culture supernatant (i.e., about 1,000x more than ORF2i), and are the most abundant ORF2 forms detected in patient sera, where they are likely targeted by patient antibodies (<a href="#bib46" class="usa-link" aria-describedby="bib46">Montpellier et al, 2018</a>; <a href="#bib2" class="usa-link" aria-describedby="bib2">Ankavay et al, 2019</a>). Thus, ORF2g/c forms may act as a humoral decoy that inhibits antibody-mediated neutralization because of their antigenic overlap with HEV virions (<a href="#bib79" class="usa-link" aria-describedby="bib79">Yin et al, 2018</a>). Whether ORF2g/c proteins play a specific role in the HEV life cycle, and how ORF2 forms regulate the innate immune responses need to be elucidated. ORF2 forms are produced from different pathways, including a major one in which the ORF2 proteins are directed to the secretion pathway, where they undergo maturation and glycosylation, and are subsequently released in significant quantities. A fraction of cytosolic ORF2i proteins are delivered to the virion assembly sites (<a href="#bib7" class="usa-link" aria-describedby="bib7">Bentaleb et al, 2022</a>; <a href="#bib29" class="usa-link" aria-describedby="bib29">Hervouet et al, 2022</a>), whereas another fraction of ORF2i proteins translocate into the nucleus of infected cells, presumably to regulate host immune responses (<a href="#bib36" class="usa-link" aria-describedby="bib36">Lenggenhager et al, 2017</a>; <a href="#bib2" class="usa-link" aria-describedby="bib2">Ankavay et al, 2019</a>; <a href="#bib29" class="usa-link" aria-describedby="bib29">Hervouet et al, 2022</a>).</p>
<p>Viral genomes can be recognized by cytosolic sensors such as RIG-I, MDA5, and LGP2 (<a href="#bib55" class="usa-link" aria-describedby="bib55">Rehwinkel &amp; Gack, 2020</a>), whose activation leads to the induction of type I and III IFN (IFN-I/III) pathways. HEV has developed many mechanisms to inhibit the IFN response via its three ORFs, described as follows. The HEV gt1 ORF1 protein blocks RIG-I and Tank-binding kinase 1 ubiquitination in hepatoma cells, thereby suppressing the pathway of IFN induction (<a href="#bib49" class="usa-link" aria-describedby="bib49">Nan et al, 2014</a>). The amino-terminal region of HEV gt3 ORF1, harboring a putative methyltransferase (Met) and a papain-like cysteine protease functional domain, inhibits IFN-stimulated response element promoter activation by inhibiting STAT1 nuclear translocation and phosphorylation (<a href="#bib5" class="usa-link" aria-describedby="bib5">Bagdassarian et al, 2018</a>). Moreover, HEV gt1 and gt3 ORF2 proteins antagonize IFN induction in HEV-replicating hepatocytes by inhibiting phosphorylation of the transcriptional regulator IRF3 (<a href="#bib39" class="usa-link" aria-describedby="bib39">Lin et al, 2019</a>). The ORF3 protein of the HEV gt3 can also suppress IFN response by blocking STAT1 phosphorylation (<a href="#bib19" class="usa-link" aria-describedby="bib19">Dong et al, 2012</a>).</p>
<p>In human hepatoma cells and primary hepatocytes, HEV infection induces only IFN-III production (<a href="#bib76" class="usa-link" aria-describedby="bib76">Wu et al, 2018</a>; <a href="#bib79" class="usa-link" aria-describedby="bib79">Yin et al, 2018</a>), which is comparatively less potent at lower doses and at early time points than IFN-I (<a href="#bib33" class="usa-link" aria-describedby="bib33">Lazear et al, 2019</a>). However, the elevated expression of IFN-stimulated genes (ISGs), which are effectors of IFN-I/III, was detected in the whole blood of HEV-infected patients (<a href="#bib80" class="usa-link" aria-describedby="bib80">Yu et al, 2010</a>; <a href="#bib45" class="usa-link" aria-describedby="bib45">Moal et al, 2013</a>; <a href="#bib58" class="usa-link" aria-describedby="bib58">Sayed et al, 2017</a>), as well as in experimentally infected mice engrafted with human hepatocytes and chimpanzee (<a href="#bib80" class="usa-link" aria-describedby="bib80">Yu et al, 2010</a>; <a href="#bib58" class="usa-link" aria-describedby="bib58">Sayed et al, 2017</a>). It must also be noted that the infectious HEV, with a complete replication cycle, has been shown to be more sensitive to IFN-α treatment than the subgenomic replicon (<a href="#bib83" class="usa-link" aria-describedby="bib83">Zhou et al, 2016</a>). Therefore, the host immune system can presumably mount an immune response to fight off HEV despite the attenuation of immune signaling within the host cells. Plasmacytoid dendritic cells (pDCs), which are key producers of IFN, could be instrumental in effectively counteracting the evasion strategies employed by HEV within the liver microenvironment. As pDCs are resistant to virtually all viruses (<a href="#bib64" class="usa-link" aria-describedby="bib64">Silvin et al, 2017</a>), these do not express viral proteins that may block IFN induction. PDCs are an immune cell type known to produce up to 1,000-fold more IFNs than any other cell type (<a href="#bib56" class="usa-link" aria-describedby="bib56">Reizis, 2019</a>). They are thus pivotal for host control of viral infections (<a href="#bib73" class="usa-link" aria-describedby="bib73">Webster et al, 2016</a>; <a href="#bib56" class="usa-link" aria-describedby="bib56">Reizis, 2019</a>; <a href="#bib81" class="usa-link" aria-describedby="bib81">Yun et al, 2021</a>; <a href="#bib71" class="usa-link" aria-describedby="bib71">Venet et al, 2023</a>). They also recruit NK cells at the site of viral replication, favor virus-specific T-cell responses (<a href="#bib68" class="usa-link" aria-describedby="bib68">Swiecki et al, 2010</a>; <a href="#bib14" class="usa-link" aria-describedby="bib14">Cervantes-Barragan et al, 2012</a>; <a href="#bib73" class="usa-link" aria-describedby="bib73">Webster et al, 2016</a>, <a href="#bib74" class="usa-link" aria-describedby="bib74">2018</a>; <a href="#bib56" class="usa-link" aria-describedby="bib56">Reizis, 2019</a>), and secrete a large panel of proinflammatory cytokines, for example, TNF-α and IL-6 (<a href="#bib56" class="usa-link" aria-describedby="bib56">Reizis, 2019</a>). PDC stimulation is mediated by the recognition of viral nucleic acid by TLR7 and TLR9, which localize in the endolysosomal compartment (<a href="#bib56" class="usa-link" aria-describedby="bib56">Reizis, 2019</a>). In the case of RNA viruses, TLR7-ligand engagement results in the formation of a signal complex comprising IRAK1 (IL-1 receptor–associated kinase 1), IRAK4, and TRAF6 (TNF receptor–associated factor 6), in a MyD88-dependent (myeloid differentiation primary-response gene 88) manner. This further triggers the translocation of IRF7 (IFN regulatory factor 7) into the nucleus, where it can induce the transcription of type I IFN genes. In addition, the NF-κB (nuclear factor-κB)– and MAPK (mitogen-activated protein kinase)-mediated signaling are also activated (<a href="#bib25" class="usa-link" aria-describedby="bib25">Gilliet et al, 2008</a>). It has been shown that liver pDCs retain the ability to produce copious amounts of IFN-α (<a href="#bib20" class="usa-link" aria-describedby="bib20">Doyle et al, 2019</a>). Whether liver-resident pDCs contribute to the control of HEV replication is unknown. Here, using hepatocyte-derived cell lines selected for their immunocompetence, we investigated the ability of HEV-replicating cells to stimulate pDCs, and defined how ORF2 forms contribute to this process.</p></section><section id="s2"><h2 class="pmc_sec_title">Results</h2>
<section id="s3"><h3 class="pmc_sec_title">Immunocompetence of HEV cellular models</h3>
<p>The magnitude of the type I and III IFN (herein referred to as IFN-I/III) responses against viruses varies among cell types. We thus first assessed the immunological robustness of three cell lines, namely, Huh-7.5, HepG2/C3A, and PLC3 cells, which are permissive to HEV (<a href="#bib62" class="usa-link" aria-describedby="bib62">Shukla et al, 2011</a>; <a href="#bib59" class="usa-link" aria-describedby="bib59">Schemmerer et al, 2016</a>; <a href="#bib46" class="usa-link" aria-describedby="bib46">Montpellier et al, 2018</a>). Huh 7.5 cells are a subclone of the hepatoma Huh-7 cells that express an inactivate version of RIG-I and thus display an increased permissiveness to viral infection (<a href="#bib9" class="usa-link" aria-describedby="bib9">Blight et al, 2002</a>; <a href="#bib67" class="usa-link" aria-describedby="bib67">Sumpter et al, 2005</a>). HepG2/C3A cells resemble liver parenchymal cells and were selected from the primary hepatoblastoma–derived cell line HepG2 for cell contact inhibition, leading to a more hepatocyte-like phenotype compared with the parental line (<a href="#bib31" class="usa-link" aria-describedby="bib31">Knowles et al, 1980</a>). PLC3 cells are a subclone of the PLC/PRF/5 hepatoma cell line broadly used to study HEV (<a href="#bib46" class="usa-link" aria-describedby="bib46">Montpellier et al, 2018</a>). We measured mRNA levels of two IFN-stimulated genes (<em>MXA</em> and <em>ISG15</em>) and two cytokines (<em>TNF</em> and <em>IL-6</em>) in the three cell lines upon stimulation with polyI:C (<a href="#fig1" class="usa-link">Fig 1A–C</a>), which is a synthetic analog of dsRNA that activates TLR3, when added to the cell culture, and it activates the cytosolic sensors RIG-I and MDA5 when transfected into cells. <em>MXA</em> and <em>ISG15</em> are representatives of the IFN-I/III responses induced via IRF3-mediated signaling, whereas <em>TNF</em> and <em>IL-6</em> are induced by NF-κB signaling. All three cell types were deficient in TLR3 activity as none of them responded to treatment with polyI:C (<a href="#fig1" class="usa-link">Fig 1A–C</a>). The expression of <em>MXA</em>, <em>ISG15</em>, <em>TNF</em>, and <em>IL-6</em> remained unaffected upon polyI:C transfection in Huh-7.5 cells (<a href="#fig1" class="usa-link">Fig 1A</a>). In contrast, the expression of these four genes was up-regulated upon activation of RIG-I and MDA5 in both HepG2/C3A and PLC3 cells (<a href="#fig1" class="usa-link">Fig 1B and C</a>). Treatment with different IFN types and TNF-α has been shown to inhibit HEV replication to a certain extent (<a href="#bib70" class="usa-link" aria-describedby="bib70">Todt et al, 2016</a>; <a href="#bib72" class="usa-link" aria-describedby="bib72">Wang et al, 2016</a>; <a href="#bib83" class="usa-link" aria-describedby="bib83">Zhou et al, 2016</a>; <a href="#bib47" class="usa-link" aria-describedby="bib47">Murata et al, 2020</a>). Consequently, we determined whether the three cell lines responded to stimulation by three recombinant cytokines: IFN-β, IFN-λ3 (i.e., representatives of type I and III IFNs, respectively), or TNF-α. We found that IFN-β treatment induced the expression of <em>MXA</em> and <em>ISG15</em> in all three cell lines but barely induced the proinflammatory cytokines, <em>TNF</em> and <em>IL-6</em> (<a href="#fig1" class="usa-link">Fig 1A–C</a>). The three cell lines responded to the treatment with recombinant IFN-λ with an up-regulation of <em>MXA</em> and <em>ISG15</em> mRNA (<a href="#fig1" class="usa-link">Fig 1A–C</a>). However, only HepG2/C3A cells showed significant induction of <em>IL-6</em> upon IFN-λ treatment (<a href="#fig1" class="usa-link">Fig 1B</a>). Treatment with TNF-α mainly up-regulated the proinflammatory cytokine <em>TNF</em> in the three cell lines (<a href="#fig1" class="usa-link">Fig 1A–C</a>). Together, our results show that HepG2/C3A and PLC3 cells are sufficiently immunocompetent and were therefore selected for further investigation.</p>
<figure class="fig xbox font-sm" id="fig1"><h4 class="obj_head">Figure 1. Immunoresponsiveness of cell types selected as highly susceptible to HEV infection.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11966012_LSA-2025-03256_Fig1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b662/11966012/aefe1074661f/LSA-2025-03256_Fig1.jpg" loading="lazy" height="756" width="667" alt="Figure 1."></a></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A, B, C)</strong> Quantification of transcript expression levels of representatives of the IFN-I/III–related pathway, that is, <em>MXA</em> and <em>ISG15</em>, NF-KB–induced signaling, that is, <em>TNF</em> and <em>IL-6</em>, upon stimulation by agonists of TLR3, that is, addition of polyI:C; RIG-I/MDA-5 cytosolic sensors by transfection of polyI:C LMW and addition of recombinant IFN-β, IFN-λ3, and TNF at the indicated concentrations for 6-hour incubation, determined by RT–qPCR. <strong>(A, B, C)</strong> Analyses were performed in Huh-7.5 cells (A), HepG2/C3A cells (B), and PLC3 cells (C); bars represent fold expression; means ± SD; each dot represents one independent experiment, n = 4 for treatments including polyI:C addition/transfection and IFN-β treatment, n = 3 for treatment with IFN-λ and TNF; statistical analysis was performed using the paired pairwise Wilcoxon test; <em>P</em>-values: * ≤ 0.05, ** ≤ 0.005, and *** ≤ 0.0005.</p></figcaption></figure><p>Next, PLC3 and HepG2/C3A cells were electroporated with capped RNA of the HEV gt3 p6 strain (<a href="#bib63" class="usa-link" aria-describedby="bib63">Shukla et al, 2012</a>). RT–qPCR analyses showed that HepG2/C3A and PLC3 cells produced a significant amount of viral RNA (<a href="#figS1" class="usa-link">Fig S1A and B</a>) compared with mock-electroporated cells (control). Immunostaining of the ORF2 protein in PLC3 and HepG2/C3A cells at 6 days post-electroporation (d.p.e.) confirmed viral replication (<a href="#figS1" class="usa-link">Fig S1C</a>). In addition, we optimized a staining protocol to quantify ORF2-positive cells by flow cytometry. Flow cytometry analyses showed that about 50% of PLC3 cells were positive for ORF2 at 6 d.p.e. (<a href="#figS1" class="usa-link">Fig S1D</a>). Overall, these results indicated that HepG2/C3A and PLC3 cells are suitable models to perform immunological studies in the context of HEV replication.</p>
<figure class="fig xbox font-sm" id="figS1"><h4 class="obj_head">Figure S1. HEV-replicating cell-types and HEV mutants.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11966012_LSA-2025-03256_FigS1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b662/11966012/1b8a275384a6/LSA-2025-03256_FigS1.jpg" loading="lazy" height="1055" width="781" alt="Figure S1."></a></p>
<div class="p text-right font-secondary"><a href="figure/figS1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A, B)</strong> Quantification of the replication levels of HEV at 6 d post-electroporation with 10 μg RNA [HEV cells] or mock electroporation without RNA [cont cells] (as described in the Materials and Methods section) by RT–qPCR detection of HEV RNA (amplicon/primers designed in ORF2) in HepG2/C3A (A) and PLC3 cells (B), respectively; means ± SD; each dot represents one independent experiment; n = 4 for PLC3 cells and HepG2/C3A cells; statistical analysis was done using the Wilcoxon rank-sum test with continuity correction and <em>P</em>-value adjustment with the Bonferroni method; <em>P</em>-values: * ≤ 0.05, ** ≤ 0.005, and *** ≤ 0.0005. <strong>(C)</strong> HEV infection levels determined by immunodetection of ORF2 protein using 1E6 antibody are presented as representative images in HEV-producing PLC3 and HepG2/C3A cells versus control cells [cont] transfected with no RNA as a negative control, at 6 d.p.e. <strong>(D)</strong> Quantification by flow cytometry of the frequency of HEV ORF2-positive PLC3 cells. Statistical analysis was done using the Wilcoxon rank-sum test with continuity correction and <em>P</em>-value adjustment with the FDR method; <em>P</em>-values: * ≤ 0.05, ** ≤ 0.005, and *** ≤ 0.0005. <strong>(E)</strong> Illustrative table describing the differences between WT and mutants in terms of intracellular and extracellular viral products in electroporated cells.</p></figcaption></figure></section><section id="s4"><h3 class="pmc_sec_title">Activation of pDC upon contact with HEV-replicating cells leads to antiviral response</h3>
<p>Next, we looked at expression levels of three ISGs (<em>MXA</em>, <em>ISG15</em>, and <em>OAS2</em>) and three cytokines (<em>TNF</em>, <em>IL-6</em>, and <em>IFN-λ1</em>) in HEV-replicating HepG2/C3A and PLC3 cells (<a href="#fig2" class="usa-link">Fig 2A and B</a>). We found a significant increase in mRNA levels of <em>ISG15</em>, <em>IL-6</em>, and <em>IFN-λ1</em> in HepG2/C3A cells at 6 d.p.e. (<a href="#fig2" class="usa-link">Fig 2A</a>; left panel). <em>OAS2</em>, whose encoded protein is known to activate RNase L antiviral activity, was also induced in HepG2/C3A (albeit not significantly). In PLC3 cells, mRNA abundance of <em>MXA</em>, <em>ISG15</em>, <em>IL-6</em>, <em>OAS2</em>, and <em>IFN-λ1</em> remained unchanged (<a href="#fig2" class="usa-link">Fig 2B</a>; left panel). These results are in accordance with the higher responsiveness of HepG2/C3A cells to activation of cytosolic sensors and cytokines as compared to PLC3 cells (<a href="#fig1" class="usa-link">Fig 1B and C</a>). Notably, the level of <em>IFN-λ1</em> mRNA, which has a known antiviral role against HEV in mice (<a href="#bib57" class="usa-link" aria-describedby="bib57">Sari et al, 2021</a>), increased upon viral replication only in HepG2/C3A cells (<a href="#fig2" class="usa-link">Fig 2A and B</a>; left panels). In addition, <em>TNF</em> expression was slightly inhibited in HEV-replicating PLC3 cells (<a href="#fig2" class="usa-link">Fig 2B</a>; left panel) in accordance with previous results (<a href="#bib29" class="usa-link" aria-describedby="bib29">Hervouet et al, 2022</a>). These results showed that although HepG2/C3A and PLC3 cells are immunocompetent upon stimulation (<a href="#fig1" class="usa-link">Fig 1</a>), IFN-I/III response is only modestly induced upon HEV replication in HepG2/C3A cells and marginally in PLC3 cells. In line with previous in vitro studies (<a href="#bib19" class="usa-link" aria-describedby="bib19">Dong et al, 2012</a>; <a href="#bib49" class="usa-link" aria-describedby="bib49">Nan et al, 2014</a>; <a href="#bib5" class="usa-link" aria-describedby="bib5">Bagdassarian et al, 2018</a>; <a href="#bib39" class="usa-link" aria-describedby="bib39">Lin et al, 2019</a>), these results suggest that HEV is a poor inducer of IFN response.</p>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">Figure 2. pDC response against HEV-infected cells.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11966012_LSA-2025-03256_Fig2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b662/11966012/f0962eb21640/LSA-2025-03256_Fig2.jpg" loading="lazy" height="766" width="739" alt="Figure 2."></a></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A, B)</strong> Quantification of the transcript expression levels of representatives of IFN-I/III signaling (i.e., <em>MXA</em>, <em>ISG15</em>, <em>OAS2</em>, and <em>IFN-λ1</em>) and NF-KB–related pathway (i.e., <em>TNF</em> and <em>IL-6</em>) determined at 6 d.p.e. of HepG2/C3A (A) and PLC3 (B) cells that were electroporated with 10 μg RNA [HEV cells] or mock-electroporated without RNA [cont cells], in the absence (left panels) or in co-culture with pDCs for 18 h (right panels), as indicated. No pDC and pDC co-culture conditions have been separated into left and right panels as steady-state levels of gene expression are different in these two datasets, and therefore, cross-comparisons between the two panels must be avoided. Bars represent copy number per μg total RNA; means ± SD; each dot represents one independent experiment (n = 3–8). Statistical analysis was done using the Wilcoxon rank-sum test with continuity correction; <em>P</em>-values: * ≤ 0.05, ** ≤ 0.005, and *** ≤ 0.0005. <strong>(C, D)</strong> pDCs were co-cultured with HEV-electroporated PLC3 or HepG2/C3A cells for 18 and 48 h. <strong>(C, D)</strong> Quantification of IL-29/28A/28B in supernatants of pDCs co-cultured with HEV-replicating HepG2/C3A (C) and PLC3 cells (D); n ≥ 3; statistical analysis was done using the Wilcoxon rank-sum test with continuity correction; <em>P</em>-values: * ≤ 0.05, ** ≤ 0.005, and *** ≤ 0.0005. <strong>(E, F, G)</strong> pDCs were co-cultured with HEV-electroporated PLC3 cells, and their supernatants, in various settings, or treated by inhibitors, as indicated, for 18 h. <strong>(E)</strong> Quantification of IFN-α in supernatants of pDCs co-cultured with HEV-replicating PLC3 cells [HEV cells] or treated with supernatants from HEV-infected cells [HEV SN] versus in the absence of pDCs [no pDC]; the uninfected [cont] cells were electroporated without HEV RNA and used as a negative control. Bars represent means ± SD, and each dot represents one independent experiment (n = 4). <strong>(F)</strong> Quantification of IFN-α in supernatants of pDCs in co-culture or separated from HEV-electroporated PLC3 cells by a permeable membrane (0.4 μm) of transwell [transwell setting]. The TLR7 agonist, imiquimod [IMQ], was used as a positive control. <strong>(E)</strong> Results presented as in (E); n = 4. <strong>(G)</strong> Co-culture of pDCs and HEV-electroporated PLC3 cells was treated by inhibitors of TLR7 [IRS661], blocking antibodies against α<sub>L</sub>β<sub>2</sub>-integrin and ICAM-1, followed by the quantification of IFN-α in supernatants of the co-cultures. <strong>(E)</strong> Results presented as in (E); n &gt; 3 independent experiments. Statistical analysis was done using the Wilcoxon rank-sum test with continuity correction; <em>P</em>-values: * ≤ 0.05, ** ≤ 0.005, and *** ≤ 0.0005.</p></figcaption></figure><p>In vivo studies of HEV infection (i.e., at the tissue level) have, nonetheless, demonstrated induction of antiviral responses (<a href="#bib80" class="usa-link" aria-describedby="bib80">Yu et al, 2010</a>; <a href="#bib45" class="usa-link" aria-describedby="bib45">Moal et al, 2013</a>; <a href="#bib58" class="usa-link" aria-describedby="bib58">Sayed et al, 2017</a>). We thus thought to investigate whether human primary pDCs can mount the IFN-I/III response. The human primary pDCs were stimulated upon co-culture for 18 h with HEV-replicating cells. As in aforementioned experiments, we analyzed mRNA abundance of <em>MXA</em>, <em>ISG15</em>, <em>IFN-λ1</em>, <em>OAS2</em>, <em>TNF</em>, and <em>IL-6</em> by RT–qPCR (<a href="#fig2" class="usa-link">Fig 2A and B</a>; right panels). We observed up-regulated expression of all these effectors, and among these, <em>MXA</em>, <em>ISG15</em>, <em>IFN-λ1</em>, and <em>OAS2</em> were significantly up-regulated when pDCs were co-cultured with HEV-replicating HepG2/C3A cells, as compared to basal levels with uninfected cells (<a href="#fig2" class="usa-link">Fig 2A and B</a>; right panels). Overall, likely because of the inhibitory mechanisms of HEV viral products in PLC3 cells targeting IFN-I/III amplification pathways, there was a poor activation of downstream effectors, ISGs, and cytokines in the mixed cell culture, with the exception of <em>IFN-λ1</em>. The induction of <em>IFN-λ1</em> was common to the pDC co-cultures of HEV-replicating HepG2/C3A and PLC3 cells. We tested whether this was true at the protein level by ELISA for lambda IFNs or type III IFNs (IFN-λ1 [IL-29], IFN-λ2 [IL-28A], and IFN-λ3 [IL-28B]). We observed that IFN-III protein levels were significantly up-regulated only at 48 h post-pDC co-culture with HEV-replicating HepG2/C3A cells, as compared to control cells co-cultured with pDCs (<a href="#fig2" class="usa-link">Fig 2C</a>). Even though <em>IFN-λ1</em> mRNA levels were up-regulated upon co-culture of pDCs with HEV-replicating PLC3 cells as early as 18 h of co-culture (<a href="#fig2" class="usa-link">Fig 2B</a>; right panel), only a small amount of secreted IFN-III was detectable at the protein level 48 h post-co-culture (<a href="#fig2" class="usa-link">Fig 2D</a>). This is possibly because of the lesser sensitivity of the ELISA over RT–qPCR analyses (i.e., lower dynamic range). Further analyses with more sensitive/advanced approaches for IFN-III detection will be needed. Therefore, IFN-III is more robustly up-regulated in pDC co-culture of HEV-replicating HepG2/C3A cells.</p>
<p>Next, we tested whether pDCs can produce IFNs in response to HEV-replicating cells by quantifying the secreted levels of multiple subtypes of IFN-α as representative of IFN-I signaling (<a href="#fig2" class="usa-link">Fig 2E</a>). HEV-replicating PLC3 cells alone did not produce detectable IFN-α levels, in accordance with previous findings, showing that HEV replication does not induce IFN-I secretion (<a href="#bib80" class="usa-link" aria-describedby="bib80">Yu et al, 2010</a>; <a href="#bib45" class="usa-link" aria-describedby="bib45">Moal et al, 2013</a>; <a href="#bib49" class="usa-link" aria-describedby="bib49">Nan et al, 2014</a>; <a href="#bib58" class="usa-link" aria-describedby="bib58">Sayed et al, 2017</a>). In contrast, when pDCs were co-cultured with HEV-replicating PLC3 cells, more than 1,000 pg/ml of IFN-α was secreted, whereas no IFN-α was detected when pDCs were co-cultured with uninfected control cells (<a href="#fig2" class="usa-link">Fig 2E</a>). We also treated pDCs with cell-free HEV to verify whether circulating virus particles can also activate pDCs in a cell-independent manner. PDCs exposed to the filtered supernatant [SN] collected from HEV-replicating cells very modestly secreted IFN-α (<a href="#fig2" class="usa-link">Fig 2E</a>), suggesting that physical contact between pDCs and HEV-replicating cells is required for the robust pDC IFN-I secretion. Therefore, we assessed whether HEV-replicating PLC3 cells activated pDCs when the two cell types were separated by a 0.4-μm permeable membrane, which allows diffusion of virions but not cells. In this transwell setting, pDCs placed in the top chamber did not produce any IFN-α, validating that physical contact between the cells is required for pDC stimulation (<a href="#fig2" class="usa-link">Fig 2F</a>). Imiquimod [IMQ], which is a soluble agonist of TLR7 (<a href="#bib24" class="usa-link" aria-describedby="bib24">Gibson et al, 2002</a>), was used as an additional positive control to rule out nonspecific effect of the setting (<a href="#fig2" class="usa-link">Fig 2F</a>). Furthermore, when the co-cultures were treated with blocking antibodies against ICAM-I and α<sub>L</sub>β<sub>2</sub>-integrin, two cell–cell adhesion proteins that mediate cellular contacts (<a href="#bib43" class="usa-link" aria-describedby="bib43">Marlin &amp; Springer, 1987</a>; <a href="#bib4" class="usa-link" aria-describedby="bib4">Assil et al, 2019b</a>), pDC response to infected cells was significantly reduced as compared to untreated pDCs in co-culture with HEV-replicating cells (<a href="#fig2" class="usa-link">Fig 2G</a>). Because TLR7 is responsible for sensing replicative viral RNAs in pDCs (<a href="#bib73" class="usa-link" aria-describedby="bib73">Webster et al, 2016</a>; <a href="#bib56" class="usa-link" aria-describedby="bib56">Reizis, 2019</a>; <a href="#bib4" class="usa-link" aria-describedby="bib4">Assil et al, 2019b</a>), we tested its contribution in pDC response to HEV-replicating cells. When the co-cultures were treated with the TLR7 inhibitor IRS661, no IFN-α was secreted (<a href="#fig2" class="usa-link">Fig 2G</a>), suggesting that stimulation of pDCs by HEV-replicating cells is mainly mediated by TLR7. Together, our results show that direct contacts mediated by ICAM-I and, at least in part, with α<sub>L</sub>β<sub>2</sub>-integrin, enable pDCs to sense and respond to HEV-replicating cells in a TLR7-dependent manner.</p>
<p>We then tested whether IFN-α produced by pDCs inhibited viral propagation in either HepG2/C3A or PLC3 cells (<a href="#fig3" class="usa-link">Fig 3A–C</a>). Upon 18 h of co-culture of pDCs with HEV-replicating HepG2/C3A cells, we observed a 1-log decrease in viral RNA yield (readout for viral replication) compared with the absence of pDCs (<a href="#fig3" class="usa-link">Fig 3A</a>). At this early time of co-culture, the impact of pDCs on viral RNA replication was not yet observed for PLC3 cells (<a href="#fig3" class="usa-link">Fig 3B</a>). This may result from a stronger up-regulation of ISGs in HEV-replicating HepG2/C3A cells (in the absence of pDCs) compared with PLC3 cells, which may result in the synergy of the antiviral impact of pDCs, leading to an earlier decrease in viral replication in HepG2/C3A cells. As we did not observe pDC-mediated viral control in HEV-replicating PLC3 cells at this early time point, we thus studied the effect of pDCs when co-cultured for 48 h. After a longer co-culture of 48 h with pDCs, we observed a 50% reduction of HEV-replicating cells as compared to cultures without pDCs (<a href="#fig3" class="usa-link">Fig 3C</a>).</p>
<figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">Figure 3. Control of HEV infection by pDCs.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11966012_LSA-2025-03256_Fig3.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b662/11966012/548400e65319/LSA-2025-03256_Fig3.jpg" loading="lazy" height="546" width="738" alt="Figure 3."></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A, B)</strong> Quantification of HEV RNA replication levels in HepG2/C3A (A) and PLC3(B) cells, in the absence or in co-culture with pDCs, as indicated; means ± SD; each dot represents one independent experiment in terms of fold expression compared with [no pDC] condition; n = 4 for HepG2/C3A cells and PLC3 cells. Statistical analysis was done using the Wilcoxon rank-sum test with continuity correction and <em>P</em>-value adjustment with the Bonferroni method; <em>P</em>-values: * ≤ 0.05, ** ≤ 0.005, and *** ≤ 0.0005. <strong>(C)</strong> Quantification of ORF2-expressing cells in the presence or absence of pDCs by flow cytometry; fold change in HEV-replicating (ORF2+) PLC3 cells in the absence and presence of pDCs for 48 h (n = 4). <strong>(D, E)</strong> HEV-replicating PLC3 cells (GFP− ORF2+) and uninfected cells (GFP+ ORF2−) were co-cultured in the presence and absence of pDCs for 48 h. <strong>(D)</strong> Quantification of ORF2- and GFP-expressing cells by flow cytometry; (D) results are presented as representative dot blots. <strong>(E)</strong> Fold change in ORF2+ GFP− and ORF2+ GFP+ cells was quantified by flow cytometry. Bars represent means ± SD, and each dot represents one independent experiment (n = 5). Statistical analysis was done using the Wilcoxon rank-sum test with continuity correction and <em>P</em>-value adjustment with the Bonferroni method; <em>P</em>-values: * ≤ 0.05, ** ≤ 0.005, and *** ≤ 0.0005.</p></figcaption></figure><p>To further examine whether the pDC-mediated IFN response decreases the percentage of newly infected cells, we co-cultured a mixture of PLC3 cells, electroporated with HEV RNA and not expressing GFP (ORF2+/GFP− cells) along with uninfected cells positive for GFP (ORF2−/GFP+) in the presence or absence of pDCs for 48 h. The impact of pDC response on HEV spread was assessed by flow cytometry to quantify the percentage of newly infected cells, that is, GFP+ cells that became ORF2+ because of viral spread (ORF2+/GFP+) in the presence versus absence of pDCs (<a href="#fig3" class="usa-link">Fig 3D and E</a>). Infected cells were distinguished from pDCs on the basis of size (FSC-SSC gating), and then, the infected cell type (PLC3 cells) was gated for the expression of ORF2 and/or GFP (<a href="#fig3" class="usa-link">Fig 3D</a>). The results demonstrated that pDC response reduced viral replication in cells replicating HEV (<a href="#fig3" class="usa-link">Fig 3E</a>, blue bars) and controlled viral spread to naive GFP+ cells by half, upon prolonged co-culture (<a href="#fig3" class="usa-link">Fig 3E</a>, green bars). Collectively, our results showed an effective control of HEV spread by pDC-mediated antiviral activities involving IFN-I production.</p></section><section id="s5"><h3 class="pmc_sec_title">Modulation of pDC response by HEV-expressing ORF2 mutant is dependent on the host cell type</h3>
<p>PDCs responded to contact with HEV-replicating cells via TLR7 recognition (<a href="#fig2" class="usa-link">Fig 2E</a>), suggesting that the HEV genome is transferred from infected cells to the endosomal compartments of pDCs, where TLR7 localizes (<a href="#bib21" class="usa-link" aria-describedby="bib21">Dreux et al, 2012</a>). The distinctive feature of the HEV ORF2 capsid protein is its generation in three distinct forms (ORF2i, ORF2g, and ORF2c), each with different localization and trafficking patterns within infected cells (<a href="#bib36" class="usa-link" aria-describedby="bib36">Lenggenhager et al, 2017</a>; <a href="#bib2" class="usa-link" aria-describedby="bib2">Ankavay et al, 2019</a>; <a href="#bib29" class="usa-link" aria-describedby="bib29">Hervouet et al, 2022</a>). To address whether these distinct ORF2 forms impact the host immune responses in infected cells, and consequently and/or in addition the sensing of infected cells by pDCs, we tested a series of ORF2 protein mutants of the p6 HEV strain (WT) (<a href="#figS1" class="usa-link">Fig S1E</a>). The first mutant, called 5R/5A mutant, expresses an ORF2 in which the arginine-rich motif located in the ORF2 N-terminal region, serving as a nuclear localization signal, was mutated and thus prevents ORF2 translocation into the nucleus (<a href="#bib29" class="usa-link" aria-describedby="bib29">Hervouet et al, 2022</a>). The nuclear export signal (NES) mutant expresses an ORF2 that lacks one of its NES, leading to its retention in the nucleus (<a href="#bib29" class="usa-link" aria-describedby="bib29">Hervouet et al, 2022</a>). Glycosylated proteins can influence immunological functions via their recognition by C-type lectin receptor (<a href="#bib13" class="usa-link" aria-describedby="bib13">Cambi et al, 2005</a>), either by weakening TLR7- and TLR9-mediated IFN-I/III response (<a href="#bib44" class="usa-link" aria-describedby="bib44">Meyer-Wentrup et al, 2008</a>; <a href="#bib23" class="usa-link" aria-describedby="bib23">Florentin et al, 2012</a>) or by contributing to cell interaction and viral uptake (<a href="#bib8" class="usa-link" aria-describedby="bib8">Bermejo-Jambrina et al, 2018</a>). Therefore, we generated an additional mutant, called STOP mutant, expressing an ORF2 protein with a nonfunctional signal peptide, preventing its translocation into the endoplasmic reticulum and thus production of glycosylated ORF2g/c forms, as recently described (<a href="#bib48" class="usa-link" aria-describedby="bib48">Nagashima et al, 2023</a>). This mutant carries a stop codon in the ORF2 signal peptide, which does not affect ORF3 expression. In the STOP mutant, ORF2 protein translation starts at the first initiation codon (Met1), stops at the stop codon (*10), and restarts at the second initiation codon (Met16) (<a href="#figS2" class="usa-link">Fig S2A</a>). PLC3 cells expressing this mutant produce intracellular ORF2i form (<a href="#figS2" class="usa-link">Fig S2B</a>; right panel), efficiently replicate HEV genome (<a href="#figS2" class="usa-link">Fig S2D</a>), and produce HEV particles (<a href="#figS2" class="usa-link">Fig S2C</a>, IP P1H1, and <a href="#figS2" class="usa-link">Fig S2E and F</a>), but no ORF2g/c proteins (<a href="#figS2" class="usa-link">Fig S2C</a>, IP P3H2). It is noteworthy that we observed the absence of ORF2g/c production in cell culture supernatants of the STOP mutant expressed in both PLC3 and HepG2/C3A cells (<a href="#figS2" class="usa-link">Fig S2G</a>). Finally, a previously characterized replication-defective p6 mutant (GAD) was also included in the analysis (<a href="#bib22" class="usa-link" aria-describedby="bib22">Emerson et al, 2013</a>).</p>
<figure class="fig xbox font-sm" id="figS2"><h4 class="obj_head">Figure S2. Determinant of the production of ORF2g/c forms.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11966012_LSA-2025-03256_FigS2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b662/11966012/2764711e8a1b/LSA-2025-03256_FigS2.jpg" loading="lazy" height="1992" width="741" alt="Figure S2."></a></p>
<div class="p text-right font-secondary"><a href="figure/figS2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> Construction: a stop codon introduced in the ORF2 signal peptide without affecting ORF3 expression. <strong>(B)</strong> WB detection of ORF2 in the supernatant [SN] and cell lysates [cells] of HEV-producing PLC3 cells (using 1E6 mAb: detect all ORF2 species, mostly ORF2g/c in WT SN). <strong>(C)</strong> Immunoprecipitation in SN of all ORF2 forms versus ORF2i using P3H2 and P1H1 mAbs, respectively. <strong>(D, E)</strong>. Quantification of HEV RNA levels in SN and HEV-infected PLC3 cells and extracellular infectious titers; mean ± SD; n ≥ 3. <strong>(F)</strong> Imaging of ORF2 (1E6 mAb) in Huh-7.5 cells infected by WT and STOP mutant. <strong>(G)</strong> Western blot detection of ORF2 in the SNs of HEV-replicating HepG2/C3A and PLC3 cells 6 d.p.e. with HEV WT or STOP RNA in two independent experiments (using 1E6, which detects all ORF2 species, mostly ORF2g/c in WT SN).</p></figcaption></figure><p>The subcellular distribution of ORF2 mutants, compared with WT ORF2, was first characterized using confocal microscopic analyses. The WT ORF2 was found in both cytoplasm and nucleus, in both HepG2/C3A and PLC3 cells (<a href="#fig4" class="usa-link">Fig 4A and B</a>). The 5R/5A mutant was excluded from the nucleus, whereas the NES mutant accumulated in the nucleus in both cell types (<a href="#fig4" class="usa-link">Fig 4A and B</a>), in accordance with previous observations (<a href="#bib2" class="usa-link" aria-describedby="bib2">Ankavay et al, 2019</a>; <a href="#bib29" class="usa-link" aria-describedby="bib29">Hervouet et al, 2022</a>). The STOP mutant exhibited localization similar to that of WT ORF2 (<a href="#fig4" class="usa-link">Fig 4A and B</a>). As expected, the GAD mutant, in which the polymerase active site was mutated and thus is replication-defective, did not produce ORF2 (<a href="#fig4" class="usa-link">Fig 4A and B</a>). We then assessed viral RNA yield produced in cells electroporated with the four mutant genomes or with the WT genome using RT–qPCR analysis. This attested comparable levels of HEV RNA, which is a prerequisite for testing the impact of this mutant panel on the response of co-cultured pDCs. The HEV RNA levels were similar across the mutants and WT genomes at 6 d.p.e. in the HepG2/C3A cells (<a href="#fig4" class="usa-link">Fig 4C</a>). PLC3 cells electroporated with the WT, 5R/5A, NES, and STOP mutant genomes also yielded similar levels of viral RNAs at 6 d.p.e. (<a href="#fig4" class="usa-link">Fig 4D</a>). About two-log less viral RNA was recovered in HepG2/C3A and PLC3 cells expressing the GAD genome mutant (<a href="#fig4" class="usa-link">Fig 4C and D</a>). Viral RNA detected in these cells likely represents “input” RNA, that is, electroporated viral RNA.</p>
<figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">Figure 4. Impact of ORF2 nuclear localization on pDC activation by studying co-cultures with specific HEV mutants.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11966012_LSA-2025-03256_Fig4.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b662/11966012/cc1b6feb8be3/LSA-2025-03256_Fig4.jpg" loading="lazy" height="977" width="730" alt="Figure 4."></a></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Mutations were designed to inactivate i) nuclear translocation of ORF2 as [5R5A] mutant, ii) the ORF2 export from nucleus as [NES] mutant, and iii) ORF2g/c secretion as [STOP] mutant. Cells harboring the [GAD] mutant that is deficient for viral replication and/or uninfected [cont] cells were used as negative controls. Plasmids containing the above-mentioned mutations in the HEV genome versus WT served as templates for in vitro transcription into HEV RNA, then transfected in either HepG2/C3A or PLC3 cells, as indicated. <strong>(A, B)</strong> Intracellular distribution of ORF2 (1E6 antibody, followed by targeting secondary antibody with Alexa Fluor 488) in HepG2/C3A (A) or PLC3 (B) cells electroporated with different mutants of HEV. <strong>(C, D)</strong> Quantification of HEV replication levels of mutants versus WT in HepG2/C3A (n = 4) <strong>(</strong>C) or PLC3 cells (n = 3–6) (D). Results represent HEV RNA copy number (primer/amplicon designed in ORF2 RNA) per μg total RNA, detected by RT–qPCR; bars represent means ± SD, and dot represents independent experiments. <strong>(E, F)</strong> Quantification of IFN-α in supernatants of pDCs that were co-cultured with cells harboring HEV with the indicated mutant genome versus WT, and GAD or uninfected, as negative controls; for 18 h; 6 d.p.e. of HepG2/C3A cells (n = 5) (E) and PLC3 cells (n = 3–8) (F); bars represent means ± SD, and dots represent independent experiments. Statistical analysis was done using the Wilcoxon rank-sum test; <em>P</em>-values: * ≤ 0.05, ** ≤ 0.005, and *** ≤ 0.0005.</p></figcaption></figure><p>Quantification of IFN-α production by pDCs co-cultured with HepG2/C3A expressing the different mutants demonstrated significant differences. First, pDCs co-cultured with HepG2/C3A cells expressing the GAD mutant did not trigger a pDC response (<a href="#fig4" class="usa-link">Fig 4E</a>), nor in the context of PLC3 cells (<a href="#fig4" class="usa-link">Fig 4F</a>). Because HEV RNA levels were reduced for the GAD mutant, it is tempting to hypothesize that active viral replication is required for mounting a pDC-mediated viral response. Next, pDCs co-cultured with HepG2/C3A expressing 5R/5A demonstrated a significant reduction in IFN-α production as compared to WT, and 5R/5A versus NES (<a href="#fig4" class="usa-link">Fig 4E</a>). These results suggested that in this cellular model, the nuclear localization of ORF2 influences pDC stimulation. Along the same lines, pDC-mediated IFN-α response was completely abolished upon co-culture with PLC3 cells expressing the 5R/5A mutant (<a href="#fig4" class="usa-link">Fig 4F</a>). Because comparable HEV RNA levels were quantified in NES versus 5R/5A mutant (<a href="#fig4" class="usa-link">Fig 4D</a>), the nuclear localization of ORF2, which is abrogated for the 5R/5A mutant, may affect pDC response against HEV. Because the 5R/5A mutant also does not produce infectious viruses (<a href="#bib29" class="usa-link" aria-describedby="bib29">Hervouet et al, 2022</a>), pDC response against HEV might be modulated by viral particle release and/or nuclear ORF2.</p>
<p>IFN-α production was lower when pDCs were co-cultured with PLC3 cells expressing the STOP mutant, as compared to cells expressing WT ORF2, suggesting that ORF2g/c forms contribute to the pDC IFN-I response in these cells. The ORF2g/c forms produced by the WT ORF2 may be recognized by the C-type lectin receptors of pDCs, contributing to an enhanced response to immunostimulatory RNA. Alternatively, ORF2g/c expression may facilitate viral RNA transfer to pDCs and its subsequent recognition by TLR7 in endosomes. However, the decrease in pDC response to the STOP mutant was not observed for HepG2/C3A cells (<a href="#fig4" class="usa-link">Fig 4E</a>).</p>
<p>Here, we showed that the pDCs exhibited reduced IFN-⍺ response to HepG2/C3A expressing the 5R/5A mutant, and a complete absence of this response in PLC3 cells. This is likely because of differences in immune signaling among the two cell lines with the 5R/5A mutant. We observed that HEV infection impacted, in a cell type–dependent manner, the expression of <em>TNF</em> (<a href="#fig2" class="usa-link">Fig 2A</a>), a representative cytokine of NF-κB signaling, known to modulate the expression of regulators of pDC adhesion and/or recruitment, including ICAM-I (<a href="#bib30" class="usa-link" aria-describedby="bib30">Kim et al, 2008</a>) and various inflammatory chemokines (<a href="#bib61" class="usa-link" aria-describedby="bib61">Sedger &amp; McDermott, 2014</a>). We thus decided to analyze the effect of 5R/5A mutation on <em>TNF</em> expression in these two cell lines (<a href="#figS3" class="usa-link">Fig S3A and B</a>). We found that the 5R/5A mutant significantly reduced <em>TNF</em> induction in PLC3 cells but not in HepG2/C3A cells as compared to HEV WT, thus potentially explaining the cell type–specific regulation by ORF2i. Taken together, our results show that ORF2 protein expression and localization in producer cells modulate pDC response, with a magnitude that is cell type–dependent.</p>
<figure class="fig xbox font-sm" id="figS3"><h4 class="obj_head">Figure S3. Differences in HEV-induced immune signaling among cell types.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11966012_LSA-2025-03256_FigS3.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b662/11966012/058e75a01b7e/LSA-2025-03256_FigS3.jpg" loading="lazy" height="407" width="763" alt="Figure S3."></a></p>
<div class="p text-right font-secondary"><a href="figure/figS3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A, B)</strong> Quantification of the transcript expression levels of <em>TNF</em> determined at 6 d.p.e. in HepG2/C3A (A) and PLC3 (B) cells. Bars represent the copy number per μg total RNA; means ± SD; each dot represents one independent experiment (n = 3–5). Statistical analysis was done using the Wilcoxon rank-sum test with continuity correction; <em>P</em>-values: * ≤ 0.05, ** ≤ 0.005, and *** ≤ 0.0005.</p></figcaption></figure></section><section id="s6"><h3 class="pmc_sec_title">ORF2 protein forms contribute to the robustness of contact between replicating cells and pDCs</h3>
<p>We sought to determine whether HEV ORF2-mediated regulation of the strength and duration of cell-to-cell contacts could impact pDC response to infected cells. To achieve this, we developed a confocal imaging pipeline that quantifies cell proximity between pDCs and infected cells (<a href="#figS4" class="usa-link">Fig S4A</a>). Because more contrasting effects among ORF2 mutants were observed in PLC3 cells than in HepG2/C3A cells (<a href="#fig4" class="usa-link">Fig 4</a>), we selected PLC3 cells for investigating the role of cell-to-cell contacts in pDC-mediated response. The pDCs and HEV-replicating cells were stained using CellTrace Violet (CTV) and CellTracker Red (CMTPX), respectively, and then co-cultured and fixed after 4 h (<a href="#fig5" class="usa-link">Fig 5A</a>). HEV-replicating cells were identified using the P3H2 anti-ORF2 antibody, which recognizes all forms of ORF2 (ORF2g/c/i) (<a href="#bib7" class="usa-link" aria-describedby="bib7">Bentaleb et al, 2022</a>). The raw images were analyzed by an ImageJ macro-driven automatic analysis of cell–cell contacts. The two criteria for selection of contacts between pDCs and infected cells were the distance between the two cell types (1 µm), and a contact area (&gt;0 µm<sup>2</sup>) between the surfaces of the two cell types (<a href="#figS4" class="usa-link">Fig S4B</a>).</p>
<figure class="fig xbox font-sm" id="figS4"><h4 class="obj_head">Figure S4. Confocal imaging pipeline to quantify cell proximity between pDCs and infected cells.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11966012_LSA-2025-03256_FigS4.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b662/11966012/2f9ed8b32ba8/LSA-2025-03256_FigS4.jpg" loading="lazy" height="575" width="776" alt="Figure S4."></a></p>
<div class="p text-right font-secondary"><a href="figure/figS4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> Illustrative pipeline explaining steps in acquisition and treatment of confocal images used to quantify infected cells, enumerate the contacts between infected cells and pDCs, and measure the contact area. <strong>(B)</strong> Illustration describing criteria of contact enumeration as set in the macro used to analyze the confocal images. <strong>(C)</strong> Violin plots represent the area of contact/proximity of PLC3-infected cells and pDCs from automatic tracking (i.e., defined within a cell-to-cell distance &lt; 1 μm; contact area&gt; 0 µm<sup>2</sup>) with detection of CTV/pDCs and CMTPX+ ORF2+/infected cells, as a reference; violin plots present the median and each dot for independent image (n &gt; 6) from three distinct experiments. Statistical analysis was done using the Wilcoxon rank-sum test.</p></figcaption></figure><figure class="fig xbox font-sm" id="fig5"><h4 class="obj_head">Figure 5. Effect of ORF2 expression on localization on physical contacts between HEV cells and pDCs.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11966012_LSA-2025-03256_Fig5.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b662/11966012/8a1173613e28/LSA-2025-03256_Fig5.jpg" loading="lazy" height="1076" width="725" alt="Figure 5."></a></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A, B, C)</strong> pDCs co-cultured with HEV-replicating PLC3 cells 6 d.p.e, bearing the indicated mutations versus WT and GAD, as reference and negative control, respectively. Confocal imaging of pDCs and HEV co-culture performed after 4-h incubation. <strong>(A)</strong> Representative confocal stack imaging of ORF2 immunostaining (green) in infected PLC3 cells, which were stained with CellTracker Red before co-cultures (CMTPX, red), combined with pDCs stained with Cell-Tracer Violet before co-culture (CTV; blue). <strong>(B)</strong> Frequency of HEV-replicating (ORF2+) PLC3 cells detected among the cells defined as non-pDC (CTV−/CMTPX+); bars present means ± SD and each dot for independent image (n ≥ 8) from four distinct experiments. <strong>(C)</strong> Contact/proximity of PLC3-infected cells and pDCs with detection of CTV/pDCs and CMTPX+ ORF2+/infected cells, as reference; bars present means ± SD and each dot for independent image (n ≥ 8) from four distinct experiments. Statistical analysis was done using the Wilcoxon rank-sum test; <em>P</em>-values: * ≤ 0.05, ** ≤ 0.005, and *** ≤ 0.0005.</p></figcaption></figure><p>The percentage of ORF2-positive cells was within a comparable range among WT and the mutants in PLC3 cells (<a href="#fig5" class="usa-link">Fig 5B</a>), in agreement with their similar RNA levels (<a href="#fig4" class="usa-link">Fig 4D</a>). As compared to co-culture of pDCs with cells expressing WT ORF2, fewer cell-to-cell contacts (<a href="#fig5" class="usa-link">Fig 5C</a>) were observed between pDCs and cells expressing the 5R/5A (i.e., reproducible but not significant), and a significant 40% decline in pDC contacts with cells expressing the STOP mutant. Taking into consideration that the STOP mutant displayed a reduced induction of pDC response (<a href="#fig4" class="usa-link">Fig 4F</a>), this decline in pDC response to the STOP mutant could be due to reduced cell-to-cell contacts. Nevertheless, no difference was observed in the quality or strength of contacts, expressed in terms of contact area, across the mutants and WT (<a href="#figS4" class="usa-link">Fig S4C</a>). This indicates that pDC function is likely influenced by frequency of contact formation rather than the area of contact between the infected cell and the pDC, which was found to be consistent across the WT and mutants.</p>
<p>Next, to further investigate the effect of ORF2 nuclear localization on immune signaling, we carried out single-cell imaging flow cytometry experiments in HepG2/C3A cells (<a href="#fig6" class="usa-link">Fig 6A</a>), which had more robustly up-regulated ISGs upon HEV infection independently of pDCs (<a href="#fig2" class="usa-link">Fig 2A</a>; left panel). Single-cell imaging flow cytometry allowed us to distinguish infected cells with (trORF2+) or without ORF2 nuclear translocation (trORF2−). We then examined these cells for IRF3 nuclear translocation, which is known to up-regulate the expression of several cytokines, including IFN-I, IFN-III, and CXCL10 (<a href="#bib12" class="usa-link" aria-describedby="bib12">Brownell et al, 2014</a>), as well as surface molecules. Secretion of chemokines can lead to the recruitment of certain immune cell subsets. For instance, CXCL10 secretion triggers pDC recruitment (<a href="#bib17" class="usa-link" aria-describedby="bib17">Di Domizio et al, 2020</a>).</p>
<figure class="fig xbox font-sm" id="fig6"><h4 class="obj_head">Figure 6. Single-cell analysis of IRF3 nuclear localization in HEV-infected HepG2/C3A cells.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11966012_LSA-2025-03256_Fig6.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b662/11966012/16904db8cebf/LSA-2025-03256_Fig6.jpg" loading="lazy" height="702" width="724" alt="Figure 6."></a></p>
<div class="p text-right font-secondary"><a href="figure/fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>HepG2/C3A cells expressing IRF3-GFP were electroporated with WT HEV RNA and fixed 2 d.p.e. after staining with viability stain (Zombie aqua), and were then stained for ORF2 (APC) and nucleus (Hoechst). Single-cell images of viable ORF2+ cells were obtained and categorized according to the localization of IRF3 and ORF2. <strong>(A)</strong> Cells with translocation of ORF2 and IRF3 into the nucleus are represented by TrORF2+ and TrIRF3+. Cells without translocation of ORF2 and IRF3 into the nucleus are represented by TrORF2− and TrIRF3−. <strong>(B)</strong> Percentage of HEV-replicating or ORF2+ cells with nuclear IRF3 translocation (ORF2+ TrIRF3+). <strong>(C)</strong> Percentage of ORF2 nuclear translocation (TrORF2+) among TrIRF3+ cells. Bars represent means ± SD, and each dot represents one independent experiment (n = 4). Statistical analysis was performed using a two-sided Wilcoxon rank-sum test with continuity correction; <em>P</em>-values: * ≤ 0.05, ** ≤ 0.005, and *** ≤ 0.0005.</p></figcaption></figure><p>We observed that around 10% of ORF2+ cells displayed IRF3 nuclear translocation (TrIRF3+) (<a href="#fig6" class="usa-link">Fig 6B</a>). Further, around 80% of TrIRF3+ cells exhibited TrORF2+ or an ORF2 nuclear signal (<a href="#fig6" class="usa-link">Fig 6C</a>). These results revealed that the immune status of IRF3 in HepG2/C3A-infected cells can differ depending on the nuclear translocation of HEV ORF2, further validating the impact of ORF2 nuclear localization on the immune status of the cells.</p>
<p>Together, our results suggest that ORF2g/c production likely modulates the number of cell–cell contacts and, subsequently, pDC-mediated IFN-I response. In the absence of ORF2g/c (STOP mutant), there is a significant decline in pDC-infected cell contacts. On the other hand, nuclear ORF2 can dictate the immune status in the infected cell via the IRF3 signaling axis, possibly fueling the pDC response.</p></section></section><section id="s7"><h2 class="pmc_sec_title">Discussion</h2>
<p>Several studies have highlighted the suppressive impact of HEV on IFN response (<a href="#bib16" class="usa-link" aria-describedby="bib16">Devhare et al, 2021</a>). Here, we investigated whether these inhibitory mechanisms can be overcome by pDCs, which act as immune sentinels against viral infections. Given the limited number of studies on cellular systems that effectively replicate HEV, we first determined the ability of three hepatoma cell lines to respond to several immune stimuli and replicate the viral genome. Based on their robust response to stimuli and their ability to replicate gt3 p6 strain HEV RNA, we selected PLC3 and HepG2/C3A cells for further analyses. Upon 6 d of HEV RNA electroporation, we observed a modest up-regulation of IRF3- and NF-KB–mediated gene expression in HepG2/C3A cells. The genes that were significantly up-regulated in HEV-replicating HepG2/C3A cells included <em>ISG15</em>, <em>IFN-λ1</em>, <em>OAS-2</em>, and <em>IL-6</em>. The up-regulation of <em>IFN-λ1</em> expression in HepG2/C3A cells was in agreement with studies recognizing type III IFN response as a fundamental anti-HEV cytokine in hepatocytes both clinically (<a href="#bib47" class="usa-link" aria-describedby="bib47">Murata et al, 2020</a>) and in vitro (<a href="#bib78" class="usa-link" aria-describedby="bib78">Yin et al, 2017</a>; <a href="#bib76" class="usa-link" aria-describedby="bib76">Wu et al, 2018</a>). Even though HEV can persist in the presence of a sustained type III IFN response (<a href="#bib78" class="usa-link" aria-describedby="bib78">Yin et al, 2017</a>), this response prevents inflammation and helps maintain barrier integrity (<a href="#bib11" class="usa-link" aria-describedby="bib11">Broggi et al, 2020</a>). Importantly, <em>TNF</em> up-regulation was reduced by HEV in PLC3 cells, in agreement with previous data obtained in PLC3 cells (<a href="#bib29" class="usa-link" aria-describedby="bib29">Hervouet et al, 2022</a>). This was not observed for HepG2/C3A cells, suggesting that HEV modulates NF-KB–mediated signaling in a cell type–specific manner. While exhibiting strong antiviral effects (<a href="#bib72" class="usa-link" aria-describedby="bib72">Wang et al, 2016</a>), TNF-α is also implicated in proinflammatory processes exacerbating the severity of liver disease upon HEV infection (<a href="#bib6" class="usa-link" aria-describedby="bib6">Behrendt et al, 2017</a>). Importantly, the differences in immune profiles of PLC3 cells and HepG2/C3A may be due to the expression of hepatitis B surface antigen (HBsAg) by PLC3 cells, a known limitation of this cell type (<a href="#bib42" class="usa-link" aria-describedby="bib42">MacNab et al, 1976</a>). Nevertheless, these cells may relevantly mimic the immune characteristics of individuals with preexisting or previous infections or medical condition, predisposing them to HEV infection. In accordance—in the absence of pDC—secreted type III IFNs were undetectable in HEV-replicating PLC3 cells alone, and HEV-replicating cells also did not secrete any detectable IFN-α, consistent with previous studies using HepG2 cells (<a href="#bib78" class="usa-link" aria-describedby="bib78">Yin et al, 2017</a>) and iPSC-HLCs (<a href="#bib76" class="usa-link" aria-describedby="bib76">Wu et al, 2018</a>).</p>
<p>Upon co-culture of HEV-replicating HepG2/C3A cells with pDCs, the three ISGs <em>MXA</em>, <em>ISG15</em>, and <em>OAS2</em>, as well as <em>IFN-λ1</em>, were robustly up-regulated. In fact, IFN-III response was also evident at the protein level after 48 h of pDC co-culture with HEV-replicating HepG2/C3A cells. However, the expression of ISGs barely increased at 18 h post-co-culture with HEV-replicating PLC3 cells. Upon pDC co-culture of HEV-replicating PLC3 cells, only <em>IFN-λ1 </em>was up-regulated significantly at the mRNA level and yet poorly detected as a secreted protein, likely owing to the sensitivity limit of the currently available ELISA detection and thus requiring analysis with more sensitive techniques. <em>IFN-λ1</em> expression by pDCs has been documented to actively participate in pDC response, for example, against human cytomegalovirus and SARS-CoV-2 (<a href="#bib81" class="usa-link" aria-describedby="bib81">Yun et al, 2021</a>; <a href="#bib71" class="usa-link" aria-describedby="bib71">Venet et al, 2023</a>). Whether the source of <em>IFN-λ1</em> up-regulation was HEV-replicating cells, pDCs, or both cell types in concert remains to be determined. Because the expression of these effectors is crucial for restricting viral replication (<a href="#bib60" class="usa-link" aria-describedby="bib60">Schneider et al, 2014</a>), the absence of ISG up-regulation presents a possible Achilles’ heel in host immune defense against HEV as this might contribute to HEV persistence in culture and its elimination might be challenging even after prolonged exposure to high doses of IFNs (<a href="#bib70" class="usa-link" aria-describedby="bib70">Todt et al, 2016</a>). The lack of ISG expression was overcome by pDCs against HEV in the form of a strong IFN-I response when co-cultured with infected cells. This could activate ISG expression in neighboring cells (<a href="#bib10" class="usa-link" aria-describedby="bib10">Bourdon et al, 2020</a>) and also facilitate adaptive immune responses (<a href="#bib15" class="usa-link" aria-describedby="bib15">Crouse et al, 2015</a>). Along the same line, the IFN-I response against HEV was dependent on pDC-infected cell contact formation. This was orchestrated by ICAM-I and α<sub>L</sub>β<sub>2</sub>-integrin and TLR7, in agreement with results obtained for other viral infections (<a href="#bib78" class="usa-link" aria-describedby="bib78">Yin et al, 2017</a>). The contact between pDCs and infected cells is beneficial for host defense, considering that HEV is resistant to exogenous IFN treatment (<a href="#bib70" class="usa-link" aria-describedby="bib70">Todt et al, 2016</a>; <a href="#bib72" class="usa-link" aria-describedby="bib72">Wang et al, 2016</a>). On a different note, a prior study has shown that HEV-ORF3 can attenuate TLR7 expression (<a href="#bib35" class="usa-link" aria-describedby="bib35">Lei et al, 2018</a>); this effect is less likely to be observed for pDCs as these are nonpermissive to most viruses (<a href="#bib64" class="usa-link" aria-describedby="bib64">Silvin et al, 2017</a>; <a href="#bib56" class="usa-link" aria-describedby="bib56">Reizis, 2019</a>; <a href="#bib71" class="usa-link" aria-describedby="bib71">Venet et al, 2023</a>). PDC response failed to control HEV replication after 18 h of co-culture with PLC3 cells but effectively controlled HEV replication in HepG2/C3A under the same conditions. We observed more efficient or faster viral control in HepG2/C3A cells compared with PLC3 cells. This was likely due to a significant up-regulation of ISGs (<em>ISG15</em>, <em>IFN-λ1</em>, and <em>OAS2</em>)—even in the absence of pDCs—in HEV-replicating HepG2/C3A cells versus an inexistent one in HEV-replicating PLC3 cells. We hypothesize that the preexisting ISG response in HEV-replicating HepG2/C3A, produced in the absence of pDCs, potentiates a more effective viral control in the presence of pDCs. Therefore, the initial ISG expression in infected cells may synergize the antiviral response by pDCs. Upon further scrutiny, pDC-mediated IFN response reduced HEV production by 50% 48 h post-co-culture in PLC3 cells. This also contributed to an overall decline in viral spread to neighboring cells. These results confirmed the need for a prolonged IFN response for controlling HEV replication.</p>
<p>In the absence of ORF2 nuclear translocation (5R/5A mutant), pDC IFN-I response was diminished in HepG2/C3A cells and abolished in PLC3 cells. Furthermore, when ORF2 accumulated in the nucleus (NES mutant), there was no difference in pDC response as compared to the WT. In addition, the depletion of ORF2g/c secretion (STOP mutant) lowered the pDC response in PLC3 cells but not in HepG2/C3A cells. Altogether, these observations suggest that ORF2 nuclear translocation and secretion of glycosylated ORF2g/c forms might modulate pDC response in a cell type–dependent manner. Because many adhesion molecules are ISGs (<a href="#bib52" class="usa-link" aria-describedby="bib52">Parr &amp; Parr, 2000</a>; <a href="#bib40" class="usa-link" aria-describedby="bib40">Ma et al, 2022a</a>) and can impact the ability of HEV mutant harboring cells to form cell-to-cell contact, we compared their pDC contact–forming propensity. Only cells harboring the STOP mutant differed from WT HEV cells in their ability to form contacts with pDCs. These results support the hypothesis of a reduced attraction of pDCs toward STOP-expressing PLC3 cells, as compared to control cells, because of the absence of ORF2g/c secretion. We did not see the same effect for HepG2/C3A cells, probably because of robust immune induction upon HEV infection. Alternative effector(s) favoring the contacts between pDCs and infected cells could be better expressed in HepG2/C3A cells compared with PLC3 cells, possibly because of the potent immune status, which may mask the effect of ORF2g/c on contact formation. However, this aspect requires further investigation. We should also take into account the decrease in viral infectivity in the absence of ORF2g/c (<a href="#figS2" class="usa-link">Fig S2E</a>), which could in turn lead to reduced pDC IFN response. Even though incubation with purified ORF2g/c does not impact HEV entry (<a href="#bib79" class="usa-link" aria-describedby="bib79">Yin et al, 2018</a>), our results do raise further questions on whether its secretion can indirectly offer an advantage to HEV for spreading to the neighboring cell by improving cell–cell adhesion, a relatively easier route of viral transmission (<a href="#bib82" class="usa-link" aria-describedby="bib82">Zhong et al, 2013</a>). Many studies have demonstrated that glycosylation is a common factor affecting cell–cell adhesion (<a href="#bib51" class="usa-link" aria-describedby="bib51">Ohtsubo &amp; Marth, 2006</a>; <a href="#bib28" class="usa-link" aria-describedby="bib28">Gu et al, 2012</a>). When rhesus macaques were infected with the HEV variant that did not express glycosylated ORF2 forms, viral replication was attenuated and viral shedding in feces declined significantly as compared to infection with WT HEV (<a href="#bib53" class="usa-link" aria-describedby="bib53">Ralfs et al, 2023</a>). Therefore, ORF2 g/c may not be essential for viral replication, but it may be instrumental for efficient viral production.</p>
<p>Our findings emphasize the significance of the ongoing co-evolution between hosts and pathogens, allowing the virus to disseminate while concurrently enhancing its detection by immune cells. The difference in pDC response among HEV WT, NES versus 5R/5A still remains enigmatic, but it paves a direction for further exploration in this exciting field. The presence of pDC response to 5R/5A-expressing HepG2/C3A cells may imply regulation of pDC response at an additional level, that is, cell type–dependent regulation. Because pDCs mounted an IFN-I response against 5R/5A in HepG2/C3A cells, we propose that this detection could have been fueled by a successful up-regulation of ISGs within HepG2/C3A cells compared with PLC3 cells upon HEV infection. Importantly, TNF signaling is induced in HepG2/C3A cells and inhibited in PLC3 cells. Because TNF signaling regulates cell adhesion via ICAM-1 (<a href="#bib54" class="usa-link" aria-describedby="bib54">Reglero-Real et al, 2014</a>) and activates an autocrine loop with low and sustained production of type I IFNs (<a href="#bib77" class="usa-link" aria-describedby="bib77">Yarilina et al, 2008</a>), it could contribute to differential pDC response for the two cell types. The difference in pDC-mediated IFN-I response to WT and 5R/5A mutant is particularly interesting and was observed for both cell lines, even though at different magnitudes. The 5R/5A mutant, which is characterized by an absence of nuclear ORF2, displayed a reduced or abolished pDC response, that is, depending on the cell line. We then uncovered a cell line–specific induction of <em>TNF</em>, depending on ORF2 nuclear translocation. It indicates a possible differential modulation of downstream effector(s), including those involved in pDC adhesion/recruitment, and eventually leading to a weaker interferogenic synapse between PLC3 cells and pDCs, but not at play in HepG2/C3A cells. Further investigation will be needed to identify the effector(s) at play. We also investigated whether ORF2 nuclear localization could be involved in the immune state of the infected cells. We showed that IRF3 nuclear translocation tends to occur more in HEV-replicating cells with nuclear ORF2 than in HEV-replicating cells without nuclear ORF2. There could be several possible reasons for this activation of immune signaling, that is, intrusion of nuclear membrane, sensing of the viral ORF2 in the nucleus, or sensing of accidentally shuttled HEV RNA in the nucleus. It would be interesting to investigate this further. The activation of IRF3 signaling could, in turn, aid pDCs in viral sensing via cytokines and up-regulation of other effector molecules.</p>
<p>Globally, the HEV mutants advanced our comprehension about viral regulation of the IFN response and also mechanistic nuances of viral recognition by pDCs. In summary, our findings suggest that pDC response is likely regulated by the (i) transfer of viral entities to pDCs for recognition, (ii) number of cell-to-cell contacts, and (iii) intrinsic immune state of the infected cells.</p></section><section id="s8"><h2 class="pmc_sec_title">Materials and Methods</h2>
<section id="s9"><h3 class="pmc_sec_title">Reagents and antibodies</h3>
<p>Reagents used for pDC isolation are as follows: Ficoll-Hypaque (GE Healthcare Life Sciences), BDCA-4 magnetic beads (MACS Miltenyi Biotec), LS Columns (MACS Miltenyi Biotec), and bottle-top vacuum filters 0.22 μm (Nalgene). Other reagents included polyI:C (LMW; InvivoGen) as TLR3 agonist, imiquimod as TLR7 agonist (InvivoGen), and TLR7 antagonist (IRS661, 5′-TGCTTGCAAGCTTGCAAGCA-3′) synthesized on a phosphorothioate backbone (MWG Biotech). The following antibodies were used: mouse anti-α<sub>L</sub>β<sub>2</sub> integrin (clone 38; Antibodies Online), mouse anti-ICAM-1 (clone LB-2; BD Bioscience), mouse anti-HEV ORF2 1E6 (antibody registry #AB-827236; Millipore), mouse anti-ORF2i/g/c P3H2 (<a href="#bib7" class="usa-link" aria-describedby="bib7">Bentaleb et al, 2022</a>), and mouse anti-ORF2i P1H1 (<a href="#bib7" class="usa-link" aria-describedby="bib7">Bentaleb et al, 2022</a>). Goat anti-Mouse IgG2b Cross-Adsorbed Secondary Antibody Alexa Fluor 488 (Catalog #A-21141; Thermo Fisher Scientific) was used for flow cytometry analysis.</p></section><section id="s10"><h3 class="pmc_sec_title">Cell lines</h3>
<p>PLC3 cells, a subclone of PLC3/PRF/5 cells (<a href="#bib46" class="usa-link" aria-describedby="bib46">Montpellier et al, 2018</a>), Huh-7.5 cells (RRID:CVCL_7927), which are hepatoma cells derived from Huh-7 cells (<a href="#bib9" class="usa-link" aria-describedby="bib9">Blight et al, 2002</a>), and HepG2/C3A cells (ATCC CRL-3581) (provided by Dr. V.L. Dao Thi and Dr. D. Moradpour) were used. PLC3 and Huh-7.5 cells were maintained in DMEM supplemented with 10% FBS, 100 U/ml penicillin, 100 mg/ml streptomycin, 2 mM L-glutamine, nonessential amino acids, and 1 mM sodium pyruvate (Life Technologies) at 37°C/5% CO<sub>2</sub>. HepG2/C3A cells were maintained in DMEM (GlutaMAX, Pyr-) supplemented with 10% FBS, 100 /ml penicillin at 37°C/5% CO<sub>2</sub>.</p></section><section id="s11"><h3 class="pmc_sec_title">pDC isolation and culture</h3>
<p>pDCs were isolated from blood from healthy adult human volunteers, which were obtained from the “Etablissement Francais du Sang” (EFS; Auvergne-Rhone-Alpes, France) under the convention EFS 16–2066 and according to procedures approved by the EFS committee. Informed consent was obtained from all subjects in accordance with the Declaration of Helsinki. Information on sex and age was available for all subjects, yet we previously showed that pDC responses are within the same range for all donors <a href="#bib84" class="usa-link" aria-describedby="bib84">Décembre et al, 2014</a>. PBMCs were isolated using Ficoll-Hypaque density centrifugation, and pDCs were positively selected from PBMCs using BDCA-4-magnetic beads (MACS Miltenyi Biotec), as previously described (<a href="#bib21" class="usa-link" aria-describedby="bib21">Dreux et al, 2012</a>; Decembre et al, 2014; <a href="#bib71" class="usa-link" aria-describedby="bib71">Venet et al, 2023</a>). The typical yields of PBMCs and pDCs were 500–800 × 10<sup>6</sup> and 1–2 × 10<sup>6</sup> cells, respectively, with a typical purity of &gt;95% pDCs. Isolated pDCs were maintained in RPMI 1640 medium (Life Technologies) supplemented with 10% FBS, 10 mM Hepes, 100 U/ml penicillin, 100 mg/ml streptomycin, 2 mM L-glutamine, nonessential amino acids, and 1 mM sodium pyruvate at 37°C/5% CO<sub>2</sub>.</p></section><section id="s12"><h3 class="pmc_sec_title">Stimulation of cells for immune response using agonists and recombinant cytokines</h3>
<p>PLC3 cells were seeded in 24-well plates at a concentration of 8.104 cells/well. After 24 h, cells were incubated with polyI:C LMW in complete medium at a concentration of 25 μg/ml and 100 μg/ml, in a total volume of 0.5 ml medium. Alternatively, cells were transfected with polyI:C LMW at a concentration of 0.3 μg/ml and 2 μg/ml. Lipofectamine 2000 (Invitrogen) in Opti-MEM I reduced serum media was used for transfection according to the manufacturer’s protocol. IFN-β subtype 1a (Catalog #PHC4244; Invitrogen), IFN-λ3/IL-28B (IL-28B) (PBL IFN Source #11820-1), and TNF (#16769; Cell Signaling Technology) were added at a concentration of 100 U/ml, 100 ng/ml, and 535 U/ml, respectively, in a total volume of 0.5 ml medium per well. The cells were harvested 6 h post-treatment for RNA extraction.</p></section><section id="s13"><h3 class="pmc_sec_title">Analysis of transcriptional levels by RT–qPCR</h3>
<p>RNAs were isolated from cells harvested in guanidinium thiocyanate citrate buffer (GTC) by phenol/chloroform extraction procedure, as described previously (<a href="#bib3" class="usa-link" aria-describedby="bib3">Assil et al, 2019a</a>). The mRNA levels of human <em>MXA</em>, <em>ISG15</em>, <em>IFN-λ1</em>, <em>IL-6</em>, <em>TNF</em>, <em>ORF2</em>, and glyceraldehyde-3-phosphate dehydrogenase (<em>GADPH</em>) were determined by RT–qPCR using High-Capacity cDNA Reverse Transcription Kit (ref# 4368813) and PowerUp SYBR Green Master Mix (ref# A25778) for RT–qPCR run on QuantStudio 6 PCR system and analyzed using QuantStudio Design and Analysis Software (Thermo Fisher Scientific). The mRNA levels were normalized to <em>GADPH</em> mRNA levels. The sequences of the primers used for RT–qPCR are listed in Table S1.</p>
<section class="sm xbox font-sm" id="SM1"><div class="caption p"><p>Table S1. <a href="/articles/instance/11966012/bin/LSA-2025-03256_TableS1.docx" class="usa-link" id="d67e1722">Primer used for RT–qPCR.</a><sup> (18.2KB, docx) </sup></p></div></section></section><section id="s14"><h3 class="pmc_sec_title">Plasmids and electroporation for HEV infectious system</h3>
<p>The plasmid pBlueScript SK(+) carrying the DNA of the full-length genome of adapted gt3 Kernow C-1 p6 strain (GenBank accession number <a href="https://ncbi.nlm.nih.gov/nucleotide/JQ679013" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">JQ679013</a>, kindly provided by Dr SU Emerson) was used as the WT genome. The 5R/5A and NES mutant plasmids were previously described (<a href="#bib29" class="usa-link" aria-describedby="bib29">Hervouet et al, 2022</a>). The STOP mutant was generated by site-directed mutagenesis using the following primers:</p>
<p>P6/mutstop-F: TGTTCTGCTGCTGTagTTCGTGTTTCTGCCTATGC.</p>
<p>P6/mutstop-R: GGCAGAAACACGAActACAGCAGCAGAACAACCC.</p>
<p>The Stop mutation was introduced by sequential PCR steps, as previously described (<a href="#bib2" class="usa-link" aria-describedby="bib2">Ankavay et al, 2019</a>), and verified by DNA sequencing. The non-replicative GAD mutant plasmid was a gift from Dr VL Dao Thi (<a href="#bib22" class="usa-link" aria-describedby="bib22">Emerson et al, 2013</a>). To prepare genomic HEV RNAs (capped RNA), the WT and mutant pBlueScript SK(+) HEV plasmids were linearized at their 3′ end with the MluI restriction enzyme (NEB) and transcribed with the mMESSAGE mMACHINE T7 Transcription kit (#1344; Ambion). 10 μg capped RNAs were next delivered to cells by electroporation using Gene Pulser Xcell 617BR 11218 (Bio-Rad).</p></section><section id="s15"><h3 class="pmc_sec_title">Co-culture of infected cells with isolated pDCs and ELISA</h3>
<p>Unless otherwise indicated, 5 × 10<sup>4</sup> pDCs were co-cultured with 6 × 10<sup>4</sup> PLC3 or HepG2/C3A cells, electroporated or not, or, as a control condition, filtered supernatants (0.4 μm). Infected cells were electroporated for 48, 72, or 144 h (as indicated) in a 200 μl final volume in 96-well round-bottom plates incubated at 37°C/5% CO<sub>2</sub>. When indicated, cells were co-cultured in 96-well format transwell chambers (Corning) with a 0.4-μm permeable membrane. 16–18 h later (as specified), cell culture supernatants were collected and the levels of IFN-α were measured using a commercially available specific ELISA kit (PBL IFN Source Catalog #411052) with an assay range of 12.5–500 pg/ml.</p></section><section id="s16"><h3 class="pmc_sec_title">Flow cytometry–based viral spread assays and imaging flow cytometry by ImageStreamX technology</h3>
<p>PLC3 cells were transduced with lentiviral-based vector pseudotyped with vesicular stomatitis virus glycoprotein to stably express GFP. After immunoisolation, pDCs were stained for 20 min at 37°C in the dark. Labeled pDCs were then spun down and resuspended in pDC culture medium. 5 × 10<sup>4</sup> pDCs were co-cultured with 3 × 10<sup>4</sup> HEV-replicating cells (electroporated 6 d before co-culture) and with 3 × 10<sup>4</sup> GFP+-uninfected cells for 48 h at 37°C/5% CO<sub>2</sub>. The level of viral spread from HEV-replicating cells (ORF2+) to uninfected cells (GFP+) during co-culture was determined by flow cytometry analysis as the frequency of infected cells (ORF2+/GFP+ population) among the GFP+ cell population and similarly in GFP- populations. At the indicated times, harvested cells were resuspended using 0.48 mM EDTA/PBS solution for the co-culture with pDCs. Cells were then incubated with 1 μl/ml viability marker diluted in PBS for 20 min at RT. Cytoperm/Cytofix and permeabilization/wash solutions (BD Bioscience) were used in the subsequent stages of the ORF2 staining protocol optimized in-house for flow cytometry. Cells were fixed with Cytofix for 20 min at 4°C and were then washed twice and resuspended in 1x Permwash. The fixed cells were treated with cold methanol (−20°C) for 45 min and washed twice with 1x Permwash. The cells were then incubated with 1E6 anti-HEV antibody for a minimum of 3 h. After another wash, cells were incubated with a secondary antibody, that is, Goat anti-Mouse IgG2b Cross-Adsorbed Secondary Antibody Alexa Fluor 488 (Catalog #A-21141; Thermo Fisher Scientific) for 2 h. The samples were acquired after final two washes. Flow cytometry analysis was performed using Canto II Becton Dickinson using BD FACSDIVA v8.1 software, and the data were analyzed using Flow Jo 10.8.1 software (Tree Star). The CTV Cell Proliferation kit (Life Technologies) was used to stain pDCs. LIVE/DEAD Fixable Near-IR Dead Cell Stain Kit (ref #<a href="https://www.ncbi.nlm.nih.gov/nuccore/L10119" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">L10119</a>; Thermo Fisher Scientific) was used to check viability. For imaging flow cytometry, cells expressing IRF3-GFP (construct was kindly provided by Dr. Marco Binder) (<a href="#bib75" class="usa-link" aria-describedby="bib75">Willemsen et al, 2017</a>) were electroporated with HEV WT RNA. The cells were then harvested at 48 h.p.e., followed by viability staining with Zombie Aqua (Cat # 423101; BioLegend), and then ORF2 staining protocol. At the final step, Hoechst (ref #H1399; Thermo Fisher Scientific) was used to stain the nuclei of the cells. After acquisition by ImageStreamX Mark II (Amnis—Millipore), analysis was done using IDEAS software.</p></section><section id="s17"><h3 class="pmc_sec_title">Microscopy and analysis of images</h3>
<p>At 4 h post-co-culture, CTV-stained pDCs and infected PLC3 and HepG2/C3A cells were fixed with PFA 4%, followed by immunostaining using anti-ORF2 P3H2 antibodies with a protocol optimized previously (<a href="#bib7" class="usa-link" aria-describedby="bib7">Bentaleb et al, 2022</a>). Confocal imaging was performed using Zeiss LSM980 scanning confocal microscopy. Co-cultured cells were automatically segmented based on the CMTPX staining for the cell line used and CTV labeling for pDCs. Next, the infected cells were identified by ORF2 expression, and number/frequency of infected cells were quantified using one of two similar versions of a home-made ImageJ macro (<a href="https://github.com/jbrocardplatim/PDC-contacts" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/jbrocardplatim/PDC-contacts</a>), adapted to HepG2/C3A or PLC3 cells. The frequency of pDCs within 3 μm of any cell was calculated, as well as the number of pDCs within 3 μm of any infected cell (from 0 to 3+). An additional macro was also used to measure the frequency of pDCs closer than 1 μm to infected cells and/or in direct contact; the individual contact areas (in μm<sup>2</sup>) were measured as well (<a href="https://github.com/jbrocardplatim/PDC-contacts" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/jbrocardplatim/PDC-contacts</a>). The calculation formula for contact frequencies considered variations in the number of infected cells to mitigate potential biases:</p>
<table class="disp-formula p" id="une1"><tr><td class="formula"><math id="m1" display="block" overflow="linebreak"><mrow><mfrac><mrow><mtext>Number</mtext><mo> </mo><mtext>of</mtext><mo> </mo><mtext>pDC</mtext><mo>−</mo><mtext>infected</mtext><mo> </mo><mtext>cell</mtext><mo> </mo><mtext>contact</mtext></mrow><mrow><mtext>Isolated</mtext><mo> </mo><mtext>pDCs</mtext><mo>+</mo><mtext>isolated</mtext><mo> </mo><mtext>infected</mtext><mo> </mo><mtext>cells</mtext><mo>+</mo><mtext>pDC</mtext><mo>−</mo><mtext>infected</mtext><mo> </mo><mtext>cell</mtext><mo> </mo><mtext>contact</mtext></mrow></mfrac><mo>.</mo></mrow></math></td></tr></table>
<p>To compare the ORF2 localization in mutants and WT bearing cells, nuclei were stained with NucSpot Live 650 (Catalog #40082; Biotium) and HEV-ORF2 was stained with 1E6 antibody. For all microscopy experiments, cells were seeded in 96-Well Optical-Bottom Plate (Thermo Fisher Scientific), coated with poly-L-lysine (P6282; Sigma-Aldrich).</p></section><section id="s18"><h3 class="pmc_sec_title">Western blotting and immunoprecipitation analyses</h3>
<p>Western blotting (WB) and immunoprecipitation (IP) analyses were performed as described previously (<a href="#bib7" class="usa-link" aria-describedby="bib7">Bentaleb et al, 2022</a>). For WB, supernatants and lysates of WT- and STOP-expressing PLC3 cells were heated for 20 min at 80°C in the presence of reducing Laemmli buffer. Samples were then separated by 10% SDS–PAGE and transferred onto nitrocellulose membranes (Hybond ECL). ORF2 proteins were detected with 1E6 antibody and peroxidase-conjugated anti-mouse antibodies. The detection of proteins was done by chemiluminescence analysis (ECL). For IP, P3H2 and P1H1 antibodies were bound to M-280 Dynabeads (Thermo Fisher Scientific) overnight at 37°C following the manufacturer’s recommendations. Beads were washed and then incubated for 1 h at room temperature with supernatants (heat-inactivated) or cell lysates. Beads were washed and then heated at 80°C for 20 min in reducing Laemmli buffer. ORF2 proteins were detected by WB using the 1E6 antibody.</p></section><section id="s19"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p>Statistical analysis was performed using R software environment for statistical computing and graphics (version 3.3.2). For quantifications by ELISA, RT–qPCR, and flow cytometry, analyses were performed using the Wilcoxon rank-sum exact test and <em>P</em>-value adjustment method: Bonferroni or FDR was applied when mentioned. The figures were prepared using PRISM software (version 10.2.1).</p></section></section><section id="sec20"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="d67e356"><div class="caption p"><span>Reviewer comments</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11966012/bin/LSA-2025-03256_review_history.pdf" data-ga-action="click_feat_suppl" class="usa-link">LSA-2025-03256_review_history.pdf</a><sup> (1.4MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We would like to thank ANRS-MIE for providing financial support (ANRS-MIE grant number: AAP2020-2/ECTZ133955) and “Contrats doctoraux Lyon 1 dédiés à l’International” from Université Lyon 1 for G Joshi’s PhD fellowship. This work was also supported by grants from the Agence Nationale de la Recherche (ANRJCJC-iSYN); the Agence Nationale pour la Recherche contre le SIDA et les Hépatites Virales (ANRS—N21006CR and N19017CR); and the UDL/ANR IA ELAN ERC (G19005CC) to M Dreux. We acknowledge Dr. Julie Lucifora, Dr. David Durantel, Dr. Elena Tomasello, Dr. Alexandre Belot, Dr. Søren Riis Paludan, Dr. Marco Binder, and Dr. Antoine Marçais for scientific discussions. We thank SFR Biosciences (UMS3444/CNRS, US8/Inserm, ENS de Lyon, UCBL) including the PLATIM and AniRA-cytometry facilities, for technical assistance in imaging and FACS analyses, respectively. Special thanks to Dr. Marion Delphin, Dr. Vladimir Goncalves Magalhaes, and Roxanne Fouille for their contribution in supporting experiments. The contribution of the EFS Confluence/Decine-Lyon is also noteworthy. Finally, we thank current and former members of the VIV team for helpful discussions and support: Dr. Margarida Sa Ribeiro, Célia Nuovo, Matteo Agostini, and Manon Venet.</p>
<section id="s20"><h3 class="pmc_sec_title">Author Contributions</h3>
<ul class="list" style="list-style-type:none">
<li><p>G Joshi: conceptualization, data curation, formal analysis, funding acquisition, investigation, methodology, and writing—original draft, review, and editing.</p></li>
<li><p>E Décembre: investigation and methodology.</p></li>
<li><p>J Brocard: formal analysis, validation, and methodology.</p></li>
<li><p>C Montpellier: resources, investigation, and methodology.</p></li>
<li><p>M Ferrié: resources, investigation, and methodology.</p></li>
<li><p>O Allatif: formal analysis and validation.</p></li>
<li><p>A-K Mehnert: investigation.</p></li>
<li><p>J Pons: investigation.</p></li>
<li><p>D Galiana: formal analysis, supervision, investigation, and writing—review and editing.</p></li>
<li><p>VL Dao Thi: resources, supervision, methodology, and writing—review and editing.</p></li>
<li><p>N Jouvenet: formal analysis, supervision, validation, and writing—review and editing.</p></li>
<li><p>L Cocquerel: conceptualization, resources, formal analysis, supervision, validation, methodology, and writing—review and editing.</p></li>
<li><p>M Dreux: conceptualization, supervision, funding acquisition, and writing—original draft, review, and editing.</p></li>
</ul></section><section id="s21"><h3 class="pmc_sec_title">Conflict of Interest Statement</h3>
<p>The authors declare that they have no conflict of interest.</p></section></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="bib1">
<cite>Andonov A, Robbins M, Borlang J, Cao J, Hatchette T, Stueck A, Deschambault Y, Murnaghan K, Varga J, Johnston L (2019) Rat hepatitis E virus linked to severe acute hepatitis in an immunocompetent patient. J Infect Dis
220: 951–955. 10.1093/infdis/jiz025
</cite> [<a href="https://doi.org/10.1093/infdis/jiz025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30649379/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Infect%20Dis&amp;title=Rat%20hepatitis%20E%20virus%20linked%20to%20severe%20acute%20hepatitis%20in%20an%20immunocompetent%20patient&amp;volume=220&amp;publication_year=2019&amp;pages=951-955&amp;pmid=30649379&amp;doi=10.1093/infdis/jiz025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib2">
<cite>Ankavay M, Montpellier C, Sayed IM, Saliou JM, Wychowski C, Saas L, Duvet S, Aliouat-Denis CM, Farhat R, de Masson d’Autume V, et al.  (2019) New insights into the ORF2 capsid protein, a key player of the hepatitis E virus lifecycle. Sci Rep
9: 6243. 10.1038/s41598-019-42737-2
</cite> [<a href="https://doi.org/10.1038/s41598-019-42737-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6472401/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31000788/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=New%20insights%20into%20the%20ORF2%20capsid%20protein,%20a%20key%20player%20of%20the%20hepatitis%20E%20virus%20lifecycle&amp;volume=9&amp;publication_year=2019&amp;pages=6243&amp;pmid=31000788&amp;doi=10.1038/s41598-019-42737-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib3">
<cite>Assil S, Coléon S, Dong C, Décembre E, Sherry L, Allatif O, Webster B, Dreux M (2019a) Plasmacytoid dendritic cells and infected cells form an interferogenic synapse required for antiviral responses. Cell Host Microbe
25: 730–745.e6. 10.1016/j.chom.2019.03.005
</cite> [<a href="https://doi.org/10.1016/j.chom.2019.03.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31003939/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Host%20Microbe&amp;title=Plasmacytoid%20dendritic%20cells%20and%20infected%20cells%20form%20an%20interferogenic%20synapse%20required%20for%20antiviral%20responses&amp;volume=25&amp;publication_year=2019a&amp;pages=730-745.e6&amp;pmid=31003939&amp;doi=10.1016/j.chom.2019.03.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib4">
<cite>Assil S, Futsch N, Décembre E, Alais S, Gessain A, Cosset FL, Mahieux R, Dreux M, Dutartre H (2019b) Sensing of cell-associated HTLV by plasmacytoid dendritic cells is regulated by dense β-galactoside glycosylation. PLoS Pathog
15: e1007589. 10.1371/journal.ppat.1007589
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1007589" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6413949/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30818370/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog&amp;title=Sensing%20of%20cell-associated%20HTLV%20by%20plasmacytoid%20dendritic%20cells%20is%20regulated%20by%20dense%20%CE%B2-galactoside%20glycosylation&amp;volume=15&amp;publication_year=2019b&amp;pages=e1007589&amp;pmid=30818370&amp;doi=10.1371/journal.ppat.1007589&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib5">
<cite>Bagdassarian E, Doceul V, Pellerin M, Demange A, Meyer L, Jouvenet N, Pavio N (2018) The amino-terminal region of hepatitis E virus ORF1 containing a methyltransferase (met) and a papain-like cysteine protease (PCP) domain counteracts type I interferon response. Viruses
10: 726. 10.3390/v10120726
</cite> [<a href="https://doi.org/10.3390/v10120726" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6315852/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30567349/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Viruses&amp;title=The%20amino-terminal%20region%20of%20hepatitis%20E%20virus%20ORF1%20containing%20a%20methyltransferase%20(met)%20and%20a%20papain-like%20cysteine%20protease%20(PCP)%20domain%20counteracts%20type%20I%20interferon%20response&amp;volume=10&amp;publication_year=2018&amp;pages=726&amp;pmid=30567349&amp;doi=10.3390/v10120726&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib6">
<cite>Behrendt P, Lüth S, Dammermann W, Drave S, Brown RJP, Todt D, Schnoor U, Steinmann E, Wedemeyer H, Pischke S, et al.  (2017) Exacerbation of hepatitis E virus infection during anti-TNFα treatment. Joint Bone Spine
84: 217–219. 10.1016/j.jbspin.2016.09.017
</cite> [<a href="https://doi.org/10.1016/j.jbspin.2016.09.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27836355/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Joint%20Bone%20Spine&amp;title=Exacerbation%20of%20hepatitis%20E%20virus%20infection%20during%20anti-TNF%CE%B1%20treatment&amp;volume=84&amp;publication_year=2017&amp;pages=217-219&amp;pmid=27836355&amp;doi=10.1016/j.jbspin.2016.09.017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib7">
<cite>Bentaleb C, Hervouet K, Montpellier C, Camuzet C, Ferrié M, Burlaud-Gaillard J, Bressanelli S, Metzger K, Werkmeister E, Ankavay M, et al.  (2022) The endocytic recycling compartment serves as a viral factory for hepatitis E virus. Cell Mol Life Sci
79: 615. 10.1007/s00018-022-04646-y
</cite> [<a href="https://doi.org/10.1007/s00018-022-04646-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9718719/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36460928/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Mol%20Life%20Sci&amp;title=The%20endocytic%20recycling%20compartment%20serves%20as%20a%20viral%20factory%20for%20hepatitis%20E%20virus&amp;volume=79&amp;publication_year=2022&amp;pages=615&amp;pmid=36460928&amp;doi=10.1007/s00018-022-04646-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib8">
<cite>Bermejo-Jambrina M, Eder J, Helgers LC, Hertoghs N, Nijmeijer BM, Stunnenberg M, Geijtenbeek TBH (2018) C-type Lectin receptors in antiviral immunity and viral escape. Front Immunol
9: 590. 10.3389/fimmu.2018.00590
</cite> [<a href="https://doi.org/10.3389/fimmu.2018.00590" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5879224/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29632536/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Immunol&amp;title=C-type%20Lectin%20receptors%20in%20antiviral%20immunity%20and%20viral%20escape&amp;volume=9&amp;publication_year=2018&amp;pages=590&amp;pmid=29632536&amp;doi=10.3389/fimmu.2018.00590&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib9">
<cite>Blight KJ, McKeating JA, Rice CM (2002) Highly permissive cell lines for subgenomic and genomic hepatitis C virus RNA replication. J Virol
76: 13001–13014. 10.1128/JVI.76.24.13001-13014.2002
</cite> [<a href="https://doi.org/10.1128/JVI.76.24.13001-13014.2002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC136668/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12438626/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Virol&amp;title=Highly%20permissive%20cell%20lines%20for%20subgenomic%20and%20genomic%20hepatitis%20C%20virus%20RNA%20replication&amp;volume=76&amp;publication_year=2002&amp;pages=13001-13014&amp;pmid=12438626&amp;doi=10.1128/JVI.76.24.13001-13014.2002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib10">
<cite>Bourdon M, Manet C, Montagutelli X (2020) Host genetic susceptibility to viral infections: The role of type I interferon induction. Genes Immun
21: 365–379. 10.1038/s41435-020-00116-2
</cite> [<a href="https://doi.org/10.1038/s41435-020-00116-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7677911/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33219336/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Immun&amp;title=Host%20genetic%20susceptibility%20to%20viral%20infections:%20The%20role%20of%20type%20I%20interferon%20induction&amp;volume=21&amp;publication_year=2020&amp;pages=365-379&amp;pmid=33219336&amp;doi=10.1038/s41435-020-00116-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib11">
<cite>Broggi A, Granucci F, Zanoni I (2020) Type III interferons: Balancing tissue tolerance and resistance to pathogen invasion. J Exp Med
217: e20190295. 10.1084/jem.20190295
</cite> [<a href="https://doi.org/10.1084/jem.20190295" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7037241/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31821443/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Exp%20Med&amp;title=Type%20III%20interferons:%20Balancing%20tissue%20tolerance%20and%20resistance%20to%20pathogen%20invasion&amp;volume=217&amp;publication_year=2020&amp;pages=e20190295&amp;pmid=31821443&amp;doi=10.1084/jem.20190295&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib12">
<cite>Brownell J, Bruckner J, Wagoner J, Thomas E, Loo Y-M, Gale M, Liang TJ, Polyak SJ (2014) Direct, interferon-independent activation of the CXCL10 promoter by NF-κB and interferon regulatory factor 3 during hepatitis C virus infection. J Virol
88: 1582–1590. 10.1128/JVI.02007-13
</cite> [<a href="https://doi.org/10.1128/JVI.02007-13" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3911583/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24257594/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Virol&amp;title=Direct,%20interferon-independent%20activation%20of%20the%20CXCL10%20promoter%20by%20NF-%CE%BAB%20and%20interferon%20regulatory%20factor%203%20during%20hepatitis%20C%20virus%20infection&amp;volume=88&amp;publication_year=2014&amp;pages=1582-1590&amp;pmid=24257594&amp;doi=10.1128/JVI.02007-13&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib13">
<cite>Cambi A, Koopman M, Figdor CG (2005) How C-type lectins detect pathogens: C-Type lectins and pathogens. Cell Microbiol
7: 481–488. 10.1111/j.1462-5822.2005.00506.x
</cite> [<a href="https://doi.org/10.1111/j.1462-5822.2005.00506.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15760448/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Microbiol&amp;title=How%20C-type%20lectins%20detect%20pathogens:%20C-Type%20lectins%20and%20pathogens&amp;volume=7&amp;publication_year=2005&amp;pages=481-488&amp;pmid=15760448&amp;doi=10.1111/j.1462-5822.2005.00506.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib14">
<cite>Cervantes-Barragan L, Lewis KL, Firner S, Thiel V, Hugues S, Reith W, Ludewig B, Reizis B (2012) Plasmacytoid dendritic cells control T-cell response to chronic viral infection. Proc Natl Acad Sci U S A
109: 3012–3017. 10.1073/pnas.1117359109
</cite> [<a href="https://doi.org/10.1073/pnas.1117359109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3286988/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22315415/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Plasmacytoid%20dendritic%20cells%20control%20T-cell%20response%20to%20chronic%20viral%20infection&amp;volume=109&amp;publication_year=2012&amp;pages=3012-3017&amp;pmid=22315415&amp;doi=10.1073/pnas.1117359109&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib15">
<cite>Crouse J, Kalinke U, Oxenius A (2015) Regulation of antiviral T cell responses by type I interferons. Nat Rev Immunol
15: 231–242. 10.1038/nri3806
</cite> [<a href="https://doi.org/10.1038/nri3806" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25790790/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Immunol&amp;title=Regulation%20of%20antiviral%20T%20cell%20responses%20by%20type%20I%20interferons&amp;volume=15&amp;publication_year=2015&amp;pages=231-242&amp;pmid=25790790&amp;doi=10.1038/nri3806&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib84">
<cite>Décembre E, Assil S, Hillaire MLB, Dejnirattisai W, Mongkolsapaya J, Screaton GR, Davidson AD, Dreux M (2014) Sensing of immature particles produced by dengue virus infected cells induces an antiviral response by plasmacytoid dendritic cells. PLoS Pathog
10: e1004434. 10.1371/journal.ppat.1004434
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1004434" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4207819/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25340500/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog&amp;title=Sensing%20of%20immature%20particles%20produced%20by%20dengue%20virus%20infected%20cells%20induces%20an%20antiviral%20response%20by%20plasmacytoid%20dendritic%20cells&amp;volume=10&amp;publication_year=2014&amp;pages=e1004434&amp;pmid=25340500&amp;doi=10.1371/journal.ppat.1004434&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib16">
<cite>Devhare P, Madiyal M, Mukhopadhyay C, Shetty S, Shastry S (2021) Interplay between hepatitis E virus and host cell pattern recognition receptors. Int J Mol Sci
22: 9259. 10.3390/ijms22179259
</cite> [<a href="https://doi.org/10.3390/ijms22179259" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8431321/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34502167/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&amp;title=Interplay%20between%20hepatitis%20E%20virus%20and%20host%20cell%20pattern%20recognition%20receptors&amp;volume=22&amp;publication_year=2021&amp;pages=9259&amp;pmid=34502167&amp;doi=10.3390/ijms22179259&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib17">
<cite>Di Domizio J, Belkhodja C, Chenuet P, Fries A, Murray T, Mondéjar PM, Demaria O, Conrad C, Homey B, Werner S, et al.  (2020) The commensal skin microbiota triggers type I IFN–dependent innate repair responses in injured skin. Nat Immunol
21: 1034–1045. 10.1038/s41590-020-0721-6
</cite> [<a href="https://doi.org/10.1038/s41590-020-0721-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32661363/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Immunol&amp;title=The%20commensal%20skin%20microbiota%20triggers%20type%20I%20IFN%E2%80%93dependent%20innate%20repair%20responses%20in%20injured%20skin&amp;volume=21&amp;publication_year=2020&amp;pages=1034-1045&amp;pmid=32661363&amp;doi=10.1038/s41590-020-0721-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib18">
<cite>Doceul V, Bagdassarian E, Demange A, Pavio N (2016) Zoonotic hepatitis E virus: Classification, animal reservoirs and transmission routes. Viruses
8: 270. 10.3390/v8100270
</cite> [<a href="https://doi.org/10.3390/v8100270" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5086606/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27706110/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Viruses&amp;title=Zoonotic%20hepatitis%20E%20virus:%20Classification,%20animal%20reservoirs%20and%20transmission%20routes&amp;volume=8&amp;publication_year=2016&amp;pages=270&amp;pmid=27706110&amp;doi=10.3390/v8100270&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib19">
<cite>Dong C, Zafrullah M, Mixson-Hayden T, Dai X, Liang J, Meng J, Kamili S (2012) Suppression of interferon-α signaling by hepatitis E virus. Hepatology
55: 1324–1332. 10.1002/hep.25530
</cite> [<a href="https://doi.org/10.1002/hep.25530" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22183878/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Hepatology&amp;title=Suppression%20of%20interferon-%CE%B1%20signaling%20by%20hepatitis%20E%20virus&amp;volume=55&amp;publication_year=2012&amp;pages=1324-1332&amp;pmid=22183878&amp;doi=10.1002/hep.25530&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib20">
<cite>Doyle EH, Rahman A, Aloman C, Klepper AL, El-Shamy A, Eng F, Rocha C, Kim S, Haydel B, Florman SS, et al.  (2019) Individual liver plasmacytoid dendritic cells are capable of producing IFNα and multiple additional cytokines during chronic HCV infection. PLoS Pathog
15: e1007935. 10.1371/journal.ppat.1007935
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1007935" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6687199/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31356648/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog&amp;title=Individual%20liver%20plasmacytoid%20dendritic%20cells%20are%20capable%20of%20producing%20IFN%CE%B1%20and%20multiple%20additional%20cytokines%20during%20chronic%20HCV%20infection&amp;volume=15&amp;publication_year=2019&amp;pages=e1007935&amp;pmid=31356648&amp;doi=10.1371/journal.ppat.1007935&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib21">
<cite>Dreux M, Garaigorta U, Boyd B, Décembre E, Chung J, Whitten-Bauer C, Wieland S, Chisari FV (2012) Short-range exosomal transfer of viral RNA from infected cells to plasmacytoid dendritic cells triggers innate immunity. Cell Host Microbe
12: 558–570. 10.1016/j.chom.2012.08.010
</cite> [<a href="https://doi.org/10.1016/j.chom.2012.08.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3479672/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23084922/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Host%20Microbe&amp;title=Short-range%20exosomal%20transfer%20of%20viral%20RNA%20from%20infected%20cells%20to%20plasmacytoid%20dendritic%20cells%20triggers%20innate%20immunity&amp;volume=12&amp;publication_year=2012&amp;pages=558-570&amp;pmid=23084922&amp;doi=10.1016/j.chom.2012.08.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib22">
<cite>Emerson SU, Nguyen HT, Torian U, Mather K, Firth AE (2013) An essential RNA element resides in a central region of hepatitis E virus ORF2. J Gen Virol
94: 1468–1476. 10.1099/vir.0.051870-0
</cite> [<a href="https://doi.org/10.1099/vir.0.051870-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3709636/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23515023/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Gen%20Virol&amp;title=An%20essential%20RNA%20element%20resides%20in%20a%20central%20region%20of%20hepatitis%20E%20virus%20ORF2&amp;volume=94&amp;publication_year=2013&amp;pages=1468-1476&amp;pmid=23515023&amp;doi=10.1099/vir.0.051870-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib23">
<cite>Florentin J, Aouar B, Dental C, Thumann C, Firaguay G, Gondois-Rey F, Soumelis V, Baumert TF, Nunès JA, Olive D, et al.  (2012) HCV glycoprotein E2 is a novel BDCA-2 ligand and acts as an inhibitor of IFN production by plasmacytoid dendritic cells. Blood
120: 4544–4551. 10.1182/blood-2012-02-413286
</cite> [<a href="https://doi.org/10.1182/blood-2012-02-413286" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23053572/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=HCV%20glycoprotein%20E2%20is%20a%20novel%20BDCA-2%20ligand%20and%20acts%20as%20an%20inhibitor%20of%20IFN%20production%20by%20plasmacytoid%20dendritic%20cells&amp;volume=120&amp;publication_year=2012&amp;pages=4544-4551&amp;pmid=23053572&amp;doi=10.1182/blood-2012-02-413286&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib24">
<cite>Gibson SJ, Lindh JM, Riter TR, Gleason RM, Rogers LM, Fuller AE, Oesterich JL, Gorden KB, Qiu X, McKane SW, et al.  (2002) Plasmacytoid dendritic cells produce cytokines and mature in response to the TLR7 agonists, imiquimod and resiquimod. Cell Immunol
218: 74–86. 10.1016/S0008-8749(02)00517-8
</cite> [<a href="https://doi.org/10.1016/S0008-8749(02)00517-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12470615/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Immunol&amp;title=Plasmacytoid%20dendritic%20cells%20produce%20cytokines%20and%20mature%20in%20response%20to%20the%20TLR7%20agonists,%20imiquimod%20and%20resiquimod&amp;volume=218&amp;publication_year=2002&amp;pages=74-86&amp;pmid=12470615&amp;doi=10.1016/S0008-8749(02)00517-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib25">
<cite>Gilliet M, Cao W, Liu Y-J (2008) Plasmacytoid dendritic cells: Sensing nucleic acids in viral infection and autoimmune diseases. Nat Rev Immunol
8: 594–606. 10.1038/nri2358
</cite> [<a href="https://doi.org/10.1038/nri2358" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18641647/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Immunol&amp;title=Plasmacytoid%20dendritic%20cells:%20Sensing%20nucleic%20acids%20in%20viral%20infection%20and%20autoimmune%20diseases&amp;volume=8&amp;publication_year=2008&amp;pages=594-606&amp;pmid=18641647&amp;doi=10.1038/nri2358&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib26">
<cite>Graff J, Torian U, Nguyen H, Emerson SU (2006) A bicistronic subgenomic mRNA encodes both the ORF2 and ORF3 proteins of hepatitis E virus. J Virol
80: 5919–5926. 10.1128/JVI.00046-06
</cite> [<a href="https://doi.org/10.1128/JVI.00046-06" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1472559/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16731930/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Virol&amp;title=A%20bicistronic%20subgenomic%20mRNA%20encodes%20both%20the%20ORF2%20and%20ORF3%20proteins%20of%20hepatitis%20E%20virus&amp;volume=80&amp;publication_year=2006&amp;pages=5919-5926&amp;pmid=16731930&amp;doi=10.1128/JVI.00046-06&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib27">
<cite>Grange ZL, Goldstein T, Johnson CK, Anthony S, Gilardi K, Daszak P, Olival KJ, O’Rourke T, Murray S, Olson SH, et al.  (2021) Ranking the risk of animal-to-human spillover for newly discovered viruses. Proc Natl Acad Sci U S A
118: e2002324118. 10.1073/pnas.2002324118
</cite> [<a href="https://doi.org/10.1073/pnas.2002324118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8053939/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33822740/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Ranking%20the%20risk%20of%20animal-to-human%20spillover%20for%20newly%20discovered%20viruses&amp;volume=118&amp;publication_year=2021&amp;pages=e2002324118&amp;pmid=33822740&amp;doi=10.1073/pnas.2002324118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib28">
<cite>Gu J, Isaji T, Xu Q, Kariya Y, Gu W, Fukuda T, Du Y (2012) Potential roles of N-glycosylation in cell adhesion. Glycoconj J
29: 599–607. 10.1007/s10719-012-9386-1
</cite> [<a href="https://doi.org/10.1007/s10719-012-9386-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22565826/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Glycoconj%20J&amp;title=Potential%20roles%20of%20N-glycosylation%20in%20cell%20adhesion&amp;volume=29&amp;publication_year=2012&amp;pages=599-607&amp;pmid=22565826&amp;doi=10.1007/s10719-012-9386-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib29">
<cite>Hervouet K, Ferrié M, Ankavay M, Montpellier C, Camuzet C, Alexandre V, Dembélé A, Lecoeur C, Foe AT, Bouquet P, et al.  (2022) An Arginine-Rich Motif in the ORF2 capsid protein regulates the hepatitis E virus lifecycle and interactions with the host cell. PLoS Pathog
18: e1010798. 10.1371/journal.ppat.1010798
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1010798" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9451086/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36007070/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog&amp;title=An%20Arginine-Rich%20Motif%20in%20the%20ORF2%20capsid%20protein%20regulates%20the%20hepatitis%20E%20virus%20lifecycle%20and%20interactions%20with%20the%20host%20cell&amp;volume=18&amp;publication_year=2022&amp;pages=e1010798&amp;pmid=36007070&amp;doi=10.1371/journal.ppat.1010798&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib30">
<cite>Kim H, Hwang J-S, Woo C-H, Kim E-Y, Kim T-H, Cho K-J, Kim JH, Seo JM, Lee SS (2008) TNF-alpha-induced up-regulation of intercellular adhesion molecule-1 is regulated by a Rac-ROS-dependent cascade in human airway epithelial cells. Exp Mol Med
40: 167–175. 10.3858/emm.2008.40.2.167
</cite> [<a href="https://doi.org/10.3858/emm.2008.40.2.167" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2679306/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18446055/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Exp%20Mol%20Med&amp;title=TNF-alpha-induced%20up-regulation%20of%20intercellular%20adhesion%20molecule-1%20is%20regulated%20by%20a%20Rac-ROS-dependent%20cascade%20in%20human%20airway%20epithelial%20cells&amp;volume=40&amp;publication_year=2008&amp;pages=167-175&amp;pmid=18446055&amp;doi=10.3858/emm.2008.40.2.167&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib31">
<cite>Knowles BB, Howe CC, Aden DP (1980) Human hepatocellular carcinoma cell lines secrete the major plasma proteins and hepatitis B surface antigen. Science
209: 497–499. 10.1126/science.6248960
</cite> [<a href="https://doi.org/10.1126/science.6248960" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6248960/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Human%20hepatocellular%20carcinoma%20cell%20lines%20secrete%20the%20major%20plasma%20proteins%20and%20hepatitis%20B%20surface%20antigen&amp;volume=209&amp;publication_year=1980&amp;pages=497-499&amp;pmid=6248960&amp;doi=10.1126/science.6248960&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib32">
<cite>Koonin EV, Gorbalenya AE, Purdy MA, Rozanov MN, Reyes GR, Bradley DW (1992) Computer-assisted assignment of functional domains in the nonstructural polyprotein of hepatitis E virus: Delineation of an additional group of positive-strand RNA plant and animal viruses. Proc Natl Acad Sci U S A
89: 8259–8263. 10.1073/pnas.89.17.8259
</cite> [<a href="https://doi.org/10.1073/pnas.89.17.8259" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC49897/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1518855/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Computer-assisted%20assignment%20of%20functional%20domains%20in%20the%20nonstructural%20polyprotein%20of%20hepatitis%20E%20virus:%20Delineation%20of%20an%20additional%20group%20of%20positive-strand%20RNA%20plant%20and%20animal%20viruses&amp;volume=89&amp;publication_year=1992&amp;pages=8259-8263&amp;pmid=1518855&amp;doi=10.1073/pnas.89.17.8259&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib33">
<cite>Lazear HM, Schoggins JW, Diamond MS (2019) Shared and distinct functions of type I and type III interferons. Immunity
50: 907–923. 10.1016/j.immuni.2019.03.025
</cite> [<a href="https://doi.org/10.1016/j.immuni.2019.03.025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6839410/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30995506/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Shared%20and%20distinct%20functions%20of%20type%20I%20and%20type%20III%20interferons&amp;volume=50&amp;publication_year=2019&amp;pages=907-923&amp;pmid=30995506&amp;doi=10.1016/j.immuni.2019.03.025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib34">
<cite>Lee G-H, Tan B-H, Chi-Yuan Teo E, Lim S-G, Dan Y-Y, Wee A, Kim Aw PP, Zhu Y, Hibberd ML, Tan C-K, et al.  (2016) Chronic infection with camelid hepatitis E virus in a liver transplant recipient who regularly consumes camel meat and milk. Gastroenterology
150: 355–357.e3. 10.1053/j.gastro.2015.10.048
</cite> [<a href="https://doi.org/10.1053/j.gastro.2015.10.048" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26551551/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=Chronic%20infection%20with%20camelid%20hepatitis%20E%20virus%20in%20a%20liver%20transplant%20recipient%20who%20regularly%20consumes%20camel%20meat%20and%20milk&amp;volume=150&amp;publication_year=2016&amp;pages=355-357.e3&amp;pmid=26551551&amp;doi=10.1053/j.gastro.2015.10.048&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib35">
<cite>Lei Q, Li L, Zhang S, Li T, Zhang X, Ding X, Qin B (2018) HEV ORF3 downregulates TLR7 to inhibit the generation of type I interferon via impairment of multiple signaling pathways. Sci Rep
8: 8585. 10.1038/s41598-018-26975-4
</cite> [<a href="https://doi.org/10.1038/s41598-018-26975-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5988675/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29872132/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=HEV%20ORF3%20downregulates%20TLR7%20to%20inhibit%20the%20generation%20of%20type%20I%20interferon%20via%20impairment%20of%20multiple%20signaling%20pathways&amp;volume=8&amp;publication_year=2018&amp;pages=8585&amp;pmid=29872132&amp;doi=10.1038/s41598-018-26975-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib36">
<cite>Lenggenhager D, Gouttenoire J, Malehmir M, Bawohl M, Honcharova-Biletska H, Kreutzer S, Semela D, Neuweiler J, Hürlimann S, Aepli P, et al.  (2017) Visualization of hepatitis E virus RNA and proteins in the human liver. J Hepatol
67: 471–479. 10.1016/j.jhep.2017.04.002
</cite> [<a href="https://doi.org/10.1016/j.jhep.2017.04.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28412294/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&amp;title=Visualization%20of%20hepatitis%20E%20virus%20RNA%20and%20proteins%20in%20the%20human%20liver&amp;volume=67&amp;publication_year=2017&amp;pages=471-479&amp;pmid=28412294&amp;doi=10.1016/j.jhep.2017.04.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib37">
<cite>Lhomme S, Marion O, Abravanel F, Izopet J, Kamar N (2020) Clinical manifestations, pathogenesis and treatment of hepatitis E virus infections. J Clin Med
9: 331. 10.3390/jcm9020331
</cite> [<a href="https://doi.org/10.3390/jcm9020331" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7073673/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31991629/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Med&amp;title=Clinical%20manifestations,%20pathogenesis%20and%20treatment%20of%20hepatitis%20E%20virus%20infections&amp;volume=9&amp;publication_year=2020&amp;pages=331&amp;pmid=31991629&amp;doi=10.3390/jcm9020331&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib38">
<cite>Li P, Liu J, Li Y, Su J, Ma Z, Bramer WM, Cao W, De Man RA, Peppelenbosch MP, Pan Q (2020) The global epidemiology of hepatitis E virus infection: A systematic review and meta-analysis. Liver Int
40: 1516–1528. 10.1111/liv.14468
</cite> [<a href="https://doi.org/10.1111/liv.14468" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7384095/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32281721/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Liver%20Int&amp;title=The%20global%20epidemiology%20of%20hepatitis%20E%20virus%20infection:%20A%20systematic%20review%20and%20meta-analysis&amp;volume=40&amp;publication_year=2020&amp;pages=1516-1528&amp;pmid=32281721&amp;doi=10.1111/liv.14468&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib39">
<cite>Lin S, Yang Y, Nan Y, Ma Z, Yang L, Zhang Y-J (2019) The capsid protein of hepatitis E virus inhibits interferon induction via its N-terminal arginine-rich motif. Viruses
11: 1050. 10.3390/v11111050
</cite> [<a href="https://doi.org/10.3390/v11111050" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6928999/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31717991/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Viruses&amp;title=The%20capsid%20protein%20of%20hepatitis%20E%20virus%20inhibits%20interferon%20induction%20via%20its%20N-terminal%20arginine-rich%20motif&amp;volume=11&amp;publication_year=2019&amp;pages=1050&amp;pmid=31717991&amp;doi=10.3390/v11111050&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib40">
<cite>Ma N, Lu J, Pei Y, Robertson ES (2022a) Transcriptome reprogramming of Epstein-Barr virus infected epithelial and B cells reveals distinct host-virus interaction profiles. Cell Death Dis
13: 894. 10.1038/s41419-022-05327-1
</cite> [<a href="https://doi.org/10.1038/s41419-022-05327-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9588026/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36272970/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Dis&amp;title=Transcriptome%20reprogramming%20of%20Epstein-Barr%20virus%20infected%20epithelial%20and%20B%20cells%20reveals%20distinct%20host-virus%20interaction%20profiles&amp;volume=13&amp;publication_year=2022a&amp;pages=894&amp;pmid=36272970&amp;doi=10.1038/s41419-022-05327-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib41">
<cite>Ma Z, De Man RA, Kamar N, Pan Q (2022b) Chronic hepatitis E: Advancing research and patient care. J Hepatol
77: 1109–1123. 10.1016/j.jhep.2022.05.006
</cite> [<a href="https://doi.org/10.1016/j.jhep.2022.05.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35605741/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&amp;title=Chronic%20hepatitis%20E:%20Advancing%20research%20and%20patient%20care&amp;volume=77&amp;publication_year=2022b&amp;pages=1109-1123&amp;pmid=35605741&amp;doi=10.1016/j.jhep.2022.05.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib42">
<cite>MacNab GM, Alexander JJ, Lecatsas G, Bey EM, Urbanowicz JM (1976) Hepatitis B surface antigen produced by a human hepatoma cell line. Br J Cancer
34: 509–515. 10.1038/bjc.1976.205
</cite> [<a href="https://doi.org/10.1038/bjc.1976.205" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2025201/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/187208/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&amp;title=Hepatitis%20B%20surface%20antigen%20produced%20by%20a%20human%20hepatoma%20cell%20line&amp;volume=34&amp;publication_year=1976&amp;pages=509-515&amp;pmid=187208&amp;doi=10.1038/bjc.1976.205&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib43">
<cite>Marlin SD, Springer TA (1987) Purified intercellular adhesion molecule-1 (ICAM-1) is a ligand for lymphocyte function-associated antigen 1 (LFA-1). Cell
51: 813–819. 10.1016/0092-8674(87)90104-8
</cite> [<a href="https://doi.org/10.1016/0092-8674(87)90104-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3315233/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Purified%20intercellular%20adhesion%20molecule-1%20(ICAM-1)%20is%20a%20ligand%20for%20lymphocyte%20function-associated%20antigen%201%20(LFA-1)&amp;volume=51&amp;publication_year=1987&amp;pages=813-819&amp;pmid=3315233&amp;doi=10.1016/0092-8674(87)90104-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib44">
<cite>Meyer-Wentrup F, Benitez-Ribas D, Tacken PJ, Punt CJA, Figdor CG, De Vries IJM, Adema GJ (2008) Targeting DCIR on human plasmacytoid dendritic cells results in antigen presentation and inhibits IFN-alpha production. Blood
111: 4245–4253. 10.1182/blood-2007-03-081398
</cite> [<a href="https://doi.org/10.1182/blood-2007-03-081398" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18258799/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Targeting%20DCIR%20on%20human%20plasmacytoid%20dendritic%20cells%20results%20in%20antigen%20presentation%20and%20inhibits%20IFN-alpha%20production&amp;volume=111&amp;publication_year=2008&amp;pages=4245-4253&amp;pmid=18258799&amp;doi=10.1182/blood-2007-03-081398&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib45">
<cite>Moal V, Textoris J, Ben Amara A, Mehraj V, Berland Y, Colson P, Mege J-L (2013) Chronic hepatitis E virus infection is specifically associated with an interferon-related transcriptional program. J Infect Dis
207: 125–132. 10.1093/infdis/jis632
</cite> [<a href="https://doi.org/10.1093/infdis/jis632" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23072754/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Infect%20Dis&amp;title=Chronic%20hepatitis%20E%20virus%20infection%20is%20specifically%20associated%20with%20an%20interferon-related%20transcriptional%20program&amp;volume=207&amp;publication_year=2013&amp;pages=125-132&amp;pmid=23072754&amp;doi=10.1093/infdis/jis632&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib46">
<cite>Montpellier C, Wychowski C, Sayed IM, Meunier J-C, Saliou J-M, Ankavay M, Bull A, Pillez A, Abravanel F, Helle F, et al.  (2018) Hepatitis E virus lifecycle and identification of 3 forms of the ORF2 capsid protein. Gastroenterology
154: 211–223.e8. 10.1053/j.gastro.2017.09.020
</cite> [<a href="https://doi.org/10.1053/j.gastro.2017.09.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28958858/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=Hepatitis%20E%20virus%20lifecycle%20and%20identification%20of%203%20forms%20of%20the%20ORF2%20capsid%20protein&amp;volume=154&amp;publication_year=2018&amp;pages=211-223.e8&amp;pmid=28958858&amp;doi=10.1053/j.gastro.2017.09.020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib47">
<cite>Murata K, Kang J-H, Nagashima S, Matsui T, Karino Y, Yamamoto Y, Atarashi T, Oohara M, Uebayashi M, Sakata H, et al.  (2020) IFN-λ3 as a host immune response in acute hepatitis E virus infection. Cytokine
125: 154816. 10.1016/j.cyto.2019.154816
</cite> [<a href="https://doi.org/10.1016/j.cyto.2019.154816" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31465972/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cytokine&amp;title=IFN-%CE%BB3%20as%20a%20host%20immune%20response%20in%20acute%20hepatitis%20E%20virus%20infection&amp;volume=125&amp;publication_year=2020&amp;pages=154816&amp;pmid=31465972&amp;doi=10.1016/j.cyto.2019.154816&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib48">
<cite>Nagashima S, Primadharsini PP, Nishiyama T, Takahashi M, Murata K, Okamoto H (2023) Development of a HiBiT-tagged reporter hepatitis E virus and its utility as an antiviral drug screening platform. J Virol
97: e0050823. 10.1128/jvi.00508-23
</cite> [<a href="https://doi.org/10.1128/jvi.00508-23" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10537679/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37681960/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Virol&amp;title=Development%20of%20a%20HiBiT-tagged%20reporter%20hepatitis%20E%20virus%20and%20its%20utility%20as%20an%20antiviral%20drug%20screening%20platform&amp;volume=97&amp;publication_year=2023&amp;pages=e0050823&amp;pmid=37681960&amp;doi=10.1128/jvi.00508-23&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib49">
<cite>Nan Y, Yu Y, Ma Z, Khattar SK, Fredericksen B, Zhang Y-J (2014) Hepatitis E virus inhibits type I interferon induction by ORF1 products. J Virol
88: 11924–11932. 10.1128/jvi.01935-14
</cite> [<a href="https://doi.org/10.1128/jvi.01935-14" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4178743/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25100852/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Virol&amp;title=Hepatitis%20E%20virus%20inhibits%20type%20I%20interferon%20induction%20by%20ORF1%20products&amp;volume=88&amp;publication_year=2014&amp;pages=11924-11932&amp;pmid=25100852&amp;doi=10.1128/jvi.01935-14&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib50">
<cite>Nimgaonkar I, Ding Q, Schwartz RE, Ploss A (2018) Hepatitis E virus: Advances and challenges. Nat Rev Gastroenterol Hepatol
15: 96–110. 10.1038/nrgastro.2017.150
</cite> [<a href="https://doi.org/10.1038/nrgastro.2017.150" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11329273/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29162935/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Gastroenterol%20Hepatol&amp;title=Hepatitis%20E%20virus:%20Advances%20and%20challenges&amp;volume=15&amp;publication_year=2018&amp;pages=96-110&amp;pmid=29162935&amp;doi=10.1038/nrgastro.2017.150&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib51">
<cite>Ohtsubo K, Marth JD (2006) Glycosylation in cellular mechanisms of health and disease. Cell
126: 855–867. 10.1016/j.cell.2006.08.019
</cite> [<a href="https://doi.org/10.1016/j.cell.2006.08.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16959566/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Glycosylation%20in%20cellular%20mechanisms%20of%20health%20and%20disease&amp;volume=126&amp;publication_year=2006&amp;pages=855-867&amp;pmid=16959566&amp;doi=10.1016/j.cell.2006.08.019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib52">
<cite>Parr MB, Parr EL (2000) Interferon-gamma up-regulates intercellular adhesion molecule-1 and vascular cell adhesion molecule-1 and recruits lymphocytes into the vagina of immune mice challenged with herpes simplex virus-2. Immunology
99: 540–545. 10.1046/j.1365-2567.2000.00980.x
</cite> [<a href="https://doi.org/10.1046/j.1365-2567.2000.00980.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2327183/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10792501/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunology&amp;title=Interferon-gamma%20up-regulates%20intercellular%20adhesion%20molecule-1%20and%20vascular%20cell%20adhesion%20molecule-1%20and%20recruits%20lymphocytes%20into%20the%20vagina%20of%20immune%20mice%20challenged%20with%20herpes%20simplex%20virus-2&amp;volume=99&amp;publication_year=2000&amp;pages=540-545&amp;pmid=10792501&amp;doi=10.1046/j.1365-2567.2000.00980.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib53">
<cite>Ralfs P, Holland B, Salinas E, Bremer B, Wang M, Zhu J, Ambardekar C, Blasczyk H, Walker CM, Feng Z, et al.  (2023) Soluble ORF2 protein enhances HEV replication and induces long-lasting antibody response and protective immunity in vivo. Hepatology
78: 1867–1881. 10.1097/HEP.0000000000000421
</cite> [<a href="https://doi.org/10.1097/HEP.0000000000000421" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10603212/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37185268/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Hepatology&amp;title=Soluble%20ORF2%20protein%20enhances%20HEV%20replication%20and%20induces%20long-lasting%20antibody%20response%20and%20protective%20immunity%20in%20vivo&amp;volume=78&amp;publication_year=2023&amp;pages=1867-1881&amp;pmid=37185268&amp;doi=10.1097/HEP.0000000000000421&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib54">
<cite>Reglero-Real N, Álvarez-Varela A, Cernuda-Morollón E, Feito J, Marcos-Ramiro B, Fernández-Martín L, Gómez-Lechón MJ, Muntané J, Sandoval P, Majano PL, et al.  (2014) Apicobasal polarity controls lymphocyte adhesion to hepatic epithelial cells. Cell Rep
8: 1879–1893. 10.1016/j.celrep.2014.08.007
</cite> [<a href="https://doi.org/10.1016/j.celrep.2014.08.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25242329/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&amp;title=Apicobasal%20polarity%20controls%20lymphocyte%20adhesion%20to%20hepatic%20epithelial%20cells&amp;volume=8&amp;publication_year=2014&amp;pages=1879-1893&amp;pmid=25242329&amp;doi=10.1016/j.celrep.2014.08.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib55">
<cite>Rehwinkel J, Gack MU (2020) RIG-I-like receptors: Their regulation and roles in RNA sensing. Nat Rev Immunol
20: 537–551. 10.1038/s41577-020-0288-3
</cite> [<a href="https://doi.org/10.1038/s41577-020-0288-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7094958/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32203325/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Immunol&amp;title=RIG-I-like%20receptors:%20Their%20regulation%20and%20roles%20in%20RNA%20sensing&amp;volume=20&amp;publication_year=2020&amp;pages=537-551&amp;pmid=32203325&amp;doi=10.1038/s41577-020-0288-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib56">
<cite>Reizis B (2019) Plasmacytoid dendritic cells: Development, regulation, and function. Immunity
50: 37–50. 10.1016/j.immuni.2018.12.027
</cite> [<a href="https://doi.org/10.1016/j.immuni.2018.12.027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6342491/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30650380/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Plasmacytoid%20dendritic%20cells:%20Development,%20regulation,%20and%20function&amp;volume=50&amp;publication_year=2019&amp;pages=37-50&amp;pmid=30650380&amp;doi=10.1016/j.immuni.2018.12.027&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib57">
<cite>Sari G, Mulders CE, Zhu J, Van Oord GW, Feng Z, Kreeft-Voermans JJC, Boonstra A, Vanwolleghem T (2021) Treatment induced clearance of hepatitis E viruses by interferon-lambda in liver-humanized mice. Liver Int
41: 2866–2873. 10.1111/liv.15033
</cite> [<a href="https://doi.org/10.1111/liv.15033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9291846/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34392598/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Liver%20Int&amp;title=Treatment%20induced%20clearance%20of%20hepatitis%20E%20viruses%20by%20interferon-lambda%20in%20liver-humanized%20mice&amp;volume=41&amp;publication_year=2021&amp;pages=2866-2873&amp;pmid=34392598&amp;doi=10.1111/liv.15033&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib58">
<cite>Sayed IM, Verhoye L, Cocquerel L, Abravanel F, Foquet L, Montpellier C, Debing Y, Farhoudi A, Wychowski C, Dubuisson J, et al.  (2017) Study of hepatitis E virus infection of genotype 1 and 3 in mice with humanised liver. Gut
66: 920–929. 10.1136/gutjnl-2015-311109
</cite> [<a href="https://doi.org/10.1136/gutjnl-2015-311109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27006186/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=Study%20of%20hepatitis%20E%20virus%20infection%20of%20genotype%201%20and%203%20in%20mice%20with%20humanised%20liver&amp;volume=66&amp;publication_year=2017&amp;pages=920-929&amp;pmid=27006186&amp;doi=10.1136/gutjnl-2015-311109&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib59">
<cite>Schemmerer M, Apelt S, Trojnar E, Ulrich RG, Wenzel JJ, Johne R (2016) Enhanced replication of hepatitis E virus strain 47832C in an A549-derived subclonal cell line. Viruses
8: 267. 10.3390/v8100267
</cite> [<a href="https://doi.org/10.3390/v8100267" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5086603/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27690085/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Viruses&amp;title=Enhanced%20replication%20of%20hepatitis%20E%20virus%20strain%2047832C%20in%20an%20A549-derived%20subclonal%20cell%20line&amp;volume=8&amp;publication_year=2016&amp;pages=267&amp;pmid=27690085&amp;doi=10.3390/v8100267&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib60">
<cite>Schneider WM, Chevillotte MD, Rice CM (2014) Interferon-stimulated genes: A complex web of host defenses. Annu Rev Immunol
32: 513–545. 10.1146/annurev-immunol-032713-120231
</cite> [<a href="https://doi.org/10.1146/annurev-immunol-032713-120231" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4313732/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24555472/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Immunol&amp;title=Interferon-stimulated%20genes:%20A%20complex%20web%20of%20host%20defenses&amp;volume=32&amp;publication_year=2014&amp;pages=513-545&amp;pmid=24555472&amp;doi=10.1146/annurev-immunol-032713-120231&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib61">
<cite>Sedger LM, McDermott MF (2014) TNF and TNF-receptors: From mediators of cell death and inflammation to therapeutic giants – past, present and future. Cytokine Growth Factor Rev
25: 453–472. 10.1016/j.cytogfr.2014.07.016
</cite> [<a href="https://doi.org/10.1016/j.cytogfr.2014.07.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25169849/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cytokine%20Growth%20Factor%20Rev&amp;title=TNF%20and%20TNF-receptors:%20From%20mediators%20of%20cell%20death%20and%20inflammation%20to%20therapeutic%20giants%20%E2%80%93%20past,%20present%20and%20future&amp;volume=25&amp;publication_year=2014&amp;pages=453-472&amp;pmid=25169849&amp;doi=10.1016/j.cytogfr.2014.07.016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib62">
<cite>Shukla P, Nguyen HT, Torian U, Engle RE, Faulk K, Dalton HR, Bendall RP, Keane FE, Purcell RH, Emerson SU (2011) Cross-species infections of cultured cells by hepatitis E virus and discovery of an infectious virus-host recombinant. Proc Natl Acad Sci U S A
108: 2438–2443. 10.1073/pnas.1018878108
</cite> [<a href="https://doi.org/10.1073/pnas.1018878108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3038723/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21262830/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Cross-species%20infections%20of%20cultured%20cells%20by%20hepatitis%20E%20virus%20and%20discovery%20of%20an%20infectious%20virus-host%20recombinant&amp;volume=108&amp;publication_year=2011&amp;pages=2438-2443&amp;pmid=21262830&amp;doi=10.1073/pnas.1018878108&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib63">
<cite>Shukla P, Nguyen HT, Faulk K, Mather K, Torian U, Engle RE, Emerson SU (2012) Adaptation of a genotype 3 hepatitis E virus to efficient growth in cell culture depends on an inserted human gene segment acquired by recombination. J Virol
86: 5697–5707. 10.1128/JVI.00146-12
</cite> [<a href="https://doi.org/10.1128/JVI.00146-12" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3347312/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22398290/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Virol&amp;title=Adaptation%20of%20a%20genotype%203%20hepatitis%20E%20virus%20to%20efficient%20growth%20in%20cell%20culture%20depends%20on%20an%20inserted%20human%20gene%20segment%20acquired%20by%20recombination&amp;volume=86&amp;publication_year=2012&amp;pages=5697-5707&amp;pmid=22398290&amp;doi=10.1128/JVI.00146-12&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib64">
<cite>Silvin A, Yu CI, Lahaye X, Imperatore F, Brault J-B, Cardinaud S, Becker C, Kwan W-H, Conrad C, Maurin M, et al.  (2017) Constitutive resistance to viral infection in human CD141<sup>+</sup> dendritic cells. Sci Immunol
2: eaai8071. 10.1126/sciimmunol.aai8071
</cite> [<a href="https://doi.org/10.1126/sciimmunol.aai8071" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5749640/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28783704/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Immunol&amp;title=Constitutive%20resistance%20to%20viral%20infection%20in%20human%20CD141+%20dendritic%20cells&amp;volume=2&amp;publication_year=2017&amp;pages=eaai8071&amp;pmid=28783704&amp;doi=10.1126/sciimmunol.aai8071&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib65">
<cite>Songtanin B, Molehin AJ, Brittan K, Manatsathit W, Nugent K (2023) Hepatitis E virus infections: Epidemiology, genetic diversity, and clinical considerations. Viruses
15: 1389. 10.3390/v15061389
</cite> [<a href="https://doi.org/10.3390/v15061389" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10304420/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37376687/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Viruses&amp;title=Hepatitis%20E%20virus%20infections:%20Epidemiology,%20genetic%20diversity,%20and%20clinical%20considerations&amp;volume=15&amp;publication_year=2023&amp;pages=1389&amp;pmid=37376687&amp;doi=10.3390/v15061389&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib66">
<cite>Sridhar S, Yip CCY, Wu S, Cai J, Zhang AJ-X, Leung K-H, Chung TWH, Chan JFW, Chan W-M, Teng JLL, et al.  (2018) Rat hepatitis E virus as cause of persistent hepatitis after liver transplant. Emerg Infect Dis
24: 2241–2250. 10.3201/eid2412.180937
</cite> [<a href="https://doi.org/10.3201/eid2412.180937" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6256372/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30457530/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Emerg%20Infect%20Dis&amp;title=Rat%20hepatitis%20E%20virus%20as%20cause%20of%20persistent%20hepatitis%20after%20liver%20transplant&amp;volume=24&amp;publication_year=2018&amp;pages=2241-2250&amp;pmid=30457530&amp;doi=10.3201/eid2412.180937&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib67">
<cite>Sumpter R, Loo Y-M, Foy E, Li K, Yoneyama M, Fujita T, Lemon SM, Gale M (2005) Regulating intracellular antiviral defense and permissiveness to hepatitis C virus RNA replication through a cellular RNA helicase, RIG-I. J Virol
79: 2689–2699. 10.1128/JVI.79.5.2689-2699.2005
</cite> [<a href="https://doi.org/10.1128/JVI.79.5.2689-2699.2005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC548482/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15708988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Virol&amp;title=Regulating%20intracellular%20antiviral%20defense%20and%20permissiveness%20to%20hepatitis%20C%20virus%20RNA%20replication%20through%20a%20cellular%20RNA%20helicase,%20RIG-I&amp;volume=79&amp;publication_year=2005&amp;pages=2689-2699&amp;pmid=15708988&amp;doi=10.1128/JVI.79.5.2689-2699.2005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib68">
<cite>Swiecki M, Gilfillan S, Vermi W, Wang Y, Colonna M (2010) Plasmacytoid dendritic cell ablation impacts early interferon responses and antiviral NK and CD8+ T cell accrual. Immunity
33: 955–966. 10.1016/j.immuni.2010.11.020
</cite> [<a href="https://doi.org/10.1016/j.immuni.2010.11.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3588567/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21130004/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Plasmacytoid%20dendritic%20cell%20ablation%20impacts%20early%20interferon%20responses%20and%20antiviral%20NK%20and%20CD8+%20T%20cell%20accrual&amp;volume=33&amp;publication_year=2010&amp;pages=955-966&amp;pmid=21130004&amp;doi=10.1016/j.immuni.2010.11.020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib69">
<cite>Tam AW, Smith MM, Guerra ME, Huang C-C, Bradley DW, Fry KE, Reyes GR (1991) Hepatitis E virus (HEV): Molecular cloning and sequencing of the full-length viral genome. Virology
185: 120–131. 10.1016/0042-6822(91)90760-9
</cite> [<a href="https://doi.org/10.1016/0042-6822(91)90760-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7130833/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1926770/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Virology&amp;title=Hepatitis%20E%20virus%20(HEV):%20Molecular%20cloning%20and%20sequencing%20of%20the%20full-length%20viral%20genome&amp;volume=185&amp;publication_year=1991&amp;pages=120-131&amp;pmid=1926770&amp;doi=10.1016/0042-6822(91)90760-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib70">
<cite>Todt D, François C, Anggakusuma, Behrendt P, Engelmann M, Knegendorf L, Vieyres G, Wedemeyer H, Hartmann R, Pietschmann T, et al.  (2016) Antiviral activities of different interferon types and subtypes against hepatitis E virus replication. Antimicrob Agents Chemother
60: 2132–2139. 10.1128/AAC.02427-15
</cite> [<a href="https://doi.org/10.1128/AAC.02427-15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4808167/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26787701/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&amp;title=Antiviral%20activities%20of%20different%20interferon%20types%20and%20subtypes%20against%20hepatitis%20E%20virus%20replication&amp;volume=60&amp;publication_year=2016&amp;pages=2132-2139&amp;pmid=26787701&amp;doi=10.1128/AAC.02427-15&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib71">
<cite>Venet M, Ribeiro MS, Décembre E, Bellomo A, Joshi G, Nuovo C, Villard M, Cluet D, Perret M, Pescamona R, et al.  (2023) Severe COVID-19 patients have impaired plasmacytoid dendritic cell-mediated control of SARS-CoV-2. Nat Commun
14: 694. 10.1038/s41467-023-36140-9
</cite> [<a href="https://doi.org/10.1038/s41467-023-36140-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9907212/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36755036/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Severe%20COVID-19%20patients%20have%20impaired%20plasmacytoid%20dendritic%20cell-mediated%20control%20of%20SARS-CoV-2&amp;volume=14&amp;publication_year=2023&amp;pages=694&amp;pmid=36755036&amp;doi=10.1038/s41467-023-36140-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib72">
<cite>Wang W, Xu L, Brandsma JH, Wang Y, Hakim MS, Zhou X, Yin Y, Fuhler GM, Van Der Laan LJW, Van Der Woude CJ, et al.  (2016) Convergent transcription of interferon-stimulated genes by TNF-α and IFN-α augments antiviral activity against HCV and HEV. Sci Rep
6: 25482. 10.1038/srep25482
</cite> [<a href="https://doi.org/10.1038/srep25482" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4858707/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27150018/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Convergent%20transcription%20of%20interferon-stimulated%20genes%20by%20TNF-%CE%B1%20and%20IFN-%CE%B1%20augments%20antiviral%20activity%20against%20HCV%20and%20HEV&amp;volume=6&amp;publication_year=2016&amp;pages=25482&amp;pmid=27150018&amp;doi=10.1038/srep25482&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib73">
<cite>Webster B, Assil S, Dreux M (2016) Cell-cell sensing of viral infection by plasmacytoid dendritic cells. J Virol
90: 10050–10053. 10.1128/jvi.01692-16
</cite> [<a href="https://doi.org/10.1128/jvi.01692-16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5105643/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27605675/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Virol&amp;title=Cell-cell%20sensing%20of%20viral%20infection%20by%20plasmacytoid%20dendritic%20cells&amp;volume=90&amp;publication_year=2016&amp;pages=10050-10053&amp;pmid=27605675&amp;doi=10.1128/jvi.01692-16&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib74">
<cite>Webster B, Werneke SW, Zafirova B, This S, Coléon S, Décembre E, Paidassi H, Bouvier I, Joubert P-E, Duffy D, et al.  (2018) Plasmacytoid dendritic cells control dengue and Chikungunya virus infections via IRF7-regulated interferon responses. Elife
7: e34273. 10.7554/eLife.34273
</cite> [<a href="https://doi.org/10.7554/eLife.34273" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6008049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29914621/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Elife&amp;title=Plasmacytoid%20dendritic%20cells%20control%20dengue%20and%20Chikungunya%20virus%20infections%20via%20IRF7-regulated%20interferon%20responses&amp;volume=7&amp;publication_year=2018&amp;pages=e34273&amp;pmid=29914621&amp;doi=10.7554/eLife.34273&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib75">
<cite>Willemsen J, Wicht O, Wolanski JC, Baur N, Bastian S, Haas DA, Matula P, Knapp B, Meyniel-Schicklin L, Wang C, et al.  (2017) Phosphorylation-dependent feedback inhibition of RIG-I by DAPK1 identified by kinome-wide siRNA screening. Mol Cell
65: 403–415.e8. 10.1016/j.molcel.2016.12.021
</cite> [<a href="https://doi.org/10.1016/j.molcel.2016.12.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28132841/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Cell&amp;title=Phosphorylation-dependent%20feedback%20inhibition%20of%20RIG-I%20by%20DAPK1%20identified%20by%20kinome-wide%20siRNA%20screening&amp;volume=65&amp;publication_year=2017&amp;pages=403-415.e8&amp;pmid=28132841&amp;doi=10.1016/j.molcel.2016.12.021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib76">
<cite>Wu X, Dao Thi VL, Liu P, Takacs CN, Xiang K, Andrus L, Gouttenoire J, Moradpour D, Rice CM (2018) Pan-genotype hepatitis E virus replication in stem cell–derived hepatocellular systems. Gastroenterology
154: 663–674.e7. 10.1053/j.gastro.2017.10.041
</cite> [<a href="https://doi.org/10.1053/j.gastro.2017.10.041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5811326/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29277559/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=Pan-genotype%20hepatitis%20E%20virus%20replication%20in%20stem%20cell%E2%80%93derived%20hepatocellular%20systems&amp;volume=154&amp;publication_year=2018&amp;pages=663-674.e7&amp;pmid=29277559&amp;doi=10.1053/j.gastro.2017.10.041&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib77">
<cite>Yarilina A, Park-Min K-H, Antoniv T, Hu X, Ivashkiv LB (2008) TNF activates an IRF1-dependent autocrine loop leading to sustained expression of chemokines and STAT1-dependent type I interferon–response genes. Nat Immunol
9: 378–387. 10.1038/ni1576
</cite> [<a href="https://doi.org/10.1038/ni1576" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18345002/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Immunol&amp;title=TNF%20activates%20an%20IRF1-dependent%20autocrine%20loop%20leading%20to%20sustained%20expression%20of%20chemokines%20and%20STAT1-dependent%20type%20I%20interferon%E2%80%93response%20genes&amp;volume=9&amp;publication_year=2008&amp;pages=378-387&amp;pmid=18345002&amp;doi=10.1038/ni1576&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib78">
<cite>Yin X, Li X, Ambardekar C, Hu Z, Lhomme S, Feng Z (2017) Hepatitis E virus persists in the presence of a type III interferon response. PLoS Pathog
13: e1006417. 10.1371/journal.ppat.1006417
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1006417" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5466342/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28558073/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog&amp;title=Hepatitis%20E%20virus%20persists%20in%20the%20presence%20of%20a%20type%20III%20interferon%20response&amp;volume=13&amp;publication_year=2017&amp;pages=e1006417&amp;pmid=28558073&amp;doi=10.1371/journal.ppat.1006417&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib79">
<cite>Yin X, Ying D, Lhomme S, Tang Z, Walker CM, Xia N, Zheng Z, Feng Z (2018) Origin, antigenicity, and function of a secreted form of ORF2 in hepatitis E virus infection. Proc Natl Acad Sci U S A
115: 4773–4778. 10.1073/pnas.1721345115
</cite> [<a href="https://doi.org/10.1073/pnas.1721345115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5939091/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29669922/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Origin,%20antigenicity,%20and%20function%20of%20a%20secreted%20form%20of%20ORF2%20in%20hepatitis%20E%20virus%20infection&amp;volume=115&amp;publication_year=2018&amp;pages=4773-4778&amp;pmid=29669922&amp;doi=10.1073/pnas.1721345115&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib80">
<cite>Yu C, Boon D, McDonald SL, Myers TG, Tomioka K, Nguyen H, Engle RE, Govindarajan S, Emerson SU, Purcell RH (2010) Pathogenesis of hepatitis E virus and hepatitis C virus in chimpanzees: Similarities and differences. J Virol
84: 11264–11278. 10.1128/JVI.01205-10
</cite> [<a href="https://doi.org/10.1128/JVI.01205-10" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2953165/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20739520/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Virol&amp;title=Pathogenesis%20of%20hepatitis%20E%20virus%20and%20hepatitis%20C%20virus%20in%20chimpanzees:%20Similarities%20and%20differences&amp;volume=84&amp;publication_year=2010&amp;pages=11264-11278&amp;pmid=20739520&amp;doi=10.1128/JVI.01205-10&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib81">
<cite>Yun TJ, Igarashi S, Zhao H, Perez OA, Pereira MR, Zorn E, Shen Y, Goodrum F, Rahman A, Sims PA, et al.  (2021) Human plasmacytoid dendritic cells mount a distinct antiviral response to virus-infected cells. Sci Immunol
6: eabc7302. 10.1126/sciimmunol.abc7302
</cite> [<a href="https://doi.org/10.1126/sciimmunol.abc7302" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8221820/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33811059/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Immunol&amp;title=Human%20plasmacytoid%20dendritic%20cells%20mount%20a%20distinct%20antiviral%20response%20to%20virus-infected%20cells&amp;volume=6&amp;publication_year=2021&amp;pages=eabc7302&amp;pmid=33811059&amp;doi=10.1126/sciimmunol.abc7302&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib82">
<cite>Zhong P, Agosto LM, Ilinskaya A, Dorjbal B, Truong R, Derse D, Uchil PD, Heidecker G, Mothes W (2013) Cell-to-Cell transmission can overcome multiple donor and target cell barriers imposed on cell-free HIV. PLoS One
8: e53138. 10.1371/journal.pone.0053138
</cite> [<a href="https://doi.org/10.1371/journal.pone.0053138" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3538641/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23308151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Cell-to-Cell%20transmission%20can%20overcome%20multiple%20donor%20and%20target%20cell%20barriers%20imposed%20on%20cell-free%20HIV&amp;volume=8&amp;publication_year=2013&amp;pages=e53138&amp;pmid=23308151&amp;doi=10.1371/journal.pone.0053138&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib83">
<cite>Zhou X, Xu L, Wang W, Watashi K, Wang Y, Sprengers D, De Ruiter PE, Van Der Laan LJW, Metselaar HJ, Kamar N, et al.  (2016) Disparity of basal and therapeutically activated interferon signalling in constraining hepatitis E virus infection. J Viral Hepat
23: 294–304. 10.1111/jvh.12491
</cite> [<a href="https://doi.org/10.1111/jvh.12491" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26620360/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Viral%20Hepat&amp;title=Disparity%20of%20basal%20and%20therapeutically%20activated%20interferon%20signalling%20in%20constraining%20hepatitis%20E%20virus%20infection&amp;volume=23&amp;publication_year=2016&amp;pages=294-304&amp;pmid=26620360&amp;doi=10.1111/jvh.12491&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><p>Table S1. <a href="/articles/instance/11966012/bin/LSA-2025-03256_TableS1.docx" class="usa-link">Primer used for RT–qPCR.</a><sup> (18.2KB, docx) </sup></p></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><span>Reviewer comments</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11966012/bin/LSA-2025-03256_review_history.pdf" data-ga-action="click_feat_suppl" class="usa-link">LSA-2025-03256_review_history.pdf</a><sup> (1.4MB, pdf) </sup>
</div></div></section></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Life Science Alliance are provided here courtesy of <strong>Life Science Alliance LLC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.26508/lsa.202503256"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/LSA-2025-03256.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.8 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/11966012/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/11966012/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11966012%2F%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11966012/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC11966012/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC11966012/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40175091/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC11966012/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40175091/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC11966012/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/11966012/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="IpRecD8YG3rvqSuyeguGjBuQTZyJJRLkMTxNibxlDEpbaQbprobZCKsAELJ41oRN">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-B7-tgZZA.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-BLbQGH8u.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-DKf1UY8S.js"></script>
    
    

    </body>
</html>
