
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/base_style-CPPDhfna.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-BepKMsUl.js"></script>

  <link rel="stylesheet" href="/static/assets/article_style-DUfI1jZp.css" />
<link rel="stylesheet" href="/static/assets/style-BOrsu84o.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-D3wJHomh.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Transforming Pharmacovigilance With Pharmacogenomics: Toward Personalized Risk Management - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="18DE241B930AEF3300241B0048505AC8.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="clpt">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12598900/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Clinical Pharmacology and Therapeutics">
<meta name="citation_title" content="Transforming Pharmacovigilance With Pharmacogenomics: Toward Personalized Risk Management">
<meta name="citation_author" content="Claire Spahn">
<meta name="citation_author_institution" content="Department of Pharmacy, Stanford Health Care, Stanford, California, USA">
<meta name="citation_author" content="Nanase Toda">
<meta name="citation_author_institution" content="Department of Clinical Pharmacy, University of California, San Francisco, California, USA">
<meta name="citation_author" content="Blaine Groat">
<meta name="citation_author_institution" content="Personalized Medicine, Healthcare Quality and Safety Center of Excellence, United States Pharmacopeia, Rockville, Maryland, USA">
<meta name="citation_author" content="Omar Aimer">
<meta name="citation_author_institution" content="InnoVigilance, Laval, Québec, Canada">
<meta name="citation_author_institution" content="International Society of Pharmacovigilance, Geneva, Switzerland">
<meta name="citation_author" content="Sara Rogers">
<meta name="citation_author_institution" content="Department of Translational Medical Sciences, School of Medicine, Texas A&amp;M University, Bryan, Texas, USA">
<meta name="citation_author_institution" content="Institute of Biosciences and Technology, Center for Genomic and Precision Medicine, Texas A&amp;M University, Houston, Texas, USA">
<meta name="citation_author_institution" content="American Society of Pharmacovigilance, Houston, Texas, USA">
<meta name="citation_author" content="Akinyemi Oni‐Orisan">
<meta name="citation_author_institution" content="Department of Clinical Pharmacy, University of California, San Francisco, California, USA">
<meta name="citation_author" content="Andrew Monte">
<meta name="citation_author_institution" content="Rocky Mountain Poison &amp; Drug Safety, Denver Health &amp; Hospital Authority, Denver, Colorado, USA">
<meta name="citation_author" content="Nancy Hakooz">
<meta name="citation_author_institution" content="Department of Biopharmaceutics and Clinical Pharmacy, School of Pharmacy, University of Jordan, Amman, Jordan">
<meta name="citation_author" content="the Pharmacogenomics Global Research Network (PGRN) Publications Committee">
<meta name="citation_publication_date" content="2025 Oct 27">
<meta name="citation_volume" content="118">
<meta name="citation_issue" content="6">
<meta name="citation_firstpage" content="1286">
<meta name="citation_doi" content="10.1002/cpt.70095">
<meta name="citation_pmid" content="41146533">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12598900/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12598900/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12598900/pdf/CPT-118-1286.pdf">
<meta name="description" content="Pharmacovigilance is a critical component of medication safety. Despite rigorous evaluation of new drugs during clinical trials, some adverse effects might only be identified once pharmaceuticals are used by a larger population for a longer ...">
<meta name="og:title" content="Transforming Pharmacovigilance With Pharmacogenomics: Toward Personalized Risk Management">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Pharmacovigilance is a critical component of medication safety. Despite rigorous evaluation of new drugs during clinical trials, some adverse effects might only be identified once pharmaceuticals are used by a larger population for a longer ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12598900/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://pmc.ncbi.nlm.nih.gov/search/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12598900">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/cpt.70095"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/CPT-118-1286.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12598900%2F%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12598900/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12598900/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12598900/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-clpt.png" alt="Clinical Pharmacology and Therapeutics logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Clinical Pharmacology and Therapeutics" title="Link to Clinical Pharmacology and Therapeutics" shape="default" href="https://doi.org/10.1002/cpt.70095" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Clin Pharmacol Ther</button></div>. 2025 Oct 27;118(6):1286–1296. doi: <a href="https://doi.org/10.1002/cpt.70095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/cpt.70095</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://pmc.ncbi.nlm.nih.gov/search/?term=%22Clin%20Pharmacol%20Ther%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Clin%20Pharmacol%20Ther%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Clin%20Pharmacol%20Ther%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Clin%20Pharmacol%20Ther%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Transforming Pharmacovigilance With Pharmacogenomics: Toward Personalized Risk Management</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Spahn%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Claire Spahn</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Claire Spahn</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Pharmacy, Stanford Health Care, Stanford, California, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Spahn%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Claire Spahn</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Toda%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Nanase Toda</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Nanase Toda</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Clinical Pharmacy, University of California, San Francisco, California, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Toda%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nanase Toda</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Groat%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Blaine Groat</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Blaine Groat</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Personalized Medicine, Healthcare Quality and Safety Center of Excellence, United States Pharmacopeia, Rockville, Maryland, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Groat%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Blaine Groat</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aimer%20O%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Omar Aimer</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Omar Aimer</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>InnoVigilance, Laval, Québec, Canada</div>
<div class="p">
<sup>
<sup>5</sup>
</sup>International Society of Pharmacovigilance, Geneva, Switzerland</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aimer%20O%22%5BAuthor%5D" class="usa-link"><span class="name western">Omar Aimer</span></a>
</div>
</div>
<sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rogers%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Sara Rogers</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Sara Rogers</span></h3>
<div class="p">
<sup>
<sup>6</sup>
</sup>Department of Translational Medical Sciences, School of Medicine, Texas A&amp;M University, Bryan, Texas, USA</div>
<div class="p">
<sup>
<sup>7</sup>
</sup>Institute of Biosciences and Technology, Center for Genomic and Precision Medicine, Texas A&amp;M University, Houston, Texas, USA</div>
<div class="p">
<sup>
<sup>8</sup>
</sup>American Society of Pharmacovigilance, Houston, Texas, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rogers%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sara Rogers</span></a>
</div>
</div>
<sup>6,</sup><sup>7,</sup><sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Oni%E2%80%90Orisan%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Akinyemi Oni‐Orisan</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Akinyemi Oni‐Orisan</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Clinical Pharmacy, University of California, San Francisco, California, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Oni%E2%80%90Orisan%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Akinyemi Oni‐Orisan</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Monte%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Andrew Monte</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Andrew Monte</span></h3>
<div class="p">
<sup>
<sup>9</sup>
</sup>Rocky Mountain Poison &amp; Drug Safety, Denver Health &amp; Hospital Authority, Denver, Colorado, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Monte%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Andrew Monte</span></a>
</div>
</div>
<sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hakooz%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Nancy Hakooz</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Nancy Hakooz</span></h3>
<div class="p">
<sup>
<sup>10</sup>
</sup>Department of Biopharmaceutics and Clinical Pharmacy, School of Pharmacy, University of Jordan, Amman, Jordan</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hakooz%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nancy Hakooz</span></a>
</div>
</div>
<sup>10</sup>; <span class="collab">the Pharmacogenomics Global Research Network (PGRN) Publications Committee</span>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="cpt70095-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Pharmacy, Stanford Health Care, Stanford, California, USA</div>
<div id="cpt70095-aff-0002">
<sup>
<sup>2</sup>
</sup>Department of Clinical Pharmacy, University of California, San Francisco, California, USA</div>
<div id="cpt70095-aff-0003">
<sup>
<sup>3</sup>
</sup>Personalized Medicine, Healthcare Quality and Safety Center of Excellence, United States Pharmacopeia, Rockville, Maryland, USA</div>
<div id="cpt70095-aff-0004">
<sup>
<sup>4</sup>
</sup>InnoVigilance, Laval, Québec, Canada</div>
<div id="cpt70095-aff-0005">
<sup>
<sup>5</sup>
</sup>International Society of Pharmacovigilance, Geneva, Switzerland</div>
<div id="cpt70095-aff-0006">
<sup>
<sup>6</sup>
</sup>Department of Translational Medical Sciences, School of Medicine, Texas A&amp;M University, Bryan, Texas, USA</div>
<div id="cpt70095-aff-0007">
<sup>
<sup>7</sup>
</sup>Institute of Biosciences and Technology, Center for Genomic and Precision Medicine, Texas A&amp;M University, Houston, Texas, USA</div>
<div id="cpt70095-aff-0008">
<sup>
<sup>8</sup>
</sup>American Society of Pharmacovigilance, Houston, Texas, USA</div>
<div id="cpt70095-aff-0009">
<sup>
<sup>9</sup>
</sup>Rocky Mountain Poison &amp; Drug Safety, Denver Health &amp; Hospital Authority, Denver, Colorado, USA</div>
<div id="cpt70095-aff-0010">
<sup>
<sup>10</sup>
</sup>Department of Biopharmaceutics and Clinical Pharmacy, School of Pharmacy, University of Jordan, Amman, Jordan</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

Correspondence: Claire Spahn (<span>cspahn@stanfordhealthcare.org</span>)
</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 17; Accepted 2025 Sep 26; Collection date 2025 Dec.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Clinical Pharmacology &amp; Therapeutics</em> published by Wiley Periodicals LLC on behalf of American Society for Clinical Pharmacology and Therapeutics.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by-nc/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited and is not used for commercial purposes.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12598900  NIHMSID: <a href="/articles/mid/NIHMS2119900/" class="usa-link">NIHMS2119900</a>  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/41146533/" class="usa-link">41146533</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Pharmacovigilance is a critical component of medication safety. Despite rigorous evaluation of new drugs during clinical trials, some adverse effects might only be identified once pharmaceuticals are used by a larger population for a longer duration. Adverse drug reactions cause negative healthcare outcomes and in severe cases, may lead to hospital admissions, delayed hospital discharges, or deaths. Adverse event reports submitted to pharmacovigilance programs by healthcare professionals and consumers are a key source of information regarding previously unrecognized detrimental effects. Pharmacogenetic markers that indicate how particular genes impact an individual’s response to medication can help explain some idiosyncratic adverse reactions. Incorporating pharmacogenomic guidance in prescribing is proven to decrease the incidence of adverse reactions and improve clinical outcomes. However, this information is not yet routinely included in incident reports. In this era of precision medicine, when prescribing can be tailored to the individual, pharmacogenomic test results yield valuable data that can enhance both individual and population health. Furthermore, advanced artificial intelligence (AI) and machine learning (ML) methods facilitate analysis of complex genetic data, revealing insights not previously available. This white paper outlines current pharmacovigilance and pharmacogenomic practices and recommends that pharmacovigilance programs include pharmacogenomics as a crucial data point in their investigations.</p></section><hr class="headless">
<p>Pharmacovigilance (PV) is defined by the World Health Organization (WHO) as the science and activities related to the detection, assessment, understanding, and prevention of adverse effects and other drug‐related problems.<a href="#cpt70095-bib-0001" class="usa-link" aria-describedby="cpt70095-bib-0001">
<sup>1</sup>
</a> The WHO‐Uppsala Monitoring Centre was established in Sweden in 1978 with the goal of advancing PV efforts globally. Initially a pilot project to foster international collaboration, it evolved into an enriched database (Vigibase)<a href="#cpt70095-bib-0002" class="usa-link" aria-describedby="cpt70095-bib-0002">
<sup>2</sup>
</a> housing millions of adverse drug reaction (ADR) reports submitted by healthcare providers and patients from member nations. Over the past several decades, PV programs have played a key role in enhancing medication safety worldwide, with their findings resulting in critical interventions, including market withdrawals (e.g., rofecoxib, 2004; a nonsteroidal anti‐inflammatory drug approved for conditions such as acute pain and inflammation, found to increase risk for heart attack and stroke, particularly with long‐term use),<a href="#cpt70095-bib-0003" class="usa-link" aria-describedby="cpt70095-bib-0003">
<sup>3</sup>
</a> mandated boxed warnings (e.g., fluoroquinolones, 2008; approved for infections, e.g., of the respiratory tract or urinary tract, and subsequently noted to be associated with potentially permanent injury of the tendons, muscles, joints, and nerves),<a href="#cpt70095-bib-0004" class="usa-link" aria-describedby="cpt70095-bib-0004">
<sup>4</sup>
</a> and safety‐related labeling changes (e.g., lecanemab‐irmb, 2025; approved for treatment of Alzheimer’s disease, the drug’s boxed warning was revised to alert clinicians to the potential for the side effect of neurologic deficits to mimic signs of ischemic stroke and to test for variants in the Apolipoprotien E gene (ApoE) prior to initiating therapy).<a href="#cpt70095-bib-0003" class="usa-link" aria-describedby="cpt70095-bib-0003">
<sup>3</sup>
</a>
</p>
<p>Over time, advancements in the field of precision medicine have refined medical practice from a “one‐size‐fits‐all” approach toward “right drug, right dose, right patient.” Integral to this updated practice is the inclusion of genetics, ranging from biomarkers of disease to pharmacogenetic indicators of drug effects used to target therapies toward greater efficacy and safety. Pharmacogenomic (PGx) information is increasingly incorporated into drug labeling to enhance patient safety and therapeutic efficacy. Recognizing the increasing prevalence of pharmacogenetic guidance in drug labeling, the FDA launched the Table of Pharmacogenetic Associations in 2020,<a href="#cpt70095-bib-0005" class="usa-link" aria-describedby="cpt70095-bib-0005">
<sup>5</sup>
</a> including 22 distinct drug–gene pairs with data indicating potential impact on safety or response. Additionally, guidelines from collaborations such as the Clinical Pharmacogenomics Implementation Consortium (CPIC)<a href="#cpt70095-bib-0006" class="usa-link" aria-describedby="cpt70095-bib-0006">
<sup>6</sup>
</a> and the Dutch Pharmacogenetics Working Group (DPWG)<a href="#cpt70095-bib-0007" class="usa-link" aria-describedby="cpt70095-bib-0007">
<sup>7</sup>
</a> direct clinicians to optimize prescriptions based on their patient’s pharmacogenomic markers. In addition, drug labeling increasingly incorporates pharmacogenomic information to enhance patient safety and therapeutic efficacy. Clopidogrel, an anti‐platelet medication used to prevent blood clots in people at risk for heart attack or stroke, provides a notable example. In 2013, the FDA approved the drug label for clopidogrel (Plavix®) warning that patients who are CYP2C19 poor metabolizers may have diminished effectiveness of the drug and advising healthcare providers to consider alternative treatments for impacted individuals.<a href="#cpt70095-bib-0008" class="usa-link" aria-describedby="cpt70095-bib-0008">
<sup>8</sup>
</a> Such label updates highlight the importance of PGx in precision medicine.</p>
<p>As genetic technology and accessibility continue to evolve, so too has the capability of AI/ML to search, detect, and analyze signals in large, complex data sets. These models can be trained to support highly complex genetic data, improving identification of ADRs linked to specific genotypes. As our understanding of genetic contributors to ADRs grows, PV efforts can be increasingly tailored toward specific patient cohorts, potentially resulting in more individualized risk management strategies.</p>
<section id="cpt70095-sec-0002"><h2 class="pmc_sec_title">BASIC PRINCIPLES</h2>
<section id="cpt70095-sec-0003"><h3 class="pmc_sec_title">Adverse drug reactions</h3>
<p>The WHO defines an ADR as “a response to a medicine which is noxious and unintended, and which occurs at doses normally used in man.” ADRs pose a significant public health risk, contributing to mortality and hospitalizations. One landmark study estimated serious ADRs to rank as high as the fourth leading cause of death among hospitalized patients in the United States (U.S.).<a href="#cpt70095-bib-0008" class="usa-link" aria-describedby="cpt70095-bib-0008">
<sup>8</sup>
</a> Less serious ADRs do not result in hospitalizations or death but still may significantly impact a patient’s health.<a href="#cpt70095-bib-0009" class="usa-link" aria-describedby="cpt70095-bib-0009">
<sup>9</sup>
</a> Effective prevention strategies are essential to reducing the number and severity of these potential negative outcomes.</p>
<p>ADRs were initially classified into two types: Type A (dose‐related) and Type B (non‐dose‐related) before the classification system expanded to six to account for additional clinical scenarios.<a href="#cpt70095-bib-0010" class="usa-link" aria-describedby="cpt70095-bib-0010">
<sup>10</sup>
</a> For the purposes of this review, we will focus on the original two types (<strong>Table</strong>
<a href="#cpt70095-tbl-0001" class="usa-link">
<strong>1</strong>
</a>). Type A reactions are the most common, accounting for 85–90% of all ADRs.<a href="#cpt70095-bib-0011" class="usa-link" aria-describedby="cpt70095-bib-0011">
<sup>11</sup>
</a> These reactions are usually dose‐dependent and predictable based on the known pharmacology of the drug. In contrast, Type B reactions are inherently unpredictable. They do not follow a simple dose–response relationship and may occur in only a small subset of patients.<a href="#cpt70095-bib-0012" class="usa-link" aria-describedby="cpt70095-bib-0012">
<sup>12</sup>
</a>
</p>
<section class="tw xbox font-sm" id="cpt70095-tbl-0001"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Type A and Type B ADRs, adapted<a href="#cpt70095-bib-0011" class="usa-link" aria-describedby="cpt70095-bib-0011">
<sup>11</sup>
</a>
</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Reaction description and type<a href="#cpt70095-note-0001" class="usa-link"><sup>a</sup></a>
</th>
<th align="center" rowspan="1" colspan="1">Details</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Drug overdoses [A]</td>
<td align="left" rowspan="1" colspan="1">
<p>Excessive drug intake, e.g., liver failure with acetaminophen overdose.</p>
<p>Side Effects: Undesirable effects unrelated to the drug’s primary purpose, e.g., gastritis with nonsteroidal anti‐inflammatory drugs (NSAIDs)</p>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Drug interactions [A]</td>
<td align="left" rowspan="1" colspan="1">Reactions due to interaction with other substances, e.g., reduced anticoagulant effects of warfarin with increased vitamin K intake</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Hypersensitivity reactions [B]</td>
<td align="left" rowspan="1" colspan="1">
<p>Immune‐mediated responses classified into Types I–IV, ranging from immediate (e.g., anaphylaxis) to delayed (e.g., Stevens‐Johnson syndrome)</p>
<p>Other Immunological Reactions: Autoimmune disorders (e.g., lupus‐like syndrome) or fixed drug eruptions</p>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Idiosyncratic reactions [B]</td>
<td align="left" rowspan="1" colspan="1">Rare reactions linked to genetic factors, such as hemolysis in glucose‐6‐phosphate dehydrogenase (G6PD) deficiency</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Pseudo allergic reactions [B]</td>
<td align="left" rowspan="1" colspan="1">Resembling allergic reactions but non‐immunologic, e.g., vancomycin flushing syndrome</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/cpt70095-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="cpt70095-note-0001">
<sup>
<sup>a</sup>
</sup><p class="display-inline">While these classifications help in understanding and managing ADRs, overlapping symptoms can occasionally complicate differentiation between types.</p>
</div></div></section><p>It is important to emphasize that ADRs differ from medication errors in that they arise despite correct prescription, proper administration, and good adherence. Notably, systematic procedural changes and error‐prevention processes for drug delivery systems that improve drug safety by mitigating medication errors may not address ADRs.<a href="#cpt70095-bib-0013" class="usa-link" aria-describedby="cpt70095-bib-0013">
<sup>13</sup>
</a> Pharmacovigilance systems, on the other hand, are commonly used to prevent ADRs through detection and reporting.<a href="#cpt70095-bib-0014" class="usa-link" aria-describedby="cpt70095-bib-0014">
<sup>14</sup>
</a>
</p></section></section><section id="cpt70095-sec-0004"><h2 class="pmc_sec_title">PHARMACOVIGILANCE CONCEPTS</h2>
<section id="cpt70095-sec-0005"><h3 class="pmc_sec_title">Development of pharmacovigilance</h3>
<p>The need for PV became evident in the 1950s following widespread birth defects caused by thalidomide use during pregnancy. This tragedy ultimately led to the enactment of the Kefauver–Harris Act of 1962, which granted the FDA greater oversight of drug safety.<a href="#cpt70095-bib-0015" class="usa-link" aria-describedby="cpt70095-bib-0015">
<sup>15</sup>
</a> The FDA then mandated manufacturers to report ADRs.<a href="#cpt70095-bib-0016" class="usa-link" aria-describedby="cpt70095-bib-0016">
<sup>16</sup>
</a>, <a href="#cpt70095-bib-0017" class="usa-link" aria-describedby="cpt70095-bib-0017">
<sup>17</sup>
</a> In 1993, the FDA established MedWatch, a program allowing voluntary reporting of ADRs by healthcare professionals and the public.<a href="#cpt70095-bib-0018" class="usa-link" aria-describedby="cpt70095-bib-0018">
<sup>18</sup>
</a> Data from MedWatch and pharmaceutical companies are compiled in FAERS, which supports safety monitoring and transparency through resources such as the FAERS Public Dashboard—for example, tracking ADRs related to COVID‐19 therapies.<a href="#cpt70095-bib-0019" class="usa-link" aria-describedby="cpt70095-bib-0019">
<sup>19</sup>
</a>, <a href="#cpt70095-bib-0020" class="usa-link" aria-describedby="cpt70095-bib-0020">
<sup>20</sup>
</a>
</p>
<p>As PV specifically aims to improve medication safety, patient care, and public health,<a href="#cpt70095-bib-0021" class="usa-link" aria-describedby="cpt70095-bib-0021">
<sup>21</sup>
</a> it spans the life cycle of the drug development process, beginning in preclinical research and extending into post‐marketing settings. In the United States, the FDA mandates PV efforts in both the pre‐ and post‐marketing phases.<a href="#cpt70095-bib-0014" class="usa-link" aria-describedby="cpt70095-bib-0014">
<sup>14</sup>
</a> Although pre‐marketing PV during clinical trials may contribute to the detection of ADRs, these trials are limited by relatively small sample sizes, strict inclusion/exclusion criteria, and short durations. Consequently, certain ADRs may only become evident after prolonged use of drugs in heterogeneous, real‐world populations. Thus, post‐marketing surveillance serves as a vital component of PV.<a href="#cpt70095-bib-0022" class="usa-link" aria-describedby="cpt70095-bib-0022">
<sup>22</sup>
</a>
</p>
<p>High‐quality reporting is critical for effective PV. The clinical data collection process for PV begins with gathering individual case safety reports (ICSRs), which can originate from either solicited or unsolicited sources (<strong>Figure</strong>
<a href="#cpt70095-fig-0001" class="usa-link">
<strong>1</strong>
</a>). An ICSR is defined as “a description of an adverse event / adverse drug reaction or other observation in an individual patient at a specific point in time.”<a href="#cpt70095-bib-0023" class="usa-link" aria-describedby="cpt70095-bib-0023">
<sup>23</sup>
</a> Solicited reports are gathered by organizations actively requesting information from patients or study participants at structured timepoints (e.g., clinical trials, post‐marketing surveillance). Unsolicited reports are gathered by organizations passively where the patient taking the drug or their healthcare professional provides the information without a request at spontaneous timepoints dictated by when the ADR occurs (e.g., reporting systems such as MedWatch).<a href="#cpt70095-bib-0024" class="usa-link" aria-describedby="cpt70095-bib-0024">
<sup>24</sup>
</a> Once collected, ICSRs are stored in databases maintained by the collecting organization such as the FDA Adverse Event Reporting System (FAERS) in the United States and EudraVigilance in Europe.<a href="#cpt70095-bib-0014" class="usa-link" aria-describedby="cpt70095-bib-0014">
<sup>14</sup>
</a>, <a href="#cpt70095-bib-0025" class="usa-link" aria-describedby="cpt70095-bib-0025">
<sup>25</sup>
</a> In the United States, the FDA outlines “Good Pharmacovigilance Practices,” emphasizing characteristics of a quality case report, such as detailed clinical descriptions and timelines (<strong>Table</strong>
<a href="#cpt70095-tbl-0002" class="usa-link">
<strong>2</strong>
</a>). The FDA’s guidance for industry on PV planning underscores the importance of genetic testing in identifying patient subpopulations at higher risk for ADRs. It directs that safety specifications should include data on “sub‐populations carrying known and relevant genetic polymorphism,” as genetic testing can provide critical insights into susceptibility to adverse reactions.<a href="#cpt70095-bib-0026" class="usa-link" aria-describedby="cpt70095-bib-0026">
<sup>26</sup>
</a> Internationally, organizations such as the European Medicines Agency (EMA) and the WHO have implemented similar pharmacovigilance frameworks to promote global drug safety.<a href="#cpt70095-bib-0027" class="usa-link" aria-describedby="cpt70095-bib-0027">
<sup>27</sup>
</a>, <a href="#cpt70095-bib-0028" class="usa-link" aria-describedby="cpt70095-bib-0028">
<sup>28</sup>
</a>
</p>
<figure class="fig xbox font-sm" id="cpt70095-fig-0001"><h4 class="obj_head">Figure 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/68a4/12641070/1eb647164fa4/CPT-118-1286-g002.jpg" loading="lazy" id="jats-graphic-1" height="370" width="709" alt="Figure 1"></p>
<div class="p text-right font-secondary"><a href="figure/cpt70095-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The pharmacovigilance process.</p></figcaption></figure><section class="tw xbox font-sm" id="cpt70095-tbl-0002"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Characteristics of a good case report. Adapted from: U.S. Food and Drug Administration (FDA). Good Pharmacovigilance Practices and Pharmacoepidemiologic Assessment. Silver Springs, MD: FDA, 2018<a href="#cpt70095-bib-0029" class="usa-link" aria-describedby="cpt70095-bib-0029">
<sup>29</sup>
</a>
</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Element</th>
<th align="center" rowspan="1" colspan="1">Additional details</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Description of the adverse events or disease experience</td>
<td align="left" rowspan="1" colspan="1">Time to onset of signs or symptoms</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Suspected and concomitant product therapy details</td>
<td align="left" rowspan="1" colspan="1">Dose, lot number, schedule, dates, duration, concomitant products including over‐the‐counter medications, dietary supplements, and recently discontinued medications</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Patient characteristics</td>
<td align="left" rowspan="1" colspan="1">Demographic information (e.g., age, race, sex), baseline medical condition prior to product therapy, co‐morbid conditions, use of concomitant medications, relevant family history of disease, and presence of other risk factors</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Documentation of the diagnosis of the events</td>
<td align="left" rowspan="1" colspan="1">Methods used to make the diagnosis</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Clinical course of the event and patient outcomes</td>
<td align="left" rowspan="1" colspan="1">Hospitalization, death etc.</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Relevant therapeutic measures and laboratory data</td>
<td align="left" rowspan="1" colspan="1">At baseline, during therapy, and after therapy, including blood levels, as appropriate</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Dechallenge and/or rechallenge</td>
<td align="left" rowspan="1" colspan="1">Information about patient response</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Any other relevant information</td>
<td align="left" rowspan="1" colspan="1">Other details relating to the event or information on benefits received by the patient, if important to the assessment of the event</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/cpt70095-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>Globally, the WHO’s VigiBase serves as the largest repository of ICSRs, encompassing reports from more than 170 countries.<a href="#cpt70095-bib-0030" class="usa-link" aria-describedby="cpt70095-bib-0030">
<sup>30</sup>
</a> These reporting mechanisms are instrumental in identifying emerging safety concerns and guiding subsequent investigations (<strong>Figure</strong>
<a href="#cpt70095-fig-0001" class="usa-link">
<strong>1</strong>
</a>). Outcomes from such efforts may include withdrawal of drugs, as exemplified by sibutramine (FDA‐approved for weight loss in 1997, withdrawn from the market in 2010 following evidence of increased risk for major adverse cardiovascular events),<a href="#cpt70095-bib-0031" class="usa-link" aria-describedby="cpt70095-bib-0031">
<sup>31</sup>
</a> label updates to include black box warnings as well as warnings and precautions, as seen with natalizumab (FDA‐approved for multiple sclerosis in 2024, temporarily withdrawn in Feb 2005 following 2 cases of a serious brain disease, progressive multifocal leukoencephalopathy),<a href="#cpt70095-bib-0032" class="usa-link" aria-describedby="cpt70095-bib-0032">
<sup>32</sup>
</a> and updates to dose limitations, such as the restriction of 80 mg simvastatin due to myopathy risks (FDA‐approved for hyperlipidemia in 1991, subsequently noted to increase risk for skeletal muscle weakness, stiffness, and pain).<a href="#cpt70095-bib-0033" class="usa-link" aria-describedby="cpt70095-bib-0033">
<sup>33</sup>
</a>
</p></section></section><section id="cpt70095-sec-0006"><h2 class="pmc_sec_title">THE CURRENT PHARMACOVIGILANCE LANDSCAPE</h2>
<p>The current PV landscape reflects a multifaceted approach to ensuring drug safety. Data collection sources include ICSRs submitted by healthcare professionals to regulatory authorities,<a href="#cpt70095-bib-0034" class="usa-link" aria-describedby="cpt70095-bib-0034">
<sup>34</sup>
</a> ADR case reports in medical literature,<a href="#cpt70095-bib-0035" class="usa-link" aria-describedby="cpt70095-bib-0035">
<sup>35</sup>
</a> pharmacoepidemiologic studies, and observational databases such as electronic health records (EHRs).<a href="#cpt70095-bib-0036" class="usa-link" aria-describedby="cpt70095-bib-0036">
<sup>36</sup>
</a> AI and ML offer potential for enhanced signal detection and analysis, yet challenges remain in data quality and integration.<a href="#cpt70095-bib-0037" class="usa-link" aria-describedby="cpt70095-bib-0037">
<sup>37</sup>
</a>
</p>
<p>Key FDA resources, such as drug labels and Table of Pharmacogenetic Associations, exemplify PV efforts to educate stakeholders and optimize drug use beyond addressing ADR reports.<a href="#cpt70095-bib-0038" class="usa-link" aria-describedby="cpt70095-bib-0038">
<sup>38</sup>
</a> Tangible impacts include improved safety warnings, label changes, and Risk Evaluation and Mitigation Strategies (REMS), underscoring PV’s role in advancing precision medicine by matching drugs to appropriate patient populations.<a href="#cpt70095-bib-0039" class="usa-link" aria-describedby="cpt70095-bib-0039">
<sup>39</sup>
</a>
</p>
<p>Globally, PV programs are at different stages of development. While developed countries prioritize extensive post‐market surveillance, others are still expanding their infrastructure to include mandatory reporting, regulatory support, and surveillance mechanisms.<a href="#cpt70095-bib-0040" class="usa-link" aria-describedby="cpt70095-bib-0040">
<sup>40</sup>
</a> These differences influence signal detection and the incorporation of PGx insights.<a href="#cpt70095-bib-0041" class="usa-link" aria-describedby="cpt70095-bib-0041">
<sup>41</sup>
</a> International efforts by organizations such as the International Society of Pharmacovigilance and the American Society of Pharmacovigilance aim to integrate PGx into PV practices, addressing gaps in global data collection and application.<a href="#cpt70095-bib-0042" class="usa-link" aria-describedby="cpt70095-bib-0042">
<sup>42</sup>
</a>
</p>
<p>Despite progress, limitations persist, such as underreporting, reliance on voluntary submissions, and inconsistent data standards across jurisdictions. These disparities can obscure or delay the detection of pharmacogenomic safety signals, particularly when rare genetic variants or population‐specific allele frequencies are involved. In regions with limited infrastructure, genetic data may be absent altogether, while in others, variations in case reporting thresholds or coding systems can hinder data aggregation. Such heterogeneity challenges the generalizability and reproducibility of PGx‐based safety signals. Addressing these gaps requires harmonized global efforts, including standardized adverse event reporting systems, broader adoption of structured genomic data in regulatory submissions, and the use of advanced analytics to reconcile cross‐border data variability.<a href="#cpt70095-bib-0042" class="usa-link" aria-describedby="cpt70095-bib-0042">
<sup>42</sup>
</a>
</p></section><section id="cpt70095-sec-0007"><h2 class="pmc_sec_title">THE CURRENT PHARMACOGENOMICS LANDSCAPE</h2>
<p>Pharmacogenomics is the science and clinical practice of using patient genetic data to improve drug safety and efficacy. More than 95% of patients have been found to have an actionable pharmacogenetic variant that would change treatment.<a href="#cpt70095-bib-0043" class="usa-link" aria-describedby="cpt70095-bib-0043">
<sup>43</sup>
</a>, <a href="#cpt70095-bib-0044" class="usa-link" aria-describedby="cpt70095-bib-0044">
<sup>44</sup>
</a>, <a href="#cpt70095-bib-0045" class="usa-link" aria-describedby="cpt70095-bib-0045">
<sup>45</sup>
</a>, <a href="#cpt70095-bib-0046" class="usa-link" aria-describedby="cpt70095-bib-0046">
<sup>46</sup>
</a> In reality, it is likely that every patient has a pertinent pharmacogenetic variant, though we do not yet have genetic information available in every patient, nor has the scientific community discovered all important PGx variants. Increasing specificity of polygenic risk scores will certainly leverage smaller impact genetic variants together, further improving the predictability of drug response. Availability of PGx data is becoming more widespread in clinical practice, with test results now available through multiple sources. Several major health organizations in the United States now have a PGx program.<a href="#cpt70095-bib-0047" class="usa-link" aria-describedby="cpt70095-bib-0047">
<sup>47</sup>
</a>, <a href="#cpt70095-bib-0048" class="usa-link" aria-describedby="cpt70095-bib-0048">
<sup>48</sup>
</a> Moreover, the FDA has approved direct‐to‐consumer testing that provides results that can be used to change prescribing. These tests have increased patient awareness and interest in PGx. For example, 23andMe, with almost 13 million people genotyped, has developed more than 55 reports that meet FDA requirements.<a href="#cpt70095-bib-0049" class="usa-link" aria-describedby="cpt70095-bib-0049">
<sup>49</sup>
</a>, <a href="#cpt70095-bib-0050" class="usa-link" aria-describedby="cpt70095-bib-0050">
<sup>50</sup>
</a> Thus, PGx results should be assessed if available, or ordered if not, when considering the etiology of adverse drug events.</p>
<p>Genetic determinants of pharmacokinetics and pharmacodynamics are increasingly understood by researchers and clinicians, as evidenced by the &gt; 23,000 drug/gene‐variant annotations curated in the ClinPGx knowledgebase. Importantly, over the past decade, the CPIC (<a href="http://www.cpicpgx.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.cpicpgx.org</a>) has published 28 peer‐reviewed guidelines and 17 updates involving 160 drugs and 34 genes to facilitate the translation of pharmacogenomic data into actionable prescribing algorithms.<a href="#cpt70095-bib-0051" class="usa-link" aria-describedby="cpt70095-bib-0051">
<sup>51</sup>
</a> The US Department of Veterans Affairs uses these data extensively through its Pharmacogenomics testing for veterans program (PHASER)<a href="#cpt70095-bib-0052" class="usa-link" aria-describedby="cpt70095-bib-0052">
<sup>52</sup>
</a>, <a href="#cpt70095-bib-0053" class="usa-link" aria-describedby="cpt70095-bib-0053">
<sup>53</sup>
</a> and several major health insurance carriers, including Medicare, cover PGx testing. Further, the US FDA Division of Translational and Precision Medicine maintains a growing list of PGx biomarkers in drug labeling that contains 541 drugs as of January 2025.<a href="#cpt70095-bib-0054" class="usa-link" aria-describedby="cpt70095-bib-0054">
<sup>54</sup>
</a> Genetic variability is now assessed as part of the new drug approval process,<a href="#cpt70095-bib-0055" class="usa-link" aria-describedby="cpt70095-bib-0055">
<sup>55</sup>
</a> with PGx data used to maximize efficacy and minimize adverse drug events (ADEs).</p></section><section id="cpt70095-sec-0008"><h2 class="pmc_sec_title">PROPOSAL FOR INCORPORATION OF PGx INTO PV PROGRAMS</h2>
<p>Although PGx is both evidence‐based and guideline‐supported, it is not yet routinely implemented in PV programs. Several novel research studies outline successful approaches to including PGx in PV analysis. The following three studies provide distinct approaches, highlighting the feasibility of this goal:
</p>
<ul id="cpt70095-list-0001" class="list" style="list-style-type:disc">
<li id="cpt70095-li-0001"><p>A 2003 study by Aronson and Ferner proposed a new framework for ADR classification, emphasizing the integration of pharmacogenetic (PGx) data into PV practices. The three‐dimensional system, known as DoTS (Dose‐Time‐Susceptibility), classified ADRs based on dose dependence, timing, and patient susceptibility. By incorporating PGx insights into the susceptibility dimension, the framework highlighted how genetic polymorphisms, such as those affecting cytochrome P450 enzymes or drug transporters, could influence individual responses to medications. This approach bridged gaps in traditional ADR classification systems, which often failed to account for patient‐specific factors. The DoTS system enabled personalized PV by identifying high‐risk populations and providing mechanistic insights into ADRs. This integration of PGx data guided drug monitoring strategies, informed clinical decisions, and supported the development of tailored therapies. By addressing individual variability in ADR risk, this framework enhanced PV’s predictive and preventive capabilities, paving the way for safer, more effective, and personalized drug use.<a href="#cpt70095-bib-0056" class="usa-link" aria-describedby="cpt70095-bib-0056">
<sup>56</sup>
</a>
</p></li>
<li id="cpt70095-li-0002"><p>A 2004 study by Clark <em>et al</em>. introduced a novel methodology for integrating pharmacogenetics (PGx) into PV by linking genetic research with prescription event monitoring (PEM) studies. It highlighted the potential of PGx to identify genetic factors, such as variations in P‐glycoprotein (P‐gp) and CYP2C9, that contributed to individual susceptibility to ADRs. Using minimally invasive buccal swabs for DNA collection and a nested case–control study design, the research demonstrated the feasibility of investigating genetic risks for ADRs in real‐world settings. While the pilot study had limited statistical power, it set the stage for larger scale, population‐based research, particularly in diverse genetic cohorts, to refine the role of PGx in ADR prevention. Ultimately, this integration of PGx into pharmacovigilance offered a powerful tool for enhancing drug safety, developing guidelines for pre‐treatment genotyping, and ensuring more effective and precision medication use.<a href="#cpt70095-bib-0057" class="usa-link" aria-describedby="cpt70095-bib-0057">
<sup>57</sup>
</a>
</p></li>
<li id="cpt70095-li-0003"><p>A 2008 study by Koh <em>et al</em>. introduced a quantitative solution for assessing ADR causality using a genetic algorithm to optimize scoring criteria, providing precise probability scores for ADR likelihood. Tested on cases from Singapore’s PV center, the algorithm achieved high sensitivity (83.8%) and specificity (71.0%), outperforming conventional methods. It offered a robust, objective tool for differentiating true ADR signals from noise, particularly in settings with limited data or unprecedented ADRs, such as clinical trials. The algorithm enhanced inter‐ and intra‐rater agreement, aiding clinicians, regulatory agencies, and pharmaceutical companies in ADR prediction and management. Its adaptability for incorporating new data makes it a valuable addition to PV systems, with potential future integration of pharmacogenetic data to address genetic variability in ADR susceptibility.<a href="#cpt70095-bib-0058" class="usa-link" aria-describedby="cpt70095-bib-0058">
<sup>58</sup>
</a>
</p></li>
</ul>
<p>While these studies involved limited populations, they all identified opportunities for future research and expansion. Building on this research, we propose incorporating PGx into PV in three key areas:
</p>
<ul id="cpt70095-list-0002" class="list" style="list-style-type:disc">
<li id="cpt70095-li-0004"><p>Data input and collection,</p></li>
<li id="cpt70095-li-0005"><p>Data analysis and risk evaluation,</p></li>
<li id="cpt70095-li-0006"><p>Data output and action.</p></li>
</ul>
<section id="cpt70095-sec-0009"><h3 class="pmc_sec_title">Data input and collection</h3>
<section id="cpt70095-sec-0010"><h4 class="pmc_sec_title">Current state</h4>
<p>PV data come from various sources, including spontaneous reporting systems such as FAERS, pharmaceutical company reports, clinical trials, literature, and collaborations with biobanks.</p>
<p>Patients and healthcare providers use tools such as FDA Form 3500 (healthcare providers) or Form 3500B (consumers) to report ADRs and interactions with pharmaceutical manufacturer representatives to share essential details about the event, the drug, and the patient.</p>
<p>Most PV data are collected by pharmaceutical companies, which generate ICSRs and initiate PV activities through internal teams or outsourced vendors. ICSRs are a vital source of post‐marketing drug safety data. As of March 2025, FAERS had received more than 30 million reports, of which almost 17 million were serious reports and almost 3 million were death reports.<a href="#cpt70095-bib-0019" class="usa-link" aria-describedby="cpt70095-bib-0019">
<sup>19</sup>
</a> The FAERS database contains both structured and unstructured data; structured data regarding ADRs are coded with a uniform vocabulary (MedDRA, developed by the International Conference on Harmonization of Technical Requirements for Pharmaceuticals for Human use, ICH) to facilitate data search and analysis. Utilization of standardized coding terminology also allows for collation of data from various international databases. Specific unstructured data may be coded by MedDRA and added to structured data fields.<a href="#cpt70095-bib-0059" class="usa-link" aria-describedby="cpt70095-bib-0059">
<sup>59</sup>
</a> These data are, however, associated with limitations, for example, reporting is voluntary, so not all events are captured. Therefore, it is not possible to calculate the incidence or prevalence of an ADR from PV data alone. Furthermore, reports may be incomplete or even duplicated. Reporting ratios of the adverse event reports vs. estimated drug utilization may be considered an estimation of the adverse event, but it is not an accurate representation of the true event rate.<a href="#cpt70095-bib-0060" class="usa-link" aria-describedby="cpt70095-bib-0060">
<sup>60</sup>
</a>
</p>
<p>Clinical trials also contribute significantly to PV efforts, exemplified by the identification of genetic associations such as HLA‐B*5701 for abacavir (an antiviral drug, approved for the treatment of HIV) and HLA‐B*5801 for carbamazepine (FDA‐approved for seizure, bipolar disorder, and neuropathic pain).<a href="#cpt70095-bib-0061" class="usa-link" aria-describedby="cpt70095-bib-0061">
<sup>61</sup>
</a>
</p>
<p>EMA mandates the inclusion of PGx considerations in Risk Management Plans (RMPs), requiring genomic data collection and monitoring during both clinical trials and the post‐authorization phase. This ensures the integration of genetic factors into drug safety assessments, addressing variability in patient responses.<a href="#cpt70095-bib-0062" class="usa-link" aria-describedby="cpt70095-bib-0062">
<sup>62</sup>
</a>
</p>
<p>Literature sources, including case reports and pharmacoepidemiologic studies, further enhance PV by identifying rare or idiosyncratic reactions and uncovering patterns in drug use and outcomes across larger populations.<a href="#cpt70095-bib-0063" class="usa-link" aria-describedby="cpt70095-bib-0063">
<sup>63</sup>
</a> To ensure quality case reports, authors should follow the 13‐point checklist outlined by the CARE case report guidelines.<a href="#cpt70095-bib-0064" class="usa-link" aria-describedby="cpt70095-bib-0064">
<sup>64</sup>
</a> Developed by international experts, these guidelines promote the accuracy, transparency, and usefulness of case reports.</p>
<p>Databases such as ClinPGx, which curate information on the impact of genetic variation on drug response, are invaluable resources for integrating PGx into PV. ClinPGx provides summaries of gene‐drug relationships, genotype–phenotype associations, and clinical guidelines, helping PV systems identify genetic factors in ADRs and recommend appropriate PGx testing. For instance, if an ADR with a known genetic association is reported, ClinPGx can guide the selection of relevant genes or variants to investigate.<a href="#cpt70095-bib-0065" class="usa-link" aria-describedby="cpt70095-bib-0065">
<sup>65</sup>
</a>
</p></section><section id="cpt70095-sec-0011"><h4 class="pmc_sec_title">Incorporating pharmacogenomics</h4>
<p>Enhancing PGx integration into PV processes could be achieved by modifying FDA Form 3500/3500B to include a dedicated field for PGx information, such as genetic test results, genotype/phenotype details, or whether PGx testing has been conducted. The current form does not have any fields specifically for PGx; therefore, any PGx data collected thus far are gathered from free‐text fields including descriptions/commentary and laboratory data. Adding prompts like, “Has PGx testing been performed for this patient? If not, consider a PGx test for drugs with known genetic associations,” would encourage systematic collection of genetic data. For example, in cases of clopidogrel‐related stent thrombosis, PGx testing could identify at‐risk individuals with genetic variants affecting drug response, enabling improved RMPs and patient safety.</p>
<p>The EMA stresses systematic genomic biospecimen collection and signal detection for serious ADRs or unexpected drug responses. This approach supports continuous evaluation of genetic factors in real‐world settings, ensuring pharmacogenomic insights remain central to ADR prevention and management.<a href="#cpt70095-bib-0062" class="usa-link" aria-describedby="cpt70095-bib-0062">
<sup>62</sup>
</a> Genetic differences across populations further highlight the need for PGx data in PV systems. For example, G6PD deficiency, prevalent in African and Mediterranean populations, illustrates the importance of considering ethnicity in PV workflows. Incorporating such data enables improved ADR stratification by ethnic subpopulations, allowing targeted safety monitoring and reducing health disparities.<a href="#cpt70095-bib-0066" class="usa-link" aria-describedby="cpt70095-bib-0066">
<sup>66</sup>
</a>
</p>
<p>Soliciting additional data to understand ADR risk factors are a key aspect of PV. PGx testing is particularly useful for identifying genetic variants linked to drug responses, especially comprehensive panel testing. PGx panel testing may be warranted when genetic factors are suspected, supported by evidence or emerging patterns. PV systems, designed to detect rare outcomes like drug‐induced liver injury (DILI), can benefit from including genetic insights alongside additional confounding factors such as phenoconversion and drug–drug interactions.</p>
<p>Dynamic PV frameworks are required to address the challenges of transient changes in metabolic phenotypes due to drug interactions or conditions such as inflammation. These temporary alterations can impact genotype–phenotype concordance, underscoring the importance of real‐time data collection and integration into PV systems. For instance, a patient’s phenotype may temporarily shift due to inflammation, affecting drug metabolism and potentially masking or exacerbating ADR signals. Such scenarios necessitate flexible PV systems capable of adapting to dynamic patient conditions.<a href="#cpt70095-bib-0067" class="usa-link" aria-describedby="cpt70095-bib-0067">
<sup>67</sup>
</a>
</p>
<p>Pharmacovigilance data often lead to the categorization of extreme phenotypes, such as ultra‐rapid metabolizers (UMs) or poor metabolizers (PMs). PGx data enhance population‐level analyses of these rare phenotypes, enabling comparisons of ADR rates with background allele frequencies by ethnicity. This approach helps refine drug labeling and mitigate risks. Incorporating both genotype and phenotype data in PV reporting supports comprehensive risk assessments and improves understanding of genetic contributions to drug safety.</p>
<p>In addition to FAERS and ICSRs, PV data are derived from collaborations with biobanks, case studies, and consortiums, which provide opportunities to identify rare genetic variants and refine safety profiles. By integrating PGx considerations into ADR reporting, ICSRs, and clinical trial protocols, pharmaceutical companies can improve the prediction, prevention, and management of ADRs (<strong>Figure</strong>
<a href="#cpt70095-fig-0002" class="usa-link">
<strong>2</strong>
</a>). This integration not only advances precision medicine while aligning with regulatory expectations but also enhances patient safety by leveraging genetic insights to mitigate risks effectively.</p>
<figure class="fig xbox font-sm" id="cpt70095-fig-0002"><h5 class="obj_head">Figure 2.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/68a4/12641070/b3dcd61b53da/CPT-118-1286-g001.jpg" loading="lazy" id="jats-graphic-3" height="237" width="709" alt="Figure 2"></p>
<div class="p text-right font-secondary"><a href="figure/cpt70095-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Sources and processes for pharmacovigilance data generation. PV data are generated through a variety of sources and processes aimed at monitoring the safety and efficacy of medicines and detecting adverse drug reactions (ADRs).</p></figcaption></figure></section></section><section id="cpt70095-sec-0012"><h3 class="pmc_sec_title">Data analysis and risk evaluation</h3>
<section id="cpt70095-sec-0013"><h4 class="pmc_sec_title">Data availability</h4>
<p>While PGx data have become more available, not all patients are genotyped. PGx tests can and should be recommended as part of a causal assessment when data support clinical implementation. Patients can request such testing, which is increasingly reimbursed by insurers, but may not realize they already have access to these data through direct‐to‐consumer (DTC) providers. Thus, both patients and their treating clinicians must be queried about the availability of these results through clinical laboratories or DTC providers. Patients may be referred for testing when the clinical case suggests the need for these results.</p></section><section id="cpt70095-sec-0014"><h4 class="pmc_sec_title">Adequacy of genotyping</h4>
<p>To ensure accurate results, a PGx test must cover all relevant genetic variants. The PGx community has only recently begun to specify which variants are necessary to claim adequate coverage of a gene.<a href="#cpt70095-bib-0068" class="usa-link" aria-describedby="cpt70095-bib-0068">
<sup>68</sup>
</a>, <a href="#cpt70095-bib-0069" class="usa-link" aria-describedby="cpt70095-bib-0069">
<sup>69</sup>
</a> Full sequencing with adequate phasing to capture variation on both DNA strands provides the most complete picture of a patient’s genome.<a href="#cpt70095-bib-0070" class="usa-link" aria-describedby="cpt70095-bib-0070">
<sup>70</sup>
</a> This technique is infrequently performed in clinical practice due to concerns about the cost and resources required for interpretation and data storage. The sequence data for an entire genome requires approximately 150 gigabytes of storage space, meaning exabytes of data storage with robust deidentification security measures are required to store these data on a population level. However, leading companies, such as Illumina and PacBio,<a href="#cpt70095-bib-0071" class="usa-link" aria-describedby="cpt70095-bib-0071">
<sup>71</sup>
</a> are making these data more available and whole genomic sequencing cost has decreased to as low as $100.<a href="#cpt70095-bib-0072" class="usa-link" aria-describedby="cpt70095-bib-0072">
<sup>72</sup>
</a>
</p>
<p>As a cost‐effective alternative to full sequencing, most clinical laboratories now utilize microarray platforms that allow capture of a large number of single nucleotide variants (SNVs). With these tests, each individual base pair variant can be stored in a discrete data field, which requires minimal storage and facilitates assessment, accurate interpretation, and discovery when new clinical phenotype patterns emerge. When paired with long‐range reads, this technique can capture a large amount of genomic variability with adequate coverage of common clinically actionable variants. The Association for Molecular Pathology, College of American Pathologists, Dutch Pharmacogenetics Working Group of the Royal Dutch Pharmacists Association, and European Society for Pharmacogenomics and Personalized Therapy provide excellent resources with a list of minimum variants that must be captured when considering whether a test is sufficient for clinical interpretation.<a href="#cpt70095-bib-0073" class="usa-link" aria-describedby="cpt70095-bib-0073">
<sup>73</sup>
</a> Assessment of the variants covered by the test provider is necessary to prevent misclassification of predicted phenotypes and false attribution when patients are insufficiently genotyped.</p></section><section id="cpt70095-sec-0015"><h4 class="pmc_sec_title">Pharmacovigilance signal strength</h4>
<p>The goal of data screening is to identify new safety signals, which suggest causal associations between the drug and the adverse event. While every ICSR cannot be individually screened, complementary approaches are employed for more efficient signal identification. FDA then categorizes the causality as either <em>probable, possible, unassessable</em>, or <em>unlikely</em>.<a href="#cpt70095-bib-0074" class="usa-link" aria-describedby="cpt70095-bib-0074">
<sup>74</sup>
</a> Data mining typically relies on disproportionality analysis to identify drug and adverse event pairs that occur more frequently than other combinations in the database.<a href="#cpt70095-bib-0074" class="usa-link" aria-describedby="cpt70095-bib-0074">
<sup>74</sup>
</a> Extreme discordant phenotypes<a href="#cpt70095-bib-0075" class="usa-link" aria-describedby="cpt70095-bib-0075">
<sup>75</sup>
</a> that result in severe ADRs are most likely to result in the identification of PGx associations. Initial integration should focus on analyzing available genetic data first, then genotyping patients with these severe ADRs when genotype data are not available. Combining data across PV providers will be necessary for low‐frequency genetic variants and less common race/ethnicities. Signal strength can also be increased through the distillation of variants into predicted phenotypes. There must be a pipeline to provide phenotypic prediction based on the genetic variants assessed. Furthermore, robust analysis of the impact of PGx results incorporated into ICSR reports would necessitate comparison with population allele frequencies already cataloged in PGx databases such as ClinPGx.<a href="#cpt70095-bib-0076" class="usa-link" aria-describedby="cpt70095-bib-0076">
<sup>76</sup>
</a> This approach is already documented via published case studies outlining rare variants linked with adverse drug reactions.<a href="#cpt70095-bib-0077" class="usa-link" aria-describedby="cpt70095-bib-0077">
<sup>77</sup>
</a>
</p></section></section><section id="cpt70095-sec-0016"><h3 class="pmc_sec_title">Data output and action</h3>
<p>A comprehensive PV program should encompass both risk identification and risk mitigation strategies to ensure drug safety and efficacy. Risk identification involves the detection of ADRs and potential PGx associations, while risk mitigation includes implementing appropriate strategies, such as preemptive testing, targeted prescribing, and updating product information. When an ADR is reported, the PV process involves assessing the event, submitting a detailed report to regulatory authorities such as the FDA or the EMA, and developing a targeted risk mitigation strategy. If a PGx link is identified, risk mitigation should focus on developing preemptive testing strategies and tailoring recommendations to target populations.<a href="#cpt70095-bib-0055" class="usa-link" aria-describedby="cpt70095-bib-0055">
<sup>55</sup>
</a> For example, in severe reactions such as Stevens–Johnson Syndrome (SJS), the question arises whether routine preemptive HLA testing should be performed in all patients at risk.</p>
<p>To improve the integration of PGx data into PV, ADR reporting forms should include fields for PGx information. This could involve leveraging existing lab result fields or introducing a specific PGx section to capture genetic test results, genotype/phenotype details, or patient testing history. During the initial ADR report or follow‐up investigations, healthcare providers or pharmacists managing the PV process should query whether PGx testing has been performed or if the patient is willing to submit to testing. The FDA—or equivalent regulatory agency, as applicable—could recommend <em>post hoc</em> PGx testing for specific cases, ensuring ADR investigations are thorough and evidence based.</p>
<p>Over time, a PGx‐enriched PV database (or biobank) could be developed, incorporating PGx samples from patients with idiosyncratic drug reactions. This database would enable comparison of genetic profiles between patients who experience ADRs and those who do not, potentially identifying new causative genes or genetic markers. Such data could be mined for research purposes, aiding in the discovery of novel PGx associations and enhancing risk stratification strategies.<a href="#cpt70095-bib-0078" class="usa-link" aria-describedby="cpt70095-bib-0078">
<sup>78</sup>
</a>
</p>
<p>Effective risk mitigation plans could include product label updates with PGx information, recommendations for genetic testing (mandatory or optional), dosing modifications, or warnings for at‐risk populations. For example, if a PGx link suggests a dosing adjustment for poor metabolizers, this information should be clearly communicated to healthcare providers, regulatory bodies, and potentially consumers to improve safety and decision making.</p></section></section><section id="cpt70095-sec-0017"><h2 class="pmc_sec_title">LEVERAGING AI/ML</h2>
<p>As more PGx data become integrated into PV systems, advanced tools such as data mining and AI have the potential to significantly enhance safety signal detection, risk factor identification, and patient stratification based on genetic risk profiles. Recent advancements in ML, particularly in natural language processing (NLP) and deep learning, offer promising approaches to analyze the vast and heterogeneous datasets involved in PV. For instance, NLP can efficiently process unstructured data, such as electronic health records (EHRs), case reports, and social media posts, extracting key insights about ADRs and gene‐drug interactions. A recent scoping review by Golder <em>et al</em>.<a href="#cpt70095-bib-0035" class="usa-link" aria-describedby="cpt70095-bib-0035">
<sup>35</sup>
</a> identified the growing application of NLP and ML in extracting actionable pharmacogenomic information from clinical text to support adverse event detection, particularly in unstructured EHR‐linked datasets.</p>
<p>Moreover, ML models, such as Bayesian Confidence Propagation Neural Networks (BCPNNs) and modern deep learning architectures (e.g., transformer‐based models), can identify complex patterns in safety data, such as ADR clusters, by incorporating genetic and phenotypic features.<a href="#cpt70095-bib-0079" class="usa-link" aria-describedby="cpt70095-bib-0079">
<sup>79</sup>
</a> For example, Bayesian Confidence Propagation Neural Networks have been successfully used in spontaneous reporting databases to detect subtle PGx‐associated safety signals that might be overlooked using traditional disproportionality analysis.<a href="#cpt70095-bib-0080" class="usa-link" aria-describedby="cpt70095-bib-0080">
<sup>80</sup>
</a> These tools have been shown to support the identification of rare, PGx‐related adverse events, such as SJS linked to HLA‐B*1502 or drug‐induced hepatotoxicity associated with CYP polymorphisms, by enhancing sensitivity in signal detection frameworks.<a href="#cpt70095-bib-0079" class="usa-link" aria-describedby="cpt70095-bib-0079">
<sup>79</sup>
</a> Another recent study used a novel message‐passing neural network model that integrates drug–gene expression and structural data to accurately predict drug–adverse reaction associations, offering a cost‐effective approach to enhance drug safety assessments.<a href="#cpt70095-bib-0081" class="usa-link" aria-describedby="cpt70095-bib-0081">
<sup>81</sup>
</a>
</p>
<p>Integrating PGx into PV workflows enables personalized safety monitoring by identifying populations at heightened genetic risk for ADRs. PGx‐guided ML models can incorporate genetic variations affecting drug metabolism—such as CYP2D6, CYP2C19, or SLCO1B1 polymorphisms—to refine safety signal detection and stratify patients based on susceptibility to specific ADRs. In addition to EHR and claims data, AI/ML systems can link PGx data to real‐world safety events from post‐market surveillance databases, facilitating proactive identification of high‐risk patients and more nuanced benefit–risk assessments.<a href="#cpt70095-bib-0082" class="usa-link" aria-describedby="cpt70095-bib-0082">
<sup>82</sup>
</a> Leveraging advanced algorithms and external data sources, such as molecular structures, geographical reporting trends, and longitudinal health data, enables the detection of subtle safety signals and stratifies patients with greater precision.<a href="#cpt70095-bib-0083" class="usa-link" aria-describedby="cpt70095-bib-0083">
<sup>83</sup>
</a>
</p>
<p>The integration of PGx data into PV systems also enhances the utility of real‐world data (RWD). AI/ML algorithms can process PGx data alongside RWD from diverse sources—such as EHRs, insurance claims, and patient‐reported outcomes—to identify patterns of ADRs that traditional methods may overlook. These systems can refine risk predictions by accounting for genetic factors, environmental influences, and drug–drug interactions, thereby supporting precision medicine approaches in drug safety monitoring. Additionally, causal inference techniques can establish robust links between genetic predispositions and observed ADRs, ensuring actionable insights for clinical decision making.<a href="#cpt70095-bib-0084" class="usa-link" aria-describedby="cpt70095-bib-0084">
<sup>84</sup>
</a>
</p>
<p>Emerging digital tools, such as mobile apps and sensors, are increasingly used to collect safety data closer to the point of care. AI/ML models can process these rich data streams to monitor medication adherence, genetic markers, and associated ADRs in real time, improving the scalability and accuracy of PGx‐based PV systems. Mobile apps, such as the WEB‐RADR App, allow patients to report adverse drug reactions directly, improving the timeliness and quality of safety data. Wearable electrochemical sensors can noninvasively monitor drug levels and physiological parameters in real time, while AI‐supported web applications help reduce polypharmacy‐related side effects in geriatric patients. The AiRDRUG Web App, for example, supports rational drug use by analyzing drug interactions and patient‐specific data to flag high‐risk combinations. These tools enhance PV by capturing safety data closer to the point of care.</p>
<p>Governance and regulatory alignment remain critical challenges in realizing the full potential of AI/ML for PGx‐enriched PV. Adopting Good Machine Learning Practices (GMLP) and establishing frameworks for data transparency, standardization, and reproducibility will help address concerns about explainability and accountability in ML‐driven PGx PV systems. By ensuring regulatory compliance and fostering stakeholder trust, AI/ML systems can drive the adoption of PGx‐informed safety monitoring and risk mitigation strategies.<a href="#cpt70095-bib-0085" class="usa-link" aria-describedby="cpt70095-bib-0085">
<sup>85</sup>
</a>
</p></section><section id="cpt70095-sec-0018"><h2 class="pmc_sec_title">CHALLENGES AND CONTROVERSIES</h2>
<p>Despite its critical role in drug safety, PV faces numerous challenges, particularly in the integration of PGx into its framework. The inclusion of PGx data in PV holds promise for improving drug safety and efficacy, yet several uncertainties must be addressed to achieve its full potential. Specific challenges include:</p>
<p>
<em>Education and awareness</em>: Many clinicians are unfamiliar with how to interpret PGx data and its clinical relevance, resulting in low uptake and utilization, particularly in regions with limited healthcare resources.<a href="#cpt70095-bib-0086" class="usa-link" aria-describedby="cpt70095-bib-0086">
<sup>86</sup>
</a> Increasing awareness through targeted education programs and integrating PGx knowledge into medical curricula is essential to foster widespread adoption.</p>
<p>
<em>Limited availability and utilization of PGx data in PV systems</em>: Current PV frameworks often lack standardized processes for collecting, storing, and analyzing PGx information. The absence of specific fields for PGx data in ADR reports limits its integration into safety signal detection and risk management strategies.<a href="#cpt70095-bib-0087" class="usa-link" aria-describedby="cpt70095-bib-0087">
<sup>87</sup>
</a> Moreover, the lack of prospective clinical trial data demonstrating the clinical and economic value of PGx in optimizing drug dosing and treatment outcomes creates barriers to broader implementation and insurance coverage.<a href="#cpt70095-bib-0088" class="usa-link" aria-describedby="cpt70095-bib-0088">
<sup>88</sup>
</a>
</p>
<p>
<em>Gene effect size</em>: Effective size and predictive value metrics, such as positive predictive value (PPV), remain inadequately explored for many PGx variants. This gap hinders the ability to robustly correlate PGx data with ADRs, making it difficult to derive actionable insights.<a href="#cpt70095-bib-0089" class="usa-link" aria-describedby="cpt70095-bib-0089">
<sup>89</sup>
</a>
</p>
<p>
<em>Laboratory standardization</em>: Uniform phenotype calls, robust tests, and the inclusion of all clinically actionable genes with CPIC level A/B evidence and Tier 1/2 alleles are crucial for reliable PGx data generation.<a href="#cpt70095-bib-0090" class="usa-link" aria-describedby="cpt70095-bib-0090">
<sup>90</sup>
</a>
</p>
<p>
<em>High dimensionality of genomic information</em>: Data complexity impacts the storage, analysis, and mining of PGx data for safety signal detection. Incorporating PGx into PV requires advanced data‐sharing frameworks and collaborative initiatives to build large, diverse datasets that can be used to identify safety signals. AI and ML tools, which the FDA has already begun employing for traditional data, must evolve to handle the complexity of PGx data. These tools could revolutionize signal detection by uncovering previously undetected PGx‐associated risks.<a href="#cpt70095-bib-0082" class="usa-link" aria-describedby="cpt70095-bib-0082">
<sup>82</sup>
</a>
</p>
<p>
<em>Ethical, legal, and social implications (ELSI)</em>: Privacy concerns surrounding PGx data, particularly regarding data sharing and cybersecurity, pose barriers to its widespread use in PV.<a href="#cpt70095-bib-0091" class="usa-link" aria-describedby="cpt70095-bib-0091">
<sup>91</sup>
</a> In particular, the reuse of PGx data in post‐market PV raises questions about informed consent—especially when such data are used retrospectively for purposes not originally disclosed to patients. Informed consent frameworks must be revisited to address the secondary use of data, ensuring transparency, patient autonomy, and compliance with evolving legal standards. Addressing these concerns through robust data protection protocols and clear regulatory guidelines will be essential to building trust among stakeholders. The growing use of biobanking and large‐scale genomic integration into PV systems also amplifies these concerns, requiring strict governance mechanisms for data access, usage, and long‐term storage.</p>
<p>
<em>Engagement with healthcare providers and patients</em>: Providers must understand the value of PGx in enhancing drug safety, while patients should be empowered to participate in PGx testing and share data for PV purposes. Meaningful engagement hinges on culturally sensitive communication strategies that respect diverse perspectives on genetic testing and data sharing. This includes transparent discussions about benefits, risks, and data stewardship responsibilities.<a href="#cpt70095-bib-0092" class="usa-link" aria-describedby="cpt70095-bib-0092">
<sup>92</sup>
</a>
</p>
<section class="bt xbox font-sm" id="cpt70095-blkfxd-0001"><p>
<strong>Suggested modifications for consideration in PV programs to include PGx data:</strong>
</p>
<ul id="cpt70095-list-0003" class="list" style="list-style-type:disc">
<li id="cpt70095-li-0007"><p>Update case report forms (e.g., FDA Form 3500) to include a specific field for PGx test results.</p></li>
<li id="cpt70095-li-0008"><p>Add prompts to the case report forms to query whether PGx testing has been completed.</p></li>
<li id="cpt70095-li-0009"><p>If PGx test results are not available, query whether the patient is willing/able to undergo testing if the drug has an associated PGx recommendation.</p></li>
<li id="cpt70095-li-0010"><p>Establish a methodology to confirm assigned phenotypes based on reported genotypes and other dynamic conditions such as drug–gene interactions and phenoconversion.</p></li>
<li id="cpt70095-li-0011"><p>Develop an enriched biobank over time, which will facilitate identification of pharmacogenetic factors predisposing to increased risk of ADRs.</p></li>
<li id="cpt70095-li-0012"><p>Mine the biobank to identify additional at‐risk populations for targeted testing to preempt ADRs.</p></li>
</ul></section></section><section id="cpt70095-sec-0019"><h2 class="pmc_sec_title">CONCLUSION</h2>
<p>PGx is emerging as a critical component of comprehensive analysis of ADRs. An abundance of resources supports the use of PGx, ranging from clinical guidelines and primary literature to guidance from drug regulatory agencies. With increasing testing and data availability, PV programs now have both the opportunity and responsibility to include vital patient‐specific data in the investigation of ADRs.</p>
<p>To date, several research studies have highlighted the benefit of including PGx parameters in such investigations and outlined methodologies to successfully incorporate this data into PV programs. Furthermore, with rapid advances in AI’s ability to analyze complex data, it is now feasible for PV programs to incorporate PGx pharmacogenomics into the process. As data accumulates, these PGx‐enriched databases may yield ADR signals that will help inform healthcare providers and pharmaceutical manufacturers to prevent future adverse events.</p>
<p>Studies consistently demonstrate that the vast majority of the population carries at least one clinically actionable pharmacogenetic variant.<a href="#cpt70095-bib-0043" class="usa-link" aria-describedby="cpt70095-bib-0043">
<sup>43</sup>
</a> As awareness of patient heterogeneity and inter‐individual differences in drug response has evolved over time, so too has our duty to investigate pharmacogenetic factors contributing to ADRs. While we acknowledge challenges to integration, we propose that PV programs solicit pharmacogenomic data as standard practice so that ADRs can be analyzed in the context of possible genetic links. Regulatory bodies can then make precision medicine‐directed recommendations or updates based on the findings. In addition to addressing individual ICSRs, aggregating and analyzing PGx data in PV biobanks will prompt the future discovery of novel pharmacogenetic biomarkers for the benefit of the greater population.</p></section><section id="cpt70095-sec-0021"><h2 class="pmc_sec_title">FUNDING</h2>
<p>A.M. is funded by NIGMS, grant #: R35GM152157. No financial support was received for this work.</p></section><section id="cpt70095-sec-0022"><h2 class="pmc_sec_title">CONFLICTS OF INTEREST</h2>
<p>A.M. is employed by Rocky Mountain Poison &amp; Drug Safety (RMPDS), a division of Denver Health &amp; Hospital Authority. RMPDS provides pharmacovigilance services to pharmaceutical companies. N.H., C.S., and A.O.‐O. are PGRN committee members. All other authors declare no competing interests for this work.</p></section><section id="cpt70095-sec-0020" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>We thank the Pharmacogenomics Global Research Network (PGRN) Publications Committee for their thoughtful review and feedback on this manuscript.</p></section><section id="cpt70095-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="cpt70095-bibl-0001_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="cpt70095-bib-0001">
<span class="label">1.</span><cite>
WHO
. What is pharmacovigilance? &lt;<a href="https://www.who.int/teams/regulation-prequalification/regulation-and-safety/pharmacovigilance" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.who.int/teams/regulation‐prequalification/regulation‐and‐safety/pharmacovigilance</a>&gt; Accessed February 6, 2025.</cite>
</li>
<li id="cpt70095-bib-0002">
<span class="label">2.</span><cite>
Centre UM
. About VigiBase. &lt;<a href="https://who-umc.org/vigibase/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://who‐umc.org/vigibase/</a>&gt; Accessed February 10, 2025.</cite>
</li>
<li id="cpt70095-bib-0003">
<span class="label">3.</span><cite>

Sibbald, B.

Rofecoxib (Vioxx) voluntarily withdrawn from market. Can. Med. Assoc. J.
171, 1027–1028 (2004).
</cite> [<a href="https://doi.org/10.1503/cmaj.1041606" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC526313/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15505253/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Can.%20Med.%20Assoc.%20J.&amp;title=Rofecoxib%20(Vioxx)%20voluntarily%20withdrawn%20from%20market&amp;author=B.%20Sibbald&amp;volume=171&amp;publication_year=2004&amp;pages=1027-1028&amp;pmid=15505253&amp;doi=10.1503/cmaj.1041606&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0004">
<span class="label">4.</span><cite>

Tanne, J.H.

FDA adds “black box” warning label to fluoroquinolone antibiotics. BMJ
337, a816 (2008).
</cite> [<a href="https://doi.org/10.1136/bmj.a816" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2483892/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18632714/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMJ&amp;title=FDA%20adds%20%E2%80%9Cblack%20box%E2%80%9D%20warning%20label%20to%20fluoroquinolone%20antibiotics&amp;author=J.H.%20Tanne&amp;volume=337&amp;publication_year=2008&amp;pages=a816&amp;pmid=18632714&amp;doi=10.1136/bmj.a816&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0005">
<span class="label">5.</span><cite>
FDA
. Table of pharmacogenetic associations &lt;<a href="https://www.fda.gov/medical-devices/precision-medicine/table-pharmacogenetic-associations" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.fda.gov/medical‐devices/precision‐medicine/table‐pharmacogenetic‐associations</a>&gt; (October 26, 2022). Accessed February 12, 2025.</cite>
</li>
<li id="cpt70095-bib-0006">
<span class="label">6.</span><cite>
Guidelines – CPIC. &lt;<a href="https://cpicpgx.org/guidelines/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cpicpgx.org/guidelines/</a>&gt; Accessed February 6, 2025.</cite>
</li>
<li id="cpt70095-bib-0007">
<span class="label">7.</span><cite>
ClinPGx
. Curation of DPWG content into ClinPGx. &lt;<a href="https://www.clinpgx.org/page/dpwgMapping" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.clinpgx.org/page/dpwgMapping</a>&gt; Accessed July 29, 2025.</cite>
</li>
<li id="cpt70095-bib-0008">
<span class="label">8.</span><cite>

Lazarou, J.
, 
Pomeranz, B.H.
 &amp; 
Corey, P.N.

Incidence of adverse drug reactions in hospitalized patients: a meta‐analysis of prospective studies. JAMA
279, 1200–1205 (1998).
</cite> [<a href="https://doi.org/10.1001/jama.279.15.1200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9555760/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Incidence%20of%20adverse%20drug%20reactions%20in%20hospitalized%20patients:%20a%20meta%E2%80%90analysis%20of%20prospective%20studies&amp;author=J.%20Lazarou&amp;author=B.H.%20Pomeranz&amp;author=P.N.%20Corey&amp;volume=279&amp;publication_year=1998&amp;pages=1200-1205&amp;pmid=9555760&amp;doi=10.1001/jama.279.15.1200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0009">
<span class="label">9.</span><cite>

Pavan, G.
, 
Kumar, M.
, 
Murti, K.
, 
Dhingra, S.
 &amp; 
Ravichandiran, V.

Exploring the factors influencing the health‐related quality of life in patients experiencing adverse drug reactions: a cross‐sectional study. J. Patient Rep. Outcomes
8, 112 (2024).
</cite> [<a href="https://doi.org/10.1186/s41687-024-00790-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11436490/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39333400/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Patient%20Rep.%20Outcomes&amp;title=Exploring%20the%20factors%20influencing%20the%20health%E2%80%90related%20quality%20of%20life%20in%20patients%20experiencing%20adverse%20drug%20reactions:%20a%20cross%E2%80%90sectional%20study&amp;author=G.%20Pavan&amp;author=M.%20Kumar&amp;author=K.%20Murti&amp;author=S.%20Dhingra&amp;author=V.%20Ravichandiran&amp;volume=8&amp;publication_year=2024&amp;pages=112&amp;pmid=39333400&amp;doi=10.1186/s41687-024-00790-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0010">
<span class="label">10.</span><cite>

Edwards, I.R.
 &amp; 
Aronson, J.K.

Adverse drug reactions: definitions, diagnosis, and management. Lancet
356, 1255–1259 (2000).
</cite> [<a href="https://doi.org/10.1016/S0140-6736(00)02799-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11072960/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Adverse%20drug%20reactions:%20definitions,%20diagnosis,%20and%20management&amp;author=I.R.%20Edwards&amp;author=J.K.%20Aronson&amp;volume=356&amp;publication_year=2000&amp;pages=1255-1259&amp;pmid=11072960&amp;doi=10.1016/S0140-6736(00)02799-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0011">
<span class="label">11.</span><cite>

Kommu, S.
, 
Carter, C.
 &amp; 
Whitfield, P.

Adverse drug reactions. In StatPearls (StatPearls Publishing; ). &lt;<a href="http://www.ncbi.nlm.nih.gov/books/NBK599521/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.ncbi.nlm.nih.gov/books/NBK599521/</a>&gt; (2025). Accessed January 24, 2025.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/38261714/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=StatPearls&amp;author=S.%20Kommu&amp;author=C.%20Carter&amp;author=P.%20Whitfield&amp;publication_year=2025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0012">
<span class="label">12.</span><cite>

Pirmohamed, M.

Pharmacogenetics of idiosyncratic adverse drug reactions. Handb. Exp. Pharmacol.
196, 477–491 (2010).</cite> [<a href="https://doi.org/10.1007/978-3-642-00663-0_17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20020273/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Handb.%20Exp.%20Pharmacol.&amp;title=Pharmacogenetics%20of%20idiosyncratic%20adverse%20drug%20reactions&amp;author=M.%20Pirmohamed&amp;volume=196&amp;publication_year=2010&amp;pages=477-491&amp;pmid=20020273&amp;doi=10.1007/978-3-642-00663-0_17&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0013">
<span class="label">13.</span><cite>

Jiang, Y.
 &amp; 
Gong, Y.

Enhancing safety via medication self‐management and timely reporting. Stud. Health Technol. Inform.
284, 463–468 (2021).
</cite> [<a href="https://doi.org/10.3233/SHTI210773" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8979547/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34920572/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Stud.%20Health%20Technol.%20Inform.&amp;title=Enhancing%20safety%20via%20medication%20self%E2%80%90management%20and%20timely%20reporting&amp;author=Y.%20Jiang&amp;author=Y.%20Gong&amp;volume=284&amp;publication_year=2021&amp;pages=463-468&amp;pmid=34920572&amp;doi=10.3233/SHTI210773&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0014">
<span class="label">14.</span><cite>

Lucas, S.
, 
Ailani, J.
, 
Smith, T.R.
, 
Abdrabboh, A.
, 
Xue, F.
 &amp; 
Navetta, M.S.

Pharmacovigilance: reporting requirements throughout a product’s lifecycle. Ther. Adv. Drug Saf.
13, 20420986221125006 (2022).
</cite> [<a href="https://doi.org/10.1177/20420986221125006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9520146/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36187302/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ther.%20Adv.%20Drug%20Saf.&amp;title=Pharmacovigilance:%20reporting%20requirements%20throughout%20a%20product%E2%80%99s%20lifecycle&amp;author=S.%20Lucas&amp;author=J.%20Ailani&amp;author=T.R.%20Smith&amp;author=A.%20Abdrabboh&amp;author=F.%20Xue&amp;volume=13&amp;publication_year=2022&amp;pages=20420986221125006&amp;pmid=36187302&amp;doi=10.1177/20420986221125006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0015">
<span class="label">15.</span><cite>
STATUTE‐76‐Pg780.pdf &lt;<a href="https://www.govinfo.gov/content/pkg/STATUTE-76/pdf/STATUTE-76-Pg780.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.govinfo.gov/content/pkg/STATUTE‐76/pdf/STATUTE‐76‐Pg780.pdf</a>&gt;.</cite>
</li>
<li id="cpt70095-bib-0016">
<span class="label">16.</span><cite>
Commissioner Office of the Milestones in U.S. Food and Drug Law, FDA &lt;<a href="https://www.fda.gov/about-fda/fda-history/milestones-us-food-and-drug-law" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.fda.gov/about‐fda/fda‐history/milestones‐us‐food‐and‐drug‐law</a>&gt; (January 30,2023). Accessed February 6, 2025.</cite>
</li>
<li id="cpt70095-bib-0017">
<span class="label">17.</span><cite>

Kim, J.H.
 &amp; 
Scialli, A.R.

Thalidomide: the tragedy of birth defects and the effective treatment of disease. Toxicol. Sci.
122, 1–6 (2011).
</cite> [<a href="https://doi.org/10.1093/toxsci/kfr088" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21507989/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Toxicol.%20Sci.&amp;title=Thalidomide:%20the%20tragedy%20of%20birth%20defects%20and%20the%20effective%20treatment%20of%20disease&amp;author=J.H.%20Kim&amp;author=A.R.%20Scialli&amp;volume=122&amp;publication_year=2011&amp;pages=1-6&amp;pmid=21507989&amp;doi=10.1093/toxsci/kfr088&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0018">
<span class="label">18.</span><cite>
United States Food and Drug Administration
. MedWatch: The FDA safety information and adverse event reporting program. &lt;<a href="https://www.fda.gov/safety/medwatch-fda-safety-information-and-adverse-event-reporting-program" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.fda.gov/safety/medwatch‐fda‐safety‐information‐and‐adverse‐event‐reporting‐program</a>&gt;. Accessed January 29, 2025.</cite>
</li>
<li id="cpt70095-bib-0019">
<span class="label">19.</span><cite>
FDA
. Research C for DE and FDA Adverse Event Reporting System (FAERS) Public Dashboard. &lt;<a href="https://www.fda.gov/drugs/fdas-adverse-event-reporting-system-faers/fda-adverse-event-reporting-system-faers-public-dashboard" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.fda.gov/drugs/fdas‐adverse‐event‐reporting‐system‐faers/fda‐adverse‐event‐reporting‐system‐faers‐public‐dashboard</a>&gt; (December 7,2023). Accessed February 6, 2025.</cite>
</li>
<li id="cpt70095-bib-0020">
<span class="label">20.</span><cite>

Rosenbaum, L.

The untold toll – the Pandemic’s effects on patients without Covid‐19. N. Engl. J. Med.
382, 2368–2371 (2020).
</cite> [<a href="https://doi.org/10.1056/NEJMms2009984" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32302076/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&amp;title=The%20untold%20toll%20%E2%80%93%20the%20Pandemic%E2%80%99s%20effects%20on%20patients%20without%20Covid%E2%80%9019&amp;author=L.%20Rosenbaum&amp;volume=382&amp;publication_year=2020&amp;pages=2368-2371&amp;pmid=32302076&amp;doi=10.1056/NEJMms2009984&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0021">
<span class="label">21.</span><cite>
Word Health Organization
. The Safety of medicines in public health programmes: pharmacovigilance an essential tool (2006) <a href="https://iris.who.int/bitstream/handle/10665/43384/9241593911_eng.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://iris.who.int/bitstream/handle/10665/43384/9241593911_eng.pdf</a>.</cite>
</li>
<li id="cpt70095-bib-0022">
<span class="label">22.</span><cite>

Sultana, J.
, 
Cutroneo, P.
 &amp; 
Trifirò, G.

Clinical and economic burden of adverse drug reactions. J. Pharmacol. Pharmacother.
4(Suppl 1), S73–S77 (2013).
</cite> [<a href="https://doi.org/10.4103/0976-500X.120957" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3853675/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24347988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Pharmacother.&amp;title=Clinical%20and%20economic%20burden%20of%20adverse%20drug%20reactions&amp;author=J.%20Sultana&amp;author=P.%20Cutroneo&amp;author=G.%20Trifir%C3%B2&amp;volume=4&amp;issue=Suppl%201&amp;publication_year=2013&amp;pages=S73-S77&amp;pmid=24347988&amp;doi=10.4103/0976-500X.120957&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0023">
<span class="label">23.</span><cite>
FDA
. E2D(R1) post‐approval safey data: definitions and standards for management and reporting of individual case study reports. &lt;<a href="https://www.fda.gov/media/176977/download" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.fda.gov/media/176977/download</a>&gt;. Accessed July 21, 2025.</cite>
</li>
<li id="cpt70095-bib-0024">
<span class="label">24.</span><cite>

Gliklich, R.E.
, 
Dreyer, N.A.
 &amp; 
Leavy, M.B.

Adverse event detection, processing, and reporting. In Registries for Evaluating Patient Outcomes: A User’s Guide
3rd edn. (Agency for Healthcare Research and Quality; ). &lt;<a href="https://www.ncbi.nlm.nih.gov/books/NBK208615/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/books/NBK208615/</a>&gt; (2014). Accessed February 10, 2025.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/24945055/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Registries%20for%20Evaluating%20Patient%20Outcomes:%20A%20User%E2%80%99s%20Guide&amp;author=R.E.%20Gliklich&amp;author=N.A.%20Dreyer&amp;author=M.B.%20Leavy&amp;publication_year=2014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0025">
<span class="label">25.</span><cite>

Postigo, R.

<em>et al</em>. EudraVigilance medicines safety database: publicly accessible data for research and public health protection. Drug Saf.
41, 665–675 (2018).
</cite> [<a href="https://doi.org/10.1007/s40264-018-0647-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5990579/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29520645/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Drug%20Saf.&amp;title=EudraVigilance%20medicines%20safety%20database:%20publicly%20accessible%20data%20for%20research%20and%20public%20health%20protection&amp;author=R.%20Postigo&amp;volume=41&amp;publication_year=2018&amp;pages=665-675&amp;pmid=29520645&amp;doi=10.1007/s40264-018-0647-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0026">
<span class="label">26.</span><cite>
Research C for DE and E2E Pharmacovigilance Planning. &lt;<a href="https://www.fda.gov/regulatory-information/search-fda-guidance-documents/e2e-pharmacovigilance-planning" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.fda.gov/regulatory‐information/search‐fda‐guidance‐documents/e2e‐pharmacovigilance‐planning</a>&gt; April 20, (2020). Accessed February 10, 2025.</cite>
</li>
<li id="cpt70095-bib-0027">
<span class="label">27.</span><cite>
European Medicines Agency (EMA)
. Pharmacovigilance: overview|European Medicines Agency (EMA). &lt;<a href="https://www.ema.europa.eu/en/human-regulatory-overview/pharmacovigilance-overview" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ema.europa.eu/en/human‐regulatory‐overview/pharmacovigilance‐overview</a>&gt; (April 19, 2022). Accessed February 6, 2025.</cite>
</li>
<li id="cpt70095-bib-0028">
<span class="label">28.</span><cite>
Snapshot. &lt;<a href="https://www.who.int/teams/regulation-prequalification/pharmacovigilance" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.who.int/teams/regulation‐prequalification/pharmacovigilance</a>&gt; Accessed February 6, 2025.</cite>
</li>
<li id="cpt70095-bib-0029">
<span class="label">29.</span><cite>
Good pharmacovigilance practices and pharmacoepidemiologic assessment. Published online (2025).</cite>
</li>
<li id="cpt70095-bib-0030">
<span class="label">30.</span><cite>

Jeong, Y.D.

<em>et al</em>. Global burden of vaccine‐associated Guillain‐Barré syndrome over 170 countries from 1967 to 2023. Sci. Rep.
14, 24561 (2024).
</cite> [<a href="https://doi.org/10.1038/s41598-024-74729-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11490553/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39427003/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Global%20burden%20of%20vaccine%E2%80%90associated%20Guillain%E2%80%90Barr%C3%A9%20syndrome%20over%20170%20countries%20from%201967%20to%202023&amp;author=Y.D.%20Jeong&amp;volume=14&amp;publication_year=2024&amp;pages=24561&amp;pmid=39427003&amp;doi=10.1038/s41598-024-74729-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0031">
<span class="label">31.</span><cite>

Li, M.F.
 &amp; 
Cheung, B.M.

Rise and fall of anti‐obesity drugs. World J. Diabetes
2, 19–23 (2011).
</cite> [<a href="https://doi.org/10.4239/wjd.v2.i2.19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3083904/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21537456/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=World%20J.%20Diabetes&amp;title=Rise%20and%20fall%20of%20anti%E2%80%90obesity%20drugs&amp;author=M.F.%20Li&amp;author=B.M.%20Cheung&amp;volume=2&amp;publication_year=2011&amp;pages=19-23&amp;pmid=21537456&amp;doi=10.4239/wjd.v2.i2.19&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0032">
<span class="label">32.</span><cite>

Hagan, J.B.
, 
Ender, E.
, 
Divekar, R.D.
, 
Pongdee, T.
 &amp; 
Rank, M.A.

Risk for Postmarket Black box warnings in FDA‐approved monoclonal antibodies. Mayo Clin Proc Innov Qual Outcomes.
6, 69–76 (2022).
</cite> [<a href="https://doi.org/10.1016/j.mayocpiqo.2021.11.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8724853/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35024565/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mayo%20Clin%20Proc%20Innov%20Qual%20Outcomes.&amp;title=Risk%20for%20Postmarket%20Black%20box%20warnings%20in%20FDA%E2%80%90approved%20monoclonal%20antibodies&amp;author=J.B.%20Hagan&amp;author=E.%20Ender&amp;author=R.D.%20Divekar&amp;author=T.%20Pongdee&amp;author=M.A.%20Rank&amp;volume=6&amp;publication_year=2022&amp;pages=69-76&amp;pmid=35024565&amp;doi=10.1016/j.mayocpiqo.2021.11.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0033">
<span class="label">33.</span><cite>
SEARCH Study Collaborative Group
, 
Bowman, L.
, 
Armitage, J.
, 
Bulbulia, R.
, 
Parish, S.
 &amp; 
Collins, R.

Study of the effectiveness of additional reductions in cholesterol and homocysteine (SEARCH): characteristics of a randomized trial among 12064 myocardial infarction survivors. Am. Heart J.
154, 815–823 (2007).
</cite> [<a href="https://doi.org/10.1016/j.ahj.2007.06.034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17967584/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Am.%20Heart%20J.&amp;title=Study%20of%20the%20effectiveness%20of%20additional%20reductions%20in%20cholesterol%20and%20homocysteine%20(SEARCH):%20characteristics%20of%20a%20randomized%20trial%20among%2012064%20myocardial%20infarction%20survivors&amp;author=L.%20Bowman&amp;author=J.%20Armitage&amp;author=R.%20Bulbulia&amp;author=S.%20Parish&amp;author=R.%20Collins&amp;volume=154&amp;publication_year=2007&amp;pages=815-823&amp;pmid=17967584&amp;doi=10.1016/j.ahj.2007.06.034&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0034">
<span class="label">34.</span><cite>

Norén, G.N.
 &amp; 
Edwards, I.R.

Modern methods of pharmacovigilance: detecting adverse effects of drugs. Clin. Med. (Lond.)
9, 486–489 (2009).
</cite> [<a href="https://doi.org/10.7861/clinmedicine.9-5-486" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4953463/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19886114/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Med.%20(Lond.)&amp;title=Modern%20methods%20of%20pharmacovigilance:%20detecting%20adverse%20effects%20of%20drugs&amp;author=G.N.%20Nor%C3%A9n&amp;author=I.R.%20Edwards&amp;volume=9&amp;publication_year=2009&amp;pages=486-489&amp;pmid=19886114&amp;doi=10.7861/clinmedicine.9-5-486&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0035">
<span class="label">35.</span><cite>

Golder, S.
, 
Xu, D.
, 
O’Connor, K.
, 
Wang, Y.
, 
Batra, M.
 &amp; 
Hernandez, G.G.

Leveraging natural language processing and machine learning methods for adverse drug event detection in electronic health/medical records: a scoping review. Drug Saf.
48, 321–337 (2025).
</cite> [<a href="https://doi.org/10.1007/s40264-024-01505-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11903561/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39786481/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Drug%20Saf.&amp;title=Leveraging%20natural%20language%20processing%20and%20machine%20learning%20methods%20for%20adverse%20drug%20event%20detection%20in%20electronic%20health/medical%20records:%20a%20scoping%20review&amp;author=S.%20Golder&amp;author=D.%20Xu&amp;author=K.%20O%E2%80%99Connor&amp;author=Y.%20Wang&amp;author=M.%20Batra&amp;volume=48&amp;publication_year=2025&amp;pages=321-337&amp;pmid=39786481&amp;doi=10.1007/s40264-024-01505-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0036">
<span class="label">36.</span><cite>

de Bie, S.

<em>et al</em>. The role of electronic healthcare record databases in paediatric drug safety surveillance: a retrospective cohort study. Br. J. Clin. Pharmacol.
80, 304–314 (2015).
</cite> [<a href="https://doi.org/10.1111/bcp.12610" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4541978/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25683723/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&amp;title=The%20role%20of%20electronic%20healthcare%20record%20databases%20in%20paediatric%20drug%20safety%20surveillance:%20a%20retrospective%20cohort%20study&amp;author=S.%20de%20Bie&amp;volume=80&amp;publication_year=2015&amp;pages=304-314&amp;pmid=25683723&amp;doi=10.1111/bcp.12610&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0037">
<span class="label">37.</span><cite>

Salvo, F.

<em>et al</em>. Will the future of pharmacovigilance be more automated?
Expert Opin. Drug Saf.
22, 541–548 (2023).
</cite> [<a href="https://doi.org/10.1080/14740338.2023.2227091" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37435796/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Expert%20Opin.%20Drug%20Saf.&amp;title=Will%20the%20future%20of%20pharmacovigilance%20be%20more%20automated?&amp;author=F.%20Salvo&amp;volume=22&amp;publication_year=2023&amp;pages=541-548&amp;pmid=37435796&amp;doi=10.1080/14740338.2023.2227091&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0038">
<span class="label">38.</span><cite>

Schuck, R.N.
 &amp; 
Grillo, J.A.

Pharmacogenomic biomarkers: an FDA perspective on utilization in biological product labeling. AAPS J.
18, 573–577 (2016).
</cite> [<a href="https://doi.org/10.1208/s12248-016-9891-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5256609/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26912182/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=AAPS%20J.&amp;title=Pharmacogenomic%20biomarkers:%20an%20FDA%20perspective%20on%20utilization%20in%20biological%20product%20labeling&amp;author=R.N.%20Schuck&amp;author=J.A.%20Grillo&amp;volume=18&amp;publication_year=2016&amp;pages=573-577&amp;pmid=26912182&amp;doi=10.1208/s12248-016-9891-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0039">
<span class="label">39.</span><cite>

Balian, J.D.
, 
Wherry, J.C.
, 
Malhotra, R.
 &amp; 
Perentesis, V.

Roadmap to risk evaluation and mitigation strategies (REMS) success. Ther. Adv. Drug Saf.
1, 21–38 (2010).
</cite> [<a href="https://doi.org/10.1177/2042098610381419" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4110798/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25083193/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ther.%20Adv.%20Drug%20Saf.&amp;title=Roadmap%20to%20risk%20evaluation%20and%20mitigation%20strategies%20(REMS)%20success&amp;author=J.D.%20Balian&amp;author=J.C.%20Wherry&amp;author=R.%20Malhotra&amp;author=V.%20Perentesis&amp;volume=1&amp;publication_year=2010&amp;pages=21-38&amp;pmid=25083193&amp;doi=10.1177/2042098610381419&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0040">
<span class="label">40.</span><cite>

Shinde, S.V.D.

Comparative review of pharmacovigilance system in different countries. Published online February 19, (2025). 10.5281/ZENODO.14892698.</cite> [<a href="https://doi.org/10.5281/ZENODO.14892698" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="cpt70095-bib-0041">
<span class="label">41.</span><cite>

Wakefield, J.
, 
Skrivankova, V.
, 
Hsu, F.C.
, 
Sale, M.
 &amp; 
Heagerty, P.

Detecting signals in pharmacogenomic genome‐wide association studies. Pharmacogenomics J.
14, 309–315 (2014).
</cite> [<a href="https://doi.org/10.1038/tpj.2013.44" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4085158/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24394200/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&amp;title=Detecting%20signals%20in%20pharmacogenomic%20genome%E2%80%90wide%20association%20studies&amp;author=J.%20Wakefield&amp;author=V.%20Skrivankova&amp;author=F.C.%20Hsu&amp;author=M.%20Sale&amp;author=P.%20Heagerty&amp;volume=14&amp;publication_year=2014&amp;pages=309-315&amp;pmid=24394200&amp;doi=10.1038/tpj.2013.44&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0042">
<span class="label">42.</span><cite>

Yue, Q.Y.

The International Society of Pharmacovigilance (ISoP) pharmacogenomic special interest group: pharmacogenomics in pharmacovigilance. Drug Saf.
44, 615–617 (2021).
</cite> [<a href="https://doi.org/10.1007/s40264-021-01068-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8184708/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33964001/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Drug%20Saf.&amp;title=The%20International%20Society%20of%20Pharmacovigilance%20(ISoP)%20pharmacogenomic%20special%20interest%20group:%20pharmacogenomics%20in%20pharmacovigilance&amp;author=Q.Y.%20Yue&amp;volume=44&amp;publication_year=2021&amp;pages=615-617&amp;pmid=33964001&amp;doi=10.1007/s40264-021-01068-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0043">
<span class="label">43.</span><cite>

Chanfreau‐Coffinier, C.

<em>et al</em>. Projected prevalence of actionable pharmacogenetic variants and level a drugs prescribed among US veterans health administration pharmacy users. JAMA Netw. Open
2, e195345 (2019).
</cite> [<a href="https://doi.org/10.1001/jamanetworkopen.2019.5345" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6563578/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31173123/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=JAMA%20Netw.%20Open&amp;title=Projected%20prevalence%20of%20actionable%20pharmacogenetic%20variants%20and%20level%20a%20drugs%20prescribed%20among%20US%20veterans%20health%20administration%20pharmacy%20users&amp;author=C.%20Chanfreau%E2%80%90Coffinier&amp;volume=2&amp;publication_year=2019&amp;pages=e195345&amp;pmid=31173123&amp;doi=10.1001/jamanetworkopen.2019.5345&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0044">
<span class="label">44.</span><cite>

Bush, W.

<em>et al</em>. Genetic variation among 82 pharmacogenes: the PGRNseq data from the eMERGE network. Clin. Pharmacol. Therap.
100, 160–169 (2016).
</cite> [<a href="https://doi.org/10.1002/cpt.350" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5010878/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26857349/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Therap.&amp;title=Genetic%20variation%20among%2082%20pharmacogenes:%20the%20PGRNseq%20data%20from%20the%20eMERGE%20network&amp;author=W.%20Bush&amp;volume=100&amp;publication_year=2016&amp;pages=160-169&amp;pmid=26857349&amp;doi=10.1002/cpt.350&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0045">
<span class="label">45.</span><cite>

Ji, Y.

<em>et al</em>. Preemptive pharmacogenomic testing for precision medicine. J. Mol. Diagn.
18, 438–445 (2016).
</cite> [<a href="https://doi.org/10.1016/j.jmoldx.2016.01.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4851731/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26947514/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Diagn.&amp;title=Preemptive%20pharmacogenomic%20testing%20for%20precision%20medicine&amp;author=Y.%20Ji&amp;volume=18&amp;publication_year=2016&amp;pages=438-445&amp;pmid=26947514&amp;doi=10.1016/j.jmoldx.2016.01.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0046">
<span class="label">46.</span><cite>

Van Driest, S.

<em>et al</em>. Clinically actionable genotypes among 10,000 patients with preemptive pharmacogenomic testing. Clin. Pharmacol. Ther.
95, 423–431 (2014).
</cite> [<a href="https://doi.org/10.1038/clpt.2013.229" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3961508/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24253661/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&amp;title=Clinically%20actionable%20genotypes%20among%2010,000%20patients%20with%20preemptive%20pharmacogenomic%20testing&amp;author=S.%20Van%20Driest&amp;volume=95&amp;publication_year=2014&amp;pages=423-431&amp;pmid=24253661&amp;doi=10.1038/clpt.2013.229&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0047">
<span class="label">47.</span><cite>

Aquilante, C.L.

<em>et al</em>. Clinical implementation of pharmacogenomics via a health system‐wide research biobank: the University of Colorado experience. Pharmacogenomics
21, 375–386 (2020).
</cite> [<a href="https://doi.org/10.2217/pgs-2020-0007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7226704/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32077359/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&amp;title=Clinical%20implementation%20of%20pharmacogenomics%20via%20a%20health%20system%E2%80%90wide%20research%20biobank:%20the%20University%20of%20Colorado%20experience&amp;author=C.L.%20Aquilante&amp;volume=21&amp;publication_year=2020&amp;pages=375-386&amp;pmid=32077359&amp;doi=10.2217/pgs-2020-0007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0048">
<span class="label">48.</span><cite>
Implementation – CPIC. &lt;<a href="https://cpicpgx.org/implementation/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cpicpgx.org/implementation/</a>&gt; Accessed February 6, 2025.</cite>
</li>
<li id="cpt70095-bib-0049">
<span class="label">49.</span><cite>
23andMe
. About us – 23andMe. &lt;<a href="https://www.23andme.com/about/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.23andme.com/about/</a>&gt; Accessed February 6, 2025.</cite>
</li>
<li id="cpt70095-bib-0050">
<span class="label">50.</span><cite>
23andMe
. Information about Pharmacogenetics reports –23andMe. &lt;<a href="https://www.23andme.com/test-info/pharmacogenetics/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.23andme.com/test‐info/pharmacogenetics/</a>&gt;. Accessed February 6, 2025.</cite>
</li>
<li id="cpt70095-bib-0051">
<span class="label">51.</span><cite>
Genes‐Drugs – CPIC. &lt;<a href="https://cpicpgx.org/genes-drugs/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cpicpgx.org/genes‐drugs/</a>&gt; Accessed February 6, 2025.</cite>
</li>
<li id="cpt70095-bib-0052">
<span class="label">52.</span><cite>

Oslin, D.W.

<em>et al</em>. Effect of pharmacogenomic testing for drug‐gene interactions on medication selection and remission of symptoms in major depressive disorder: the PRIME care randomized clinical trial. JAMA
328, 151–161 (2022).
</cite> [<a href="https://doi.org/10.1001/jama.2022.9805" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9277497/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35819423/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Effect%20of%20pharmacogenomic%20testing%20for%20drug%E2%80%90gene%20interactions%20on%20medication%20selection%20and%20remission%20of%20symptoms%20in%20major%20depressive%20disorder:%20the%20PRIME%20care%20randomized%20clinical%20trial&amp;author=D.W.%20Oslin&amp;volume=328&amp;publication_year=2022&amp;pages=151-161&amp;pmid=35819423&amp;doi=10.1001/jama.2022.9805&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0053">
<span class="label">53.</span><cite>
Diffusion marketplace. &lt;<a href="https://marketplace.va.gov/innovations/phaser" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://marketplace.va.gov/innovations/phaser</a>&gt; Accessed February 6, 2025.</cite>
</li>
<li id="cpt70095-bib-0054">
<span class="label">54.</span><cite>
FDA
. Research C for DE and table of pharmacogenomic biomarkers in drug labeling. &lt;<a href="https://www.fda.gov/drugs/science-and-research-drugs/table-pharmacogenomic-biomarkers-drug-labeling" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.fda.gov/drugs/science‐and‐research‐drugs/table‐pharmacogenomic‐biomarkers‐drug‐labeling</a>&gt; (September 23, 2024). Accessed February 6, 2025.</cite>
</li>
<li id="cpt70095-bib-0055">
<span class="label">55.</span><cite>
Research C for DE and pharmacogenomic data submissions. &lt;<a href="https://www.fda.gov/regulatory-information/search-fda-guidance-documents/pharmacogenomic-data-submissions" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.fda.gov/regulatory‐information/search‐fda‐guidance‐documents/pharmacogenomic‐data‐submissions</a>&gt; (April 27, 2020). Accessed February 6, 2025.</cite>
</li>
<li id="cpt70095-bib-0056">
<span class="label">56.</span><cite>

Aronson, J.K.
 &amp; 
Ferner, R.E.

Joining the DoTS: new approach to classifying adverse drug reactions. BMJ
327, 1222–1225 (2003).
</cite> [<a href="https://doi.org/10.1136/bmj.327.7425.1222" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC274067/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14630763/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMJ&amp;title=Joining%20the%20DoTS:%20new%20approach%20to%20classifying%20adverse%20drug%20reactions&amp;author=J.K.%20Aronson&amp;author=R.E.%20Ferner&amp;volume=327&amp;publication_year=2003&amp;pages=1222-1225&amp;pmid=14630763&amp;doi=10.1136/bmj.327.7425.1222&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0057">
<span class="label">57.</span><cite>

Clark, D.W.J.
, 
Donnelly, E.
, 
Coulter, D.M.
, 
Roberts, R.L.
 &amp; 
Kennedy, M.A.

Linking pharmacovigilance with pharmacogenetics. Drug Saf.
27, 1171–1184 (2004).
</cite> [<a href="https://doi.org/10.2165/00002018-200427150-00002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15588114/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Drug%20Saf.&amp;title=Linking%20pharmacovigilance%20with%20pharmacogenetics&amp;author=D.W.J.%20Clark&amp;author=E.%20Donnelly&amp;author=D.M.%20Coulter&amp;author=R.L.%20Roberts&amp;author=M.A.%20Kennedy&amp;volume=27&amp;publication_year=2004&amp;pages=1171-1184&amp;pmid=15588114&amp;doi=10.2165/00002018-200427150-00002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0058">
<span class="label">58.</span><cite>

Koh, Y.
, 
Yap, C.W.
 &amp; 
Li, S.C.

A quantitative approach of using genetic algorithm in designing a probability scoring system of an adverse drug reaction assessment system. Int. J. Med. Inform.
77, 421–430 (2008).
</cite> [<a href="https://doi.org/10.1016/j.ijmedinf.2007.08.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17921048/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Med.%20Inform.&amp;title=A%20quantitative%20approach%20of%20using%20genetic%20algorithm%20in%20designing%20a%20probability%20scoring%20system%20of%20an%20adverse%20drug%20reaction%20assessment%20system&amp;author=Y.%20Koh&amp;author=C.W.%20Yap&amp;author=S.C.%20Li&amp;volume=77&amp;publication_year=2008&amp;pages=421-430&amp;pmid=17921048&amp;doi=10.1016/j.ijmedinf.2007.08.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0059">
<span class="label">59.</span><cite>

Potter, E.
, 
Reyes, M.
, 
Naples, J.
 &amp; 
Dal Pan, G.

FDA adverse event reporting system (FAERS) essentials: a guide to understanding, applying, and interpreting adverse event data reported to FAERS. Clin. Pharmacol. Ther.
118, 567–582 (2025).
</cite> [<a href="https://doi.org/10.1002/cpt.3701" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12393772/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40384638/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&amp;title=FDA%20adverse%20event%20reporting%20system%20(FAERS)%20essentials:%20a%20guide%20to%20understanding,%20applying,%20and%20interpreting%20adverse%20event%20data%20reported%20to%20FAERS&amp;author=E.%20Potter&amp;author=M.%20Reyes&amp;author=J.%20Naples&amp;author=G.%20Dal%20Pan&amp;volume=118&amp;publication_year=2025&amp;pages=567-582&amp;pmid=40384638&amp;doi=10.1002/cpt.3701&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0060">
<span class="label">60.</span><cite>
Good pharmacovigilance practices and pharmacoepidemiologic assessment. &lt;<a href="https://www.fda.gov/regulatory-information/search-fda-guidance-documents/good-pharmacovigilance-practices-and-pharmacoepidemiologic-assessment" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.fda.gov/regulatory‐information/search‐fda‐guidance‐documents/good‐pharmacovigilance‐practices‐and‐pharmacoepidemiologic‐assessment</a>&gt; Accessed July 22, 2025.</cite>
</li>
<li id="cpt70095-bib-0061">
<span class="label">61.</span><cite>

Chung, W.H.
, 
Hung, S.I.
 &amp; 
Chen, Y.T.

Human leukocyte antigens and drug hypersensitivity. Curr. Opin. Allergy Clin. Immunol.
7, 317–323 (2007).
</cite> [<a href="https://doi.org/10.1097/ACI.0b013e3282370c5f" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17620823/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Allergy%20Clin.%20Immunol.&amp;title=Human%20leukocyte%20antigens%20and%20drug%20hypersensitivity&amp;author=W.H.%20Chung&amp;author=S.I.%20Hung&amp;author=Y.T.%20Chen&amp;volume=7&amp;publication_year=2007&amp;pages=317-323&amp;pmid=17620823&amp;doi=10.1097/ACI.0b013e3282370c5f&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0062">
<span class="label">62.</span><cite>

Ehmann, F.
, 
Caneva, L.
 &amp; 
Papaluca, M.

European medicines agency initiatives and perspectives on pharmacogenomics. Br. J. Clin. Pharmacol.
77, 612–617 (2014).
</cite> [<a href="https://doi.org/10.1111/bcp.12319" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3971978/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24433361/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&amp;title=European%20medicines%20agency%20initiatives%20and%20perspectives%20on%20pharmacogenomics&amp;author=F.%20Ehmann&amp;author=L.%20Caneva&amp;author=M.%20Papaluca&amp;volume=77&amp;publication_year=2014&amp;pages=612-617&amp;pmid=24433361&amp;doi=10.1111/bcp.12319&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0063">
<span class="label">63.</span><cite>

Rogers, S.

<em>et al</em>. Case report: life‐threatening fluoxetine‐linked postoperative bleeding informed by pharmacogenetic evaluation. Drugs R D.
24, 117–121 (2024).
</cite> [<a href="https://doi.org/10.1007/s40268-023-00451-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11035520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38413481/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Drugs%20R%20D.&amp;title=Case%20report:%20life%E2%80%90threatening%20fluoxetine%E2%80%90linked%20postoperative%20bleeding%20informed%20by%20pharmacogenetic%20evaluation&amp;author=S.%20Rogers&amp;volume=24&amp;publication_year=2024&amp;pages=117-121&amp;pmid=38413481&amp;doi=10.1007/s40268-023-00451-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0064">
<span class="label">64.</span><cite>
CARE guidelines. &lt;<a href="https://www.care-statement.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.care‐statement.org/</a>&gt; Accessed September 7, 2025.</cite>
</li>
<li id="cpt70095-bib-0065">
<span class="label">65.</span><cite>
ClinPGx. &lt;<a href="https://www.clinpgx.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.clinpgx.org/</a>&gt; Accessed July 29, 2025.</cite>
</li>
<li id="cpt70095-bib-0066">
<span class="label">66.</span><cite>

Brockmöller, J.
 &amp; 
Tzvetkov, M.V.

Pharmacogenetics: data, concepts and tools to improve drug discovery and drug treatment. Eur. J. Clin. Pharmacol.
64, 133–157 (2008).
</cite> [<a href="https://doi.org/10.1007/s00228-007-0424-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2235910/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18224312/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Clin.%20Pharmacol.&amp;title=Pharmacogenetics:%20data,%20concepts%20and%20tools%20to%20improve%20drug%20discovery%20and%20drug%20treatment&amp;author=J.%20Brockm%C3%B6ller&amp;author=M.V.%20Tzvetkov&amp;volume=64&amp;publication_year=2008&amp;pages=133-157&amp;pmid=18224312&amp;doi=10.1007/s00228-007-0424-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0067">
<span class="label">67.</span><cite>

Maggo, S.D.S.
, 
Savage, R.L.
 &amp; 
Kennedy, M.A.

Impact of new genomic technologies on understanding adverse drug reactions. Clin. Pharmacokinet.
55, 419–436 (2016).
</cite> [<a href="https://doi.org/10.1007/s40262-015-0324-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4791456/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26369774/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacokinet.&amp;title=Impact%20of%20new%20genomic%20technologies%20on%20understanding%20adverse%20drug%20reactions&amp;author=S.D.S.%20Maggo&amp;author=R.L.%20Savage&amp;author=M.A.%20Kennedy&amp;volume=55&amp;publication_year=2016&amp;pages=419-436&amp;pmid=26369774&amp;doi=10.1007/s40262-015-0324-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0068">
<span class="label">68.</span><cite>

Pratt, V.M.

<em>et al</em>. Recommendations for clinical CYP2D6 genotyping allele selection. J. Mol. Diagn.
23, 1047–1064 (2021).
</cite> [<a href="https://doi.org/10.1016/j.jmoldx.2021.05.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8579245/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34118403/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Diagn.&amp;title=Recommendations%20for%20clinical%20CYP2D6%20genotyping%20allele%20selection&amp;author=V.M.%20Pratt&amp;volume=23&amp;publication_year=2021&amp;pages=1047-1064&amp;pmid=34118403&amp;doi=10.1016/j.jmoldx.2021.05.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0069">
<span class="label">69.</span><cite>

Tayeh, M.K.

<em>et al</em>. Clinical pharmacogenomic testing and reporting: a technical standard of the American College of Medical Genetics and Genomics (ACMG). Genet. Med.
24, 759–768 (2022).
</cite> [<a href="https://doi.org/10.1016/j.gim.2021.12.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35177334/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genet.%20Med.&amp;title=Clinical%20pharmacogenomic%20testing%20and%20reporting:%20a%20technical%20standard%20of%20the%20American%20College%20of%20Medical%20Genetics%20and%20Genomics%20(ACMG)&amp;author=M.K.%20Tayeh&amp;volume=24&amp;publication_year=2022&amp;pages=759-768&amp;pmid=35177334&amp;doi=10.1016/j.gim.2021.12.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0070">
<span class="label">70.</span><cite>

Tewhey, R.
, 
Bansal, V.
, 
Torkamani, A.
, 
Topol, E.J.
 &amp; 
Schork, N.J.

The importance of phase information for human genomics. Nat. Rev. Genet.
12, 215–223 (2011).
</cite> [<a href="https://doi.org/10.1038/nrg2950" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3753045/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21301473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&amp;title=The%20importance%20of%20phase%20information%20for%20human%20genomics&amp;author=R.%20Tewhey&amp;author=V.%20Bansal&amp;author=A.%20Torkamani&amp;author=E.J.%20Topol&amp;author=N.J.%20Schork&amp;volume=12&amp;publication_year=2011&amp;pages=215-223&amp;pmid=21301473&amp;doi=10.1038/nrg2950&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0071">
<span class="label">71.</span><cite>
PacBio
. Sequencing 101: ploidy, haplotypes, and phasing — how to get more from your sequencing data. PacBio. &lt;<a href="https://www.pacb.com/blog/ploidy-haplotypes-and-phasing/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.pacb.com/blog/ploidy‐haplotypes‐and‐phasing/</a>&gt; (December 10, 2020). Accessed February 10, 2025.</cite>
</li>
<li id="cpt70095-bib-0072">
<span class="label">72.</span><cite>
Ultima genomics delivers the $100 genome. Emerges from stealth with $600 million in funding. &lt;<a href="https://www.ultimagenomics.com/blog/ultima-genomics-delivers-the-usd100-genome/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ultimagenomics.com/blog/ultima‐genomics‐delivers‐the‐usd100‐genome/</a>&gt; Accessed July 29, 2025.</cite>
</li>
<li id="cpt70095-bib-0073">
<span class="label">73.</span><cite>
AMP’s minimum sets of alleles for PGx testing. ClinPGx. &lt;<a href="https://www.clinpgx.org/ampAllelesToTest" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.clinpgx.org/ampAllelesToTest</a>&gt; Accessed July 29, 2025.</cite>
</li>
<li id="cpt70095-bib-0074">
<span class="label">74.</span><cite>
Best practices for FDA staff in the postmarketing safety surveillance of human drug and biological products. &lt;<a href="https://www.fda.gov/media/130216/download?attachment" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.fda.gov/media/130216/download?attachment</a>&gt; Accessed July 22, 2025.</cite>
</li>
<li id="cpt70095-bib-0075">
<span class="label">75.</span><cite>

Nebert, D.W.

Extreme discordant phenotype methodology: an intuitive approach to clinical pharmacogenetics. Eur. J. Pharmacol.
410, 107–120 (2000).
</cite> [<a href="https://doi.org/10.1016/s0014-2999(00)00809-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11134663/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Pharmacol.&amp;title=Extreme%20discordant%20phenotype%20methodology:%20an%20intuitive%20approach%20to%20clinical%20pharmacogenetics&amp;author=D.W.%20Nebert&amp;volume=410&amp;publication_year=2000&amp;pages=107-120&amp;pmid=11134663&amp;doi=10.1016/s0014-2999(00)00809-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0076">
<span class="label">76.</span><cite>
Allele Frequencies. &lt;<a href="https://www.clinpgx.org/alleleFrequency" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.clinpgx.org/alleleFrequency</a>&gt; Accessed July 29, 2025.</cite>
</li>
<li id="cpt70095-bib-0077">
<span class="label">77.</span><cite>

De Falco, V.

<em>et al</em>. A case report of a severe fluoropyrimidine‐related toxicity due to an uncommon DPYD variant. Medicine (Baltimore)
98, e15759 (2019).
</cite> [<a href="https://doi.org/10.1097/MD.0000000000015759" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6571425/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31124962/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Medicine%20(Baltimore)&amp;title=A%20case%20report%20of%20a%20severe%20fluoropyrimidine%E2%80%90related%20toxicity%20due%20to%20an%20uncommon%20DPYD%20variant&amp;author=V.%20De%20Falco&amp;volume=98&amp;publication_year=2019&amp;pages=e15759&amp;pmid=31124962&amp;doi=10.1097/MD.0000000000015759&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0078">
<span class="label">78.</span><cite>

Diao, X.

<em>et al</em>. Automated ICD coding for primary diagnosis via clinically interpretable machine learning. Int. J. Med. Inform.
153, 104543 (2021).
</cite> [<a href="https://doi.org/10.1016/j.ijmedinf.2021.104543" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34391016/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Med.%20Inform.&amp;title=Automated%20ICD%20coding%20for%20primary%20diagnosis%20via%20clinically%20interpretable%20machine%20learning&amp;author=X.%20Diao&amp;volume=153&amp;publication_year=2021&amp;pages=104543&amp;pmid=34391016&amp;doi=10.1016/j.ijmedinf.2021.104543&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0079">
<span class="label">79.</span><cite>

Kompa, B.

<em>et al</em>. Artificial intelligence based on machine learning in pharmacovigilance: a scoping review. Drug Saf.
45, 477–491 (2022).
</cite> [<a href="https://doi.org/10.1007/s40264-022-01176-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9883349/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35579812/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Drug%20Saf.&amp;title=Artificial%20intelligence%20based%20on%20machine%20learning%20in%20pharmacovigilance:%20a%20scoping%20review&amp;author=B.%20Kompa&amp;volume=45&amp;publication_year=2022&amp;pages=477-491&amp;pmid=35579812&amp;doi=10.1007/s40264-022-01176-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0080">
<span class="label">80.</span><cite>

Caster, O.
, 
Aoki, Y.
, 
Gattepaille, L.M.
 &amp; 
Grundmark, B.

Disproportionality analysis for pharmacovigilance signal detection in small databases or subsets: recommendations for limiting false‐positive associations. Drug Saf.
43, 479–487 (2020).
</cite> [<a href="https://doi.org/10.1007/s40264-020-00911-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7165139/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32008183/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Drug%20Saf.&amp;title=Disproportionality%20analysis%20for%20pharmacovigilance%20signal%20detection%20in%20small%20databases%20or%20subsets:%20recommendations%20for%20limiting%20false%E2%80%90positive%20associations&amp;author=O.%20Caster&amp;author=Y.%20Aoki&amp;author=L.M.%20Gattepaille&amp;author=B.%20Grundmark&amp;volume=43&amp;publication_year=2020&amp;pages=479-487&amp;pmid=32008183&amp;doi=10.1007/s40264-020-00911-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0081">
<span class="label">81.</span><cite>

Li, S.

<em>et al</em>. BiMPADR: a deep learning framework for predicting adverse drug reactions in new drugs. Molecules
29, 1784 (2024).
</cite> [<a href="https://doi.org/10.3390/molecules29081784" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11051887/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38675604/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=BiMPADR:%20a%20deep%20learning%20framework%20for%20predicting%20adverse%20drug%20reactions%20in%20new%20drugs&amp;author=S.%20Li&amp;volume=29&amp;publication_year=2024&amp;pages=1784&amp;pmid=38675604&amp;doi=10.3390/molecules29081784&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0082">
<span class="label">82.</span><cite>

Abdelhalim, H.

<em>et al</em>. Artificial intelligence, healthcare, clinical genomics, and pharmacogenomics approaches in precision medicine. Front. Genet.
13, 929736 (2022).
</cite> [<a href="https://doi.org/10.3389/fgene.2022.929736" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9299079/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35873469/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Genet.&amp;title=Artificial%20intelligence,%20healthcare,%20clinical%20genomics,%20and%20pharmacogenomics%20approaches%20in%20precision%20medicine&amp;author=H.%20Abdelhalim&amp;volume=13&amp;publication_year=2022&amp;pages=929736&amp;pmid=35873469&amp;doi=10.3389/fgene.2022.929736&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0083">
<span class="label">83.</span><cite>

Salas, M.

<em>et al</em>. The use of artificial intelligence in pharmacovigilance: a systematic review of the literature. Pharmaceut Med.
36, 295–306 (2022).
</cite> [<a href="https://doi.org/10.1007/s40290-022-00441-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35904529/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmaceut%20Med.&amp;title=The%20use%20of%20artificial%20intelligence%20in%20pharmacovigilance:%20a%20systematic%20review%20of%20the%20literature&amp;author=M.%20Salas&amp;volume=36&amp;publication_year=2022&amp;pages=295-306&amp;pmid=35904529&amp;doi=10.1007/s40290-022-00441-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0084">
<span class="label">84.</span><cite>

Bate, A.
 &amp; 
Luo, Y.

Artificial intelligence and machine learning for safe medicines. Drug Saf.
45, 403–405 (2022).
</cite> [<a href="https://doi.org/10.1007/s40264-022-01177-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9112276/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35579805/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Drug%20Saf.&amp;title=Artificial%20intelligence%20and%20machine%20learning%20for%20safe%20medicines&amp;author=A.%20Bate&amp;author=Y.%20Luo&amp;volume=45&amp;publication_year=2022&amp;pages=403-405&amp;pmid=35579805&amp;doi=10.1007/s40264-022-01177-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0085">
<span class="label">85.</span><cite>

Glaser, M.
 &amp; 
Littlebury, R.

Governance of artificial intelligence and machine learning in pharmacovigilance: what works today and what more is needed?
Ther. Adv. Drug Saf.
15, 20420986241293303 (2024).
</cite> [<a href="https://doi.org/10.1177/20420986241293303" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11528645/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39493927/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ther.%20Adv.%20Drug%20Saf.&amp;title=Governance%20of%20artificial%20intelligence%20and%20machine%20learning%20in%20pharmacovigilance:%20what%20works%20today%20and%20what%20more%20is%20needed?&amp;author=M.%20Glaser&amp;author=R.%20Littlebury&amp;volume=15&amp;publication_year=2024&amp;pages=20420986241293303&amp;pmid=39493927&amp;doi=10.1177/20420986241293303&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0086">
<span class="label">86.</span><cite>

Siamoglou, S.
, 
Koromina, M.
, 
Moy, F.M.
, 
Mitropoulou, C.
, 
Patrinos, G.P.
 &amp; 
Vasileiou, K.

What do students in pharmacy and medicine think about pharmacogenomics and personalized medicine education? Awareness, attitudes, and perceptions in Malaysian health sciences. OMICS
25, 52–59 (2021).
</cite> [<a href="https://doi.org/10.1089/omi.2020.0178" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33170085/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=OMICS&amp;title=What%20do%20students%20in%20pharmacy%20and%20medicine%20think%20about%20pharmacogenomics%20and%20personalized%20medicine%20education?%20Awareness,%20attitudes,%20and%20perceptions%20in%20Malaysian%20health%20sciences&amp;author=S.%20Siamoglou&amp;author=M.%20Koromina&amp;author=F.M.%20Moy&amp;author=C.%20Mitropoulou&amp;author=G.P.%20Patrinos&amp;volume=25&amp;publication_year=2021&amp;pages=52-59&amp;pmid=33170085&amp;doi=10.1089/omi.2020.0178&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0087">
<span class="label">87.</span><cite>

Bright, D.
, 
Petry, N.
, 
Roath, E.
, 
Reckow, E.
 &amp; 
Chavour, S.

Barriers, solutions, and effect of using pharmacogenomics data to support opioid prescribing. J. Manag. Care Spec. Pharm.
26, 1597–1602 (2020).
</cite> [<a href="https://doi.org/10.18553/jmcp.2020.26.12.1597" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10390958/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33252002/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Manag.%20Care%20Spec.%20Pharm.&amp;title=Barriers,%20solutions,%20and%20effect%20of%20using%20pharmacogenomics%20data%20to%20support%20opioid%20prescribing&amp;author=D.%20Bright&amp;author=N.%20Petry&amp;author=E.%20Roath&amp;author=E.%20Reckow&amp;author=S.%20Chavour&amp;volume=26&amp;publication_year=2020&amp;pages=1597-1602&amp;pmid=33252002&amp;doi=10.18553/jmcp.2020.26.12.1597&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0088">
<span class="label">88.</span><cite>

Berm, E.J.J.

<em>et al</em>. Economic evaluations of pharmacogenetic and pharmacogenomic screening tests: a systematic review. Second update of the literature. PLoS One
11, e0146262 (2016).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0146262" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4709231/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26752539/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Economic%20evaluations%20of%20pharmacogenetic%20and%20pharmacogenomic%20screening%20tests:%20a%20systematic%20review.%20Second%20update%20of%20the%20literature&amp;author=E.J.J.%20Berm&amp;volume=11&amp;publication_year=2016&amp;pages=e0146262&amp;pmid=26752539&amp;doi=10.1371/journal.pone.0146262&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0089">
<span class="label">89.</span><cite>

Proietti, R.

<em>et al</em>. Pharmacogenomic profile of a central European urban random population‐Czech population. PLoS One
18, e0284386 (2023).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0284386" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10118108/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37079615/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Pharmacogenomic%20profile%20of%20a%20central%20European%20urban%20random%20population%E2%80%90Czech%20population&amp;author=R.%20Proietti&amp;volume=18&amp;publication_year=2023&amp;pages=e0284386&amp;pmid=37079615&amp;doi=10.1371/journal.pone.0284386&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0090">
<span class="label">90.</span><cite>

Alessandrini, M.
, 
Chaudhry, M.
, 
Dodgen, T.M.
 &amp; 
Pepper, M.S.

Pharmacogenomics and global precision medicine in the context of adverse drug reactions: top 10 opportunities and challenges for the next decade. OMICS
20, 593–603 (2016).
</cite> [<a href="https://doi.org/10.1089/omi.2016.0122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5072285/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27643672/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=OMICS&amp;title=Pharmacogenomics%20and%20global%20precision%20medicine%20in%20the%20context%20of%20adverse%20drug%20reactions:%20top%2010%20opportunities%20and%20challenges%20for%20the%20next%20decade&amp;author=M.%20Alessandrini&amp;author=M.%20Chaudhry&amp;author=T.M.%20Dodgen&amp;author=M.S.%20Pepper&amp;volume=20&amp;publication_year=2016&amp;pages=593-603&amp;pmid=27643672&amp;doi=10.1089/omi.2016.0122&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0091">
<span class="label">91.</span><cite>

Franc, M.A.

<em>et al</em>. Coding of DNA samples and data in the pharmaceutical industry: current practices and future directions‐‐perspective of the I‐PWG. Clin. Pharmacol. Ther.
89, 537–545 (2011).
</cite> [<a href="https://doi.org/10.1038/clpt.2010.306" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21346752/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&amp;title=Coding%20of%20DNA%20samples%20and%20data%20in%20the%20pharmaceutical%20industry:%20current%20practices%20and%20future%20directions%E2%80%90%E2%80%90perspective%20of%20the%20I%E2%80%90PWG&amp;author=M.A.%20Franc&amp;volume=89&amp;publication_year=2011&amp;pages=537-545&amp;pmid=21346752&amp;doi=10.1038/clpt.2010.306&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cpt70095-bib-0092">
<span class="label">92.</span><cite>

Hicks, J.K.

<em>et al</em>. A call for clear and consistent communications regarding the role of pharmacogenetics in antidepressant pharmacotherapy. Clin. Pharmacol. Ther.
107, 50–52 (2020).
</cite> [<a href="https://doi.org/10.1002/cpt.1661" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6925627/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31664715/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&amp;title=A%20call%20for%20clear%20and%20consistent%20communications%20regarding%20the%20role%20of%20pharmacogenetics%20in%20antidepressant%20pharmacotherapy&amp;author=J.K.%20Hicks&amp;volume=107&amp;publication_year=2020&amp;pages=50-52&amp;pmid=31664715&amp;doi=10.1002/cpt.1661&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>While PGx data have become more available, not all patients are genotyped. PGx tests can and should be recommended as part of a causal assessment when data support clinical implementation. Patients can request such testing, which is increasingly reimbursed by insurers, but may not realize they already have access to these data through direct‐to‐consumer (DTC) providers. Thus, both patients and their treating clinicians must be queried about the availability of these results through clinical laboratories or DTC providers. Patients may be referred for testing when the clinical case suggests the need for these results.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Clinical Pharmacology and Therapeutics are provided here courtesy of <strong>Wiley and American Society for Clinical Pharmacology and Therapeutics</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/cpt.70095"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/CPT-118-1286.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (641.2 KB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12598900/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12598900/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12598900%2F%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12598900/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12598900/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12598900/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/41146533/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12598900/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/41146533/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12598900/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12598900/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="UOeKRbAuvj3r6Ekbagj2LZK7r2cDM8Jl89LCgjFwCSqQe3jOd0LhFYdujdWyCFSB">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-B7-tgZZA.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-BLbQGH8u.js"></script>
    
    

    </body>
</html>
