
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/base_style-CPPDhfna.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-BepKMsUl.js"></script>

  <link rel="stylesheet" href="/static/assets/article_style-DUfI1jZp.css" />
<link rel="stylesheet" href="/static/assets/style-BOrsu84o.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-D3wJHomh.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Explainable artificial intelligence of DNA methylation-based brain tumor diagnostics - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="E0CA2651930B2FC3002651000AD265B3.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11842776/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Explainable artificial intelligence of DNA methylation-based brain tumor diagnostics">
<meta name="citation_author" content="Salvatore Benfatto">
<meta name="citation_author_institution" content="Department of Pediatric Oncology, Dana-Farber Cancer Institute, Boston, MA USA">
<meta name="citation_author_institution" content="Division of Hematology/Oncology, Boston Children’s Hospital, Boston, MA USA">
<meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA USA">
<meta name="citation_author" content="Martin Sill">
<meta name="citation_author_institution" content="Division of Pediatric Neurooncology, Hopp Children’s Cancer Center (KiTZ), Heidelberg, Germany">
<meta name="citation_author_institution" content="National Center for Tumor Diseases (NCT), NCT Heidelberg, a partnership between DKFZ and Heidelberg University Hospital, Heidelberg, Germany">
<meta name="citation_author_institution" content="German Cancer Research Center (DKFZ) and German Cancer Consortium (DKTK), Heidelberg, Germany">
<meta name="citation_author" content="David T W Jones">
<meta name="citation_author_institution" content="National Center for Tumor Diseases (NCT), NCT Heidelberg, a partnership between DKFZ and Heidelberg University Hospital, Heidelberg, Germany">
<meta name="citation_author_institution" content="German Cancer Research Center (DKFZ) and German Cancer Consortium (DKTK), Heidelberg, Germany">
<meta name="citation_author_institution" content="Division of Pediatric Glioma Research, Hopp Children’s Cancer Center (KiTZ), Heidelberg, Germany">
<meta name="citation_author" content="Stefan M Pfister">
<meta name="citation_author_institution" content="Division of Pediatric Neurooncology, Hopp Children’s Cancer Center (KiTZ), Heidelberg, Germany">
<meta name="citation_author_institution" content="National Center for Tumor Diseases (NCT), NCT Heidelberg, a partnership between DKFZ and Heidelberg University Hospital, Heidelberg, Germany">
<meta name="citation_author_institution" content="German Cancer Research Center (DKFZ) and German Cancer Consortium (DKTK), Heidelberg, Germany">
<meta name="citation_author_institution" content="Department of Pediatric Oncology, Hematology &amp; Immunology, Heidelberg University Hospital, Heidelberg, Germany">
<meta name="citation_author" content="Felix Sahm">
<meta name="citation_author_institution" content="Department of Neuropathology, Heidelberg University Hospital, Heidelberg, Germany">
<meta name="citation_author_institution" content="Clinical Cooperation Unit Neuropathology, German Cancer Research Center (DKFZ) and German Cancer Consortium (DKTK), Heidelberg, Germany">
<meta name="citation_author" content="Andreas von Deimling">
<meta name="citation_author_institution" content="Department of Neuropathology, Heidelberg University Hospital, Heidelberg, Germany">
<meta name="citation_author_institution" content="Clinical Cooperation Unit Neuropathology, German Cancer Research Center (DKFZ) and German Cancer Consortium (DKTK), Heidelberg, Germany">
<meta name="citation_author" content="David Capper">
<meta name="citation_author_institution" content="Department of Neuropathology, Charité - Universitätsmedizin Berlin, corporate member of Freie Universität Berlin and Humboldt-Universität zu Berlin, Berlin, Germany">
<meta name="citation_author_institution" content="German Cancer Consortium (DKTK), Partner Site Berlin, German Cancer Research Center (DKFZ), Heidelberg, Germany">
<meta name="citation_author" content="Volker Hovestadt">
<meta name="citation_author_institution" content="Department of Pediatric Oncology, Dana-Farber Cancer Institute, Boston, MA USA">
<meta name="citation_author_institution" content="Division of Hematology/Oncology, Boston Children’s Hospital, Boston, MA USA">
<meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA USA">
<meta name="citation_publication_date" content="2025 Feb 20">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="1787">
<meta name="citation_doi" content="10.1038/s41467-025-57078-0">
<meta name="citation_pmid" content="39979307">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11842776/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11842776/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11842776/pdf/41467_2025_Article_57078.pdf">
<meta name="description" content="We have recently developed a machine learning classifier that enables fast, accurate, and affordable classification of brain tumors based on genome-wide DNA methylation profiles that is widely employed in the clinic. Neuro-oncology research would ...">
<meta name="og:title" content="Explainable artificial intelligence of DNA methylation-based brain tumor diagnostics">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="We have recently developed a machine learning classifier that enables fast, accurate, and affordable classification of brain tumors based on genome-wide DNA methylation profiles that is widely employed in the clinic. Neuro-oncology research would ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11842776/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://pmc.ncbi.nlm.nih.gov/search/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="11842776">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-57078-0"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_57078.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11842776%2F%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/11842776/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/11842776/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11842776/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Feb 20;16:1787. doi: <a href="https://doi.org/10.1038/s41467-025-57078-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-57078-0</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://pmc.ncbi.nlm.nih.gov/search/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Explainable artificial intelligence of DNA methylation-based brain tumor diagnostics</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Benfatto%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Salvatore Benfatto</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Salvatore Benfatto</span></h3>
<div class="p">
<sup>1</sup>Department of Pediatric Oncology, Dana-Farber Cancer Institute, Boston, MA USA </div>
<div class="p">
<sup>2</sup>Division of Hematology/Oncology, Boston Children’s Hospital, Boston, MA USA </div>
<div class="p">
<sup>3</sup>Broad Institute of MIT and Harvard, Cambridge, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Benfatto%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Salvatore Benfatto</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sill%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Martin Sill</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Martin Sill</span></h3>
<div class="p">
<sup>4</sup>Division of Pediatric Neurooncology, Hopp Children’s Cancer Center (KiTZ), Heidelberg, Germany </div>
<div class="p">
<sup>5</sup>National Center for Tumor Diseases (NCT), NCT Heidelberg, a partnership between DKFZ and Heidelberg University Hospital, Heidelberg, Germany </div>
<div class="p">
<sup>6</sup>German Cancer Research Center (DKFZ) and German Cancer Consortium (DKTK), Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sill%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Martin Sill</span></a>
</div>
</div>
<sup>4,</sup><sup>5,</sup><sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jones%20DTW%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">David T W Jones</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">David T W Jones</span></h3>
<div class="p">
<sup>5</sup>National Center for Tumor Diseases (NCT), NCT Heidelberg, a partnership between DKFZ and Heidelberg University Hospital, Heidelberg, Germany </div>
<div class="p">
<sup>6</sup>German Cancer Research Center (DKFZ) and German Cancer Consortium (DKTK), Heidelberg, Germany </div>
<div class="p">
<sup>7</sup>Division of Pediatric Glioma Research, Hopp Children’s Cancer Center (KiTZ), Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jones%20DTW%22%5BAuthor%5D" class="usa-link"><span class="name western">David T W Jones</span></a>
</div>
</div>
<sup>5,</sup><sup>6,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pfister%20SM%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Stefan M Pfister</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Stefan M Pfister</span></h3>
<div class="p">
<sup>4</sup>Division of Pediatric Neurooncology, Hopp Children’s Cancer Center (KiTZ), Heidelberg, Germany </div>
<div class="p">
<sup>5</sup>National Center for Tumor Diseases (NCT), NCT Heidelberg, a partnership between DKFZ and Heidelberg University Hospital, Heidelberg, Germany </div>
<div class="p">
<sup>6</sup>German Cancer Research Center (DKFZ) and German Cancer Consortium (DKTK), Heidelberg, Germany </div>
<div class="p">
<sup>8</sup>Department of Pediatric Oncology, Hematology &amp; Immunology, Heidelberg University Hospital, Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pfister%20SM%22%5BAuthor%5D" class="usa-link"><span class="name western">Stefan M Pfister</span></a>
</div>
</div>
<sup>4,</sup><sup>5,</sup><sup>6,</sup><sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sahm%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Felix Sahm</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Felix Sahm</span></h3>
<div class="p">
<sup>9</sup>Department of Neuropathology, Heidelberg University Hospital, Heidelberg, Germany </div>
<div class="p">
<sup>10</sup>Clinical Cooperation Unit Neuropathology, German Cancer Research Center (DKFZ) and German Cancer Consortium (DKTK), Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sahm%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Felix Sahm</span></a>
</div>
</div>
<sup>9,</sup><sup>10</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22von%20Deimling%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Andreas von Deimling</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Andreas von Deimling</span></h3>
<div class="p">
<sup>9</sup>Department of Neuropathology, Heidelberg University Hospital, Heidelberg, Germany </div>
<div class="p">
<sup>10</sup>Clinical Cooperation Unit Neuropathology, German Cancer Research Center (DKFZ) and German Cancer Consortium (DKTK), Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22von%20Deimling%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Andreas von Deimling</span></a>
</div>
</div>
<sup>9,</sup><sup>10</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Capper%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">David Capper</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">David Capper</span></h3>
<div class="p">
<sup>11</sup>Department of Neuropathology, Charité - Universitätsmedizin Berlin, corporate member of Freie Universität Berlin and Humboldt-Universität zu Berlin, Berlin, Germany </div>
<div class="p">
<sup>12</sup>German Cancer Consortium (DKTK), Partner Site Berlin, German Cancer Research Center (DKFZ), Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Capper%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">David Capper</span></a>
</div>
</div>
<sup>11,</sup><sup>12</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hovestadt%20V%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Volker Hovestadt</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Volker Hovestadt</span></h3>
<div class="p">
<sup>1</sup>Department of Pediatric Oncology, Dana-Farber Cancer Institute, Boston, MA USA </div>
<div class="p">
<sup>2</sup>Division of Hematology/Oncology, Boston Children’s Hospital, Boston, MA USA </div>
<div class="p">
<sup>3</sup>Broad Institute of MIT and Harvard, Cambridge, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hovestadt%20V%22%5BAuthor%5D" class="usa-link"><span class="name western">Volker Hovestadt</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Pediatric Oncology, Dana-Farber Cancer Institute, Boston, MA USA </div>
<div id="Aff2">
<sup>2</sup>Division of Hematology/Oncology, Boston Children’s Hospital, Boston, MA USA </div>
<div id="Aff3">
<sup>3</sup>Broad Institute of MIT and Harvard, Cambridge, MA USA </div>
<div id="Aff4">
<sup>4</sup>Division of Pediatric Neurooncology, Hopp Children’s Cancer Center (KiTZ), Heidelberg, Germany </div>
<div id="Aff5">
<sup>5</sup>National Center for Tumor Diseases (NCT), NCT Heidelberg, a partnership between DKFZ and Heidelberg University Hospital, Heidelberg, Germany </div>
<div id="Aff6">
<sup>6</sup>German Cancer Research Center (DKFZ) and German Cancer Consortium (DKTK), Heidelberg, Germany </div>
<div id="Aff7">
<sup>7</sup>Division of Pediatric Glioma Research, Hopp Children’s Cancer Center (KiTZ), Heidelberg, Germany </div>
<div id="Aff8">
<sup>8</sup>Department of Pediatric Oncology, Hematology &amp; Immunology, Heidelberg University Hospital, Heidelberg, Germany </div>
<div id="Aff9">
<sup>9</sup>Department of Neuropathology, Heidelberg University Hospital, Heidelberg, Germany </div>
<div id="Aff10">
<sup>10</sup>Clinical Cooperation Unit Neuropathology, German Cancer Research Center (DKFZ) and German Cancer Consortium (DKTK), Heidelberg, Germany </div>
<div id="Aff11">
<sup>11</sup>Department of Neuropathology, Charité - Universitätsmedizin Berlin, corporate member of Freie Universität Berlin and Humboldt-Universität zu Berlin, Berlin, Germany </div>
<div id="Aff12">
<sup>12</sup>German Cancer Consortium (DKTK), Partner Site Berlin, German Cancer Research Center (DKFZ), Heidelberg, Germany </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Jan 8; Accepted 2025 Feb 7; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC11842776  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/39979307/" class="usa-link">39979307</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">We have recently developed a machine learning classifier that enables fast, accurate, and affordable classification of brain tumors based on genome-wide DNA methylation profiles that is widely employed in the clinic. Neuro-oncology research would benefit greatly from understanding the underlying artificial intelligence decision process, which currently remains unclear. Here, we describe an interpretable framework to explain the classifier’s decisions. We show that functional genomic regions of various sizes are predominantly employed to distinguish between different tumor classes, ranging from enhancers and CpG islands to large-scale heterochromatic domains. We detect a high degree of genomic redundancy, with many genes distinguishing individual tumor classes, explaining the robustness of the classifier and revealing potential targets for further therapeutic investigation. We anticipate that our resource will build up trust in machine learning in clinical settings, foster biomarker discovery and development of compact point-of-care assays, and enable further epigenome research of brain tumors. Our interpretable framework is accessible to the research community via an interactive web application (<a href="https://hovestadtlab.shinyapps.io/shinyMNP/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://hovestadtlab.shinyapps.io/shinyMNP/</a>).</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> CNS cancer, Cancer genomics, Machine learning, Diagnostic markers, Oncogenes</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">AI-guided epigenetic classification is integral for accurate brain tumour diagnosis. Here, the authors introduce an interpretable framework to explain the AI process, revealing biological insights and potential therapeutic targets.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Cancer diagnosis has a pivotal role in clinical oncology. Precise tumor classification is required for making accurate prognoses and adequate treatment decisions. For many tumor types, histopathological evaluation represents the standard diagnostic method. However, over the last decades, characterization of genomic alterations has enabled more refined tumor classification and molecularly stratified treatments. Today, high throughput technologies generate genetic and epigenetic tumor landscapes at unprecedented resolution. In addition, the arrival of “big data” in medicine, and particularly in cancer diagnostics, has paved the way for the employment of machine learning (ML) algorithms in clinical routines<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. Trained on extensive reference datasets, these algorithms automate complex tasks while often surpassing human accuracy. Several recent ML applications have been successfully employed in the context of cancer diagnostics, for instance in the early detection of tumors from radiologic images<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup> or circulating cell-free tumor DNA<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Artificial intelligence (AI) will have a growing impact on diagnostic pathology in the future<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>.</p>
<p id="Par4">We and colleagues have recently developed a ML-based approach that utilizes array-based genome-wide DNA methylation profiles to enable fast, accurate, and affordable classification of central nervous system (CNS) tumors (known as the “Heidelberg brain tumor classifier”)<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. Over 100 different molecular brain tumor classes have been recognized<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup> and their precise classification remains difficult even for experienced neuropathologists. Our widely employed ML-based approach supports clinicians in making precise diagnoses, and it is particularly helpful to solve challenging cases<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>–<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. Similar DNA methylation-based classifiers have been developed for other tumor types, including sarcoma, meningioma, sinonasal tumors, and intrahepatic bile duct cancer<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>–<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, demonstrating generalizability and reliability of the method<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>.</p>
<p id="Par5">DNA methylation is a key epigenetic mark that plays an important role in regulating the phenotypic landscape during normal development and cancer<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>–<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. The etiology of CNS tumors is deeply connected to the development of the nervous system, and their high diversity mirrors the complexity of cellular phenotypes in the human brain. There is mounting evidence showing that the DNA methylation patterns of tumors reflect their respective cell of origin<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>, in addition to superimposed somatic epigenetic alterations that are specific to tumors<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. While genome-wide DNA methylation profiles are the basis for ML-based tumor classification, it remains largely unclear which specific patterns are being used for distinguishing classes<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>.</p>
<p id="Par6">Here, we present the development of an explainable artificial intelligence (XAI) framework to interpret the Heidelberg brain tumor classifier. Generally, XAI is motivated by the need for human interpretable models to better understand automated predictions, to gain insights into the underlying data, and to build up trust in AI<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>,<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Our XAI framework enables the identification of tumor-specific epigenetic signatures, may reveal biomarkers, and provides insights into brain tumor biology. Moreover, we provide a publicly accessible web application for easy exploration between tumor classes, methylation sites, and associated genes to facilitate a detailed understanding of the models’ decisions.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Development of an interpretable AI framework</h3>
<p id="Par7">The Heidelberg brain tumor classifier and other DNA methylation-based classifiers are based on the Random Forest (RF) algorithm which has been shown to be a very suitable ML algorithm to utilize high-dimensional genomic datasets<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>–<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. RF is an ensemble method that utilizes a multitude of binary decision trees trained on a subset of samples (“in-bag” samples) and features<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. For each splitting node of every decision tree, the algorithm selects the feature (i.e., a DNA methylation probe) that provides the best binary split. In this way RF naturally calculates feature importances, which has been shown to be a powerful metric to identify complex biological associations<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>–<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. We reasoned that quantifying the number of times a feature is selected during training provides us with a simple metric to create an interpretable framework that is capable of highlighting class-specific DNA methylation patterns. To this end, we retrieved the original dataset and the RF models of the Heidelberg brain tumor classifier. The dataset contains DNA methylation array profiles of 2801 samples corresponding to 82 tumor and 9 normal control classes (Fig. <a href="#Fig1" class="usa-link">1a</a>; an overview of classes and their abbreviations is provided in Supplementary Fig. <a href="#MOESM1" class="usa-link">1a</a> and Table <a href="#MOESM1" class="usa-link">S1</a>). For each sample, the DNA methylation status of 428,799 genomic sites (high-quality probes) is measured.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Overview of the explainable AI workflow.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11842776_41467_2025_57078_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9890/11842776/6a28e23ac5d7/41467_2025_57078_Fig1_HTML.jpg" loading="lazy" id="d33e471" height="606" width="682" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Plot shows a t-SNE dimensionality reduction of the reference dataset consisting of 2801 DNA methylation profiles from 82 CNS tumor classes and 9 healthy control classes. Color indicates different classes. Data were generated using Infinium HumanMethylation BeadChip arrays (428,799 probes). CNS central nervous system. Figure adapted from ref. <sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. <strong>b</strong> Graph illustration of the RF classifier trained on the reference dataset. Four out of 10,000 binary decision trees are shown. Magnification exemplifies four binary splitting nodes (probes) and five terminal nodes (tumor classes) of a single tree. Color of the edges indicates if the methylation value is higher (hypermethylated, red) or lower (hypomethylated, blue) than the threshold value of the preceding splitting node. <strong>c</strong> Illustration of the pairwise probe usage extracted from the RF classifier for each pair of reference samples and aggregated by sample class. Probe usage is stored in a 3D array in which the first two dimensions represent all possible class combinations and the third dimension represents all probes. <strong>d</strong> The information stored in the 3D array is used to build an interpretable framework that is accessible in an interactive web application.</p></figcaption></figure><p id="Par8">In the original work, a RF classifier was trained using all 428,799 probes (“outer” classifier) to select the 10,000 most informative probes for the final (“inner”) classifier (Fig. <a href="#Fig1" class="usa-link">1b</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">1b</a>)<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. We first investigated the outer RF model. Specifically, we analyzed each pairwise combination of tumor samples that were used for training a given tree (eight in-bag samples per class) and summarized the probe usage as the number of times a probe was selected to split their respective classes (see “Methods”). We used a positive sign if the probe was hypermethylated and a negative sign if the probe was hypomethylated (higher or lower than the threshold value, respectively) in the samples at the split. Probe usage was aggregated across all trees into a three-dimensional array of all 8281 (91 by 91) possible class combinations for each of the 428,799 probes (Fig. <a href="#Fig1" class="usa-link">1c</a>). In total, the array is composed of 3.55 × 10<sup>9</sup> data points. The absolute probe usage across all data points is 2.24 × 10<sup>11</sup>, representing the total amount of splits between in-bag samples over all trees. This 3D array serves as the starting point for all the analyses and for the development of an interactive web application described below (Fig. <a href="#Fig1" class="usa-link">1d</a>).</p></section><section id="Sec4"><h3 class="pmc_sec_title">Global patterns of differential probe usage</h3>
<p id="Par9">We first investigated the global patterns of probe usage of the outer RF classifier. We found that a relatively small subset of probes contributed the majority of the total usage. The top 10,000 or 25,000 probes (2.3% or 5.8% of all probes) contributed to 61.2% or 78.1% of the probe usage across all class combinations, respectively (Fig. <a href="#Fig2" class="usa-link">2a</a>). In contrast, the 250,000 least used probes (58.3%) contributed to only 1.3% of the total usage. Looking at probes that separated individual classes from all others, the top 10,000 probes contributed between 96.7% and 55.7% of the total usage (classes LIPN and LGG_PA_MID, respectively; see Table <a href="#MOESM1" class="usa-link">S1</a> for extended class annotations; Fig. <a href="#Fig2" class="usa-link">2b</a>). There was high inequality in the probe usage for each class, as described by Gini indexes ranging from 0.99 for LIPN to 0.89 for LGG_PA/GG_ST (Supplementary Fig. <a href="#MOESM1" class="usa-link">2a</a>). These analyses indicate that the contribution of a majority of probes is negligible, while few probes are highly informative to distinguish between tumor classes. Among those, fewer probes are selected for the classification of biologically very distinct classes (e.g., LIPN or ETMR), while for more closely related classes the model makes use of a larger number of probes to discriminate between those classes (e.g., the members of the low grade glioma methylation class family; Supplementary Fig. <a href="#MOESM1" class="usa-link">2b</a>). Similar results were observed in an independent RF classifier trained on the same dataset, demonstrating a high stability of probe usage values across different models (Supplementary Fig. <a href="#MOESM1" class="usa-link">2c–f</a>). Moreover, probe usage values showed high concordance with feature importance values calculated using the SHAP approach<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>, highlighting the robustness of our method in assessing features importances compared to established methods (Supplementary Fig. <a href="#MOESM1" class="usa-link">3a–c</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Class-specific probe usage across functional genomic regions.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11842776_41467_2025_57078_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9890/11842776/c05559b3e6f2/41467_2025_57078_Fig2_HTML.jpg" loading="lazy" id="d33e550" height="1015" width="752" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Plot shows the overall probe usage with probes ranked from the most to least used. Vertical lines indicate the fraction of the total usage of the first 10,000 and 25,000 probes. <strong>b</strong> Plot shows the cumulative probe usage for each tumor class. The probes are ranked by their usage. The vertical line indicates the first 10,000 probes. <strong>c</strong> Heatmap indicates the average usage of probes within CpG islands, Sea, Shelf, and Shore regions for each tumor class. Number of overlapping probes is indicated above. The color code indicates if probes are predominantly hypermethylated (red) or hypomethylated (blue). <strong>d</strong> Similar heatmap indicating the mean probe usage within annotated DHS and enhancers (overlapping CpG islands). DHS DNase I hypersensitive sites. <strong>e</strong> Similar heatmap indicating the mean probe usage within large-scale LADs and heterochromatic domains. LAD lamina associated domains. <strong>f</strong> Heatmaps show the probe usage in 250 bp windows surrounding the TSS of all annotated genes (5 kb up- and downstream). Heatmaps represent the total (left) or the average (right) probe usage. Histogram on top of the left heatmap indicates the number of microarray probes overlapping each window. TSS transcription start site. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par10">We then asked if probes displayed differential usage according to their functional genomic localization and grouped probes according to the following annotations: CpG islands (regions of high CpG density), shores (regions within 2 kb from CpG island), shelves (regions 2–4 kb from CpG island), and open sea regions (the rest of the genome). Usage distributions showed major differences between classes. For instance, probes falling within CpG islands were frequently used to distinguish IDH-mutant gliomas (classes O_IDH, A_IDH, and A_IDH_HG) from other classes (52.0% of the average probe usage; Fig. <a href="#Fig2" class="usa-link">2c</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">4a</a>). These probes were predominantly hypermethylated in IDH-mutant gliomas, in accordance with the previously reported CpG island methylator phenotype (CIMP)<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. Interestingly, similar patterns were observed for tumor classes LYMPHO and ETMR (80.6% and 56.3%, respectively). Conversely, probes that were used to distinguish PITAD and LIPN classes were mostly located in shelf and open sea regions and were frequently hypomethylated (78.6% and 57.7%, respectively). These classes are characterized by low overall DNA methylation levels (Supplementary Fig. <a href="#MOESM1" class="usa-link">4b</a>).</p>
<p id="Par11">Other functional genomic regions that often show differential DNA methylation between cell types and disease states are enhancer regions and large-scale heterochromatic domains. Hypomethylation within enhancer regions is generally associated with enhancer activation and transcription factor binding<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. Heterochromatic domains, which frequently are positioned at the nuclear lamina, are condensed and transcriptionally silent regions in which DNA methylation is believed to be lost in a passive process in many different types of cancer<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. To investigate probe usage in these genomic regions of the RF classifier, we overlapped probes with annotated enhancers (defined across ENCODE cell lines) and DNase I hypersensitive sites (DHS; Fig. <a href="#Fig2" class="usa-link">2d</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">4c, d</a>). In addition, we overlapped probes with large-scale heterochromatic domains (H3K9me3-positive domains, defined in K562 lymphoblast cells and H1 embryonic stem cells) and lamina-associated domains (LADs, defined in Tig3 fibroblast cells; Fig. <a href="#Fig2" class="usa-link">2e</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">4e, f</a>). We found that ETMR is often classified by hypermethylated probes located within DHS and CpG island enhancers. High usage of hypermethylated probes within enhancers that are not overlapping CpG islands was observed for different classes of ATRTs, supporting a previous study showing that H3K27ac enhancer landscapes distinguish between ATRT subgroups<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. While probe usage patterns in LADs largely mirrored those observed in heterochromatic domains defined in K562 cells, we observed more pronounced patterns in H1 cells. Hypomethylated probes in those regions were frequently employed to classify LGG_MYB, MELAN, EFT_CIC, and CNS_NB_FOXR2 classes. Hypermethylated probes within LADs and heterochromatin domains were used for multiple GBM classes. These results indicate that genomic regions of different sizes (sub-kilobase to megabases) are employed to distinguish between different tumor classes, with potential links to their biology.</p>
<p id="Par12">Finally, we focused on promoter regions which are often described as the primary region of transcriptional regulation by DNA methylation. Promoter hypermethylation is generally associated with gene silencing. To this aim, we grouped probes in 250 bp windows within 10 kb regions centered around annotated transcription starting sites (TSS) of all annotated genes. We observed highest probe usage in close proximity of the TSS in the majority of classes (Fig. <a href="#Fig2" class="usa-link">2f</a>). To test if this enrichment was due to the higher probe coverage of promoters on the Illumina DNA methylation array (32.5% of available probes are located within 1 kb of annotated TSS), we also considered the average probe usage within windows. While hypermethylated probes in proximity of TSS showed the highest usage for IDH-mutant tumors, this analysis showed that most other tumor classes relied on probes that were located further upstream or downstream of the TSS (Fig. <a href="#Fig2" class="usa-link">2f</a>). This analysis suggests that probes located distal to promoter regions (e.g., in enhancers, gene bodies) are more informative for classification of most tumor classes.</p></section><section id="Sec5"><h3 class="pmc_sec_title">High genomic redundancy of informative probes</h3>
<p id="Par13">For further analyses we focused on the inner RF model that is based on 10,000 probes selected from the outer RF model and is used for generating predictions with the Heidelberg brain tumor classifier<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. Using this model, we extracted and aggregated pairwise probe usage values similarly as for the outer classier. We first asked if informative probes separated multiple classes from each other or were specific to individual classes (i.e., separated a single class from all others). We also asked to which degree there is redundancy between informative probes and if they map in close proximity to each other. To this aim, we performed unsupervised clustering and t-SNE dimensionality reduction of all 10,000 probes according to their usage (using Pearson’s correlation coefficient as the distance measure). Our analysis indicated clearly defined groups of probes that were separated into a total of 88 clusters (Fig. <a href="#Fig3" class="usa-link">3a</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">5a</a>). The number of probes per cluster was highly variable and ranged from 2 to 211 (median of 117; Supplementary Fig. <a href="#MOESM1" class="usa-link">5b</a>). Interestingly, we found that the majority of clusters were associated with a single tumor class, indicating high class specificity of most selected probes (Fig. <a href="#Fig3" class="usa-link">3b</a>). Specifically, 71 of 88 clusters (80.7%) were associated with a specific tumor class. For instance, to classify ETMR samples, the model predominantly employed probes belonging to cluster 27 (hypermethylated, <em>n</em> = 136 probes) and cluster 78 (hypomethylated, <em>n</em> = 44 probes; Fig. <a href="#Fig3" class="usa-link">3c, d</a>). We also identified 17 clusters (19.3%) that were associated with 2 or 3 tumor classes. Among these were clusters associated with GBM, IDH-mutant glioma, and PITAD methylation class families of closely related tumor classes (Fig. <a href="#Fig3" class="usa-link">3b</a>). For example, probes belonging to cluster 1 (hypermethylated, <em>n</em> = 211) were associated with classes O_IDH, A_IDH, and A_IDH_HG (Fig. <a href="#Fig3" class="usa-link">3e, f</a>). When looking at probes that distinguish these three classes from each other, we found that a smaller set of probes within cluster 1 was specifically used to distinguish O_IDH from the other two, while few other probes scattered across different clusters were employed to split A_IDH and A_IDH_HG (Supplementary Fig. <a href="#MOESM1" class="usa-link">5c</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Unsupervised clustering of probes by their usage.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11842776_41467_2025_57078_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9890/11842776/e9c76b59bf69/41467_2025_57078_Fig3_HTML.jpg" loading="lazy" id="d33e685" height="1016" width="730" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> t-SNE dimensionality reduction and unsupervised clustering of 10,000 selected probes according to their usage (Pearson’s distance). A total of 88 clusters are identified. Individual clusters of probes are numbered. <strong>b</strong> Heatmap shows the total probe usage by class and by cluster. Blue color indicates clusters in which probes are predominantly hypomethylated in a given tumor class, red color indicates clusters which are predominantly hypermethylated. Clusters (columns) and tumor classes (rows) are ordered by hierarchical clustering. Values are scaled by column. <strong>c</strong> t-SNE as in panel (a) showing the probe usage in ETMR compared to all other classes. The color code indicates hypomethylated (blue) and hypermethylated (red) probes. <strong>d</strong> Scatterplot showing the usage of probes belonging to cluster 78 (<em>n</em> = 44) for each tumor class. <strong>e</strong> t-SNE as in panel (<strong>a</strong>) showing the probe usage in A IDH compared to all other classes. <strong>f</strong> Scatterplot showing the usage of probes belonging to cluster 1. <strong>g</strong> Karyotype plot showing the genomic location of the probes belonging to cluster 78. <strong>h</strong> Similar karyotype plot showing probes belonging to cluster 1. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par14">We next asked if probes belonging to a given cluster mapped in close proximity to each other (e.g., multiple probes associated with the same gene). We found that probes from each cluster were distributed over many regions across the genome and did not show a specific enrichment towards a particular chromosome or region (Supplementary Fig. <a href="#MOESM1" class="usa-link">5d</a>). The 44 probes belonging to cluster 78 (ETMR-specific probes) mapped to 32 distinct genomic regions (Fig. <a href="#Fig3" class="usa-link">3g</a>). For cluster 1 (IDH-mutant glioma-specific probes, <em>n</em> = 211), we identified 106 genomic regions (Fig. <a href="#Fig3" class="usa-link">3h</a>). These results indicate a high genomic redundancy of probes employed by the classifier that may mitigate potential intra-class variability between individual patient samples (e.g., due to stochastic copy-number changes). A main characteristic of the RF algorithm is that it derives predictions across a large number of decision trees. Making use of nearly redundant probes across the genome may explain the high robustness of DNA methylation array-based tumor classification.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Interpretable AI yields insights into tumor biology</h3>
<p id="Par15">To make our interpretable framework accessible to the research community, we developed a user-friendly web application utilizing the shiny R package (“shinyMNP”, <a href="https://hovestadtlab.shinyapps.io/shinyMNP/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://hovestadtlab.shinyapps.io/shinyMNP/</a>). Our application consists of four main panels that allow the user to query and explore the dataset in different ways. In the first panel, users can explore the top probes employed by the classifier to identify a given class (Supplementary Fig. <a href="#MOESM1" class="usa-link">6a</a>). The second panel of the app allows users to identify probes that distinguish any two classes (Supplementary Fig. <a href="#MOESM1" class="usa-link">6b</a>). The third panel visualizes the total probe usage for select genes of interest across all classes (Supplementary Fig. <a href="#MOESM1" class="usa-link">6c</a>). The interactive heatmap shows the usage of associated probes across all possible class combinations. The final panel visualizes the top genes associated with the most used probes for each class, as represented as a directed network in which arrows connect genes to associated tumor classes (Fig. <a href="#Fig4" class="usa-link">4a</a>).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. shinyMNP interface and illustrative class-specific genes.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11842776_41467_2025_57078_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9890/11842776/748211548dbd/41467_2025_57078_Fig4_HTML.jpg" loading="lazy" id="d33e765" height="1012" width="716" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Screenshot of the shinyMNP web application shows a network illustration of the top five probe-associated genes for each tumor class. Black and colored nodes represent genes and tumor classes, respectively. Width of edges indicate the total probe usage for each gene. Some genes may be associated with multiple tumor classes. <strong>b</strong> Genome plot of a 12 kb region surrounding the <em>SHPRH</em> promoter highlights probes that are specifically hypermethylated in ETMR. Red indicates methylated probes (beta-value of 1), blue indicates unmethylated probes (beta-value of 0) for each tumor class. Total probe usage and exact location of probes are indicated on top. <em>SHPRH</em> is silenced in ETMR. <strong>c</strong> Genome plot of the <em>PWWP3A</em> gene highlighting pronounced hypomethylation across the entire gene body specific to HGNET_MN1. <em>PWWP3A</em> is highly expressed in this tumor class. <strong>d</strong> Genome plot of the <em>TBX19</em> gene showing hypomethylation of the promoter specific in PITAD_ACTH. <strong>e</strong> Genome plot of the <em>RET</em> oncogene showing hypermethylation of a small region near the fourth exon specific to HGNET_BCOR. <em>RET</em> is highly expressed in this tumor class.</p></figcaption></figure><p id="Par16">Using our shinyMNP web application, we investigated class-specific probes and their associated genes for potential biological discovery. We selected four illustrative examples that demonstrate the application of our interpretable framework. As a first example, we identified the promoter region of <em>SHPRH</em> that was specifically hypermethylated in ETMR, a tumor class that is characterized by chromosomal instability and high levels of R-loops<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup> (Fig. <a href="#Fig4" class="usa-link">4b</a>). In accordance, gene expression analysis highlighted pronounced <em>SHPRH</em> downregulation specifically in ETMR tumors (Supplementary Fig. <a href="#MOESM1" class="usa-link">6d</a>). The E3-ligase <em>SHPRH</em> poly-ubiquitinates PCNA to prevent genomic instability from stalled replication forks that may be caused by R-loops<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>–<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. Our observation suggests that silencing of <em>SHPRH</em> may contribute to the chromosomal instability and high levels of R-loops in ETMR. Second, we identified a number of probes that were specifically hypomethylated throughout the entire gene body of <em>PWWP3A</em> (also known as <em>MUM1</em>) specific to the recently defined tumor class HGNET_MN1 (Fig. <a href="#Fig4" class="usa-link">4c</a>). <em>PWWP3A</em> is exclusively expressed in this tumor class (Supplementary Fig. <a href="#MOESM1" class="usa-link">6e</a>) and is involved in DNA damage repair and chromatin organization<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. The third example highlights the hypomethylation of the broader promoter region of the <em>TBX19</em> transcription factor (also known as <em>TPIT</em>) in PITAD_ACTH and relates to the pituitary origin of this tumor class (Fig. <a href="#Fig4" class="usa-link">4d</a>). Immunohistochemistry for <em>TPIT</em> has been established as a specific marker for the identification of PITAD_ACTH in routine diagnostics<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. Inactivating mutations of <em>TBX19</em> are associated with isolated deficiency of adrenocorticotropic hormone (ACTH), indicating a crucial role of this transcription factor in the regulation of the pituitary corticotroph lineage<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>,<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. As a last example, we identified pronounced hypermethylation of a single probe near the fourth exon of the proto-oncogene <em>RET</em> specific to the recently defined tumor class HGNET_BCOR in which <em>RET</em> is highly expressed (Fig. <a href="#Fig4" class="usa-link">4e</a>; Supplementary Fig. <a href="#MOESM1" class="usa-link">6f</a>)<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. <em>RET</em> encodes a transmembrane receptor of the tyrosine protein kinase family and is an important gene in CNS development<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>–<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. While it is unknown how the observed hypermethylation is associated with the high expression of <em>RET</em> in HGNET_BCOR, we postulate that overexpression may be due to hypermethylation of a regulatory element, as multiple distal enhancers have been identified in this genomic region. <em>RET</em> may represent a HGNET_BCOR-specific vulnerability as potent targeted drugs have recently been FDA-approved for the treatment of other <em>RET</em>-altered tumors<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>.</p>
<p id="Par17">In summary, our interpretable framework reveals associations between specific genes and tumor classes that may be employed for future studies in the context of biomarker discovery, therapeutic target identification, and cancer biology research of CNS tumors. Our publicly accessible web application of the interpretable framework enables researchers of the scientific community to perform similar analyses across all classes included in the Heidelberg brain tumor classifier.</p></section></section><section id="Sec7"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par18">The ability to classify CNS tumors based on their DNA methylation profiles using artificial intelligence approaches has irrevocably changed brain tumor classification in clinical practice and in research. Despite the usefulness of the Heidelberg brain tumor classifier, a clear understanding of the classifier’s inner decision-making process has been missing. To this aim, we developed an interpretable framework to better explain its underlying working rules.</p>
<p id="Par19">By calculating the pairwise probe usage between classes over all RF trees, we simultaneously enable both global and local interpretability. DNA methylation patterns that are globally associated with individual tumor classes are readily identified, such as CpG island hypermethylation or hypomethylation in broad heterochromatic regions. On the other hand, our framework allows us to pinpoint probes that are locally important to distinguish between pairs of select tumor classes. Results may sometimes overlap, as a probe can have both high global and local importance for the same class. For instance, for some methylation class families, such as the IDH-mutant glioma classes, a group of redundant probes are used to separate these classes from all the other classes at the global level. After this high-level separation, the algorithm selects some of the few class-specific probes, as identifiable with our pairwise analysis. In the research context, this approach can be applied to annotate DNA methylation patterns that are shared among different classes and families of brain tumors and at the same time to uncover genomic sites that are unique to a single class. In addition to their value as biomarkers for classification and diagnostic relevance, tumor-associated genes could reveal valuable targets for the development of precision therapies, such as the association between <em>RET</em> and HGNET_BCOR, for which targeted inhibitors have recently become available.</p>
<p id="Par20">Importantly, our interpretable framework can be readily adapted to future versions of the Heidelberg brain tumor classifier, for example to discover relevant probes and DNA methylation patterns associated with additional tumor classes. Our approach can also be transferred to other tumor classifiers that are based on DNA methylation profiling and the RF algorithm<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. For example, applying our approach to a RF classifier of a recently published cohort sarcoma samples<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup> showed that the main principles unveiled from the CNS tumor classifier also hold true in other DNA methylation-based tumor classifiers (Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>). Furthermore, motivated by recent advances using nanopore sequencing technology<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>–<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>, we foresee the implementation of point-of-care assays to allow for more affordable and rapid diagnosis that make use of reduced subsets of informative and non-redundant CpG sites based on our findings. Our results also have potential applications in the context of liquid biopsies for early cancer detection, classification, and disease monitoring over time from circulating tumor DNA<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. Our study provides a unique reference for incorporating DNA methylation profiling into these studies.</p>
<p id="Par21">Overall, our interpretable framework provides a better understanding of the underlying working rules of the Heidelberg brain tumor classifier. Our resource will facilitate the discovery of disease biomarkers and therapeutic targets, and support the development of bioinformatic pipelines, machine learning models and point-of-care assays for rapid diagnostics, early detection, and disease monitoring.</p></section><section id="Sec8"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec9"><h3 class="pmc_sec_title">Datasets</h3>
<p id="Par22">DNA methylation profiles (from Illumina Infinium HumanMethylation450K BeadChip arrays) from 2,801 samples of the reference dataset of the Heidelberg brain tumor classifier<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup> were downloaded as IDAT files from GEO (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE109381" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE109381</a> [<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE109381" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE109381</a>]). Samples were associated with 82 molecular tumor classes and 9 control classes. Processed gene expression data (Affymetrix U133 Plus 2.0 Array) were retrieved from GEO (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73038" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE73038</a> [<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73038" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73038</a>]) and annotations were obtained from the original publication<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. The following probes were selected: 211421_s_at (<em>RET</em>); 221290_s_at (<em>MUM1</em>/<em>PWWP3A</em>); 226366_at (<em>SHPRH</em>). All computational analyses were performed in R (v.4.1.3). The pre-processed DNA methylation array data of the sarcoma cohort<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup> were retrieved from GEO (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE140686" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE140686</a> [<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE140686" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE140686</a>]).</p></section><section id="Sec10"><h3 class="pmc_sec_title">Random forest models</h3>
<p id="Par23">Random forests consist of many binary decision trees, which are simple ML algorithms that resemble the human cognitive process of decision-making. The algorithm constructs individual trees by selecting features that offer the best split between remaining samples, resulting in a flowchart-like structure that makes them easy to understand and interpret. Terminal nodes represent different classes that can be predicted. Decision trees are composed of a limited number of features and are readily interpretable. At the same time, individual trees suffer from low accuracy in complex scenarios and are prone to overfitting. In other words, their rigid, but transparent structure limits their ability to learn and generalize. To overcome those limitations, RF aggregates the prediction of multiple decision trees on a subset of samples and features.</p>
<p id="Par24">In order to make the Heidelberg CNS tumor classifier interpretable, we used the trained RF models (“outer” and “inner” classifier) from the original publication (v.11b2)<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. The inner classifier was trained using 10,000 probes that were selected from the outer classifier trained on all 428,799 high quality probes. Probes were selected based on their per-class importance (MeanDecreaseAcuarcy value). We first calculated the rank of each probe within every class (low rank = high MeanDecreaseAcuarcy), and subsequently selected the 10,000 probes with the lowest rank across classes<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>.</p>
<p id="Par25">To re-train the outer RF model (Supplementary Fig. <a href="#MOESM1" class="usa-link">2c–f</a>), we used the randomForest R package (v.4.7-1.1) and the following parameters: ntree = 10000, mtry = 654, sampsize = rep(8, 91), importance = T. To train an outer RF model for the sarcoma cohort (Supplementary Fig. <a href="#MOESM1" class="usa-link">6</a>) we used the following parameters: ntree = 10,000, mtry = 654, sampsize = rep (7, 65). The inner RF sarcoma model was trained using the 10,000 most important features (overall MeanDecreaseAccuracy value from the outer classifier) and using mtry = 100. All the RF models were trained with the option keep.inbag = T, required for the next steps.</p>
<p id="Par26">To calculate SHAP feature importances, we trained an independent random forest model in Python (v.3.12.5) with the scikit-learn library (v.1.5.1) using the same 10,000 selected features, achieving an out-of-bag error of 0.04. We calculated Shapley values using the shap library (v.0.45.1). To reduce computing time, Shapley values were calculated for a single sample in each of the 91 classes (similarly using one sample per class as the background).</p></section><section id="Sec11"><h3 class="pmc_sec_title">Extraction of random forest probe usage</h3>
<p id="Par27">We assumed that the number of times a feature (i.e., a DNA methylation probe) is selected to perform a split between tumor classes reflects the importance of that particular feature in that context. Since this information is not directly accessible from the innate structure of a trained RF model object, we calculated the probe usage as follows: In an iterative process over each of the 10,000 trees, all paths from the root node to each terminal node were extracted. Next, those paths were compared in a pairwise manner, annotating the probe ID and its methylation status at the node that splits two terminal nodes (annotated as −1 for hypomethylated and 1 for hypermethylated). From these pre-computed comparisons, we retrieved the splitting nodes for all 529,984 possible combinations between in-bag samples (8 samples for each of the 91 classes). Subsequently, splitting nodes were aggregated at the tumor class level across all trees (representing the probe usage). Probe usage values were stored as a 3D array in which each layer (428,799 probes) is composed of a symmetric matrix representing the probe usage of all 91 by 91 (8281) possible class comparisons. The probe usage can be summarized as the total or average probe usage per class or across all classes. For these steps the data.table (v.1.14.2) and iterpc (v.0.4.2) R packages were used. The code for extracting probe usages from a trained RF model is accessible at <a href="https://github.com/hovestadt/shinyMNP" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/hovestadt/shinyMNP</a>.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Analysis of functional genomic regions</h3>
<p id="Par28">Probes were grouped by tumor classes and mean probe usage was calculated for each category of functional genomic regions. CpG island, shore, shelf, sea, DHS, and enhancers annotations were extracted from the Illumina array annotation file (HumanMethylation450 manifest file v1.2, downloaded from the Illumina website). Lamina associated domains (LADs, identified in human Tig3 human fibroblasts) and heterochromatin domains (identified in H1-hESC and K562 cell lines) positions were downloaded from the ENCODE portal. The human genome assembly hg19 and RefSeq gene annotation were retrieved from the UCSC genome browser website. Probes were mapped to the nearest TSS using functions from the GenomicRanges Bioconductor package (v.1.46.1). Probes with a distance greater than 5000 bp were removed. The total and average probe usage was plotted as a function of the distance to the nearest TSS for each tumor class in 250 bp windows.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Dimensionality reduction and clustering</h3>
<p id="Par29">Vectorized probe usages were employed to calculate the 10,000 by 10,000 Pearson’s distance matrix. This matrix was used as input for a t-SNE dimensionality reduction calculated using the Rtsne R package (v.0.16) and the following parameters: theta = 0.1, pca = F, num_threads = 0, is_distance = T, max_iter = 5000. For unsupervised clustering, we first calculated the k-nearest neighbors and constructed the SNN graph from the distance matrix. Then, clusters were identified using the Louvain algorithm to optimize the modularity function. The Seurat Bioconductor package (v.4.1.1) was employed for this step. To associate each cluster of probes to tumor classes we first calculated the fraction of probe usage over the total for each cluster and class. A tumor class was associated with a cluster if the fraction was greater than 0.1 (10%). The genomic location of probes within clusters was plotted using the karyoploteR Bioconductor package (v.1.20.3).</p></section><section id="Sec14"><h3 class="pmc_sec_title">Development of the shinyMNP web application</h3>
<p id="Par30">To create an interactive web application that allows researchers to access our interpretable framework we used the same 3D probe usage array as described above and precomputed different summaries for rapid access. The app was developed using the shiny (v.1.7.1), tidyverse (v.1.3.1), ggplot2 (v.3.3.5), shinythemes (v.1.2.0), rhdf5 (v.2.38.1), plotly (v.4.10.0), heatmaply (v.1.3.0), igraph (v.1.3.1) and visNetwork (v.2.1.0) R/Bioconductor packages.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par31">Further information on research design is available in the <a href="#MOESM2" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec16"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/11842776/bin/41467_2025_57078_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/11842776/bin/41467_2025_57078_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (1.9MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/11842776/bin/41467_2025_57078_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (1.8MB, pdf) </sup>
</div></div></section></section><section id="Sec17"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/11842776/bin/41467_2025_57078_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (1.2MB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank the members of the Hovestadt lab for insightful discussions and critical feedback on the paper. This work was supported by the Charles H. Hood Foundation, the Children’s Cancer Research Fund, the V Foundation, and the Claudia Adams Barr Program in Innovative Cancer Research (V.H.).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>S.B. and V.H. developed the explainable AI approach and investigated results; S.B. created the web application; M.S. and V.H. developed the random forest classification algorithm; D.T.W.J., S.M.P., F.S., A.v.D., and D.C. generated the reference cohort and defined methylation classes; D.T.W.J., S.M.P., and V.H. conceptualized the study; S.B. and V.H. wrote the paper with comments from all authors. All authors approved the final version of the paper.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par32"><em>Nature Communications</em> thanks Sylvain Cussat-Blanc, Xiao-Nan Li and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>No new datasets have been generated as part of this study. The DNA methylation reference dataset and the gene expression dataset are accessible from GEO (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE109381" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE109381</a> [<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE109381" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE109381</a>]) and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73038" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE73038</a>, respectively). <a href="#Sec17" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="notes4"><h2 class="pmc_sec_title">Code availability</h2>
<p>The code to calculate probe usages from the RF model and perform subsequent analyses is available at: <a href="https://github.com/hovestadt/shinyMNP" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/hovestadt/shinyMNP</a>.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par33">M.S., D.T.W.J., S.M.P., F.S., A.v.D., D.C., and V.H. are listed as inventors on the patent ‘DNA-methylation based method for classifying tumor species’ (EP3268492B1) filed by Deutsches Krebsforschungszentrum Stiftung des öffentlichen Rechts and Ruprecht-Karls-Universität Heidelberg. M.S., D.T.W.J., S.M.P., F.S., A.v.D., and D.C. are shareholders in and co-founders of Heidelberg Epignostix GmbH. The remaining authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="sec18"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-57078-0.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Topol, E. J. High-performance medicine: the convergence of human and artificial intelligence. <em>Nat. Med.</em><strong>25</strong>, 44–56 (2019).
</cite> [<a href="https://doi.org/10.1038/s41591-018-0300-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30617339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Topol,%20E.%20J.%20High-performance%20medicine:%20the%20convergence%20of%20human%20and%20artificial%20intelligence.%20Nat.%20Med.25,%2044%E2%80%9356%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Stenzinger, A. et al. Artificial intelligence and pathology: from principles to practice and future applications in histomorphology and molecular profiling. <em>Semin. Cancer Biol.</em><strong>84</strong>, 129–143 (2022).
</cite> [<a href="https://doi.org/10.1016/j.semcancer.2021.02.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33631297/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stenzinger,%20A.%20et%20al.%20Artificial%20intelligence%20and%20pathology:%20from%20principles%20to%20practice%20and%20future%20applications%20in%20histomorphology%20and%20molecular%20profiling.%20Semin.%20Cancer%20Biol.84,%20129%E2%80%93143%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Yala, A., Lehman, C., Schuster, T., Portnoi, T. &amp; Barzilay, R. A deep learning mammography-based model for improved breast cancer risk prediction. <em>Radiology</em><strong>292</strong>, 60–66 (2019).
</cite> [<a href="https://doi.org/10.1148/radiol.2019182716" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31063083/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yala,%20A.,%20Lehman,%20C.,%20Schuster,%20T.,%20Portnoi,%20T.%20&amp;%20Barzilay,%20R.%20A%20deep%20learning%20mammography-based%20model%20for%20improved%20breast%20cancer%20risk%20prediction.%20Radiology292,%2060%E2%80%9366%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Jamshidi, A. et al. Evaluation of cell-free DNA approaches for multi-cancer early detection. <em>Cancer Cell</em><strong>40</strong>, 1537–1549.e12 (2022).
</cite> [<a href="https://doi.org/10.1016/j.ccell.2022.10.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36400018/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jamshidi,%20A.%20et%20al.%20Evaluation%20of%20cell-free%20DNA%20approaches%20for%20multi-cancer%20early%20detection.%20Cancer%20Cell40,%201537%E2%80%931549.e12%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Capper, D. et al. DNA methylation-based classification of central nervous system tumours. <em>Nature</em><strong>555</strong>, 469–474 (2018).
</cite> [<a href="https://doi.org/10.1038/nature26000" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6093218/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29539639/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Capper,%20D.%20et%20al.%20DNA%20methylation-based%20classification%20of%20central%20nervous%20system%20tumours.%20Nature555,%20469%E2%80%93474%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Louis, D. N. et al. The 2021 WHO Classification of Tumors of the Central Nervous System: a summary. <em>Neuro. Oncol.</em><strong>23</strong>, 1231–1251 (2021).
</cite> [<a href="https://doi.org/10.1093/neuonc/noab106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8328013/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34185076/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Louis,%20D.%20N.%20et%20al.%20The%202021%20WHO%20Classification%20of%20Tumors%20of%20the%20Central%20Nervous%20System:%20a%20summary.%20Neuro.%20Oncol.23,%201231%E2%80%931251%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Jaunmuktane, Z. et al. Methylation array profiling of adult brain tumours: diagnostic outcomes in a large, single centre. <em>Acta Neuropathol. Commun.</em><strong>7</strong>, 1–18 (2019).</cite> [<a href="https://doi.org/10.1186/s40478-019-0668-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6381711/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30786920/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jaunmuktane,%20Z.%20et%20al.%20Methylation%20array%20profiling%20of%20adult%20brain%20tumours:%20diagnostic%20outcomes%20in%20a%20large,%20single%20centre.%20Acta%20Neuropathol.%20Commun.7,%201%E2%80%9318%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Karimi, S. et al. The central nervous system tumor methylation classifier changes neuro-oncology practice for challenging brain tumor diagnoses and directly impacts patient care. <em>Clin. Epigenetics</em><strong>11</strong>, 1–10 (2019).
</cite> [<a href="https://doi.org/10.1186/s13148-019-0766-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6896594/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31806041/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Karimi,%20S.%20et%20al.%20The%20central%20nervous%20system%20tumor%20methylation%20classifier%20changes%20neuro-oncology%20practice%20for%20challenging%20brain%20tumor%20diagnoses%20and%20directly%20impacts%20patient%20care.%20Clin.%20Epigenetics11,%201%E2%80%9310%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Priesterbach-Ackley, L. P. et al. Brain tumour diagnostics using a DNA methylation-based classifier as a diagnostic support tool. <em>Neuropathol. Appl. Neurobiol.</em><strong>46</strong>, 478–492 (2020).
</cite> [<a href="https://doi.org/10.1111/nan.12610" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7496466/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32072658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Priesterbach-Ackley,%20L.%20P.%20et%20al.%20Brain%20tumour%20diagnostics%20using%20a%20DNA%20methylation-based%20classifier%20as%20a%20diagnostic%20support%20tool.%20Neuropathol.%20Appl.%20Neurobiol.46,%20478%E2%80%93492%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>White, C. L. et al. Implementation of DNA methylation array profiling in pediatric central nervous system tumors - the AIM BRAIN Project: an Australian and New Zealand Children’s Haematology and Oncology (ANZCHOG) Group study. <em>J. Mol. Diagn.</em>10.1016/j.jmoldx.2023.06.013 (2023).
</cite> [<a href="https://doi.org/10.1016/j.jmoldx.2023.06.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37517472/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?White,%20C.%20L.%20et%20al.%20Implementation%20of%20DNA%20methylation%20array%20profiling%20in%20pediatric%20central%20nervous%20system%20tumors%20-%20the%20AIM%20BRAIN%20Project:%20an%20Australian%20and%20New%20Zealand%20Children%E2%80%99s%20Haematology%20and%20Oncology%20(ANZCHOG)%20Group%20study.%20J.%20Mol.%20Diagn.10.1016/j.jmoldx.2023.06.013%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Capper, D. et al. Practical implementation of DNA methylation and copy-number-based CNS tumor diagnostics: the Heidelberg experience. <em>Acta Neuropathol.</em><strong>136</strong>, 181–210 (2018).
</cite> [<a href="https://doi.org/10.1007/s00401-018-1879-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6060790/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29967940/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Capper,%20D.%20et%20al.%20Practical%20implementation%20of%20DNA%20methylation%20and%20copy-number-based%20CNS%20tumor%20diagnostics:%20the%20Heidelberg%20experience.%20Acta%20Neuropathol.136,%20181%E2%80%93210%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Sturm, D. et al. Multiomic neuropathology improves diagnostic accuracy in pediatric neuro-oncology. <em>Nat. Med.</em><strong>29</strong>, 917–926 (2023).
</cite> [<a href="https://doi.org/10.1038/s41591-023-02255-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10115638/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36928815/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sturm,%20D.%20et%20al.%20Multiomic%20neuropathology%20improves%20diagnostic%20accuracy%20in%20pediatric%20neuro-oncology.%20Nat.%20Med.29,%20917%E2%80%93926%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Pickles, J. C. et al. DNA methylation-based profiling for paediatric CNS tumour diagnosis and treatment: a population-based study. <em>Lancet Child Adolesc. Health</em><strong>4</strong>, 121–130 (2020).
</cite> [<a href="https://doi.org/10.1016/S2352-4642(19)30342-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31786093/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pickles,%20J.%20C.%20et%20al.%20DNA%20methylation-based%20profiling%20for%20paediatric%20CNS%20tumour%20diagnosis%20and%20treatment:%20a%20population-based%20study.%20Lancet%20Child%20Adolesc.%20Health4,%20121%E2%80%93130%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Jurmeister, P. et al. DNA methylation-based classification of sinonasal tumors. <em>Nat. Commun.</em><strong>13</strong>, 7148 (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-022-34815-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9705411/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36443295/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jurmeister,%20P.%20et%20al.%20DNA%20methylation-based%20classification%20of%20sinonasal%20tumors.%20Nat.%20Commun.13,%207148%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Koelsche, C. et al. Sarcoma classification by DNA methylation profiling. <em>Nat. Commun.</em><strong>12</strong>, 498 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-020-20603-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7819999/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33479225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koelsche,%20C.%20et%20al.%20Sarcoma%20classification%20by%20DNA%20methylation%20profiling.%20Nat.%20Commun.12,%20498%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Sahm, F. et al. DNA methylation-based classification and grading system for meningioma: a multicentre, retrospective analysis. <em>Lancet Oncol.</em><strong>18</strong>, 682–694 (2017).
</cite> [<a href="https://doi.org/10.1016/S1470-2045(17)30155-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28314689/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sahm,%20F.%20et%20al.%20DNA%20methylation-based%20classification%20and%20grading%20system%20for%20meningioma:%20a%20multicentre,%20retrospective%20analysis.%20Lancet%20Oncol.18,%20682%E2%80%93694%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Dragomir, M. P. et al. DNA methylation-based classifier differentiates intrahepatic pancreato-biliary tumours. <em>EBioMedicine</em><strong>93</strong>, 104657 (2023).
</cite> [<a href="https://doi.org/10.1016/j.ebiom.2023.104657" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10333440/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37348162/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dragomir,%20M.%20P.%20et%20al.%20DNA%20methylation-based%20classifier%20differentiates%20intrahepatic%20pancreato-biliary%20tumours.%20EBioMedicine93,%20104657%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Koelsche, C. &amp; von Deimling, A. Methylation classifiers: brain tumors, sarcomas, and what’s next. <em>Genes Chromosomes Cancer</em><strong>61</strong>, 346–355 (2022).
</cite> [<a href="https://doi.org/10.1002/gcc.23041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35388566/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koelsche,%20C.%20&amp;%20von%20Deimling,%20A.%20Methylation%20classifiers:%20brain%20tumors,%20sarcomas,%20and%20what%E2%80%99s%20next.%20Genes%20Chromosomes%20Cancer61,%20346%E2%80%93355%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Mattei, A. L., Bailly, N. &amp; Meissner, A. DNA methylation: a historical perspective. <em>Trends Genet.</em><strong>38</strong>, 676–707 (2022).
</cite> [<a href="https://doi.org/10.1016/j.tig.2022.03.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35504755/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mattei,%20A.%20L.,%20Bailly,%20N.%20&amp;%20Meissner,%20A.%20DNA%20methylation:%20a%20historical%20perspective.%20Trends%20Genet.38,%20676%E2%80%93707%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Li, E. &amp; Zhang, Y. DNA methylation in mammals. <em>Cold Spring Harb. Perspect. Biol.</em><strong>6</strong>, a019133 (2014).
</cite> [<a href="https://doi.org/10.1101/cshperspect.a019133" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3996472/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24789823/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20E.%20&amp;%20Zhang,%20Y.%20DNA%20methylation%20in%20mammals.%20Cold%20Spring%20Harb.%20Perspect.%20Biol.6,%20a019133%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Greenberg, M. V. C. &amp; Bourc’his, D. The diverse roles of DNA methylation in mammalian development and disease. <em>Nat. Rev. Mol. Cell Biol.</em><strong>20</strong>, 590–607 (2019).
</cite> [<a href="https://doi.org/10.1038/s41580-019-0159-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31399642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Greenberg,%20M.%20V.%20C.%20&amp;%20Bourc%E2%80%99his,%20D.%20The%20diverse%20roles%20of%20DNA%20methylation%20in%20mammalian%20development%20and%20disease.%20Nat.%20Rev.%20Mol.%20Cell%20Biol.20,%20590%E2%80%93607%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Smith, K. S. et al. Unified rhombic lip origins of group 3 and group 4 medulloblastoma. <em>Nature</em><strong>609</strong>, 1012–1020 (2022).
</cite> [<a href="https://doi.org/10.1038/s41586-022-05208-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9748853/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36131015/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smith,%20K.%20S.%20et%20al.%20Unified%20rhombic%20lip%20origins%20of%20group%203%20and%20group%204%20medulloblastoma.%20Nature609,%201012%E2%80%931020%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Hovestadt, V. et al. Decoding the regulatory landscape of medulloblastoma using DNA methylation sequencing. <em>Nature</em><strong>510</strong>, 537–541 (2014).
</cite> [<a href="https://doi.org/10.1038/nature13268" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24847876/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hovestadt,%20V.%20et%20al.%20Decoding%20the%20regulatory%20landscape%20of%20medulloblastoma%20using%20DNA%20methylation%20sequencing.%20Nature510,%20537%E2%80%93541%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Klauschen, F. et al. Toward explainable artificial intelligence for precision pathology. <em>Annu. Rev. Pathol.</em><strong>19</strong>, 541–570 (2024).
</cite> [<a href="https://doi.org/10.1146/annurev-pathmechdis-051222-113147" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37871132/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Klauschen,%20F.%20et%20al.%20Toward%20explainable%20artificial%20intelligence%20for%20precision%20pathology.%20Annu.%20Rev.%20Pathol.19,%20541%E2%80%93570%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Azodi, C. B., Tang, J. &amp; Shiu, S.-H. Opening the Black Box: interpretable machine learning for geneticists. <em>Trends Genet.</em><strong>36</strong>, 442–455 (2020).
</cite> [<a href="https://doi.org/10.1016/j.tig.2020.03.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32396837/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Azodi,%20C.%20B.,%20Tang,%20J.%20&amp;%20Shiu,%20S.-H.%20Opening%20the%20Black%20Box:%20interpretable%20machine%20learning%20for%20geneticists.%20Trends%20Genet.36,%20442%E2%80%93455%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Nussberger, A.-M., Luo, L., Celis, L. E. &amp; Crockett, M. J. Public attitudes value interpretability but prioritize accuracy in Artificial Intelligence. <em>Nat. Commun.</em><strong>13</strong>, 5821 (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-022-33417-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9528860/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36192416/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nussberger,%20A.-M.,%20Luo,%20L.,%20Celis,%20L.%20E.%20&amp;%20Crockett,%20M.%20J.%20Public%20attitudes%20value%20interpretability%20but%20prioritize%20accuracy%20in%20Artificial%20Intelligence.%20Nat.%20Commun.13,%205821%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Touw, W. G. et al. Data mining in the Life Sciences with Random Forest: a walk in the park or lost in the jungle? <em>Brief. Bioinform.</em><strong>14</strong>, 315–326 (2013).
</cite> [<a href="https://doi.org/10.1093/bib/bbs034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3659301/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22786785/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Touw,%20W.%20G.%20et%20al.%20Data%20mining%20in%20the%20Life%20Sciences%20with%20Random%20Forest:%20a%20walk%20in%20the%20park%20or%20lost%20in%20the%20jungle?%20Brief.%20Bioinform.14,%20315%E2%80%93326%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Chen, X. &amp; Ishwaran, H. Random forests for genomic data analysis. <em>Genomics</em><strong>99</strong>, 323–329 (2012).
</cite> [<a href="https://doi.org/10.1016/j.ygeno.2012.04.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3387489/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22546560/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20X.%20&amp;%20Ishwaran,%20H.%20Random%20forests%20for%20genomic%20data%20analysis.%20Genomics99,%20323%E2%80%93329%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Maros, M. E. et al. Machine learning workflows to estimate class probabilities for precision cancer diagnostics on DNA methylation microarray data. <em>Nat. Protoc.</em><strong>15</strong>, 479–512 (2020).
</cite> [<a href="https://doi.org/10.1038/s41596-019-0251-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31932775/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Maros,%20M.%20E.%20et%20al.%20Machine%20learning%20workflows%20to%20estimate%20class%20probabilities%20for%20precision%20cancer%20diagnostics%20on%20DNA%20methylation%20microarray%20data.%20Nat.%20Protoc.15,%20479%E2%80%93512%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Breiman, L. Random Forests. <em>Mach. Learn.</em><strong>45</strong>, 5–32 (2001).</cite> [<a href="https://scholar.google.com/scholar_lookup?Breiman,%20L.%20Random%20Forests.%20Mach.%20Learn.45,%205%E2%80%9332%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Díaz-Uriarte, R. &amp; Alvarez de Andrés, S. Gene selection and classification of microarray data using random forest. <em>BMC Bioinforma.</em><strong>7</strong>, 3 (2006).</cite> [<a href="https://doi.org/10.1186/1471-2105-7-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1363357/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16398926/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?D%C3%ADaz-Uriarte,%20R.%20&amp;%20Alvarez%20de%20Andr%C3%A9s,%20S.%20Gene%20selection%20and%20classification%20of%20microarray%20data%20using%20random%20forest.%20BMC%20Bioinforma.7,%203%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Petralia, F., Wang, P., Yang, J. &amp; Tu, Z. Integrative random forest for gene regulatory network inference. <em>Bioinformatics</em><strong>31</strong>, i197–i205 (2015).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv268" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4542785/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26072483/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Petralia,%20F.,%20Wang,%20P.,%20Yang,%20J.%20&amp;%20Tu,%20Z.%20Integrative%20random%20forest%20for%20gene%20regulatory%20network%20inference.%20Bioinformatics31,%20i197%E2%80%93i205%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Basu, S., Kumbier, K., Brown, J. B. &amp; Yu, B. Iterative random forests to discover predictive and stable high-order interactions. <em>Proc. Natl Acad. Sci. USA</em><strong>115</strong>, 1943–1948 (2018).
</cite> [<a href="https://doi.org/10.1073/pnas.1711236115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5828575/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29351989/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Basu,%20S.,%20Kumbier,%20K.,%20Brown,%20J.%20B.%20&amp;%20Yu,%20B.%20Iterative%20random%20forests%20to%20discover%20predictive%20and%20stable%20high-order%20interactions.%20Proc.%20Natl%20Acad.%20Sci.%20USA115,%201943%E2%80%931948%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Benfatto, S. et al. Uncovering cancer vulnerabilities by machine learning prediction of synthetic lethality. <em>Mol. Cancer</em><strong>20</strong>, 111 (2021).
</cite> [<a href="https://doi.org/10.1186/s12943-021-01405-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8401190/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34454516/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Benfatto,%20S.%20et%20al.%20Uncovering%20cancer%20vulnerabilities%20by%20machine%20learning%20prediction%20of%20synthetic%20lethality.%20Mol.%20Cancer20,%20111%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Lundberg, S. &amp; Lee, S.-I. <em>A Unified Approach to Interpreting Model Predictions</em>. (2017).</cite>
</li>
<li id="CR36">
<span class="label">36.</span><cite>Turcan, S. et al. IDH1 mutation is sufficient to establish the glioma hypermethylator phenotype. <em>Nature</em><strong>483</strong>, 479–483 (2012).
</cite> [<a href="https://doi.org/10.1038/nature10866" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3351699/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22343889/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Turcan,%20S.%20et%20al.%20IDH1%20mutation%20is%20sufficient%20to%20establish%20the%20glioma%20hypermethylator%20phenotype.%20Nature483,%20479%E2%80%93483%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Noushmehr, H. et al. Identification of a CpG island methylator phenotype that defines a distinct subgroup of glioma. <em>Cancer Cell</em><strong>17</strong>, 510–522 (2010).
</cite> [<a href="https://doi.org/10.1016/j.ccr.2010.03.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2872684/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20399149/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Noushmehr,%20H.%20et%20al.%20Identification%20of%20a%20CpG%20island%20methylator%20phenotype%20that%20defines%20a%20distinct%20subgroup%20of%20glioma.%20Cancer%20Cell17,%20510%E2%80%93522%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Zhou, W. et al. DNA methylation loss in late-replicating domains is linked to mitotic cell division. <em>Nat. Genet.</em><strong>50</strong>, 591–602 (2018).
</cite> [<a href="https://doi.org/10.1038/s41588-018-0073-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5893360/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29610480/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou,%20W.%20et%20al.%20DNA%20methylation%20loss%20in%20late-replicating%20domains%20is%20linked%20to%20mitotic%20cell%20division.%20Nat.%20Genet.50,%20591%E2%80%93602%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Johann, P. D. et al. Atypical teratoid/rhabdoid tumors are comprised of three epigenetic subgroups with distinct enhancer landscapes. <em>Cancer Cell</em><strong>29</strong>, 379–393 (2016).
</cite> [<a href="https://doi.org/10.1016/j.ccell.2016.02.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26923874/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Johann,%20P.%20D.%20et%20al.%20Atypical%20teratoid/rhabdoid%20tumors%20are%20comprised%20of%20three%20epigenetic%20subgroups%20with%20distinct%20enhancer%20landscapes.%20Cancer%20Cell29,%20379%E2%80%93393%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Lambo, S. et al. The molecular landscape of ETMR at diagnosis and relapse. <em>Nature</em><strong>576</strong>, 274–280 (2019).
</cite> [<a href="https://doi.org/10.1038/s41586-019-1815-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6908757/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31802000/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lambo,%20S.%20et%20al.%20The%20molecular%20landscape%20of%20ETMR%20at%20diagnosis%20and%20relapse.%20Nature576,%20274%E2%80%93280%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Lambo, S., von Hoff, K., Korshunov, A., Pfister, S. M. &amp; Kool, M. ETMR: a tumor entity in its infancy. <em>Acta Neuropathol.</em><strong>140</strong>, 249–266 (2020).
</cite> [<a href="https://doi.org/10.1007/s00401-020-02182-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7423804/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32601913/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lambo,%20S.,%20von%20Hoff,%20K.,%20Korshunov,%20A.,%20Pfister,%20S.%20M.%20&amp;%20Kool,%20M.%20ETMR:%20a%20tumor%20entity%20in%20its%20infancy.%20Acta%20Neuropathol.140,%20249%E2%80%93266%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Sood, R. et al. Cloning and characterization of a novel gene, SHPRH, encoding a conserved putative protein with SNF2/helicase and PHD-finger domains from the 6q24 region. <em>Genomics</em><strong>82</strong>, 153–161 (2003).
</cite> [<a href="https://doi.org/10.1016/s0888-7543(03)00121-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12837266/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sood,%20R.%20et%20al.%20Cloning%20and%20characterization%20of%20a%20novel%20gene,%20SHPRH,%20encoding%20a%20conserved%20putative%20protein%20with%20SNF2/helicase%20and%20PHD-finger%20domains%20from%20the%206q24%20region.%20Genomics82,%20153%E2%80%93161%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Motegi, A. et al. Human SHPRH suppresses genomic instability through proliferating cell nuclear antigen polyubiquitination. <em>J. Cell Biol.</em><strong>175</strong>, 703–708 (2006).
</cite> [<a href="https://doi.org/10.1083/jcb.200606145" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2064669/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17130289/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Motegi,%20A.%20et%20al.%20Human%20SHPRH%20suppresses%20genomic%20instability%20through%20proliferating%20cell%20nuclear%20antigen%20polyubiquitination.%20J.%20Cell%20Biol.175,%20703%E2%80%93708%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Gan, W. et al. R-loop-mediated genomic instability is caused by impairment of replication fork progression. <em>Genes Dev.</em><strong>25</strong>, 2041–2056 (2011).
</cite> [<a href="https://doi.org/10.1101/gad.17010011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3197203/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21979917/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gan,%20W.%20et%20al.%20R-loop-mediated%20genomic%20instability%20is%20caused%20by%20impairment%20of%20replication%20fork%20progression.%20Genes%20Dev.25,%202041%E2%80%932056%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Motegi, A. et al. Polyubiquitination of proliferating cell nuclear antigen by HLTF and SHPRH prevents genomic instability from stalled replication forks. <em>Proc. Natl Acad. Sci. USA</em><strong>105</strong>, 12411–12416 (2008).
</cite> [<a href="https://doi.org/10.1073/pnas.0805685105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2518831/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18719106/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Motegi,%20A.%20et%20al.%20Polyubiquitination%20of%20proliferating%20cell%20nuclear%20antigen%20by%20HLTF%20and%20SHPRH%20prevents%20genomic%20instability%20from%20stalled%20replication%20forks.%20Proc.%20Natl%20Acad.%20Sci.%20USA105,%2012411%E2%80%9312416%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Huen, M. S. Y. et al. Regulation of chromatin architecture by the PWWP domain-containing DNA damage-responsive factor EXPAND1/MUM1. <em>Mol. Cell</em><strong>37</strong>, 854–864 (2010).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2009.12.040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3695488/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20347427/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huen,%20M.%20S.%20Y.%20et%20al.%20Regulation%20of%20chromatin%20architecture%20by%20the%20PWWP%20domain-containing%20DNA%20damage-responsive%20factor%20EXPAND1/MUM1.%20Mol.%20Cell37,%20854%E2%80%93864%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Lloyd, R. V., Osamura, R. Y., Klöppel, G. &amp; Rosai, J. <em>WHO Classification of Tumours of Endocrine Organs</em>. (WHO Classification of Tumours, 2017).</cite>
</li>
<li id="CR48">
<span class="label">48.</span><cite>Metherell, L. A. et al. TPIT mutations are associated with early-onset, but not late-onset isolated ACTH deficiency. <em>Eur. J. Endocrinol.</em>10.1530/eje.0.1510463 (2004).</cite> [<a href="https://doi.org/10.1530/eje.0.1510463" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15476446/" class="usa-link">PubMed</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Oh, J. Y. et al. Transcriptomic profiles of normal pituitary cells and pituitary neuroendocrine tumor cells. <em>Cancers</em><strong>15</strong>, 110 (2022).
</cite> [<a href="https://doi.org/10.3390/cancers15010110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9817686/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36612109/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Oh,%20J.%20Y.%20et%20al.%20Transcriptomic%20profiles%20of%20normal%20pituitary%20cells%20and%20pituitary%20neuroendocrine%20tumor%20cells.%20Cancers15,%20110%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Sturm, D. et al. New brain tumor entities emerge from molecular classification of CNS-PNETs. <em>Cell</em><strong>164</strong>, 1060–1072 (2016).
</cite> [<a href="https://doi.org/10.1016/j.cell.2016.01.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5139621/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26919435/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sturm,%20D.%20et%20al.%20New%20brain%20tumor%20entities%20emerge%20from%20molecular%20classification%20of%20CNS-PNETs.%20Cell164,%201060%E2%80%931072%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Takahashi, M., Ritz, J. &amp; Cooper, G. M. Activation of a novel human transforming gene, ret, by DNA rearrangement. <em>Cell</em><strong>42</strong>, 581–588 (1985).
</cite> [<a href="https://doi.org/10.1016/0092-8674(85)90115-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2992805/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Takahashi,%20M.,%20Ritz,%20J.%20&amp;%20Cooper,%20G.%20M.%20Activation%20of%20a%20novel%20human%20transforming%20gene,%20ret,%20by%20DNA%20rearrangement.%20Cell42,%20581%E2%80%93588%20(1985)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Takahashi, M. et al. Cloning and expression of the ret proto-oncogene encoding a tyrosine kinase with two potential transmembrane domains. <em>Oncogene</em><strong>3</strong>, 571–578 (1988).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/3078962/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Takahashi,%20M.%20et%20al.%20Cloning%20and%20expression%20of%20the%20ret%20proto-oncogene%20encoding%20a%20tyrosine%20kinase%20with%20two%20potential%20transmembrane%20domains.%20Oncogene3,%20571%E2%80%93578%20(1988)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Treanor, J. J. et al. Characterization of a multicomponent receptor for GDNF. <em>Nature</em><strong>382</strong>, 80–83 (1996).
</cite> [<a href="https://doi.org/10.1038/382080a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8657309/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Treanor,%20J.%20J.%20et%20al.%20Characterization%20of%20a%20multicomponent%20receptor%20for%20GDNF.%20Nature382,%2080%E2%80%9383%20(1996)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Trupp, M. et al. Functional receptor for GDNF encoded by the c-ret proto-oncogene. <em>Nature</em><strong>381</strong>, 785–789 (1996).
</cite> [<a href="https://doi.org/10.1038/381785a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8657281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Trupp,%20M.%20et%20al.%20Functional%20receptor%20for%20GDNF%20encoded%20by%20the%20c-ret%20proto-oncogene.%20Nature381,%20785%E2%80%93789%20(1996)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Mehta, G. U. et al. US Food and Drug Administration regulatory updates in neuro-oncology. <em>J. Neurooncol.</em><strong>153</strong>, 375–381 (2021).
</cite> [<a href="https://doi.org/10.1007/s11060-021-03789-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8218275/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34156585/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mehta,%20G.%20U.%20et%20al.%20US%20Food%20and%20Drug%20Administration%20regulatory%20updates%20in%20neuro-oncology.%20J.%20Neurooncol.153,%20375%E2%80%93381%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Kuschel, L. P. et al. Robust methylation-based classification of brain tumours using nanopore sequencing. <em>Neuropathol. Appl. Neurobiol</em>. <strong>49</strong>, e12856 (2022).</cite> [<a href="https://doi.org/10.1111/nan.12856" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36269599/" class="usa-link">PubMed</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Patel, A. et al. Rapid-CNS: rapid comprehensive adaptive nanopore-sequencing of CNS tumors, a proof-of-concept study. <em>Acta Neuropathol.</em><strong>143</strong>, 609–612 (2022).
</cite> [<a href="https://doi.org/10.1007/s00401-022-02415-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9038836/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35357562/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Patel,%20A.%20et%20al.%20Rapid-CNS:%20rapid%20comprehensive%20adaptive%20nanopore-sequencing%20of%20CNS%20tumors,%20a%20proof-of-concept%20study.%20Acta%20Neuropathol.143,%20609%E2%80%93612%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Vermeulen, C. et al. Ultra-fast deep-learned CNS tumour classification during surgery. <em>Nature</em><strong>622</strong>, 842–849 (2023).
</cite> [<a href="https://doi.org/10.1038/s41586-023-06615-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10600004/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37821699/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vermeulen,%20C.%20et%20al.%20Ultra-fast%20deep-learned%20CNS%20tumour%20classification%20during%20surgery.%20Nature622,%20842%E2%80%93849%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Djirackor, L. et al. Intraoperative DNA methylation classification of brain tumors impacts neurosurgical strategy. <em>Neurooncol. Adv.</em><strong>3</strong>, vdab149 (2021).
</cite> [<a href="https://doi.org/10.1093/noajnl/vdab149" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8557693/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34729487/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Djirackor,%20L.%20et%20al.%20Intraoperative%20DNA%20methylation%20classification%20of%20brain%20tumors%20impacts%20neurosurgical%20strategy.%20Neurooncol.%20Adv.3,%20vdab149%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Euskirchen, P. et al. Same-day genomic and epigenomic diagnosis of brain tumors using real-time nanopore sequencing. <em>Acta Neuropathol.</em><strong>134</strong>, 691–703 (2017).
</cite> [<a href="https://doi.org/10.1007/s00401-017-1743-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5645447/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28638988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Euskirchen,%20P.%20et%20al.%20Same-day%20genomic%20and%20epigenomic%20diagnosis%20of%20brain%20tumors%20using%20real-time%20nanopore%20sequencing.%20Acta%20Neuropathol.134,%20691%E2%80%93703%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Liu, A. P. Y. et al. Serial assessment of measurable residual disease in medulloblastoma liquid biopsies. <em>Cancer Cell</em><strong>39</strong>, 1519–1530.e4 (2021).
</cite> [<a href="https://doi.org/10.1016/j.ccell.2021.09.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9620970/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34678152/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20A.%20P.%20Y.%20et%20al.%20Serial%20assessment%20of%20measurable%20residual%20disease%20in%20medulloblastoma%20liquid%20biopsies.%20Cancer%20Cell39,%201519%E2%80%931530.e4%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/11842776/bin/41467_2025_57078_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/11842776/bin/41467_2025_57078_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (1.9MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/11842776/bin/41467_2025_57078_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (1.8MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/11842776/bin/41467_2025_57078_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (1.2MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>No new datasets have been generated as part of this study. The DNA methylation reference dataset and the gene expression dataset are accessible from GEO (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE109381" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE109381</a> [<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE109381" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE109381</a>]) and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73038" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE73038</a>, respectively). <a href="#Sec17" class="usa-link">Source data</a> are provided with this paper.</p>
<p>The code to calculate probe usages from the RF model and perform subsequent analyses is available at: <a href="https://github.com/hovestadt/shinyMNP" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/hovestadt/shinyMNP</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-57078-0"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_57078.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/11842776/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/11842776/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11842776%2F%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11842776/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC11842776/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC11842776/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/39979307/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC11842776/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/39979307/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC11842776/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/11842776/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="Bsn7wJQI7qhnDEtN9FJYJxfxCTThT60kIF4Zybs2oYNqOeg36IrcRbjzdok9L2bM">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-B7-tgZZA.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-BLbQGH8u.js"></script>
    
    

    </body>
</html>
